<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09723">cd09723</a></td><td>132</td><td>113</td><td>99.2</td><td>1.1E-10</td><td>[&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as csx13 family</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09741">cd09741</a></td><td>219</td><td>111</td><td>99.1</td><td>3.5E-10</td><td>[&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as NE0113 family</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09623">pfam09623</a></td><td>225</td><td>131</td><td>99.0</td><td>2.2E-09</td><td>[&nbsp;---------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_NE0113</td><td>CRISPR-associated protein NE0113 (Cas_NE0113). Members of this minor CRISPR-associated (Cas) protein family are encoded in cas gene clusters in Vibrio vulnificus YJ016, Nitrosomonas europaea ATCC 19718, Mannheimia succiniciproducens MBEL55E, and Verrucomicrobium spinosum.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03642">TIGR03642</a></td><td>124</td><td>108</td><td>99.0</td><td>8.7E-09</td><td>[&nbsp;&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_csx14</td><td>CRISPR-associated protein, Csx14 family. This model describes a protein N-terminal protein sequence domain strictly associated with CRISPR and CRISPR-associated protein systems. This model and TIGR02584 identify two separate clades from a larger homology domain family, both CRISPR-associated, while other homologs are found that may not be. Members are found in bacteria that include Pelotomaculum thermopropionicum SI, Thermoanaerobacter tengcongensis MB4, and Roseiflexus sp. RS-1, and in archaea that include Thermoplasma volcanium, Picrophilus torridus, and Methanospirillum hungatei. The molecular function is unknown.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09747">cd09747</a></td><td>378</td><td>87</td><td>95.7</td><td>0.02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as Cas02710 family</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09670">pfam09670</a></td><td>379</td><td>87</td><td>95.1</td><td>0.025</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_Cas02710</td><td>CRISPR-associated protein (Cas_Cas02710). Members of this family are found, exclusively in the vicinity of CRISPR repeats and other CRISPR-associated (cas) genes, in Methanothermobacter thermautotrophicus (Methanobacterium thermoformicicum), Thermus thermophilus (Deinococcus-Thermus), Chloroflexus aurantiacus (Chloroflexi), and Thermomicrobium roseum (Thermomicrobia).</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09732">cd09732</a></td><td>221</td><td>107</td><td>94.1</td><td>0.18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as TM1812 family</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09455">pfam09455</a></td><td>372</td><td>73</td><td>92.5</td><td>0.37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_DxTHG</td><td>CRISPR-associated (Cas) DxTHG family. CRISPR is a term for Clustered Regularly Interspaced Short Palidromic Repeats. A number of protein families appear only in association with these repeats and are designated Cas (CRISPR associated) proteins. The family describes Cas proteins of about 400 residues that include the motif </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13580">pfam13580</a></td><td>138</td><td>41</td><td>88.6</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SIS_2</td><td>SIS domain. SIS (Sugar ISomerase) domains are found in many phosphosugar isomerases and phosphosugar binding proteins. SIS domains are also found in proteins that regulate the expression of genes involved in synthesis of phosphosugars.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09742">cd09742</a></td><td>183</td><td>89</td><td>86.9</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csm6_III-A</td><td>CRISPR/Cas system-associated protein Csm6. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; loosely associated with CRISPR/Cas systems; also known as APE2256 family</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07201">PRK07201</a></td><td>657</td><td>53</td><td>82.4</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07201</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09651">pfam09651</a></td><td>136</td><td>83</td><td>81.9</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_APE2256</td><td>CRISPR-associated protein (Cas_APE2256). This entry represents a conserved region of about 150 amino acids found in at least five archaeal and three bacterial species. These species all contain CRISPRs (Clustered Regularly Interspaced Short Palindromic Repeats). In six of eight species, the protein is encoded the vicinity of a CRISPR/Cas locus.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09702">cd09702</a></td><td>378</td><td>57</td><td>81.6</td><td>0.62</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as TIGR02710 family</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02710">TIGR02710</a></td><td>380</td><td>42</td><td>81.5</td><td>0.89</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02710</td><td>CRISPR-associated protein, TIGR02710 family. Members of this family are found, exclusively in the vicinity of CRISPR repeats and other CRISPR-associated (cas) genes, in Methanothermobacter thermautotrophicus (Archaea), Thermus thermophilus (Deinococcus-Thermus), Chloroflexus aurantiacus (Chloroflexi), and Thermomicrobium roseum (Thermomicrobia).</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05276">cd05276</a></td><td>323</td><td>59</td><td>76.6</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>p53_inducible_oxidoreductase</td><td>PIG3 p53-inducible quinone oxidoreductase. PIG3 p53-inducible quinone oxidoreductase, a medium chain dehydrogenase/reductase family member, acts in the apoptotic pathway. PIG3 reduces ortho-quinones, but its apoptotic activity has been attributed to oxidative stress generation, since overexpression of PIG3 accumulates reactive oxygen species. PIG3 resembles the MDR family member quinone reductases, which catalyze the reduction of quinone to hydroxyquinone. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes or ketones.  Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation.  ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form.  The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide.  A GxGxxG motif after the first mononucleotide contact half allows the close contact of the coenzyme with the ADH backbone.  The N-terminal catalytic domain has a distant homology  to GroES.  These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site, and a structural zinc in a lobe of the catalytic domain.  NAD(H) binding occurs in the cleft between the catalytic and coenzyme-binding domains at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding. In human ADH catalysis, the zinc ion helps coordinate the alcohol, followed by deprotonation of  a histidine, the ribose of NAD, a serine, then the alcohol, which allows the transfer of a hydride to NAD+, creating NADH and a zinc-bound aldehyde or ketone. In yeast and some bacteria, the active site zinc binds an aldehyde, polarizing it, and leading to the reverse reaction.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14078">pfam14078</a></td><td>127</td><td>77</td><td>76.6</td><td>2.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;]</td><td>DUF4259</td><td>Domain of unknown function (DUF4259). This family of proteins is functionally uncharacterized. This family of proteins is found in bacteria and eukaryotes. Proteins in this family are typically between 118 and 145 amino acids in length.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04882">cd04882</a></td><td>65</td><td>46</td><td>75.0</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ACT_Bt0572_2</td><td>C-terminal ACT domain of a novel protein composed of just two ACT domains. Included in this CD is the C-terminal ACT domain of a novel protein composed of just two ACT domains, as seen in the yet uncharacterized structure (pdb 2F06) of the Bt0572 protein from Bacteroides thetaiotaomicron and related proteins. Members of this CD belong to the superfamily of ACT regulatory domains.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01884">TIGR01884</a></td><td>203</td><td>76</td><td>72.2</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_HTH</td><td>CRISPR locus-related DNA-binding protein. Most but not all examples of this family are associated with CRISPR loci, a combination of DNA repeats and characteristic proteins encoded near the repeat cluster. The C-terminal region of this protein is homologous to DNA-binding helix-turn-helix domains with predicted transcriptional regulatory activity. </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08419">PRK08419</a></td><td>298</td><td>75</td><td>71.6</td><td>1.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;]</td><td>PRK08419</td><td>lipid A biosynthesis lauroyl acyltransferase; Reviewed</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08290">cd08290</a></td><td>341</td><td>50</td><td>70.6</td><td>9.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ETR</td><td>2-enoyl thioester reductase (ETR). 2-enoyl thioester reductase (ETR) catalyzes the NADPH-dependent conversion of trans-2-enoyl acyl carrier protein/coenzyme A (ACP/CoA) to acyl-(ACP/CoA) in fatty acid synthesis. 2-enoyl thioester reductase activity has been linked in  Candida tropicalis as essential in maintaining mitiochondrial respiratory function. This ETR family is a part of the medium chain dehydrogenase/reductase family, but lack the zinc coordination sites characteristic of the alcohol dehydrogenases in this family. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.  Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation.   ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form. The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide.  The N-terminal catalytic domain has a distant homology to GroES. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site, and a structural zinc in a lobe of the catalytic domain.  NAD(H) binding occurs in the cleft between the catalytic and coenzyme-binding domains, at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding. Candida tropicalis enoyl thioester reductase (Etr1p) catalyzes the NADPH-dependent reduction of trans-2-enoyl thioesters in mitochondrial fatty acid synthesis. Etr1p forms homodimers, with each subunit containing a nucleotide-binding Rossmann fold domain and a catalytic domain.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04239">cd04239</a></td><td>229</td><td>60</td><td>69.2</td><td>6.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAK_UMPK-like</td><td>AAK_UMPK-like: UMP kinase (UMPK)-like, the microbial/chloroplast uridine monophosphate kinase (uridylate kinase) enzyme that catalyzes UMP phosphorylation and plays a key role in pyrimidine nucleotide biosynthesis. Regulation of this process is via feed-back control and via gene repression of carbamoyl phosphate synthetase (the first enzyme of the pyrimidine biosynthesis pathway). The UMP kinases of E. coli (Ec) and Pyrococcus furiosus (Pf) are known to function as homohexamers, with GTP and UTP being allosteric effectors. Like other related enzymes (carbamate kinase, aspartokinase, and N-acetylglutamate kinase) the E. coli and most bacterial UMPKs have a conserved, N-terminal, lysine residue proposed to function in the catalysis of the phosphoryl group transfer, whereas most archaeal UMPKs appear to lack this residue and the Pyrococcus furiosus structure has an additional Mg ion bound to the ATP molecule which is proposed to function as the catalysis instead. Also included in this CD are the alpha and beta subunits of the Mo storage protein (MosA and MosB) characterized as an alpha4-beta4 octamer containing an ATP-dependent, polynuclear molybdenum-oxide cluster. These and related  sequences in this CD are members of the Amino Acid Kinase Superfamily (AAK).</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02808">cd02808</a></td><td>392</td><td>35</td><td>68.8</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GltS_FMN</td><td>Glutamate synthase (GltS) FMN-binding domain.  GltS is a complex iron-sulfur flavoprotein that catalyzes the reductive synthesis of L-glutamate from 2-oxoglutarate and L-glutamine via intramolecular channelling of ammonia, a reaction in the plant, yeast and bacterial pathway for ammonia assimilation. It is a multifunctional enzyme that functions through three distinct active centers, carrying out  L-glutamine hydrolysis, conversion of 2-oxoglutarate into L-glutamate, and electron uptake from an electron donor.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05006">cd05006</a></td><td>177</td><td>42</td><td>65.0</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SIS_GmhA</td><td>Phosphoheptose isomerase is a member of the SIS (Sugar ISomerase) superfamily. Phosphoheptose isomerase catalyzes the isomerization of sedoheptulose 7-phosphate into D-glycero-D-mannoheptose 7-phosphate. This is the first step of the biosynthesis of gram-negative bacteria inner core lipopolysaccharide precursor, L-glycero-D-mannoheptose (Gmh).</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04123">cd04123</a></td><td>162</td><td>39</td><td>63.1</td><td>8.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rab21</td><td>Rab GTPase family 21 (Rab21). The localization and function of Rab21 are not clearly defined, with conflicting data reported. Rab21 has been reported to localize in the ER in human intestinal epithelial cells, with partial colocalization with alpha-glucosidase, a late endosomal/lysosomal marker. More recently, Rab21 was shown to colocalize with and affect the morphology of early endosomes. In Dictyostelium, GTP-bound Rab21, together with two novel LIM domain proteins, LimF and ChLim, has been shown to regulate phagocytosis. GTPase activating proteins (GAPs) interact with GTP-bound Rab and accelerate the hydrolysis of GTP to GDP. Guanine nucleotide exchange factors (GEFs) interact with GDP-bound Rabs to promote the formation of the GTP-bound state. Rabs are further regulated by guanine nucleotide dissociation inhibitors (GDIs), which facilitate Rab recycling by masking C-terminal lipid binding and promoting cytosolic localization. Most Rab GTPases contain a lipid modification site at the C-terminus, with sequence motifs CC, CXC, or CCX. Lipid binding is essential for membrane attachment, a key feature of most Rab proteins. Due to the presence of truncated sequences in this CD, the lipid modification site is not available for annotation.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05005">cd05005</a></td><td>179</td><td>41</td><td>62.1</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SIS_PHI</td><td>Hexulose-6-phosphate isomerase (PHI). PHI is a member of the SIS (Sugar ISomerase domain) superfamily. In the ribulose monophosphate pathway of formaldehyde fixation, hexulose-6-phosphate synthase catalyzes the condensation of ribulose-5-phosphate with formadelhyde to become hexulose-6-phosphate, which is then isomerized to fructose-6-phosphate by PHI.</td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13766">PRK13766</a></td><td>773</td><td>105</td><td>61.7</td><td>31</td><td>[&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13766</td><td>Hef nuclease; Provisional</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05530">cd05530</a></td><td>372</td><td>57</td><td>60.7</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>POLBc_B1</td><td>DNA polymerase type-B B1 subfamily catalytic domain. Archaeal proteins that are involved in DNA replication are similar to those from eukaryotes. Some archaeal members also possess multiple family B DNA polymerases (B1, B2 and B3). So far there is no specific function(s) has been assigned for different members of the archaea type B DNA polymerases. Phylogenetic analyses of eubacterial, archaeal, and eukaryotic family B DNA polymerases are support independent gene duplications during the evolution of archaeal and eukaryotic family B DNA polymerases.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06523">PRK06523</a></td><td>260</td><td>35</td><td>60.5</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06523</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03168">TIGR03168</a></td><td>303</td><td>94</td><td>60.3</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------]</td><td>1-PFK</td><td>hexose kinase, 1-phosphofructokinase family. This family consists largely of 1-phosphofructokinases, but also includes tagatose-6-kinases and 6-phosphofructokinases.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09655">cd09655</a></td><td>198</td><td>75</td><td>56.9</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CasRa_I-A</td><td>CRISPR/Cas system-associated transcriptional regulator CasRa. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Predicted transcriptional regulator of CRISPR/Cas system</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1611">COG1611</a></td><td>205</td><td>50</td><td>56.7</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YgdH</td><td>Predicted Rossmann fold nucleotide-binding protein </td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06125">PRK06125</a></td><td>259</td><td>52</td><td>55.8</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06125</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05195">cd05195</a></td><td>293</td><td>62</td><td>55.7</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>enoyl_red</td><td>enoyl reductase of polyketide synthase. Putative enoyl reductase of polyketide synthase. Polyketide synthases produce polyketides in step by step mechanism that is similar to fatty acid synthesis. Enoyl reductase reduces a double to single bond. Erythromycin is one example of a polyketide generated by 3 complex enzymes (megasynthases). 2-enoyl thioester reductase (ETR) catalyzes the NADPH-dependent dependent conversion of trans-2-enoyl acyl carrier protein/coenzyme A (ACP/CoA) to acyl-(ACP/CoA) in fatty acid synthesis. 2-enoyl thioester reductase activity has been linked in  Candida tropicalis as essential in maintaining mitiochondrial respiratory function. This ETR family is a part of the medium chain dehydrogenase/reductase family, but lack the zinc coordination sites characteristic of the alcohol dehydrogenases in this family. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes or ketones. Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation.   ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form.  The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide. The N-terminal catalytic domain has a distant homology  to GroES. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site, and a structural zinc in a lobe of the catalytic domain. NAD(H) binding occurs in the cleft between the catalytic  and coenzyme-binding domains, at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09002">pfam09002</a></td><td>379</td><td>94</td><td>52.9</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1887</td><td>Domain of unknown function (DUF1887). This domain is found in a set of hypothetical bacterial proteins.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02824">TIGR02824</a></td><td>325</td><td>110</td><td>52.4</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>quinone_pig3</td><td>putative NAD(P)H quinone oxidoreductase, PIG3 family. Members of this family are putative quinone oxidoreductases that belong to the broader superfamily (modeled by Pfam pfam00107) of zinc-dependent alcohol (of medium chain length) dehydrogenases and quinone oxiooreductases. The alignment shows no motif of conserved Cys residues as are found in zinc-binding members of the superfamily, and members are likely to be quinone oxidoreductases instead. A member of this family in Homo sapiens, PIG3, is induced by p53 but is otherwise uncharacterized. </td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03152">pfam03152</a></td><td>176</td><td>40</td><td>51.8</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UFD1</td><td>Ubiquitin fusion degradation protein UFD1. Post-translational ubiquitin-protein conjugates are recognized for degradation by the ubiquitin fusion degradation (UFD) pathway. Several proteins involved in this pathway have been identified. This family includes UFD1, a 40kD protein that is essential for vegetative cell viability. The human UFD1 gene is expressed at high levels during embryogenesis, especially in the eyes and in the inner ear primordia and is thought to be important in the determination of ectoderm-derived structures, including neural crest cells. In addition, this gene is deleted in the CATCH-22 (cardiac defects, abnormal facies, thymic hypoplasia, cleft palate and hypocalcaemia with deletions on chromosome 22) syndrome. This clinical syndrome is associated with a variety of developmental defects, all characterized by microdeletions on 22q11.2. Two such developmental defects are the DiGeorge syndrome OMIM:188400, and the velo-cardio- facial syndrome OMIM:145410. Several of the abnormalities associated with these conditions are thought to be due to defective neural crest cell differentiation.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09851">pfam09851</a></td><td>31</td><td>15</td><td>51.5</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SHOCT</td><td>Short C-terminal domain. </td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03910">TIGR03910</a></td><td>347</td><td>16</td><td>51.3</td><td>6.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pyrrolys_PylB</td><td>pyrrolysine biosynthesis radical SAM protein. This model describes a radical SAM protein, PylB, that is part of the three-gene cassette sufficient for the biosynthesis of pyrrolysine (the twenty-second amino acid) when expressed heterologously in E. coli. The pyrrolysine next is ligated to its own tRNA and incorporated at special UAG codons. </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08266">cd08266</a></td><td>342</td><td>56</td><td>50.9</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Zn_ADH_like1</td><td>Alcohol dehydrogenases of the MDR family. This group contains proteins related to the zinc-dependent  alcohol dehydrogenases. However, while the group has structural zinc site characteristic of these enzymes, it lacks the consensus site for a catalytic zinc. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.   Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation. ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form.  The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide. A GxGxxG motif after the first mononucleotide contact half allows the close contact of the coenzyme with the ADH backbone. The N-terminal catalytic domain has a distant homology to GroES. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria),  and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site, and a structural zinc in a lobe of the catalytic domain. NAD(H)-binding occurs in the cleft between the catalytic  and coenzyme-binding domains at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding. In human ADH catalysis, the zinc ion helps coordinate the alcohol, followed by deprotonation of  a histidine, the ribose of NAD, a serine, then the alcohol, which allows the transfer of a hydride to NAD+, creating NADH and a zinc-bound aldehyde or ketone. In yeast and some bacteria, the active site zinc binds an aldehyde, polarizing it, and leading to the reverse reaction.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0528">COG0528</a></td><td>238</td><td>41</td><td>50.9</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PyrH</td><td>Uridylate kinase </td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08268">cd08268</a></td><td>328</td><td>41</td><td>50.6</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MDR2</td><td>Medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family. This group is a member of the medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, but lacks the zinc-binding sites of the zinc-dependent alcohol dehydrogenases. The medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, which contains the zinc-dependent alcohol dehydrogenase (ADH-Zn) and related proteins, is a diverse group of proteins related to the first identified member, class I mammalian ADH.  MDRs display a broad range of activities and are distinguished from the smaller short chain dehydrogenases (~ 250 amino acids vs. the ~ 350 amino acids of the MDR).  The MDR proteins have 2 domains: a C-terminal NAD(P)-binding Rossmann fold domain of a beta-alpha form and an N-terminal catalytic domain with distant homology to GroES.  The MDR group contains a host of activities, including the founding alcohol dehydrogenase (ADH), quinone reductase, sorbitol dehydrogenase, formaldehyde dehydrogenase, butanediol DH, ketose reductase, cinnamyl reductase, and numerous others. The zinc-dependent alcohol dehydrogenases (ADHs) catalyze the  NAD(P)(H)-dependent interconversion of alcohols to aldehydes or ketones.  Active site zinc has a catalytic role, while structural zinc aids in stability.  ADH-like proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and generally have 2 tightly bound zinc atoms per subunit. The active site zinc is coordinated by a histidine, two cysteines, and a water molecule. The second zinc seems to play a structural role, affects subunit interactions, and is typically coordinated by 4 cysteines.</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04254">cd04254</a></td><td>231</td><td>54</td><td>49.9</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAK_UMPK-PyrH-Ec</td><td>UMP kinase (UMPK)-Ec, the microbial/chloroplast uridine monophosphate kinase (uridylate kinase) enzyme that catalyzes UMP phosphorylation and plays a key role in pyrimidine nucleotide biosynthesis; regulation of this process is via feed-back control and via gene repression of carbamoyl phosphate synthetase (the first enzyme of the pyrimidine biosynthesis pathway). The UMP kinase of E. coli (Ec) is known to function as a homohexamer, with GTP and UTP being allosteric effectors. Like other related enzymes (carbamate kinase, aspartokinase, and N-acetylglutamate kinase) the E. coli and most bacterial and chloroplast UMPKs (this CD) have a conserved, N-terminal, lysine residue proposed to function in the catalysis of the phosphoryl group transfer, whereas most archaeal UMPKs appear to lack this residue and the Pyrococcus furiosus structure has an additional Mg ion bound to the ATP molecule which is proposed to function as the catalysis instead. Members of this CD belong to the Amino Acid Kinase Superfamily (AAK).</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05188">cd05188</a></td><td>271</td><td>51</td><td>49.2</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MDR</td><td>Medium chain reductase/dehydrogenase (MDR)/zinc-dependent alcohol dehydrogenase-like family. The medium chain reductase/dehydrogenases (MDR)/zinc-dependent alcohol dehydrogenase-like family, which contains the zinc-dependent alcohol dehydrogenase (ADH-Zn) and related proteins, is a diverse group of proteins related to the first identified member, class I mammalian ADH.  MDRs display a broad range of activities and are distinguished from the smaller short chain dehydrogenases  (~ 250 amino acids vs. the ~ 350 amino acids of the MDR).  The MDR proteins have 2 domains: a C-terminal NAD(P) binding-Rossmann fold domain of a beta-alpha form and an N-terminal catalytic domain with distant homology to GroES.  The MDR group contains a host of activities, including the founding alcohol dehydrogenase (ADH) , quinone reductase, sorbitol dehydrogenase, formaldehyde dehydrogenase, butanediol DH, ketose reductase, cinnamyl reductase, and numerous others. The zinc-dependent alcohol dehydrogenases (ADHs) catalyze the NAD(P)(H)-dependent interconversion of alcohols to aldehydes or ketones. ADH-like proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and generally have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site and a structural zinc in a lobe of the catalytic domain. The active site zinc is coordinated by a histidine, two cysteines, and a water molecule. The second zinc seems to play a structural role, affects subunit interactions, and is typically coordinated by 4 cysteines. Other MDR members have only a catalytic zinc, and some contain no coordinated zinc.</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08243">cd08243</a></td><td>320</td><td>53</td><td>49.0</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>quinone_oxidoreductase_like_1</td><td>Quinone oxidoreductase (QOR). NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.  The medium chain alcohol dehydrogenase family (MDR) have a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form. The N-terminal region typically has an all-beta catalytic domain. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit.</td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12211">pfam12211</a></td><td>253</td><td>17</td><td>48.0</td><td>8.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LMWSLP_N</td><td>Low molecular weight S layer protein N terminal. This family of proteins is found in bacteria. Proteins in this family are typically between 328 and 381 amino acids in length. There is a conserved LGDG sequence motif. Clostridial species have a layer of surface proteins surrounding their membrane. This layer is comprised of a high molecular weight protein and a low molecular weight protein. This domain is the N terminal domain of the low molecular weight protein. It is a structural domain.</td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10256">pfam10256</a></td><td>118</td><td>42</td><td>46.8</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;]</td><td>Erf4</td><td>Golgin subfamily A member 7/ERF4 family. This family of proteins includes Golgin subfamily A member 7 proteins as well as Ras modification protein ERF4.</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03828">TIGR03828</a></td><td>304</td><td>94</td><td>46.6</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------]</td><td>pfkB</td><td>1-phosphofructokinase. This enzyme acts in concert with the fructose-specific phosphotransferase system (PTS) which imports fructose as fructose-1-phosphate. The action of 1-phosphofructokinase results in beta-D-fructose-1,6-bisphosphate and is an entry point into glycolysis (GenProp0688).</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07677">PRK07677</a></td><td>252</td><td>52</td><td>45.9</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07677</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02593">pfam02593</a></td><td>215</td><td>103</td><td>45.6</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------&nbsp;&nbsp;]</td><td>dTMP_synthase</td><td>Thymidylate synthase. This family catalyses the synthesis of thymidine monophosphate (dTMP) from deoxyuridine monophosphate (dUMP). The physiological co-substrate has not yet been identified.</td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4064">COG4064</a></td><td>75</td><td>22</td><td>44.7</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MtrG</td><td>Tetrahydromethanopterin S-methyltransferase, subunit G  </td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01746">pfam01746</a></td><td>185</td><td>41</td><td>44.4</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tRNA_m1G_MT</td><td>tRNA (Guanine-1)-methyltransferase. This is a family of tRNA (Guanine-1)-methyltransferases EC:2.1.1.31. In E.coli K12 this enzyme catalyses the conversion of a guanosine residue to N1-methylguanine in position 37, next to the anticodon, in tRNA.</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05360">cd05360</a></td><td>233</td><td>49</td><td>44.2</td><td>47</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SDR_c3</td><td>classical (c) SDR, subgroup 3. These proteins are members of the classical SDR family, with a canonical active site triad (and also active site Asn) and a typical Gly-rich NAD-binding motif. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRS are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes have a 3-glycine N-terminal NAD(P)(H)-binding pattern (typically, TGxxxGxG in classical SDRs and TGxxGxxG in extended SDRs), while substrate binding is in the C-terminal region. A critical catalytic Tyr residue (Tyr-151, human 15-hydroxyprostaglandin dehydrogenase (15-PGDH) numbering), is often found in a conserved YXXXK pattern. In addition to the Tyr and Lys, there is often an upstream Ser (Ser-138, 15-PGDH numbering) and/or an Asn (Asn-107, 15-PGDH numbering) or additional Ser, contributing to the active site. Substrates for these enzymes include sugars, steroids, alcohols, and aromatic compounds. The standard reaction mechanism is a proton relay involving the conserved Tyr and Lys, as well as Asn (or Ser). Some SDR family members, including 17 beta-hydroxysteroid dehydrogenase contain an additional helix-turn-helix motif that is not generally found among SDRs.</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01164">cd01164</a></td><td>289</td><td>49</td><td>42.4</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FruK_PfkB_like</td><td>1-phosphofructokinase (FruK), minor 6-phosphofructokinase (pfkB) and related sugar kinases. FruK plays an important role in the predominant pathway for fructose utilisation.This group also contains tagatose-6-phophate kinase, an enzyme of the tagatose 6-phosphate pathway, which responsible for breakdown of the galactose moiety during lactose metabolism by bacteria such as L. lactis.</td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08267">cd08267</a></td><td>319</td><td>44</td><td>42.4</td><td>70</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MDR1</td><td>Medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family. This group is a member of the medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, but lacks the zinc-binding sites of the zinc-dependent alcohol dehydrogenases. The medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, which contains the zinc-dependent alcohol dehydrogenase (ADH-Zn) and related proteins, is a diverse group of proteins related to the first identified member, class I mammalian ADH.  MDRs display a broad range of activities and are distinguished from the smaller short chain dehydrogenases (~ 250 amino acids vs. the ~ 350 amino acids of the MDR).  The MDR proteins have 2 domains: a C-terminal NAD(P)-binding Rossmann fold domain of a beta-alpha form and an N-terminal catalytic domain with distant homology to GroES.  The MDR group contains a host of activities, including the founding alcohol dehydrogenase (ADH), quinone reductase, sorbitol dehydrogenase, formaldehyde dehydrogenase, butanediol DH, ketose reductase, cinnamyl reductase, and numerous others. The zinc-dependent alcohol dehydrogenases (ADHs) catalyze the  NAD(P)(H)-dependent interconversion of alcohols to aldehydes or ketones.  Active site zinc has a catalytic role, while structural zinc aids in stability.  ADH-like proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and generally have 2 tightly bound zinc atoms per subunit. The active site zinc is coordinated by a histidine, two cysteines, and a water molecule. The second zinc seems to play a structural role, affects subunit interactions, and is typically coordinated by 4 cysteines.</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01026">PRK01026</a></td><td>77</td><td>27</td><td>42.1</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01026</td><td>tetrahydromethanopterin S-methyltransferase subunit G; Provisional</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11213">pfam11213</a></td><td>70</td><td>33</td><td>42.0</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3006</td><td>Protein of unknown function (DUF3006). This family of proteins has no known function.</td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05653">PRK05653</a></td><td>246</td><td>64</td><td>41.7</td><td>88</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fabG</td><td>3-ketoacyl-(acyl-carrier-protein) reductase; Validated</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02947">PRK02947</a></td><td>246</td><td>34</td><td>41.3</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK02947</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04198">pfam04198</a></td><td>255</td><td>60</td><td>40.9</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Sugar-bind</td><td>Putative sugar-binding domain. This probable domain is found in bacterial transcriptional regulators such as DeoR and SorC. These proteins have an amino-terminal helix-turn-helix pfam00325 that binds to DNA. This domain is probably the ligand regulator binding region. SorC is regulated by sorbose and other members of this family are likely to be regulated by other sugar substrates.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01012">cd01012</a></td><td>157</td><td>42</td><td>40.4</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YcaC_related</td><td>YcaC related amidohydrolases; E.coli YcaC is an homooctameric hydrolase with unknown specificity. Despite its weak sequence similarity, it is structurally related to other amidohydrolases and shares conserved active site residues with them. Multimerisation interface seems not to be conserved in all members.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08274">PRK08274</a></td><td>466</td><td>27</td><td>40.3</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08274</td><td>tricarballylate dehydrogenase; Validated</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05233">cd05233</a></td><td>234</td><td>37</td><td>39.0</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SDR_c</td><td>classical (c) SDRs. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRs are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes catalyze a wide range of activities including the metabolism of steroids, cofactors, carbohydrates, lipids, aromatic compounds, and amino acids, and act in redox sensing. Classical SDRs have an TGXXX</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09936">PRK09936</a></td><td>296</td><td>85</td><td>38.8</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09936</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4821">COG4821</a></td><td>243</td><td>57</td><td>38.4</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4821</td><td>Uncharacterized protein, contains SIS (Sugar ISomerase) phosphosugar binding domain  </td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04210">pfam04210</a></td><td>70</td><td>22</td><td>38.0</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MtrG</td><td>Tetrahydromethanopterin S-methyltransferase, subunit G. The N5-methyltetrahydromethanopterin: coenzyme M (EC:2.1.1.86) of Methanosarcina mazei Go1 is a membrane-associated, corrinoid-containing protein that uses a transmethylation reaction to drive an energy-conserving sodium ion pump.</td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05289">cd05289</a></td><td>309</td><td>47</td><td>38.0</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MDR_like_2</td><td>alcohol dehydrogenase and quinone reductase-like medium chain degydrogenases/reductases. Members identified as zinc-dependent alcohol dehydrogenases and quinone oxidoreductase. QOR catalyzes the conversion of a quinone + NAD(P)H to a hydroquinone + NAD(P)+. Quinones are cyclic diones derived from aromatic compounds.  Membrane bound QOR actin the respiratory chains of bacteria and mitochondria, while soluble QOR acts to protect from toxic quinones (e.g. DT-diaphorase) or as a soluble eye-lens protein in some vertebrates (e.g. zeta-crystalin). QOR reduces quinones through a semi-quinone intermediate via a NAD(P)H-dependent single electron transfer. QOR is a member of the medium chain dehydrogenase/reductase family, but lacks the zinc-binding sites of the prototypical alcohol dehydrogenases of this group. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.  Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation.  ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form. The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide.  A GxGxxG motif after the first mononucleotide contact half allows the close contact of the coenzyme with the ADH backbone.  The N-terminal catalytic domain has a distant homology to GroES. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site and a structural zinc in a lobe of the catalytic domain.  NAD(H) binding occurs in the cleft between the catalytic and coenzyme-binding domains at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding. In human ADH catalysis, the zinc ion helps coordinate the alcohol, followed by deprotonation of  a histidine, the ribose of NAD, a serine, then the alcohol, which allows the transfer of a hydride to NAD+, creating NADH and a zinc-bound aldehyde or ketone. In yeast and some bacteria, the active site zinc binds an aldehyde, polarizing it, and leading to the reverse reaction.</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05074">cd05074</a></td><td>284</td><td>28</td><td>37.3</td><td>28</td><td>[&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PTKc_Tyro3</td><td>Catalytic domain of the Protein Tyrosine Kinase, Tyro3. PTKs catalyze the transfer of the gamma-phosphoryl group from ATP to tyrosine (tyr) residues in protein substrates. Tyro3 (or Sky) is predominantly expressed in the central nervous system and the brain, and functions as a neurotrophic factor. It is also expressed in osteoclasts and has a role in bone resorption. Tyro3 is a member of the TAM subfamily, composed of receptor PTKs (RTKs) containing an extracellular ligand-binding region with two immunoglobulin-like domains followed by two fibronectin type III repeats, a transmembrane segment, and an intracellular catalytic domain. Binding to their ligands, Gas6 and protein S, leads to receptor dimerization, autophosphorylation, activation, and intracellular signaling. The Tyro3 subfamily is part of a larger superfamily that includes the catalytic domains of other kinases such as serine/threonine kinases, RIO kinases, aminoglycoside phosphotransferase, choline kinase, and phosphoinositide 3-kinase.</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02359">pfam02359</a></td><td>87</td><td>25</td><td>36.6</td><td>3.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CDC48_N</td><td>Cell division protein 48 (CDC48), N-terminal domain. This domain has a double psi-beta barrel fold and includes VCP-like ATPase and N-ethylmaleimide sensitive fusion protein N-terminal domains. Both the VAT and NSF N-terminal functional domains consist of two structural domains of which this is at the N-terminus. The VAT-N domain found in AAA ATPases pfam00004 is a substrate 185-residue recognition domain.</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0069">COG0069</a></td><td>485</td><td>54</td><td>36.4</td><td>73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GltB2</td><td>Glutamate synthase domain 2 </td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06180">pfam06180</a></td><td>256</td><td>123</td><td>36.1</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;---------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CbiK</td><td>Cobalt chelatase (CbiK). This family consists of several bacterial cobalt chelatase (CbiK) proteins (EC:4.99.1.-).</td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04139">cd04139</a></td><td>163</td><td>53</td><td>34.4</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RalA_RalB</td><td>Ral (Ras-like) family containing highly homologous RalA and RalB. The Ral (Ras-like) subfamily consists of the highly homologous RalA and RalB. Ral proteins are believed to play a crucial role in tumorigenesis, metastasis, endocytosis, and actin cytoskeleton dynamics. Despite their high sequence similarity (>80% sequence identity), nonoverlapping and opposing functions have been assigned to RalA and RalBs in tumor migration. In human bladder and prostate cancer cells, RalB promotes migration while RalA inhibits it. A Ral-specific set of GEFs has been identified that are activated by Ras binding. This RalGEF activity is enhanced by Ras binding to another of its target proteins, phosphatidylinositol 3-kinase (PI3K). Ral effectors include RLIP76/RalBP1, a Rac/cdc42 GAP, and the exocyst (Sec6/8) complex, a heterooctomeric protein complex that is involved in tethering vesicles to specific sites on the plasma membrane prior to exocytosis. In rat kidney cells, RalB is required for functional assembly of the exocyst and for localizing the exocyst to the leading edge of migrating cells. In human cancer cells, RalA is required to support anchorage-independent proliferation and RalB is required to suppress apoptosis. RalA has been shown to localize to the plasma membrane while RalB is localized to the intracellular vesicles. Most Ras proteins contain a lipid modification site at the C-terminus, with a typical sequence motif CaaX, where a = an aliphatic amino acid and X = any amino acid. Lipid binding is essential for membrane attachment, a key feature of most Ras proteins. Due to the presence of truncated sequences in this CD, the lipid modification site is not available for annotation.</td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00725">TIGR00725</a></td><td>159</td><td>51</td><td>34.0</td><td>71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00725</td><td>TIGR00725 family protein. This model represents one branch of a subfamily of uncharacterized proteins. Both PSI-BLAST and weak hits by this model show a low level of similarity and suggest an evolutionary relationship of the subfamily to the DprA/Smf family of DNA-processing proteins involved in chromosomal transformation with foreign DNA. Both Aquifex aeolicus and Mycobacterium leprae have one member in each of two branches of this subfamily, suggesting the branches may have distinct functions. This family is one of several families within the scope of pfam03641, several members of which are annotated as lysine decarboxylases. That larger family, and the branch described by this model, have a well-conserved motif PGGXGTXXE. </td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06924">PRK06924</a></td><td>251</td><td>28</td><td>33.9</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06924</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0604">COG0604</a></td><td>326</td><td>45</td><td>33.7</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Qor</td><td>NADPH:quinone reductase or related Zn-dependent oxidoreductase </td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06500">PRK06500</a></td><td>249</td><td>55</td><td>33.5</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06500</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03102">pfam03102</a></td><td>240</td><td>94</td><td>33.4</td><td>2.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NeuB</td><td>NeuB family. NeuB is the prokaryotic N-acetylneuraminic acid (Neu5Ac) synthase. It catalyses the direct formation of Neu5Ac (the most common sialic acid) by condensation of phosphoenolpyruvate (PEP) and N-acetylmannosamine (ManNAc). This reaction has only been observed in prokaryotes; eukaryotes synthesize the 9-phosphate form, Neu5Ac-9-P, and utilize ManNAc-6-P instead of ManNAc. Such eukaryotic enzymes are not present in this family. This family also contains SpsE spore coat polysaccharide biosynthesis proteins.</td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14478">PRK14478</a></td><td>475</td><td>46</td><td>33.4</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14478</td><td>nitrogenase molybdenum-cofactor biosynthesis protein NifE; Provisional</td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05974">cd05974</a></td><td>433</td><td>59</td><td>33.2</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MACS_like_1</td><td>Uncharacterized subfamily of medium-chain acyl-CoA synthetase (MACS). MACS catalyzes the two-step activation of medium chain fatty acids (containing 4-12 carbons). The carboxylate substrate first reacts with ATP to form an acyl-adenylate intermediate, which then reacts with CoA to produce an acyl-CoA ester. MACS enzymes are localized to mitochondria.</td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03127">TIGR03127</a></td><td>179</td><td>37</td><td>33.0</td><td>67</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RuMP_HxlB</td><td>6-phospho 3-hexuloisomerase. Members of this protein family are 6-phospho 3-hexuloisomerase (PHI), or the PHI domain of a fusion protein. This enzyme is part of the ribulose monophosphate (RuMP) pathway, which in one direction removes the toxic metabolite formaldehyde by assimilation into fructose-6-phosphate. In the other direction, in species lacking a complete pentose phosphate pathway, the RuMP pathway yields ribulose-5-phosphate, necessary for nucleotide biosynthesis, at the cost of also yielding formaldehyde. These latter species tend usually have a formaldehyde-activating enzyme to attach formaldehyde to the C1 carrier tetrahydromethanopterin.</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00809">PRK00809</a></td><td>144</td><td>67</td><td>32.8</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00809</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07099">cd07099</a></td><td>453</td><td>75</td><td>32.5</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ALDH_DDALDH</td><td>Methylomonas sp. 4,4'-diapolycopene-dialdehyde dehydrogenase-like. The 4,4'-diapolycopene-dialdehyde dehydrogenase (DDALDH) involved in C30 carotenoid synthesis in Methylomonas sp. strain 16a and other similar sequences are present in this CD. DDALDH converts 4,4'-diapolycopene-dialdehyde into 4,4'-diapolycopene-diacid.</td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08290">cd08290</a></td><td>341</td><td>74</td><td>32.1</td><td>43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ETR</td><td>2-enoyl thioester reductase (ETR). 2-enoyl thioester reductase (ETR) catalyzes the NADPH-dependent conversion of trans-2-enoyl acyl carrier protein/coenzyme A (ACP/CoA) to acyl-(ACP/CoA) in fatty acid synthesis. 2-enoyl thioester reductase activity has been linked in  Candida tropicalis as essential in maintaining mitiochondrial respiratory function. This ETR family is a part of the medium chain dehydrogenase/reductase family, but lack the zinc coordination sites characteristic of the alcohol dehydrogenases in this family. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.  Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation.   ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form. The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide.  The N-terminal catalytic domain has a distant homology to GroES. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site, and a structural zinc in a lobe of the catalytic domain.  NAD(H) binding occurs in the cleft between the catalytic and coenzyme-binding domains, at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding. Candida tropicalis enoyl thioester reductase (Etr1p) catalyzes the NADPH-dependent reduction of trans-2-enoyl thioesters in mitochondrial fatty acid synthesis. Etr1p forms homodimers, with each subunit containing a nucleotide-binding Rossmann fold domain and a catalytic domain.</td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01487">pfam01487</a></td><td>222</td><td>62</td><td>32.1</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DHquinase_I</td><td>Type I 3-dehydroquinase. Type I 3-dehydroquinase, (3-dehydroquinate dehydratase or DHQase.) Catalyses the cis-dehydration of 3-dehydroquinate via a covalent imine intermediate giving dehydroshikimate. Dehydroquinase functions in the shikimate pathway which is involved in the biosynthesis of aromatic amino acids. Type II 3-dehydroquinase catalyses the trans-dehydration of 3-dehydroshikimate see pfam01220.</td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4013">COG4013</a></td><td>91</td><td>31</td><td>31.5</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4013</td><td>Uncharacterized protein </td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02524">TIGR02524</a></td><td>358</td><td>95</td><td>31.0</td><td>77</td><td>[&nbsp;&nbsp;&nbsp;-----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>dot_icm_DotB</td><td>Dot/Icm secretion system ATPase DotB. Members of this protein family are the DotB component of Dot/Icm secretion systems, as found in obligate intracellular pathogens Legionella pneumophila and Coxiella burnetii. While this system resembles type IV secretion systems and has been called a form of type IV, the liturature now seems to favor calling this the Dot/Icm system. This family is most closely related to TraJ proteins of plasmid transfer, rather than to proteins of other type IV secretion systems.</td></tr>
<tr><td>86</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1899">COG1899</a></td><td>318</td><td>87</td><td>30.8</td><td>19</td><td>[&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DYS1</td><td>Deoxyhypusine synthase  </td></tr>
<tr><td>87</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3268">COG3268</a></td><td>382</td><td>37</td><td>30.0</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG3268</td><td>Uncharacterized conserved protein, related to short-chain dehydrogenases </td></tr>
<tr><td>88</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00885">cd00885</a></td><td>170</td><td>62</td><td>29.5</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cinA</td><td>Competence-damaged protein. CinA is the first gene in the competence- inducible (cin) operon and is thought to be specifically required at some stage in the process of transformation. This domain is closely related to a domain, found in a variety of proteins involved in biosynthesis of molybdopterin cofactor, where the domain is presumed to bind molybdopterin.</td></tr>
<tr><td>89</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00297">pfam00297</a></td><td>199</td><td>16</td><td>29.4</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ribosomal_L3</td><td>Ribosomal protein L3. </td></tr>
<tr><td>90</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08253">cd08253</a></td><td>325</td><td>45</td><td>29.3</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>zeta_crystallin</td><td>Zeta-crystallin with NADP-dependent quinone reductase activity (QOR). Zeta-crystallin is a eye lens protein with NADP-dependent quinone reductase activity (QOR). It has been cited as a structural component in mammalian eyes, but also has homology to quinone reductases in unrelated species. QOR catalyzes the conversion of a quinone and NAD(P)H to a hydroquinone and NAD(P+. Quinones are cyclic diones derived from aromatic compounds. Membrane bound QOR acts in the respiratory chains of bacteria and mitochondria, while soluble QOR acts to protect from toxic quinones (e.g. DT-diaphorase) or as a soluble eye-lens protein in some vertebrates (e.g. zeta-crystalin). QOR reduces quinones through a semi-quinone intermediate via a NAD(P)H-dependent single electron transfer. QOR is a member of the medium chain dehydrogenase/reductase family, but lacks the zinc-binding sites of the prototypical alcohol dehydrogenases of this group.  Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation.  ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form.  The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide.  The N-terminal catalytic domain has a distant homology to GroES. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site, and a structural zinc in a lobe of the catalytic domain.  NAD(H)-binding occurs in the cleft between the catalytic and coenzyme-binding domains at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding. In human ADH catalysis, the zinc ion helps coordinate the alcohol, followed by deprotonation of  a histidine, the ribose of NAD, a serine, then the alcohol, which allows the transfer of a hydride to NAD+, creating NADH and a zinc-bound aldehyde or ketone. In yeast and some bacteria, the active site zinc binds an aldehyde, polarizing it, and leading to the reverse reaction.</td></tr>
<tr><td>91</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK15482">PRK15482</a></td><td>285</td><td>45</td><td>29.1</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK15482</td><td>transcriptional regulator MurR; Provisional</td></tr>
<tr><td>92</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd15854">cd15854</a></td><td>59</td><td>26</td><td>29.0</td><td>60</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SNARE_SNAP47C</td><td>C-terminal SNARE motif of SNAP47. C-terminal SNARE motif of SNAP47, a member of the Qb/Qc subfamily of SNARE (soluble N-ethylmaleimide-sensitive factor attachment protein receptor) proteins. The exact funtion of SNAP47 is unknown. Qb/Qc SNAREs consist of 2 coiled-coil helices (called SNARE motifs, one belonging to the Qb subgroup and one belonging to the Qc subgroup), which mediate the interactions with other SNARE proteins, and a transmembrane domain. In general, the SNARE complex mediates membrane fusion, important for trafficking of newly synthesized proteins, recycling of pre-existing proteins and organelle formation. SNARE proteins are classified into four groups, Qa-, Qb-, Qc- and R-SNAREs, depending on whether the residue in the hydrophilic center layer of the four-helical bundle is a glutamine (Q) or arginine (R). Qa-, as well as Qb- and Qc-SNAREs, are localized to target organelle membranes, while R-SNARE is localized to vesicle membranes. They form unique complexes consisting of one member of each subgroup, that mediate fusion between a specific type of vesicles and their target organelle. Their SNARE motifs form twisted and parallel heterotetrameric helix bundles. Other members of the Qb/Qc SNAREs are SNAP23, SNAP25, SNAP29 and SEC9.</td></tr>
<tr><td>93</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05565">PRK05565</a></td><td>247</td><td>47</td><td>28.7</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fabG</td><td>3-ketoacyl-(acyl-carrier-protein) reductase; Provisional</td></tr>
<tr><td>94</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07549">pfam07549</a></td><td>31</td><td>7</td><td>28.1</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Sec_GG</td><td>SecD/SecF GG Motif. This family consists of various prokaryotic SecD and SecF protein export membrane proteins.  This SecD and SecF proteins are part of the multimeric protein export complex comprising SecA, D, E, F, G, Y, and YajC. SecD and SecF are required to maintain a proton motive force. This alignment encompasses a -GG- motif typically found in N-terminal half of the SecD/SecF proteins.</td></tr>
<tr><td>95</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01261">pfam01261</a></td><td>202</td><td>33</td><td>28.1</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AP_endonuc_2</td><td>Xylose isomerase-like TIM barrel. This TIM alpha/beta barrel structure is found in xylose isomerase and in endonuclease IV (EC:3.1.21.2). This domain is also found in the N termini of bacterial myo-inositol catabolism proteins. These are involved in the myo-inositol catabolism pathway, and is required for growth on myo-inositol in Rhizobium leguminosarum bv. viciae.</td></tr>
<tr><td>96</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09694">cd09694</a></td><td>181</td><td>95</td><td>28.0</td><td>64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csm6_III-A</td><td>CRISPR/Cas system-associated protein Csm6. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; loosely associated with CRISPR/Cas systems</td></tr>
<tr><td>97</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01011">TIGR01011</a></td><td>225</td><td>65</td><td>27.9</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rpsB_bact</td><td>ribosomal protein S2, bacterial type. This model describes the bacterial, ribosomal, and chloroplast forms of ribosomal protein S2. TIGR01012 describes the archaeal and cytosolic forms. </td></tr>
<tr><td>98</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07985">PRK07985</a></td><td>294</td><td>48</td><td>27.8</td><td>59</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07985</td><td>oxidoreductase; Provisional</td></tr>
<tr><td>99</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05359">cd05359</a></td><td>242</td><td>47</td><td>27.3</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ChcA_like_SDR_c</td><td>1-cyclohexenylcarbonyl_coenzyme A_reductase (ChcA)_like, classical (c) SDRs. This subgroup contains classical SDR proteins, including members identified as 1-cyclohexenylcarbonyl coenzyme A reductase. ChcA of Streptomyces collinus is implicated in the final reduction step of shikimic acid to ansatrienin. ChcA shows sequence similarity to the SDR family of NAD-binding proteins, but it lacks the conserved Tyr of the characteristic catalytic site. This subgroup also contains the NADH-dependent enoyl-</td></tr>
<tr><td>100</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1105">COG1105</a></td><td>310</td><td>48</td><td>26.8</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FruK</td><td>Fructose-1-phosphate kinase or kinase (PfkB) </td></tr>
<tr><td>101</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07899">PRK07899</a></td><td>486</td><td>30</td><td>26.5</td><td>65</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rpsA</td><td>30S ribosomal protein S1; Reviewed</td></tr>
<tr><td>102</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01149">TIGR01149</a></td><td>70</td><td>22</td><td>26.4</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>mtrG</td><td>N5-methyltetrahydromethanopterin:coenzyme M methyltransferase subunit G. This model describes N5-methyltetrahydromethanopterin: coenzyme M methyltransferase subunit G in methanogenic archaea. This methyltranfersae is membrane-associated enzyme complex that uses methyl-transfer reaction to drive sodium-ion pump. Archaea have evolved energy-yielding pathways marked by one-carbon biochemistry featuring novel cofactors and enzymes. This transferase is involved in the transfer of 'methyl' group from N5-methyltetrahydromethanopterin to coenzyme M. In an accompanying reaction, methane is produced by two-electron reduction of the methyl moiety in methyl-coenzyme M by another enzyme methyl-coenzyme M reductase. </td></tr>
<tr><td>103</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14713">pfam14713</a></td><td>234</td><td>77</td><td>26.4</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4464</td><td>Domain of unknown function (DUF4464). This family of proteins is found in eukaryotes. Proteins in this family are typically between 224 and 241 amino acids in length. There is a conserved YID sequence motif.</td></tr>
<tr><td>104</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07656">PRK07656</a></td><td>513</td><td>46</td><td>25.9</td><td>77</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07656</td><td>long-chain-fatty-acid--CoA ligase; Validated</td></tr>
<tr><td>105</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08939">cd08939</a></td><td>239</td><td>48</td><td>25.8</td><td>71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KDSR-like_SDR_c</td><td>3-ketodihydrosphingosine reductase (KDSR) and related proteins, classical (c) SDR. These proteins include members identified as KDSR, ribitol type dehydrogenase, and others. The group shows strong conservation of the active site tetrad and glycine rich NAD-binding motif of the classical SDRs. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRs are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes catalyze a wide range of activities including the metabolism of steroids, cofactors, carbohydrates, lipids, aromatic compounds, and amino acids, and act in redox sensing. Classical SDRs have an TGXXX</td></tr>
<tr><td>106</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2089">COG2089</a></td><td>347</td><td>87</td><td>25.8</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SpsE</td><td>Sialic acid synthase SpsE, contains C-terminal SAF domain </td></tr>
<tr><td>107</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12535">pfam12535</a></td><td>58</td><td>22</td><td>25.6</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Nudix_N</td><td>Hydrolase of X-linked nucleoside diphosphate N terminal. This family of proteins is found in eukaryotes. Proteins in this family are typically between 847 and 5344 amino acids in length. These enzymes hydrolyse the molecular motif of a nucleoside diphosphate linked to some other moiety, X.</td></tr>
<tr><td>108</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01818">pfam01818</a></td><td>120</td><td>15</td><td>25.2</td><td>42</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Translat_reg</td><td>Bacteriophage translational regulator. The translational regulator protein regA is encoded by the T4 bacteriophage and binds to a region of messenger RNA (mRNA) that includes the initiator codon. RegA is unusual in that it represses the translation of about 35 early T4 mRNAs but does not affect nearly 200 other mRNAs.</td></tr>
<tr><td>109</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05327">cd05327</a></td><td>269</td><td>47</td><td>25.0</td><td>2.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>retinol-DH_like_SDR_c_like</td><td>retinol dehydrogenase (retinol-DH), Light dependent Protochlorophyllide (Pchlide) OxidoReductase (LPOR) and related proteins, classical (c) SDRs. Classical SDR subgroup containing retinol-DHs, LPORs, and related proteins. Retinol is processed by a medium chain alcohol dehydrogenase followed by retinol-DHs. Pchlide reductases act in chlorophyll biosynthesis. There are distinct enzymes that catalyze Pchlide reduction in light or dark conditions. Light-dependent reduction is via an NADP-dependent SDR, LPOR. Proteins in this subfamily share the glycine-rich NAD-binding motif of the classical SDRs, have a partial match to the canonical active site tetrad, but lack the typical active site Ser. This subgroup includes the human proteins: retinol dehydrogenase -12, -13 ,and -14, dehydrogenase/reductase SDR family member (DHRS)-12 , -13 and -X (a DHRS on chromosome X), and WWOX (WW domain-containing oxidoreductase), as well as a Neurospora crassa SDR encoded by the blue light inducible bli-4 gene. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRs are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes catalyze a wide range of activities including the metabolism of steroids, cofactors, carbohydrates, lipids, aromatic compounds, and amino acids, and act in redox sensing. Classical SDRs have an TGXXX</td></tr>
<tr><td>110</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05329">cd05329</a></td><td>251</td><td>50</td><td>24.9</td><td>2.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TR_SDR_c</td><td>tropinone reductase-I and II (TR-1, and TR-II)-like, classical (c) SDRs. This subgroup includes TR-I and TR-II; these proteins are members of the SDR family. TRs catalyze the NADPH-dependent reductions of the 3-carbonyl group of tropinone, to a beta-hydroxyl group. TR-I and TR-II produce different stereoisomers from tropinone, TR-I produces tropine (3alpha-hydroxytropane), and TR-II, produces pseudotropine (sigma-tropine, 3beta-hydroxytropane).  SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRs are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes catalyze a wide range of activities including the metabolism of steroids, cofactors, carbohydrates, lipids, aromatic compounds, and amino acids, and act in redox sensing. Classical SDRs have an TGXXX</td></tr>
<tr><td>111</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04545">TIGR04545</a></td><td>339</td><td>44</td><td>24.8</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;]</td><td>rSAM_ahbD_hemeb</td><td>heme b synthase. Members of this family are AhbD (alternative heme biosynthetic protein D), a radical SAM enzyme in sulfate-reducing bacteria and methanogens that performs the last decarboxylations to synthesize heme b from Fe-coproporphyrin III. Members include DVU_0855, previously included in error in TIGR04055, the NirJ2 family thought to be involved in heme d1 biosynthesis. </td></tr>
<tr><td>112</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00919">PRK00919</a></td><td>307</td><td>64</td><td>24.4</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00919</td><td>GMP synthase subunit B; Validated</td></tr>
<tr><td>113</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3462">COG3462</a></td><td>117</td><td>39</td><td>24.0</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG3462</td><td>Uncharacterized membrane protein  </td></tr>
<tr><td>114</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05341">cd05341</a></td><td>247</td><td>45</td><td>23.7</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3beta-17beta-HSD_like_SDR_c</td><td>3beta17beta hydroxysteroid dehydrogenase-like, classical (c) SDRs. This subgroup includes members identified as 3beta17beta hydroxysteroid dehydrogenase, 20beta hydroxysteroid dehydrogenase, and R-alcohol dehydrogenase. These proteins exhibit the canonical active site tetrad and glycine rich NAD(P)-binding motif of the classical SDRs. 17beta-dehydrogenases are a group of isozymes that catalyze activation and inactivation of estrogen and androgens, and include members of the SDR family. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRS are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes have a 3-glycine N-terminal NAD(P)(H)-binding pattern (typically, TGxxxGxG in classical SDRs and TGxxGxxG in extended SDRs), while substrate binding is in the C-terminal region. A critical catalytic Tyr residue (Tyr-151, human 15-hydroxyprostaglandin dehydrogenase (15-PGDH) numbering), is often found in a conserved YXXXK pattern. In addition to the Tyr and Lys, there is often an upstream Ser (Ser-138, 15-PGDH numbering) and/or an Asn (Asn-107, 15-PGDH numbering) or additional Ser, contributing to the active site. Substrates for these enzymes include sugars, steroids, alcohols, and aromatic compounds. The standard reaction mechanism is a proton relay involving the conserved Tyr and Lys, as well as Asn (or Ser). Some SDR family members, including 17 beta-hydroxysteroid dehydrogenase contain an additional helix-turn-helix motif that is not generally found among SDRs.</td></tr>
<tr><td>115</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05355">cd05355</a></td><td>270</td><td>46</td><td>23.3</td><td>74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SDR_c1</td><td>classical (c) SDR, subgroup 1. These proteins are members of the classical SDR family, with a canonical active site tetrad and a typical Gly-rich NAD-binding motif. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRs are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes catalyze a wide range of activities including the metabolism of steroids, cofactors, carbohydrates, lipids, aromatic compounds, and amino acids, and act in redox sensing. Classical SDRs have an TGXXX</td></tr>
<tr><td>116</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03792">pfam03792</a></td><td>192</td><td>47</td><td>23.2</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBC</td><td>PBC domain. The PBC domain is a member of the TALE (three-amino-acid loop extension) superclass of homeodomain proteins.</td></tr>
<tr><td>117</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00358">PRK00358</a></td><td>231</td><td>39</td><td>23.1</td><td>86</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pyrH</td><td>uridylate kinase; Provisional</td></tr>
<tr><td>118</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05513">cd05513</a></td><td>98</td><td>17</td><td>22.9</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Bromo_brd7_like</td><td>Bromodomain, brd7_like subgroup. The BRD7 gene encodes a nuclear protein that has been shown to inhibit cell growth and the progression of the cell cycle by regulating cell-cycle genes at the transcriptional level. BRD7 has been identified as a gene involved in nasopharyngeal carcinoma. The protein interacts with acetylated histone H3 via its bromodomain. Bromodomains are 110 amino acid long domains that are found in many chromatin associated proteins. Bromodomains can interact specifically with acetylated lysine.</td></tr>
<tr><td>119</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09205">cd09205</a></td><td>143</td><td>11</td><td>22.5</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLDc_N_DEXD_b3</td><td>N-terminal putative catalytic domain of uncharacterized prokaryotic and archeal HKD family nucleases fused to a DEAD/DEAH box helicase domain. N-terminal putative catalytic domain of uncharacterized prokaryotic and archeal HKD family nucleases fused to a DEAD/DEAH box helicase domain. All members of this subfamily are uncharacterized. Other characterized members of the superfamily that have a related domain architecture ( containing a DEAD/DEAH box helicase domain), include the DNA/RNA helicase superfamily II (SF2) and Res-subunit of type III restriction endonucleases. In addition to the helicase-like region, members of this subfamily also contain one copy of the conserved HKD motif (H-x-K-x(4)-D, where x represents any amino acid residue) in the N-terminal putative catalytic domain. The HKD motif characterizes the phospholipase D (PLD, EC 3.1.4.4) superfamily.  A few family members contain additional domains, like a C-terminal peptidase S24-like domain.</td></tr>
<tr><td>120</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01421">cd01421</a></td><td>187</td><td>57</td><td>22.5</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>IMPCH</td><td>Inosine monophosphate cyclohydrolase domain. This is the N-terminal domain in the purine biosynthesis pathway protein ATIC (purH). The bifunctional ATIC protein contains a C-terminal  ATIC formylase domain that formylates 5-aminoimidazole-4-carboxamide-ribonucleotide. The IMPCH domain then converts the formyl-5-aminoimidazole-4-carboxamide-ribonucleotide to inosine monophosphate. This is the final step in de novo purine production.</td></tr>
<tr><td>121</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05013">cd05013</a></td><td>139</td><td>34</td><td>22.0</td><td>88</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SIS_RpiR</td><td>RpiR-like protein. RpiR contains a SIS (Sugar ISomerase) domain, which is found in many phosphosugar isomerases and phosphosugar binding proteins. In E. coli, rpiR negatively regulates the expression of rpiB gene. Both rpiB and rpiA are ribose phosphate isomerases that catalyze the reversible reactions of ribose 5-phosphate into ribulose 5-phosphate.</td></tr>
<tr><td>122</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00994">pfam00994</a></td><td>141</td><td>27</td><td>21.9</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MoCF_biosynth</td><td>Probable molybdopterin binding domain. This domain is found a variety of proteins involved in biosynthesis of molybdopterin cofactor. The domain is presumed to bind molybdopterin. The structure of this domain is known, and it forms an alpha/beta structure. In the known structure of Gephyrin this domain mediates trimerization.</td></tr>
<tr><td>123</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09664">PRK09664</a></td><td>415</td><td>19</td><td>21.7</td><td>21</td><td>[&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09664</td><td>tryptophan permease TnaB; Provisional</td></tr>
<tr><td>124</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03156">cd03156</a></td><td>114</td><td>34</td><td>21.7</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>uroplakin_I_like_LEL</td><td>Tetraspanin, extracellular domain or large extracellular loop (LEL), uroplakin_I_like family. Tetraspanins are trans-membrane proteins with 4 trans-membrane segments. Both the N- and C-termini lie on the intracellular side of the membrane. This alignment model spans the extracellular domain between the 3rd and 4th trans-membrane segment. Tetraspanins are involved in diverse processes and their various functions may relate to their ability to act as molecular facilitators. Tetraspanins associate laterally with one another and cluster dynamically with numerous parnter domains in membrane microdomains, forming a network of multimolecular complexes, the "tetraspanin web". Uroplakin Ia and Ib are components of the 16nm protein particles, which are packed hexagonally to form 2D crystals of asymmetric unit membranes, and cover the apical surface of mammalian urothelium, contributing to the urinay bladder's permeability barrier function. Uroplakins Ia and Ib are maturation facilitators. They trigger conformational changes in their single-transmembrane-domain binding partner proteins uroplakin II and IIIa, which in turn may lead to ER-exit, stabilization, and cell-surface expression.</td></tr>
<tr><td>125</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05369">cd05369</a></td><td>249</td><td>59</td><td>21.5</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;]</td><td>TER_DECR_SDR_a</td><td>Trans-2-enoyl-CoA reductase (TER) and 2,4-dienoyl-CoA reductase (DECR), atypical (a) SDR. TTER is a peroxisomal protein with a proposed role in fatty acid elongation. Fatty acid synthesis is known to occur in the both endoplasmic reticulum and mitochondria; peroxisomal TER has been proposed as an additional fatty acid elongation system, it reduces the double bond at C-2 as the last step of elongation.  This system resembles the mitochondrial system in that acetyl-CoA is used as a carbon donor. TER may also function in phytol metabolism, reducting phytenoyl-CoA to phytanoyl-CoA in peroxisomes. DECR processes double bonds in fatty acids to increase their utility in fatty acid metabolism; it reduces 2,4-dienoyl-CoA to an enoyl-CoA. DECR is active in mitochondria and peroxisomes. This subgroup has the Gly-rich NAD-binding motif of the classical SDR family, but does not display strong identity to the canonical active site tetrad, and lacks the characteristic Tyr at the usual position. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRS are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes have a 3-glycine N-terminal NAD(P)(H)-binding pattern (typically, TGxxxGxG in classical SDRs and TGxxGxxG in extended SDRs), while substrate binding is in the C-terminal region. A critical catalytic Tyr residue (Tyr-151, human 15-hydroxyprostaglandin dehydrogenase (15-PGDH) numbering), is often found in a conserved YXXXK pattern. In addition to the Tyr and Lys, there is often an upstream Ser (Ser-138, 15-PGDH numbering) and/or an Asn (Asn-107, 15-PGDH numbering) or additional Ser, contributing to the active site. Substrates for these enzymes include sugars, steroids, alcohols, and aromatic compounds. The standard reaction mechanism is a proton relay involving the conserved Tyr and Lys, as well as Asn (or Ser). Some SDR family members, including 17 beta-hydroxysteroid dehydrogenase contain an additional helix-turn-helix motif that is not generally found among SDRs.</td></tr>
<tr><td>126</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09861">pfam09861</a></td><td>203</td><td>12</td><td>21.3</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2088</td><td>Domain of unknown function (DUF2088). This domain, found in various hypothetical prokaryotic proteins, has no known function.</td></tr>
<tr><td>127</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07060">PRK07060</a></td><td>245</td><td>43</td><td>21.3</td><td>1.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07060</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>128</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00433">TIGR00433</a></td><td>296</td><td>46</td><td>21.3</td><td>2.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>bioB</td><td>biotin synthase. Catalyzes the last step of the biotin biosynthesis pathway. All members of the seed alignment are in the immediate gene neighborhood of a bioA gene. </td></tr>
<tr><td>129</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04024">TIGR04024</a></td><td>330</td><td>64</td><td>21.2</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_NP1902A</td><td>coenzyme F420-dependent oxidoreductase, NP1902A family. This subfamily of the luciferase-like monooxygenases is restricted to the order Halobacteriales. SIMBAL analysis strongly suggests this oxidoreductase binds coenzyme F420 rather than FMN. Occasional annotations of members of this family as N5,N10-methylenetetrahydromethanopterin reductase appear to represent overly aggressive transfer of annotation. </td></tr>
<tr><td>130</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14394">pfam14394</a></td><td>171</td><td>30</td><td>21.1</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4423</td><td>Domain of unknown function (DUF4423). This presumed domain is functionally uncharacterized. This domain family is found in bacteria, and is approximately 170 amino acids in length.</td></tr>
<tr><td>131</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06395">pfam06395</a></td><td>89</td><td>35</td><td>20.9</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CDC24</td><td>CDC24 Calponin. Is a calponin homology domain.</td></tr>
<tr><td>132</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00071">pfam00071</a></td><td>162</td><td>29</td><td>20.8</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ras</td><td>Ras family. Includes sub-families Ras, Rab, Rac, Ral, Ran, Rap Ypt1 and more. Shares P-loop motif with GTP_EFTU, arf and myosin_head. See pfam00009 pfam00025, pfam00063. As regards Rab GTPases, these are important regulators of vesicle formation, motility and fusion. They share a fold in common with all Ras GTPases: this is a six-stranded beta-sheet surrounded by five alpha-helices.</td></tr>
<tr><td>133</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2379">COG2379</a></td><td>422</td><td>31</td><td>20.7</td><td>79</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GckA</td><td>Glycerate-2-kinase  </td></tr>
<tr><td>134</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03802">TIGR03802</a></td><td>562</td><td>56</td><td>20.6</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Asp_Ala_antiprt</td><td>aspartate-alanine antiporter. All members of the seed alignment for this model are asparate-alanine anti-transporters (AspT) encoded next to the gene for aspartate 4-decarboxylase (AspD), which converts asparate to alanine, releasing CO2. The exchange of Asp for Ala is electrogenic, so the AspD/AspT system confers a proton-motive force. This transporter contains two copies of the AspT/YidE/YbjL antiporter duplication domain (TIGR01625).</td></tr>
<tr><td>135</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12583">cd12583</a></td><td>75</td><td>26</td><td>20.4</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM2_hnRNPD</td><td>RNA recognition motif 2 in heterogeneous nuclear ribonucleoprotein D0 (hnRNP D0) and similar proteins. This subgroup corresponds to the RRM2 of hnRNP D0, also termed AU-rich element RNA-binding protein 1, a UUAG-specific nuclear RNA binding protein that may be involved in pre-mRNA splicing and telomere elongation. hnRNP D0 contains two RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains), in the middle and an RGG box rich in glycine and arginine residues in the C-terminal part. Each of RRMs can bind solely to the UUAG sequence specifically. </td></tr>
<tr><td>136</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14803">cd14803</a></td><td>97</td><td>23</td><td>20.0</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RAP</td><td>Receptor-associated protein (RAP). Receptor-associated protein, RAP, is an antagonist and a specialized chaperone in the endoplasmic reticulum that binds tightly to members of the low-density lipoprotein (LDL) receptor family and prevents them from associating with other ligands. RAP associates with (LDL) receptor-related protein (LRP) early in the secretory pathway, reducing its ligand binding capacity, and then dissociates from LRP in the low-pH environment of the Golgi; studies have shown that histidine residues in RAP D3 serve as a switch that facilitates its uncoupling from the receptor. RAP is a modular protein identified as having an internal triplication, with domains, D1, D2, and D3, each thought to have distinct functions; these domains are independent and do not interact. The carboxyl-terminal domain (D3) of RAP is required for folding and trafficking of LRP, while the amino-terminal tandem D1D2 domains of RAP are essential for blocking LRP from binding of certain ligands, such as activated forms of alpha2-macroglobulin.</td></tr>
</table>
</body>
