<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09742">cd09742</a></td><td>183</td><td>127</td><td>100.0</td><td>1.2E-40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csm6_III-A</td><td>CRISPR/Cas system-associated protein Csm6. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; loosely associated with CRISPR/Cas systems; also known as APE2256 family</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09651">pfam09651</a></td><td>136</td><td>116</td><td>100.0</td><td>4.4E-39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_APE2256</td><td>CRISPR-associated protein (Cas_APE2256). This entry represents a conserved region of about 150 amino acids found in at least five archaeal and three bacterial species. These species all contain CRISPRs (Clustered Regularly Interspaced Short Palindromic Repeats). In six of eight species, the protein is encoded the vicinity of a CRISPR/Cas locus.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4006">COG4006</a></td><td>278</td><td>136</td><td>100.0</td><td>3.1E-30</td><td>[&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4006</td><td>CRISPR/Cas system-associated protein Csm6, COG1517 family </td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02619">TIGR02619</a></td><td>149</td><td>123</td><td>100.0</td><td>2.7E-30</td><td>[&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02619</td><td>putative CRISPR-associated protein, APE2256 family. This model represents a conserved domain of about 150 amino acids found in at least five archaeal species and three bacterial species, exclusively in species with CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats). In six of eight species, the member of this family is in the vicinity of a CRISPR/Cas locus.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09694">cd09694</a></td><td>181</td><td>133</td><td>100.0</td><td>1.1E-29</td><td>[&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csm6_III-A</td><td>CRISPR/Cas system-associated protein Csm6. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; loosely associated with CRISPR/Cas systems</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12870">PRK12870</a></td><td>290</td><td>205</td><td>97.5</td><td>4.5E-07</td><td>[&nbsp;&nbsp;------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ubiA</td><td>4-hydroxybenzoate polyprenyltransferase; Reviewed</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03642">TIGR03642</a></td><td>124</td><td>80</td><td>97.3</td><td>0.0033</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_csx14</td><td>CRISPR-associated protein, Csx14 family. This model describes a protein N-terminal protein sequence domain strictly associated with CRISPR and CRISPR-associated protein systems. This model and TIGR02584 identify two separate clades from a larger homology domain family, both CRISPR-associated, while other homologs are found that may not be. Members are found in bacteria that include Pelotomaculum thermopropionicum SI, Thermoanaerobacter tengcongensis MB4, and Roseiflexus sp. RS-1, and in archaea that include Thermoplasma volcanium, Picrophilus torridus, and Methanospirillum hungatei. The molecular function is unknown.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09723">cd09723</a></td><td>132</td><td>81</td><td>97.3</td><td>0.0034</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as csx13 family</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09623">pfam09623</a></td><td>225</td><td>111</td><td>97.1</td><td>0.012</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_NE0113</td><td>CRISPR-associated protein NE0113 (Cas_NE0113). Members of this minor CRISPR-associated (Cas) protein family are encoded in cas gene clusters in Vibrio vulnificus YJ016, Nitrosomonas europaea ATCC 19718, Mannheimia succiniciproducens MBEL55E, and Verrucomicrobium spinosum.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09747">cd09747</a></td><td>378</td><td>82</td><td>96.1</td><td>0.03</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as Cas02710 family</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09670">pfam09670</a></td><td>379</td><td>82</td><td>96.0</td><td>0.037</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_Cas02710</td><td>CRISPR-associated protein (Cas_Cas02710). Members of this family are found, exclusively in the vicinity of CRISPR repeats and other CRISPR-associated (cas) genes, in Methanothermobacter thermautotrophicus (Methanobacterium thermoformicicum), Thermus thermophilus (Deinococcus-Thermus), Chloroflexus aurantiacus (Chloroflexi), and Thermomicrobium roseum (Thermomicrobia).</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09741">cd09741</a></td><td>219</td><td>108</td><td>96.0</td><td>0.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as NE0113 family</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09002">pfam09002</a></td><td>379</td><td>85</td><td>94.4</td><td>0.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1887</td><td>Domain of unknown function (DUF1887). This domain is found in a set of hypothetical bacterial proteins.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09655">cd09655</a></td><td>198</td><td>103</td><td>93.5</td><td>0.82</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CasRa_I-A</td><td>CRISPR/Cas system-associated transcriptional regulator CasRa. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Predicted transcriptional regulator of CRISPR/Cas system</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02710">TIGR02710</a></td><td>380</td><td>82</td><td>90.6</td><td>0.64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02710</td><td>CRISPR-associated protein, TIGR02710 family. Members of this family are found, exclusively in the vicinity of CRISPR repeats and other CRISPR-associated (cas) genes, in Methanothermobacter thermautotrophicus (Archaea), Thermus thermophilus (Deinococcus-Thermus), Chloroflexus aurantiacus (Chloroflexi), and Thermomicrobium roseum (Thermomicrobia).</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01884">TIGR01884</a></td><td>203</td><td>56</td><td>86.9</td><td>4.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_HTH</td><td>CRISPR locus-related DNA-binding protein. Most but not all examples of this family are associated with CRISPR loci, a combination of DNA repeats and characteristic proteins encoded near the repeat cluster. The C-terminal region of this protein is homologous to DNA-binding helix-turn-helix domains with predicted transcriptional regulatory activity. </td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09702">cd09702</a></td><td>378</td><td>53</td><td>77.1</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as TIGR02710 family</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09398">pfam09398</a></td><td>81</td><td>19</td><td>55.8</td><td>9.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FOP_dimer</td><td>FOP N terminal dimerization domain. Fibroblast growth factor receptor 1 (FGFR1) oncogene partner (FOP) is a centrosomal protein that is involved in anchoring microtubules to subcellular structures. This domain includes a Lis-homology motif. It forms an alpha helical bundle and is involved in dimerization.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03272">cd03272</a></td><td>243</td><td>32</td><td>55.4</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC3_euk</td><td>ATP-binding cassette domain of eukaryotic SMC3 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03785">cd03785</a></td><td>350</td><td>32</td><td>42.9</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GT1_MurG</td><td>MurG is an N-acetylglucosaminyltransferase, the last enzyme involved in the intracellular phase of peptidoglycan biosynthesis. It transfers N-acetyl-D-glucosamine (GlcNAc) from UDP-GlcNAc to the C4 hydroxyl of a lipid-linked N-acetylmuramoyl pentapeptide (NAM). The resulting disaccharide is then transported across the cell membrane, where it is polymerized into NAG-NAM cell-wall repeat structure. MurG belongs to the GT-B structural superfamily of glycoslytransferases, which have characteristic N- and C-terminal domains, each containing a typical Rossmann fold. The two domains have high structural homology despite minimal sequence homology.  The large cleft that separates the two domains includes the catalytic center and permits a high degree of flexibility.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01133">TIGR01133</a></td><td>348</td><td>32</td><td>42.2</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>murG</td><td>undecaprenyldiphospho-muramoylpentapeptide beta-N-acetylglucosaminyltransferase. RM 8449890 RT The final step of peptidoglycan subunit assembly in Escherichia coli occurs in the cytoplasm. RA Bupp K, van Heijenoort J. RL J Bacteriol 1993 Mar;175(6):1841-3 </td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02463">pfam02463</a></td><td>1162</td><td>36</td><td>42.0</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SMC_N</td><td>RecF/RecN/SMC N terminal domain. This domain is found at the N terminus of SMC proteins. The SMC (structural maintenance of chromosomes) superfamily proteins have ATP-binding domains at the N- and C-termini, and two extended coiled-coil domains separated by a hinge in the middle. The eukaryotic SMC proteins form two kind of heterodimers: the SMC1/SMC3 and the SMC2/SMC4 types. These heterodimers constitute an essential part of higher order complexes, which are involved in chromatin and DNA dynamics.   This family also includes the RecF and RecN proteins that are involved in DNA metabolizm and recombination.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03278">cd03278</a></td><td>197</td><td>33</td><td>41.3</td><td>33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC_barmotin</td><td>ATP-binding cassette domain of barmotin, a member of the SMC protein family. Barmotin is a tight junction-associated protein expressed in rat epithelial cells which is thought to have an important regulatory role in tight junction barrier function. Barmotin belongs to the SMC protein family. SMC proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03239">cd03239</a></td><td>178</td><td>32</td><td>35.5</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC_head</td><td>The SMC head domain belongs to the ATP-binding cassette superfamily. The structural maintenance of chromosomes (SMC) proteins are essential for successful chromosome transmission during replication and segregation of the genome in all organisms. SMCs are generally present as single proteins in bacteria, and as at least six distinct proteins in eukaryotes. The proteins range in size from approximately 110 to 170 kDa, and each has five distinct domains: amino- and carboxy-terminal globular domains, which contain sequences characteristic of ATPases, two coiled-coil regions separating the terminal domains , and a central flexible hinge. SMC proteins function together with other proteins in a range of chromosomal transactions, including chromosome condensation, sister-chromatid cohesion, recombination, DNA repair, and epigenetic silencing of gene expression.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01536">cd01536</a></td><td>267</td><td>86</td><td>34.3</td><td>2.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP1_ABC_sugar_binding_like</td><td>Periplasmic sugar-binding domain of active transport systems that are members of the type I periplasmic binding protein (PBP1) superfamily. Periplasmic sugar-binding domain of active transport systems that are members of the type I periplasmic binding protein (PBP1) superfamily. The members of this family function as the primary receptors for chemotaxis and transport of many sugar based solutes in bacteria and archaea. The sugar binding domain is also homologous to the ligand-binding domain of eukaryotic receptors such as glutamate receptor (GluR) and DNA-binding transcriptional repressors such as LacI and GalR. Moreover, this periplasmic binding domain, also known as Venus flytrap domain, undergoes transition from an open to a closed conformational state upon the binding of ligands such as lactose, ribose, fructose, xylose, arabinose, galactose/glucose, and other sugars. This family also includes the periplasmic binding domain of autoinducer-2 (AI-2) receptors such as LsrB and LuxP which are highly homologous to periplasmic pentose/hexose sugar-binding proteins.</td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06309">cd06309</a></td><td>273</td><td>85</td><td>33.9</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP1_YtfQ_like</td><td>Periplasmic binding domain of ABC-type YtfQ-like transport systems. Periplasmic binding domain of ABC-type YtfQ-like transport systems. The YtfQ protein from Escherichia coli is up-regulated under glucose-limited conditions and shares homology with a family of pentose/hexose sugar-binding proteins of the type I periplasmic binding protein superfamily. Members of this group are predicted to be involved in the transport of sugar-containing molecules across cellular and organellar membranes; however their ligand specificity is not determined experimentally.</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07503">PRK07503</a></td><td>403</td><td>60</td><td>33.6</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07503</td><td>methionine gamma-lyase; Provisional</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0707">COG0707</a></td><td>357</td><td>32</td><td>33.5</td><td>2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MurG</td><td>UDP-N-acetylglucosamine:LPS N-acetylglucosamine transferase </td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06355">cd06355</a></td><td>348</td><td>67</td><td>32.0</td><td>90</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP1_FmdD_like</td><td>Periplasmic component (FmdD) of an active transport system for short-chain amides and urea (FmdDEF). This group includes the periplasmic component (FmdD) of an active transport system for short-chain amides and urea (FmdDEF), found in Methylophilus methylotrophus, and its homologs from other bacteria. FmdD, a type I periplasmic binding protein, is induced by short-chain amides and urea and repressed by excess ammonia, while FmdE and FmdF are hydrophobic transmembrane proteins. FmdDEF is predicted to be an ATP-dependent transporter and closely resembles the periplasmic binding protein and the two transmembrane proteins present in various hydrophobic amino acid-binding transport systems.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09455">pfam09455</a></td><td>372</td><td>74</td><td>31.6</td><td>2.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_DxTHG</td><td>CRISPR-associated (Cas) DxTHG family. CRISPR is a term for Clustered Regularly Interspaced Short Palidromic Repeats. A number of protein families appear only in association with these repeats and are designated Cas (CRISPR associated) proteins. The family describes Cas proteins of about 400 residues that include the motif </td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06331">cd06331</a></td><td>333</td><td>65</td><td>31.4</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP1_AmiC_like</td><td>Type I periplasmic components of amide-binding protein (AmiC) and the active transport system for short-chain and urea (FmdDEF). This group includes the type I periplasmic components of amide-binding protein (AmiC) and the active transport system for short-chain and urea (FmdDEF), found in bacteria and Archaea. AmiC controls expression of the amidase operon by a ligand-triggered conformational switch. In the absence of ligand or presence of butyramide (repressor), AmiC (the ligand sensor and negative regulator) adopts an open conformation and inhibits the transcription antitermination function of AmiR by direct protein-protein interaction.  In the presence of inducing ligands such as acetamide, AmiC adopts a closed conformation which disrupts a silencing AmiC-AmiR complex and the expression of amidase and other genes of the operon is induced. FmdDEF is predicted to be an ATP-dependent transporter and closely resembles the periplasmic binding protein and the two transmembrane proteins present in various hydrophobic amino acid-binding transport systems.</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13433">pfam13433</a></td><td>363</td><td>66</td><td>30.2</td><td>80</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peripla_BP_5</td><td>Periplasmic binding protein domain. This domain is found in a variety of bacterial periplasmic binding proteins.</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04565">pfam04565</a></td><td>68</td><td>44</td><td>29.0</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNA_pol_Rpb2_3</td><td>RNA polymerase Rpb2, domain 3. RNA polymerases catalyse the DNA dependent polymerization of RNA. Prokaryotes contain a single RNA polymerase compared to three in eukaryotes (not including mitochondrial. and chloroplast polymerases). Domain 3, s also known as the fork domain and is proximal to catalytic site.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09732">cd09732</a></td><td>221</td><td>72</td><td>28.6</td><td>3.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as TM1812 family</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03273">cd03273</a></td><td>251</td><td>32</td><td>28.4</td><td>69</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC2_euk</td><td>ATP-binding cassette domain of eukaryotic SMC2 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10683">pfam10683</a></td><td>68</td><td>43</td><td>28.0</td><td>88</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DBD_Tnp_Hermes</td><td>Hermes transposase DNA-binding domain. This domain confers specific DNA-binding on Hermes transposase.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03237">cd03237</a></td><td>246</td><td>66</td><td>28.0</td><td>3.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_RNaseL_inhibitor_domain2</td><td>The ATP-binding cassette domain 2 of RNase L inhibitor. The ABC ATPase, RNase L inhibitor (RLI), is a key enzyme in ribosomal biogenesis, formation of translation preinitiation complexes, and assembly of HIV capsids. RLI's are not transport proteins and thus cluster with a group of soluble proteins that lack the transmembrane components commonly found in other members of the family. Structurally, RLI's have an N-terminal Fe-S domain and two nucleotide-binding domains which are arranged to form two composite active sites in their interface cleft. RLI is one of the most conserved enzymes between archaea and eukaryotes with a sequence identity of more than 48%. The high degree of evolutionary conservation suggests that RLI performs a central role in archaeal and eukaryotic physiology.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1196">COG1196</a></td><td>1163</td><td>33</td><td>25.0</td><td>63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Smc</td><td>Chromosome segregation ATPase </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04283">TIGR04283</a></td><td>220</td><td>26</td><td>24.7</td><td>50</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>glyco_like_mftF</td><td>transferase 2, rSAM/selenodomain-associated. This enzyme may transfer a nucleotide, or it sugar moiety, as part of a biosynthetic pathway. Other proposed members of the pathway include another transferase (TIGR04282), a phosphoesterase, and a radical SAM enzyme (TIGR04167) whose C-terminal domain (pfam12345) frequently contains a selenocysteine. </td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07524">PRK07524</a></td><td>535</td><td>67</td><td>24.7</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07524</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09267">PRK09267</a></td><td>169</td><td>25</td><td>23.8</td><td>84</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09267</td><td>flavodoxin FldA; Validated</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13458">pfam13458</a></td><td>343</td><td>44</td><td>23.2</td><td>2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peripla_BP_6</td><td>Periplasmic binding protein. This family includes a diverse range of periplasmic binding proteins.</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04028">pfam04028</a></td><td>74</td><td>47</td><td>23.0</td><td>2.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF374</td><td>Domain of unknown function (DUF374). Bacterial domain of unknown function.</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04400">cd04400</a></td><td>190</td><td>71</td><td>22.8</td><td>2.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RhoGAP_fBEM3</td><td>RhoGAP_fBEM3: RhoGAP (GTPase-activator </td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15643">pfam15643</a></td><td>100</td><td>20</td><td>22.3</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Tox-PL-2</td><td>Papain fold toxin 2. A papain fold toxin domain found in bacterial polymorphic toxin systems.</td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06379">cd06379</a></td><td>377</td><td>44</td><td>21.7</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP1_iGluR_NMDA_NR1</td><td>N-terminal leucine/isoleucine/valine-binding protein (LIVBP)-like domain of the NR1, an essential channel-forming subunit of the NMDA receptor. N-terminal leucine/isoleucine/valine-binding protein (LIVBP)-like domain of the NR1, an essential channel-forming subunit of the NMDA receptor. The ionotropic N-methyl-d-asparate (NMDA) subtype of glutamate receptor serves critical functions in neuronal development, functioning, and degeneration in the mammalian central nervous system. The functional NMDA receptor is a heterotetramer ccomposed of two NR1 and two NR2 (A, B, C, and D) or of NR3 (A and B) subunits.  The receptor controls a cation channel that is highly permeable to monovalent ions and calcium and exhibits voltage-dependent inhibition by magnesium. Dual agonists, glutamate and glycine, are required for efficient activation of the NMDA receptor.  When co-expressed with NR1, the NR3 subunits form receptors that are activated by glycine alone and therefore can be classified as excitatory glycine receptors. NR1/NR3 receptors are calcium-impermeable and unaffected by ligands acting at the NR2 glutamate-binding site</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01548">pfam01548</a></td><td>143</td><td>21</td><td>21.1</td><td>2.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DEDD_Tnp_IS110</td><td>Transposase. Transposase proteins are necessary for efficient DNA transposition. This family includes an amino-terminal region of the pilin gene inverting protein (PIVML) and members of the IS111A/IS1328/IS1533 family of transposases.</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03274">cd03274</a></td><td>212</td><td>36</td><td>20.9</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC4_euk</td><td>ATP-binding cassette domain of eukaryotic SMC4 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11910">pfam11910</a></td><td>67</td><td>28</td><td>20.8</td><td>97</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NdhO</td><td>Cyanobacterial and plant NDH-1 subunit O. The proton-pumping NADH:ubiquinone oxidoreductase catalyzes the electron transfer from NADH to ubiquinone linked with proton translocation across the membrane. It is the largest, most complex and least understood of the respiratory chain enzymes and is referred to as Complex I. The subunit composition of the enzyme varies between groups of organisms. Complex I originating from mammalian mitochondria contains 45 different proteins, whereas in bacteria, the corresponding complex NDH-1 consists of 14 different polypeptides. Homologues of these 14 proteins are found among subunits of the mitochondrial complex I, and therefore bacterial NDH-1 might be considered a model proton-pumping NADH dehydrogenase with a minimal set of subunits. Escherichia coli NDH-1 readily disintegrates into 3 subcomplexes: a water-soluble NADH dehydrogenase fragment (NuoE, -F, and -G),the connecting fragment (NuoB, -C, -D, and -I), and the membrane fragment (NuoA, -H, -J, -K, -L, -M, -N). In cyanobacteria and their descendants, the chloroplasts of green plants, the subunit composition of NDH-1 remains obscure. The genes for eleven subunits NdhA-NdhK, homologous to the NuoA-NuoD and NuoH-NuoN of the Escherichia coli complex, have been found in the genome of Synechocystis sp. PCC 6803 which has a family of 6 ndhD genes and a family of 3 ndhF genes. Two reported multisubunit complexes, NDH-1L and NDH-1M, represent distinct NDH-1 complexes in the thylakoid membrane of Synechocystis 6803 -cyanobacterium. NDH-1L was shown to be essential for photoheterotrophic cell growth, whereas expression of NDH-1M was a prerequisite for CO2 uptake and played an important role in growth of cells at low CO2. Here we report the subunit composition of these two complexes. Fifteen proteins were discovered in NDH-1L including NdhL, a new component of the membrane fragment, and Ssl1690 (designated as NdhO), a novel peripheral subunit. The three nuclear-encoded subunits NdhM,NdhN and NdhO are vital for the functional integrity of the plastidial complex.</td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05355">cd05355</a></td><td>270</td><td>112</td><td>20.1</td><td>5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SDR_c1</td><td>classical (c) SDR, subgroup 1. These proteins are members of the classical SDR family, with a canonical active site tetrad and a typical Gly-rich NAD-binding motif. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRs are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes catalyze a wide range of activities including the metabolism of steroids, cofactors, carbohydrates, lipids, aromatic compounds, and amino acids, and act in redox sensing. Classical SDRs have an TGXXX</td></tr>
</table>
</body>
