<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14076">pfam14076</a></td><td>73</td><td>27</td><td>62.5</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4258</td><td>Domain of unknown function (DUF4258). This family of proteins is functionally uncharacterized. This family of proteins is found in bacteria, archaea and eukaryotes. Proteins in this family are typically between 95 and 124 amino acids in length.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00965">pfam00965</a></td><td>181</td><td>52</td><td>51.3</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------]</td><td>TIMP</td><td>Tissue inhibitor of metalloproteinase. Members of this family are common in extracellular regions of vertebrate species</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11467">pfam11467</a></td><td>106</td><td>66</td><td>48.0</td><td>7.1</td><td>[----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LEDGF</td><td>Lens epithelium-derived growth factor (LEDGF). LEDGF is a chromatin-associated protein that protects cells from stress-induced apoptosis. It is the binding partner of HIV-1 integrase in human cells. The integrase binding domain (IBD) of LEDGF is a compact right-handed bundle composed of five alpha-helices. The residues essential for the interaction with the integrase are present in the inter-helical loop regions of the bundle structure.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14779">cd14779</a></td><td>140</td><td>26</td><td>42.0</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FHP_Ae-globin_like</td><td>Globin domain of Alcaligenes eutrophus flavohemoglobin (FHP) and related proteins. Flavohemoglobins (flavoHbs) function primarily as nitric oxide dioxygenases (NODs, EC 1.14.12.17), converting NO and O2 to inert NO3- (nitrate). They have an N-terminal globin domain and a C-terminal ferredoxin reductase-like NAD- and FAD-binding domain, and use the reducing power of cellular NAD(P)H to drive regeneration of the ferrous heme. They protect from nitrosative stress (the broad range of cellular toxicities caused by NO), and modulate NO signaling pathways. NO scavenging by flavoHb maintains Medicago truncatula-Sinorhizobium meliloti symbiosis. Alcaligenes eutrophus FHP contains a phospholipid-binding site.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04699">pfam04699</a></td><td>153</td><td>17</td><td>24.7</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>P16-Arc</td><td>ARP2/3 complex 16 kDa subunit (p16-Arc). The Arp2/3 protein complex has been implicated in the control of actin polymerization. The human complex consists of seven subunits which include the actin related proteins Arp2 and Arp3, and five others referred to as p41-Arc, p34-Arc, p21-Arc, p20-Arc, and p16-Arc. The precise function of p16-Arc is currently unknown. Its structure consists of a single domain containing a bundle of seven alpha helices.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14776">cd14776</a></td><td>138</td><td>28</td><td>24.5</td><td>88</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HmpEc-globin_like</td><td>Globin domain of Escherichia coli flavohemoglobin (Hmp) and related proteins. Flavohemoglobins (flavoHbs) function primarily as nitric oxide dioxygenases (NODs, EC 1.14.12.17), converting NO and O2 to inert NO3- (nitrate). They have an N-terminal globin domain and a C-terminal ferredoxin reductase-like NAD- and FAD-binding domain, and use the reducing power of cellular NAD(P)H to drive regeneration of the ferrous heme. They protect from nitrosative stress (the broad range of cellular toxicities caused by NO), and modulate NO signaling pathways. This subfamily includes Vibrio fischeri Hmp and E.coli Hmp. NO scavenging by flavoHb affects the swarming behavior of Escherichia coli, and protects against NO during initiation of the squid-Vibrio symbiosis. E.coli Hmp can catalyze the reduction of several alkylhydroperoxide substrates into their corresponding alcohols using NADH as an electron donor, and it has been suggested that it participates in the repair of the lipid membrane oxidative damage generated during oxidative/nitrosative stress.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12509">cd12509</a></td><td>81</td><td>18</td><td>22.7</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM3_ESRPs_Fusilli</td><td>RNA recognition motif 3 in epithelial splicing regulatory protein ESRP1, ESRP2, Drosophila RNA-binding protein Fusilli and similar proteins. This subfamily corresponds to the RRM3 of ESRPs and Fusilli. ESRP1 (also termed RBM35A) and ESRP2 (also termed RBM35B) are epithelial-specific RNA binding proteins that promote splicing of the epithelial variant of the fibroblast growth factor receptor 2 (FGFR2), ENAH (also termed hMena), CD44 and CTNND1 (also termed p120-Catenin) transcripts. They are highly conserved paralogs and specifically bind to GU-rich binding site. ESRP1 and ESRP2 contain three RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains). The family also includes Drosophila fusilli (fus) gene encoding RNA-binding protein Fusilli. Loss of fusilli activity causes lethality during embryogenesis in flies. Drosophila Fusilli can regulate endogenous FGFR2 splicing and functions as a splicing factor. Fusilli shows high sequence homology to ESRPs and contains three RRMs as well. It also has an N-terminal domain with unknown function and a C-terminal domain particularly rich in alanine, glutamine, and serine. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07660">pfam07660</a></td><td>51</td><td>16</td><td>22.7</td><td>76</td><td>[-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>STN</td><td>Secretin and TonB N terminus short domain. This is a short domain found at the N-terminus of the Secretins of the bacterial type II/III secretory system as well as the TonB-dependent receptor proteins. These proteins are involved in TonB-dependent active uptake of selective substrates.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11195">pfam11195</a></td><td>75</td><td>18</td><td>22.0</td><td>43</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2829</td><td>Protein of unknown function (DUF2829). This is a uncharacterized family of proteins found in bacteria and bacteriphages.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04104">pfam04104</a></td><td>217</td><td>53</td><td>19.6</td><td>1.2E+02</td><td>[----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DNA_primase_lrg</td><td>Eukaryotic and archaeal DNA primase, large subunit. DNA primase is the polymerase that synthesizes small RNA primers for the Okazaki fragments made during discontinuous DNA replication. DNA primase is a heterodimer of two subunits, the small subunit Pri1 (48 kDa in yeast), and the large subunit Pri2 (58 kDa in the yeast S. cerevisiae). The large subunit of DNA primase forms interactions with the small subunit and the structure implicates that it is not directly involved in catalysis, but plays roles in correctly positioning the primase/DNA complex, and in the transfer of RNA to DNA polymerase.</td></tr>
</table>
</body>
