<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09652">pfam09652</a></td><td>93</td><td>92</td><td>100.0</td><td>7.8E-56</td><td>[&nbsp;&nbsp;-----------------------------------------------]</td><td>Cas_VVA1548</td><td>Putative CRISPR-associated protein (Cas_VVA1548). This entry represents a conserved region of about 95 amino acids found exclusively in species with CRISPRs (Clustered Regularly Interspaced Short Palindromic Repeats). In all bacterial species that contain this entry, the genes encoding the proteins are in the midst of a cluster of cas (CRISPR-associated) genes.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09743">cd09743</a></td><td>90</td><td>90</td><td>100.0</td><td>1.9E-55</td><td>[&nbsp;&nbsp;----------------------------------------------&nbsp;]</td><td>Csx16_III-U</td><td>CRISPR/Cas system-associated protein Csx16. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Small protein often seen in proximity to Csx1 (COG1517) family; also known as VVA1548 family</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02620">TIGR02620</a></td><td>93</td><td>92</td><td>100.0</td><td>2E-46</td><td>[&nbsp;&nbsp;-----------------------------------------------]</td><td>cas_VVA1548</td><td>putative CRISPR-associated protein, VVA1548 family. This model represents a conserved domain of about 95 amino acids exclusively in species with CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats). In all bacterial species with members so far (Vibrio vulnificus YJ016, Mannheimia succiniciproducens MBEL55E, and Nitrosomonas europaea ATCC 19718) and but not in the archaeon Methanothermobacter thermautotrophicus str. Delta H, the gene for this protein is in the midst of a cluster of Cas protein gene near CRISPR repeats.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09695">cd09695</a></td><td>76</td><td>76</td><td>100.0</td><td>8.9E-34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------]</td><td>Csx16_III-U</td><td>CRISPR/Cas system-associated protein Csx16. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Small protein often seen in proximity to Csx1 (COG1517) family; also known as VVA1548 family</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02639">pfam02639</a></td><td>130</td><td>43</td><td>88.6</td><td>0.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF188</td><td>Uncharacterized BCR, YaiI/YqxD family COG1671. </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06953">pfam06953</a></td><td>121</td><td>29</td><td>69.5</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArsD</td><td>Arsenical resistance operon trans-acting repressor ArsD. This family consists of several bacterial arsenical resistance operon trans-acting repressor ArsD proteins. ArsD is a trans-acting repressor of the arsRDABC operon that confers resistance to arsenicals and antimonials in Escherichia coli. It possesses two-pairs of vicinal cysteine residues, Cys(12)-Cys(13) and Cys(112)-Cys(113), that potentially form separate binding sites for the metalloids that trigger dissociation of ArsD from the operon. However, as a homodimer it has four vicinal cysteine pairs.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03807">pfam03807</a></td><td>93</td><td>51</td><td>66.0</td><td>22</td><td>[&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_oxidored</td><td>NADP oxidoreductase coenzyme F420-dependent. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0394">COG0394</a></td><td>139</td><td>48</td><td>65.0</td><td>4.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Wzb</td><td>Protein-tyrosine-phosphatase </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15508">pfam15508</a></td><td>94</td><td>21</td><td>58.1</td><td>3.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NAAA-beta</td><td>beta subunit of N-acylethanolamine-hydrolyzing acid amidase. NAAA-beta is a family of vertebral sequences that form the beta subunit of vertebral N-acylethanolamine-hydrolyzing acid amidase, a member of the choloylglycine hydrolase acid ceramidase family. The alpha subunit is represented by family CBAH, pfam02275.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08506">PRK08506</a></td><td>472</td><td>38</td><td>58.0</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08506</td><td>replicative DNA helicase; Provisional</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00402">PRK00402</a></td><td>418</td><td>32</td><td>57.1</td><td>4.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00402</td><td>3-isopropylmalate dehydratase large subunit; Reviewed</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08220">pfam08220</a></td><td>57</td><td>29</td><td>56.8</td><td>5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_DeoR</td><td>DeoR-like helix-turn-helix domain. </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10787">pfam10787</a></td><td>149</td><td>17</td><td>56.7</td><td>5.4</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YfmQ</td><td>Uncharacterized protein from bacillus cereus group. This family is conserved in the Bacillus cereus group. Several members are called YfmQ but the function is not known.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00977">pfam00977</a></td><td>229</td><td>42</td><td>55.9</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>His_biosynth</td><td>Histidine biosynthesis protein. Proteins involved in steps 4 and 6 of the histidine biosynthesis pathway are contained in this family. Histidine is formed by several complex and distinct biochemical reactions catalysed by eight enzymes. The enzymes in this Pfam entry are called His6 and His7 in eukaryotes and HisA and HisF in prokaryotes. The structure of HisA is known to be a TIM barrel fold. In some archaeal HisA proteins the TIM barrel is composed of two tandem repeats of a half barrel. This family belong to the common phosphate binding site TIM barrel family.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05710">cd05710</a></td><td>120</td><td>42</td><td>53.4</td><td>9</td><td>[&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SIS_1</td><td>A subgroup of the SIS domain. SIS (Sugar ISomerase) domains are found in many phosphosugar isomerases and phosphosugar binding proteins. SIS domains are also found in proteins that regulate the expression of genes involved in synthesis of phosphosugars.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06264">PRK06264</a></td><td>210</td><td>62</td><td>52.9</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cbiC</td><td>precorrin-8X methylmutase; Validated</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5661">COG5661</a></td><td>210</td><td>19</td><td>51.9</td><td>0.68</td><td>[&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG5661</td><td>Predicted secreted Zn-dependent protease  </td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09328">pfam09328</a></td><td>264</td><td>18</td><td>51.8</td><td>6.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Phytochelatin_C</td><td>Domain of unknown function (DUF1984). Members of this family of functionally uncharacterized domains are found at the C-terminus of plant phytochelatin synthases.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03168">TIGR03168</a></td><td>303</td><td>38</td><td>51.7</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>1-PFK</td><td>hexose kinase, 1-phosphofructokinase family. This family consists largely of 1-phosphofructokinases, but also includes tagatose-6-kinases and 6-phosphofructokinases.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10477">PRK10477</a></td><td>177</td><td>25</td><td>49.2</td><td>9.2</td><td>[&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10477</td><td>outer membrane lipoprotein Blc; Provisional</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01254">PRK01254</a></td><td>707</td><td>33</td><td>48.4</td><td>8.5</td><td>[&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01254</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12476">PRK12476</a></td><td>612</td><td>35</td><td>47.1</td><td>8.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK12476</td><td>putative fatty-acid--CoA ligase; Provisional</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01451">pfam01451</a></td><td>138</td><td>49</td><td>46.9</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LMWPc</td><td>Low molecular weight phosphotyrosine protein phosphatase. </td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1671">COG1671</a></td><td>150</td><td>41</td><td>46.6</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YaiI</td><td>Uncharacterized conserved protein YaiI, UPF0178 family </td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13883">pfam13883</a></td><td>167</td><td>21</td><td>45.3</td><td>1.3</td><td>[&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pyrid_oxidase_2</td><td>Pyridoxamine 5'-phosphate oxidase. </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00115">cd00115</a></td><td>141</td><td>45</td><td>45.3</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LMWPc</td><td>Substituted updates: Aug 22, 2001</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3040">COG3040</a></td><td>174</td><td>25</td><td>44.5</td><td>13</td><td>[&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Blc</td><td>Bacterial lipocalin </td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00984">cd00984</a></td><td>242</td><td>39</td><td>43.9</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DnaB_C</td><td>DnaB helicase C terminal domain. The hexameric helicase DnaB unwinds the DNA duplex at the  chromosome replication fork. Although the mechanism by which DnaB both couples ATP hydrolysis to translocation along DNA and denatures the duplex is unknown, a change in the quaternary structure of the protein involving dimerization of the N-terminal domain has been observed and may occur during the enzymatic cycle. This C-terminal domain contains an ATP-binding site and is therefore probably the site of ATP hydrolysis.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14125">PRK14125</a></td><td>103</td><td>25</td><td>42.8</td><td>7.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14125</td><td>cell division suppressor protein YneA; Provisional</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00748">PRK00748</a></td><td>233</td><td>37</td><td>42.8</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00748</td><td>1-(5-phosphoribosyl)-5-</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05789">cd05789</a></td><td>86</td><td>10</td><td>42.7</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>S1_Rrp4</td><td>S1_Rrp4: Rrp4 S1-like RNA-binding domain. S1-like RNA-binding domains are found in a wide variety of RNA-associated proteins. Rrp4 protein is a subunit of the exosome complex. The exosome plays a central role in 3' to 5' RNA processing and degradation in eukarytes and archaea. Its functions include the removal of incorrectly processed RNA and the maintenance of proper levels of mRNA, rRNA and a number of small RNA species. In Saccharomyces cerevisiae, the exosome includes nine core components, six of which are homologous to bacterial RNase PH. These form a hexameric ring structure. The other three subunits (RrP4, Rrp40, and Csl4) contain an S1 RNA binding domain and are part of the "S1 pore structure"td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0099">COG0099</a></td><td>121</td><td>39</td><td>42.4</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;]</td><td>RpsM</td><td>Ribosomal protein S13 </td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04016">pfam04016</a></td><td>147</td><td>32</td><td>39.6</td><td>71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF364</td><td>Domain of unknown function (DUF364). This domain of unknown function has a PLP-dependent transferase-like fold. Its genomic context suggests that it may have a role in anaerobic vitamin B12 biosynthesis. This domain is often found at the C-terminus of proteins containing DUF4213, pfam13938.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09542">cd09542</a></td><td>63</td><td>16</td><td>38.9</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;]</td><td>SAM_EPH-A1</td><td>SAM domain of EPH-A1 subfamily of tyrosine kinase receptors. SAM (sterile alpha motif) domain of EPH-A1 subfamily of the receptor tyrosine kinases is a C-terminal protein-protein interaction domain. This domain is located in the cytoplasmic region of EPH-A1 receptors and appears to mediate cell-cell initiated signal transduction. Activation of these receptors leads to inhibition of cell spreading and migration in a RhoA-ROCK-dependent manner. EPH-A1 receptors are known to bind ILK (integrin-linked kinase) which is the mediator of interactions between integrin and the actin cytoskeleton. However SAM is not sufficient for this interaction; it rather plays an ancillary role.  SAM domains of Eph-A1 receptors do not form homo/hetero dimers/oligomers. EphA1 gene was found expressed widely in differentiated epithelial cells. In a number of different malignant tumors EphA1 genes are downregulated. In breast carcinoma the downregulation is associated with invasive behavior of the cell.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05179">PRK05179</a></td><td>122</td><td>42</td><td>38.4</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;]</td><td>rpsM</td><td>30S ribosomal protein S13; Validated</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00124">PRK00124</a></td><td>151</td><td>42</td><td>38.4</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00124</td><td>hypothetical protein; Validated</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05943">cd05943</a></td><td>616</td><td>46</td><td>38.1</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AACS</td><td>Acetoacetyl-CoA synthetase (acetoacetate-CoA ligase, AACS). AACS is a cytosolic ligase that specifically activates acetoacetate to its coenzyme A ester by a two-step reaction. Acetoacetate first reacts with ATP to form an acyl-adenylate intermediate, which then reacts with CoA to produce an acyl-CoA ester. This is the first step of the mevalonate pathway of isoprenoid biosynthesis via isopentenyl diphosphate. Isoprenoids are a large class of compounds found in all living organisms. AACS is widely distributed in bacteria, archaea and eukaryotes. In bacteria, AACS is known to exhibit an important role in the metabolism of poly-b-hydroxybutyrate, an intracellular reserve of organic carbon and chemical energy by some microorganisms. In mammals, AACS influences the rate of ketone body utilization for the formation of physiologically important fatty acids and cholesterol.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03631">TIGR03631</a></td><td>113</td><td>42</td><td>37.4</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;]</td><td>uS13_bact</td><td>ribosomal protein uS13, bacterial form. This model describes bacterial ribosomal protein S13, to the exclusion of the homologous archaeal S13P and eukaryotic ribosomal protein S18. This model identifies some (but not all) instances of chloroplast and mitochondrial S13, which is of bacterial type. </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02153">pfam02153</a></td><td>258</td><td>54</td><td>37.3</td><td>59</td><td>[&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PDH</td><td>Prephenate dehydrogenase. Members of this family are prephenate dehydrogenases EC:1.3.1.12 involved in tyrosine biosynthesis.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02073">pfam02073</a></td><td>404</td><td>39</td><td>36.9</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_M29</td><td>Thermophilic metalloprotease (M29). </td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11739">pfam11739</a></td><td>207</td><td>32</td><td>36.4</td><td>8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DctA-YdbH</td><td>Dicarboxylate transport. In certain bacterial families this protein is expressed from the ydbH gene, and there is a suggestion that this is a form of DctA or dicarboxylate transport protein. Dicarboxylate transport proteins are found in aerobic bacteria which grow on succinate or other C4-dicarboxylates.</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02196">cd02196</a></td><td>297</td><td>33</td><td>35.4</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PurM</td><td>PurM (Aminoimidazole Ribonucleotide </td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14821">PRK14821</a></td><td>184</td><td>57</td><td>34.6</td><td>6.9</td><td>[-----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14821</td><td>putative deoxyribonucleotide triphosphate pyrophosphatase; Provisional</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02198">TIGR02198</a></td><td>315</td><td>17</td><td>34.4</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rfaE_dom_I</td><td>rfaE bifunctional protein, domain I. RfaE is a protein involved in the biosynthesis of ADP-L-glycero-D-manno-heptose, a precursor for LPS inner core biosynthesis. RfaE is a bifunctional protein in E. coli, and separate proteins in some other genome. The longer, N-terminal domain I (this family) is suggested to act in D-glycero-D-manno-heptose 1-phosphate biosynthesis, while domain II (TIGR02199) adds ADP to yield ADP-D-glycero-D-manno-heptose. </td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00631">TIGR00631</a></td><td>655</td><td>63</td><td>34.2</td><td>86</td><td>[&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>uvrb</td><td>excinuclease ABC, B subunit. All proteins in this family for wich functions are known are DNA helicases that function in the nucleotide excision repair and are endonucleases that make the 3' incision next to DNA damage. They are part of a pathway requiring UvrA, UvrB, UvrC, and UvrD homologs. This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University) </td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03755">pfam03755</a></td><td>159</td><td>29</td><td>33.8</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YicC_N</td><td>YicC-like family, N-terminal region. Family of bacterial proteins. Although poorly characterized, the members of this protein family have been demonstrated to play a role in stationary phase survival. These proteins are not essential during stationary phase.</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03584">PRK03584</a></td><td>655</td><td>34</td><td>33.6</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK03584</td><td>acetoacetyl-CoA synthetase; Provisional</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08350">pfam08350</a></td><td>64</td><td>15</td><td>33.6</td><td>29</td><td>[&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1724</td><td>Domain of unknown function (DUF1724). This domain of unknown function has so far only been found at the C-terminus of archaean proteins, including several transcriptional regulators of the ArsR family (see pfam01022).</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01164">cd01164</a></td><td>289</td><td>38</td><td>33.4</td><td>47</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FruK_PfkB_like</td><td>1-phosphofructokinase (FruK), minor 6-phosphofructokinase (pfkB) and related sugar kinases. FruK plays an important role in the predominant pathway for fructose utilisation.This group also contains tagatose-6-phophate kinase, an enzyme of the tagatose 6-phosphate pathway, which responsible for breakdown of the galactose moiety during lactose metabolism by bacteria such as L. lactis.</td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14319">pfam14319</a></td><td>111</td><td>20</td><td>32.8</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Zn_Tnp_IS91</td><td>Transposase zinc-binding domain. This domain is likely to be a zinc-binding domain. It is found at the N-terminus of transposases belonging to the IS91 family.</td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13352">PRK13352</a></td><td>431</td><td>28</td><td>32.7</td><td>38</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13352</td><td>thiamine biosynthesis protein ThiC; Provisional</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09287">PRK09287</a></td><td>459</td><td>25</td><td>32.6</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09287</td><td>6-phosphogluconate dehydrogenase; Validated</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04732">cd04732</a></td><td>234</td><td>35</td><td>32.3</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HisA</td><td>HisA.  Phosphoribosylformimino-5-aminoimidazole carboxamide ribonucleotide (ProFAR) isomerase catalyzes the fourth step in histidine biosynthesis, an isomerisation of the aminoaldose moiety of ProFAR to the aminoketose of PRFAR (N-(5'-phospho-D-1'-ribulosylformimino)-5-amino-1-(5''-phospho-ribosyl)-4-imidazolecarboxamide). In bacteria and archaea, ProFAR isomerase is encoded by the HisA gene.</td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08284">pfam08284</a></td><td>135</td><td>38</td><td>31.5</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RVP_2</td><td>Retroviral aspartyl protease. Single domain aspartyl proteases from retroviruses, retrotransposons, and badnaviruses (plant dsDNA viruses). These proteases are generally part of a larger polyprotein; usually pol, more rarely gag. Retroviral proteases appear to be homologous to a single domain of the two-domain eukaryotic aspartyl proteases</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07368">cd07368</a></td><td>277</td><td>43</td><td>31.4</td><td>3.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PhnC_Bs_like</td><td>PhnC is a Class III Extradiol ring-cleavage dioxygenase involved in the polycyclic aromatic hydrocarbon (PAH) catabolic pathway. This subfamily is composed of Burkholderia sp. PhnC and similar poteins. PhnC is one of nine protein products encoded by the phn locus. These proteins are involved in the polycyclic aromatic hydrocarbon (PAH) catabolic pathway. PhnC is a member of the class III extradiol dioxygenase family, a group os enzymes which use a non-heme Fe(II) to cleave aromatic rings between a hydroxylated carbon and an adjacent non-hydroxylated carbon. LigAB-like enzymes are usually composed of two subunits, designated A and B, which form a tetramer composed of two copies of each subunit. This model represents the catalytic subunit, B.</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1469">COG1469</a></td><td>289</td><td>17</td><td>31.2</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FolE2</td><td>GTP cyclohydrolase FolE2 </td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03904">TIGR03904</a></td><td>559</td><td>30</td><td>31.0</td><td>27</td><td>[&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SAM_YgiQ</td><td>uncharacterized radical SAM protein YgiQ. Members of this family are fairly widespread uncharacterized radical SAM family proteins, many of which are designated YgiQ. </td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06413">cd06413</a></td><td>191</td><td>17</td><td>30.9</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GH25_muramidase_1</td><td>Uncharacterized bacterial muramidase containing a glycosyl hydrolase family 25 (GH25) catalytic domain.  Endo-N-acetylmuramidases are lysozymes (also referred to as peptidoglycan hydrolases) that degrade bacterial cell walls by catalyzing the hydrolysis of 1,4-beta-linkages between N-acetylmuramic acid and N-acetyl-D-glucosamine residues.</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13815">pfam13815</a></td><td>118</td><td>32</td><td>30.9</td><td>23</td><td>[&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Dzip-like_N</td><td>Iguana/Dzip1-like DAZ-interacting protein N-terminal. The DAZ gene-product - Deleted in Azoospermia - and a closely related sequence are required early in germ-cell development in order to maintain germ-cell populations. This family is the N-terminal region that is the only part of the protein in some fungi and lower metazoa.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00665">TIGR00665</a></td><td>432</td><td>38</td><td>30.5</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DnaB</td><td>replicative DNA helicase. This model describes the helicase DnaB, a homohexameric protein required for DNA replication. The homohexamer can form a ring around a single strand of DNA near a replication fork. An intein of > 400 residues is found at a conserved location in DnaB of Synechocystis PCC6803, Rhodothermus marinus (both experimentally confirmed), and Mycobacterium tuberculosis. The intein removes itself by a self-splicing reaction. The seed alignment contains inteins so that the model built from the seed alignment will model a low cost at common intein insertion sites. </td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02194">TIGR02194</a></td><td>72</td><td>17</td><td>30.2</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GlrX_NrdH</td><td>Glutaredoxin-like protein NrdH. NrdH-redoxin is a representative of a class of small redox proteins that contain a conserved CXXC motif and are characterized by a glutaredoxin-like amino acid sequence and thioredoxin-like activity profile. Unlike other the glutaredoxins to which it is most closely related, NrdH aparrently does not interact with glutathione/glutathione reductase, but rather with thioredoxin reductase to catalyze the reduction of ribonucleotide reductase.</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02887">pfam02887</a></td><td>117</td><td>10</td><td>30.1</td><td>61</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PK_C</td><td>Pyruvate kinase, alpha/beta domain. As well as being found in pyruvate kinase this family is found as an isolated domain in some bacterial proteins.</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01048">pfam01048</a></td><td>228</td><td>15</td><td>29.4</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PNP_UDP_1</td><td>Phosphorylase superfamily. Members of this family include: purine nucleoside phosphorylase (PNP) Uridine phosphorylase (UdRPase) 5'-methylthioadenosine phosphorylase (MTA phosphorylase)</td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02607">TIGR02607</a></td><td>78</td><td>38</td><td>28.9</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>antidote_HigA</td><td>addiction module antidote protein, HigA family. Members of this family form a distinct clade within the larger family HTH_3 of helix-turn-helix proteins, described by pfam01381. Members of this clade are strictly bacterial and nearly always shorter than 110 amino acids. This family includes the characterized member HigA, without which the killer protein HigB cannot be cloned. The hig (host inhibition of growth) system is noted to be unusual in that killer protein is uncoded by the upstream member of the gene pair. </td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12176">cd12176</a></td><td>304</td><td>28</td><td>28.6</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PGDH_3</td><td>Phosphoglycerate dehydrogenases, NAD-binding and catalytic domains. Phosphoglycerate dehydrogenases (PGDHs) catalyze the initial step in the biosynthesis of L-serine from D-3-phosphoglycerate. PGDHs come in 3 distinct structural forms, with this first group being related to 2-hydroxy acid dehydrogenases, sharing structural similarity to formate and glycerate dehydrogenases. PGDH in E. coli and Mycobacterium tuberculosis form tetramers, with subunits containing a Rossmann-fold NAD binding domain. Formate/glycerate and related dehydrogenases of the D-specific 2-hydroxyacid dehydrogenase superfamily include groups such as formate dehydrogenase, glycerate dehydrogenase, L-alanine dehydrogenase, and S-adenosylhomocysteine hydrolase. Despite often low sequence identity, these proteins typically have a characteristic arrangement of 2 similar subdomains of the alpha/beta Rossmann fold NAD+ binding form. The NAD+ binding domain is inserted within the linear sequence of the mostly N-terminal catalytic domain, which has a similar domain structure to the internal NAD binding domain. Structurally, these domains are connected by extended alpha helices and create a cleft in which NAD is bound, primarily to the C-terminal portion of the 2nd (internal) domain. Some related proteins have similar structural subdomain but with a tandem arrangement of the catalytic and NAD-binding subdomains in the linear sequence.</td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01096">cd01096</a></td><td>315</td><td>19</td><td>27.8</td><td>48</td><td>[&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Alkanal_monooxygenase</td><td>Alkanal monooxygenase are flavin monoxygenases. Molecular oxygen is activated by reaction with reduced flavin mononucleotide (FMNH2) and reacts with an aldehyde to yield the carboxylic acid, oxidized flavin (FMN) and a blue-green light. Bacterial luciferases are heterodimers made of alpha and beta subunits which are homologous. The single activer center is on the alpha subunit. The alpha subunit has a stretch of 30 amino acid residues that is not present in the beta subunit. The beta subunit does not contain the active site and is required for the formation of the fully active heterodimer. The beta subunit does not contribute anything directly to the active site. Its role is probably to stabilize the high quantum yield conformation of the alpha subunit through interactionbs across the subunit interface.</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11316">PRK11316</a></td><td>473</td><td>17</td><td>27.3</td><td>8.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11316</td><td>bifunctional heptose 7-phosphate kinase/heptose 1-phosphate adenyltransferase; Provisional</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08245">PRK08245</a></td><td>240</td><td>23</td><td>26.9</td><td>57</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08245</td><td>hypothetical protein; Validated</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07338">pfam07338</a></td><td>56</td><td>18</td><td>26.8</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1471</td><td>Protein of unknown function (DUF1471). This family consists of several hypothetical Enterobacterial proteins of around 90 residues in length. Some members of this family are annotated as ydgH precursors and contain two copies of this region, one at the N-terminus and the other at the C-terminus. The function of this family is unknown.</td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05385">PRK05385</a></td><td>327</td><td>42</td><td>26.7</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05385</td><td>phosphoribosylaminoimidazole synthetase; Provisional</td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4821">COG4821</a></td><td>243</td><td>34</td><td>26.6</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4821</td><td>Uncharacterized protein, contains SIS (Sugar ISomerase) phosphosugar binding domain  </td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08030">pfam08030</a></td><td>152</td><td>16</td><td>26.4</td><td>47</td><td>[&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NAD_binding_6</td><td>Ferric reductase NAD binding domain. </td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13463">PRK13463</a></td><td>203</td><td>14</td><td>26.1</td><td>35</td><td>[&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13463</td><td>phosphatase PhoE; Provisional</td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11883">PRK11883</a></td><td>451</td><td>75</td><td>25.9</td><td>23</td><td>[&nbsp;&nbsp;----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11883</td><td>protoporphyrinogen oxidase; Reviewed</td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13200">cd13200</a></td><td>109</td><td>27</td><td>25.8</td><td>7.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FERM_C_KCBP</td><td>FERM domain C-lobe of Kinesin-like calmodulin binding protein. KCBPs (also called KIPK/Kinesin-like Calmodulin-Binding Protein-Interacting Protein Kinase), a member of the Kinesin-14 family, is a C-terminal microtubule motor with three unique domains including a myosin tail homology region 4 (MyTH4), a talin-like domain, and a calmodulin-binding domain (CBD). Binding of the Ca2+-activated calmodulin to KCBP causes the motor to dissociate from microtubules. The microtubule binding of KCBP is controlled by the calcium binding protein KIC containing a single EF-hand motif. KCBPs are unique to land plants and green algae. The MyTH4 and talin-like domains are not found in other kinesins, while the CBD domain is also only found in Strongylocentrotus purpuratus kinesin-C (SpKinC). The FERM domain has a cloverleaf tripart structure composed of: (1) FERM_N (A-lobe or F1); (2) FERM_M (B-lobe, or F2); and (3) FERM_C (C-lobe or F3). The C-lobe/F3 within the FERM domain is part of the PH domain family. The FERM domain is found in the cytoskeletal-associated proteins such as ezrin, moesin, radixin, 4.1R, and merlin. These proteins provide a link between the membrane and cytoskeleton and are involved in signal transduction pathways. The FERM domain is also found in protein tyrosine phosphatases (PTPs), the tyrosine kinases FAK and JAK, in addition to other proteins involved in signaling. This domain is structurally similar to the PH and PTB domains and consequently is capable of binding to both peptides and phospholipids at different sites.</td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1240">COG1240</a></td><td>261</td><td>29</td><td>25.7</td><td>63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ChlD</td><td>Mg-chelatase subunit ChlD </td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09521">PRK09521</a></td><td>189</td><td>14</td><td>25.1</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09521</td><td>exosome complex RNA-binding protein Csl4; Provisional</td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00026">PRK00026</a></td><td>244</td><td>47</td><td>25.1</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>trmD</td><td>tRNA (guanine-N(1)-)-methyltransferase; Reviewed</td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08267">cd08267</a></td><td>319</td><td>28</td><td>25.0</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MDR1</td><td>Medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family. This group is a member of the medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, but lacks the zinc-binding sites of the zinc-dependent alcohol dehydrogenases. The medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, which contains the zinc-dependent alcohol dehydrogenase (ADH-Zn) and related proteins, is a diverse group of proteins related to the first identified member, class I mammalian ADH.  MDRs display a broad range of activities and are distinguished from the smaller short chain dehydrogenases (~ 250 amino acids vs. the ~ 350 amino acids of the MDR).  The MDR proteins have 2 domains: a C-terminal NAD(P)-binding Rossmann fold domain of a beta-alpha form and an N-terminal catalytic domain with distant homology to GroES.  The MDR group contains a host of activities, including the founding alcohol dehydrogenase (ADH), quinone reductase, sorbitol dehydrogenase, formaldehyde dehydrogenase, butanediol DH, ketose reductase, cinnamyl reductase, and numerous others. The zinc-dependent alcohol dehydrogenases (ADHs) catalyze the  NAD(P)(H)-dependent interconversion of alcohols to aldehydes or ketones.  Active site zinc has a catalytic role, while structural zinc aids in stability.  ADH-like proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and generally have 2 tightly bound zinc atoms per subunit. The active site zinc is coordinated by a histidine, two cysteines, and a water molecule. The second zinc seems to play a structural role, affects subunit interactions, and is typically coordinated by 4 cysteines.</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0150">COG0150</a></td><td>345</td><td>29</td><td>24.8</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PurM</td><td>Phosphoribosylaminoimidazole (AIR) synthetase </td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03293">pfam03293</a></td><td>160</td><td>13</td><td>24.8</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pox_RNA_pol</td><td>Poxvirus DNA-directed RNA polymerase, 18 kD subunit. </td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02819">TIGR02819</a></td><td>393</td><td>51</td><td>24.5</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fdhA_non_GSH</td><td>formaldehyde dehydrogenase, glutathione-independent. Members of this family represent a distinct clade within the larger family of zinc-dependent dehydrogenases of medium chain alcohols, a family that also includes the so-called glutathione-dependent formaldehyde dehydrogenase. Members of this protein family have a tightly bound NAD that can act as a true cofactor, rather than a cosubstrate in dehydrogenase reactions, in dismutase reactions for some aldehydes. The name given to this family, however, is formaldehyde dehydrogenase, glutathione-independent. </td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04454">cd04454</a></td><td>82</td><td>10</td><td>24.5</td><td>42</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>S1_Rrp4_like</td><td>S1_Rrp4_like: Rrp4-like, S1-like RNA-binding domain. S1-like RNA-binding domains are found in a wide variety of RNA-associated proteins. Rrp4 protein, and Rrp40 and Csl4 proteins, also represented in this group, are subunits of the exosome complex. The exosome plays a central role in 3' to 5' RNA processing and degradation in eukarytes and archaea. Its functions include the removal of incorrectly processed RNA and the maintenance of proper levels of mRNA, rRNA and a number of small RNA species. In Saccharomyces cerevisiae, the exosome includes nine core components, six of which are homologous to bacterial RNase PH. These form a hexameric ring structure. The other three subunits (RrP4, Rrp40, and Csl4) contain an S1 RNA binding domain and are part of the "S1 pore structure"td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10952">pfam10952</a></td><td>140</td><td>13</td><td>24.3</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2753</td><td>Protein of unknown function (DUF2753). This bacterial family of proteins has no known function.</td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13146">pfam13146</a></td><td>78</td><td>27</td><td>24.2</td><td>68</td><td>[&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TRL</td><td>TRL-like protein family. This family includes the TRL protein, which is found in a locus that includes several tRNAs. The function of this protein is not known. The proteins in this family usually have a lipoprotein attachment site at their N-terminus.</td></tr>
<tr><td>86</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02228">TIGR02228</a></td><td>158</td><td>36</td><td>23.5</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>sigpep_I_arch</td><td>signal peptidase I, archaeal type. This model represents signal peptidase I from most archaea, a subunit of the eukaryotic endoplasmic reticulum signal peptidase I complex, and an apparent signal peptidase I from a small number of bacteria. It is related to but does not overlap in hits with TIGR02227, the bacterial and mitochondrial signal peptidase I.</td></tr>
<tr><td>87</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03828">TIGR03828</a></td><td>304</td><td>37</td><td>22.9</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pfkB</td><td>1-phosphofructokinase. This enzyme acts in concert with the fructose-specific phosphotransferase system (PTS) which imports fructose as fructose-1-phosphate. The action of 1-phosphofructokinase results in beta-D-fructose-1,6-bisphosphate and is an entry point into glycolysis (GenProp0688).</td></tr>
<tr><td>88</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01476">pfam01476</a></td><td>43</td><td>29</td><td>22.9</td><td>74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LysM</td><td>LysM domain. The LysM (lysin motif) domain is about 40 residues long. It is found in a variety of enzymes involved in bacterial cell wall degradation. This domain may have a general peptidoglycan binding function. The structure of this domain is known.</td></tr>
<tr><td>89</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01964">pfam01964</a></td><td>421</td><td>28</td><td>22.8</td><td>73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ThiC</td><td>ThiC family. ThiC is found within the thiamine biosynthesis operon. ThiC is involved in pyrimidine biosynthesis. The precise catalytic function of ThiC is still not known. ThiC participates in the formation of 4-Amino-5-hydroxymethyl-2-methylpyrimidine from AIR, an intermediate in the de novo pyrimidine biosynthesis.</td></tr>
<tr><td>90</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02947">PRK02947</a></td><td>246</td><td>34</td><td>22.6</td><td>50</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK02947</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>91</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00809">PRK00809</a></td><td>144</td><td>41</td><td>22.5</td><td>86</td><td>[---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00809</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>92</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03111">cd03111</a></td><td>106</td><td>28</td><td>22.5</td><td>79</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CpaE_like</td><td>This protein family consists of proteins similar to the cpaE protein of the Caulobacter pilus assembly and the orf4 protein of Actinobacillus pilus formation gene cluster. The function of these proteins are unkown. The Caulobacter pilus assembly contains 7 genes: pilA, cpaA, cpaB, cpaC, cpaD, cpaE and cpaF. These genes are clustered together on chromosome.</td></tr>
<tr><td>93</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01217">TIGR01217</a></td><td>652</td><td>33</td><td>22.3</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ac_ac_CoA_syn</td><td>acetoacetyl-CoA synthase. This enzyme catalyzes the first step of the mevalonate pathway of IPP biosynthesis. Most bacteria do not use this pathway, but rather the deoxyxylulose pathway. </td></tr>
<tr><td>94</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03600">TIGR03600</a></td><td>420</td><td>43</td><td>21.9</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>phage_DnaB</td><td>phage replicative helicase, DnaB family, HK022 subfamily. Members of this family are phage (or prophage-region) homologs of the bacterial homohexameric replicative helicase DnaB. Some phage may rely on host DnaB, while others encode their own verions. This model describes the largest phage-specific clade among the close homologs of DnaB, but there are, or course, other DnaB homologs from phage that fall outside the scope of this model. </td></tr>
<tr><td>95</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10502">pfam10502</a></td><td>138</td><td>22</td><td>21.8</td><td>85</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_S26</td><td>Signal peptidase, peptidase S26. This is a family of membrane signal serine endopeptidases which function in the processing of newly-synthesized secreted proteins. Peptidase S26 removes the hydrophobic, N-terminal, signal peptides as proteins are translocated across membranes. The active site residues take the form of a catalytic dyad that is Ser, Lys in subfamily S26A; the Ser is the nucleophile in catalysis, and the Lys is the general base.</td></tr>
<tr><td>96</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08282">cd08282</a></td><td>375</td><td>22</td><td>21.8</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PFDH_like</td><td>Pseudomonas putida aldehyde-dismutating formaldehyde dehydrogenase (PFDH). Formaldehyde dehydrogenase (FDH) is a member of the zinc-dependent/medium chain alcohol dehydrogenase family.  Unlike typical FDH, Pseudomonas putida aldehyde-dismutating FDH (PFDH) is glutathione-independent.  PFDH converts 2 molecules of aldehydes to corresponding carboxylic acid and alcohol.  MDH family uses NAD(H) as a cofactor in the interconversion of alcohols and aldehydes, or ketones. Like the zinc-dependent alcohol dehydrogenases (ADH) of the medium chain alcohol dehydrogenase/reductase family (MDR), these tetrameric FDHs have a catalytic zinc that resides between the catalytic and NAD(H)binding domains and a structural zinc in a lobe of the catalytic domain. Unlike ADH, where NAD(P)(H) acts as a cofactor, NADH in FDH is a tightly bound redox cofactor (similar to nicotinamide proteins).  The medium chain alcohol dehydrogenase family (MDR) has a NAD(P)(H)-binding domain in a Rossmann fold of an beta-alpha form. The N-terminal region typically has an all-beta catalytic domain. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit.</td></tr>
<tr><td>97</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04311">TIGR04311</a></td><td>423</td><td>22</td><td>21.8</td><td>24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rSAM_Geo_metal</td><td>putative metalloenzyme radical SAM/SPASM domain maturase. This model describes a family of radical SAM/SPASM enzymes largely from the deltaproteobacteria. The family is most closely related to radical SAM enzyme family regularly in archaea in the vicinity of tungsten-containing oxidoreductases. A single member of the family in archaea may correspond to multiple tungsten enzymes, e.g. five in Pyrococcus furiosus. Therefore, the lack of a conserved gene neighborhood for members of this family in deltaprotebacteria suggests members may be involved in the maturation of multiple metalloenzymes.</td></tr>
<tr><td>98</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1411">COG1411</a></td><td>229</td><td>30</td><td>21.6</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1411</td><td>Uncharacterized protein related to proFAR isomerase (HisA) </td></tr>
<tr><td>99</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07794">cd07794</a></td><td>470</td><td>57</td><td>21.3</td><td>67</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FGGY_GK_like_proteobact</td><td>Proteobacterial glycerol kinase-like proteins; belongs to the FGGY family of carbohydrate kinases. This subgroup corresponds to a small group of proteobacterial glycerol kinase (GK)-like proteins, including the glycerol kinase from Pseudomonas aeruginosa. Most bacteria, such as Escherichia coli, take up glycerol passively by facilitated diffusion. In contrast, P. aeruginosa may also utilize a binding protein-dependent active transport system to mediate glycerol transportation. The glycerol kinase subsequently phosphorylates the intracellular glycerol to glycerol 3-phosphate (G3P). GKs belong to the FGGY family of carbohydrate kinases, the monomers of which contain two large domains, which are separated by a deep cleft that forms the active site. This model includes both the N-terminal domain, which adopts a ribonuclease H-like fold, and the structurally related C-terminal domain.</td></tr>
<tr><td>100</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11617">pfam11617</a></td><td>31</td><td>10</td><td>21.3</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cu-binding_MopE</td><td>Protein metal binding site. This family of proteins represents a unique protein copper binding site that involves a tryptophan metabolite, kynurenine in the protein MopE. The production of kyneurenin by modification of tryptophan and its involvement in copper binding is an innate property of MopE.</td></tr>
<tr><td>101</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05304">cd05304</a></td><td>363</td><td>47</td><td>21.2</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rubrum_tdh</td><td>Rubrum transdehydrogenase NAD-binding and catalytic domains. Transhydrogenases found in bacterial and inner mitochondrial membranes link NAD(P)(H)-dependent redox reactions to proton translocation. The energy of the proton electrochemical gradient (delta-p), generated by the respiratory electron transport chain, is consumed by transhydrogenase in NAD(P)+ reduction. Transhydrogenase is likely involved in the regulation of the citric acid cycle. Rubrum transhydrogenase has 3 components, dI, dII, and dIII. dII spans the membrane while dI and dIII protrude on the cytoplasmic/matrix side. DI contains 2 domains in Rossmann-like folds, linked by a long alpha helix, and contains a NAD binding site. Two dI polypeptides (represented in this sub-family) spontaneously form a heterotrimer with dIII in the absence of dII. In the heterotrimer, both dI chains may bind NAD, but only one is well-ordered. dIII also binds a well-ordered NADP, but in a different orientation than a classical Rossmann domain.</td></tr>
<tr><td>102</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08464">cd08464</a></td><td>200</td><td>45</td><td>21.2</td><td>81</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_DntR_like_2</td><td>The C-terminal substrate binding domain of an uncharacterized LysR-type transcriptional regulator similar to DntR, which is involved in the catabolism of dinitrotoluene; contains the type 2 periplasmic binding fold. This CD includes an uncharacterized LysR-type transcriptional regulator similar to DntR, NahR, and LinR, which are involved in the degradation of aromatic compounds. The transcription of the genes encoding enzymes involved in such degradation is regulated and expression of these enzymes is enhanced by inducers, which are either an intermediate in the metabolic pathway or compounds to be degraded.  This substrate-binding domain shows significant homology to the type 2 periplasmic binding proteins (PBP2), which are responsible for the uptake of a variety of substrates such as phosphate, sulfate, polysaccharides, lysine/arginine/ornithine, and histidine. The PBP2 bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two cytoplasmically located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>103</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02699">pfam02699</a></td><td>83</td><td>11</td><td>21.1</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YajC</td><td>Preprotein translocase subunit. See.</td></tr>
<tr><td>104</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2870">COG2870</a></td><td>467</td><td>17</td><td>21.1</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RfaE</td><td>ADP-heptose synthase, bifunctional sugar kinase/adenylyltransferase </td></tr>
<tr><td>105</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03483">cd03483</a></td><td>127</td><td>18</td><td>21.1</td><td>86</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MutL_Trans_MLH1</td><td>MutL_Trans_MLH1: transducer domain, having a ribosomal S5 domain 2-like fold, found in proteins similar to yeast and human MLH1 (MutL homologue 1). This transducer domain is homologous to the second domain of the DNA gyrase B subunit, which is known to be important in nucleotide hydrolysis and the transduction of structural signals from ATP-binding site to the DNA breakage/reunion regions of the enzymes. MLH1 forms heterodimers with PMS2, PMS1 and MLH3. These three complexes have distinct functions in meiosis. hMLH1-hPMS2 also participates in the repair of all DNA mismatch repair (MMR) substrates. Roles for hMLH1-hPMS1 or hMLH1-hMLH3 in MMR have not been established. Cells lacking hMLH1 have a strong mutator phenotype and display microsatellite instability (MSI). Mutation in hMLH1 causes predisposition to HNPCC, Muir-Torre syndrome and Turcot syndrome (HNPCC variant). Mutation in hMLH1 accounts for a large fraction of HNPCC families.</td></tr>
<tr><td>106</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07324">PRK07324</a></td><td>373</td><td>27</td><td>20.7</td><td>59</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07324</td><td>transaminase; Validated</td></tr>
<tr><td>107</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02691">TIGR02691</a></td><td>129</td><td>65</td><td>20.7</td><td>84</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>arsC_pI258_fam</td><td>arsenate reductase (thioredoxin). This family describes the well-studied thioredoxin-dependent arsenate reductase of Staphylococcus aureaus plasmid pI258 and other mechanistically similar arsenate reductases. The mechanism involves an intramolecular disulfide bond cascade, and aligned members of this family have four absolutely conserved Cys residues. This group of arsenate reductases belongs to the low-molecular weight protein-tyrosine phosphatase family (pfam01451), as does a group of glutathione/glutaredoxin type arsenate reductases (TIGR02689). At least two other, non-homologous groups of arsenate reductases involved in arsenical resistance are also known. This enzyme reduces arsenate to arsenite, which may be more toxic but which is more easily exported. </td></tr>
<tr><td>108</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0422">COG0422</a></td><td>432</td><td>33</td><td>20.7</td><td>62</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ThiC</td><td>Thiamine biosynthesis protein ThiC </td></tr>
<tr><td>109</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06482">PRK06482</a></td><td>276</td><td>54</td><td>20.2</td><td>2.3E+02</td><td>[&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06482</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>110</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4043">COG4043</a></td><td>111</td><td>15</td><td>20.0</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ASCH</td><td>ASC-1 homology (ASCH) domain, predicted RNA-binding domain </td></tr>
</table>
</body>
