<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12496">PRK12496</a></td><td>164</td><td>138</td><td>99.5</td><td>1.5E-14</td><td>[------------------------------------------------&nbsp;]</td><td>PRK12496</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1439">COG1439</a></td><td>177</td><td>140</td><td>99.4</td><td>3.6E-13</td><td>[------------------------------------------------&nbsp;]</td><td>Nob1</td><td>rRNA maturation endonuclease Nob1 </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09861">cd09861</a></td><td>123</td><td>117</td><td>97.2</td><td>0.00067</td><td>[&nbsp;------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_VapC-like</td><td>PIN domain of ribonucleases (toxins), VapC, FitB, and PAE0151 of bacterial and archaeal toxin/antitoxin-like operons, and other similar homologs. PIN (PilT N terminus) domain of ribonucleases (toxins) of prokaryotic toxin/antitoxin (TA) operons, involved in growth inhibition by regulating translation, are included in this family. They include the Mycobacterium tuberculosis VapC of the VapBC (virulence associated proteins) TA operon, and Neisseria gonorrhoeae FitB of the FitAB (fast intracellular trafficking) TA operon. Also included in this family are the uncharacterized Mycobacterium bovis UPF0110 and Synechocystis sp. PCC 6803 Sll0205 proteins, as well as, the archaeal Pyrobaculum aerophilum PAE0151 protein. PIN domain-containing toxins, such as, VapC, are nearly always co-expressed with an antitoxin, a cognate protein inhibitor (VapB) forming an inert protein complex. Disassociation of the protein complex activates the ribonuclease activity of the toxin by an, as yet undefined mechanism. VapC-like PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this group typically contain three or four highly conserved acidic residues that cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and are essential for single-stranded ribonuclease activity. VapC-like PIN domains are single domain proteins that form dimers.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09876">cd09876</a></td><td>109</td><td>69</td><td>97.1</td><td>0.00056</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Nob1</td><td>PIN domain of eukaryotic ribosome assembly factor Nob1 and archaeal UPF0129 protein Ta0041-like homologs. PIN (PilT N terminus) domain of the Saccharomyces cerevisiae ribosome assembly factor, Nob1 (Nin one binding) protein, the Thermoplasma acidophilum DSM 1728, UPF0129 protein Ta0041, and similar eukaryotic and archaeal homologs are included in this family.  The Nob1 PIN domain binds the single-stranded cleavage site D at the 3#end of 18S rRNA. Recombinant Nob1 binds as a tetramer to pre-18S rRNA fragments containing cleavage site D and believed to cleave at this site. These PIN domains are homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this subgroup contain three highly conserved acidic residues (putative metal-binding, active site residues).</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01850">pfam01850</a></td><td>120</td><td>85</td><td>96.7</td><td>0.0047</td><td>[--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN</td><td>PIN domain. </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09881">cd09881</a></td><td>127</td><td>112</td><td>95.0</td><td>0.029</td><td>[&nbsp;----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_VapC-FitB</td><td>PIN domain of ribonucleases (toxins), VapC and FitB, of prokaryotic toxin/antitoxin operons, Pyrococcus horikoshii protein PH0500, and other similar bacterial and archaeal homologs. PIN (PilT N terminus) domain-containing proteins of prokaryotic toxin/antitoxin (TA) operons, such as, Mycobacterium tuberculosis VapC of the VapBC (virulence associated proteins) TA operon, and Neisseria gonorrhoeae FitB of the FitAB (fast intracellular trafficking) TA operon, as well as, the archaeal Pyrococcus horikoshii protein PH0500 are included in this family. Toxins of TA operons are believed to be involved in growth inhibition by regulating translation and are nearly always co-expressed with an antitoxin, a cognate protein inhibitor, forming an inert protein complex. Disassociation of the complex activates the ribonuclease activity of the toxin. In N. gonorrhoeae, FitA and FitB form a heterodimer: FitA is the DNA binding subunit and FitB contains a ribonuclease activity that is blocked by the presence of FitA. A tetramer of FitAB heterodimers binds DNA from the fitAB upstream promoter region with high affinity. This results in both sequestration of FitAB and repression of fitAB transcription.  It is thought that FitAB release from the DNA and subsequent dissociation both slows N. gonorrhoeae replication and transcytosis by an as yet undefined mechanism.  The toxin M. tuberculosis VapC is a structural homolog of N. gonorrhoeae FitB, but their antitoxin partners, VapB and FitA, respectively, differ structurally.  The M. tuberculosis VapC-5 is proposed to be both an endoribonuclease and an exoribonuclease that can act on free RNA in a similar manner to the endo and exonuclease flap endonuclease-1 (FEN1). VapC-like toxins are structural homologs of FEN1-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this group typically contain three or four conserved acidic residues that cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and be essential for single-stranded ribonuclease activity. VapC-like PIN domains are single domain proteins that form dimers and dimerization configures the active sites in a groove along the long-axis of the structure.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09872">cd09872</a></td><td>127</td><td>118</td><td>94.2</td><td>0.12</td><td>[&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Sll0205</td><td>PIN domain of the VapC-like Sll0205 protein and homologs. Virulence associated protein C (VapC)-like PIN (PilT N terminus) domain of the Synechocystis sp. (strain PCC 6803) Sll0205 protein and other uncharacterized homologs are included in this subfamily. They are similar to the PIN domains of the Mycobacterium tuberculosis VapC and Neisseria gonorrhoeae FitB toxins of the prokaryotic toxin/antitoxin operons, VapBC and FitAB, respectively, which are believed to be involved in growth inhibition by regulating translation. These toxins are nearly always co-expressed with an antitoxin, a cognate protein inhibitor, forming an inert protein complex. Disassociation of the protein complex activates the ribonuclease activity of the toxin by an, as yet undefined mechanism. VapC-like PIN domains are homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region,  seen in FEN1-like PIN domains.  PIN domains within this subgroup contain four highly conserved acidic residues. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and be essential for single-stranded ribonuclease activity.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09875">cd09875</a></td><td>121</td><td>109</td><td>93.5</td><td>0.12</td><td>[&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_VapC-FitB-like</td><td>PIN domain of ribonucleases (toxins), VapC and FitB, of prokaryotic toxin/antitoxin operons, Pyrococcus horikoshii protein PH0500, and other similar bacterial and archaeal homologs. PIN (PilT N terminus) domain-containing proteins of prokaryotic toxin/antitoxin (TA) operons, such as, Mycobacterium tuberculosis VapC of the VapBC (virulence associated proteins) TA operon, and Neisseria gonorrhoeae FitB of the FitAB (fast intracellular trafficking) TA operon, as well as, the archaeal Pyrococcus horikoshii protein PH0500 are included in this family. Toxins of TA operons are believed to be involved in growth inhibition by regulating translation and are nearly always co-expressed with an antitoxin, a cognate protein inhibitor, forming an inert protein complex. Disassociation of the complex activates the ribonuclease activity of the toxin. In N. gonorrhoeae, FitA and FitB form a heterodimer: FitA is the DNA binding subunit and FitB contains a ribonuclease activity that is blocked by the presence of FitA. A tetramer of FitAB heterodimers binds DNA from the fitAB upstream promoter region with high affinity. This results in both sequestration of FitAB and repression of fitAB transcription.  It is thought that FitAB release from the DNA and subsequent dissociation both slows N. gonorrhoeae replication and transcytosis by an as yet undefined mechanism.  The toxin M. tuberculosis VapC is a structural homolog of N. gonorrhoeae FitB, but their antitoxin partners, VapB and FitA, respectively, differ structurally.  The M. tuberculosis VapC-5 is proposed to be both an endoribonuclease and an exoribonuclease that can act on free RNA in a similar manner to the endo and exonuclease flap endonuclease-1 (FEN1). VapC-like toxins are structural homologs of FEN1-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this group typically contain three or four conserved acidic residues that cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and be essential for single-stranded ribonuclease activity. VapC-like PIN domains are single domain proteins that form dimers and dimerization configures the active sites in a groove along the long-axis of the structure.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09882">cd09882</a></td><td>128</td><td>54</td><td>93.4</td><td>0.15</td><td>[&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_MtRv0301</td><td>PIN domain of the Rv0301 toxin of Mycobacterium tuberculosis and other uncharacterized, annotated PilT protein domain proteins. Virulence associated protein C (VapC)-like PIN (PilT N terminus) domain of Mycobacterium tuberculosis protein Rv0301 and similar bacterial proteins are included in this subfamily. They are PIN domain homologs of the Mycobacterium tuberculosis VapC and Neisseria gonorrhoeae FitB toxins of the prokaryotic toxin/antitoxin operons, VapBC and FitAB, respectively, which are believed to be involved in growth inhibition by regulating translation. These toxins are nearly always co-expressed with an antitoxin, a cognate protein inhibitor, forming an inert protein complex. Disassociation of the protein complex activates the ribonuclease activity of the toxin by an, as yet undefined mechanism. VapC-like PIN domains are homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this subgroup contain four highly conserved acidic residues. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and be essential for single-stranded ribonuclease activity.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1487">COG1487</a></td><td>133</td><td>47</td><td>92.6</td><td>0.28</td><td>[---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VapC</td><td>Predicted nucleic acid-binding protein, contains PIN domain  </td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1848">COG1848</a></td><td>140</td><td>126</td><td>87.0</td><td>0.61</td><td>[-------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1848</td><td>Predicted nucleic acid-binding protein, contains PIN domain  </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01018">cd01018</a></td><td>266</td><td>39</td><td>86.6</td><td>0.84</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ZntC</td><td>Metal binding protein ZntC.  These proteins are predicted to function as initial receptors in ABC transport of metal ions.  They belong to the TroA superfamily of helical backbone metal receptor proteins that share a distinct fold and ligand binding mechanism.  They are comprised of two globular subdomains connected by a long alpha helix and bind their specific ligands in the cleft between these domains.  In addition, many of these proteins possess a metal-binding histidine-rich motif (repetitive HDH sequence).</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09855">cd09855</a></td><td>128</td><td>102</td><td>81.9</td><td>4.9</td><td>[&nbsp;------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_VapC-Smg6-like</td><td>PIN domains of VapC and Smg6 ribonucleases, ribosome assembly factor NOB1, Archaeoglobus fulgidus AF0591 protein and homologs. PIN (PilT N terminus) domains of such ribonucleases as the toxins of prokaryotic toxin/antitoxin operons FitAB and VapBC, as well as, eukaryotic ribonucleases such as Smg6, ribosome assembly factor NOB1, exosome subunit Rrp44 endoribonuclease are included in this family. Also included are the PIN domains of the Pyrobaculum aerophilum Pea0151 and Archaeoglobus fulgidus AF0591 proteins and other similar archaeal homologs. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domains typically contain three or four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09871">cd09871</a></td><td>128</td><td>100</td><td>77.3</td><td>7.6</td><td>[&nbsp;------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_UPF0110</td><td>PIN domain of the VapC-like UPF0110 protein Mb0640 and homologs. Virulence associated protein C (VapC)-like PIN (PilT N terminus) domain of the Mycobacterium bovis UPF0110 protein Mb0640 and other uncharacterized homologs are included in this subfamily. They are similar to the PIN domains of the Mycobacterium tuberculosis VapC and Neisseria gonorrhoeae FitB toxins of the prokaryotic toxin/antitoxin operons, VapBC and FitAB, respectively, which are believed to be involved in growth inhibition by regulating translation. These toxins are nearly always co-expressed with an antitoxin, a cognate protein inhibitor, forming an inert protein complex. Disassociation of the protein complex activates the ribonuclease activity of the toxin by an, as yet undefined mechanism. VapC-like PIN domains are homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region,  seen in FEN1-like PIN domains.  PIN domains within this subgroup contain three highly conserved acidic residues. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and be essential for single-stranded ribonuclease activity.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0803">COG0803</a></td><td>303</td><td>109</td><td>71.9</td><td>3.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ZnuA</td><td>ABC-type Zn uptake system ZnuABC, Zn-binding component ZnuA </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09873">cd09873</a></td><td>127</td><td>120</td><td>68.5</td><td>14</td><td>[&nbsp;------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Pae0151</td><td>PIN domain of the Pyrobaculum aerophilum Pae0151 and Pae2754 proteins and homologs. Virulence associated protein C (VapC)-like PIN (PilT N terminus) domain of the Pyrobaculum aerophilum proteins, Pae0151 and Pae2754, and homologs are included in this subfamily. They are similar to the PIN domains of the Mycobacterium tuberculosis VapC and Neisseria gonorrhoeae FitB toxins of the prokaryotic toxin/antitoxin operons, VapBC and FitAB, respectively, which are believed to be involved in growth inhibition by regulating translation. These toxins are nearly always co-expressed with an antitoxin, a cognate protein inhibitor, forming an inert protein complex. Disassociation of the protein complex activates the ribonuclease activity of the toxin by an, as yet undefined mechanism. VapC-like PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this subgroup contain four highly conserved acidic residues. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and be essential for single-stranded ribonuclease activity.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09874">cd09874</a></td><td>135</td><td>85</td><td>67.3</td><td>15</td><td>[&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_MT3492</td><td>PIN domain of the hypothetical protein MT3492 of Mycobacterium tuberculosis CDC1551 and other uncharacterized, annotated PilT protein domain proteins. Virulence associated protein C (VapC)-like PIN (PilT N terminus) domain of Mycobacterium tuberculosis CDC1551, hypothetical protein MT3492, and similar bacterial and archaeal proteins are included in this subfamily. They are PIN domain homologs of the Mycobacterium tuberculosis VapC and Neisseria gonorrhoeae FitB toxins of the prokaryotic toxin/antitoxin operons, VapBC and FitAB, respectively, which are believed to be involved in growth inhibition by regulating translation. These toxins are nearly always co-expressed with an antitoxin, a cognate protein inhibitor, forming an inert protein complex. Disassociation of the protein complex activates the ribonuclease activity of the toxin by an, as yet undefined mechanism. VapC-like PIN domains are homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region,  seen in FEN1-like PIN domains.  PIN domains within this subgroup contain four highly conserved acidic residues. These putative active site residues are thought to bind Mg2+ and/or Mn2+ ions and be essential for single-stranded ribonuclease activity.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0424">COG0424</a></td><td>193</td><td>68</td><td>65.8</td><td>3.4</td><td>[&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Maf</td><td>Predicted house-cleaning NTP pyrophosphatase, Maf/HAM1 superfamily </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11377">cd11377</a></td><td>139</td><td>25</td><td>63.8</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pro-peptidase_S53</td><td>Activation domain of S53 peptidases. Members of this family are found in various subtilase propeptides, such as  pro-kumamolysin and tripeptidyl peptidase I, and adopt a ferredoxin-like fold, with an alpha+beta sandwich. Cleavage of the domain results in activation of the peptidase.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11290">pfam11290</a></td><td>171</td><td>16</td><td>60.8</td><td>3.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;]</td><td>DUF3090</td><td>Protein of unknown function (DUF3090). This family of proteins with unknown function appears to be restricted to Actinobacteria.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11829">pfam11829</a></td><td>97</td><td>33</td><td>60.2</td><td>5.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3349</td><td>Protein of unknown function (DUF3349). This family of proteins are functionally uncharacterized. This protein is found in bacteria. Proteins in this family are typically between 99 to 124 amino acids in length.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07228">PRK07228</a></td><td>445</td><td>91</td><td>57.8</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07228</td><td>N-ethylammeline chlorohydrolase; Provisional</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03847">TIGR03847</a></td><td>177</td><td>25</td><td>57.0</td><td>4.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;]</td><td>TIGR03847</td><td>conserved hypothetical protein. The conserved hypothetical protein described here occurs as part of the trio of uncharacterized proteins common in the Actinobacteria.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14354">pfam14354</a></td><td>56</td><td>12</td><td>56.2</td><td>4.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;]</td><td>Lar_restr_allev</td><td>Restriction alleviation protein Lar. This family includes the restriction alleviation protein Lar encoded by the Rac prophage of Escherichia coli. This protein modulates the activity of the Escherichia coli restriction and modification system.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00334">pfam00334</a></td><td>135</td><td>75</td><td>54.6</td><td>7.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NDK</td><td>Nucleoside diphosphate kinase. </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09778">pfam09778</a></td><td>212</td><td>48</td><td>54.2</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Guanylate_cyc_2</td><td>Guanylylate cyclase. Members of this family of proteins catalyse the conversion of guanosine triphosphate (GTP) to 3',5'-cyclic guanosine monophosphate (cGMP) and pyrophosphate.</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04423">pfam04423</a></td><td>54</td><td>15</td><td>47.8</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;]</td><td>Rad50_zn_hook</td><td>Rad50 zinc hook motif. The Mre11 complex (Mre11 Rad50 Nbs1) is central to chromosomal maintenance and functions in homologous recombination, telomere maintenance and sister chromatid association. The Rad50 coiled-coil region contains a dimer interface at the apex of the coiled coils in which pairs of conserved Cys-X-X-Cys motifs form interlocking hooks that bind one Zn ion. This alignment includes the zinc hook motif and a short stretch of coiled-coil on either side.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08130">PRK08130</a></td><td>213</td><td>72</td><td>45.1</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08130</td><td>putative aldolase; Validated</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01298">cd01298</a></td><td>411</td><td>84</td><td>44.9</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ATZ_TRZ_like</td><td>TRZ/ATZ family contains enzymes from the atrazine degradation pathway and related hydrolases. Atrazine, a chlorinated herbizide, can be catabolized by a variety of different bacteria. The first three steps of the atrazine dehalogenation pathway are catalyzed by atrazine chlorohydrolase (AtzA), hydroxyatrazine ethylaminohydrolase (AtzB), and N-isopropylammelide N-isopropylaminohydrolase (AtzC). All three enzymes belong to the superfamily of metal dependent hydrolases. AtzA and AtzB, beside other related enzymes are represented in this CD.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15531">pfam15531</a></td><td>128</td><td>35</td><td>42.9</td><td>12</td><td>[&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Toxin_51</td><td>Putative toxin 51. A predicted RNase toxin found in bacterial polymorphic toxin systems. The toxin possesses an alpha+beta fold and conserved aspartate and glutamate residues, and an RxW motif. In bacterial polymorphic toxin systems, the toxin is exported by the type 2 or type 7 secretion system.</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09710">PRK09710</a></td><td>64</td><td>14</td><td>42.3</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;]</td><td>lar</td><td>restriction alleviation and modification protein; Reviewed</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01151">cd01151</a></td><td>386</td><td>33</td><td>39.2</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GCD</td><td>Glutaryl-CoA dehydrogenase. Glutaryl-CoA dehydrogenase (GCD). GCD is an acyl-CoA dehydrogenase, which catalyzes the oxidative decarboxylation of glutaryl-CoA to crotonyl-CoA and carbon dioxide in the catabolism of lysine, hydroxylysine, and tryptophan. It uses electron transfer flavoprotein (ETF) as an electron acceptor. GCD is a homotetramer. GCD deficiency leads to a severe neurological disorder in humans.</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13470">pfam13470</a></td><td>80</td><td>44</td><td>38.1</td><td>37</td><td>[--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_3</td><td>PIN domain. Members of this family of bacterial domains are predicted to be RNases (from similarities to 5'-exonucleases).</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00555">cd00555</a></td><td>180</td><td>68</td><td>35.4</td><td>20</td><td>[&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Maf</td><td>Nucleotide binding protein Maf. Maf has been implicated in inhibition of septum formation in eukaryotes, bacteria and archaea, but homologs in B.subtilis and S.cerevisiae are nonessential for cell division. Maf has been predicted to be a nucleotide- or nucleic acid-binding protein with structural similarity to the hypoxanthine/xanthine NTP pyrophosphatase Ham1 from Methanococcus jannaschii, RNase H from Escherichia coli, and some other nucleotide or RNA-binding proteins.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0031">COG0031</a></td><td>300</td><td>44</td><td>34.0</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CysK</td><td>Cysteine synthase </td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11123">pfam11123</a></td><td>82</td><td>49</td><td>33.0</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DNA_Packaging_2</td><td>DNA packaging protein. This DNA packaging protein is also referred to as gene 18 product (gp18). This protein is required for DNA packaging and functions in a complex with gp19.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09877">cd09877</a></td><td>127</td><td>94</td><td>32.9</td><td>1.8E+02</td><td>[------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_YacL</td><td>PIN domain of Thermus Thermophilus Hb8, uncharacterized Bacillus subtilis YacL, and other bacterial homologs. PIN (PilT N terminus) domain of the conserved membrane protein of unknown function of Thermus Thermophilus Hb8, Bacillus subtilis YacL and other similar homologs are included in this family. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domain homologs within this group contain four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.  Proteins in this group have a C-terminal TRAM domain whose function is unknown but predicted to be a RNA-binding domain common to tRNA uracil methylation and adenine thiolation enzymes.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03655">TIGR03655</a></td><td>53</td><td>9</td><td>31.0</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;]</td><td>anti_R_Lar</td><td>restriction alleviation protein, Lar family. Restriction alleviation proteins provide a countermeasure to host cell restriction enzyme defense against foreign DNA such as phage or plasmids. This family consists of homologs to the phage antirestriction protein Lar, and most members belong to phage genomes or prophage regions of bacterial genomes. </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04904">cd04904</a></td><td>74</td><td>24</td><td>30.9</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ACT_AAAH</td><td>ACT domain of the nonheme iron-dependent, aromatic amino acid hydroxylases (AAAH). ACT domain of the nonheme iron-dependent, aromatic amino acid hydroxylases (AAAH): Phenylalanine hydroxylases (PAH), tyrosine hydroxylases (TH) and tryptophan hydroxylases (TPH), both peripheral (TPH1) and neuronal (TPH2) enzymes. This family of enzymes shares a common catalytic mechanism, in which dioxygen is used by an active site containing a single, reduced iron atom to hydroxylate an unactivated aromatic substrate, concomitant with a two-electron oxidation of tetrahydropterin (BH4) cofactor to its quinonoid dihydropterin form. PAH catalyzes the hydroxylation of L-Phe to L-Tyr, the first step in the catabolic degradation of L-Phe; TH catalyses the hydroxylation of L-Tyr to 3,4-dihydroxyphenylalanine, the rate limiting step in the biosynthesis of catecholamines; and TPH catalyses the hydroxylation of L-Trp to 5-hydroxytryptophan, the rate limiting step in the biosynthesis of 5-hydroxytryptamine (serotonin) and the first reaction in the synthesis of melatonin. Eukaryotic AAAHs have an N-terminal  ACT (regulatory) domain, a middle catalytic domain and a C-terminal domain which is responsible for the oligomeric state of the enzyme forming a domain-swapped tetrameric coiled-coil. The PAH, TH, and TPH enzymes contain highly conserved catalytic domains but distinct N-terminal ACT domains (this CD) and differ in their mechanisms of regulation. One commonality is that all three eukaryotic enzymes are regulated in part by the phosphorylation of serine residues N-terminal of the ACT domain. Members of this CD belong to the superfamily of ACT regulatory domains.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00032">PRK00032</a></td><td>190</td><td>68</td><td>30.8</td><td>13</td><td>[&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00032</td><td>Maf-like protein; Reviewed</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02116">cd02116</a></td><td>60</td><td>28</td><td>30.6</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ACT</td><td>ACT domains are commonly involved in specifically binding an amino acid or other small ligand leading to regulation of the enzyme. Members of this CD belong to the superfamily of ACT regulatory domains. Pairs of ACT domains are commonly involved in specifically binding an amino acid or other small ligand leading to regulation of the enzyme. The ACT domain has been detected in a number of diverse proteins; some of these proteins are involved in amino acid and purine biosynthesis, phenylalanine hydroxylation, regulation of bacterial metabolism and transcription, and many remain to be characterized. ACT domain-containing enzymes involved in amino acid and purine synthesis are in many cases allosteric enzymes with complex regulation enforced by the binding of ligands. The ACT domain is commonly involved in the binding of a small regulatory molecule, such as the amino acids L-Ser and L-Phe in the case of D-3-phosphoglycerate dehydrogenase and the bifunctional chorismate mutase-prephenate dehydratase enzyme (P-protein), respectively. Aspartokinases typically consist of two C-terminal ACT domains in a tandem repeat, but  the second ACT domain is inserted within the first, resulting in, what is normally the terminal beta strand of ACT2, formed from a region N-terminal of ACT1. ACT domain repeats have been shown to have nonequivalent ligand-binding sites with complex regulatory patterns such as those seen in the bifunctional enzyme, aspartokinase-homoserine dehydrogenase (ThrA). In other enzymes, such as phenylalanine hydroxylases, the ACT domain appears to function as a flexible small module providing allosteric regulation via transmission of conformational changes, these conformational changes are not necessarily initiated by regulatory ligand binding at the ACT domain itself. ACT domains are present either singularly, N- or C-terminal, or in pairs present C-terminal or between two catalytic domains. Unique to cyanobacteria are four ACT domains C-terminal to an aspartokinase domain. A few proteins are composed almost entirely of ACT domain repeats as seen in the four ACT domain protein, the ACR protein, found in higher plants; and the two ACT domain protein, the glycine cleavage system transcriptional repressor (GcvR) protein, found in some bacteria. Also seen are single ACT domain proteins similar to the Streptococcus pneumoniae ACT domain protein (uncharacterized pdb structure 1ZPV) found in both bacteria and archaea. Purportedly, the ACT domain is an evolutionarily mobile ligand binding regulatory module that has been fused to different enzymes at various times.</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01303">cd01303</a></td><td>429</td><td>121</td><td>30.5</td><td>73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GDEase</td><td>Guanine deaminase (GDEase). Guanine deaminase is an aminohydrolase responsible for the conversion of guanine to xanthine and ammonia, the first step to utilize guanine as a nitrogen source. This reaction also removes the guanine base from the pool and therefore can play a role in the regulation of cellular GTP and the guanylate nucleotide pool.</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13638">pfam13638</a></td><td>130</td><td>30</td><td>29.8</td><td>80</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_4</td><td>PIN domain. Members of this family of bacterial domains are predicted to be RNases (from similarities to 5'-exonucleases).</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3744">COG3744</a></td><td>130</td><td>120</td><td>29.2</td><td>1.9E+02</td><td>[------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG3744</td><td>PIN domain nuclease, a component of toxin-antitoxin system (PIN domain)  </td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14737">PRK14737</a></td><td>186</td><td>18</td><td>29.1</td><td>28</td><td>[------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gmk</td><td>guanylate kinase; Provisional</td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01388">pfam01388</a></td><td>92</td><td>59</td><td>28.5</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ARID</td><td>ARID/BRIGHT DNA binding domain. This domain is know as ARID for AT-Rich Interaction Domain, and also known as the BRIGHT domain.</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09286">pfam09286</a></td><td>142</td><td>24</td><td>28.1</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pro-kuma_activ</td><td>Pro-kumamolisin, activation domain. Members of this family are found in various subtilase propeptides, and adopt a ferredoxin-like fold, with an alpha+beta sandwich. Cleavage of the domain results in activation of the peptide.</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09802">PRK09802</a></td><td>269</td><td>56</td><td>28.0</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09802</td><td>DNA-binding transcriptional regulator AgaR; Provisional</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4113">COG4113</a></td><td>134</td><td>120</td><td>26.4</td><td>1.7E+02</td><td>[-------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4113</td><td>Predicted nucleic acid-binding protein, contains PIN domain  </td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03119">pfam03119</a></td><td>27</td><td>13</td><td>26.0</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----]</td><td>DNA_ligase_ZBD</td><td>NAD-dependent DNA ligase C4 zinc finger domain. DNA ligases catalyse the crucial step of joining the breaks in duplex DNA during DNA replication, repair and recombination, utilizing either ATP or NAD(+) as a cofactor. This family is a small zinc binding motif that is presumably DNA binding. IT is found only in NAD dependent DNA ligases.</td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13721">pfam13721</a></td><td>101</td><td>42</td><td>26.0</td><td>82</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SecD-TM1</td><td>SecD export protein N-terminal TM region. This domain appears to be the fist transmembrane region of the SecD export protein. SecD is directly involved in protein secretion and important for the release of proteins that have been translocated across the cytoplasmic membrane.</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07265">cd07265</a></td><td>122</td><td>59</td><td>25.9</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>2_3_CTD_N</td><td>N-terminal domain of catechol 2,3-dioxygenase. This subfamily contains the N-terminal, non-catalytic, domain of catechol 2,3-dioxygenase. Catechol 2,3-dioxygenase  (2,3-CTD, catechol:oxygen 2,3-oxidoreductase) catalyzes an extradiol cleavage of catechol to form 2-hydroxymuconate semialdehyde with the insertion of two atoms of oxygen. The enzyme is a homotetramer and contains catalytically essential Fe(II) . The reaction proceeds by an ordered bi-unit mechanism. First, catechol binds to the enzyme, this is then followed by the binding of dioxygen to form a tertiary complex, and then the aromatic ring is cleaved to produce 2-hydroxymuconate semialdehyde. Catechol 2,3-dioxygenase belongs to the type I extradiol dioxygenase family. The subunit comprises the N- and C-terminal domains of similar structure fold, resulting from an ancient gene duplication. The active site is located in a funnel-shaped space of the C-terminal domain. This subfamily represents the N-terminal domain.</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00075">pfam00075</a></td><td>127</td><td>31</td><td>25.9</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNase_H</td><td>RNase H. RNase H digests the RNA strand of an RNA/DNA hybrid. Important enzyme in retroviral replication cycle, and often found as a domain associated with reverse transcriptases. Structure is a mixed alpha+beta fold with three a/b/a layers.</td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0419">COG0419</a></td><td>908</td><td>13</td><td>25.3</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;]</td><td>SbcC</td><td>DNA repair exonuclease SbcCD ATPase subunit </td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05580">PRK05580</a></td><td>679</td><td>17</td><td>25.1</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;]</td><td>PRK05580</td><td>primosome assembly protein PriA; Validated</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1569">COG1569</a></td><td>142</td><td>18</td><td>24.5</td><td>29</td><td>[-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1569</td><td>Predicted nucleic acid-binding protein, contains PIN domain  </td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10087">pfam10087</a></td><td>96</td><td>32</td><td>24.5</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2325</td><td>Uncharacterized protein conserved in bacteria (DUF2325). Members of this family of hypothetical bacterial proteins have no known function.</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09278">cd09278</a></td><td>139</td><td>44</td><td>23.9</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNase_HI_prokaryote_like</td><td>RNase HI family found mainly in prokaryotes. Ribonuclease H (RNase H) is classified into two evolutionarily unrelated families, type 1 (prokaryotic RNase HI, eukaryotic RNase H1 and viral RNase H) and type 2 (prokaryotic RNase HII and HIII, and eukaryotic RNase H2). RNase H is an endonuclease that cleaves the RNA strand of an RNA/DNA hybrid in a sequence non-specific manner. RNase H is involved in DNA replication, repair and transcription. RNase H is widely present in various organisms, including bacteria, archaea and eukaryotes and most prokaryotic and eukaryotic genomes contain multiple RNase H genes. Despite the lack of amino acid sequence homology, type 1 and type 2 RNase H share a main-chain fold and steric configurations of the four acidic active-site (DEDD), residues and have the same catalytic mechanism and functions in cells. One of the important functions of RNase H is to remove Okazaki fragments during DNA replication. Prokaryotic RNase H varies greatly in domain structures and substrate specificities. Prokaryotes and some single-cell eukaryotes do not require RNase H for viability.</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05034">cd05034</a></td><td>248</td><td>16</td><td>23.9</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PTKc_Src_like</td><td>Catalytic domain of Src kinase-like Protein Tyrosine Kinases. PTKs catalyze the transfer of the gamma-phosphoryl group from ATP to tyrosine (tyr) residues in protein substrates. Src subfamily members include Src, Lck, Hck, Blk, Lyn, Fgr, Fyn, Yrk, and Yes. Src (or c-Src) proteins are cytoplasmic (or non-receptor) PTKs which are anchored to the plasma membrane. They contain an N-terminal SH4 domain with a myristoylation site, followed by SH3 and SH2 domains, a tyr kinase domain, and a regulatory C-terminal region containing a conserved tyr. They are activated by autophosphorylation at the tyr kinase domain, but are negatively regulated by phosphorylation at the C-terminal tyr by Csk (C-terminal Src Kinase). Src proteins are involved in signaling pathways that regulate cytokine and growth factor responses, cytoskeleton dynamics, cell proliferation, survival, and differentiation. They were identified as the first proto-oncogene products, and they regulate cell adhesion, invasion, and motility in cancer cells and tumor vasculature, contributing to cancer progression and metastasis. Src kinases are overexpressed in a variety of human cancers, making them attractive targets for therapy. They are also implicated in acute inflammatory responses and osteoclast function. Src, Fyn, Yes, and Yrk are widely expressed, while Blk, Lck, Hck, Fgr, and Lyn show a limited expression pattern. The Src-like subfamily is part of a larger superfamily that includes the catalytic domains of other kinases such as serine/threonine kinases, RIO kinases, aminoglycoside phosphotransferase, choline kinase, and phosphoinositide 3-kinase.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04880">cd04880</a></td><td>75</td><td>23</td><td>23.2</td><td>69</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ACT_AAAH-PDT-like</td><td>ACT domain of the nonheme iron-dependent, aromatic amino acid hydroxylases (AAAH). ACT domain of the nonheme iron-dependent, aromatic amino acid hydroxylases (AAAH): Phenylalanine hydroxylases (PAH), tyrosine hydroxylases (TH) and tryptophan hydroxylases (TPH), both peripheral (TPH1) and neuronal (TPH2) enzymes. This family of enzymes shares a common catalytic mechanism, in which dioxygen is used by an active site containing a single, reduced iron atom to hydroxylate an unactivated aromatic substrate, concomitant with a two-electron oxidation of tetrahydropterin (BH4) cofactor to its quinonoid dihydropterin form. Eukaryotic AAAHs have an N-terminal  ACT (regulatory) domain, a middle catalytic domain and a C-terminal domain which is responsible for the oligomeric state of the enzyme forming a domain-swapped tetrameric coiled-coil. The PAH, TH, and TPH enzymes contain highly conserved catalytic domains but distinct N-terminal ACT domains and differ in their mechanisms of regulation. One commonality is that all three eukaryotic enzymes appear to be regulated, in part, by the phosphorylation of serine residues N-terminal of the ACT domain. Also included in this CD are the C-terminal ACT domains of the bifunctional chorismate mutase-prephenate dehydratase (CM-PDT) enzyme and the prephenate dehydratase (PDT) enzyme found in plants, fungi, bacteria, and archaea. The P-protein of Escherichia coli (CM-PDT) catalyzes the conversion of chorismate to prephenate and then the decarboxylation and dehydration to form phenylpyruvate. These are the first two steps in the biosynthesis of L-Phe and L-Tyr via the shikimate pathway in microorganisms and plants. The E. coli P-protein (CM-PDT) has three domains with an N-terminal domain with chorismate mutase activity, a middle domain with prephenate dehydratase activity, and an ACT regulatory C-terminal domain. The prephenate dehydratase enzyme has a PDT and ACT domain. The ACT domain is essential to bring about the negative allosteric regulation by L-Phe binding. L-Phe binds with positive cooperativity; with this binding, there is a shift in the protein to less active tetrameric and higher oligomeric forms from a more active dimeric form. Members of this CD belong to the superfamily of ACT regulatory domains.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02817">pfam02817</a></td><td>39</td><td>15</td><td>22.9</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>E3_binding</td><td>e3 binding domain. This family represents a small domain of the E2 subunit of 2-oxo-acid dehydrogenases responsible for the binding of the E3 subunit.</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04163">TIGR04163</a></td><td>428</td><td>43</td><td>22.8</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;]</td><td>rSAM_cobopep</td><td>peptide-modifying radical SAM enzyme CbpB. Members of this family are radical SAM enzymes that modify a short peptide encoded by an upstream gene. A role in metal chelation is suggested.</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5548">COG5548</a></td><td>105</td><td>26</td><td>22.7</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;]</td><td>COG5548</td><td>Uncharacterized membrane protein, UPF0136 family </td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01297">pfam01297</a></td><td>263</td><td>39</td><td>22.6</td><td>77</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TroA</td><td>Periplasmic solute binding protein family. This family includes periplasmic solute binding proteins such as TroA that interacts with an ATP-binding cassette transport system in Treponema pallidum.</td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01850">pfam01850</a></td><td>120</td><td>69</td><td>22.3</td><td>2.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN</td><td>PIN domain. </td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03686">pfam03686</a></td><td>127</td><td>49</td><td>22.0</td><td>59</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UPF0146</td><td>Uncharacterized protein family (UPF0146). The function of this family of proteins is unknown.</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09879">cd09879</a></td><td>117</td><td>52</td><td>22.0</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_AF0591</td><td>PIN domain of Archaeoglobus fulgidus AF0591 protein and other similar archaeal homologs. PIN (PilT N terminus) domain of Archaeoglobus fulgidus AF0591 protein and other similar uncharacterized archaeal homologs are included in this family. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domains within this subgroup contain four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01924">pfam01924</a></td><td>355</td><td>11</td><td>21.7</td><td>50</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;]</td><td>HypD</td><td>Hydrogenase formation hypA family. HypD is involved in hydrogenase formation. It contains many possible metal binding residues, which may bind to nickel. Transposon Tn5 insertions into hypD resulted in R. leguminosarum mutants that lacked any hydrogenase activity in symbiosis with peas.</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05786">PRK05786</a></td><td>238</td><td>47</td><td>21.7</td><td>70</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fabG</td><td>3-ketoacyl-(acyl-carrier-protein) reductase; Provisional</td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10246">PRK10246</a></td><td>1047</td><td>14</td><td>21.6</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;]</td><td>PRK10246</td><td>exonuclease subunit SbcC; Provisional</td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1198">COG1198</a></td><td>730</td><td>16</td><td>21.4</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;]</td><td>PriA</td><td>Primosomal protein N' (replication factor Y) - superfamily II helicase </td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05370">pfam05370</a></td><td>87</td><td>27</td><td>21.2</td><td>76</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF749</td><td>Domain of unknown function (DUF749). Archaeal domain of unknown function. This domain has been solved as part of a structural genomics project and comprises of segregated helical and anti-parallel beta sheet regions.</td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5611">COG5611</a></td><td>130</td><td>101</td><td>21.0</td><td>59</td><td>[&nbsp;-------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG5611</td><td>Predicted nucleic-acid-binding protein, contains PIN domain  </td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00043">PRK00043</a></td><td>212</td><td>29</td><td>21.0</td><td>87</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>thiE</td><td>thiamine-phosphate pyrophosphorylase; Reviewed</td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00985">cd00985</a></td><td>131</td><td>44</td><td>20.3</td><td>48</td><td>[&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Maf_Ham1</td><td>Maf_Ham1. Maf, a nucleotide binding protein, has been implicated in inhibition of septum formation in eukaryotes, bacteria and archaea. A Ham1-related protein from Methanococcus jannaschii is a novel NTPase that has been shown to hydrolyze nonstandard nucleotides, such as hypoxanthine/xanthine NTP, but not standard nucleotides.</td></tr>
</table>
</body>
