<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3183">COG3183</a></td><td>272</td><td>106</td><td>99.6</td><td>3.3E-16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------------------&nbsp;]</td><td>COG3183</td><td>Predicted restriction endonuclease, HNH family </td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01844">pfam01844</a></td><td>47</td><td>45</td><td>90.5</td><td>0.11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HNH</td><td>HNH endonuclease. </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00085">cd00085</a></td><td>57</td><td>44</td><td>62.3</td><td>2.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HNHc</td><td>HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and anaredoxins.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08491">PRK08491</a></td><td>263</td><td>25</td><td>55.9</td><td>6.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08491</td><td>NADH dehydrogenase subunit C; Provisional</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07180">cd07180</a></td><td>204</td><td>22</td><td>53.0</td><td>5.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNase_HII_archaea_like</td><td>Archaeal Ribonuclease HII. This family includes type 2 RNases H from archaea, some of which show broad divalent cation specificity. It is proposed that three of the four acidic residues at the active site are involved in metal binding and the fourth one is involved in the catalytic process in archaea. Most archaeal genomes contain multiple RNase H genes. Despite a lack of evidence for homology from sequence comparisons, type I and type II RNase H share a common fold and similar steric configurations of the four acidic active-site residues, suggesting identical or very similar catalytic mechanisms. It appears that type I and type II RNases H also have overlapping functions in cells, as over-expression of Escherichia coli RNase HII can complement an RNase HI deletion phenotype in E. coli. RNase H is classified into two families, type I (prokaryotic RNase HI, eukaryotic RNase H1 and viral RNase H) and type II (prokaryotic RNase HII and HIII, archaeal RNase HII and eukaryotic RNase H2/HII). RNase H endonucleolytically hydrolyzes an RNA strand when it is annealed to a complementary DNA strand in the presence of divalent cations, in DNA replication or repair.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07776">pfam07776</a></td><td>74</td><td>53</td><td>47.8</td><td>9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>zf-AD</td><td>Zinc-finger associated domain (zf-AD). The zf-AD domain, also known as ZAD, forms an atypical treble-cleft-like zinc co-ordinating fold. The zf-AD domain is thought to be involved in mediating dimer formation, but does not bind to DNA.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00729">TIGR00729</a></td><td>206</td><td>17</td><td>47.5</td><td>8.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00729</td><td>ribonuclease H, mammalian HI/archaeal HII subfamily. This enzyme cleaves RNA from DNA-RNA hybrids. Archaeal members of this subfamily of RNase H are designated RNase HII and one has been shown to be active as a monomer. A member from Homo sapiens was characterized as RNase HI, large subunit. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04423">pfam04423</a></td><td>54</td><td>24</td><td>41.7</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rad50_zn_hook</td><td>Rad50 zinc hook motif. The Mre11 complex (Mre11 Rad50 Nbs1) is central to chromosomal maintenance and functions in homologous recombination, telomere maintenance and sister chromatid association. The Rad50 coiled-coil region contains a dimer interface at the apex of the coiled coils in which pairs of conserved Cys-X-X-Cys motifs form interlocking hooks that bind one Zn ion. This alignment includes the zinc hook motif and a short stretch of coiled-coil on either side.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14663">pfam14663</a></td><td>115</td><td>54</td><td>38.9</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------]</td><td>RasGEF_N_2</td><td>Rapamycin-insensitive companion of mTOR RasGEF_N domain. Rictor appears to serve as a scaffolding protein that is important for maintaining mTORC2 integrity. The mammalian target of rapamycin (mTOR) is a conserved Ser/Thr kinase that forms two functionally distinct complexes, mTROC1 and mTORC2, important for nutrient and growth-factor signalling. This region is the more conserved central section that may include several individual domains. Rictor can be inhibited in the short-term by rapamycin.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09482">cd09482</a></td><td>54</td><td>15</td><td>38.4</td><td>4.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LIM2_CRP3</td><td>The second LIM domain of Cysteine Rich Protein 3 (CRP3/MLP). The second LIM domain of Cysteine Rich Protein 3 (CRP3/MLP):  Cysteine-rich proteins (CRPs) are characterized by the presence of two LIM domains linked to short glycine-rich repeats (GRRs). The CRP family members include CRP1, CRP2, CRP3/MLP and TLPCRP1, CRP2 and CRP3 share a conserved nuclear targeting signal (K/R-K/R-Y-G-P-K), which supports the fact that these proteins function not only in the cytoplasm but also in the nucleus. CRPs control regulatory pathways during cellular differentiation, and involve in complex transcription circuits, and the organization as well as the arrangement of the myofibrillar/cytoskeletal network.CRP3 also called Muscle LIM Protein (MLP), which is a striated muscle-specific factor that enhances myogenic differentiation. The second LIM domain of CRP3/MLP interacts with cytoskeletal protein beta-spectrin. CRP3/MLP also interacts with the basic helix-loop-helix myogenic transcription factors MyoD, myogenin, and MRF4 thereby increasing their affinity for specific DNA regulatory elements. LIM domains are 50-60 amino acids in size and share two characteristic zinc finger motifs. The two zinc fingers contain eight conserved residues, mostly cysteines and histidines, which coordinately bond to two zinc atoms. LIM domains function as adaptors or scaffolds to support the assembly of multimeric protein complexes.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05582">PRK05582</a></td><td>650</td><td>51</td><td>35.9</td><td>4.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05582</td><td>DNA topoisomerase I; Validated</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1655">COG1655</a></td><td>267</td><td>58</td><td>35.2</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1655</td><td>Uncharacterized protein, DUF2225 family </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd15630">cd15630</a></td><td>49</td><td>38</td><td>34.0</td><td>3.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------]</td><td>PHD_BAZ2B</td><td>PHD finger found in bromodomain adjacent to zinc finger domain protein 2B (BAZ2B). BAZ2B, also termed WALp4, is a bromodomain-containing protein whose biological role is still elusive. It shows high sequence similarly with BAZ2A, which is the large subunit of the SNF2h-containing chromatin-remodeling complex NoRC that induces nucleosome sliding in an ATP-and histone H4 tail-dependent fashion. BAZ2B contains a TAM (TIP5/ARBP/MBD) domain, an Apolipophorin-III like domain, a DDT domain, four AT-hooks, BAZ 1 and BAZ 2 motifs, a WAKZ (WSTF/Acf1/KIAA0314/ZK783.4) motif, a plant homeodomain (PHD) finger, and a bromodomain.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09445">cd09445</a></td><td>53</td><td>26</td><td>31.7</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LIM_Mical_like_2</td><td>This domain belongs to the LIM domain family which are found on Mical (molecule interacting with CasL) like proteins. The LIM domain on proteins of unknown function: This domain belongs to the LIM domain family which are found on Mical (molecule interacting with CasL)-like proteins. Known members of the Mical-like family includes single LIM domain containing proteins, Mical (molecule interacting with CasL), pollen specific protein SF3, Eplin, xin actin-binding repeat-containing protein 2 (XIRP2), and Ltd-1. The members of this family function mainly at the cytoskeleton and focal adhesions. They interact with transcription factors or other signaling molecules to play roles in muscle development, neuronal differentiation, cell growth, and mobility.  As in other LIM domains, this domain family is 50-60 amino acids in size and shares two characteristic zinc finger motifs. The two zinc fingers contain eight conserved residues, mostly cysteines and histidines, which coordinately bond to two zinc atoms. LIM domains function as adaptors or scaffolds to support the assembly of multimeric protein.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12141">pfam12141</a></td><td>488</td><td>42</td><td>30.3</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3589</td><td>Protein of unknown function (DUF3589). This family of proteins is found in eukaryotes. Proteins in this family are typically between 541 and 717 amino acids in length. The function of this family is not known,</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09086">pfam09086</a></td><td>98</td><td>33</td><td>30.0</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;]</td><td>DUF1924</td><td>Domain of unknown function (DUF1924). This domain is found in a set of bacterial proteins, including Cytochrome c-type protein. It is functionally uncharacterized.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13790">pfam13790</a></td><td>38</td><td>19</td><td>26.7</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4182</td><td>Domain of unknown function (DUF4182). This protein of unknown function contains a number of highly conserved cysteine residues, which may form disulphide bonds.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12494">PRK12494</a></td><td>172</td><td>22</td><td>24.1</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK12494</td><td>NADH dehydrogenase subunit J; Provisional</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13451">pfam13451</a></td><td>49</td><td>19</td><td>24.0</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>zf-trcl</td><td>Probable zinc-binding domain. This is a probable zinc-binding domain with two CxxC sequence motifs, found in various families of bacteria.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09452">cd09452</a></td><td>52</td><td>35</td><td>24.0</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LIM1_Enigma</td><td>The first LIM domain of Enigma. The first LIM domain of Enigma: Enigma was initially characterized in humans as a protein containing three LIM domains at the C-terminus and a PDZ domain at N-terminus.  The third LIM domain specifically interacts with the insulin receptor and the second LIM domain interacts with the receptor tyrosine kinase Ret and the adaptor protein APS. Thus Enigma is implicated in signal transduction processes such as mitogenic activity, insulin related actin organization, and glucose metabolism. Enigma is expressed in multiple tissues, such as skeletal muscle, heart, bone and brain.  LIM domains are 50-60 amino acids in size and share two characteristic zinc finger motifs. The two zinc fingers contain eight conserved residues, mostly cysteines and histidines, which coordinately bond to two zinc atoms. LIM domains function as adaptors or scaffolds to support the assembly of multimeric protein complexes.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2075">COG2075</a></td><td>66</td><td>53</td><td>22.8</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;]</td><td>RPL24A</td><td>Ribosomal protein L24E  </td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3677">COG3677</a></td><td>129</td><td>20</td><td>21.9</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>InsA</td><td>Transposase </td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00019">PRK00019</a></td><td>72</td><td>9</td><td>20.7</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rpmE</td><td>50S ribosomal protein L31; Reviewed</td></tr>
</table>
</body>
