<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04210">cd04210</a></td><td>111</td><td>18</td><td>43.7</td><td>12</td><td>[&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cupredoxin_like_1</td><td>Uncharacterized Cupredoxin-like subfamily. Cupredoxins contain type I copper centers and are involved in inter-molecular electron transfer reactions. Cupredoxins are blue copper proteins because they have an intense blue color due to the presence of a mononuclear type 1 (T1) copper site. Structurally, the cupredoxin-like fold consists of a beta-sandwich with 7 strands in 2 beta-sheets, which is arranged in a Greek-key beta-barrel. Some of these proteins have lost the ability to bind copper. Majority of family members contain multiple cupredoxin domain repeats; ceruloplasmin and coagulation factors V/VIII have six repeats; Laccase, ascorbate oxidase, and spore coat protein A, and multicopper oxidase CueO contain three repeats; and nitrite reductase has two repeats. Others are mono-domain cupredoxins, such as plastocyanin, pseudoazurin, plantacyanin, azurin, rusticyanin, stellacyanin, quinol oxidase and the periplasmic domain of cytochrome c oxidase subunit II.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07833">pfam07833</a></td><td>93</td><td>40</td><td>29.5</td><td>1.1E+02</td><td>[&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cu_amine_oxidN1</td><td>Copper amine oxidase N-terminal domain. Copper amine oxidases catalyse the oxidative deamination of primary amines to the corresponding aldehydes, while reducing molecular oxygen to hydrogen peroxide. These enzymes are dimers of identical subunits, each comprising four domains. The N-terminal domain, which is absent in some amine oxidases, consists of a five-stranded antiparallel beta sheet twisted around an alpha helix. The D1 domains from the two subunits comprise the 'stalk' of the mushroom-shaped dimer, and interact with each other but do not pack tightly against each other.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14558">pfam14558</a></td><td>140</td><td>18</td><td>28.5</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TRP_N</td><td>ML-like domain. This domain is distantly similar to pfam02221 and conserves its pattern of conserved cysteines. This suggests that this domain may be involved in lipid binding.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1036">COG1036</a></td><td>187</td><td>14</td><td>28.2</td><td>47</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;]</td><td>COG1036</td><td>Archaeal flavoprotein  </td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04462">TIGR04462</a></td><td>289</td><td>35</td><td>28.0</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>endura_MppP</td><td>enduracididine biosynthesis enzyme MppP. Members of this family are MppP, one of three enzymes involved in synthesizing enduracididine, a non-proteinogenic amino acid used in non-ribosomal peptide synthases to make natural products such as enduracidin from Streptomyces fungicidicus ATCC 21013. MppP is a PLP-dependent enzyme, predicted by homology to be an aminotransferase.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03423">TIGR03423</a></td><td>592</td><td>36</td><td>25.7</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pbp2_mrdA</td><td>penicillin-binding protein 2. Members of this protein family are penicillin-binding protein 2 (PBP-2), a protein whose gene (designated pbpA or mrdA) generally is found next to the gene for RodA, a protein required for rod (bacillus) shape in many bacteria. PBP-2 acts as a transpeptidase for cell elongation (hence, rod-shape). </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12686">pfam12686</a></td><td>227</td><td>22</td><td>24.4</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3800</td><td>Protein of unknown function (DUF3800). This family of proteins is functionally uncharacterized. This family of proteins is found in bacteria, archaea, eukaryotes and viruses. Proteins in this family are typically between 215 and 302 amino acids in length. There is a DE motif at the N-terminus and a QXXD motif at the C-terminus that may be functionally important.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01780">cd01780</a></td><td>93</td><td>7</td><td>22.5</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLC_epsilon_RA</td><td>Ubiquitin-like domain of Phosphatidylinositide-specific phospholipase. PLC_epsilon_RA   Phosphatidylinositide-specific phospholipase C (PLC) is a signaling enzyme that hydrolyzes membrane phospholipids to generate inositol triphosphate.   PLC-epsilon represents a novel forth class of PLC that has a PLC catalytic core domain, a CDC25 guanine nucleotide exchange factor domain and two RA (Ras-association) domains of which the second is critical for Ras activation of the enzyme.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07142">cd07142</a></td><td>476</td><td>25</td><td>21.1</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ALDH_F2BC</td><td>Arabidosis aldehyde dehydrogenase family 2 B4, B7, C4-like. Included in this CD is the Arabidosis aldehyde dehydrogenase family 2 members B4 and B7 (EC=1.2.1.3),  which are mitochondrial homotetramers that oxidize acetaldehyde and glycolaldehyde, but not L-lactaldehyde. Also in this group, is the Arabidosis cytosolic, homotetramer ALDH2C4 (EC=1.2.1.3), an enzyme involved in the oxidation of sinapalehyde and coniferaldehyde.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09695">cd09695</a></td><td>76</td><td>25</td><td>20.7</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx16_III-U</td><td>CRISPR/Cas system-associated protein Csx16. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Small protein often seen in proximity to Csx1 (COG1517) family; also known as VVA1548 family</td></tr>
</table>
</body>
