<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11860">cd11860</a></td><td>63</td><td>43</td><td>56.1</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH3_DLG5</td><td>Src homology 3 domain of Disks Large homolog 5. DLG5 is a multifunctional scaffold protein that is located at sites of cell-cell contact and is involved in the maintenance of cell shape and polarity. Mutations in the DLG5 gene are associated with Crohn's disease (CD) and inflammatory bowel disease (IBD). DLG5 is a member of the MAGUK (membrane-associated guanylate kinase) protein family, which is characterized by the presence of a core of three domains: PDZ, SH3, and guanylate kinase (GuK). The GuK domain in MAGUK proteins is enzymatically inactive; instead, the domain mediates protein-protein interactions and associates intramolecularly with the SH3 domain. DLG5 contains 4 PDZ domains as well as an N-terminal domain of unknown function. SH3 domains are protein interaction domains that bind to proline-rich ligands with moderate affinity and selectivity, preferentially to PxxP motifs. They play versatile and diverse roles in the cell including the regulation of enzymes, changing the subcellular localization of signaling pathway components, and mediating the formation of multiprotein complex assemblies.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00230">TIGR00230</a></td><td>234</td><td>45</td><td>39.8</td><td>21</td><td>[----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>sfsA</td><td>sugar fermentation stimulation protein. probable regulatory factor involved in maltose metabolism contains a putative DNA binding domain. Isolated as a gene which enabled E.coli strain MK2001 to use maltose. </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12306">PRK12306</a></td><td>519</td><td>72</td><td>28.9</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>uvrC</td><td>excinuclease ABC subunit C; Reviewed</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12725">cd12725</a></td><td>86</td><td>29</td><td>23.6</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM2_CPEB1</td><td>RNA recognition motif 2 in cytoplasmic polyadenylation element-binding protein 1 (CPEB-1) and similar proteins. This subgroup corresponds to the RRM2 of CPEB-1 (also termed CPE-BP1 or CEBP), an RNA-binding protein that interacts with the cytoplasmic polyadenylation element (CPE), a short U-rich motif in the 3' untranslated regions (UTRs) of certain mRNAs. It functions as a translational regulator that plays a major role in the control of maternal CPE-containing mRNA in oocytes, as well as of subsynaptic CPE-containing mRNA in neurons. Once phosphorylated and recruiting the polyadenylation complex, CPEB-1 may function as a translational activator stimulating polyadenylation and translation. Otherwise, it may function as a translational inhibitor when dephosphorylated and bound to a protein such as maskin or neuroguidin, which blocks translation initiation through interfering with the assembly of eIF-4E and eIF-4G. Although CPEB-1 is mainly located in cytoplasm, it can shuttle between nucleus and cytoplasm. CPEB-1 contains an N-terminal unstructured region, two RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains), and a Zn-finger motif. Both of the RRMs and the Zn finger are required for CPEB-1 to bind CPE. The N-terminal regulatory region may be responsible for CPEB-1 interacting with other proteins. </td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13312">cd13312</a></td><td>103</td><td>57</td><td>23.4</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PH_USP37_like</td><td>Pleckstrin homology-like domain of Ubiquitin carboxyl-terminal hydrolase 37. Members here include USP37, USP29, and USP26. All of these contain a single PH-like domain. USP37 (also called ubiquitin carboxyl-terminal hydrolase 37, ubiquitin thiolesterase 37, deubiquitinating enzyme 37, and tmp_locus_50) is a deubiquitinase that antagonizes the anaphase-promoting complex (APC/C) during G1/S transition by mediating deubiquitination of cyclin-A (CCNA1 and CCNA2), resulting in promoting S phase entry. USP37 mediates deubiquitination of 'Lys-11'-linked polyubiquitin chains, a specific ubiquitin-linkage type mediated by the APC/C complex and 'Lys-48'-linked polyubiquitin chains in vitro. Phosphorylation at Ser-628 during G1/S phase maximizes the deubiquitinase activity, leading to prevent degradation of cyclin-A (CCNA1 and CCNA2). USP29 (also called ubiquitin carboxyl-terminal hydrolase 29, ubiquitin thiolesterase 29, deubiquitinating enzyme 29, and HOM-TES-84/86) plays a role in apoptosis and oxidative stress. In response to oxidative stress, JTV1 dissociates from the ARS complex, translocates to the nucleus, associates with far upstream element binding protein (FBP) and co-activates the transcription of USP29 which binds to, cleaves poly-ubiquitin chains from, and stabilizes p53 leading to apoptosis. The X-linked deubiquitination enzyme USP26 (also called ubiquitin carboxyl-terminal hydrolase 26, ubiquitin thiolesterase 26, and deubiquitinating enzyme 26) is a regulator of androgen receptor (AR) signaling. It binds to AR using three nuclear receptor interaction motifs (LXXLL, FXXLF and FXXFF) and modulates AR ubiquitination. Polymorphism of Usp26 correlates with idiopathic male infertility. PH domains have diverse functions, but in general are involved in targeting proteins to the appropriate cellular location or in the interaction with a binding partner. They share little sequence conservation, but all have a common fold, which is electrostatically polarized. Less than 10% of PH domains bind phosphoinositide phosphates (PIPs) with high affinity and specificity. PH domains are distinguished from other PIP-binding domains by their specific high-affinity binding to PIPs with two vicinal phosphate groups: PtdIns(3,4)P2, PtdIns(4,5)P2 or PtdIns(3,4,5)P3 which results in targeting some PH domain proteins to the plasma membrane. A few display strong specificity in lipid binding. Any specificity is usually determined by loop regions or insertions in the N-terminus of the domain, which are not conserved across all PH domains. PH domains are found in cellular signaling proteins such as serine/threonine kinase, tyrosine kinases, regulators of G-proteins, endocytotic GTPases, adaptors, as well as cytoskeletal associated molecules and in lipid associated enzymes.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14667">cd14667</a></td><td>90</td><td>34</td><td>19.9</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3D_containing_proteins</td><td>Non-mltA associated 3D domain containing proteins, named for 3 conserved aspartate residues. This family contains the 3D domain, named for its 3 conserved aspartates, including similar uncharacterized proteins. These proteins contain the critical active site aspartate of mltA-like lytic transglycosylases where the 3D domain forms a larger domain with the N-terminal region. This domain is also found in conjunction with numerous other domains such as the Escherichia coli MltA, a membrane-bound lytic transglycosylase comprised of 2 domains separated by a large groove, where the peptidoglycan strand binds. Domain A is made up of an N-terminal and a C-terminal portion, corresponding to the 3D domain and Domain B is inserted within the linear sequence of domain A.  MltA is distinct from other bacterial LTs, which are similar to each other. Escherichia coli peptidoglycan lytic transglycosylase (LT) initiates cell wall recycling in response to damage, during bacterial fission, and cleaves peptidoglycan (PG) to create functional spaces in its wall. PG chains (also known as murein), the major components of the bacterial cell wall, are comprised of alternating beta-1-4-linked N-acetylmuramic acid (MurNAc) and N-acetyl-D-glucosamine (GlcNAc), and lytic transglycosylases cleave this beta-1-4 bond.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12042">cd12042</a></td><td>68</td><td>43</td><td>16.9</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH3_CACNB3</td><td>Src Homology 3 domain of Voltage-dependent L-type calcium channel subunit beta3. The beta3 subunit of voltage-dependent calcium channels (Ca(V)s) is one of four beta subunits present in vertebrates. It is the main beta subunit present in smooth muscles and is strongly expressed in the brain; it is predominant in the olfactory bulb, cortex, and hippocampus. It may play a role in regulating the NMDAR (N-methyl-d-aspartate receptor) activity in the hippocampus and thus, activity-dependent synaptic plasticity and cognitive behaviors. Ca(V)s are multi-protein complexes that regulate the entry of calcium into cells. They impact muscle contraction, neuronal migration, hormone and neurotransmitter release, and the activation of calcium-dependent signaling pathways. They are composed of four subunits: alpha1, alpha2delta, beta, and gamma. The beta subunit is a soluble and intracellular protein that interacts with the transmembrane alpha1 subunit. It facilitates the trafficking and proper localization of the alpha1 subunit to the cellular plasma membrane. Vertebrates contain four different beta subunits from distinct genes (beta1-4); each exists as multiple splice variants. All are expressed in the brain while other tissues show more specific expression patterns. The beta subunits show similarity to MAGUK (membrane-associated guanylate kinase) proteins in that they contain SH3 and inactive guanylate kinase (GuK) domains; however, they do not appear to contain a PDZ domain. SH3 domains are protein interaction domains that bind to proline-rich ligands with moderate affinity and selectivity, preferentially to PxxP motifs. They play versatile and diverse roles in the cell including the regulation of enzymes, changing the subcellular localization of signaling pathway components, and mediating the formation of multiprotein complex assemblies.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09183">pfam09183</a></td><td>65</td><td>14</td><td>15.8</td><td>80</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1947</td><td>Domain of unknown function (DUF1947). Members of this family are found in a set of hypothetical Archaeal proteins. Their exact function has not, as yet, been defined.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10049">pfam10049</a></td><td>50</td><td>37</td><td>14.3</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2283</td><td>Protein of unknown function (DUF2283). Members of this family of hypothetical bacterial proteins have no known function.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03866">TIGR03866</a></td><td>310</td><td>86</td><td>13.0</td><td>2.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PQQ_ABC_repeats</td><td>PQQ-dependent catabolism-associated beta-propeller protein. Members of this protein family consist of seven repeats each of the YVTN family beta-propeller repeat (see TIGR02276). Members occur invariably as part of a transport operon that is associated with PQQ-dependent catabolism of alcohols such as phenylethanol.</td></tr>
</table>
</body>
