<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03698">TIGR03698</a></td><td>107</td><td>95</td><td>100.0</td><td>1.3E-28</td><td>[&nbsp;&nbsp;&nbsp;---------------------------------------------&nbsp;]</td><td>clan_AA_DTGF</td><td>clan AA aspartic protease, AF_0612 family. Members of this protein family are clan AA aspartic proteases, related to family TIGR02281. These proteins resemble retropepsins, pepsin-like proteases of retroviruses such as HIV. Members of this family are found in archaea and bacteria. </td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5550">COG5550</a></td><td>125</td><td>98</td><td>99.4</td><td>5.8E-13</td><td>[&nbsp;&nbsp;&nbsp;----------------------------------------------]</td><td>COG5550</td><td>Predicted aspartyl protease  </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05484">cd05484</a></td><td>91</td><td>85</td><td>97.7</td><td>0.00086</td><td>[&nbsp;----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>retropepsin_like_LTR_2</td><td>Retropepsins_like_LTR, pepsin-like aspartate proteases. Retropepsin of retrotransposons with long terminal repeats are pepsin-like aspartate proteases. While fungal and mammalian pepsins are bilobal proteins with structurally related N- and C-termini, retropepsins are half as long as their fungal and mammalian counterparts. The monomers are structurally related to one lobe of the pepsin molecule and retropepsins function as homodimers. The active site aspartate occurs within a motif (Asp-Thr/Ser-Gly), as it does in pepsin. Retroviral aspartyl protease is synthesized as part of the POL polyprotein that contains an aspartyl protease, a reverse transcriptase, RNase H, and an integrase. The POL polyprotein undergoes specific enzymatic cleavage to yield the mature proteins. In aspartate peptidases, Asp residues are ligands of an activated water molecule in all examples where catalytic residues have been identified. This group of aspartate peptidases is classified by MEROPS as the peptidase family A2 (retropepsin family, clan AA), subfamily A2A.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13650">pfam13650</a></td><td>89</td><td>64</td><td>97.6</td><td>0.00055</td><td>[&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Asp_protease_2</td><td>Aspartyl protease. This family consists of predicted aspartic proteases, typically from 180 to 230 amino acids in length, in MEROPS clan AA. This model describes the well-conserved 121-residue C-terminal region. The poorly conserved, variable length N-terminal region usually contains a predicted transmembrane helix.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00077">pfam00077</a></td><td>99</td><td>90</td><td>97.0</td><td>0.0034</td><td>[&nbsp;--------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RVP</td><td>Retroviral aspartyl protease. Single domain aspartyl proteases from retroviruses, retrotransposons, and badnaviruses (plant dsDNA viruses). These proteases are generally part of a larger polyprotein; usually pol, more rarely gag. Retroviral proteases appear to be homologous to a single domain of the two-domain eukaryotic aspartyl proteases such as pepsins, cathepsins, and renins (pfam00026).</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00303">cd00303</a></td><td>92</td><td>74</td><td>96.9</td><td>0.012</td><td>[&nbsp;------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>retropepsin_like</td><td>Retropepsins; pepsin-like aspartate proteases. The family includes pepsin-like aspartate proteases from retroviruses, retrotransposons and retroelements, as well as eukaryotic dna-damage-inducible proteins (DDIs), and bacterial aspartate peptidases. While fungal and mammalian pepsins are bilobal proteins with structurally related N and C-terminals, retropepsins are half as long as their fungal and mammalian counterparts. The monomers are structurally related to one lobe of the pepsin molecule and retropepsins function as homodimers. The active site aspartate occurs within a motif (Asp-Thr/Ser-Gly), as it does in pepsin. Retroviral aspartyl protease is synthesized as part of the POL polyprotein that contains an aspartyl protease, a reverse transcriptase, RNase H, and an integrase. The POL polyprotein undergoes specific enzymatic cleavage to yield the mature proteins. In aspartate peptidases, Asp residues are ligands of an activated water molecule in all examples where catalytic residues have been identified. This group of aspartate peptidases is classified by MEROPS as the peptidase family A2 (retropepsin family, clan AA), subfamily A2A.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13975">pfam13975</a></td><td>72</td><td>56</td><td>96.5</td><td>0.01</td><td>[&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gag-asp_proteas</td><td>gag-polyprotein putative aspartyl protease. This family of putative aspartyl proteases is found pre-dominantly in retroviral proteins.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02281">TIGR02281</a></td><td>121</td><td>90</td><td>95.6</td><td>0.17</td><td>[-------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>clan_AA_DTGA</td><td>clan AA aspartic protease, TIGR02281 family. This family consists of predicted aspartic proteases, typically from 180 to 230 amino acids in length, in MEROPS clan AA. This model describes the well-conserved 121-residue C-terminal region. The poorly conserved, variable length N-terminal region usually contains a predicted transmembrane helix. Sequences in the seed alignment and those scoring above the trusted cutoff are Proteobacterial; homologs scroing between trusted and noise are found in Pyrobaculum aerophilum str. IM2 (archaeal), Pirellula sp. (Planctomycetes), and Nostoc sp. PCC 7120 (Cyanobacteria). </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05483">cd05483</a></td><td>96</td><td>63</td><td>94.8</td><td>0.28</td><td>[&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>retropepsin_like_bacteria</td><td>Bacterial aspartate proteases, retropepsin-like protease family. This family of bacteria aspartate proteases is a subfamily of retropepsin-like protease family, which includes enzymes from retrovirus and retrotransposons. While fungal and mammalian pepsin-like aspartate proteases are bilobal proteins with structurally related N- and C-termini, this family of bacteria aspartate proteases is half as long as their fungal and mammalian counterparts. The monomers are structurally related to one lobe of the pepsin molecule and function as homodimers. The active site aspartate occurs within a motif (Asp-Thr/Ser-Gly), as it does in pepsin. In aspartate peptidases, Asp residues are ligands of an activated water molecule in all examples where catalytic residues have been identified. This group of aspartate proteases is classified by MEROPS as the peptidase family A2 (retropepsin family, clan AA), subfamily A2A.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3577">COG3577</a></td><td>215</td><td>89</td><td>93.9</td><td>0.36</td><td>[-------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG3577</td><td>Predicted aspartyl protease  </td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05481">cd05481</a></td><td>93</td><td>69</td><td>93.6</td><td>0.42</td><td>[&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>retropepsin_like_LTR_1</td><td>Retropepsins_like_LTR; pepsin-like aspartate protease from retrotransposons with long terminal repeats. Retropepsin of retrotransposons with long terminal repeats are pepsin-like aspartate proteases. While fungal and mammalian pepsins are bilobal proteins with structurally related N and C-terminals, retropepsins are half as long as their fungal and mammalian counterparts. The monomers are structurally related to one lobe of the pepsin molecule and retropepsins function as homodimers. The active site aspartate occurs within a motif (Asp-Thr/Ser-Gly), as it does in pepsin. Retroviral aspartyl protease is synthesized as part of the POL polyprotein that contains an aspartyl protease, a reverse transcriptase, RNase H, and an integrase. The POL polyprotein undergoes specific enzymatic cleavage to yield the mature proteins. In aspartate peptidases, Asp residues are ligands of an activated water molecule in all examples where catalytic residues have been identified. This group of aspartate peptidases is classified by MEROPS as the peptidase family A2 (retropepsin family, clan AA), subfamily A2A.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05479">cd05479</a></td><td>124</td><td>90</td><td>93.5</td><td>0.49</td><td>[&nbsp;-----------------------------------------------&nbsp;]</td><td>RP_DDI</td><td>RP_DDI; retropepsin-like domain of DNA damage inducible protein. The family represents the retropepsin-like domain of DNA damage inducible protein. DNA damage inducible protein has a retropepsin-like domain and an amino-terminal ubiquitin-like domain and/or a UBA (ubiquitin-associated) domain. This CD represents the retropepsin-like domain of DDI.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07419">PRK07419</a></td><td>304</td><td>14</td><td>79.9</td><td>0.54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07419</td><td>1,4-dihydroxy-2-naphthoate octaprenyltransferase; Provisional</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02235">TIGR02235</a></td><td>285</td><td>13</td><td>78.8</td><td>0.65</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>menA_cyano-plnt</td><td>1,4-dihydroxy-2-naphthoate phytyltransferase. This family of phytyltransferases, found in plants and cyanobacteria, are involved in the biosythesis of phylloquinone (Vitamin K1). Phylloquinone is a critical component of photosystem I. The closely related MenA enzyme from bacteria transfers a prenyl group (which only differs in the saturation of the isoprenyl groups) in the biosynthesis of menaquinone. Activity towards both substrates in certain organisms should be considered a possibility. </td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13975">pfam13975</a></td><td>72</td><td>37</td><td>70.7</td><td>6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gag-asp_proteas</td><td>gag-polyprotein putative aspartyl protease. This family of putative aspartyl proteases is found pre-dominantly in retroviral proteins.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06095">cd06095</a></td><td>86</td><td>18</td><td>59.2</td><td>7.8</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RP_RTVL_H_like</td><td>Retropepsin of the RTVL_H family of human endogenous retrovirus-like elements. This family includes aspartate proteases from retroelements with LTR (long terminal repeats) including the RTVL_H family of human endogenous retrovirus-like elements. While fungal and mammalian pepsins are bilobal proteins with structurally related N- and C-termini, retropepsins are half as long as their fungal and mammalian counterparts. The monomers are structurally related to one lobe of the pepsin molecule and retropepsins function as homodimers. The active site aspartate occurs within a motif (Asp-Thr/Ser-Gly), as it does in pepsin. Retroviral aspartyl protease is synthesized as part of the POL polyprotein that contains an aspartyl protease, a reverse transcriptase, RNase H, and an integrase. The POL polyprotein undergoes specific enzymatic cleavage to yield the mature proteins. In aspartate peptidases, Asp residues are ligands of an activated water molecule in all examples where catalytic residues have been identified. This group of aspartate peptidases is classified by MEROPS as the peptidase family A2 (retropepsin family, clan AA), subfamily A2A.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02070">pfam02070</a></td><td>25</td><td>15</td><td>58.6</td><td>4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NMU</td><td>Neuromedin U. </td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08757">pfam08757</a></td><td>298</td><td>52</td><td>52.0</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CotH</td><td>CotH protein. Members of this family include the spore coat protein H (cotH).</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09668">pfam09668</a></td><td>124</td><td>89</td><td>50.8</td><td>13</td><td>[&nbsp;--------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Asp_protease</td><td>Aspartyl protease. This family of eukaryotic aspartyl proteases have a fold similar to retroviral proteases which implies they function proteolytically during regulated protein turnover.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05482">cd05482</a></td><td>87</td><td>84</td><td>47.7</td><td>84</td><td>[&nbsp;----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HIV_retropepsin_like</td><td>Retropepsins, pepsin-like aspartate proteases. This is a subfamily of retropepsins. The family includes pepsin-like aspartate proteases from retroviruses, retrotransposons and retroelements. While fungal and mammalian pepsins are bilobal proteins with structurally related N- and C-termini, retropepsins are half as long as their fungal and mammalian counterparts. The monomers are structurally related to one lobe of the pepsin molecule and retropepsins function as homodimers. The active site aspartate occurs within a motif (Asp-Thr/Ser-Gly), as it does in pepsin. Retroviral aspartyl protease is synthesized as part of the POL polyprotein that contains an aspartyl protease, a reverse transcriptase, RNase H, and an integrase. The POL polyprotein undergoes specific enzymatic cleavage to yield the mature proteins. In aspartate peptidases, Asp residues are ligands of an activated water molecule in all examples where catalytic residues have been identified. This group of aspartate peptidases is classified by MEROPS as the peptidase family A2 (retropepsin family, clan AA), subfamily A2A.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03419">pfam03419</a></td><td>291</td><td>79</td><td>44.9</td><td>1.2E+02</td><td>[--------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_U4</td><td>Sporulation factor SpoIIGA. </td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03807">TIGR03807</a></td><td>27</td><td>22</td><td>41.0</td><td>8.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RR_fam_repeat</td><td>putative cofactor-binding repeat. This model describes a small repeat found in a family of proteins that crosses the plasma membrane by twin-arginine translation, which usually signifies the presence of a bound cofactor. This repeat shows similarity to the beta-helical repeat, in which three beta-strands per repeat wind once per repeat around in a right-handed helical stack of parallel beta structure.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07557">cd07557</a></td><td>92</td><td>37</td><td>35.8</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>trimeric_dUTPase</td><td>Trimeric dUTP diphosphatases. Trimeric dUTP diphosphatases, or dUTPases, are the most common family of dUTPase, found in bacteria, eukaryotes, and archaea. They catalyze the hydrolysis of the dUTP-Mg complex (dUTP-Mg) into dUMP and pyrophosphate. This reaction is crucial for the preservation of chromosomal integrity as it removes dUTP and therefore reduces the cellular dUTP/dTTP ratio, and prevents dUTP from being incorporated into DNA.  It also provides dUMP as the precursor for dTTP synthesis via the thymidylate synthase pathway. dUTPases are homotrimeric, except some monomeric viral dUTPases, which have been shown to mimic a trimer. Active sites are located at the subunit interface.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13739">cd13739</a></td><td>168</td><td>67</td><td>35.1</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;]</td><td>SPRY_PRY_TRIM1</td><td>PRY/SPRY domain of tripartite motif-binding protein 1 (TRIM1) or MID2. This domain, consisting of the distinct N-terminal PRY subdomain followed by the SPRY subdomain, is found at the C-terminus of TRIM1 (also known as MID2 or midline 2). MID2 and its close homolog, TRIM18 (also known as MID1), both contain a B30.2-like domain at their C-terminus and a single fibronectin type III (FN3) motif between it and their N-terminal RBCC domain. MID2 and MID1 coiled-coil motifs mediate both homo- and heterodimerization, a prerequisite for association of the rapamycin-sensitive PP2A regulatory subunit Alpha 4 with microtubules. Mutations in MID1 have shown to cause Opitz syndrome, a disorder causing congenital anomalies such as cleft lip and palate as well as heart defects.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09856">pfam09856</a></td><td>156</td><td>41</td><td>34.6</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2083</td><td>Predicted transcriptional regulator (DUF2083). This domain is found in various prokaryotic transcriptional regulatory proteins belonging to the XRE family. Its exact function is, as yet, unknown.</td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12892">cd12892</a></td><td>177</td><td>15</td><td>34.4</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;]</td><td>SPRY_PRY_TRIM18</td><td>PRY/SPRY domain of TRIM18/MID1, also known as FXY or RNF59. This domain, consisting of the distinct N-terminal PRY subdomain followed by the SPRY subdomain, is at the C-terminus of the overall domain architecture of MID1 (also known as FXY, RNF59, TRIM18) gene represented by a RING finger domain (RING), two B-box motifs (BBOX), coiled-coil C-terminal to Bbox domain (BBC) and fibronectin type 3 domain (FN3). Mutations in the human MID1 gene result in X-linked Opitz G/BBB syndrome (OS), a disorder affecting development of midline structures, causing craniofacial, urogenital, gastrointestinal and cardiovascular abnormalities. A unique MID1 gene mutation located in a variable loop in the SPRY domain alters conformation of the binding pocket and may affect the binding affinity to the PRY/SPRY domain.</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00893">cd00893</a></td><td>286</td><td>26</td><td>33.6</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PI4Kc_III</td><td>Catalytic domain of Type III Phosphoinositide 4-kinase. PI4Ks catalyze the transfer of the gamma-phosphoryl group from ATP to the 4-hydroxyl of the inositol ring of D-myo-phosphatidylinositol (PtdIns) to generate PtdIns(4)P, the major precursor in the synthesis of other phosphoinositides including PtdIns(4,5)P2, PtdIns(3,4)P2, and PtdIns(3,4,5)P3. There are two types of PI4Ks, types II and III. Type II PI4Ks lack the characteristic catalytic kinase domain present in PI3Ks and type III PI4Ks, and are excluded from this family. Two isoforms of type III PI4K, alpha and beta, exist in most eukaryotes. The PI4K catalytic domain family is part of a larger superfamily that includes the catalytic domains of other kinases such as the typical serine/threonine/tyrosine protein kinases (PKs), aminoglycoside phosphotransferase, choline kinase, and RIO kinases.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01415">cd01415</a></td><td>230</td><td>80</td><td>33.2</td><td>69</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------------]</td><td>SAICAR_synt_PurC</td><td>bacterial and archaeal 5-aminoimidazole-4-(N-succinylcarboxamide) ribonucleotide (SAICAR) synthase. A subfamily of SAICAR synthetases represented by the Thermotoga maritima (Tm) enzyme and E. coli PurC. SAICAR synthetase catalyzes the seventh step of the de novo biosynthesis of purine nucleotides (also reported as eighth step). It converts 5-aminoimidazole-4-carboxyribonucleotide (CAIR), ATP, and L-aspartate into 5-aminoimidazole-4-(N-succinylcarboxamide) ribonucleotide (SAICAR), ADP, and phosphate.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01324">pfam01324</a></td><td>154</td><td>56</td><td>33.2</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Diphtheria_R</td><td>Diphtheria toxin, R domain. C-terminal receptor binding (R) domain - binds to cell surface receptor, permitting the toxin to enter the cell by receptor mediated endocytosis.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12280">cd12280</a></td><td>81</td><td>28</td><td>29.6</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM_FET</td><td>RNA recognition motif in the FET family of RNA-binding proteins. This subfamily corresponds to the RRM of FET (previously TET) (FUS/TLS, EWS, TAF15) family of RNA-binding proteins. This ubiquitously expressed family of similarly structured proteins predominantly localizing to the nuclear, includes FUS (also known as TLS or Pigpen or hnRNP P2), EWS (also known as EWSR1), TAF15 (also known as hTAFII68 or TAF2N or RPB56), and Drosophila Cabeza (also known as SARFH). The corresponding coding genes of these proteins are involved in deleterious genomic rearrangements with transcription factor genes in a variety of human sarcomas and acute leukemias. All FET proteins interact with each other and are therefore likely to be part of the very same protein complexes, which suggests a general bridging role for FET proteins coupling RNA transcription, processing, transport, and DNA repair. The FET proteins contain multiple copies of a degenerate hexapeptide repeat motif at the N-terminus. The C-terminal region consists of a conserved nuclear import and retention signal (C-NLS), a putative zinc-finger domain, and a conserved RNA recognition motif (RRM), also known as RBD (RNA binding domain) or RNP (ribonucleoprotein domain), which is flanked by 3 arginine-glycine-glycine (RGG) boxes. FUS and EWS might have similar sequence specificity; both bind preferentially to GGUG-containing RNAs. FUS has also been shown to bind strongly to human telomeric RNA and to small low-copy-number RNAs tethered to the promoter of cyclin D1. To date, nothing is known about the RNA binding specificity of TAF15. </td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05931">cd05931</a></td><td>547</td><td>52</td><td>29.5</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FAAL</td><td>Fatty acyl-AMP ligase (FAAL). FAAL belongs to the class I adenylate forming enzyme family and is homologous to fatty acyl-coenzyme A (CoA) ligases (FACLs). However, FAALs produce only the acyl adenylate and are unable to perform the thioester-forming reaction, while FACLs perform a two-step catalytic reaction; AMP ligation followed by CoA ligation using ATP and CoA as cofactors. FAALs have insertion motifs between the N-terminal and C-terminal subdomains that distinguish them from the FACLs. This insertion motif precludes the binding of CoA, thus preventing CoA ligation. It has been suggested that the acyl adenylates serve as substrates for multifunctional polyketide synthases to permit synthesis of complex lipids such as phthiocerol dimycocerosate, sulfolipids, mycolic acids, and mycobactin.</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05204">pfam05204</a></td><td>110</td><td>44</td><td>29.1</td><td>87</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;]</td><td>Hom_end</td><td>Homing endonuclease. Homing endonucleases are encoded by mobile DNA elements that are found inserted within host genes in all domains of life.</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14191">pfam14191</a></td><td>103</td><td>41</td><td>28.4</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YodL</td><td>YodL-like. The YodL-like protein family includes the B. subtilis YodL protein, which is functionally uncharacterized. This domain family is found in bacteria, and is approximately 100 amino acids in length. There are two completely conserved residues (Y and D) that may be functionally important.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13189">pfam13189</a></td><td>174</td><td>25</td><td>26.1</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cytidylate_kin2</td><td>Cytidylate kinase-like family. This family includes enzymes related to cytidylate kinase.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11027">pfam11027</a></td><td>103</td><td>13</td><td>26.0</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2615</td><td>Protein of unknown function (DUF2615). This small. approximately 100 residue, family is conserved from worms to humans. It is cysteine-rich with a characteristic FDxCEC sequence motif. The function is not known.</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03840">TIGR03840</a></td><td>213</td><td>16</td><td>25.4</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TMPT_Se_Te</td><td>thiopurine S-methyltransferase, Se/Te detoxification family. Members of this family are thiopurine S-methyltransferase from a branch in which at least some member proteins can perform selenium methylation as a means to detoxify selenium, or perform a related detoxification of tellurium. Note that the EC number definition does not specify a particular thiopurine, but rather represents a class of activity.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02791">cd02791</a></td><td>122</td><td>25</td><td>25.0</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MopB_CT_Nitrate-R-NapA-like</td><td>Nitrate reductases, NapA (Nitrate-R-NapA), NasA, and NarB catalyze the reduction of nitrate to nitrite. Monomeric Nas is located in the cytoplasm and participates in nitrogen assimilation. Dimeric Nap is located in the periplasm and is coupled to quinol oxidation via a membrane-anchored tetraheme cytochrome. This CD (MopB_CT_Nitrate-R-Nap) is of the conserved molybdopterin_binding C-terminal (MopB_CT) region present in many, but not all, MopB homologs</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11513">pfam11513</a></td><td>110</td><td>30</td><td>24.0</td><td>82</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TA0956</td><td>Thermoplasma acidophilum protein TA0956. TA0956 is a protein from Thermoplasma acidophilum which currently has no known function however the structure has been determined. The protein has a two-layered alpha/beta-sandwich topology and is a putative Elongation factor 1-alpha binding motif.</td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09362">PRK09362</a></td><td>238</td><td>80</td><td>23.9</td><td>86</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------------]</td><td>PRK09362</td><td>phosphoribosylaminoimidazole-succinocarboxamide synthase; Reviewed</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07582">pfam07582</a></td><td>55</td><td>15</td><td>23.8</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AP_endonuc_2_N</td><td>AP endonuclease family 2 C terminus. This highly-conserved sequence is found at the C terminus of several apurinic/apyrimidinic (AP) endonucleases. in a range of Gram-positive and Gram-negative bacteria. See also pfam01261.</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06094">cd06094</a></td><td>89</td><td>75</td><td>22.2</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RP_Saci_like</td><td>RP_Saci_like, retropepsin family. Retropepsin on retrotransposons with long terminal repeats (LTR) including Saci-1, -2 and -3 of Schistosoma mansoni. Retropepsins are related to fungal and mammalian pepsins. While fungal and mammalian pepsins are bilobal proteins with structurally related N- and C-termini, retropepsins are half as long as their fungal and mammalian counterparts. The monomers are structurally related to one lobe of the pepsin molecule and retropepsins function as homodimers. The active site aspartate occurs within a motif (Asp-Thr/Ser-Gly), as it does in pepsin. Retroviral aspartyl protease is synthesized as part of the POL polyprotein that contains an aspartyl protease, a reverse transcriptase, RNase H, and an integrase. The POL polyprotein undergoes specific enzymatic cleavage to yield the mature proteins. In aspartate peptidases, Asp residues are ligands of an activated water molecule in all examples where catalytic residues have been identified. This group of aspartate peptidases is classified by MEROPS as the peptidase family A2 (retropepsin family, clan AA), subfamily A2A.</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04134">pfam04134</a></td><td>113</td><td>15</td><td>21.1</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF393</td><td>Protein of unknown function, DUF393. Members of this family have two highly conserved cysteine residues near their N-terminus. The function of these proteins is unknown.</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0462">COG0462</a></td><td>314</td><td>48</td><td>20.7</td><td>92</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;]</td><td>PrsA</td><td>Phosphoribosylpyrophosphate synthetase </td></tr>
</table>
</body>
