<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07021">pfam07021</a></td><td>193</td><td>70</td><td>78.6</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MetW</td><td>Methionine biosynthesis protein MetW. This family consists of several bacterial and one archaeal methionine biosynthesis MetW proteins. Biosynthesis of methionine from homoserine in Pseudomonas putida takes place in three steps. The first step is the acylation of homoserine to yield an acyl-L-homoserine. This reaction is catalysed by the products of the metXW genes and is equivalent to the first step in enterobacteria, gram-positive bacteria and fungi, except that in these microorganisms the reaction is catalysed by a single polypeptide (the product of the metA gene in Escherichia coli and the met5 gene product in Neurospora crassa). In Pseudomonas putida, as in gram-positive bacteria and certain fungi, the second and third steps are a direct sulfhydrylation that converts the O-acyl-L-homoserine into homocysteine and further methylation to yield methionine. The latter reaction can be mediated by either of the two methionine synthetases present in the cells.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04085">TIGR04085</a></td><td>93</td><td>11</td><td>58.5</td><td>4</td><td>[&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rSAM_more_4Fe4S</td><td>radical SAM additional 4Fe4S-binding SPASM domain. This domain contains regions binding additional 4Fe4S clusters found in various radical SAM proteins C-terminal to the domain described by model pfam04055. Radical SAM enzymes with this domain tend to be involved in protein modification, including anaerobic sulfatase maturation proteins, a quinohemoprotein amine dehydrogenase biogenesis protein, the Pep1357-cyclizing radical SAM enzyme, and various bacteriocin biosynthesis proteins. The motif CxxCxxxxxCxxxC is nearly invariant for members of this family, although PqqE has a variant form. We name this domain SPASM for Subtilosin, PQQ, Anaerobic Sulfatase, and Mycofactocin.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12447">pfam12447</a></td><td>117</td><td>14</td><td>57.1</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3683</td><td>Protein of unknown function (DUF3683). This domain family is found in bacteria, and is approximately 120 amino acids in length. The family is found in association with pfam02754, pfam01565, pfam02913.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10974">PRK10974</a></td><td>438</td><td>29</td><td>54.4</td><td>6.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10974</td><td>glycerol-3-phosphate transporter periplasmic binding protein; Provisional</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12621">cd12621</a></td><td>74</td><td>32</td><td>53.0</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM3_TIA1</td><td>RNA recognition motif 3 in nucleolysin TIA-1 isoform p40 (p40-TIA-1) and similar proteins. This subgroup corresponds to the RRM3 of p40-TIA-1, the 40-kDa isoform of T-cell-restricted intracellular antigen-1 (TIA-1) and a cytotoxic granule-associated RNA-binding protein mainly found in the granules of cytotoxic lymphocytes. TIA-1 can be phosphorylated by a serine/threonine kinase that is activated during Fas-mediated apoptosis, and function as the granule component responsible for inducing apoptosis in cytolytic lymphocyte (CTL) targets. It is composed of three N-terminal highly homologous RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains), and a glutamine-rich C-terminal auxiliary domain containing a lysosome-targeting motif. TIA-1 interacts with RNAs containing short stretches of uridylates and its RRM2 can mediate the specific binding to uridylate-rich RNAs. </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10990">pfam10990</a></td><td>91</td><td>51</td><td>50.4</td><td>4.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2809</td><td>Protein of unknown function (DUF2809). Some members in this family of proteins are annotated as yjgA however currently no function for the protein is known.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01625">pfam01625</a></td><td>147</td><td>15</td><td>41.2</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PMSR</td><td>Peptide methionine sulfoxide reductase. This enzyme repairs damaged proteins. Methionine sulfoxide in proteins is reduced to methionine.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08789">cd08789</a></td><td>91</td><td>18</td><td>38.5</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;]</td><td>CARD_IPS-1_RIG-I</td><td>Caspase activation and recruitment domains (CARDs) found in IPS-1 and RIG-I-like RNA helicases. Caspase activation and recruitment domains (CARDs) found in IPS-1 (Interferon beta promoter stimulator protein 1) and Retinoic acid Inducible Gene I (RIG-I)-like DEAD box helicases. RIG-I-like helicases and IPS-1 play important roles in the induction of interferons in response to viral infection. They are crucial in triggering innate immunity and in developing adaptive immunity against viral pathogens. RIG-I-like helicases, including MDA5 and RIG-I, contain two N-terminal CARD domains and a C-terminal DEAD box RNA helicase domain. They are cytoplasmic RNA helicases that play an important role in host antiviral response by sensing incoming viral RNA. Upon activation, the signal is transferred to downstream pathways via the adaptor molecule IPS-1 (MAVS, VISA, CARDIF), leading to the induction of type I interferons. MDA5 and RIG-I associate with IPS-1 through a CARD-CARD interaction. In general, CARDs are death domains (DDs) found associated with caspases. They are known to be important in the signaling pathways for apoptosis, inflammation, and host-defense mechanisms. DDs are protein-protein interaction domains found in a variety of domain architectures. Their common feature is that they form homodimers by self-association or heterodimers by associating with other members of the DD superfamily including PYRIN and DED (Death Effector Domain). They serve as adaptors in signaling pathways and can recruit other proteins into signaling complexes.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02794">cd02794</a></td><td>121</td><td>34</td><td>37.8</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MopB_CT_DmsA-EC</td><td>The MopB_CT_DmsA-EC CD includes the DmsA enzyme of the dmsABC operon encoding the anaerobic dimethylsulfoxide reductase (DMSOR) of Escherichia coli and other related DMSOR-like enzymes. Unlike other DMSOR-like enzymes, this group has a  predicted N-terminal iron-sulfur </td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04762">pfam04762</a></td><td>902</td><td>29</td><td>34.5</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>IKI3</td><td>IKI3 family. Members of this family are components of the elongator multi-subunit component of a novel RNA polymerase II holoenzyme for transcriptional elongation. This region contains WD40 like repeats.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04400">pfam04400</a></td><td>45</td><td>10</td><td>33.7</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF539</td><td>Protein of unknown function (DUF539). Putative periplasmic protein.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04141">TIGR04141</a></td><td>516</td><td>113</td><td>32.5</td><td>76</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------------&nbsp;]</td><td>TIGR04141</td><td>sporadically distributed protein, TIGR04141 family. This model describes a sporadically distributed conserved hypothetical protein in which complete members average over 500 amino acids in length, although matching sequences frequently are truncated or broken into tandem ORFs. Regular co-clustering with known markers of mobility (integrases, transposases, phage proteins, restriction enzymes, etc.) suggests this family also is part of the mobilome. The function is unknown.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08415">pfam08415</a></td><td>57</td><td>36</td><td>30.9</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NRPS</td><td>Nonribosomal peptide synthase. This domain is found in bacterial nonribosomal peptide synthetases (NRPS). NRPS are megaenzymes organized as iterative modules, one for each amino acid to be built into the peptide product. NRPS modules are involved in epothilone biosynthesis (EpoB), myxothiazol biosynthesis (MtaC and MtaD), and other functions. The NRPS domain tends to be found together with the condensation domain (pfam00668) and the phosphopantetheine binding domain (pfam00550).</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12158">cd12158</a></td><td>343</td><td>116</td><td>28.9</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ErythrP_dh</td><td>D-Erythronate-4-Phosphate Dehydrogenase NAD-binding and catalytic domains. D-Erythronate-4-phosphate Dehydrogenase (E. coli gene PdxB), a D-specific 2-hydroxyacid dehydrogenase family member, catalyzes the NAD-dependent oxidation of erythronate-4-phosphate, which is followed by transamination to form 4-hydroxy-L-threonine-4-phosphate within the de novo biosynthesis pathway of vitamin B6. D-Erythronate-4-phosphate dehydrogenase has the common architecture shared with D-isomer specific 2-hydroxyacid dehydrogenases but contains an additional C-terminal dimerization domain in addition to an NAD-binding domain and the "lid" domain. The lid domain corresponds to the catalytic domain of phosphoglycerate dehydrogenase and other proteins of the D-isomer specific 2-hydroxyacid dehydrogenase family, which include groups such as formate dehydrogenase, glycerate dehydrogenase, L-alanine dehydrogenase, and S-adenosylhomocysteine hydrolase. Despite often low sequence identity, these proteins typically have a characteristic arrangement of 2 similar subdomains of the alpha/beta Rossmann fold NAD+ binding form. The NAD+ binding domain is inserted within the linear sequence of the mostly N-terminal catalytic domain, which has a similar domain structure to the internal NAD binding domain. Structurally, these domains are connected by extended alpha helices and create a cleft in which NAD is bound, primarily to the C-terminal portion of the 2nd (internal) domain. Some related proteins have similar structural subdomain but with a tandem arrangement of the catalytic and NAD-binding subdomains in the linear sequence.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02667">cd02667</a></td><td>279</td><td>50</td><td>25.5</td><td>60</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_C19K</td><td>A subfamily of Peptidase C19. Peptidase C19 contains ubiquitinyl hydrolases. They are intracellular peptidases that remove ubiquitin molecules from polyubiquinated peptides by cleavage of isopeptide bonds. They hydrolyze bonds involving the carboxyl group of the C-terminal Gly residue of ubiquitin. The purpose of the de-ubiquitination is thought to be editing of the ubiquitin conjugates, which could rescue them from degradation, as well as recycling of the ubiquitin. The ubiquitin/proteasome system is responsible for most protein turnover in the mammalian cell, and with over 50 members, family C19 is one of the largest families of peptidases in the human genome.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06395">cd06395</a></td><td>91</td><td>30</td><td>24.9</td><td>79</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PB1_Map2k5</td><td>PB1 domain is essential part of the mitogen-activated protein kinase kinase 5 (Map2k5, alias MEK5) one of the key member of the signaling kinases cascade which involved in angiogenesis and early cardiovascular development. The PB1 domain of Map2k5 interacts with the PB1 domain of another members of kinase cascade MEKK2 (or MEKK3).  A canonical PB1-PB1 interaction, involving heterodimerization of two PB1 domain, is required for the formation of macromolecular signaling complexes ensuring specificity and fidelity during cellular signaling. The interaction between two PB1 domain depends on the type of PB1. There are three types of PB1 domains: type I which contains an OPCA motif, acidic aminoacid cluster, type II which contains a basic cluster, and type I/II which contains both an OPCA motif and a basic cluster.  The Map2k5 protein contains a type I PB1 domain.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02272">TIGR02272</a></td><td>335</td><td>34</td><td>23.8</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gentisate_1_2</td><td>gentisate 1,2-dioxygenase. This family consists of gentisate 1,2-dioxygenases. This ring-opening enzyme acts in salicylate degradation that goes via gentisate rather than via catechol. It converts gentisate to maleylpyruvate. Some putative gentisate 1,2-dioxygenases are excluded by a relatively high trusted cutoff score because they are too closely related to known examples of 1-hydroxy-2-naphthoate dioxygenase. Therefore some homologs may be bona fide gentisate 1,2-dioxygenases even if they score below the given cutoffs.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00401">TIGR00401</a></td><td>149</td><td>22</td><td>23.4</td><td>43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>msrA</td><td>methionine-S-sulfoxide reductase. This model describes peptide methionine sulfoxide reductase (MsrA), a repair enzyme for proteins that have been inactivated by oxidation. The enzyme from E. coli is coextensive with this model and has enzymatic activity. However, in all completed genomes in which this module is present, a second protein module, described in TIGR00357, is also found, and in several cases as part of the same polypeptide chain: N-terminal to this module in Helicobacter pylori and Haemophilus influenzae (as in PilB of Neisseria gonorrhoeae) but C-terminal to it in Treponema pallidum. PilB, containing both domains, has been shown to be important for the expression of adhesins in certain pathogens. </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0225">COG0225</a></td><td>174</td><td>22</td><td>23.4</td><td>43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MsrA</td><td>Peptide methionine sulfoxide reductase MsrA </td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12836">pfam12836</a></td><td>65</td><td>24</td><td>23.3</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HHH_3</td><td>Helix-hairpin-helix motif. The HhH domain is a short DNA-binding domain.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02666">TIGR02666</a></td><td>334</td><td>47</td><td>22.9</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>moaA</td><td>molybdenum cofactor biosynthesis protein A, bacterial. The model for this family describes molybdenum cofactor biosynthesis protein A, or MoaA, as found in bacteria. It does not include the family of probable functional equivalent proteins from the archaea. MoaA works together with MoaC to synthesize precursor Z from guanine. </td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00667">TIGR00667</a></td><td>185</td><td>21</td><td>22.8</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>aat</td><td>leucyl/phenylalanyl-tRNA--protein transferase. The N-terminal residue controls the biological half-life of many proteins via the N-end rule pathway. This enzyme transfers a Leu or Phe to the amino end of certain proteins to enable degradation. </td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13352">PRK13352</a></td><td>431</td><td>47</td><td>22.5</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13352</td><td>thiamine biosynthesis protein ThiC; Provisional</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14086">pfam14086</a></td><td>50</td><td>9</td><td>22.0</td><td>39</td><td>[&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4266</td><td>Domain of unknown function (DUF4266). This presumed lipoprotein domain is functionally uncharacterized. This domain family is found in bacteria, and is approximately 50 amino acids in length.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14054">PRK14054</a></td><td>172</td><td>15</td><td>20.7</td><td>53</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14054</td><td>methionine sulfoxide reductase A; Provisional</td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02657">cd02657</a></td><td>305</td><td>42</td><td>20.1</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_C19A</td><td>A subfamily of Peptidase C19. Peptidase C19 contains ubiquitinyl hydrolases. They are intracellular peptidases that remove ubiquitin molecules from polyubiquinated peptides by cleavage of isopeptide bonds. They hydrolyse bonds involving the carboxyl group of the C-terminal Gly residue of ubiquitin. The purpose of the de-ubiquitination is thought to be editing of the ubiquitin conjugates, which could rescue them from degradation, as well as recycling of the ubiquitin. The ubiquitin/proteasome system is responsible for most protein turnover in the mammalian cell, and with over 50 members, family C19 is one of the largest families of peptidases in the human genome.</td></tr>
</table>
</body>
