<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03985">TIGR03985</a></td><td>248</td><td>247</td><td>100.0</td><td>5E-115</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>TIGR03985</td><td>CRISPR-associated protein, TIGR03985 family. Members of this protein family belong to CRISPR-associated (Cas) gene clusters. The majority of members are Cyanobacterial.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13280">pfam13280</a></td><td>173</td><td>157</td><td>99.1</td><td>5.5E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------]</td><td>WYL</td><td>WYL domain. This presumed domain is around 170 amino acids in length.  #=GF CC   It is found to the C-terminus of a DNA-binding helix-turn-helix domain. This domain may be involved in binding to an as yet unknown ligand that allows a transcriptional regulation response to that molecule. There are a number of proteins that contain two tandem copies of this domain. This suggests that this domain may form a dimeric arrangement.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2378">COG2378</a></td><td>311</td><td>269</td><td>97.9</td><td>7.6E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------]</td><td>YafY</td><td>Predicted DNA-binding transcriptional regulator YafY, contains an HTH and WYL domains  </td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08279">pfam08279</a></td><td>52</td><td>41</td><td>73.3</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_11</td><td>HTH domain. This family includes helix-turn-helix domains in a wide variety of proteins.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13995">PRK13995</a></td><td>203</td><td>59</td><td>68.9</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13995</td><td>potassium-transporting ATPase subunit C; Provisional</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1349">COG1349</a></td><td>253</td><td>39</td><td>49.0</td><td>7.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GlpR</td><td>DNA-binding transcriptional regulator of sugar metabolism, DeoR/GlpR family </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13556">pfam13556</a></td><td>56</td><td>20</td><td>45.6</td><td>5.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_30</td><td>PucR C-terminal helix-turn-helix domain. This helix-turn-helix domain is often found at the C-terminus of PucR-like transcriptional regulators and is likely to be DNA-binding.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5088">COG5088</a></td><td>114</td><td>21</td><td>39.4</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SOH1</td><td>Rad5p-binding protein </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05669">pfam05669</a></td><td>103</td><td>13</td><td>39.2</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Med31</td><td>SOH1. The family consists of Saccharomyces cerevisiae SOH1 homologues. SOH1 is responsible for the repression of temperature sensitive growth of the HPR1 mutant and has been found to be a component of the RNA polymerase II transcription complex. SOH1 not only interacts with factors involved in DNA repair, but transcription as well. Thus, the SOH1 protein may serve to couple these two processes.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08220">pfam08220</a></td><td>57</td><td>40</td><td>37.5</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_DeoR</td><td>DeoR-like helix-turn-helix domain. </td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01366">cd01366</a></td><td>329</td><td>86</td><td>32.8</td><td>9.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KISc_C_terminal</td><td>Kinesin motor domain, KIFC2/KIFC3/ncd-like carboxy-terminal kinesins. Kinesin motor domain, KIFC2/KIFC3/ncd-like carboxy-terminal kinesins. Ncd is a spindle motor protein necessary for chromosome segregation in meiosis. KIFC2/KIFC3-like kinesins have been implicated in motility of the Golgi apparatus as well as dentritic and axonal transport in neurons. This catalytic (head) domain has ATPase activity and belongs to the larger group of P-loop NTPases. Kinesins are microtubule-dependent molecular motors that play important roles in intracellular transport and in cell division. In this subgroup the motor domain is found at the C-terminus (C-type). C-type kinesins are (-) end-directed motors, i.e. they transport cargo towards the (-) end of the microtubule. Kinesin motor domains hydrolyze ATP at a rate of about 80 per second, and move along the microtubule at a speed of about 6400 Angstroms per second. To achieve that, kinesin head groups work in pairs. Upon replacing ADP with ATP, a kinesin motor domain increases its affinity for microtubule binding and locks in place. Also, the neck linker binds to the motor domain, which repositions the other head domain through the coiled-coil domain close to a second tubulin dimer, about 80 Angstroms along the microtubule. Meanwhile, ATP hydrolysis takes place, and when the second head domain binds to the microtubule, the first domain again replaces ADP with ATP, triggering a conformational change that pulls the first domain forward.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02677">TIGR02677</a></td><td>494</td><td>97</td><td>32.7</td><td>60</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02677</td><td>TIGR02677 family protein. Members of this protein belong to a conserved gene four-gene neighborhood found sporadically in a phylogenetically broad range of bacteria: Nocardia farcinica, Symbiobacterium thermophilum, and Streptomyces avermitilis (Actinobacteria), Geobacillus kaustophilus (Firmicutes), Azoarcus sp. EbN1 and Ralstonia solanacearum (Betaproteobacteria). </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4458">COG4458</a></td><td>821</td><td>45</td><td>31.8</td><td>1.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SrfC</td><td>Uncharacterized protein </td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10079">PRK10079</a></td><td>241</td><td>24</td><td>31.1</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10079</td><td>phosphonate metabolism transcriptional regulator PhnF; Provisional</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02234">pfam02234</a></td><td>51</td><td>18</td><td>31.0</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CDI</td><td>Cyclin-dependent kinase inhibitor. Cell cycle progression is negatively controlled by cyclin-dependent kinases inhibitors (CDIs). CDIs are involved in cell cycle arrest at the G1 phase.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05915">cd05915</a></td><td>509</td><td>100</td><td>29.2</td><td>73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ttLC_FACS_like</td><td>Fatty acyl-CoA synthetases similar to LC-FACS from Thermus thermophiles. This family includes fatty acyl-CoA synthetases that can activate medium-chain to long-chain fatty acids. They catalyze the ATP-dependent acylation of fatty acids in a two-step reaction. The carboxylate substrate first reacts with ATP to form an acyl-adenylate intermediate, which then reacts with CoA to produce an acyl-CoA ester. Fatty acyl-CoA synthetases are responsible for fatty acid degradation as well as physiological regulation of cellular functions via the production of fatty acyl-CoA esters. The fatty acyl-CoA synthetase from Thermus thermophiles in this family has been shown to catalyze the long-chain fatty acid, myristoyl acid, while another member in this family, the AlkK protein identified in Pseudomonas oleovorans, targets medium chain fatty acids. This family also includes an uncharacterized subgroup of FACS.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02812">TIGR02812</a></td><td>235</td><td>24</td><td>24.3</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fadR_gamma</td><td>fatty acid metabolism transcriptional regulator FadR. Members of this family are FadR, a transcriptional regulator of fatty acid metabolism, including both biosynthesis and beta-oxidation. It is found exclusively in a subset of Gammaproteobacteria, with strictly one copy per genome. It has an N-terminal DNA-binding domain and a less well conserved C-terminal long chain acyl-CoA-binding domain. FadR from this family heterologously expressed in Escherichia coli show differences in regulatory response and fatty acid binding profiles. The family is nevertheless designated equivalog, as all member proteins have at least nominally the same function. </td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06605">cd06605</a></td><td>265</td><td>47</td><td>23.2</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;]</td><td>PKc_MAPKK</td><td>Catalytic domain of the dual-specificity Protein Kinase, Mitogen-Activated Protein Kinase Kinase. PKs catalyze the transfer of the gamma-phosphoryl group from ATP to serine/threonine (ST) or tyrosine residues on protein substrates. MAPKKs are dual-specificity PKs that phosphorylate their downstream targets, MAPKs, at specific threonine and tyrosine residues. The MAPK signaling pathways are important mediators of cellular responses to extracellular signals. The pathways involve a triple kinase core cascade comprising the MAPK, which is phosphorylated and activated by a MAPK kinase (MAPKK or MKK or MAP2K), which itself is phosphorylated and activated by a MAPKK kinase (MAPKKK or MKKK or MAP3K). There are three MAPK subfamilies: extracellular signal-regulated kinase (ERK), c-Jun N-terminal kinase (JNK), and p38. In mammalian cells, there are seven MAPKKs (named MKK1-7) and 20 MAPKKKs. Each MAPK subfamily can be activated by at least two cognate MAPKKs and by multiple MAPKKKs. The MAPKK subfamily is part of a larger superfamily that includes the catalytic domains of other STKs, protein tyrosine kinases, RIO kinases, aminoglycoside phosphotransferase, choline kinase, and phosphoinositide 3-kinase.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03194">cd03194</a></td><td>115</td><td>37</td><td>23.1</td><td>87</td><td>[&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GST_C_3</td><td>C-terminal, alpha helical domain of an unknown subfamily 3 of Glutathione S-transferases. Glutathione S-transferase (GST) C-terminal domain family, unknown subfamily 3; composed of uncharacterized proteins with similarity to GSTs. GSTs are cytosolic dimeric proteins involved in cellular detoxification by catalyzing the conjugation of glutathione (GSH) with a wide range of endogenous and xenobiotic alkylating agents, including carcinogens, therapeutic drugs, environmental toxins, and products of oxidative stress. GSTs also show GSH peroxidase activity and are involved in the synthesis of prostaglandins and leukotrienes. The GST fold contains an N-terminal thioredoxin-fold domain and a C-terminal alpha helical domain, with an active site located in a cleft between the two domains. GSH binds to the N-terminal domain while the hydrophobic substrate occupies a pocket in the C-terminal domain.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK04984">PRK04984</a></td><td>239</td><td>24</td><td>22.6</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK04984</td><td>fatty acid metabolism regulator; Provisional</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07848">pfam07848</a></td><td>70</td><td>39</td><td>22.2</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PaaX</td><td>PaaX-like protein. This family contains proteins that are similar to the product of the paaX gene of Escherichia coli. This protein is involved in the regulation of expression of a group of proteins known to participate in the metabolizm of phenylacetic acid. In fact, some members of this family are annotated by InterPro as containing a winged helix DNA-binding domain (Interpro:IPR009058).</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09918">cd09918</a></td><td>85</td><td>43</td><td>22.1</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH2_Nterm_SPT6_like</td><td>N-terminal Src homology 2 (SH2) domain found in Spt6. N-terminal SH2 domain in Spt6. Spt6 is an essential transcription elongation factor and histone chaperone that binds the C-terminal repeat domain (CTD) of RNA polymerase II. Spt6 contains a tandem SH2 domain with a novel structure and CTD-binding mode. The tandem SH2 domain binds to a serine 2-phosphorylated CTD peptide in vitro, whereas its N-terminal SH2 subdomain does not. CTD binding requires a positively charged crevice in the C-terminal SH2 subdomain, which lacks the canonical phospho-binding pocket of SH2 domains. The tandem SH2 domain is apparently required for transcription elongation in vivo as its deletion in cells is lethal in the presence of 6-azauracil.  In general SH2 domains are involved in signal transduction.  They typically bind pTyr-containing ligands via two surface pockets, a pTyr and hydrophobic binding pocket, allowing proteins with SH2 domains to localize to tyrosine phosphorylated sites.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04181">TIGR04181</a></td><td>359</td><td>29</td><td>20.7</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NHT_00031</td><td>aminotransferase, LLPSF_NHT_00031 family. This clade of aminotransferases is a member of the pfam01041 (DegT/DnrJ/EryC1/StrS) superfamily. The family is named after the instance in Leptospira interrogans serovar Lai, str. 56601, where it is the 31st gene in the 91-gene lipopolysaccharide biosynthesis locus. Members of this family are generally found within a subcluster of seven or more genes including an epimerase/dehydratase, four genes homologous to the elements of the neuraminic (sialic) acid biosynthesis cluster (NeuABCD) and a nucleotidyl transferase. Together it is very likely that these enzymes direct the biosynthesis of a nine-carbon sugar analogous to CMP-neuraminic acid. These seven genes form the core of the cassette, although they are often accompanied by additional genes that may further modify the product sugar.</td></tr>
</table>
</body>
