<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4951">COG4951</a></td><td>361</td><td>198</td><td>99.8</td><td>3.4E-19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4951</td><td>Uncharacterized protein </td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01896">pfam01896</a></td><td>131</td><td>101</td><td>98.2</td><td>2.6E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DNA_primase_S</td><td>Eukaryotic and archaeal DNA primase small subunit. DNA primase synthesizes the RNA primers for the Okazaki fragments in lagging strand DNA synthesis. DNA primase is a heterodimer of large and small subunits. This family also includes baculovirus late expression factor 1 or LEF-1 proteins. Baculovirus LEF-1 is a DNA primase enzyme. Bacterial DNA primase adopts a different fold to archaeal and eukaryotic primases.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13414">pfam13414</a></td><td>69</td><td>64</td><td>97.9</td><td>8.6E-05</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_11</td><td>TPR repeat. </td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14559">pfam14559</a></td><td>68</td><td>60</td><td>97.6</td><td>9.7E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_19</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13432">pfam13432</a></td><td>65</td><td>56</td><td>97.5</td><td>0.00056</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_16</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13424">pfam13424</a></td><td>78</td><td>63</td><td>96.9</td><td>0.0041</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_12</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00525">cd00525</a></td><td>136</td><td>109</td><td>96.3</td><td>0.039</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AE_Prim_S_like</td><td>AE_Prim_S_like: primase domain similar to that found in the small subunit of archaeal and eukaryotic (A/E) DNA primases. The replication machineries of A/Es are distinct from that of bacteria. Primases are DNA-dependent RNA polymerases which synthesis the short RNA primers required for DNA replication. In eukaryotes, this small catalytically active primase subunit (p50) and a larger primase subunit (p60), referred to jointly as the core primase, associate with the B subunit and the DNA polymerase alpha subunit in a complex, called Pol alpha-pri. In addition to its catalytic role in replication, eukaryotic DNA primase may play a role in coupling replication to DNA damage repair and in checkpoint control during S phase. Pfu41 and Pfu46 comprise the primase complex of the archaea Pyrococcus furiosus; these proteins have sequence identity to the eukaryotic p50 and p60 primase proteins respectively. Pfu41 preferentially uses dNTPs as substrate. Pfu46 regulates the primase activity of Pfu41. Also found in this group is the primase-polymerase (primpol) domain of replicases from archaeal plasmids including the ORF904 protein of pRN1 from Sulfolobus islandicus (pRN1 primpol). The pRN1 primpol domain exhibits DNA polymerase and primase activities; a cluster of active site residues (three acidic residues, and a histidine) is required for both these activities. The pRN1 primpol primase activity prefers dNTPs to rNTPs; however incorporation of dNTPs requires rNTP as cofactor. This group also includes the Pol domain of bacterial LigD proteins such Mycobacterium tuberculosis (Mt)LigD. MtLigD contains an N-terminal Pol domain, a central phosphoesterase module, and a C-terminal ligase domain. LigD Pol plays a role in non-homologous end joining (NHEJ)-mediated repair of DNA double-strand breaks (DSB) in vivo, perhaps by filling in short 5'-overhangs with ribonucleotides; the filled in termini would be sealed by the associated LigD ligase domain. The MtLigD Pol domain is stimulated by manganese, is error-prone, and prefers adding rNTPs to dNTPs in vitro.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13431">pfam13431</a></td><td>34</td><td>33</td><td>96.2</td><td>0.0044</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_17</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13428">pfam13428</a></td><td>44</td><td>41</td><td>95.8</td><td>0.023</td><td>[&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_14</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07719">pfam07719</a></td><td>34</td><td>34</td><td>94.8</td><td>0.042</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_2</td><td>Tetratricopeptide repeat. This Pfam entry includes outlying Tetratricopeptide-like repeats (TPR) that are not matched by pfam00515.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02521">TIGR02521</a></td><td>234</td><td>91</td><td>94.8</td><td>0.21</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>type_IV_pilW</td><td>type IV pilus biogenesis/stability protein PilW. Members of this family are designated PilF and PilW. This outer membrane protein is required both for pilus stability and for pilus function such as adherence to human cells. Members of this family contain copies of the TPR (tetratricopeptide repeat) domain.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04860">cd04860</a></td><td>232</td><td>150</td><td>94.2</td><td>0.087</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AE_Prim_S</td><td>AE_Prim_S: primase domain similar to that found in the small subunit of archaeal and eukaryotic (A/E) DNA primases. Primases are DNA-dependent RNA polymerases which synthesis the short RNA primers required for DNA replication. In addition to its catalytic role in replication, DNA primase may play a role in coupling replication to DNA damage repair and in checkpoint control during S phase. In eukaryotes, this small catalytically active primase subunit (p50) and a larger primase subunit (p60), referred to jointly as the core primase, associate with the B subunit and the DNA polymerase alpha subunit in a complex, called Pol alpha-pri. The function of the larger primase subunit is unclear. Included in this group are Pfu41 and Pfu46, these two proteins comprise the primase complex of the archaea Pyrococcus furiosus; Pfu41 and Pfu46 have sequence identity to the eukaryotic p50 and p60 primase proteins respectively. Pfu41 preferentially uses dNTPs as substrate. Pfu46 regulates the primase activity of Pfu41.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02521">TIGR02521</a></td><td>234</td><td>65</td><td>93.8</td><td>0.13</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>type_IV_pilW</td><td>type IV pilus biogenesis/stability protein PilW. Members of this family are designated PilF and PilW. This outer membrane protein is required both for pilus stability and for pilus function such as adherence to human cells. Members of this family contain copies of the TPR (tetratricopeptide repeat) domain.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13181">pfam13181</a></td><td>34</td><td>32</td><td>93.7</td><td>0.099</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_8</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3063">COG3063</a></td><td>250</td><td>87</td><td>93.5</td><td>0.29</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PilF</td><td>Tfp pilus assembly protein PilF  </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00515">pfam00515</a></td><td>34</td><td>33</td><td>93.2</td><td>0.13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_1</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02917">TIGR02917</a></td><td>899</td><td>109</td><td>93.1</td><td>0.82</td><td>[---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PEP_TPR_lipo</td><td>putative PEP-CTERM system TPR-repeat lipoprotein. This protein family occurs in strictly within a subset of Gram-negative bacterial species with the proposed PEP-CTERM/exosortase system, analogous to the LPXTG/sortase system common in Gram-positive bacteria. This protein occurs in a species if and only if a transmembrane histidine kinase (TIGR02916) and a DNA-binding response regulator (TIGR02915) also occur. The present of tetratricopeptide repeats (TPR) suggests protein-protein interaction, possibly for the regulation of PEP-CTERM protein expression, since many PEP-CTERM proteins in these genomes are preceded by a proposed DNA binding site for the response regulator.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13371">pfam13371</a></td><td>73</td><td>58</td><td>92.7</td><td>0.34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_9</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13429">pfam13429</a></td><td>279</td><td>84</td><td>92.0</td><td>1</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_15</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02778">TIGR02778</a></td><td>245</td><td>105</td><td>91.9</td><td>0.46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ligD_pol</td><td>DNA ligase D, polymerase domain. DNA repair of double-stranded breaks by non-homologous end joining (NHEJ) is accomplished by a two-protein system that is present in a minority of prokaryotes. One component is the Ku protein (see TIGR02772), which binds DNA ends. The other is a DNA ligase, a protein that is a multidomain polypeptide in most of those bacteria that have NHEJ, a permuted polypeptide in Mycobacterium tuberculosis and a few other species, and the product of tandem genes in some other bacteria. This model represents the polymerase domain.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11788">PRK11788</a></td><td>389</td><td>67</td><td>91.8</td><td>0.26</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11788</td><td>tetratricopeptide repeat protein; Provisional</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13428">pfam13428</a></td><td>44</td><td>37</td><td>90.9</td><td>0.38</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_14</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3063">COG3063</a></td><td>250</td><td>64</td><td>89.9</td><td>0.66</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PilF</td><td>Tfp pilus assembly protein PilF  </td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04861">cd04861</a></td><td>227</td><td>110</td><td>89.7</td><td>0.75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LigD_Pol_like</td><td>LigD_Pol_like: Polymerase (Pol) domain of bacterial LigD proteins similar to Pseudomonas aeruginosa (Pae) LigD. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required for non-homologous end joining (NHEJ)-mediated repair of DNA double-strand breaks (DSB). NHEJ-mediated DNA DSB repair is error-prone. PaeLigD is monomeric, containing an N-terminal phosphoesterase module, a central polymerase (Pol) domain, and a C-terminal ATP-dependent ligase domain. Mycobacterium tuberculosis (Mt)LigD, also found in this group, is monomeric and contains the same modules but these are arranged differently: an N-terminal Pol domain, a central phosphoesterase module, and a C-terminal ligase domain. It has been suggested that LigD Pol contributes to NHEJ-mediated DNA DSB repair in vivo, by filling in short 5'-overhangs with ribonucleotides; the filled in termini would then be sealed by the associated LigD ligase domain, resulting in short stretches of RNA incorporated into the genomic DNA. The PaeLigD Pol domain in vitro, in a manganese-dependent fashion, catalyzes templated extensions of 5'-overhang duplex DNA, and nontemplated single-nucleotide additions to blunt-end duplex DNA; it preferentially adds single ribonucleotides at blunt DNA ends. PaeLigD Pol adds a correctly paired rNTP to the DNA primer termini more rapidly than it does a correctly paired dNTP; it has higher infidelity as an RNA polymerase than it does as a DNA polymerase, which is in keeping with the mutagenic property of NHEJ-mediated DNA DSB repair. The MtLigD Pol domain similarly is stimulated by manganese, is error-prone, and prefers adding rNTPs to dNTPs in vitro. The MtLigD Pol domain has been shown to prefer DNA gapped substrates containing a 5'-phosphate group at the gap.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2956">COG2956</a></td><td>389</td><td>51</td><td>88.7</td><td>1.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YciM</td><td>Lipopolysaccharide biosynthesis regulator YciM, contains six TPR domains and a predicted metal-binding C-terminal domain </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4235">COG4235</a></td><td>287</td><td>77</td><td>87.5</td><td>1.1</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NrfG</td><td>Cytochrome c-type biogenesis protein CcmH/NrfG </td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03939">TIGR03939</a></td><td>800</td><td>104</td><td>86.3</td><td>2</td><td>[&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PGA_TPR_OMP</td><td>poly-beta-1,6 N-acetyl-D-glucosamine export porin PgaA. Members of this protein family are the poly-beta-1,6 N-acetyl-D-glucosamine (PGA) export porin PgaA of Gram-negative bacteria. There is no counterpart in the poly-beta-1,6 N-acetyl-D-glucosamine biosynthesis systems of Gram-positive bacteria such as Staphylococcus epidermidis. The PGA polysaccharide adhesin is a critical determinant of biofilm formation. The conserved C-terminal domain of this outer membrane protein is preceded by a variable number of TPR repeats.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00515">pfam00515</a></td><td>34</td><td>31</td><td>86.3</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_1</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00540">TIGR00540</a></td><td>367</td><td>59</td><td>85.9</td><td>0.96</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_hemY_coli</td><td>heme biosynthesis-associated TPR protein. Members of this protein family are uncharacterized tetratricopeptide repeat (TPR) proteins invariably found in heme biosynthesis gene clusters. The absence of any invariant residues other than Ala argues against this protein serving as an enzyme per se. The gene symbol hemY assigned in E. coli is unfortunate in that an unrelated protein, protoporphyrinogen oxidase (HemG in E. coli) is designated HemY in Bacillus subtilis. </td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11788">PRK11788</a></td><td>389</td><td>47</td><td>85.5</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11788</td><td>tetratricopeptide repeat protein; Provisional</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13414">pfam13414</a></td><td>69</td><td>62</td><td>85.2</td><td>5.4</td><td>[-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_11</td><td>TPR repeat. </td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3285">COG3285</a></td><td>299</td><td>70</td><td>85.0</td><td>1.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LigD</td><td>Eukaryotic-type DNA primase  </td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02917">TIGR02917</a></td><td>899</td><td>69</td><td>84.8</td><td>4.5</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PEP_TPR_lipo</td><td>putative PEP-CTERM system TPR-repeat lipoprotein. This protein family occurs in strictly within a subset of Gram-negative bacterial species with the proposed PEP-CTERM/exosortase system, analogous to the LPXTG/sortase system common in Gram-positive bacteria. This protein occurs in a species if and only if a transmembrane histidine kinase (TIGR02916) and a DNA-binding response regulator (TIGR02915) also occur. The present of tetratricopeptide repeats (TPR) suggests protein-protein interaction, possibly for the regulation of PEP-CTERM protein expression, since many PEP-CTERM proteins in these genomes are preceded by a proposed DNA binding site for the response regulator.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13174">pfam13174</a></td><td>33</td><td>31</td><td>83.9</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_6</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04864">cd04864</a></td><td>228</td><td>110</td><td>83.8</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LigD_Pol_like_1</td><td>LigD_Pol_like_1: Polymerase (Pol) domain of mostly bacterial LigD proteins similar to Pseudomonas aeruginosa (Pae) LigD, subgroup 1. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required for non-homologous end joining (NHEJ)-mediated repair of DNA double-strand breaks (DSB). NHEJ-mediated DNA DSB repair is error-prone. It has been suggested that LigD Pol contributes to NHEJ-mediated DNA DSB repair in vivo, by filling in short 5'-overhangs with ribonucleotides; the filled in termini would then be sealed by the associated LigD ligase domain, resulting in short stretches of RNA incorporated into the genomic DNA. The Pol domains of PaeLigD and Mycobacterium tuberculosis (Mt)LigD are stimulated by manganese, are error-prone, and prefer adding rNTPs to dNTPs in vitro; however PaeLigD and MtLigD belong to other subgroups, proteins in this subgroup await functional characterization.</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11189">PRK11189</a></td><td>296</td><td>90</td><td>83.3</td><td>2.9</td><td>[&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11189</td><td>lipoprotein NlpI; Provisional</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13176">pfam13176</a></td><td>36</td><td>26</td><td>83.3</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_7</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02776">TIGR02776</a></td><td>552</td><td>74</td><td>83.1</td><td>2.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NHEJ_ligase_prk</td><td>DNA ligase D. Members of this protein family are DNA ligases involved in the repair of DNA double-stranded breaks by non-homologous end joining (NHEJ). The system of the bacterial Ku protein (TIGR02772) plus this DNA ligase is seen in about 20 % of bacterial genomes to date and at least one archaeon (Archeoglobus fulgidus). This model describes a central and a C-terminal domain. These two domains may be permuted, as in genus Mycobacterium, or divided into tandem ORFs, and therefore not be identified by this model. An additional N-terminal 3'-phosphoesterase (PE) domain present in some but not all examples of this ligase is not included in the seed alignment for this model; it only represents the central ATP-dependent ligase domain and the C-terminal polymerase domain. Most examples of genes for this ligase are adjacent to the gene for Ku. </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07719">pfam07719</a></td><td>34</td><td>32</td><td>82.9</td><td>2.3</td><td>[&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_2</td><td>Tetratricopeptide repeat. This Pfam entry includes outlying Tetratricopeptide-like repeats (TPR) that are not matched by pfam00515.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02552">TIGR02552</a></td><td>135</td><td>95</td><td>82.9</td><td>8.6</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LcrH_SycD</td><td>type III secretion low calcium response chaperone LcrH/SycD. Genes in this family are found in type III secretion operons. LcrH, from Yersinia is believed to have a regulatory function in the low-calcium response of the secretion system. The same protein is also known as SycD (SYC = Specific Yop Chaperone) for its chaperone role. In Pseudomonas, where the homolog is known as PcrH, the chaperone role has been demonstrated and the regulatory role appears to be absent. ScyD/LcrH contains three central tetratricopeptide-like repeats that are predicted to fold into an all-alpha-helical array.</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5010">COG5010</a></td><td>257</td><td>57</td><td>82.5</td><td>2.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TadD</td><td>Flp pilus assembly protein TadD, contains TPR repeats  </td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4235">COG4235</a></td><td>287</td><td>80</td><td>81.2</td><td>3.2</td><td>[&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NrfG</td><td>Cytochrome c-type biogenesis protein CcmH/NrfG </td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4783">COG4783</a></td><td>484</td><td>100</td><td>81.1</td><td>3</td><td>[&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YfgC</td><td>Putative Zn-dependent protease, contains TPR repeats </td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04862">cd04862</a></td><td>227</td><td>107</td><td>80.3</td><td>3.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PaeLigD_Pol_like</td><td>PaeLigD_Pol_like: Polymerase (Pol) domain of bacterial LigD proteins similar to Pseudomonas aeruginosa (Pae) LigD. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required for non-homologous end joining (NHEJ)-mediated repair of DNA double-strand breaks (DSB). NHEJ-mediated DNA DSB repair is error-prone. PaeLigD is monomeric, containing an N-terminal phosphoesterase module, a central polymerase (Pol) domain, and a C-terminal ATP-dependent ligase domain. It has been suggested that LigD Pol contributes to NHEJ-mediated DNA DSB repair in vivo, by filling in short 5'-overhangs with ribonucleotides; the filled in termini would then be sealed by the associated LigD ligase domain, resulting in short stretches of RNA incorporated into the genomic DNA. The PaeLigD Pol domain in vitro, in a manganese-dependent fashion, catalyzes templated extensions of 5'-overhang duplex DNA, and nontemplated single-nucleotide additions to blunt-end duplex DNA; it preferentially adds single ribonucleotides at blunt DNA ends. PaeLigD Pol adds a correctly paired rNTP to the DNA primer termini more rapidly than it does a correctly paired dNTP; it has higher infidelity as an RNA polymerase than it does as a DNA polymerase, which is in keeping with the mutagenic property of NHEJ-mediated DNA DSB repair.</td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2219">COG2219</a></td><td>363</td><td>71</td><td>77.0</td><td>3.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRI2</td><td>Eukaryotic-type DNA primase, large subunit  </td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13181">pfam13181</a></td><td>34</td><td>31</td><td>75.7</td><td>5.2</td><td>[&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_8</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12895">pfam12895</a></td><td>81</td><td>43</td><td>75.3</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Apc3</td><td>Anaphase-promoting complex, cyclosome, subunit 3. Apc3, otherwise known as Cdc27, is one of the subunits of the anaphase-promoting complex or cyclosome. The anaphase-promoting complex is a multiprotein subunit E3 ubiquitin ligase complex that controls segregation of chromosomes and exit from mitosis in eukaryotes. The protein members of this family contain TPR repeats just as those of Apc7 do, and it appears that these TPR units bind the C-termini of the APC co-activators CDH1 and CDC20.</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13429">pfam13429</a></td><td>279</td><td>89</td><td>74.6</td><td>6.5</td><td>[&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_15</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13432">pfam13432</a></td><td>65</td><td>63</td><td>73.7</td><td>21</td><td>[-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_16</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0457">COG0457</a></td><td>291</td><td>86</td><td>73.5</td><td>35</td><td>[&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR</td><td>Tetratricopeptide (TPR) repeat </td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2976">COG2976</a></td><td>207</td><td>70</td><td>72.1</td><td>6</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YfgM</td><td>Putative negative regulator of RcsB-dependent stress response </td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0457">COG0457</a></td><td>291</td><td>57</td><td>71.4</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR</td><td>Tetratricopeptide (TPR) repeat </td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5010">COG5010</a></td><td>257</td><td>100</td><td>71.1</td><td>36</td><td>[---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TadD</td><td>Flp pilus assembly protein TadD, contains TPR repeats  </td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11447">PRK11447</a></td><td>1157</td><td>68</td><td>70.9</td><td>4.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11447</td><td>cellulose synthase subunit BcsC; Provisional</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4785">COG4785</a></td><td>297</td><td>90</td><td>69.4</td><td>9.8</td><td>[&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NlpI</td><td>Lipoprotein NlpI, contains TPR repeats </td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4976">COG4976</a></td><td>287</td><td>57</td><td>68.5</td><td>7.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4976</td><td>Predicted methyltransferase, contains TPR repeat </td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14559">pfam14559</a></td><td>68</td><td>61</td><td>68.3</td><td>18</td><td>[------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_19</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04866">cd04866</a></td><td>223</td><td>83</td><td>67.2</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LigD_Pol_like_3</td><td>LigD_Pol_like_3: Polymerase (Pol) domain of bacterial LigD proteins similar to Pseudomonas aeruginosa (Pae) LigD, subgroup 3. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required for non-homologous end joining (NHEJ)-mediated repair of DNA double-strand breaks (DSB). NHEJ-mediated DNA DSB repair is error-prone. It has been suggested that LigD Pol contributes to NHEJ-mediated repair DSB repair in vivo, by filling in short 5'-overhangs with ribonucleotides; the filled in termini would then be sealed by the associated LigD ligase domain, resulting in short stretches of RNA incorporated into the genomic DNA. The Pol domains of PaeLigD and Mycobacterium tuberculosis (Mt)LigD are stimulated by manganese, are error-prone, and prefer adding rNTPs to dNTPs in vitro; however PaeLigD and MtLigD belong to other subgroups, proteins in this subgroup await functional characterization.</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02795">TIGR02795</a></td><td>117</td><td>56</td><td>66.3</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tol_pal_ybgF</td><td>tol-pal system protein YbgF. Members of this protein family are the product of one of seven genes regularly clustered in operons to encode the proteins of the tol-pal system, which is critical for maintaining the integrity of the bacterial outer membrane. The gene for this periplasmic protein has been designated orf2 and ybgF. All members of the seed alignment were from unique tol-pal gene regions from completed bacterial genomes. The architecture of this protein is a signal sequence, a low-complexity region usually rich in Asn and Gln, a well-conserved region with tandem repeats that resemble the tetratricopeptide (TPR) repeat, involved in protein-protein interaction.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04865">cd04865</a></td><td>228</td><td>89</td><td>65.1</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LigD_Pol_like_2</td><td>LigD_Pol_like_2: Polymerase (Pol) domain of bacterial LigD proteins similar to Pseudomonas aeruginosa (Pae) LigD, subgroup 2. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required for non-homologous end joining (NHEJ)-mediated repair of DNA double-strand breaks (DSB). NHEJ-mediated DNA DSB repair is error-prone. It has been suggested that LigD Pol contributes to NHEJ-mediated DNA DSB repair in vivo, by filling in short 5'-overhangs with ribonucleotides; the filled in termini would then be sealed by the associated LigD ligase domain, resulting in short stretches of RNA incorporated into the genomic DNA. The Pol domains of PaeLigD and Mycobacterium tuberculosis (Mt)LigD are stimulated by manganese, are error-prone, and prefer adding rNTPs to dNTPs in vitro; however PaeLigD and MtLigD belong to other subgroups, proteins in this subgroup await functional characterization.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00990">TIGR00990</a></td><td>615</td><td>65</td><td>65.0</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3a0801s09</td><td>mitochondrial precursor proteins import receptor (72 kDa mitochondrial outermembrane protein) (mitochondrial import receptor for the ADP/ATP carrier) (translocase of outermembrane tom70). </td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03939">TIGR03939</a></td><td>800</td><td>87</td><td>63.5</td><td>40</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PGA_TPR_OMP</td><td>poly-beta-1,6 N-acetyl-D-glucosamine export porin PgaA. Members of this protein family are the poly-beta-1,6 N-acetyl-D-glucosamine (PGA) export porin PgaA of Gram-negative bacteria. There is no counterpart in the poly-beta-1,6 N-acetyl-D-glucosamine biosynthesis systems of Gram-positive bacteria such as Staphylococcus epidermidis. The PGA polysaccharide adhesin is a critical determinant of biofilm formation. The conserved C-terminal domain of this outer membrane protein is preceded by a variable number of TPR repeats.</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14561">pfam14561</a></td><td>90</td><td>64</td><td>62.0</td><td>24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_20</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05804">cd05804</a></td><td>355</td><td>48</td><td>61.3</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>StaR_like</td><td>StaR_like; a well-conserved protein found in bacteria, plants, and animals. A family member from Streptomyces toyocaensis, StaR is part of a gene cluster involved in the biosynthesis of glycopeptide antibiotics (GPAs), specifically A47934. It has been speculated that StaR could be a flavoprotein hydroxylating a tyrosine sidechain. Some family members have been annotated as proteins containing tetratricopeptide (TPR) repeats, which may at least indicate mostly alpha-helical secondary structure.</td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3629">COG3629</a></td><td>280</td><td>70</td><td>61.0</td><td>22</td><td>[&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DnrI</td><td>DNA-binding transcriptional activator of the SARP family  </td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05972">PRK05972</a></td><td>860</td><td>59</td><td>59.8</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ligD</td><td>ATP-dependent DNA ligase; Reviewed</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02552">TIGR02552</a></td><td>135</td><td>52</td><td>59.4</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LcrH_SycD</td><td>type III secretion low calcium response chaperone LcrH/SycD. Genes in this family are found in type III secretion operons. LcrH, from Yersinia is believed to have a regulatory function in the low-calcium response of the secretion system. The same protein is also known as SycD (SYC = Specific Yop Chaperone) for its chaperone role. In Pseudomonas, where the homolog is known as PcrH, the chaperone role has been demonstrated and the regulatory role appears to be absent. ScyD/LcrH contains three central tetratricopeptide-like repeats that are predicted to fold into an all-alpha-helical array.</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12569">pfam12569</a></td><td>516</td><td>69</td><td>59.2</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NARP1</td><td>NMDA receptor-regulated protein 1. This domain family is found in eukaryotes, and is approximately 40 amino acids in length. The family is found in association with pfam07719, pfam00515. There is a single completely conserved residue L that may be functionally important. NARP1 is the mammalian homologue of a yeast N-terminal acetyltransferase that regulates entry into the G(0) phase of the cell cycle.</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13424">pfam13424</a></td><td>78</td><td>32</td><td>56.5</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_12</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1467">COG1467</a></td><td>341</td><td>24</td><td>55.3</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRI1</td><td>Eukaryotic-type DNA primase, catalytic (small) subunit  </td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4700">COG4700</a></td><td>251</td><td>85</td><td>55.1</td><td>99</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4700</td><td>Uncharacterized protein </td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12688">pfam12688</a></td><td>119</td><td>55</td><td>54.5</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_5</td><td>Tetratrico peptide repeat. BH0479 of Bacillus halodurans is a hypothetical protein which contains a tetratrico peptide repeat (TPR) structural motif. The TPR motif is often involved in mediating protein-protein interactions. This protein is likely to function as a dimer. The first 48 amino acids are not present in the clone construct. This Pfam entry includes tetratricopeptide-like repeats not detected by the pfam00515, pfam07719, pfam07720 and pfam07221 models.</td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12212">cd12212</a></td><td>115</td><td>61</td><td>54.5</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fis1</td><td>Mitochondrial Fission Protein Fis1, cytosolic domain. Fis1, along with Dnm1 and Mdv1, is an essential protein in mediating mitochondrial fission. Dnm1 and Fis1 are highly conserved, with a common mechanism in disparate species. In mutants of these proteins, mitochondrial fission is impaired, resulting in networks of undivided mitochondria. The Fis1 N-terminus is cytosolic and tethered to the mitochondrial outer membrane via a C-terminal transmembrane domain. Fis1 appears to act via the recruitment of division complexes to the mitochondrial outer membrane, via interactions with Mdv1 or Caf4. Fis1 has tandem Tetratricopeptide repeat (TPR) motifs which are known to mediate protein-protein interactions.</td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07720">pfam07720</a></td><td>34</td><td>30</td><td>54.5</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_3</td><td>Tetratricopeptide repeat. This Pfam entry includes tetratricopeptide-like repeats found in the LcrH/SycD-like chaperones.</td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14853">pfam14853</a></td><td>53</td><td>38</td><td>52.7</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fis1_TPR_C</td><td>Fis1 C-terminal tetratricopeptide repeat. The mitochondrial fission protein Fis1 consists of two tetratricopeptide repeats. This domain is the C-terminal tetratricopeptide repeat</td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02249">PRK02249</a></td><td>343</td><td>83</td><td>52.6</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK02249</td><td>DNA primase large subunit; Validated</td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13374">pfam13374</a></td><td>42</td><td>28</td><td>52.4</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_10</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02259">pfam02259</a></td><td>350</td><td>68</td><td>50.1</td><td>45</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FAT</td><td>FAT domain. The FAT domain is named after FRAP, ATM and TRRAP.</td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05804">cd05804</a></td><td>355</td><td>70</td><td>49.4</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>StaR_like</td><td>StaR_like; a well-conserved protein found in bacteria, plants, and animals. A family member from Streptomyces toyocaensis, StaR is part of a gene cluster involved in the biosynthesis of glycopeptide antibiotics (GPAs), specifically A47934. It has been speculated that StaR could be a flavoprotein hydroxylating a tyrosine sidechain. Some family members have been annotated as proteins containing tetratricopeptide (TPR) repeats, which may at least indicate mostly alpha-helical secondary structure.</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14267">PRK14267</a></td><td>253</td><td>41</td><td>48.4</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14267</td><td>phosphate ABC transporter ATP-binding protein; Provisional</td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1729">COG1729</a></td><td>262</td><td>56</td><td>47.9</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YbgF</td><td>Periplasmic TolA-binding protein (function unknown) </td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2956">COG2956</a></td><td>389</td><td>47</td><td>47.5</td><td>24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YciM</td><td>Lipopolysaccharide biosynthesis regulator YciM, contains six TPR domains and a predicted metal-binding C-terminal domain </td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02679">cd02679</a></td><td>79</td><td>31</td><td>47.2</td><td>21</td><td>[&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MIT_spastin</td><td>MIT: domain contained within Microtubule Interacting and Trafficking molecules. This MIT domain sub-family is found in the AAA protein spastin, a probable ATPase involved in the assembly or function of nuclear protein complexes; spastins might also be involved in microtubule dynamics. The molecular function of the MIT domain is unclear.</td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00531">pfam00531</a></td><td>81</td><td>22</td><td>46.4</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Death</td><td>Death domain. </td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00637">pfam00637</a></td><td>138</td><td>47</td><td>46.3</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Clathrin</td><td>Region in Clathrin and VPS. Each region is about 140 amino acids long. The regions are composed of multiple alpha helical repeats. They occur in the arm region of the Clathrin heavy chain.</td></tr>
<tr><td>86</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12569">pfam12569</a></td><td>516</td><td>45</td><td>46.1</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NARP1</td><td>NMDA receptor-regulated protein 1. This domain family is found in eukaryotes, and is approximately 40 amino acids in length. The family is found in association with pfam07719, pfam00515. There is a single completely conserved residue L that may be functionally important. NARP1 is the mammalian homologue of a yeast N-terminal acetyltransferase that regulates entry into the G(0) phase of the cell cycle.</td></tr>
<tr><td>87</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02603">PRK02603</a></td><td>172</td><td>54</td><td>45.6</td><td>50</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK02603</td><td>photosystem I assembly protein Ycf3; Provisional</td></tr>
<tr><td>88</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd15832">cd15832</a></td><td>278</td><td>72</td><td>45.2</td><td>60</td><td>[&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SNAP</td><td>Soluble N-ethylmaleimide-sensitive factor (NSF) Attachment Protein family. Members of the soluble NSF attachment protein (SNAP) family are involved in intracellular membrane trafficking, including vesicular transport between the endoplasmic reticulum and Golgi apparatus. Higher eukaryotes contain three isoforms of SNAPs: alpha, beta, and gamma. Alpha-SNAP is universally present in eukaryotes and acts as an adaptor protein between SNARE (integral membrane SNAP receptor) and NSF for recruitment to the 20S complex. Beta-SNAP is brain-specific and shares high sequence identity (about 85%) with alpha-SNAP. Gamma-SNAP is weakly related (about 20-25% identity) to the two other isoforms, and is ubiquitous. It may help regulate the activity of the 20S complex. The X-ray structures of vertebrate gamma-SNAP and yeast Sec17, a SNAP family member, show similar all-helical structures consisting of an N-terminal extended twisted sheet of four Tetratricopeptide repeat (TPR)-like helical hairpins and a C-terminal helical bundle.</td></tr>
<tr><td>89</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1729">COG1729</a></td><td>262</td><td>56</td><td>43.3</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YbgF</td><td>Periplasmic TolA-binding protein (function unknown) </td></tr>
<tr><td>90</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07906">cd07906</a></td><td>190</td><td>65</td><td>43.1</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Adenylation_DNA_ligase_LigD_LigC</td><td>Adenylation domain of Mycobacterium tuberculosis LigD and LigC-like ATP-dependent DNA ligases. Bacterial DNA ligases are divided into two broad classes: NAD-dependent and ATP-dependent. All bacterial species have a NAD-dependent DNA ligase (LigA). Some bacterial genomes contain multiple genes for DNA ligases that are predicted to use ATP as their cofactor, including Mycobacterium tuberculosis LigB, LigC, and LigD. This group is composed of ATP-dependent DNA ligases similar to Mycobacterium tuberculosis LigC. ATP-dependent polynucleotide ligases catalyze phosphodiester bond formation using nicked nucleic acid substrates with the high energy nucleotide of ATP as a cofactor in a three step reaction mechanism. DNA ligases play a vital role in the diverse processes of DNA replication, recombination and repair. Members of this group contain adenylation and C-terminal oligonucleotide/oligosaccharide binding (OB)-fold domains, comprising a catalytic core unit that is common to all members of the ATP-dependent DNA ligase family. The adenylation domain binds ATP and contains many of the active-site residues. The common catalytic core unit comprises six conserved sequence motifs (I, III, IIIa, IV, V and VI) that define this family of related nucleotidyltransferases. LigD consists of a central ATP-dependent DNA ligase catalytic core unit fused to a C-terminal polymerase domain and an N-terminal 3'-phosphoesterase (PE) module. LigD catalyzes the end-healing and end-sealing steps during non-homologous end joining.</td></tr>
<tr><td>91</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00540">TIGR00540</a></td><td>367</td><td>37</td><td>41.0</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_hemY_coli</td><td>heme biosynthesis-associated TPR protein. Members of this protein family are uncharacterized tetratricopeptide repeat (TPR) proteins invariably found in heme biosynthesis gene clusters. The absence of any invariant residues other than Ala argues against this protein serving as an enzyme per se. The gene symbol hemY assigned in E. coli is unfortunate in that an unrelated protein, protoporphyrinogen oxidase (HemG in E. coli) is designated HemY in Bacillus subtilis. </td></tr>
<tr><td>92</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK15179">PRK15179</a></td><td>694</td><td>68</td><td>39.9</td><td>71</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK15179</td><td>Vi polysaccharide biosynthesis protein TviE; Provisional</td></tr>
<tr><td>93</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3118">COG3118</a></td><td>304</td><td>92</td><td>37.4</td><td>3.1E+02</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YbbN</td><td>Negative regulator of GroEL, contains thioredoxin-like and TPR-like domains  </td></tr>
<tr><td>94</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02795">TIGR02795</a></td><td>117</td><td>60</td><td>37.0</td><td>97</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tol_pal_ybgF</td><td>tol-pal system protein YbgF. Members of this protein family are the product of one of seven genes regularly clustered in operons to encode the proteins of the tol-pal system, which is critical for maintaining the integrity of the bacterial outer membrane. The gene for this periplasmic protein has been designated orf2 and ybgF. All members of the seed alignment were from unique tol-pal gene regions from completed bacterial genomes. The architecture of this protein is a signal sequence, a low-complexity region usually rich in Asn and Gln, a well-conserved region with tandem repeats that resemble the tetratricopeptide (TPR) repeat, involved in protein-protein interaction.</td></tr>
<tr><td>95</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09986">pfam09986</a></td><td>214</td><td>52</td><td>35.1</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2225</td><td>Uncharacterized protein conserved in bacteria (DUF2225). This domain, found in various hypothetical bacterial proteins, has no known function.</td></tr>
<tr><td>96</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03302">TIGR03302</a></td><td>235</td><td>76</td><td>34.2</td><td>2.3E+02</td><td>[&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>OM_YfiO</td><td>outer membrane assembly lipoprotein YfiO. Members of this protein family include YfiO, a near-essential protein of the outer membrane, part of a complex involved in protein insertion into the bacterial outer membrane. Many proteins in this family are annotated as ComL, based on the involvement of this protein in natural transformation with exogenous DNA in Neisseria gonorrhoeae. This protein family shows sequence similarity to, but is distinct from, the tol-pal system protein YbgF (TIGR02795). </td></tr>
<tr><td>97</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11189">PRK11189</a></td><td>296</td><td>52</td><td>33.8</td><td>76</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11189</td><td>lipoprotein NlpI; Provisional</td></tr>
<tr><td>98</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10049">PRK10049</a></td><td>765</td><td>88</td><td>32.0</td><td>1.2E+02</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pgaA</td><td>outer membrane protein PgaA; Provisional</td></tr>
<tr><td>99</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00990">TIGR00990</a></td><td>615</td><td>54</td><td>31.9</td><td>83</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3a0801s09</td><td>mitochondrial precursor proteins import receptor (72 kDa mitochondrial outermembrane protein) (mitochondrial import receptor for the ADP/ATP carrier) (translocase of outermembrane tom70). </td></tr>
<tr><td>100</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12895">pfam12895</a></td><td>81</td><td>43</td><td>29.9</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Apc3</td><td>Anaphase-promoting complex, cyclosome, subunit 3. Apc3, otherwise known as Cdc27, is one of the subunits of the anaphase-promoting complex or cyclosome. The anaphase-promoting complex is a multiprotein subunit E3 ubiquitin ligase complex that controls segregation of chromosomes and exit from mitosis in eukaryotes. The protein members of this family contain TPR repeats just as those of Apc7 do, and it appears that these TPR units bind the C-termini of the APC co-activators CDH1 and CDC20.</td></tr>
<tr><td>101</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04863">cd04863</a></td><td>231</td><td>76</td><td>29.7</td><td>92</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MtLigD_Pol_like</td><td>MtLigD_Pol_like: Polymerase (Pol) domain of bacterial LigD proteins similar to Mycobacterium tuberculosis (Mt)LigD. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required for non-homologous end joining (NHEJ)-mediated repair of DNA double-strand breaks (DSB). NHEJ-mediated DNA DSB repair is error-prone. MtLigD is monomeric and contains an N-terminal Pol domain, a central phosphoesterase module, and a C-terminal ligase domain. It has been suggested that LigD Pol contributes to NHEJ-mediated DNA DSB repair in vivo, by filling in short 5'-overhangs with ribonucleotides; the filled in termini would then be sealed by the associated LigD ligase domain, resulting in short stretches of RNA incorporated into the genomic DNA. The MtLigD Pol domain is stimulated by manganese, is error-prone, and prefers adding rNTPs to dNTPs in vitro. The MtLigD Pol domain has been shown to prefer DNA gapped substrates containing a 5'-phosphate group at the gap.</td></tr>
<tr><td>102</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09295">pfam09295</a></td><td>395</td><td>82</td><td>29.4</td><td>3.2E+02</td><td>[-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ChAPs</td><td>ChAPs (Chs5p-Arf1p-binding proteins). ChAPs (Chs5p-Arf1p-binding proteins) are required for the export of specialized cargo from the Golgi.  They physically interact with Chs3, Chs5 and the small GTPase Arf1, and they form also interactions with each other.</td></tr>
<tr><td>103</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11690">pfam11690</a></td><td>112</td><td>73</td><td>29.4</td><td>76</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;]</td><td>DUF3287</td><td>Protein of unknown function (DUF3287). This eukaryotic family of proteins has no known function.</td></tr>
<tr><td>104</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02841">TIGR02841</a></td><td>140</td><td>39</td><td>28.2</td><td>63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>spore_YyaC</td><td>putative sporulation protein YyaC. A comparative genome analysis of all sequenced genomes of shows a number of proteins conserved strictly among the endospore-forming subset of the Firmicutes. This protein, also called YyaC, is a member of that panel and is otherwise uncharacterized. The second round of PSI-BLAST shows many similarities to the germination protease GPR, which is found in exactly the same set of organisms and has a known role in the sporulation/germination process. </td></tr>
<tr><td>105</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09250">pfam09250</a></td><td>160</td><td>25</td><td>27.3</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Prim-Pol</td><td>Bifunctional DNA primase/polymerase, N-terminal. Members of this family adopt a structure consisting of a core of antiparallel beta sheets. They are found in various bacterial hypothetical proteins, and have been shown to harbour both primase and polymerase activities.</td></tr>
<tr><td>106</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10049">PRK10049</a></td><td>765</td><td>76</td><td>27.3</td><td>1.4E+02</td><td>[&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pgaA</td><td>outer membrane protein PgaA; Provisional</td></tr>
<tr><td>107</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13371">pfam13371</a></td><td>73</td><td>62</td><td>26.7</td><td>3.1E+02</td><td>[------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_9</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>108</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01670">cd01670</a></td><td>79</td><td>53</td><td>26.7</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Death</td><td>Death Domain: a protein-protein interaction domain. Death Domains (DDs) are protein-protein interaction domains found in a variety of domain architectures. Their common feature is that they form homodimers by self-association or heterodimers by associating with other members of the DD superfamily including CARD (Caspase activation and recruitment domain), DED (Death Effector Domain), and PYRIN. Structural analysis of DD-DD complexes show that the domains interact with each other in many different ways. DD-containing proteins serve as adaptors in signaling pathways and they can recruit other proteins into signaling complexes. In mammals, they are prominent components of the programmed cell death (apoptosis) pathway and are found in a number of other signaling pathways. In invertebrates, they are involved in transcriptional regulation of zygotic patterning genes in insect embryogenesis, and are components of the ToII/NF-kappaB pathway, a conserved innate immune pathway in animal cells.</td></tr>
<tr><td>109</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06967">pfam06967</a></td><td>84</td><td>39</td><td>26.7</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Mo-nitro_C</td><td>Mo-dependent nitrogenase C-terminus. This family represents the C-terminus (approximately 80 residues) of a number of bacterial Mo-dependent nitrogenases. These are involved in nitrogen fixation in cyanobacteria. Note that many family members are hypothetical proteins.</td></tr>
<tr><td>110</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4783">COG4783</a></td><td>484</td><td>73</td><td>26.6</td><td>2E+02</td><td>[&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YfgC</td><td>Putative Zn-dependent protease, contains TPR repeats </td></tr>
<tr><td>111</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08774">cd08774</a></td><td>225</td><td>48</td><td>26.4</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>14-3-3</td><td>14-3-3 domain. 14-3-3 domain is an essential part of 14-3-3 proteins, a ubiquitous class of regulatory, phosphoserine/threonine-binding proteins found in all eukaryotic cells, including yeast, protozoa and mammalian cells. 14-3-3 proteins play important roles in many biological processes that are regulated by phosphorylation, including cell cycle regulation, cell proliferation, protein trafficking, metabolic regulation and apoptosis.  More than 300 binding partners of the 14-3-3 domain have been identified in all subcellular compartments and include transcription factors, signaling molecules, tumor suppressors, biosynthetic enzymes, cytoskeletal proteins and apoptosis factors. 14-3-3 binding can alter the conformation, localization, stability, phosphorylation state, activity as well as molecular interactions of a target protein. They function only as dimers, some preferring strictly homodimeric interaction, while others form heterodimers. Binding of the 14-3-3 domain to its target occurs in a phosphospecific manner where it binds to one of two consensus sequences of their target proteins; RSXpSXP (mode-1) and RXXXpSXP (mode-2). In some instances, 14-3-3 domain containing proteins are involved in regulation and signaling of a number of cellular processes in phosphorylation-independent manner. Many organisms express multiple isoforms: there are seven mammalian 14-3-3 family members (beta, gamma, eta, theta, epsilon, sigma, zeta), each encoded by a distinct gene, while plants contain up to 13 isoforms. The flexible C-terminal segment of 14-3-3 isoforms shows the highest sequence variability and may significantly contribute to individual isoform uniqueness by playing an important regulatory role by occupying the ligand binding groove and blocking the binding of inappropriate ligands in a distinct manner. Elevated amounts of 14-3-3 proteins are found in the cerebrospinal fluid of patients with Creutzfeldt-Jakob disease. In protozoa, like Plasmodium or Cryptosporidium parvum 14-3-3 proteins play an important role in key steps of parasite development.</td></tr>
<tr><td>112</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10370">PRK10370</a></td><td>198</td><td>54</td><td>25.9</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10370</td><td>formate-dependent nitrite reductase complex subunit NrfG; Provisional</td></tr>
<tr><td>113</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02259">pfam02259</a></td><td>350</td><td>66</td><td>25.7</td><td>3.8E+02</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FAT</td><td>FAT domain. The FAT domain is named after FRAP, ATM and TRRAP.</td></tr>
<tr><td>114</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03210">cd03210</a></td><td>126</td><td>58</td><td>25.1</td><td>87</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GST_C_Pi</td><td>C-terminal, alpha helical domain of Class Pi Glutathione S-transferases. Glutathione S-transferase (GST) C-terminal domain family, Class Pi subfamily; GSTs are cytosolic dimeric proteins involved in cellular detoxification by catalyzing the conjugation of glutathione (GSH) with a wide range of endogenous and xenobiotic alkylating agents, including carcinogens, therapeutic drugs, environmental toxins, and products of oxidative stress. The GST fold contains an N-terminal thioredoxin-fold domain and a C-terminal alpha helical domain, with an active site located in a cleft between the two domains. GSH binds to the N-terminal domain while the hydrophobic substrate occupies a pocket in the C-terminal domain. Class Pi GST is a homodimeric eukaryotic protein. The human GSTP1 is mainly found in erythrocytes, kidney, placenta and fetal liver. It is involved in stress responses and in cellular proliferation pathways as an inhibitor of JNK (c-Jun N-terminal kinase). Following oxidative stress, monomeric GSTP1 dissociates from JNK and dimerizes, losing its ability to bind JNK and causing an increase in JNK activity, thereby promoting apoptosis. GSTP1 is expressed in various tumors and is the predominant GST in a wide range of cancer cells. It has been implicated in the development of multidrug-resistant tumors.</td></tr>
<tr><td>115</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09976">pfam09976</a></td><td>145</td><td>52</td><td>24.9</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_21</td><td>Tetratricopeptide repeat. TPR repeat</td></tr>
<tr><td>116</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13281">pfam13281</a></td><td>374</td><td>65</td><td>24.6</td><td>88</td><td>[-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4071</td><td>Domain of unknown function (DUF4071). This domain is found at the N-terminus of many serine-threonine kinase-like proteins.</td></tr>
<tr><td>117</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00871">cd00871</a></td><td>175</td><td>68</td><td>24.6</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PI4Ka</td><td>Phosphoinositide 4-kinase(PI4K), accessory domain (PIK domain); PIK domain is conserved in PI3 and PI4-kinases. Its role is unclear but it has been suggested to be involved in substrate presentation. PI4K phosphorylates hydroxylgroup at position 4 on the inositol ring of phosphoinositide, the first commited step in the phosphatidylinositol cycle.</td></tr>
<tr><td>118</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04859">cd04859</a></td><td>152</td><td>78</td><td>23.6</td><td>87</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Prim_Pol</td><td>Prim_Pol: Primase-polymerase (primpol) domain of the type found in bifunctional replicases from archaeal plasmids, including ORF904 protein of the crenarchaeal plasmid pRN1 from Sulfolobus islandicus (pRN1 primpol). These primpol domains belong to the archaeal/eukaryal primase (AEP) superfamily. This group includes archaeal plasmids and bacteriophage AEPs. The ORF904 protein is a multifunctional protein having ATPase, primase and DNA polymerase activity, and may play a role in the replication of the archaeal plasmid. The pRN1 primpol domain exhibits DNA polymerase and primase activities; a cluster of active site residues (three acidic residues, and a histidine) is required for both these activities. For pRN1 primpol, the primase activity prefers dNTPs to rNTPs; incorporation of dNTPs requires rNTP as cofactor. The pRN1 primpol contains an unusual zinc-binding stem, which is not conserved in other members of this group.</td></tr>
<tr><td>119</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11447">PRK11447</a></td><td>1157</td><td>57</td><td>22.9</td><td>1.8E+02</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11447</td><td>cellulose synthase subunit BcsC; Provisional</td></tr>
<tr><td>120</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10153">PRK10153</a></td><td>517</td><td>55</td><td>22.8</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10153</td><td>DNA-binding transcriptional activator CadC; Provisional</td></tr>
<tr><td>121</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09976">pfam09976</a></td><td>145</td><td>89</td><td>22.8</td><td>2.7E+02</td><td>[--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_21</td><td>Tetratricopeptide repeat. TPR repeat</td></tr>
<tr><td>122</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02679">cd02679</a></td><td>79</td><td>31</td><td>22.3</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MIT_spastin</td><td>MIT: domain contained within Microtubule Interacting and Trafficking molecules. This MIT domain sub-family is found in the AAA protein spastin, a probable ATPase involved in the assembly or function of nuclear protein complexes; spastins might also be involved in microtubule dynamics. The molecular function of the MIT domain is unclear.</td></tr>
<tr><td>123</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10395">cd10395</a></td><td>101</td><td>43</td><td>21.8</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH2_RIN3</td><td>Src homology 2 (SH2) domain found in Ras and Rab interactor 3 (RIN3)-like proteins. RIN3, a member of the RIN (AKA Ras interaction/interference) family, have multifunctional domains including SH2 and proline-rich (PR) domains in the N-terminal region, and RIN-family homology (RH), VPS9 and Ras-association (RA) domains in the C-terminal region. RIN proteins function as Rab5-GEFs. RIN3 stimulated the formation of GTP-bound Rab31, a Rab5-subfamily GTPase, and formed enlarged vesicles and tubular structures, where it colocalized with Rab31. Transferrin appeared to be transported partly through the RIN3-positive vesicles to early endosomes. RIN3 interacts via its Pro-rich domain with amphiphysin II, which contains SH3 domain and participates in receptor-mediated endocytosis. RIN3, a Rab5 and Rab31 GEF, plays an important role in the transport pathway from plasma membrane to early endosomes. Mutations in the region between the SH2 and RH domain of RIN3 specifically abolished its GEF action on Rab31, but not Rab5. RIN3 was also found to partially translocate the cation-dependent mannose 6-phosphate receptor from the trans-Golgi network to peripheral vesicles and that this is dependent on its Rab31-GEF activity. These data indicate that RIN3 specifically acts as a GEF for Rab31. In general SH2 domains are involved in signal transduction. They typically bind pTyr-containing ligands via two surface pockets, a pTyr and hydrophobic binding pocket, allowing proteins with SH2 domains to localize to tyrosine phosphorylated sites.</td></tr>
<tr><td>124</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06423">PRK06423</a></td><td>73</td><td>50</td><td>21.6</td><td>84</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06423</td><td>phosphoribosylformylglycinamidine synthase; Provisional</td></tr>
<tr><td>125</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01046">cd01046</a></td><td>123</td><td>103</td><td>21.4</td><td>1E+02</td><td>[&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rubrerythrin_like</td><td>rubrerythrin-like, diiron-binding domain. Rubrerythrin-like domain, similar to rubrerythrin, a nonheme iron binding domain found in many air-sensitive bacteria and archaea, and member of a broad superfamily of ferritin-like diiron-carboxylate proteins. Rubrerythrin is thought to reduce hydrogen peroxide as part of an oxidative stress protection system. The rubrerythrin protein has two domains, a binuclear metal center located within a four-helix bundle of the rubrerythrin domain, and a rubredoxin domain. The Rubrerythrin-like domains in this CD are singular domains (no C-terminus rubredoxin domain) and are phylogenetically distinct from rubrerythrin domains of rubrerythrin-rubredoxin proteins.</td></tr>
<tr><td>126</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04510">TIGR04510</a></td><td>814</td><td>46</td><td>21.3</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>mod_pep_cyc</td><td>putative peptide modification system cyclase. Members of this family show homology to mononucleotidyl cyclases and to tetratricopeptide repeat (TPR) proteins. Members occur in next to two other markers of ribosomal peptide modification systems. One is a dehydrogenase related to SagB proteins from thiazole/oxazole modification systems. The other is the putative precursor, related to the nitrile hydratase-related leader peptide (NHLP) and nitrile hydratase alpha subunit families. These systems occur in many species of Xanthomonas and Stenotrophomonas, among others.</td></tr>
<tr><td>127</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08966">pfam08966</a></td><td>72</td><td>32</td><td>20.9</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1882</td><td>Domain of unknown function (DUF1882). This domain is found in a set of hypothetical bacterial proteins.</td></tr>
<tr><td>128</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12261">cd12261</a></td><td>73</td><td>36</td><td>20.7</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM1_3_MRN1</td><td>RNA recognition motif 1 and 3 in RNA-binding protein MRN1 and similar proteins. This subfamily corresponds to the RRM1 and RRM3 of MRN1, also termed multicopy suppressor of RSC-NHP6 synthetic lethality protein 1, or post-transcriptional regulator of 69 kDa, which is an RNA-binding protein found in yeast. Although its specific biological role remains unclear, MRN1 might be involved in translational regulation. Members in this family contain four copies of conserved RNA recognition motif (RRM), also known as RBD (RNA binding domain) or RNP (ribonucleoprotein domain). </td></tr>
</table>
</body>
