<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08754">cd08754</a></td><td>222</td><td>28</td><td>69.9</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;]</td><td>RGS_LARG</td><td>Regulator of G protein signaling (RGS) domain found in the leukemia-associated Rho guanine nucleotide exchange factor (RhoGEF) protein (LARG). The RGS domain is an essential part of the leukemia-associated RhoGEF protein (LARG), a member of the RhoGEF (Rho guanine nucleotide exchange factor) subfamily of the RGS protein family. The RhoGEFs are peripheral membrane proteins that regulate essential cellular processes, including cell shape, cell migration, cell cycle progression of cells, and gene transcription by linking signals from heterotrimeric G-alpha12/13 protein-coupled receptors to Rho GTPase activation, leading to various cellular responses, such as actin reorganization and gene expression. The RhoGEF subfamily includes p115RhoGEF, LARG, PDZ-RhoGEF, and its rat specific splice variant GTRAP48. The RGS domain of RhoGEFs has very little sequence similarity with the canonical RGS domain of the RGS proteins and is often refered to as RH (RGS Homology) domain. In addition to being a G-alpha13 effector, the LARG protein also functions as a GTPase-activating protein (GAP) for G-alpha13. RGS proteins play critical regulatory role as GTPase activating proteins (GAPs) of the heterotrimeric G-protein G-alpha-subunits. RGS proteins play critical regulatory role as GTPase activating proteins (GAPs) of the heterotrimeric G-protein G-alpha-subunits. RGS proteins regulate many aspects of embryonic development such as glial differentiation, embryonic axis formation, skeletal and muscle development, cell migration during early embryogenesis, as well as apoptosis, cell proliferation, and modulation of cardiac development.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12338">cd12338</a></td><td>72</td><td>15</td><td>67.0</td><td>2.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM1_SRSF1_like</td><td>RNA recognition motif 1 in serine/arginine-rich splicing factor 1 (SRSF1) and similar proteins. This subgroup corresponds to the RRM1 in three serine/arginine (SR) proteins: serine/arginine-rich splicing factor 1 (SRSF1 or ASF-1), serine/arginine-rich splicing factor 9 (SRSF9 or SRp30C), and plant pre-mRNA-splicing factor SF2 (SR1). SRSF1 is a shuttling SR protein involved in constitutive and alternative splicing, nonsense-mediated mRNA decay (NMD), mRNA export and translation. It also functions as a splicing-factor oncoprotein that regulates apoptosis and proliferation to promote mammary epithelial cell transformation. SRSF9 has been implicated in the activity of many elements that control splice site selection, the alternative splicing of the glucocorticoid receptor beta in neutrophils and in the gonadotropin-releasing hormone pre-mRNA. It can also interact with other proteins implicated in alternative splicing, including YB-1, rSLM-1, rSLM-2, E4-ORF4, Nop30, and p32. Both, SRSF1 and SRSF9, contain two N-terminal RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains), and a C-terminal RS domains rich in serine-arginine dipeptides. In contrast, SF2 contains two N-terminal RRMs and a C-terminal PSK domain rich in proline, serine and lysine residues.  </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13358">PRK13358</a></td><td>269</td><td>80</td><td>58.7</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13358</td><td>protocatechuate 4,5-dioxygenase subunit beta; Provisional</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12597">cd12597</a></td><td>73</td><td>15</td><td>55.2</td><td>5.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM1_SRSF1</td><td>RNA recognition motif 1 in serine/arginine-rich splicing factor 1 (SRSF1) and similar proteins. This subgroup corresponds to the RRM1 of SRSF1, also termed alternative-splicing factor 1 (ASF-1), or pre-mRNA-splicing factor SF2, P33 subunit. SRSF1 is a splicing regulatory serine/arginine (SR) protein involved in constitutive and alternative splicing, nonsense-mediated mRNA decay (NMD), mRNA export and translation. It also functions as a splicing-factor oncoprotein that regulates apoptosis and proliferation to promote mammary epithelial cell transformation. SRSF1 is a shuttling SR protein and contains two N-terminal RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains), separated by a long glycine-rich spacer, and a C-terminal RS domains rich in serine-arginine dipeptides. </td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12646">cd12646</a></td><td>77</td><td>14</td><td>42.1</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM_SRSF7</td><td>RNA recognition motif in vertebrate serine/arginine-rich splicing factor 7 (SRSF7). This subgroup corresponds to the RRM of SRSF7, also termed splicing factor 9G8, is a splicing regulatory serine/arginine (SR) protein that plays a crucial role in both constitutive splicing and alternative splicing of many pre-mRNAs. Its localization and functions are tightly regulated by phosphorylation. SRSF7 is predominantly present in the nuclear and can shuttle between nucleus and cytoplasm. It cooperates with the export protein, Tap/NXF1, helps mRNA export to the cytoplasm, and enhances the expression of unspliced mRNA. SRSF7 inhibits tau E10 inclusion through directly interacting with the proximal downstream intron of E10, a clustering region for frontotemporal dementia with Parkinsonism (FTDP) mutations. SRSF7 contains a single N-terminal RNA recognition motif (RRM), also termed RBD (RNA binding domain) or RNP (ribonucleoprotein domain), followed by a CCHC-type zinc knuckle motif in its median region, and a C-terminal RS domain rich in serine-arginine dipeptides. The RRM domain is involved in RNA binding, and the RS domain has been implicated in protein shuttling and protein-protein interactions. </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12645">cd12645</a></td><td>81</td><td>13</td><td>40.4</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM_SRSF3</td><td>RNA recognition motif in vertebrate serine/arginine-rich splicing factor 3 (SRSF3). This subgroup corresponds to the RRM of SRSF3, also termed pre-mRNA-splicing factor SRp20, a splicing regulatory serine/arginine (SR) protein that modulates alternative splicing by interacting with RNA cis-elements in a concentration- and cell differentiation-dependent manner. It is also involved in termination of transcription, alternative RNA polyadenylation, RNA export, and protein translation. SRSF3 is critical for cell proliferation and tumor induction and maintenance. SRSF3 can shuttle between the nucleus and cytoplasm. It contains a single N-terminal RNA recognition motif (RRM), also termed RBD (RNA binding domain) or RNP (ribonucleoprotein domain), and a C-terminal RS domain rich in serine-arginine dipeptides. The RRM domain is involved in RNA binding, and the RS domain has been implicated in protein shuttling and protein-protein interactions. </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03096">TIGR03096</a></td><td>135</td><td>16</td><td>35.0</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>nitroso_cyanin</td><td>nitrosocyanin. Nitrosocyanin, as described from the obligate chemolithoautotroph Nitrosomonas europaea, is a red copper protein of unknown function with sequence similarity to a number of blue copper redox proteins. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08142">pfam08142</a></td><td>81</td><td>12</td><td>33.5</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AARP2CN</td><td>AARP2CN (NUC121) domain. This domain is the central domain of AARP2. It is weakly similar to the GTP-binding domain of elongation factor TU.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03822">TIGR03822</a></td><td>321</td><td>28</td><td>32.9</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AblA_like_2</td><td>lysine-2,3-aminomutase-related protein. Members of this protein form a distinctive clade, homologous to lysine-2,3-aminomutase (of Bacillus, Clostridium, and methanogenic archaea) and likely similar in function. Members of this family are found in Rhodopseudomonas, Caulobacter crescentus, Bradyrhizobium, etc.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11792">pfam11792</a></td><td>43</td><td>36</td><td>32.1</td><td>47</td><td>[&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Baculo_LEF5_C</td><td>Baculoviridae late expression factor 5 C-terminal domain. This C-terminal domain is likely to be a zinc-binding domain.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12373">cd12373</a></td><td>73</td><td>15</td><td>32.0</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM_SRSF3_like</td><td>RNA recognition motif in serine/arginine-rich splicing factor 3 (SRSF3) and similar proteins. This subfamily corresponds to the RRM of two serine/arginine (SR) proteins, serine/arginine-rich splicing factor 3 (SRSF3) and serine/arginine-rich splicing factor 7 (SRSF7). SRSF3, also termed pre-mRNA-splicing factor SRp20, modulates alternative splicing by interacting with RNA cis-elements in a concentration- and cell differentiation-dependent manner. It is also involved in termination of transcription, alternative RNA polyadenylation, RNA export, and protein translation. SRSF3 is critical for cell proliferation, and tumor induction and maintenance. It can shuttle between the nucleus and cytoplasm. SRSF7, also termed splicing factor 9G8, plays a crucial role in both constitutive splicing and alternative splicing of many pre-mRNAs. Its localization and functions are tightly regulated by phosphorylation. SRSF7 is predominantly present in the nuclear and can shuttle between nucleus and cytoplasm. It cooperates with the export protein, Tap/NXF1, helps mRNA export to the cytoplasm, and enhances the expression of unspliced mRNA. Moreover, SRSF7 inhibits tau E10 inclusion through directly interacting with the proximal downstream intron of E10, a clustering region for frontotemporal dementia with Parkinsonism (FTDP) mutations. Both SRSF3 and SRSF7 contain a single N-terminal RNA recognition motif (RRM), also termed RBD (RNA binding domain) or RNP (ribonucleoprotein domain), and a C-terminal RS domain rich in serine-arginine dipeptides. The RRM domain is involved in RNA binding, and the RS domain has been implicated in protein shuttling and protein-protein interactions. </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12454">cd12454</a></td><td>80</td><td>26</td><td>30.8</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM2_RIM4_like</td><td>RNA recognition motif 2 in yeast meiotic activator RIM4 and similar proteins. This subfamily corresponds to the RRM2 of RIM4, also termed regulator of IME2 protein 4, a putative RNA binding protein that is expressed at elevated levels early in meiosis. It functions as a meiotic activator required for both the IME1- and IME2-dependent pathways of meiotic gene expression, as well as early events of meiosis, such as meiotic division and recombination, in Saccharomyces cerevisiae. RIM4 contains two RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains). The family also includes a putative RNA-binding protein termed multicopy suppressor of sporulation protein Msa1. It is a putative RNA-binding protein encoded by a novel gene, msa1, from the fission yeast Schizosaccharomyces pombe. Msa1 may be involved in the inhibition of sexual differentiation by controlling the expression of Ste11-regulated genes, possibly through the pheromone-signaling pathway. Like RIM4, Msa1 also contains two RRMs, both of which are essential for the function of Msa1. </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13351">pfam13351</a></td><td>84</td><td>43</td><td>30.1</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4099</td><td>Protein of unknown function (DUF4099). A family of uncharacterized proteins found by clustering human gut metagenomic sequences. The C-terminal repeat region of this family is DUF4098, pfam13345.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06553">PRK06553</a></td><td>308</td><td>51</td><td>29.7</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06553</td><td>lipid A biosynthesis lauroyl acyltransferase; Provisional</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05409">pfam05409</a></td><td>293</td><td>27</td><td>22.5</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_C30</td><td>Coronavirus endopeptidase C30. Corresponds to Merops family C30. These peptidases are involved in viral polyprotein processing in replication.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12451">cd12451</a></td><td>79</td><td>54</td><td>22.4</td><td>73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM2_NUCLs</td><td>RNA recognition motif 2 in nucleolin-like proteins mainly from plants. This subfamily corresponds to the RRM2 of a group of plant nucleolin-like proteins, including nucleolin 1 (also termed protein nucleolin like 1) and nucleolin 2 (also termed protein nucleolin like 2, or protein parallel like 1). They play roles in the regulation of ribosome synthesis and in the growth and development of plants. Like yeast nucleolin, nucleolin-like proteins possess two RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains).  </td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12307">cd12307</a></td><td>74</td><td>19</td><td>21.4</td><td>64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM_NIFK_like</td><td>RNA recognition motif in nucleolar protein interacting with the FHA domain of pKI-67 (NIFK) and similar proteins. This subgroup corresponds to the RRM of NIFK and Nop15p. NIFK, also termed MKI67 FHA domain-interacting nucleolar phosphoprotein, or nucleolar phosphoprotein Nopp34, is a putative RNA-binding protein interacting with the forkhead associated (FHA) domain of pKi-67 antigen in a mitosis-specific and phosphorylation-dependent manner. It is nucleolar in interphase but associates with condensed mitotic chromosomes. This family also includes Saccharomyces cerevisiae YNL110C gene encoding ribosome biogenesis protein 15 (Nop15p), also termed nucleolar protein 15. Both, NIFK and Nop15p, contain an RNA recognition motif (RRM), also termed RBD (RNA binding domain) or RNP (ribonucleoprotein domain). </td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02118">TIGR02118</a></td><td>100</td><td>67</td><td>21.3</td><td>98</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02118</td><td>conserved hypothetical protein. This model represents a small family of proteins of unknown function, each about 105 amino acids in length. Conserved sites in the multiple alignment include a pair of aromatic residues, a histidine, and an aspartate. </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12535">pfam12535</a></td><td>58</td><td>10</td><td>20.6</td><td>47</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Nudix_N</td><td>Hydrolase of X-linked nucleoside diphosphate N terminal. This family of proteins is found in eukaryotes. Proteins in this family are typically between 847 and 5344 amino acids in length. These enzymes hydrolyse the molecular motif of a nucleoside diphosphate linked to some other moiety, X.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03378">PRK03378</a></td><td>292</td><td>52</td><td>20.5</td><td>70</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ppnK</td><td>inorganic polyphosphate/ATP-NAD kinase; Provisional</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11739">pfam11739</a></td><td>207</td><td>51</td><td>20.3</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DctA-YdbH</td><td>Dicarboxylate transport. In certain bacterial families this protein is expressed from the ydbH gene, and there is a suggestion that this is a form of DctA or dicarboxylate transport protein. Dicarboxylate transport proteins are found in aerobic bacteria which grow on succinate or other C4-dicarboxylates.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10297">pfam10297</a></td><td>17</td><td>9</td><td>20.1</td><td>42</td><td>[---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Hap4_Hap_bind</td><td>Minimal binding motif of Hap4 for binding to Hap2/3/5. In Saccharomyces cerevisiae, the haem-activated protein complex Hap2/3/4/5 plays a major role in the transcription of genes involved in respiration. Hap4_Hap_bind is the essential domain of Hap4 which allows it to associate with Hap2, Hap3 and Hap5 to form the Hap complex.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1679">COG1679</a></td><td>403</td><td>38</td><td>20.1</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1679</td><td>Predicted aconitase  </td></tr>
</table>
</body>
