<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07957">cd07957</a></td><td>129</td><td>58</td><td>72.9</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------------&nbsp;&nbsp;]</td><td>Anticodon_Ia_Met</td><td>Anticodon-binding domain of methionyl tRNA synthetases. This domain is found in methionyl tRNA synthetases (MetRS), which belong to the class Ia aminoacyl tRNA synthetases. It lies C-terminal to the catalytic core domain, and recognizes and specifically binds to the tRNA anticodon (CAU). MetRS catalyzes the transfer of methionine to the 3'-end of its tRNA.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11125">pfam11125</a></td><td>54</td><td>39</td><td>66.9</td><td>9.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2830</td><td>Protein of unknown function (DUF2830). Several members in this viral family of proteins are annotated as lysis proteins.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14424">cd14424</a></td><td>37</td><td>25</td><td>60.9</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CUE_Cue1p_like</td><td>CUE domain found in yeast ubiquitin-binding protein CUE1 (Cue1p), CUE4 (Cue4p) and similar proteins. Cue1p, also called coupling of ubiquitin conjugation to ER degradation protein 1 or kinetochore-defect suppressor 4, is encoded by the open reading frame (ORF) YMR264W in yeast. It is an N-terminally membrane-anchored endoplasmic reticulum (ER) protein essential for the activity of the two major yeast RING finger ubiquitin ligases (E3s) implicated in ER-associated degradation (ERAD). It interacts with the ERAD ubiquitin-conjugating enzyme (E2) Ubc7p in vivo, stimulates Ubc7p E2 activity, and further activates ER-associated protein degradation. Cue1p contains a CUE domain which binds ubiquitin much more weakly than those of other CUE domain containing proteins. It also has an Ubc7p binding-domain at the C-terminal region which is required for Ubc7p-dependent ubiquitylation and for degradation of substrates in the ER. This family also includes Cue4p, also called coupling of ubiquitin conjugation to ER degradation protein 4. It is encoded by the open reading frame (ORF) YML101C in yeast. Cue4p contains a CUE domain which shows high level of similarity with that of Cue1p.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05592">cd05592</a></td><td>320</td><td>21</td><td>58.2</td><td>4.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>STKc_nPKC_theta_like</td><td>Catalytic domain of the Serine/Threonine Kinases, Novel Protein Kinase C theta, delta, and similar proteins. STKs catalyze the transfer of the gamma-phosphoryl group from ATP to serine/threonine residues on protein substrates. PKC-theta is selectively expressed in T-cells and plays an important and non-redundant role in several aspects of T-cell biology. PKC-delta plays a role in cell cycle regulation and programmed cell death in many cell types. PKCs are classified into three groups (classical, atypical, and novel) depending on their mode of activation and the structural characteristics of their regulatory domain. nPKCs are calcium-independent, but require DAG (1,2-diacylglycerol) and phosphatidylserine (PS) for activity. There are four nPKC isoforms, delta, epsilon, eta, and theta. The nPKC-theta-like subfamily is part of a larger superfamily that includes the catalytic domains of other STKs, protein tyrosine kinases, RIO kinases, aminoglycoside phosphotransferase, choline kinase, and phosphoinositide 3-kinase.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04100">pfam04100</a></td><td>375</td><td>61</td><td>56.4</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Vps53_N</td><td>Vps53-like, N-terminal. Vps53 complexes with Vps52 and Vps54 to form a multi- subunit complex involved in regulating membrane trafficking events.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02552">TIGR02552</a></td><td>135</td><td>29</td><td>55.5</td><td>4.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LcrH_SycD</td><td>type III secretion low calcium response chaperone LcrH/SycD. Genes in this family are found in type III secretion operons. LcrH, from Yersinia is believed to have a regulatory function in the low-calcium response of the secretion system. The same protein is also known as SycD (SYC = Specific Yop Chaperone) for its chaperone role. In Pseudomonas, where the homolog is known as PcrH, the chaperone role has been demonstrated and the regulatory role appears to be absent. ScyD/LcrH contains three central tetratricopeptide-like repeats that are predicted to fold into an all-alpha-helical array.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14559">pfam14559</a></td><td>68</td><td>28</td><td>50.3</td><td>33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;]</td><td>TPR_19</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00215">cd00215</a></td><td>97</td><td>20</td><td>49.7</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;]</td><td>PTS_IIA_lac</td><td>PTS_IIA, PTS system, lactose/cellobiose specific IIA subunit. The bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS) is a multi-protein system involved in the regulation of a variety of metabolic and transcriptional processes. This family is one of four structurally and functionally distinct group IIA PTS system cytoplasmic enzymes, necessary for the uptake of carbohydrates across the cytoplasmic membrane and their phosphorylation. This family of proteins normally function as a homotrimer, stabilized by a centrally located metal ion. Separation into subunits is thought to occur after phosphorylation.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07719">pfam07719</a></td><td>34</td><td>15</td><td>46.9</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_2</td><td>Tetratricopeptide repeat. This Pfam entry includes outlying Tetratricopeptide-like repeats (TPR) that are not matched by pfam00515.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13428">pfam13428</a></td><td>44</td><td>20</td><td>46.2</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_14</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02845">pfam02845</a></td><td>42</td><td>33</td><td>45.5</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CUE</td><td>CUE domain. CUE domains have been shown to bind ubiquitin. It has been suggested that CUE domains are related to pfam00627 and this has been confirmed by the structure of the domain. CUE domains also occur in two protein of the IL-1 signal transduction pathway, tollip and TAB2.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01471">pfam01471</a></td><td>57</td><td>32</td><td>43.0</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PG_binding_1</td><td>Putative peptidoglycan binding domain. This domain is composed of three alpha helices. This domain is found at the N or C terminus of a variety of enzymes involved in bacterial cell wall degradation. This domain may have a general peptidoglycan binding function. This family is found N-terminal to the catalytic domain of matrixins. The domain is found to bind peptidoglycan experimentally.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02186">cd02186</a></td><td>434</td><td>15</td><td>42.9</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>alpha_tubulin</td><td>The alpha-tubulin family. The tubulin superfamily includes five distinct families, the alpha-, beta-, gamma-, delta-, and epsilon-tubulins and a sixth family (zeta-tubulin) which is present only in kinetoplastid protozoa. The alpha- and beta-tubulins are the major components of microtubules, while gamma-tubulin plays a major role in the nucleation of microtubule assembly.  The delta- and epsilon-tubulins are widespread but unlike the alpha, beta, and gamma-tubulins they are not ubiquitous among eukaryotes. The alpha/beta-tubulin heterodimer is the structural subunit of microtubules.  The alpha- and beta-tubulins share 40% amino-acid sequence identity, exist in several isotype forms, and undergo a variety of posttranslational modifications.  The structures of alpha- and beta-tubulin are basically identical: each monomer is formed by a core of two beta-sheets surrounded by alpha-helices. The monomer structure is very compact, but can be divided into three regions based on function: the amino-terminal nucleotide-binding region, an intermediate taxol-binding region and the carboxy-terminal region which probably constitutes the binding surface for motor proteins.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14110">PRK14110</a></td><td>291</td><td>34</td><td>38.1</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14110</td><td>F0F1 ATP synthase subunit gamma; Provisional</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1447">COG1447</a></td><td>105</td><td>22</td><td>37.9</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;]</td><td>CelC</td><td>Phosphotransferase system cellobiose-specific component IIA </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07721">pfam07721</a></td><td>26</td><td>15</td><td>37.8</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_4</td><td>Tetratricopeptide repeat. This Pfam entry includes tetratricopeptide-like repeats not detected by the pfam00515, pfam07719 and pfam07720 models.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05578">cd05578</a></td><td>257</td><td>21</td><td>36.6</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>STKc_Yank1</td><td>Catalytic domain of the Serine/Threonine Kinase, Yank1. STKs catalyze the transfer of the gamma-phosphoryl group from ATP to serine/threonine residues on protein substrates. This subfamily contains uncharacterized STKs with similarity to the human protein designated as Yank1 or STK32A. The Yank1 subfamily is part of a larger superfamily that includes the catalytic domains of other STKs, protein tyrosine kinases, RIO kinases, aminoglycoside phosphotransferase, choline kinase, and phosphoinositide 3-kinase.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11577">pfam11577</a></td><td>68</td><td>16</td><td>36.4</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;]</td><td>NEMO</td><td>NF-kappa-B essential modulator NEMO. NEMO is a regulatory protein which is part of the IKK complex along with the catalytic IKKalpha and beta kinases. The IKK complex phosphorylates IkappaB targeting it for degradation which results in the release of NF-kappaB which initiates the inflammatory response, cell proliferation or cell differentiation. NEMO activates the IKK complex's activity by associating with the unphosphorylated IKK kinase C termini.The core domain of NEMO is a dimer which binds to two fragments of IKK.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4273">COG4273</a></td><td>135</td><td>37</td><td>36.4</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4273</td><td>Uncharacterized protein, contains metal-binding DGC domain  </td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00231">pfam00231</a></td><td>288</td><td>33</td><td>35.2</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ATP-synt</td><td>ATP synthase. </td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13181">pfam13181</a></td><td>34</td><td>19</td><td>34.2</td><td>42</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_8</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05991">pfam05991</a></td><td>165</td><td>19</td><td>33.7</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;]</td><td>NYN_YacP</td><td>YacP-like NYN domain. This family consists of bacterial proteins related to YacP. This family is uncharacterized functionally, but it has been suggested that these proteins are nucleases due to them containing a NYN domain. NYN (for N4BP1, YacP-like Nuclease) domains were discovered by Anantharaman and Aravind. Based on gene neighborhoods it was suggested that the bacterial YacP proteins interact with the Ribonuclease III and TrmH methylase in a processome complex that catalyzes the maturation of rRNA and tRNA.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05068">pfam05068</a></td><td>169</td><td>59</td><td>33.2</td><td>63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------&nbsp;&nbsp;&nbsp;]</td><td>MtlR</td><td>Mannitol repressor. The mannitol operon of Escherichia coli, encoding the mannitol-specific enzyme II of the phosphotransferase system (MtlA) and mannitol phosphate dehydrogenase (MtlD) contains an additional downstream open reading frame which encodes the mannitol repressor (MtlR).</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10769">pfam10769</a></td><td>74</td><td>37</td><td>32.9</td><td>84</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2594</td><td>Protein of unknown function (DUF2594). This family of proteins with unknown function appear to be restricted to Enterobacteriaceae.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14368">cd14368</a></td><td>41</td><td>23</td><td>32.4</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CUE_DEF1_like</td><td>CUE domain found in fungal RNA polymerase II degradation factor 1 (DEF1) and similar proteins. DEF1, also called RRM3-interacting protein 1, is a RNA Polymerase II (RNAPII) degradation factor that may be required to couple arrested RNAPII to the proteasome to facilitate its degradation. It contains a CUE domain that is responsible for the binding of ubiquitin. The family also includes many uncharacterized hypothetical proteins. They show a high level of sequence similarity with DEF1.</td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12151">cd12151</a></td><td>282</td><td>32</td><td>32.4</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F1-ATPase_gamma</td><td>mitochondrial ATP synthase gamma subunit. The F-ATPase is found in bacterial plasma membranes, mitochondrial inner membranes and in chloroplast thylakoid membranes. It has also been found in the archaea Methanosarcina barkeri. It uses a proton gradient to drive ATP synthesis and hydrolyzes ATP to build the proton gradient. The extrinisic membrane domain of F-ATPases is composed of alpha, beta, gamma, delta, and epsilon (not present in bacteria) subunits with a stoichiometry of 3:3:1:1:1. Alpha and beta subunit form the globular catalytic moiety, a hexameric ring of alternating subunits. Gamma, delta and epsilon subunits form a stalk, connecting F1 to F0, the integral membrane proton translocating domain.</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03029">TIGR03029</a></td><td>274</td><td>35</td><td>32.3</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>EpsG</td><td>chain length determinant protein tyrosine kinase EpsG. The proteins in this family are homologs of the EpsG protein found in Methylobacillus strain 12S and are generally found in operons with other Eps homologs. The protein is believed to function as the protein tyrosine kinase component of the chain length regulator (along with the transmembrane component EpsF).</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14804">pfam14804</a></td><td>52</td><td>29</td><td>32.1</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Jag_N</td><td>Jag N-terminus. This domain is found at the N-terminus of proteins containing pfam13083 and pfam01424, including the jag proteins.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02255">pfam02255</a></td><td>96</td><td>21</td><td>31.9</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;]</td><td>PTS_IIA</td><td>PTS system, Lactose/Cellobiose specific IIA subunit. The bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS) is a multi-protein system involved in the regulation of a variety of metabolic and transcriptional processes. The lactose/cellobiose-specific family are one of four structurally and functionally distinct group IIA PTS system enzymes. This family of proteins normally function as a homotrimer, stabilized by a centrally located metal ion. Separation into subunits is thought to occur after phosphorylation.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10613">PRK10613</a></td><td>74</td><td>35</td><td>29.8</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10613</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00133">PRK00133</a></td><td>673</td><td>62</td><td>29.4</td><td>3.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;&nbsp;]</td><td>metG</td><td>methionyl-tRNA synthetase; Reviewed</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06202">pfam06202</a></td><td>367</td><td>19</td><td>28.8</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;]</td><td>GDE_C</td><td>Amylo-alpha-1,6-glucosidase. This family includes human glycogen branching enzyme. This enzyme contains a number of distinct catalytic activities. It has been shown for the yeast homologue that mutations in this region disrupt the enzymes Amylo-alpha-1,6-glucosidase (EC:3.2.1.33).</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14562">PRK14562</a></td><td>204</td><td>18</td><td>28.8</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;]</td><td>PRK14562</td><td>haloacid dehalogenase superfamily protein; Provisional</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14561">pfam14561</a></td><td>90</td><td>27</td><td>28.7</td><td>53</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_20</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13423">PRK13423</a></td><td>288</td><td>34</td><td>28.2</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13423</td><td>F0F1 ATP synthase subunit gamma; Provisional</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12178">pfam12178</a></td><td>38</td><td>27</td><td>27.8</td><td>71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>INCENP_N</td><td>Chromosome passenger complex (CPC) protein INCENP N terminal. This domain family is found in eukaryotes, and is approximately 40 amino acids in length. INCENP is a regulatory protein in the chromosome passenger complex. It is involved in regulation of the catalytic protein Aurora B. It performs this function in association with two other proteins - Survivin and Borealin. These proteins form a tight three-helical bundle. The N terminal domain is the domain involved in formation of this three helical bundle.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09525">cd09525</a></td><td>67</td><td>31</td><td>27.6</td><td>42</td><td>[&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SAM_GAREM</td><td>SAM domain of GAREM subfamily. SAM (sterile alpha motif) domain of GAREM (Grb2-associated and regulator of Erk/MARK) protein subfamily (also known as FAM59A) is a putative protein-protein interaction domain. SAM domain is a widespread domain in signaling proteins. Proteins of this group have SAM at the C-terminus. Human GAREM protein is known to play a role in regulation of the EGF (Epidermal Growth Factor) receptor and of Gab or insulin preceptor substrate-1 family proteins. Grb2 (Growth factor receptor-bound) protein was identified as a binding partner of human GAREM. Proline-rich motifs and phosphorylation of two conserved tyrosines in GAREM are important for the interaction with the SH3 domains of Grb2 protein; however these motifs and residues do not belong to the SAM domain.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14484">cd14484</a></td><td>134</td><td>48</td><td>27.3</td><td>2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SPX_GDE1_like</td><td>SPX domain of Gde1 and similar proteins. This region has been named the SPX domain after (Syg1, Pho81 and XPR1). The domain is found at the amino terminus of a variety of proteins. The N-termini of several proteins involved in the regulation of phosphate transport, including the putative phosphate level sensors Pho81 from Saccharomyces cerevisiae and NUC-2 from Neurospora crassa, are also members of this family. The yeast protein Gde1/Ypl110c is similar to both, NUC-2 and Pho81, in sharing their multi-domain architecture, which includes the SPX N-terminal domain followed by several ankyrin repeats and a C-terminal glycerophosphodiester phosphodiesterase domain (GDPD). Gde1 hydrolyzes intracellular glycerophosphocholine into glycerolphosphate and choline, and plays a role in the utilization of glycerophosphocholine as a source for phosphate.</td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14048">pfam14048</a></td><td>96</td><td>16</td><td>26.7</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MBD_C</td><td>C-terminal domain of methyl-CpG binding protein 2 and 3. CpG-methylation is a frequently occurring epigenetic modification of vertebrate genomes resulting in transcriptional repression. This domain was found at the C-terminus of the methyl-CpG-binding domain (MBD) containing proteins MBD2 and MBD3, the latter was shown to not bind directly to methyl-CpG DNA but rather interact with components of the NuRD/Mi2 complex, an abundant deacetylase complex. The domain is subject to structure determination by the Joint Center of Structural Genomics.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14994">PRK14994</a></td><td>287</td><td>45</td><td>26.3</td><td>92</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14994</td><td>SAM-dependent 16S ribosomal RNA C1402 ribose 2'-O-methyltransferase; Provisional</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01935">TIGR01935</a></td><td>150</td><td>44</td><td>26.0</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NOT-MenG</td><td>RraA famliy. The E. coli member of this family has been characterized as a regulator of RNase E and its crystal structure has been analyzed. This model was initially classified as a "hypothetical equivalog" expressing the tentative hypothesis that all members might have the same function as the E. coli enzyme. Considering the second clade of enterobacterial sequences within this family, that appears to be less tenable. The function of these sequences outside of the narrow RraA equivalog model (TIGR02998) remains obscure. All of these were initially annotated as MenG, AKA S-adenosylmethionine: 2-demethylmenaquinone methyltransferase (EC 2.1.-.-). See the references characterizing this as a case of transitive annotation error in the case of the E. coli protein. </td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08542">pfam08542</a></td><td>89</td><td>13</td><td>25.4</td><td>57</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rep_fac_C</td><td>Replication factor C C-terminal domain. This is the C-terminal domain of RFC (replication factor-C) protein of the clamp loader complex which binds to the DNA sliding clamp (proliferating cell nuclear antigen, PCNA). The five modules of RFC assemble into a right-handed spiral, which results in only three of the five RFC subunits (RFC-A, RFC-B and RFC-C) making contact with PCNA, leaving a wedge-shaped gap between RFC-E and the PCNA clamp-loader complex. The C-terminal is vital for the correct orientation of RFC-E with respect to RFC-A.</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01146">TIGR01146</a></td><td>286</td><td>34</td><td>25.3</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ATPsyn_F1gamma</td><td>ATP synthase, F1 gamma subunit. This model describes the ATP synthase gamma subunit in bacteria and its equivalents in organelles, namely, mitochondria and chloroplast. F1/F0-ATP synthase is a multisubunit, membrane associated enzyme found in bacteria and organelles of higher eukaryotes, namely, mitochondria and chloroplast. This enzyme is principally involed in the synthesis of ATP from ADP and inorganic phosphate by coupling the energy derived from the proton electrochemical gradient across the biological membrane. A brief description of this multisubunit enzyme complex: F1 and F0 represent two major clusters of subunits. The gamma subunit is the part of F1 cluster. Surrounding the gamma subunit in a cylinder-like structure are three alpha and three subunits in an alternating fashion. This is the central catalytic unit whose different conformations permit the binding of ADP and inorganic phosphate and release of ATP. </td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13374">pfam13374</a></td><td>42</td><td>16</td><td>25.2</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_10</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12752">pfam12752</a></td><td>57</td><td>17</td><td>25.0</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SUZ</td><td>SUZ domain. The SUZ domain is a conserved RNA-binding domain found in eukaryotes and enriched in positively charged amino acids. It was first characterized in the C.elegans protein Szy-20 where it has been shown to bind RNA and allow their localization to the centrosome. Warning- the domain has a compositionally biased character.</td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09537">pfam09537</a></td><td>111</td><td>38</td><td>24.7</td><td>1.9E+02</td><td>[&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2383</td><td>Domain of unknown function (DUF2383). Members of this protein family are found mostly in the Proteobacteria, although one member is found in the the marine planctomycete Pirellula sp. strain 1. The function is unknown.</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08859">pfam08859</a></td><td>110</td><td>29</td><td>24.6</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DGC</td><td>DGC domain. This domain appears to be a zinc binding domain from the conservation of four potential chelating cysteines. The domain is named after a conserved central motif. The function of this domain is unknown.</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06253">PRK06253</a></td><td>529</td><td>10</td><td>23.9</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06253</td><td>O-phosphoseryl-tRNA synthetase; Reviewed</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06957">pfam06957</a></td><td>421</td><td>22</td><td>23.6</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COPI_C</td><td>Coatomer (COPI) alpha subunit C-terminus. This family represents the C-terminus (approximately 500 residues) of the eukaryotic coatomer alpha subunit. Coatomer (COPI) is a large cytosolic protein complex which forms a coat around vesicles budding from the Golgi apparatus. Such coatomer-coated vesicles have been proposed to play a role in many distinct steps of intracellular transport. Note that many family members also contain the pfam04053 domain.</td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10056">pfam10056</a></td><td>86</td><td>24</td><td>23.2</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------]</td><td>DUF2293</td><td>Uncharacterized conserved protein (DUF2293). This domain, found in various hypothetical bacterial proteins, has no known function.</td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00756">TIGR00756</a></td><td>35</td><td>18</td><td>23.0</td><td>83</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PPR</td><td>pentatricopeptide repeat domain (PPR motif). This model describes a domain called the PPR motif, or pentatricopeptide repeat. Its consensus sequence is 35 positions long and typically is found in four or more tandem copies. This family is strongly represented in plant proteins, particularly those sorted to chloroplasts or mitochondria. The pfam01535, domain of unknown function DUF17, consists of 6 copies of this repeat. This family has a similar consensus to the TPR domain (tetratricopeptide), pfam00515, a 33-residue repeat. It is predicted to form a pair of antiparallel helices similar to that of TPR.</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01000">cd01000</a></td><td>228</td><td>30</td><td>22.5</td><td>99</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_Cys_DEBP_like</td><td>Substrate-binding domain of cysteine- and aspartate/glutamate-binding proteins; the type 2 periplasmic-binding protein fold. This family comprises of the periplasmic-binding protein component of ABC transporters specific for cysteine and carboxylic amino acids, as well as their closely related proteins.  The cysteine and aspartate-glutamate binding domains belong to the type 2 periplasmic binding protein fold superfamily (PBP2), whose many  members are involved in chemotaxis and uptake of nutrients and other small molecules from the extracellular space as a primary receptor. The PBP2 proteins are typically comprised of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two receptor cytoplasmically-located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04533">pfam04533</a></td><td>212</td><td>49</td><td>22.4</td><td>1.9E+02</td><td>[&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Herpes_U44</td><td>Herpes virus U44 protein. This is a family of proteins from dsDNA beta-herpesvirinae and gamma-herpesvirinae viruses. The function is not known, and the proteins are named variously as U44, BSRF1, UL71, and M71. The family BSRF1 has been merged into this.</td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14820">cd14820</a></td><td>182</td><td>17</td><td>21.7</td><td>82</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;]</td><td>TRAX</td><td>Translin-associated factor-X (TRAX). TRAX (translin-associated factor-X) is a paralog of its binding partner protein Translin and together they form oligomeric complexes known as C3PO proteins (for component 3 promoter of  RNA-induced silencing complex or RISC). TRAX complexed with Translin is possibly involved in dendritic RNA processing and in DNA double-strand break repair as an interacting partner with C1D, an activator of the DNA-dependent protein kinase involved in the repair of DNA-double strand breaks. It has been shown that Trax subunit, but not Translin, possesses a Glu-Glu-Asp catalytic center with the capacity to digest RNA; this catalytic activity is required for passenger-strand removal and RISC activation in RNAi. In Archaeoglobus fulgidus, Trax-like-subunits assemble into an octameric structure, highly similar to human C3PO; its complex with duplex RNA reveals that the octamer entirely encapsulates a single 13-base-pair RNA duplex inside a large inner cavity. Translin and Trax participate in a variety of nucleic acid metabolism pathways in addition to RNAi and have been implicated in a wide range of biological activities, including mRNA processing, cell growth regulation, spermatogenesis, neuronal development/function, genome stability regulation and carcinogenesis; however, their precise role in some of the processes remains unclear.</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01900">pfam01900</a></td><td>104</td><td>31</td><td>21.4</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNase_P_Rpp14</td><td>Rpp14/Pop5 family. tRNA processing enzyme ribonuclease P (RNase P) consists of an RNA molecule associated with at least eight protein subunits, hPop1, Rpp14, Rpp20, Rpp25, Rpp29, Rpp30, Rpp38, and Rpp40. This protein is known as Pop5 in eukaryotes.</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3275">COG3275</a></td><td>557</td><td>74</td><td>21.0</td><td>3.6E+02</td><td>[&nbsp;&nbsp;&nbsp;---------------------------------------------&nbsp;]</td><td>LytS</td><td>Sensor histidine kinase, LytS/YehU family </td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03158">TIGR03158</a></td><td>357</td><td>38</td><td>20.9</td><td>24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas3_cyano</td><td>CRISPR-associated helicase Cas3, subtype CYANO. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) is a widespread family of prokaryotic direct repeats with spacers of unique sequence between consecutive repeats. This protein family is a CRISPR-associated (Cas) family strictly associated with the Cyano subtype of CRISPR/Cas locus, found in several species of Cyanobacteria and several archaeal species. It contains helicase motifs and appears to represent the Cas3 protein of the Cyano subtype of CRISPR/Cas system.</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00440">PRK00440</a></td><td>319</td><td>17</td><td>20.5</td><td>81</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;]</td><td>rfc</td><td>replication factor C small subunit; Reviewed</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03717">PRK03717</a></td><td>120</td><td>9</td><td>20.4</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK03717</td><td>ribonuclease P protein component 2; Provisional</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5023">COG5023</a></td><td>443</td><td>16</td><td>20.3</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG5023</td><td>Tubulin </td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10155">cd10155</a></td><td>82</td><td>22</td><td>20.2</td><td>81</td><td>[&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BsYrkD-like_DUF156</td><td>Uncharacterized protein YrkD from Bacillus subtilis and related domains; this domain superfamily was previously known as part of DUF156. This domain family contains an uncharacterized protein YrkD from Bacillus subtilis and related proteins. This family is part of a larger superfamily that contains various transcriptional regulators that respond to different stressors such as Cu(I), Ni(I), sulfite, and formaldehyde, and includes CsoRs (copper-sensitive operon repressors). CsoRs form homotetramers (dimer of dimers). In Mycobacterium tuberculosis CsoR, within each dimer, two Cys residues on opposite subunits, along with a His residue, bind the Cu(I) ion (forming a triagonal S2N coordination complex, C-H-C). These residues are conserved in the majority of members of this superfamily. In this family, however, not all these residues are conserved, there is an Asn instead of the His (C-N-C); also a conserved Tyr and a Glu residue that facilitates allosteric regulation of DNA binding for CsoRs are very poorly conserved.</td></tr>
</table>
</body>
