<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12847">pfam12847</a></td><td>106</td><td>42</td><td>97.5</td><td>0.00013</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_18</td><td>Methyltransferase domain. Protein in this family function as methyltransferases.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08242">pfam08242</a></td><td>98</td><td>37</td><td>97.4</td><td>0.00017</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_12</td><td>Methyltransferase domain. Members of this family are SAM dependent methyltransferases.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13847">pfam13847</a></td><td>151</td><td>45</td><td>97.0</td><td>0.0016</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_31</td><td>Methyltransferase domain. This family appears to be have methyltransferase activity.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08317">PRK08317</a></td><td>241</td><td>45</td><td>96.6</td><td>0.0024</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08317</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13649">pfam13649</a></td><td>97</td><td>37</td><td>96.5</td><td>0.0023</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_25</td><td>Methyltransferase domain. This family appears to be a methyltransferase domain.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00121">PRK00121</a></td><td>202</td><td>129</td><td>95.6</td><td>0.011</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>trmB</td><td>tRNA (guanine-N(7)-)-methyltransferase; Reviewed</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02440">cd02440</a></td><td>107</td><td>37</td><td>95.5</td><td>0.015</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AdoMet_MTases</td><td>S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy). There are at least five structurally distinct families of AdoMet-MTases, class I being the largest and most diverse. Within this class enzymes can be classified by different substrate specificities (small molecules, lipids, nucleic acids, etc.) and different target atoms for methylation (nitrogen, oxygen, carbon, sulfur, etc.).</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13489">pfam13489</a></td><td>157</td><td>48</td><td>93.9</td><td>0.085</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_23</td><td>Methyltransferase domain. This family appears to be a methyltransferase domain.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03534">TIGR03534</a></td><td>250</td><td>52</td><td>93.8</td><td>0.09</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RF_mod_PrmC</td><td>protein-(glutamine-N5) methyltransferase, release factor-specific. Members of this protein family are HemK (PrmC), a protein once thought to be involved in heme biosynthesis but now recognized to be a protein-glutamine methyltransferase that modifies the peptide chain release factors. All members of the seed alignment are encoded next to the release factor 1 gene (prfA) and confirmed by phylogenetic analysis. SIMBAL analysis (manuscript in prep.) shows the motif </td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13659">pfam13659</a></td><td>117</td><td>107</td><td>93.6</td><td>0.38</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_26</td><td>Methyltransferase domain. This family contains methyltransferase domains.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08241">pfam08241</a></td><td>95</td><td>36</td><td>93.4</td><td>0.06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_11</td><td>Methyltransferase domain. Members of this family are SAM dependent methyltransferases.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2226">COG2226</a></td><td>238</td><td>42</td><td>93.0</td><td>0.13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UbiE</td><td>Ubiquinone/menaquinone biosynthesis C-methylase UbiE </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00091">TIGR00091</a></td><td>194</td><td>138</td><td>92.3</td><td>0.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00091</td><td>tRNA (guanine-N(7)-)-methyltransferase. This predicted S-adenosylmethionine-dependent methyltransferase is found in a single copy in most Bacteria. It is also found, with a short amino-terminal extension in eukaryotes. Its function is unknown. In E. coli, this protein flanks the DNA repair protein MutY, also called micA. </td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02072">TIGR02072</a></td><td>240</td><td>43</td><td>92.2</td><td>0.27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BioC</td><td>malonyl-acyl carrier protein O-methyltransferase BioC. This enzyme, which is found in biotin biosynthetic gene clusters in proteobacteria, firmicutes, green-sulfur bacteria, fusobacterium and bacteroides, carries out an enzymatic step prior to the formation of pimeloyl-CoA, namely O-methylation of the malonyl group preferentially while on acyl carrier protein. The enzyme is recognizable as a methyltransferase by homology. </td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04345">TIGR04345</a></td><td>242</td><td>47</td><td>91.4</td><td>0.16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ovoA_Cterm</td><td>putative 4-mercaptohistidine N1-methyltranferase. Ovothiol A is N1-methyl-4-mercaptohistidine. In the absence of S-adenosylmethione, a methyl donor, the intermediate produced is 4-mercaptohistidine. In both Erwinia tasmaniensis and Trypanosoma cruzi, a protein occurs with 5-histidylcysteine sulfoxide synthase activity, but these two enzymes and most homologs share an additional C-terminal methyltransferase domain. Thus OvoA may be a bifunctional enzyme with 5-histidylcysteine sulfoxide synthase and 4-mercaptohistidine N1-methyltranferase activity. This model describes C-terminal putative 4-mercaptohistidine N1-methyltranferase domain. </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01683">PRK01683</a></td><td>258</td><td>51</td><td>91.0</td><td>0.22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01683</td><td>trans-aconitate 2-methyltransferase; Provisional</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0220">COG0220</a></td><td>227</td><td>64</td><td>89.8</td><td>0.25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TrmB</td><td>tRNA G46 methylase TrmB </td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0500">COG0500</a></td><td>257</td><td>36</td><td>89.7</td><td>0.54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SmtA</td><td>SAM-dependent methyltransferase </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02390">pfam02390</a></td><td>199</td><td>125</td><td>89.0</td><td>0.29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_4</td><td>Putative methyltransferase. This is a family of putative methyltransferases. The aligned region contains the GXGXG S-AdoMet binding site suggesting a putative methyltransferase activity.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02081">TIGR02081</a></td><td>194</td><td>89</td><td>88.5</td><td>0.38</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>metW</td><td>methionine biosynthesis protein MetW. This protein is found alongside MetX, of the enzyme that acylates homoserine as a first step toward methionine biosynthesis, in many species. It appears to act in methionine biosynthesis but is not fully characterized. </td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13679">pfam13679</a></td><td>143</td><td>44</td><td>88.2</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_32</td><td>Methyltransferase domain. This family appears to be a methyltransferase domain.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4106">COG4106</a></td><td>257</td><td>53</td><td>87.3</td><td>0.63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Tam</td><td>Trans-aconitate methyltransferase </td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07021">pfam07021</a></td><td>193</td><td>42</td><td>86.7</td><td>0.68</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MetW</td><td>Methionine biosynthesis protein MetW. This family consists of several bacterial and one archaeal methionine biosynthesis MetW proteins. Biosynthesis of methionine from homoserine in Pseudomonas putida takes place in three steps. The first step is the acylation of homoserine to yield an acyl-L-homoserine. This reaction is catalysed by the products of the metXW genes and is equivalent to the first step in enterobacteria, gram-positive bacteria and fungi, except that in these microorganisms the reaction is catalysed by a single polypeptide (the product of the metA gene in Escherichia coli and the met5 gene product in Neurospora crassa). In Pseudomonas putida, as in gram-positive bacteria and certain fungi, the second and third steps are a direct sulfhydrylation that converts the O-acyl-L-homoserine into homocysteine and further methylation to yield methionine. The latter reaction can be mediated by either of the two methionine synthetases present in the cells.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2230">COG2230</a></td><td>283</td><td>53</td><td>85.2</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cfa</td><td>Cyclopropane fatty-acyl-phospholipid synthase and related methyltransferases </td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4976">COG4976</a></td><td>287</td><td>39</td><td>84.6</td><td>1.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4976</td><td>Predicted methyltransferase, contains TPR repeat </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00216">PRK00216</a></td><td>239</td><td>44</td><td>82.6</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ubiE</td><td>ubiquinone/menaquinone biosynthesis methyltransferase; Reviewed</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2890">COG2890</a></td><td>280</td><td>62</td><td>82.5</td><td>2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HemK</td><td>Methylase of polypeptide chain release factors </td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00517">PRK00517</a></td><td>250</td><td>47</td><td>81.4</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>prmA</td><td>ribosomal protein L11 methyltransferase; Reviewed</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14103">PRK14103</a></td><td>255</td><td>91</td><td>81.2</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14103</td><td>trans-aconitate 2-methyltransferase; Provisional</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11312">pfam11312</a></td><td>287</td><td>147</td><td>80.3</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3115</td><td>Protein of unknown function (DUF3115). This eukaryotic family of proteins has no known function.</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07091">pfam07091</a></td><td>251</td><td>76</td><td>78.1</td><td>1.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FmrO</td><td>Ribosomal RNA methyltransferase (FmrO). This family consists of several bacterial ribosomal RNA methyltransferase (aminoglycoside-resistance methyltransferase) proteins.</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2242">COG2242</a></td><td>187</td><td>53</td><td>76.8</td><td>7.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CobL</td><td>Precorrin-6B methylase 2  </td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09328">PRK09328</a></td><td>275</td><td>53</td><td>76.0</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09328</td><td>N5-glutamine S-adenosyl-L-methionine-dependent methyltransferase; Provisional</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02469">TIGR02469</a></td><td>124</td><td>49</td><td>74.5</td><td>3.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CbiT</td><td>precorrin-6Y C5,15-methyltransferase (decarboxylating), CbiT subunit. This model recognizes the CbiT methylase which is responsible, in part (along with CbiE), for methylating precorrin-6y (or cobalt-precorrin-6y) at both the 5 and 15 positions as well as the concomitant decarbozylation at C-12. In many organisms, this protein is fused to the CbiE subunit. The fused protein, when found in organisms catalyzing the oxidative version of the cobalamin biosynthesis pathway, is called CobL. </td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02021">TIGR02021</a></td><td>219</td><td>70</td><td>74.2</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BchM-ChlM</td><td>magnesium protoporphyrin O-methyltransferase. This model represents the S-adenosylmethionine-dependent O-methyltransferase responsible for methylation of magnesium protoporphyrin IX. This step is essentiasl for the biosynthesis of both chlorophyll and bacteriochlorophyll. This model encompasses two closely related clades, from cyanobacteria (and plants) where it is called ChlM and other photosynthetic bacteria where it is known as BchM. </td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4123">COG4123</a></td><td>248</td><td>137</td><td>73.0</td><td>5.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TrmN6</td><td>tRNA1(Val) A37 N6-methylase TrmN6 </td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02636">pfam02636</a></td><td>240</td><td>59</td><td>72.4</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_28</td><td>Putative S-adenosyl-L-methionine-dependent methyltransferase. This family is a putative S-adenosyl-L-methionine (SAM)-dependent methyltransferase. In eukaryotes it plays a role in mitochondrial complex I activity.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00377">PRK00377</a></td><td>198</td><td>23</td><td>68.4</td><td>3.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cbiT</td><td>cobalt-precorrin-6Y C(15)-methyltransferase; Provisional</td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02353">pfam02353</a></td><td>273</td><td>159</td><td>67.6</td><td>3.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CMAS</td><td>Mycolic acid cyclopropane synthetase. This family consist of Cyclopropane-fatty-acyl-phospholipid synthase or CFA synthase EC:2.1.1.79 this enzyme catalyse the reaction: S-adenosyl-L-methionine + phospholipid olefinic fatty acid <=> S-adenosyl-L-homocysteine + phospholipid cyclopropane fatty acid.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03439">TIGR03439</a></td><td>319</td><td>67</td><td>67.3</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>methyl_EasF</td><td>probable methyltransferase domain, EasF family. This model represents an uncharacterized domain of about 300 amino acids with homology to S-adenosylmethionine-dependent methyltransferases. Proteins with this domain are exclusively fungal. A few, such as EasF from Neotyphodium lolii, are associated with the biosynthesis of ergot alkaloids, a class of fungal secondary metabolites. EasF may, in fact, be the AdoMet:dimethylallyltryptophan N-methyltransferase, the enzyme that follows tryptophan dimethylallyltransferase (DMATS) in ergot alkaloid biosynthesis. Several other members of this family, including mug158 (meiotically up-regulated gene 158 protein) from Schizosaccharomyces pombe, contain an additional uncharacterized domain DUF323 (pfam03781).</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01135">pfam01135</a></td><td>205</td><td>54</td><td>67.1</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PCMT</td><td>Protein-L-isoaspartate(D-aspartate) O-methyltransferase (PCMT). </td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2264">COG2264</a></td><td>300</td><td>29</td><td>67.0</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PrmA</td><td>Ribosomal protein L11 methylase PrmA </td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01983">TIGR01983</a></td><td>224</td><td>114</td><td>66.2</td><td>5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UbiG</td><td>ubiquinone biosynthesis O-methyltransferase. This model represents an O-methyltransferase believed to act at two points in the ubiquinone biosynthetic pathway in bacteria (UbiG) and fungi (COQ3). A separate methylase (MenG/UbiE) catalyzes the single C-methylation step. The most commonly used names for genes in this family do not indicate whether this gene is an O-methyl, or C-methyl transferase. </td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2227">COG2227</a></td><td>243</td><td>108</td><td>66.1</td><td>3.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UbiG</td><td>2-polyprenyl-3-methyl-5-hydroxy-6-metoxy-1,4-benzoquinol methylase </td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04371">TIGR04371</a></td><td>273</td><td>75</td><td>65.3</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>methyltran_NanM</td><td>putative sugar O-methyltransferase. Members of this family appear to be SAM-dependent O-methyltransferases acting on sugars, based on iterated sequence searches and gene context. Members occur in Leptospira O-antigen regions, as well NanM from the biosynthesis cluster for nanchangmycin, which produces 4-O-methyl-L-rhodinose as an intermediate. </td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01934">TIGR01934</a></td><td>223</td><td>44</td><td>64.4</td><td>6.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MenG_MenH_UbiE</td><td>ubiquinone/menaquinone biosynthesis methyltransferases. This model represents a family of methyltransferases involved in the biosynthesis of menaquinone and ubiqinone. Some members such as the UbiE enzyme from E. coli are believed to act in both pathways, while others may act in only the menaquinone pathway. These methyltransferases are members of the UbiE/CoQ family of methyltransferases (pfam01209) which also contains ubiquinone methyltransferases and other methyltransferases. Members of this clade include a wide distribution of bacteria and eukaryotes, but no archaea. An outgroup for this clade is provided by the phosphatidylethanolamine methyltransferase (EC 2.1.1.17) from Rhodobacter sphaeroides. Note that a number of non-orthologous genes which are members of pfam03737 have been erroneously annotated as MenG methyltransferases. </td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04543">TIGR04543</a></td><td>331</td><td>65</td><td>64.4</td><td>6.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ketoArg_3Met</td><td>2-ketoarginine methyltransferase. This SAM-dependent C-methyltransferase performs the middle step of a three step conversion from arginine to beta-methylarginine. It performs a C-methylation at position 3 of 5-guanidino-2-oxopentanoic acid (keto-arginine). An aminotransferase converts arginine to 5-guanidino-2-oxopentanoic acid, and later converts 5-guanidino-3-methyl-2-oxopentanoic acid to beta-methylarginine.</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11088">PRK11088</a></td><td>272</td><td>82</td><td>62.6</td><td>9.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rrmA</td><td>23S rRNA methyltransferase A; Provisional</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11705">PRK11705</a></td><td>383</td><td>62</td><td>62.1</td><td>4.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11705</td><td>cyclopropane fatty acyl phospholipid synthase; Provisional</td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01358">pfam01358</a></td><td>295</td><td>65</td><td>59.8</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PARP_regulatory</td><td>Poly A polymerase regulatory subunit. </td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00536">TIGR00536</a></td><td>284</td><td>61</td><td>59.6</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>hemK_fam</td><td>HemK family putative methylases. The gene hemK from E. coli was found to contribute to heme biosynthesis and originally suggested to be protoporphyrinogen oxidase. Functional analysis of the nearest homolog in Saccharomyces cerevisiae, YNL063w, finds it is not protoporphyrinogen oxidase and sequence analysis suggests that HemK homologs have S-adenosyl-methionine-dependent methyltransferase activity (Medline 99237242). Homologs are found, usually in a single copy, in nearly all completed genomes, but varying somewhat in apparent domain architecture. Both E. coli and H. influenzae have two members rather than one. The members from the Mycoplasmas have an additional C-terminal domain. </td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09243">pfam09243</a></td><td>275</td><td>111</td><td>58.8</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rsm22</td><td>Mitochondrial small ribosomal subunit Rsm22. Rsm22 has been identified as a mitochondrial small ribosomal subunit and is a methyltransferase. In Schizosaccharomyces pombe, Rsm22 is tandemly fused to Cox11 (a factor required for copper insertion into cytochrome oxidase) and the two proteins are proteolytically cleaved after import into the mitochondria.</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2263">COG2263</a></td><td>198</td><td>41</td><td>57.8</td><td>7.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG2263</td><td>Predicted RNA methylase  </td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05134">PRK05134</a></td><td>233</td><td>86</td><td>57.5</td><td>5.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05134</td><td>bifunctional 3-demethylubiquinone-9 3-methyltransferase/ 2-octaprenyl-6-hydroxy phenol methylase; Provisional</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05175">pfam05175</a></td><td>170</td><td>37</td><td>54.9</td><td>9.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MTS</td><td>Methyltransferase small domain. This domain is found in ribosomal RNA small subunit methyltransferase C as well as other methyltransferases.</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06202">PRK06202</a></td><td>232</td><td>57</td><td>54.4</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06202</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5459">COG5459</a></td><td>484</td><td>75</td><td>54.4</td><td>9.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rsm22</td><td>Ribosomal protein RSM22 (predicted mitochondrial rRNA methylase) </td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2813">COG2813</a></td><td>300</td><td>38</td><td>54.0</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RsmC</td><td>16S rRNA G1207 methylase RsmC </td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07580">PRK07580</a></td><td>230</td><td>57</td><td>53.2</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07580</td><td>Mg-protoporphyrin IX methyl transferase; Validated</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02966">cd02966</a></td><td>116</td><td>57</td><td>53.1</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TlpA_like_family</td><td>TlpA-like family; composed of  TlpA, ResA, DsbE and similar proteins. TlpA, ResA and DsbE are bacterial protein disulfide reductases with important roles in cytochrome maturation. They are membrane-anchored proteins with a soluble TRX domain containing a CXXC motif located in the periplasm. The TRX domains of this family contain an insert, approximately 25 residues in length, which correspond to an extra alpha helix and a beta strand when compared with TRX. TlpA catalyzes an essential reaction in the biogenesis of cytochrome aa3, while ResA and DsbE are essential proteins in cytochrome c maturation. Also included in this family are proteins containing a TlpA-like TRX domain with domain architectures similar to E. coli DipZ protein, and the N-terminal TRX domain of PilB protein from Neisseria which acts as a disulfide reductase that can recylce methionine sulfoxide reductases.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01728">pfam01728</a></td><td>177</td><td>15</td><td>52.8</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FtsJ</td><td>FtsJ-like methyltransferase. This family consists of FtsJ from various bacterial and archaeal sources FtsJ is a methyltransferase, but actually has no effect on cell division. FtsJ's substrate is the 23S rRNA. The 1.5 A crystal structure of FtsJ in complex with its cofactor S-adenosylmethionine revealed that FtsJ has a methyltransferase fold. This family also includes the N terminus of flaviviral NS5 protein. It has been hypothesised that the N-terminal domain of NS5 is a methyltransferase involved in viral RNA capping.</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06325">pfam06325</a></td><td>294</td><td>25</td><td>50.8</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PrmA</td><td>Ribosomal protein L11 methyltransferase (PrmA). This family consists of several Ribosomal protein L11 methyltransferase (EC:2.1.1.-) sequences.</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03291">pfam03291</a></td><td>327</td><td>178</td><td>49.5</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------&nbsp;]</td><td>Pox_MCEL</td><td>mRNA capping enzyme. This family of enzymes are related to pfam03919.</td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12558">cd12558</a></td><td>76</td><td>33</td><td>48.8</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM3_RBM15B</td><td>RNA recognition motif 3 in putative RNA-binding protein 15B (RBM15B) from vertebrate. This subgroup corresponds to the RRM3 of RBM15B, also termed one twenty-two 3 (OTT3), a paralog of RNA binding motif protein 15 (RBM15), also known as One-twenty two protein 1 (OTT1). Like RBM15, RBM15B has post-transcriptional regulatory activity. It is a nuclear protein sharing with RBM15 the association with the splicing factor compartment and the nuclear envelope as well as the binding to mRNA export factors NXF1 and Aly/REF. RBM15B belongs to the Spen (split end) protein family, which shares a domain architecture comprising of three N-terminal RNA recognition motifs (RRMs), also known as RBD (RNA binding domain) or RNP (ribonucleoprotein domain), and a C-terminal SPOC (Spen paralog and ortholog C-terminal) domain. </td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08303">pfam08303</a></td><td>168</td><td>75</td><td>48.6</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tRNA_lig_kinase</td><td>tRNA ligase kinase domain. This domain is found in fungal tRNA ligases and has kinase activity. tRNA ligases are enzymes required for the splicing of precursor tRNA molecules containing introns. This family contains a P-loop motif.</td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02968">cd02968</a></td><td>142</td><td>68</td><td>44.9</td><td>80</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SCO</td><td>SCO (an acronym for Synthesis of Cytochrome c Oxidase) family; composed of proteins similar to Sco1, a membrane-anchored protein possessing a soluble domain with a TRX fold. Members of this family are required for the proper assembly of cytochrome c oxidase (COX). They contain a metal binding motif, typically CXXXC, which is located in a flexible loop. COX, the terminal enzyme in the respiratory chain, is imbedded in the inner mitochondrial membrane of all eukaryotes and in the plasma membrane of some prokaryotes. It is composed of two subunits, COX I and COX II. It has been proposed that Sco1 specifically delivers copper to the CuA site, a dinuclear copper center, of the COX II subunit. Mutations in human Sco1 and Sco2 cause fatal infantile hepatoencephalomyopathy and cardioencephalomyopathy, respectively. Both disorders are associated with severe COX deficiency in affected tissues. More recently, it has been argued that the redox sensitivity of the copper binding properties of Sco1 implies that it participates in signaling events rather than functioning as a chaperone that transfers copper to COX II.</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04652">pfam04652</a></td><td>312</td><td>37</td><td>43.4</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF605</td><td>Vta1 like. Vta1 (VPS20-associated protein 1) is a positive regulator of Vps4. Vps4 is an ATPase that is required in the multivesicular body (MVB) sorting pathway to dissociate the endosomal sorting complex required for transport (ESCRT). Vta1 promotes correct assembly of Vps4 and stimulates its ATPase activity through its conserved Vta1/SBP1/LIP5 region.</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07199">cd07199</a></td><td>258</td><td>71</td><td>43.3</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pat17_PNPLA8_PNPLA9_like</td><td>Patatin-like phospholipase; includes PNPLA8, PNPLA9, and Pat17. Patatin is a storage protein of the potato tuber that shows Phospholipase A2 activity (PLA2; EC 3.1.1.4). Patatin catalyzes the nonspecific hydrolysis of phospholipids, glycolipids, sulfolipids, and mono- and diacylglycerols, thereby showing lipid acyl hydrolase activity. The active site includes an oxyanion hole with a conserved GGxR motif; it is found in almost all the members of this family. The catalytic dyad is formed by a serine and an aspartate. Patatin belongs to the alpha-beta hydrolase family which is identified by a characteristic nucleophile elbow with a consensus sequence of Sm-X-Nu-Sm (Sm = small residue, X = any residue and Nu = nucleophile). Members of this family have been found also in vertebrates. This family includes subfamily of PNPLA8 (iPLA2-gamma) and PNPLA9 (iPLA2-beta) like phospholipases from human as well as the Pat17 isozyme from Solanum cardiophyllum.</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK15068">PRK15068</a></td><td>322</td><td>37</td><td>41.7</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK15068</td><td>tRNA mo(5)U34 methyltransferase; Provisional</td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08003">pfam08003</a></td><td>315</td><td>89</td><td>41.4</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_9</td><td>Protein of unknown function (DUF1698). This family contains many hypothetical proteins. It also includes two putative methyltransferase proteins.</td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00452">TIGR00452</a></td><td>316</td><td>36</td><td>40.9</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00452</td><td>tRNA (mo5U34)-methyltransferase. This model describes CmoB, the enzyme tRNA (mo5U34)-methyltransferase involved in tRNA wobble base modification. </td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00406">TIGR00406</a></td><td>288</td><td>45</td><td>39.9</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>prmA</td><td>ribosomal protein L11 methyltransferase. Ribosomal protein L11 methyltransferase is an S-adenosyl-L-methionine-dependent methyltransferase required for the modification of ribosomal protein L11. This protein is found in bacteria and (with a probable transit peptide) in Arabidopsis. </td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13905">pfam13905</a></td><td>94</td><td>23</td><td>39.8</td><td>61</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Thioredoxin_8</td><td>Thioredoxin-like. Thioredoxins are small enzymes that participate in redox reactions, via the reversible oxidation of an active centre disulfide bond.</td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13480">pfam13480</a></td><td>144</td><td>58</td><td>39.4</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Acetyltransf_6</td><td>Acetyltransferase (GNAT) domain. This family contains proteins with N-acetyltransferase functions.</td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01234">pfam01234</a></td><td>261</td><td>36</td><td>37.0</td><td>60</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NNMT_PNMT_TEMT</td><td>NNMT/PNMT/TEMT family. </td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01209">pfam01209</a></td><td>233</td><td>77</td><td>36.9</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ubie_methyltran</td><td>ubiE/COQ5 methyltransferase family. </td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12335">PRK12335</a></td><td>287</td><td>39</td><td>36.4</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK12335</td><td>tellurite resistance protein TehB; Provisional</td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02875">pfam02875</a></td><td>87</td><td>29</td><td>35.7</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Mur_ligase_C</td><td>Mur ligase family, glutamate ligase domain. This family contains a number of related ligase enzymes which have EC numbers 6.3.2.*. This family includes: MurC, MurD, MurE, MurF, Mpl and FolC. MurC, MurD, Mure and MurF catalyse consecutive steps in the synthesis of peptidoglycan. Peptidoglycan consists of a sheet of two sugar derivatives, with one of these N-acetylmuramic acid attaching to a small pentapeptide. The pentapeptide is is made of L-alanine, D-glutamic acid, Meso-diaminopimelic acid and D-alanyl alanine. The peptide moiety is synthesized by successively adding these amino acids to UDP-N-acetylmuramic acid. MurC transfers the L-alanine, MurD transfers the D-glutamate, MurE transfers the diaminopimelic acid, and MurF transfers the D-alanyl alanine. This family also includes Folylpolyglutamate synthase that transfers glutamate to folylpolyglutamate.</td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14968">PRK14968</a></td><td>188</td><td>47</td><td>34.7</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14968</td><td>putative methyltransferase; Provisional</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5269">COG5269</a></td><td>379</td><td>54</td><td>34.5</td><td>92</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ZUO1</td><td>Ribosome-associated chaperone zuotin </td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02055">TIGR02055</a></td><td>191</td><td>22</td><td>34.2</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>APS_reductase</td><td>thioredoxin-dependent adenylylsulfate APS reductase. This model describes recently identified adenosine 5'-phosphosulfate (APS) reductase activity found in sulfate-assimilatory prokaryotes, thus separating it from the traditionally described phosphoadenosine 5'-phosphosulfate (PAPS) reductases found in bacteria and fungi. Homologous to PAPS reductase in enterobacteria, cyanobacteria, and yeast, APS reductase here clusters with, and demonstrates greater homology to plant APS reductase. Additionally, the presence of two conserved C-terminal motifs (CCXXRKXXPL & SXGCXXCT) distinguishes APS substrate specificity and serves as a FeS cluster. </td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01555">pfam01555</a></td><td>219</td><td>45</td><td>32.7</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>N6_N4_Mtase</td><td>DNA methylase. Members of this family are DNA methylases. The family contains both N-4 cytosine-specific DNA methylases and N-6 Adenine-specific DNA methylases.</td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05327">cd05327</a></td><td>269</td><td>49</td><td>32.6</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>retinol-DH_like_SDR_c_like</td><td>retinol dehydrogenase (retinol-DH), Light dependent Protochlorophyllide (Pchlide) OxidoReductase (LPOR) and related proteins, classical (c) SDRs. Classical SDR subgroup containing retinol-DHs, LPORs, and related proteins. Retinol is processed by a medium chain alcohol dehydrogenase followed by retinol-DHs. Pchlide reductases act in chlorophyll biosynthesis. There are distinct enzymes that catalyze Pchlide reduction in light or dark conditions. Light-dependent reduction is via an NADP-dependent SDR, LPOR. Proteins in this subfamily share the glycine-rich NAD-binding motif of the classical SDRs, have a partial match to the canonical active site tetrad, but lack the typical active site Ser. This subgroup includes the human proteins: retinol dehydrogenase -12, -13 ,and -14, dehydrogenase/reductase SDR family member (DHRS)-12 , -13 and -X (a DHRS on chromosome X), and WWOX (WW domain-containing oxidoreductase), as well as a Neurospora crassa SDR encoded by the blue light inducible bli-4 gene. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRs are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes catalyze a wide range of activities including the metabolism of steroids, cofactors, carbohydrates, lipids, aromatic compounds, and amino acids, and act in redox sensing. Classical SDRs have an TGXXX</td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00107">PRK00107</a></td><td>187</td><td>38</td><td>32.1</td><td>93</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gidB</td><td>16S rRNA methyltransferase GidB; Reviewed</td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11523">cd11523</a></td><td>72</td><td>40</td><td>30.1</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NTP-PPase</td><td>Nucleoside Triphosphate Pyrophosphohydrolase (EC 3.6.1.8) MazG-like domain superfamily. This superfamily contains enzymes that hydrolyze the alpha-beta phosphodiester bond of all canonical NTPs into monophosphate derivatives and pyrophosphate (PPi). Divalent ions, such as Mg2+ ion(s), are essential to activate a proposed water nucleophile and stabilize the charged intermediates to facilitate catalysis. These enzymes share a conserved divalent ion-binding motif EXX</td></tr>
<tr><td>86</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00312">PRK00312</a></td><td>212</td><td>43</td><td>30.0</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pcm</td><td>protein-L-isoaspartate O-methyltransferase; Reviewed</td></tr>
<tr><td>87</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01795">pfam01795</a></td><td>310</td><td>93</td><td>29.3</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_5</td><td>MraW methylase family. Members of this family are probably SAM dependent methyltransferases based on Escherichia coli rsmH. This family appears to be related to pfam01596.</td></tr>
<tr><td>88</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01208">pfam01208</a></td><td>337</td><td>37</td><td>28.5</td><td>2.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>URO-D</td><td>Uroporphyrinogen decarboxylase (URO-D). </td></tr>
<tr><td>89</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00438">TIGR00438</a></td><td>188</td><td>27</td><td>28.1</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rrmJ</td><td>cell division protein FtsJ. Methylates the 23S rRNA. Previously known as cell division protein ftsJ.// Trusted cutoff too high? </td></tr>
<tr><td>90</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10507">cd10507</a></td><td>47</td><td>16</td><td>27.9</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Zn-ribbon_RPA12</td><td>C-terminal zinc ribbon domain of RPA12 subunit of RNA polymerase I. The C-terminal zinc ribbon domain (C_ribbon) of subunit A12 (C-ribbon_RPA12) in RNA polymerase (Pol) I is involved in intrinsic transcript cleavage. Eukaryote genomes are transcribed by three nuclear RNA polymerases (Pol I, II and III) that share some subunits. RPA12 in Pol I, RPB9 in Pol II, RPC11 in Pol III and TFS in archaea are distantly related to each other and to the TFIIS elongation factor of Pol II. RPA12 has two zinc-binding domains separated by a flexible linker.</td></tr>
<tr><td>91</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0357">COG0357</a></td><td>215</td><td>42</td><td>27.1</td><td>90</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RsmG</td><td>16S rRNA G527 N7-methylase RsmG (former glucose-inhibited division protein B) </td></tr>
<tr><td>92</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01739">pfam01739</a></td><td>194</td><td>54</td><td>26.6</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CheR</td><td>CheR methyltransferase, SAM binding domain. CheR proteins are part of the chemotaxis signaling mechanism in bacteria. CheR methylates the chemotaxis receptor at specific glutamate residues. CheR is an S-adenosylmethionine- dependent methyltransferase - the C-terminal domain (this one) binds SAM.</td></tr>
<tr><td>93</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05785">PRK05785</a></td><td>226</td><td>146</td><td>25.9</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05785</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>94</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10258">PRK10258</a></td><td>251</td><td>87</td><td>25.9</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10258</td><td>biotin biosynthesis protein BioC; Provisional</td></tr>
<tr><td>95</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00424">TIGR00424</a></td><td>463</td><td>14</td><td>25.8</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>APS_reduc</td><td>5'-adenylylsulfate reductase, thioredoxin-independent. This enzyme, involved in the assimilation of inorganic sulfate, is closely related to the thioredoxin-dependent PAPS reductase of Bacteria (CysH) and Saccharomyces cerevisiae. However, it has its own C-terminal thioredoxin-like domain and is not thioredoxin-dependent. Also, it has a substrate preference for 5'-adenylylsulfate (APS) over 3'-phosphoadenylylsulfate (PAPS) so the pathway does not require an APS kinase (CysC) to convert APS to PAPS. Arabidopsis thaliana appears to have three isozymes, all able to complement E. coli CysH mutants (even in backgrounds lacking thioredoxin or APS kinase) but likely localized to different compartments in Arabidopsis. </td></tr>
<tr><td>96</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2265">COG2265</a></td><td>432</td><td>76</td><td>25.7</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TrmA</td><td>tRNA/tmRNA/rRNA uracil-C5-methylase, TrmA/RlmC/RlmD family </td></tr>
<tr><td>97</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07361">cd07361</a></td><td>266</td><td>74</td><td>25.7</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MEMO_like</td><td>Memo (mediator of ErbB2-driven cell motility) is co-precipitated with the C terminus of ErbB2, a protein involved in cell motility. This subfamily is composed of Memo (mediator of ErbB2-driven cell motility) and similar proteins. Memo is a protein that is co-precipitated with the C terminus of ErbB2, a protein involved in cell motility. It is required for the ErbB2-driven cell mobility and is found in protein complexes with cofilin, ErbB2 and PLCgamma1. However, Memo is not homologous to any known signaling proteins, and its function in ErbB2 signaling is not known. Structural studies show that Memo binds directly to a specific ErbB2-derived phosphopeptide. Memo is homologous to class III nonheme iron-dependent extradiol dioxygenases, however, no metal binding or enzymatic activity can be detected for Memo. This subfamily also contains a few members containing a C-terminal AMMECR1-like domain. The AMMECR1 protein was proposed to be a regulatory factor that is potentially involved in the development of AMME contiguous gene deletion syndrome.</td></tr>
<tr><td>98</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14896">PRK14896</a></td><td>258</td><td>89</td><td>25.7</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ksgA</td><td>16S ribosomal RNA methyltransferase KsgA/Dim1 family protein; Provisional</td></tr>
<tr><td>99</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02310">pfam02310</a></td><td>120</td><td>46</td><td>25.1</td><td>24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>B12-binding</td><td>B12 binding domain. This domain binds to B12 (adenosylcobamide), it is found in several enzymes, such as glutamate mutase, methionine synthase and methylmalonyl-CoA mutase. It contains a conserved DxHxxGx(41)SxVx(26)GG motif, which is important for B12 binding.</td></tr>
<tr><td>100</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0286">COG0286</a></td><td>489</td><td>41</td><td>25.0</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HsdM</td><td>Type I restriction-modification system, DNA methylase subunit </td></tr>
<tr><td>101</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00006">TIGR00006</a></td><td>307</td><td>62</td><td>24.9</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00006</td><td>16S rRNA (cytosine(1402)-N(4))-methyltransferase. This model describes RsmH, a 16S rRNA methyltransferase. Previously, this gene was designated MraW, known to be essential in E. coli and widely conserved in bacteria. </td></tr>
<tr><td>102</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02090">PRK02090</a></td><td>241</td><td>23</td><td>24.8</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK02090</td><td>phosphoadenosine phosphosulfate reductase; Provisional</td></tr>
<tr><td>103</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1352">COG1352</a></td><td>268</td><td>48</td><td>24.5</td><td>3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CheR</td><td>Methylase of chemotaxis methyl-accepting proteins </td></tr>
<tr><td>104</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05148">pfam05148</a></td><td>214</td><td>127</td><td>23.5</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_8</td><td>Hypothetical methyltransferase. This family consists of several uncharacterized eukaryotic proteins which are related to methyltransferases pfam01209.</td></tr>
<tr><td>105</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12747">PRK12747</a></td><td>252</td><td>25</td><td>21.2</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK12747</td><td>short chain dehydrogenase; Provisional</td></tr>
<tr><td>106</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09906">cd09906</a></td><td>105</td><td>81</td><td>21.1</td><td>88</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>H3TH_YEN1</td><td>H3TH domain of Yeast Endonuclease 1, a structure-specific, divalent-metal-ion dependent, 5' nuclease. Yeast Endonuclease 1 (YEN1): Holliday junction resolvase which promotes reciprocal exchange during mitotic recombination to maintain genome integrity in budding yeast. YEN1 is a member of the structure-specific, 5' nuclease family that catalyzes hydrolysis of DNA duplex-containing nucleic acid structures during DNA replication, repair, and recombination. Members of this subgroup include the H3TH (helix-3-turn-helix) domains of YEN1 and other similar fungal 5' nucleases. These nucleases contain a PIN (PilT N terminus) domain with a helical arch/clamp region/I domain (not included here) and inserted within the PIN domain is an atypical helix-hairpin-helix-2 (HhH2)-like region. This atypical HhH2 region, the H3TH domain, has an extended loop with at least three turns between the first two helices, and only three of the four helices appear to be conserved. Both the H3TH domain and the helical arch/clamp region are involved in DNA binding. Studies suggest that a glycine-rich loop in the H3TH domain contacts the phosphate backbone of the template strand in the downstream DNA duplex. These nucleases have a carboxylate rich active site that is involved in binding essential divalent metal ion cofactors (Mg2+ or Mn2+) required for nuclease activity. The first metal binding site is composed entirely of Asp/Glu residues from the PIN domain, whereas, the second metal binding site is composed generally of two Asp residues from the PIN domain and one Asp residue from the H3TH domain. Together with the helical arch and network of amino acids interacting with metal binding ions, the H3TH region defines a positively charged active-site DNA-binding groove in structure-specific 5' nucleases.</td></tr>
<tr><td>107</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11727">PRK11727</a></td><td>321</td><td>27</td><td>20.7</td><td>61</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11727</td><td>23S rRNA mA1618 methyltransferase; Provisional</td></tr>
<tr><td>108</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04074">TIGR04074</a></td><td>462</td><td>42</td><td>20.2</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>bacter_Hen1</td><td>3' terminal RNA ribose 2'-O-methyltransferase Hen1. Members of this protein family are bacterial Hen1, a 3' terminal RNA ribose 2'-O-methyltransferase that acts in bacterial RNA repair. All members of the seed alignment belong to a cassette with the RNA repair enzyme polynucleotide kinase-phosphatase (Pnkp). Chemically similar Hen1 in eukaryotes acts instead on small regulatory RNAs. </td></tr>
<tr><td>109</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00138">TIGR00138</a></td><td>181</td><td>90</td><td>20.2</td><td>74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rsmG_gidB</td><td>16S rRNA (guanine(527)-N(7))-methyltransferase RsmG. RsmG was previously called GidB (glucose-inhibited division protein B). It is present and a single copy in nearly all complete eubacterial genomes. It is missing only from some obligate intracellular species of various lineages (Chlamydiae, Ehrlichia, Wolbachia, Anaplasma, Buchnera, etc.). RsmG shows a methytransferase fold in its the crystal structure, and acts as a 7-methylguanosine (m(7)G) methyltransferase, apparently specific to 16S rRNA. </td></tr>
<tr><td>110</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09489">PRK09489</a></td><td>342</td><td>87</td><td>20.1</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rsmC</td><td>16S ribosomal RNA m2G1207 methyltransferase; Provisional</td></tr>
</table>
</body>
