<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0210">COG0210</a></td><td>655</td><td>303</td><td>99.9</td><td>2.7E-22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UvrD</td><td>Superfamily I DNA or RNA helicase </td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3972">COG3972</a></td><td>660</td><td>353</td><td>99.8</td><td>5.2E-19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;]</td><td>COG3972</td><td>Superfamily I DNA and RNA helicases  </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11773">PRK11773</a></td><td>721</td><td>288</td><td>99.7</td><td>1.1E-16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>uvrD</td><td>DNA-dependent helicase II; Provisional</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00609">TIGR00609</a></td><td>1087</td><td>173</td><td>99.7</td><td>6.3E-15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recB</td><td>exodeoxyribonuclease V, beta subunit. The RecBCD holoenzyme is a multifunctional nuclease with potent ATP-dependent exodeoxyribonuclease activity. Ejection of RecD, as occurs at chi recombinational hotspots, cripples exonuclease activity in favor of recombinagenic helicase activity. All proteins in this family for which functions are known are DNA-DNA helicases that are used as part of an exonuclease-helicase complex (made up of RecBCD homologs) that function to generate substrates for the initiation of recombination and recombinational repair. This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University). </td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1074">COG1074</a></td><td>1139</td><td>174</td><td>99.7</td><td>5E-16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecB</td><td>ATP-dependent exoDNAse (exonuclease V) beta subunit (contains helicase and exonuclease domains) </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01073">TIGR01073</a></td><td>726</td><td>292</td><td>99.7</td><td>5.4E-15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pcrA</td><td>ATP-dependent DNA helicase PcrA. Designed to identify pcrA members of the uvrD/rep subfamily. </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13361">pfam13361</a></td><td>342</td><td>97</td><td>99.6</td><td>1E-14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UvrD_C</td><td>UvrD-like helicase C-terminal domain. This domain is found at the C-terminus of a wide variety of helicase enzymes. This domain has a AAA-like structural fold.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01075">TIGR01075</a></td><td>715</td><td>287</td><td>99.6</td><td>1.7E-13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>uvrD</td><td>DNA helicase II. Designed to identify uvrD members of the uvrD/rep subfamily. </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01074">TIGR01074</a></td><td>664</td><td>279</td><td>99.5</td><td>5.5E-13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rep</td><td>ATP-dependent DNA helicase Rep. Designed to identify rep members of the uvrD/rep subfamily. </td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11054">PRK11054</a></td><td>684</td><td>335</td><td>99.5</td><td>4.8E-12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------]</td><td>helD</td><td>DNA helicase IV; Provisional</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3973">COG3973</a></td><td>747</td><td>199</td><td>99.4</td><td>1.1E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;]</td><td>HelD</td><td>DNA helicase IV </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02785">TIGR02785</a></td><td>1230</td><td>167</td><td>99.3</td><td>4.6E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>addA_Gpos</td><td>helicase-exonuclease AddAB, AddA subunit, Firmicutes type. AddAB, also called RexAB, substitutes for RecBCD in several bacterial lineages. These DNA recombination proteins act before synapse and are particularly important for DNA repair of double-stranded breaks by homologous recombination. The term AddAB is used broadly, with AddA homologous between the Firmicutes (as modeled here) and the alphaproteobacteria, while the partner AddB proteins show no strong homology across the two groups of species. </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00580">pfam00580</a></td><td>267</td><td>185</td><td>99.1</td><td>5E-10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UvrD-helicase</td><td>UvrD/REP helicase N-terminal domain. The Rep family helicases are composed of four structural domains. The Rep family function as dimers. REP helicases catalyse ATP dependent unwinding of double stranded DNA to single stranded DNA. Some members have large insertions near to the carboxy-terminus relative to other members of the family.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13909">PRK13909</a></td><td>910</td><td>156</td><td>98.8</td><td>1.2E-07</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13909</td><td>putative recombination protein RecB; Provisional</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13538">pfam13538</a></td><td>105</td><td>45</td><td>98.7</td><td>4.4E-09</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;]</td><td>UvrD_C_2</td><td>UvrD-like helicase C-terminal domain. This domain is found at the C-terminus of a wide variety of helicase enzymes. This domain has a AAA-like structural fold.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08378">pfam08378</a></td><td>111</td><td>86</td><td>98.7</td><td>1E-07</td><td>[&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NERD</td><td>Nuclease-related domain. The nuclease-related domain (NERD) is found in a range of bacterial as well as archaeal and plant proteins. It has distant similarity to endonucleases (hence its name) and its predicted secondary structure is helix - sheet - sheet - sheet - sheet - weak sheet/long loop - helix - sheet - sheet. The majority of NERD-containing proteins are single-domain, but in several cases proteins containing NERD have additional domains which in 75% of cases are involved in DNA processing.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00376">TIGR00376</a></td><td>636</td><td>354</td><td>98.6</td><td>9.9E-07</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------]</td><td>TIGR00376</td><td>DNA helicase, putative. The gene product may represent a DNA helicase. Eukaryotic members of this family have been characterized as binding certain single-stranded G-rich DNA sequences (GGGGT and GGGCT). A number of related proteins are characterized as helicases. </td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01447">TIGR01447</a></td><td>582</td><td>232</td><td>98.6</td><td>2E-07</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;]</td><td>recD</td><td>exodeoxyribonuclease V, alpha subunit. This family describes the exodeoxyribonuclease V alpha subunit, RecD. RecD is part of a RecBCD complex. A related family in the Gram-positive bacteria separates in a phylogenetic tree, has an additional N-terminal extension of about 200 residues, and is not supported as a member of a RecBCD complex by neighboring genes. The related family is consequently described by a different model. </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00046">cd00046</a></td><td>144</td><td>51</td><td>98.1</td><td>5.5E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DEXDc</td><td>DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13191">pfam13191</a></td><td>156</td><td>34</td><td>98.1</td><td>0.00014</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_16</td><td>AAA ATPase domain. This family of domains contain a P-loop motif that is characteristic of the AAA superfamily.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01443">pfam01443</a></td><td>227</td><td>42</td><td>98.1</td><td>2.6E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;]</td><td>Viral_helicase1</td><td>Viral (Superfamily 1) RNA helicase. Helicase activity for this family has been demonstrated and NTPase activity. This helicase has multiple roles at different stages of viral RNA replication, as dissected by mutational analysis.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13245">pfam13245</a></td><td>72</td><td>44</td><td>98.1</td><td>4.5E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_19</td><td>Part of AAA domain. </td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00009">cd00009</a></td><td>151</td><td>39</td><td>98.1</td><td>1.9E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA</td><td>The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC, RecA-like, VirD4-like, PilT-like, and SF1/2 helicases. Members of the AAA+ ATPases function as molecular chaperons, ATPase subunits of proteases, helicases, or nucleic-acid stimulated ATPases. The AAA+ proteins contain several distinct features in addition to the conserved alpha-beta-alpha core domain structure and the Walker A and B motifs of the P-loop NTPases.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10919">PRK10919</a></td><td>672</td><td>93</td><td>97.9</td><td>2.3E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10919</td><td>ATP-dependent DNA helicase Rep; Provisional</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04851">pfam04851</a></td><td>103</td><td>43</td><td>97.9</td><td>4.2E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ResIII</td><td>Type III restriction enzyme, res subunit. </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13401">pfam13401</a></td><td>124</td><td>25</td><td>97.9</td><td>0.00015</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_22</td><td>AAA domain. </td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00064">TIGR00064</a></td><td>277</td><td>130</td><td>97.8</td><td>0.00016</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ftsY</td><td>signal recognition particle-docking protein FtsY. There is a weak division between FtsY and SRP54; both are GTPases. In E.coli, ftsY is an essential gene located in an operon with cell division genes ftsE and ftsX, but its apparent function is as the signal recognition particle docking protein. </td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0507">COG0507</a></td><td>696</td><td>50</td><td>97.8</td><td>1.9E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----]</td><td>RecD</td><td>ATP-dependent exoDNAse (exonuclease V), alpha subunit, helicase superfamily I  </td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13604">pfam13604</a></td><td>195</td><td>160</td><td>97.8</td><td>0.00016</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_30</td><td>AAA domain. This family of domains contain a P-loop motif that is characteristic of the AAA superfamily. Many of the proteins in this family are conjugative transfer proteins. There is a Walker A and Walker B.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02784">TIGR02784</a></td><td>1135</td><td>110</td><td>97.8</td><td>1.8E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>addA_alphas</td><td>double-strand break repair helicase AddA, alphaproteobacterial type. AddAB, also called RexAB, substitutes for RecBCD in several bacterial lineages. These DNA recombination proteins act before synapse and are particularly important for DNA repair of double-stranded breaks by homologous recombination. The term AddAB is used broadly, with AddA homologous between the alphaproteobacteria (as modeled here) and the Firmicutes, while the partner AddB proteins show no strong homology across the two groups of species. </td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13087">pfam13087</a></td><td>197</td><td>157</td><td>97.8</td><td>0.00034</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;]</td><td>AAA_12</td><td>AAA domain. This family of domains contain a P-loop motif that is characteristic of the AAA superfamily. Many of the proteins in this family are conjugative transfer proteins.</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1112">COG1112</a></td><td>767</td><td>217</td><td>97.7</td><td>0.0029</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------]</td><td>DNA2</td><td>Superfamily I DNA and/or RNA helicase </td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13086">pfam13086</a></td><td>183</td><td>68</td><td>97.7</td><td>0.00014</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_11</td><td>AAA domain. This family of domains contain a P-loop motif that is characteristic of the AAA superfamily. Many of the proteins in this family are conjugative transfer proteins.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09848">pfam09848</a></td><td>348</td><td>152</td><td>97.6</td><td>0.00017</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2075</td><td>Uncharacterized conserved protein (DUF2075). This domain, found in various prokaryotic proteins (including putative ATP/GTP binding proteins), has no known function.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03115">cd03115</a></td><td>173</td><td>128</td><td>97.5</td><td>0.0012</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SRP</td><td>The signal recognition particle (SRP) mediates the transport to or across the plasma membrane in bacteria and the endoplasmic reticulum in eukaryotes. SRP recognizes N-terminal sighnal sequences of newly synthesized polypeptides at the ribosome. The SRP-polypeptide complex is then targeted to the membrane by an interaction between SRP and its cognated receptor (SR). In mammals, SRP consists of six protein subunits and a 7SL RNA. One of these subunits is a 54 kd protein (SRP54), which is a GTP-binding protein that interacts with the signal sequence when it emerges from the ribosome. SRP54 is a multidomain protein that consists of an N-terminal domain, followed by a central G (GTPase) domain and a C-terminal M domain.</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02035">cd02035</a></td><td>217</td><td>160</td><td>97.5</td><td>0.00052</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArsA</td><td>ArsA ATPase functionas as an efflux pump located on the inner membrane of the cell. This ATP-driven oxyanion pump catalyzes the extrusion of arsenite, antimonite and arsenate. Maintenance of a low intracellular concentration of oxyanion produces resistance to the toxic agents. The pump is composed of two subunits, the catalytic ArsA subunit and the membrane subunit ArsB, which are encoded by arsA and arsB genes respectively. Arsenic efflux in bacteria is catalyzed by either ArsB alone or by ArsAB complex. The ATP-coupled pump, however, is more efficient. ArsA is composed of two homologous halves, A1 and A2, connected by a short linker sequence.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00448">pfam00448</a></td><td>195</td><td>132</td><td>97.4</td><td>0.0023</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SRP54</td><td>SRP54-type protein, GTPase domain. This family includes relatives of the G-domain of the SRP54 family of proteins.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10416">PRK10416</a></td><td>318</td><td>130</td><td>97.3</td><td>0.0028</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10416</td><td>signal recognition particle-docking protein FtsY; Provisional</td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0552">COG0552</a></td><td>340</td><td>130</td><td>97.2</td><td>0.0027</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FtsY</td><td>Signal recognition particle GTPase </td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02562">pfam02562</a></td><td>205</td><td>53</td><td>97.2</td><td>0.0034</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PhoH</td><td>PhoH-like protein. PhoH is a cytoplasmic protein and predicted ATPase that is induced by phosphate starvation.</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01448">TIGR01448</a></td><td>720</td><td>146</td><td>97.0</td><td>0.0018</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------]</td><td>recD_rel</td><td>helicase, putative, RecD/TraA family. This model describes a family similar to RecD, the exodeoxyribonuclease V alpha chain of TIGR01447. Members of this family, however, are not found in a context of RecB and RecC and are longer by about 200 amino acids at the amino end. Chlamydia muridarum has both a member of this family and a RecD. </td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13173">pfam13173</a></td><td>127</td><td>35</td><td>96.9</td><td>0.0043</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_14</td><td>AAA domain. This family of domains contain a P-loop motif that is characteristic of the AAA superfamily.</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK04296">PRK04296</a></td><td>190</td><td>102</td><td>96.9</td><td>0.0023</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK04296</td><td>thymidine kinase; Provisional</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00270">pfam00270</a></td><td>167</td><td>49</td><td>96.8</td><td>0.011</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DEAD</td><td>DEAD/DEAH box helicase. Members of this family include the DEAD and DEAH box helicases. Helicases are involved in unwinding nucleic acids. The DEAD box helicases are involved in various aspects of RNA metabolizm, including nuclear transcription, pre mRNA splicing, ribosome biogenesis, nucleocytoplasmic transport, translation, RNA decay and organellar gene expression.</td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01983">cd01983</a></td><td>99</td><td>31</td><td>96.8</td><td>0.0014</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fer4_NifH</td><td>The Fer4_NifH superfamily contains a variety of proteins which share a common ATP-binding domain. Functionally, proteins in this superfamily use the energy from hydrolysis of NTP to transfer electron or ion.</td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01120">cd01120</a></td><td>165</td><td>39</td><td>96.7</td><td>0.015</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecA-like_NTPases</td><td>RecA-like NTPases. This family includes the NTP binding domain of F1 and V1 H+ATPases, DnaB and related helicases as well as bacterial RecA and related eukaryotic and archaeal recombinases. This group also includes bacterial conjugation proteins and related DNA transfer proteins involved in type II and type IV secretion.</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0513">COG0513</a></td><td>513</td><td>301</td><td>96.5</td><td>0.34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SrmB</td><td>Superfamily II DNA and RNA helicase </td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10876">PRK10876</a></td><td>1181</td><td>85</td><td>96.5</td><td>0.0031</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recB</td><td>exonuclease V subunit beta; Provisional</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1061">COG1061</a></td><td>442</td><td>125</td><td>96.3</td><td>0.0082</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SSL2</td><td>Superfamily II DNA or RNA helicase </td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09361">PRK09361</a></td><td>225</td><td>44</td><td>96.3</td><td>0.0098</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>radB</td><td>DNA repair and recombination protein RadB; Provisional</td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3857">COG3857</a></td><td>1108</td><td>283</td><td>96.2</td><td>0.17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AddB</td><td>ATP-dependent helicase/DNAse subunit B </td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05707">pfam05707</a></td><td>183</td><td>30</td><td>96.1</td><td>0.015</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Zot</td><td>Zonular occludens toxin (Zot). This family consists of bacterial and viral proteins which are very similar to the Zonular occludens toxin (Zot). Zot is elaborated by bacteriophages present in toxigenic strains of Vibrio cholerae. Zot is a single polypeptide chain of 44.8 kDa, with the ability to reversibly alter intestinal epithelial tight junctions, allowing the passage of macromolecules through mucosal barriers</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0541">COG0541</a></td><td>451</td><td>128</td><td>96.1</td><td>0.078</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ffh</td><td>Signal recognition particle GTPase </td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10875">PRK10875</a></td><td>615</td><td>42</td><td>96.0</td><td>0.0072</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;]</td><td>recD</td><td>exonuclease V subunit alpha; Provisional</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13207">pfam13207</a></td><td>114</td><td>19</td><td>95.9</td><td>0.0072</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_17</td><td>AAA domain. </td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00771">PRK00771</a></td><td>437</td><td>126</td><td>95.9</td><td>0.086</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00771</td><td>signal recognition particle protein Srp54; Provisional</td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13768">PRK13768</a></td><td>253</td><td>36</td><td>95.8</td><td>0.0087</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13768</td><td>GTPase; Provisional</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1435">COG1435</a></td><td>201</td><td>108</td><td>95.7</td><td>0.048</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Tdk</td><td>Thymidine kinase </td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13614">pfam13614</a></td><td>145</td><td>115</td><td>95.7</td><td>0.087</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_31</td><td>AAA domain. This family includes a wide variety of AAA domains including some that have lost essential nucleotide binding residues in the P-loop.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02034">cd02034</a></td><td>116</td><td>35</td><td>95.7</td><td>0.011</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CooC</td><td>The accessory protein CooC, which contains a nucleotide-binding domain (P-loop) near the N-terminus, participates in the maturation of the nickel center of carbon monoxide dehydrogenase (CODH). CODH from Rhodospirillum rubrum catalyzes the reversible oxidation of CO to CO2. CODH contains a nickel-iron-sulfur cluster (C-center) and an iron-sulfur cluster (B-center). CO oxidation occurs at the C-center. Three accessory proteins encoded by cooCTJ genes are involved in nickel incorporation into a nickel site. CooC functions as a nickel insertase that mobilizes nickel to apoCODH using energy released from ATP hydrolysis. CooC is a homodimer and has NTPase activities. Mutation at the P-loop abolishs its function.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1643">COG1643</a></td><td>845</td><td>158</td><td>95.4</td><td>0.033</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HrpA</td><td>HrpA-like RNA helicase </td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13481">pfam13481</a></td><td>192</td><td>47</td><td>95.4</td><td>0.063</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_25</td><td>AAA domain. This AAA domain is found in a wide variety of presumed DNA repair proteins.</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02042">cd02042</a></td><td>104</td><td>31</td><td>95.0</td><td>0.017</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ParA</td><td>ParA and ParB of Caulobacter crescentus belong to a conserved family of bacterial proteins implicated in chromosome segregation. ParB binds to DNA sequences adjacent to the origin of replication and localizes to opposite cell poles shortly following the initiation of DNA replication. ParB regulates the ParA ATPase activity by promoting nucleotide exchange in a fashion reminiscent of the exchange factors of eukaryotic G proteins. ADP-bound ParA binds single-stranded DNA, whereas the ATP-bound form dissociates ParB from its DNA binding sites. Increasing the fraction of ParA-ADP in the cell inhibits cell division, suggesting that this simple nucleotide switch may regulate cytokinesis. ParA shares sequence similarity to a conserved and widespread family of ATPases which includes the repA protein of the repABC operon in R. etli Sym plasmid. This operon is involved in the plasmid replication and partition.</td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10919">PRK10919</a></td><td>672</td><td>148</td><td>95.0</td><td>0.27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10919</td><td>ATP-dependent DNA helicase Rep; Provisional</td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13476">pfam13476</a></td><td>203</td><td>26</td><td>94.9</td><td>0.015</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_23</td><td>AAA domain. </td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03214">cd03214</a></td><td>180</td><td>119</td><td>94.9</td><td>0.11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Iron-Siderophores_B12_Hemin</td><td>ATP-binding component of iron-siderophores, vitamin B12 and hemin transporters and related proteins. ABC transporters, involved in the uptake of siderophores, heme, and vitamin B12, are widely conserved in bacteria and archaea. Only very few species lack representatives of the siderophore family transporters. The E. coli BtuCD protein is an ABC transporter mediating vitamin B12 uptake. The two ATP-binding cassettes (BtuD) are in close contact with each other, as are the two membrane-spanning subunits (BtuC); this arrangement is distinct from that observed for the E. coli lipid flippase MsbA. The BtuC subunits provide 20 transmembrane helices grouped around a translocation pathway that is closed to the cytoplasm by a gate region, whereas the dimer arrangement of the BtuD subunits resembles the ATP-bound form of the Rad50 DNA repair enzyme. A prominent cytoplasmic loop of BtuC forms the contact region with the ATP-binding cassette and represent a conserved motif among the ABC transporters.</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0467">COG0467</a></td><td>260</td><td>66</td><td>94.8</td><td>0.099</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RAD55</td><td>RecA-superfamily ATPase, KaiC/GvpD/RAD55 family </td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1203">COG1203</a></td><td>733</td><td>178</td><td>94.8</td><td>0.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas3</td><td>CRISPR/Cas system-associated endonuclease/helicase Cas3 </td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1074">COG1074</a></td><td>1139</td><td>80</td><td>94.8</td><td>0.053</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecB</td><td>ATP-dependent exoDNAse (exonuclease V) beta subunit (contains helicase and exonuclease domains) </td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02784">TIGR02784</a></td><td>1135</td><td>61</td><td>94.7</td><td>0.049</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>addA_alphas</td><td>double-strand break repair helicase AddA, alphaproteobacterial type. AddAB, also called RexAB, substitutes for RecBCD in several bacterial lineages. These DNA recombination proteins act before synapse and are particularly important for DNA repair of double-stranded breaks by homologous recombination. The term AddAB is used broadly, with AddA homologous between the alphaproteobacteria (as modeled here) and the Firmicutes, while the partner AddB proteins show no strong homology across the two groups of species. </td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03029">pfam03029</a></td><td>235</td><td>32</td><td>94.6</td><td>0.023</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ATP_bind_1</td><td>Conserved hypothetical ATP binding protein. Members of this family are found in a range of archaea and eukaryotes and have hypothesised ATP binding activity.</td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01393">cd01393</a></td><td>226</td><td>57</td><td>94.6</td><td>0.097</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recA_like</td><td>RecA is a  bacterial enzyme which has roles in homologous recombination, DNA repair, and the induction of the SOS response.  RecA couples ATP hydrolysis to DNA strand exchange. While prokaryotes have a single RecA protein, eukaryotes have multiple RecA homologs such as Rad51, DMC1 and Rad55/57.  Archaea have the RecA-like homologs radA and radB.</td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1131">COG1131</a></td><td>293</td><td>148</td><td>94.5</td><td>0.047</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CcmA</td><td>ABC-type multidrug transport system, ATPase component </td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1484">COG1484</a></td><td>254</td><td>41</td><td>94.5</td><td>0.063</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DnaC</td><td>DNA replication protein DnaC </td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3598">COG3598</a></td><td>402</td><td>73</td><td>94.3</td><td>0.098</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RepA</td><td>RecA-family ATPase  </td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02773">TIGR02773</a></td><td>1160</td><td>273</td><td>94.0</td><td>2.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>addB_Gpos</td><td>helicase-exonuclease AddAB, AddB subunit. DNA repair is accomplished by several different systems in prokaryotes. Recombinational repair of double-stranded DNA breaks involves the RecBCD pathway in some lineages, and AddAB (also called RexAB) in other. The AddA protein is conserved between the firmicutes and the alphaproteobacteria, while the partner protein is not. Nevertheless, the partner is designated AddB in both systems. This model describes the AddB protein as found Bacillus subtilis and related species. Although the RexB protein of Streptococcus and Lactococcus is considered to be orthologous, functionally equivalent, and merely named differently, all members of this protein family have a P-loop nucleotide binding motif GxxGxGK</td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0489">COG0489</a></td><td>265</td><td>136</td><td>93.9</td><td>0.06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Mrp</td><td>Chromosome partitioning ATPase, Mrp family, contains Fe-S cluster </td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0470">COG0470</a></td><td>325</td><td>55</td><td>93.9</td><td>0.23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HolB</td><td>DNA polymerase III, delta prime subunit </td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02237">TIGR02237</a></td><td>209</td><td>48</td><td>93.8</td><td>0.086</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recomb_radB</td><td>DNA repair and recombination protein RadB. This family consists exclusively of archaeal RadB protein, a homolog of bacterial RecA (TIGR02012), eukaryotic RAD51 (TIGR02239) and DMC1 (TIGR02238), and archaeal RadA (TIGR02236).</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02021">cd02021</a></td><td>150</td><td>20</td><td>93.7</td><td>0.038</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GntK</td><td>Gluconate kinase (GntK) catalyzes the phosphoryl transfer from ATP to gluconate. The resulting product gluconate-6-phoshate is an important precursor of gluconate metabolism. GntK acts as a dimmer composed of two identical subunits.</td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00004">pfam00004</a></td><td>131</td><td>61</td><td>93.6</td><td>0.22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA</td><td>ATPase family associated with various cellular activities (AAA). AAA family proteins often perform chaperone-like functions that assist in the assembly, operation, or disassembly of protein complexes.</td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1132">COG1132</a></td><td>567</td><td>69</td><td>93.6</td><td>0.24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MdlB</td><td>ABC-type multidrug transport system, ATPase and permease component </td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00959">TIGR00959</a></td><td>428</td><td>126</td><td>93.5</td><td>0.92</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ffh</td><td>signal recognition particle protein. This model represents Ffh (Fifty-Four Homolog), the protein component that forms the bacterial (and organellar) signal recognition particle together with a 4.5S RNA. Ffh is a GTPase homologous to eukaryotic SRP54 and also to the GTPase FtsY (TIGR00064) that is the receptor for the signal recognition particle. </td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0468">COG0468</a></td><td>279</td><td>46</td><td>93.4</td><td>0.18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecA</td><td>RecA/RadA recombinase </td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03264">cd03264</a></td><td>211</td><td>155</td><td>93.3</td><td>0.15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_drug_resistance_like</td><td>ABC-type multidrug transport system, ATPase component. The biological function of this family is not well characterized, but display ABC domains similar to members of ABCA subfamily. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>86</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3267">COG3267</a></td><td>269</td><td>34</td><td>93.2</td><td>0.39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ExeA</td><td>Type II secretory pathway, component ExeA (predicted ATPase)  </td></tr>
<tr><td>87</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02374">pfam02374</a></td><td>304</td><td>43</td><td>93.1</td><td>0.084</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArsA_ATPase</td><td>Anion-transporting ATPase. This Pfam family represents a conserved domain, which is sometimes repeated, in an anion-transporting ATPase. The ATPase is involved in the removal of arsenate, antimonite, and arsenate from the cell.</td></tr>
<tr><td>88</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4096">COG4096</a></td><td>875</td><td>107</td><td>93.0</td><td>1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HsdR</td><td>Type I site-specific restriction endonuclease, part of a restriction-modification system </td></tr>
<tr><td>89</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10876">PRK10876</a></td><td>1181</td><td>36</td><td>92.8</td><td>1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recB</td><td>exonuclease V subunit beta; Provisional</td></tr>
<tr><td>90</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01656">pfam01656</a></td><td>211</td><td>31</td><td>92.8</td><td>0.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CbiA</td><td>CobQ/CobB/MinD/ParA nucleotide binding domain. This family consists of various cobyrinic acid a,c-diamide synthases. These include CbiA and CbiP from S.typhimurium, and CobQ from R. capsulatus. These amidases catalyse amidations to various side chains of hydrogenobyrinic acid or cobyrinic acid a,c-diamide in the biosynthesis of cobalamin (vitamin B12) from uroporphyrinogen III. Vitamin B12 is an important cofactor and an essential nutrient for many plants and animals and is primarily produced by bacteria. The family also contains dethiobiotin synthetases as well as the plasmid partitioning proteins of the MinD/ParA family.</td></tr>
<tr><td>91</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00631">TIGR00631</a></td><td>655</td><td>52</td><td>92.7</td><td>0.12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>uvrb</td><td>excinuclease ABC, B subunit. All proteins in this family for wich functions are known are DNA helicases that function in the nucleotide excision repair and are endonucleases that make the 3' incision next to DNA damage. They are part of a pathway requiring UvrA, UvrB, UvrC, and UvrD homologs. This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University) </td></tr>
<tr><td>92</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02224">PRK02224</a></td><td>880</td><td>30</td><td>92.6</td><td>0.061</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK02224</td><td>chromosome segregation protein; Provisional</td></tr>
<tr><td>93</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13304">pfam13304</a></td><td>144</td><td>28</td><td>92.5</td><td>0.061</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_21</td><td>AAA domain. </td></tr>
<tr><td>94</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1198">COG1198</a></td><td>730</td><td>338</td><td>92.5</td><td>0.22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;]</td><td>PriA</td><td>Primosomal protein N' (replication factor Y) - superfamily II helicase </td></tr>
<tr><td>95</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05729">pfam05729</a></td><td>165</td><td>25</td><td>92.5</td><td>0.13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NACHT</td><td>NACHT domain. This NTPase domain is found in apoptosis proteins as well as those involved in MHC transcription activation. This family is closely related to pfam00931.</td></tr>
<tr><td>96</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1192">COG1192</a></td><td>259</td><td>28</td><td>92.4</td><td>0.092</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BcsQ</td><td>Cellulose biosynthesis protein BcsQ </td></tr>
<tr><td>97</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02020">cd02020</a></td><td>147</td><td>21</td><td>92.4</td><td>0.073</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CMPK</td><td>Cytidine monophosphate kinase (CMPK) catalyzes the reversible phosphorylation of cytidine monophosphate (CMP) to produce cytidine diphosphate (CDP), using ATP as the preferred phosphoryl donor.</td></tr>
<tr><td>98</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08433">pfam08433</a></td><td>267</td><td>32</td><td>92.3</td><td>0.13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KTI12</td><td>Chromatin associated protein KTI12. This is a family of chromatin associated proteins which interact with the Elongator complex, a component of the elongating form of RNA polymerase II. The Elongator complex has histone acetyltransferase activity.</td></tr>
<tr><td>99</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0556">COG0556</a></td><td>663</td><td>51</td><td>92.3</td><td>0.15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UvrB</td><td>Excinuclease UvrABC helicase subunit UvrB </td></tr>
<tr><td>100</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01394">cd01394</a></td><td>218</td><td>44</td><td>92.2</td><td>0.17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>radB</td><td>RadB. The archaeal protein radB shares similarity radA, the archaeal functional homologue to the bacterial RecA. The precise function of radB is unclear.</td></tr>
<tr><td>101</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02785">TIGR02785</a></td><td>1230</td><td>67</td><td>92.2</td><td>0.35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----]</td><td>addA_Gpos</td><td>helicase-exonuclease AddAB, AddA subunit, Firmicutes type. AddAB, also called RexAB, substitutes for RecBCD in several bacterial lineages. These DNA recombination proteins act before synapse and are particularly important for DNA repair of double-stranded breaks by homologous recombination. The term AddAB is used broadly, with AddA homologous between the Firmicutes (as modeled here) and the alphaproteobacteria, while the partner AddB proteins show no strong homology across the two groups of species. </td></tr>
<tr><td>102</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09639">cd09639</a></td><td>353</td><td>228</td><td>92.1</td><td>2.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas3_I</td><td>CRISPR/Cas system-associated protein Cas3. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; DEAD/DEAH box helicase DNA helicase cas3'; Often but not always is fused to HD nuclease domain; signature gene for Type I</td></tr>
<tr><td>103</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01313">TIGR01313</a></td><td>163</td><td>22</td><td>92.1</td><td>0.087</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>therm_gnt_kin</td><td>carbohydrate kinase, thermoresistant glucokinase family. This model represents a subfamily of proteins that includes thermoresistant and thermosensitve isozymes of gluconate kinase (gluconokinase) in E. coli and other related proteins; members of this family are often named by similarity to the thermostable isozyme. These proteins show homology to shikimate kinases and adenylate kinases but not to gluconate kinases from the FGGY family of carbohydrate kinases.</td></tr>
<tr><td>104</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05298">PRK05298</a></td><td>652</td><td>51</td><td>92.1</td><td>0.14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05298</td><td>excinuclease ABC subunit B; Provisional</td></tr>
<tr><td>105</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02036">cd02036</a></td><td>179</td><td>38</td><td>92.1</td><td>0.14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MinD</td><td>Bacterial cell division requires the formation of a septum at mid-cell. The site is determined by the min operon products MinC, MinD and MinE. MinC is a nonspecific inhibitor of the septum protein FtsZ. MinE is the supressor of MinC. MinD plays a pivotal role, selecting the mid-cell over other sites through the activation and regulation of MinC and MinE. MinD is a membrane-associated ATPase, related to nitrogenase iron protein. More distantly related proteins include flagellar biosynthesis proteins and ParA chromosome partitioning proteins. MinD is a monomer.</td></tr>
<tr><td>106</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK04040">PRK04040</a></td><td>188</td><td>31</td><td>92.0</td><td>0.11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK04040</td><td>adenylate kinase; Provisional</td></tr>
<tr><td>107</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01672">cd01672</a></td><td>200</td><td>44</td><td>92.0</td><td>0.16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TMPK</td><td>Thymidine monophosphate kinase (TMPK), also known as thymidylate kinase, catalyzes the phosphorylation of thymidine monophosphate (TMP) to thymidine diphosphate (TDP) utilizing ATP as its preferred phophoryl donor. TMPK represents the rate-limiting step in either de novo or salvage biosynthesis of thymidine triphosphate (TTP).</td></tr>
<tr><td>108</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05580">PRK05580</a></td><td>679</td><td>49</td><td>92.0</td><td>0.21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05580</td><td>primosome assembly protein PriA; Validated</td></tr>
<tr><td>109</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1703">COG1703</a></td><td>323</td><td>46</td><td>92.0</td><td>0.18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArgK</td><td>Putative periplasmic protein kinase ArgK or related GTPase of G3E family </td></tr>
<tr><td>110</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13695">PRK13695</a></td><td>174</td><td>124</td><td>91.8</td><td>0.12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13695</td><td>putative NTPase; Provisional</td></tr>
<tr><td>111</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12846">pfam12846</a></td><td>298</td><td>37</td><td>91.7</td><td>0.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_10</td><td>AAA-like domain. This family of domains contain a P-loop motif that is characteristic of the AAA superfamily. Many of the proteins in this family are conjugative transfer proteins.</td></tr>
<tr><td>112</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00643">TIGR00643</a></td><td>630</td><td>198</td><td>91.7</td><td>0.45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recG</td><td>ATP-dependent DNA helicase RecG. </td></tr>
<tr><td>113</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01129">cd01129</a></td><td>264</td><td>70</td><td>91.6</td><td>0.25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PulE-GspE</td><td>PulE/GspE The type II secretory pathway is the main terminal branch of the general secretory pathway (GSP).  It is responsible for the export the majority of Gram-negative bacterial exoenzymes and toxins. PulE is a cytoplasmic protein of the GSP, which contains an ATP binding site and a tetracysteine motif. This subgroup also includes PillB and HofB.</td></tr>
<tr><td>114</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0003">COG0003</a></td><td>322</td><td>41</td><td>91.6</td><td>0.18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArsA</td><td>Anion-transporting ATPase, ArsA/GET3 family </td></tr>
<tr><td>115</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13238">pfam13238</a></td><td>128</td><td>20</td><td>91.5</td><td>0.12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_18</td><td>AAA domain. </td></tr>
<tr><td>116</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2019">COG2019</a></td><td>189</td><td>30</td><td>91.4</td><td>0.13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AdkA</td><td>Archaeal adenylate kinase  </td></tr>
<tr><td>117</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2805">COG2805</a></td><td>353</td><td>132</td><td>91.3</td><td>0.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PilT</td><td>Tfp pilus assembly protein PilT, pilus retraction ATPase </td></tr>
<tr><td>118</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03499">TIGR03499</a></td><td>282</td><td>47</td><td>91.3</td><td>0.35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FlhF</td><td>flagellar biosynthetic protein FlhF. </td></tr>
<tr><td>119</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01637">pfam01637</a></td><td>223</td><td>187</td><td>91.2</td><td>7.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Arch_ATPase</td><td>Archaeal ATPase. This family contain a conserved P-loop motif that is involved in binding ATP. This family is almost exclusively found in archaebacteria and particularly in Methanococcus jannaschii that encodes sixteen members of this family.</td></tr>
<tr><td>120</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2909">COG2909</a></td><td>894</td><td>113</td><td>91.2</td><td>0.27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MalT</td><td>ATP-, maltotriose- and DNA-dependent transcriptional regulator MalT </td></tr>
<tr><td>121</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01967">TIGR01967</a></td><td>1283</td><td>159</td><td>91.1</td><td>0.32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DEAH_box_HrpA</td><td>RNA helicase HrpA. This model represents HrpA, one of two related but uncharacterized DEAH-box ATP-dependent helicases in many Proteobacteria and a few high-GC Gram-positive bacteria. HrpA is about 1300 amino acids long, while its paralog HrpB, also uncharacterized, is about 800 amino acids long. Related characterized eukarotic proteins are RNA helicases associated with pre-mRNA processing. The HrpA/B homolog from Borrelia is 500 amino acids shorter but appears to be derived from HrpA rather than HrpB. </td></tr>
<tr><td>122</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0529">COG0529</a></td><td>197</td><td>31</td><td>91.0</td><td>0.21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CysC</td><td>Adenylylsulfate kinase or related kinase </td></tr>
<tr><td>123</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08939">PRK08939</a></td><td>306</td><td>48</td><td>90.9</td><td>0.27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08939</td><td>primosomal protein DnaI; Reviewed</td></tr>
<tr><td>124</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1126">COG1126</a></td><td>240</td><td>42</td><td>90.9</td><td>0.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GlnQ</td><td>ABC-type polar amino acid transport system, ATPase component </td></tr>
<tr><td>125</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03277">cd03277</a></td><td>213</td><td>57</td><td>90.9</td><td>0.14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC5_euk</td><td>ATP-binding cassette domain of eukaryotic SMC5 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>126</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01131">cd01131</a></td><td>198</td><td>34</td><td>90.7</td><td>0.24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PilT</td><td>Pilus retraction ATPase PilT. PilT is a nucleotide binding protein responsible for the retraction of type IV pili, likely by pili disassembly. This retraction provides the force required for travel of bacteria in low water environments by a mechanism known as twitching motility.</td></tr>
<tr><td>127</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08116">PRK08116</a></td><td>268</td><td>33</td><td>90.7</td><td>0.22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08116</td><td>hypothetical protein; Validated</td></tr>
<tr><td>128</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1197">COG1197</a></td><td>1139</td><td>117</td><td>90.6</td><td>0.32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Mfd</td><td>Transcription-repair coupling factor (superfamily II helicase) </td></tr>
<tr><td>129</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03308">pfam03308</a></td><td>267</td><td>189</td><td>90.6</td><td>0.31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArgK</td><td>ArgK protein. The ArgK protein acts as an ATPase enzyme and as a kinase, and phosphorylates periplasmic binding proteins involved in the LAO (lysine, arginine, ornithine)/AO transport systems.</td></tr>
<tr><td>130</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02027">cd02027</a></td><td>149</td><td>30</td><td>90.5</td><td>0.23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>APSK</td><td>Adenosine 5'-phosphosulfate kinase (APSK) catalyzes the phosphorylation of adenosine 5'-phosphosulfate to form 3'-phosphoadenosine 5'-phosphosulfate (PAPS). The end-product PAPS is a biologically "activated" sulfate form important for the assimilation of inorganic sulfate.</td></tr>
<tr><td>131</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13555">pfam13555</a></td><td>60</td><td>19</td><td>90.5</td><td>0.13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_29</td><td>P-loop containing region of AAA domain. </td></tr>
<tr><td>132</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03234">cd03234</a></td><td>226</td><td>69</td><td>90.2</td><td>0.53</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCG_White</td><td>White pigment protein homolog of ABCG transporter subfamily. The White subfamily represents ABC transporters homologous to the Drosophila white gene, which acts as a dimeric importer for eye pigment precursors. The eye pigmentation of Drosophila is developed from the synthesis and deposition in the cells of red pigments, which are synthesized from guanine, and brown pigments, which are synthesized from tryptophan. The pigment precursors are encoded by the white, brown, and scarlet genes, respectively. Evidence from genetic and biochemical studies suggest that the White and Brown proteins function as heterodimers to import guanine, while the White and Scarlet proteins function to import tryptophan. However, a recent study also suggests that White may be involved in the transport of a metabolite, such as 3-hydroxykynurenine, across intracellular membranes. Mammalian ABC transporters belonging to the White subfamily (ABCG1, ABCG5, and ABCG8) have been shown to be involved in the regulation of lipid-trafficking mechanisms in macrophages, hepatocytes, and intestinal mucosa cells. ABCG1 (ABC8), the human homolog of the Drosophila white gene is induced in monocyte-derived macrophages during cholesterol influx mediated by acetylated low-density lipoprotein. It is possible that human ABCG1 forms heterodimers with several heterologous partners.</td></tr>
<tr><td>133</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00411">PRK00411</a></td><td>394</td><td>127</td><td>90.2</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cdc6</td><td>cell division control protein 6; Reviewed</td></tr>
<tr><td>134</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00176">TIGR00176</a></td><td>155</td><td>60</td><td>90.2</td><td>0.27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>mobB</td><td>molybdopterin-guanine dinucleotide biosynthesis protein MobB. This molybdenum cofactor biosynthesis enzyme is similar to the urease accessory protein UreG and to the hydrogenase accessory protein HypB, both GTP hydrolases involved in loading nickel into the metallocenters of their respective target enzymes. </td></tr>
<tr><td>135</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02399">pfam02399</a></td><td>829</td><td>53</td><td>90.1</td><td>0.33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Herpes_ori_bp</td><td>Origin of replication binding protein. This Pfam family represents the herpesvirus origin of replication binding protein, probably involved in DNA replication.</td></tr>
<tr><td>136</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03453">TIGR03453</a></td><td>387</td><td>68</td><td>90.1</td><td>0.21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>partition_RepA</td><td>plasmid partitioning protein RepA. Members of this family are the RepA (or ParA) protein involved in replicon partitioning. All known examples occur in bacterial species with two or more replicons, on a plasmid or the smaller chromosome. Note that an apparent exception may be seen as a pseudomolecule from assembly of an incompletely sequenced genome. Members of this family belong to a larger family that also includes the enzyme cobyrinic acid a,c-diamide synthase, but assignment of that name to members of this family would be in error. </td></tr>
<tr><td>137</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01420">TIGR01420</a></td><td>343</td><td>44</td><td>90.0</td><td>0.28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pilT_fam</td><td>pilus retraction protein PilT. This model represents the PilT subfamily of proteins related to GspE, a protein involved in type II secretion (also called the General Secretion Pathway). PilT is an apparent cytosolic ATPase associated with type IV pilus systems. It is not required for pilin biogenesis, but is required for twitching motility and social gliding behaviors, shown in some species, powered by pilus retraction. Members of this family may be found in some species that type IV pili but have related structures for DNA uptake and natural transformation. </td></tr>
<tr><td>138</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01695">pfam01695</a></td><td>178</td><td>47</td><td>90.0</td><td>0.55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>IstB_IS21</td><td>IstB-like ATP binding protein. This protein contains an ATP/GTP binding P-loop motif. It is found associated with IS21 family insertion sequences. The function of this protein is unknown, but it may perform a transposase function.</td></tr>
<tr><td>139</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3640">COG3640</a></td><td>255</td><td>39</td><td>90.0</td><td>0.27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CooC</td><td>CO dehydrogenase nickel-insertion accessory protein CooC1 </td></tr>
<tr><td>140</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13521">pfam13521</a></td><td>162</td><td>37</td><td>89.7</td><td>0.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_28</td><td>AAA domain. </td></tr>
<tr><td>141</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03240">cd03240</a></td><td>204</td><td>29</td><td>89.7</td><td>0.19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Rad50</td><td>ATP-binding cassette domain of Rad50. The catalytic domains of Rad50 are similar to the ATP-binding cassette of ABC transporters, but are not associated with membrane-spanning domains. The conserved ATP-binding motifs common to Rad50 and the ABC transporter family include the Walker A and Walker B motifs, the Q loop, a histidine residue in the switch region, a D-loop, and a conserved LSGG sequence. This conserved sequence, LSGG, is the most specific and characteristic motif of this family and is thus known as the ABC signature sequence.</td></tr>
<tr><td>142</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03227">cd03227</a></td><td>162</td><td>26</td><td>89.7</td><td>0.15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Class2</td><td>ATP-binding cassette domain of non-transporter proteins. ABC-type Class 2 contains systems involved in cellular processes other than transport. These families are characterized by the fact that the ABC subunit is made up of duplicated, fused ABC modules (ABC2). No known transmembrane proteins or domains are associated with these proteins.</td></tr>
<tr><td>143</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05541">PRK05541</a></td><td>176</td><td>32</td><td>89.7</td><td>0.32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05541</td><td>adenylylsulfate kinase; Provisional</td></tr>
<tr><td>144</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02173">TIGR02173</a></td><td>171</td><td>47</td><td>89.6</td><td>0.19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cyt_kin_arch</td><td>cytidylate kinase, putative. Proteins in this family are believed to be cytidylate kinase. Members of this family are found in the archaea and in spirochaetes, and differ considerably from the common bacterial form of cytidylate kinase described by TIGR00017.</td></tr>
<tr><td>145</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02028">cd02028</a></td><td>179</td><td>37</td><td>89.5</td><td>0.34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UMPK_like</td><td>Uridine monophosphate kinase_like (UMPK_like) is a family of proteins highly similar to the uridine monophosphate kinase (UMPK, EC 2.7.1.48), also known as uridine kinase or uridine-cytidine kinase (UCK).</td></tr>
<tr><td>146</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0378">COG0378</a></td><td>202</td><td>41</td><td>89.5</td><td>0.43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HypB</td><td>Ni2+-binding GTPase involved in regulation of expression and maturation of urease and hydrogenase </td></tr>
<tr><td>147</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11231">PRK11231</a></td><td>255</td><td>146</td><td>89.2</td><td>2.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fecE</td><td>iron-dicitrate transporter ATP-binding subunit; Provisional</td></tr>
<tr><td>148</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02019">cd02019</a></td><td>69</td><td>31</td><td>89.1</td><td>0.45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NK</td><td>Nucleoside/nucleotide kinase (NK) is a protein superfamily consisting of multiple families of enzymes that share structural similarity and are functionally related to the catalysis of the reversible phosphate group transfer from nucleoside triphosphates to nucleosides/nucleotides, nucleoside monophosphates, or sugars. Members of this family play a wide variety of essential roles in nucleotide metabolism, the biosynthesis of coenzymes and aromatic compounds, as well as the metabolism of sugar and sulfate.</td></tr>
<tr><td>149</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03262">cd03262</a></td><td>213</td><td>36</td><td>89.1</td><td>0.25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_HisP_GlnQ</td><td>ATP-binding cassette domain of the histidine and glutamine transporters. HisP and GlnQ are the ATP-binding components of the bacterial periplasmic histidine and glutamine permeases, respectively. Histidine permease is a multi-subunit complex containing the HisQ and HisM integral membrane subunits and two copies of HisP. HisP has properties intermediate between those of integral and peripheral membrane proteins and is accessible from both sides of the membrane, presumably by its interaction with HisQ and HisM. The two HisP subunits form a homodimer within the complex. The domain structure of the amino acid uptake systems is typical for prokaryotic extracellular solute binding protein-dependent uptake systems. All of the amino acid uptake systems also have at least one, and in a few cases, two extracellular solute binding proteins located in the periplasm of Gram-negative bacteria, or attached to the cell membrane of Gram-positive bacteria. The best-studied member of the PAAT (polar amino acid transport) family is the HisJQMP system of S. typhimurium, where HisJ is the extracellular solute binding proteins and HisP is the ABC protein.</td></tr>
<tr><td>150</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0419">COG0419</a></td><td>908</td><td>54</td><td>89.1</td><td>0.26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SbcC</td><td>DNA repair exonuclease SbcCD ATPase subunit </td></tr>
<tr><td>151</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01969">TIGR01969</a></td><td>251</td><td>38</td><td>89.0</td><td>0.27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>minD_arch</td><td>cell division ATPase MinD, archaeal. This model represents the archaeal branch of the MinD family. MinD, a weak ATPase, works in bacteria with MinC as a generalized cell division inhibitor and, through interaction with MinE, prevents septum placement inappropriate sites. Often several members of this family are found in archaeal genomes, and the function is uncharacterized. More distantly related proteins ParA chromosome partitioning proteins. The exact roles of the various archaeal MinD homologs are unknown.</td></tr>
<tr><td>152</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12402">PRK12402</a></td><td>337</td><td>34</td><td>89.0</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK12402</td><td>replication factor C small subunit 2; Reviewed</td></tr>
<tr><td>153</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00071">cd00071</a></td><td>137</td><td>16</td><td>88.7</td><td>0.21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GMPK</td><td>Guanosine monophosphate kinase (GMPK, EC 2.7.4.8), also known as guanylate kinase (GKase), catalyzes the reversible phosphoryl transfer from adenosine triphosphate (ATP) to guanosine monophosphate (GMP) to yield adenosine diphosphate (ADP) and guanosine diphosphate (GDP). It plays an essential role in the biosynthesis of guanosine triphosphate (GTP). This enzyme is also important for the activation of some antiviral and anticancer agents, such as acyclovir, ganciclovir, carbovir, and thiopurines.</td></tr>
<tr><td>154</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03116">cd03116</a></td><td>159</td><td>56</td><td>88.7</td><td>0.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MobB</td><td>Molybdenum is an essential trace element in the form of molybdenum cofactor (Moco) which is associated with the metabolism of nitrogen, carbon and sulfur by redox active enzymes. In E. coli, the synthesis of Moco involves genes from several loci: moa, mob, mod, moe and mog. The mob locus contains mobA and mobB genes. MobB catalyzes the attachment of the guanine dinucleotide to molybdopterin.</td></tr>
<tr><td>155</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1373">COG1373</a></td><td>398</td><td>32</td><td>88.7</td><td>2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1373</td><td>Predicted ATPase, AAA+ superfamily </td></tr>
<tr><td>156</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00595">TIGR00595</a></td><td>505</td><td>79</td><td>88.5</td><td>0.44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------]</td><td>priA</td><td>primosomal protein N'. All proteins in this family for which functions are known are components of the primosome which is involved in replication, repair, and recombination.This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University). </td></tr>
<tr><td>157</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03266">pfam03266</a></td><td>168</td><td>122</td><td>88.4</td><td>0.35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NTPase_1</td><td>NTPase. This domain is found across all species from bacteria to human, and the function was determined first in a hyperthermophilic bacterium to be an NTPase. The structure of one member-sequence represents a variation of the RecA fold, and implies that the function might be that of a DNA/RNA modifying enzyme. The sequence carries both a Walker A and Walker B motif which together are characteristic of ATPases or GTPases. The protein exhibits an increased expression profile in human liver cholangiocarcinoma when compared to normal tissue.</td></tr>
<tr><td>158</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14974">PRK14974</a></td><td>336</td><td>178</td><td>88.4</td><td>0.71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14974</td><td>cell division protein FtsY; Provisional</td></tr>
<tr><td>159</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2804">COG2804</a></td><td>500</td><td>48</td><td>88.3</td><td>0.56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PulE</td><td>Type II secretory pathway ATPase GspE/PulE or T4P pilus assembly pathway ATPase PilB </td></tr>
<tr><td>160</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1204">COG1204</a></td><td>766</td><td>58</td><td>88.2</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BRR2</td><td>Replicative superfamily II helicase </td></tr>
<tr><td>161</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1618">COG1618</a></td><td>179</td><td>28</td><td>88.2</td><td>0.35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>THEP1</td><td>Nucleoside-triphosphatase THEP1 </td></tr>
<tr><td>162</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00267">cd00267</a></td><td>157</td><td>33</td><td>88.2</td><td>0.22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_ATPase</td><td>ATP-binding cassette transporter nucleotide-binding domain. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide-binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>163</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00455">TIGR00455</a></td><td>184</td><td>31</td><td>88.0</td><td>0.48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>apsK</td><td>adenylyl-sulfate kinase. This protein, adenylylsulfate kinase, is often found as a fusion protein with sulfate adenylyltransferase. Important residue (active site in E.coli) is residue 100 of the seed alignment. </td></tr>
<tr><td>164</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK04182">PRK04182</a></td><td>180</td><td>37</td><td>87.9</td><td>0.32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK04182</td><td>cytidylate kinase; Provisional</td></tr>
<tr><td>165</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0283">COG0283</a></td><td>222</td><td>19</td><td>87.9</td><td>0.29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cmk</td><td>Cytidylate kinase </td></tr>
<tr><td>166</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02533">TIGR02533</a></td><td>486</td><td>75</td><td>87.8</td><td>0.68</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>type_II_gspE</td><td>type II secretion system protein E. This family describes GspE, the E protein of the type II secretion system, also called the main terminal branch of the general secretion pathway. This model separates GspE from the PilB protein of type IV pilin biosynthesis. </td></tr>
<tr><td>167</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05480">PRK05480</a></td><td>209</td><td>29</td><td>87.7</td><td>0.45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05480</td><td>uridine/cytidine kinase; Provisional</td></tr>
<tr><td>168</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0455">COG0455</a></td><td>262</td><td>220</td><td>87.7</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FlhG</td><td>MinD-like ATPase involved in chromosome partitioning or flagellar assembly </td></tr>
<tr><td>169</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01122">cd01122</a></td><td>271</td><td>49</td><td>87.6</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GP4d_helicase</td><td>GP4d_helicase is a homohexameric 5'-3' helicases. Helicases couple NTP hydrolysis to the unwinding of nucleic acid duplexes into their component strands.</td></tr>
<tr><td>170</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11784">PRK11784</a></td><td>345</td><td>30</td><td>87.5</td><td>0.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11784</td><td>tRNA 2-selenouridine synthase; Provisional</td></tr>
<tr><td>171</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1110">COG1110</a></td><td>1187</td><td>63</td><td>87.5</td><td>0.55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TopG2</td><td>Reverse gyrase  </td></tr>
<tr><td>172</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01124">cd01124</a></td><td>187</td><td>36</td><td>87.5</td><td>0.69</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KaiC</td><td>KaiC is a circadian clock protein primarily found in cyanobacteria KaiC is a RecA-like ATPase, having both Walker A and Walker B motifs. A related protein is found in archaea.</td></tr>
<tr><td>173</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1136">COG1136</a></td><td>226</td><td>156</td><td>87.4</td><td>0.29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LolD</td><td>ABC-type lipoprotein export system, ATPase component </td></tr>
<tr><td>174</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1474">COG1474</a></td><td>366</td><td>26</td><td>87.3</td><td>2.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CDC6</td><td>Cdc6-related protein, AAA superfamily ATPase  </td></tr>
<tr><td>175</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13671">pfam13671</a></td><td>143</td><td>14</td><td>87.2</td><td>0.48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_33</td><td>AAA domain. This family of domains contain only a P-loop motif, that is characteristic of the AAA superfamily. Many of the proteins in this family are just short fragments so there is no Walker B motif.</td></tr>
<tr><td>176</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00268">cd00268</a></td><td>203</td><td>52</td><td>87.2</td><td>9.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DEADc</td><td>DEAD-box helicases. A diverse family of proteins involved in ATP-dependent RNA unwinding, needed in a variety of cellular processes including splicing, ribosome biogenesis and RNA degradation. The name derives from the sequence of the Walker  B motif (motif II). This domain contains the ATP- binding region.</td></tr>
<tr><td>177</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03230">cd03230</a></td><td>173</td><td>34</td><td>86.9</td><td>0.74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_DR_subfamily_A</td><td>ATP-binding cassette domain of the drug resistance transporter and related proteins, subfamily A. This family of ATP-binding proteins belongs to a multi-subunit transporter involved in drug resistance (BcrA and DrrA), nodulation, lipid transport, and lantibiotic immunity. In bacteria and archaea, these transporters usually include an ATP-binding protein and one or two integral membrane proteins. Eukaryotic systems of the ABCA subfamily display ABC domains that are quite similar to this family. The ATP-binding domain shows the highest similarity between all members of the ABC transporter family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>178</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01054">TIGR01054</a></td><td>1171</td><td>62</td><td>86.7</td><td>0.91</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rgy</td><td>reverse gyrase. This model describes reverse gyrase, found in both archaeal and bacterial thermophiles. This enzyme, a fusion of a type I topoisomerase domain and a helicase domain, introduces positive supercoiling to increase the melting temperature of DNA double strands. Generally, these gyrases are encoded as a single polypeptide. An exception was found in Methanopyrus kandleri, where enzyme is split within the topoisomerase domain, yielding a heterodimer of gene products designated RgyB and RgyA. </td></tr>
<tr><td>179</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0125">COG0125</a></td><td>208</td><td>45</td><td>86.7</td><td>0.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Tmk</td><td>Thymidylate kinase </td></tr>
<tr><td>180</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1875">COG1875</a></td><td>436</td><td>144</td><td>86.7</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YlaK</td><td>Predicted ribonuclease YlaK, contains NYN-type RNase and PhoH-family ATPase domains  </td></tr>
<tr><td>181</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1200">COG1200</a></td><td>677</td><td>133</td><td>86.6</td><td>0.44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecG</td><td>RecG-like helicase </td></tr>
<tr><td>182</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00750">TIGR00750</a></td><td>300</td><td>46</td><td>86.5</td><td>0.81</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>lao</td><td>LAO/AO transport system ATPase. In E. coli, mutation of this kinase blocks phosphorylation of two transporter system periplasmic binding proteins and consequently inhibits those transporters. This kinase is also found in Gram-positive bacteria, archaea, and the roundworm C. elegans. It may have a more general, but still unknown function. Mutations have also been found that do not phosphorylate the periplasmic binding proteins, yet still allow transport. The ATPase activity of this protein seems to be necessary, however. </td></tr>
<tr><td>183</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02038">cd02038</a></td><td>139</td><td>38</td><td>86.5</td><td>0.71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FleN-like</td><td>FleN is a member of the Fer4_NifH superfamily. It shares the common function as an ATPase, with the ATP-binding domain at the N-terminus. In Pseudomonas aeruginosa, FleN gene is involved in regulating the number of flagella and chemotactic motility by influencing FleQ activity.</td></tr>
<tr><td>184</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11264">PRK11264</a></td><td>250</td><td>36</td><td>86.4</td><td>0.94</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11264</td><td>putative amino-acid ABC transporter ATP-binding protein YecC; Provisional</td></tr>
<tr><td>185</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02023">cd02023</a></td><td>198</td><td>29</td><td>86.3</td><td>0.62</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UMPK</td><td>Uridine monophosphate kinase (UMPK, EC 2.7.1.48), also known as uridine kinase or uridine-cytidine kinase (UCK), catalyzes the reversible phosphoryl transfer from ATP to uridine or cytidine to yield UMP or CMP. In the primidine nucleotide-salvage pathway, this enzyme combined with nucleoside diphosphate kinases further phosphorylates UMP and CMP to form UTP and CTP. This kinase also catalyzes the phosphorylation of several cytotoxic ribonucleoside analogs such as 5-flurrouridine and cyclopentenyl-cytidine.</td></tr>
<tr><td>186</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00580">TIGR00580</a></td><td>926</td><td>90</td><td>86.3</td><td>1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>mfd</td><td>transcription-repair coupling factor (mfd). All proteins in this family for which functions are known are DNA-dependent ATPases that function in the process of transcription-coupled DNA repair in which the repair of the transcribed strand of actively transcribed genes is repaired at a higher rate than the repair of non-transcribed regions of the genome and than the non-transcribed strand of the same gene. This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University). This family is closely related to the RecG and UvrB families. </td></tr>
<tr><td>187</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03268">cd03268</a></td><td>208</td><td>145</td><td>86.3</td><td>0.73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_BcrA_bacitracin_resist</td><td>ATP-binding cassette domain of the bacitracin-resistance transporter. The BcrA subfamily represents ABC transporters involved in peptide antibiotic resistance. Bacitracin is a dodecapeptide antibiotic produced by B. licheniformis and B. subtilis. The synthesis of bacitracin is non-ribosomally catalyzed by a multi-enzyme complex BcrABC. Bacitracin has potent antibiotic activity against gram-positive bacteria. The inhibition of peptidoglycan biosynthesis is the best characterized bacterial effect of bacitracin. The bacitracin resistance of B. licheniformis is mediated by the ABC transporter Bcr which is composed of two identical BcrA ATP-binding subunits and one each of the integral membrane proteins, BcrB and BcrC. B. subtilis cells carrying bcr genes on high-copy number plasmids develop collateral detergent sensitivity, a similar phenomenon in human cells with overexpressed multi-drug resistance P-glycoprotein.</td></tr>
<tr><td>188</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10818">PRK10818</a></td><td>270</td><td>36</td><td>86.1</td><td>0.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10818</td><td>cell division inhibitor MinD; Provisional</td></tr>
<tr><td>189</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5008">COG5008</a></td><td>375</td><td>85</td><td>86.1</td><td>0.93</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PilU</td><td>Tfp pilus assembly protein, ATPase PilU  </td></tr>
<tr><td>190</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09435">PRK09435</a></td><td>332</td><td>184</td><td>86.0</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09435</td><td>membrane ATPase/protein kinase; Provisional</td></tr>
<tr><td>191</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0572">COG0572</a></td><td>218</td><td>109</td><td>86.0</td><td>0.52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Udk</td><td>Uridine kinase </td></tr>
<tr><td>192</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04291">TIGR04291</a></td><td>566</td><td>46</td><td>85.9</td><td>0.84</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>arsen_driv_ArsA</td><td>arsenical pump-driving ATPase. The broader family (TIGR00345) to which the current family belongs consists of transport-energizing ATPases, including to TRC40/GET3 family involved in post-translational insertion of protein C-terminal transmembrane anchors into membranes from the cyotosolic face. This family, however, is restricted to ATPases that energize pumps that export arsenite (or antimonite).</td></tr>
<tr><td>193</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00698">PRK00698</a></td><td>205</td><td>46</td><td>85.9</td><td>0.75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tmk</td><td>thymidylate kinase; Validated</td></tr>
<tr><td>194</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01587">TIGR01587</a></td><td>359</td><td>232</td><td>85.8</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas3_core</td><td>CRISPR-associated helicase Cas3. This model represents the highly conserved core region of an alignment of Cas3, a protein found in association with CRISPR repeat elements in a broad range of bacteria and archaea. Cas3 appears to be a helicase, with regions found by pfam00270 (DEAD/DEAH box helicase) and pfam00271 (Helicase conserved C-terminal domain). Some but not all members have an N-terminal HD domain region (pfam01966) that is not included within this model.</td></tr>
<tr><td>195</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4133">COG4133</a></td><td>209</td><td>145</td><td>85.8</td><td>0.59</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CcmA</td><td>ABC-type transport system involved in cytochrome c biogenesis, ATPase component </td></tr>
<tr><td>196</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3950">COG3950</a></td><td>440</td><td>23</td><td>85.6</td><td>0.36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG3950</td><td>Predicted ATP-binding protein involved in virulence  </td></tr>
<tr><td>197</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00005">pfam00005</a></td><td>150</td><td>21</td><td>85.3</td><td>0.43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_tran</td><td>ABC transporter. ABC transporters for a large family of proteins responsible for translocation of a variety of compounds across biological membranes. ABC transporters are the largest family of proteins in many completely sequenced bacteria. ABC transporters are composed of two copies of this domain and two copies of a transmembrane domain pfam00664. These four domains may belong to a single polypeptide or belong in different polypeptide chains.</td></tr>
<tr><td>198</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03255">cd03255</a></td><td>218</td><td>37</td><td>85.2</td><td>0.35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_MJ0796_LolCDE_FtsE</td><td>ATP-binding cassette domain of the transporters involved in export of lipoprotein and macrolide, and cell division protein. This family is comprised of MJ0796 ATP-binding cassette, macrolide-specific ABC-type efflux carrier (MacAB), and proteins involved in cell division (FtsE), and release of lipoproteins from the cytoplasmic membrane (LolCDE). They are clustered together phylogenetically. MacAB is an exporter that confers resistance to macrolides, while the LolCDE system is not a transporter at all. An FtsE null mutants showed filamentous growth and appeared viable on high salt medium only, indicating a role for FtsE in cell division and/or salt transport. The LolCDE complex catalyzes the release of lipoproteins from the cytoplasmic membrane prior to their targeting to the outer membrane.</td></tr>
<tr><td>199</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01935">pfam01935</a></td><td>219</td><td>40</td><td>85.2</td><td>0.99</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF87</td><td>Domain of unknown function DUF87. The function of this prokaryotic domain is unknown. It contains several conserved aspartates and histidines that could be metal ligands.</td></tr>
<tr><td>200</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10881">pfam10881</a></td><td>127</td><td>53</td><td>85.1</td><td>1.6</td><td>[&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2726</td><td>Protein of unknown function (DUF2726). This bacterial family of proteins has no known function.</td></tr>
<tr><td>201</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03279">cd03279</a></td><td>213</td><td>116</td><td>85.0</td><td>0.59</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_sbcCD</td><td>ATP-binding cassette domain of sbcCD. SbcCD and other Mre11/Rad50 (MR) complexes are implicated in the metabolism of DNA ends. They cleave ends sealed by hairpin structures and are thought to play a role in removing protein bound to DNA termini.</td></tr>
<tr><td>202</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01583">pfam01583</a></td><td>157</td><td>31</td><td>85.0</td><td>0.95</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>APS_kinase</td><td>Adenylylsulphate kinase. Enzyme that catalyses the phosphorylation of adenylylsulphate to 3'-phosphoadenylylsulfate. This domain contains an ATP binding P-loop motif.</td></tr>
<tr><td>203</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02463">pfam02463</a></td><td>1162</td><td>25</td><td>84.7</td><td>0.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SMC_N</td><td>RecF/RecN/SMC N terminal domain. This domain is found at the N terminus of SMC proteins. The SMC (structural maintenance of chromosomes) superfamily proteins have ATP-binding domains at the N- and C-termini, and two extended coiled-coil domains separated by a hinge in the middle. The eukaryotic SMC proteins form two kind of heterodimers: the SMC1/SMC3 and the SMC2/SMC4 types. These heterodimers constitute an essential part of higher order complexes, which are involved in chromatin and DNA dynamics.   This family also includes the RecF and RecN proteins that are involved in DNA metabolizm and recombination.</td></tr>
<tr><td>204</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3265">COG3265</a></td><td>161</td><td>19</td><td>84.6</td><td>0.47</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GntK</td><td>Gluconate kinase </td></tr>
<tr><td>205</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09270">PRK09270</a></td><td>229</td><td>29</td><td>84.3</td><td>0.61</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09270</td><td>nucleoside triphosphate hydrolase domain-containing protein; Reviewed</td></tr>
<tr><td>206</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13548">PRK13548</a></td><td>258</td><td>49</td><td>84.1</td><td>0.36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>hmuV</td><td>hemin importer ATP-binding subunit; Provisional</td></tr>
<tr><td>207</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07724">pfam07724</a></td><td>168</td><td>29</td><td>84.1</td><td>4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_2</td><td>AAA domain (Cdc48 subfamily). This Pfam entry includes some of the AAA proteins not detected by the pfam00004 model.</td></tr>
<tr><td>208</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1100">COG1100</a></td><td>219</td><td>27</td><td>84.0</td><td>0.55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Gem1</td><td>GTPase SAR1 family domain </td></tr>
<tr><td>209</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03228">cd03228</a></td><td>171</td><td>34</td><td>83.9</td><td>0.48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCC_MRP_Like</td><td>ATP-binding cassette domain of multidrug resistance protein-like transporters. The MRP (Multidrug Resistance Protein)-like transporters are involved in drug, peptide, and lipid export. They belong to the subfamily C of the ATP-binding cassette (ABC) superfamily of transport proteins. The ABCC subfamily contains transporters with a diverse functional spectrum that includes ion transport, cell surface receptor, and toxin secretion activities. The MRP-like family, similar to all ABC proteins, have a common four-domain core structure constituted by two membrane-spanning domains, each composed of six transmembrane (TM) helices, and two nucleotide-binding domains (NBD). ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>210</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00910">pfam00910</a></td><td>105</td><td>24</td><td>83.9</td><td>0.83</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNA_helicase</td><td>RNA helicase. This family includes RNA helicases thought to be involved in duplex unwinding during viral RNA replication. Members of this family are found in a variety of single stranded RNA viruses.</td></tr>
<tr><td>211</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1199">COG1199</a></td><td>654</td><td>117</td><td>83.8</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------]</td><td>DinG</td><td>Rad3-related DNA helicase </td></tr>
<tr><td>212</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13869">PRK13869</a></td><td>405</td><td>29</td><td>83.7</td><td>0.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13869</td><td>plasmid-partitioning protein RepA; Provisional</td></tr>
<tr><td>213</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03225">cd03225</a></td><td>211</td><td>41</td><td>83.7</td><td>0.55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_cobalt_CbiO_domain1</td><td>First domain of the ATP-binding cassette component of cobalt transport system. Domain I of the ABC component of a cobalt transport family found in bacteria, archaea, and eukaryota. The transition metal cobalt is an essential component of many enzymes and must be transported into cells in appropriate amounts when needed. This ABC transport system of the CbiMNQO family is involved in cobalt transport in association with the cobalamin (vitamin B12) biosynthetic pathways. Most of cobalt (Cbi) transport systems possess a separate CbiN component, the cobalt-binding periplasmic protein, and they are encoded by the conserved gene cluster cbiMNQO. Both the CbiM and CbiQ proteins are integral cytoplasmic membrane proteins, and the CbiO protein has the linker peptide and the Walker A and B motifs commonly found in the ATPase components of the ABC-type transport systems.</td></tr>
<tr><td>214</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00023">PRK00023</a></td><td>225</td><td>45</td><td>83.7</td><td>0.63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cmk</td><td>cytidylate kinase; Provisional</td></tr>
<tr><td>215</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0514">COG0514</a></td><td>590</td><td>153</td><td>83.6</td><td>9.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecQ</td><td>Superfamily II DNA helicase RecQ </td></tr>
<tr><td>216</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05703">PRK05703</a></td><td>424</td><td>34</td><td>83.4</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>flhF</td><td>flagellar biosynthesis regulator FlhF; Validated</td></tr>
<tr><td>217</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1122">COG1122</a></td><td>235</td><td>168</td><td>83.4</td><td>0.29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>EcfA2</td><td>Energy-coupling factor transporter ATP-binding protein EcfA2 </td></tr>
<tr><td>218</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10236">pfam10236</a></td><td>293</td><td>34</td><td>83.4</td><td>0.91</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DAP3</td><td>Mitochondrial ribosomal death-associated protein 3. This is a family of conserved proteins which were originally described as death-associated-protein-3 (DAP-3). The proteins carry a P-loop DNA-binding motif, and induce apoptosis. DAP3 has been shown to be a pro-apoptotic factor in the mitochondrial matrix and to be crucial for mitochondrial biogenesis and so has also been designated as MRP-S29 (mitochondrial ribosomal protein subunit 29).</td></tr>
<tr><td>219</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03420">TIGR03420</a></td><td>226</td><td>37</td><td>83.3</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DnaA_homol_Hda</td><td>DnaA regulatory inactivator Hda. Members of this protein family are Hda (Homologous to DnaA). These proteins are about half the length of DnaA and homologous over length of Hda. In the model species Escherichia coli, the initiation of DNA replication requires DnaA bound to ATP rather than ADP; Hda helps facilitate the conversion of DnaA-ATP to DnaA-ADP. </td></tr>
<tr><td>220</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3839">COG3839</a></td><td>338</td><td>49</td><td>83.3</td><td>0.59</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MalK</td><td>ABC-type sugar transport system, ATPase component </td></tr>
<tr><td>221</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0444">COG0444</a></td><td>316</td><td>146</td><td>83.2</td><td>6.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DppD</td><td>ABC-type dipeptide/oligopeptide/nickel transport system, ATPase component </td></tr>
<tr><td>222</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10867">PRK10867</a></td><td>433</td><td>128</td><td>83.2</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10867</td><td>signal recognition particle protein; Provisional</td></tr>
<tr><td>223</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03167">TIGR03167</a></td><td>311</td><td>27</td><td>83.1</td><td>0.89</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tRNA_sel_U_synt</td><td>tRNA 2-selenouridine synthase. The Escherichia coli YbbB protein was shown to encode a selenophosphate-dependent tRNA 2-selenouridine synthase, essential for modification of some tRNAs to replace a sulfur atom with selenium. This enzyme works with SelD, the selenium donor protein, which also acts in selenocysteine incorporation. Although the members of this protein family show a fairly deep split, sequences from both sides of the split are supported by co-occurence with, and often proximity to, the selD gene. </td></tr>
<tr><td>224</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06851">PRK06851</a></td><td>367</td><td>32</td><td>82.9</td><td>1.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06851</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>225</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03574">TIGR03574</a></td><td>249</td><td>32</td><td>82.8</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>selen_PSTK</td><td>L-seryl-tRNA(Sec) kinase, archaeal. Members of this protein are L-seryl-tRNA(Sec) kinase. This enzyme is part of a two-step pathway in Eukaryota and Archaea for performing selenocysteine biosynthesis by changing serine misacylated on selenocysteine-tRNA to selenocysteine. This enzyme performs the first step, phosphorylation of the OH group of the serine side chain. This family represents archaeal proteins with this activity. </td></tr>
<tr><td>226</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2894">COG2894</a></td><td>272</td><td>111</td><td>82.8</td><td>1.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MinD</td><td>Septum formation inhibitor-activating ATPase MinD </td></tr>
<tr><td>227</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1102">COG1102</a></td><td>179</td><td>36</td><td>82.6</td><td>0.81</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CmkB</td><td>Cytidylate kinase  </td></tr>
<tr><td>228</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02492">pfam02492</a></td><td>178</td><td>59</td><td>82.4</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cobW</td><td>CobW/HypB/UreG, nucleotide-binding domain. This domain is found in HypB, a hydrogenase expression / formation protein, and UreG a urease accessory protein. Both these proteins contain a P-loop nucleotide binding motif. HypB has GTPase activity and is a guanine nucleotide binding protein. It is not known whether UreG binds GTP or some other nucleotide. Both enzymes are involved in nickel binding. HypB can store nickel and is required for nickel dependent hydrogenase expression. UreG is required for functional incorporation of the urease nickel metallocenter. GTP hydrolysis may required by these proteins for nickel incorporation into other nickel proteins. This family of domains also contains P47K, a Pseudomonas chlororaphis protein needed for nitrile hydratase expression, and the cobW gene product, which may be involved in cobalamin biosynthesis in Pseudomonas denitrificans.</td></tr>
<tr><td>229</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1120">COG1120</a></td><td>258</td><td>143</td><td>82.2</td><td>0.68</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FepC</td><td>ABC-type cobalamin/Fe3+-siderophores transport system, ATPase component </td></tr>
<tr><td>230</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03205">pfam03205</a></td><td>138</td><td>30</td><td>82.2</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MobB</td><td>Molybdopterin guanine dinucleotide synthesis protein B. This protein contains a P-loop.</td></tr>
<tr><td>231</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1763">COG1763</a></td><td>161</td><td>32</td><td>82.2</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MobB</td><td>Molybdopterin-guanine dinucleotide biosynthesis protein </td></tr>
<tr><td>232</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13539">PRK13539</a></td><td>207</td><td>40</td><td>82.1</td><td>0.61</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13539</td><td>cytochrome c biogenesis protein CcmA; Provisional</td></tr>
<tr><td>233</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00176">pfam00176</a></td><td>286</td><td>56</td><td>82.1</td><td>2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SNF2_N</td><td>SNF2 family N-terminal domain. This domain is found in proteins involved in a variety of processes including transcription regulation (e.g., SNF2, STH1, brahma, MOT1), DNA repair (e.g., ERCC6, RAD16, RAD5), DNA recombination (e.g., RAD54), and chromatin unwinding (e.g., ISWI) as well as a variety of other proteins with little functional information (e.g., lodestar, ETL1).</td></tr>
<tr><td>234</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1125">COG1125</a></td><td>309</td><td>184</td><td>82.1</td><td>0.93</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>OpuBA</td><td>ABC-type proline/glycine betaine transport system, ATPase component </td></tr>
<tr><td>235</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06067">PRK06067</a></td><td>234</td><td>133</td><td>81.9</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06067</td><td>flagellar accessory protein FlaH; Validated</td></tr>
<tr><td>236</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02117">cd02117</a></td><td>212</td><td>186</td><td>81.8</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NifH_like</td><td>This family contains the NifH (iron protein) of nitrogenase, L subunit (BchL/ChlL) of the  protochlorophyllide reductase and the BchX subunit of the Chlorophyllide reductase. Members of this family use energey from ATP hydrolysis and transfer electrons through a Fe4-S4 cluster to other subunit for reduction of substrate.</td></tr>
<tr><td>237</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03600">TIGR03600</a></td><td>420</td><td>109</td><td>81.8</td><td>3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>phage_DnaB</td><td>phage replicative helicase, DnaB family, HK022 subfamily. Members of this family are phage (or prophage-region) homologs of the bacterial homohexameric replicative helicase DnaB. Some phage may rely on host DnaB, while others encode their own verions. This model describes the largest phage-specific clade among the close homologs of DnaB, but there are, or course, other DnaB homologs from phage that fall outside the scope of this model. </td></tr>
<tr><td>238</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13830">PRK13830</a></td><td>818</td><td>64</td><td>81.7</td><td>0.75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13830</td><td>conjugal transfer protein TrbE; Provisional</td></tr>
<tr><td>239</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00437">pfam00437</a></td><td>273</td><td>33</td><td>81.7</td><td>2.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>T2SE</td><td>Type II/IV secretion system protein. This family contains both type II and type IV pathway secretion proteins from bacteria. VirB11 ATPase is a subunit of the Agrobacterium tumefaciens transfer DNA (T-DNA) transfer system, a type IV secretion pathway required for delivery of T-DNA and effector proteins to plant cells during infection.</td></tr>
<tr><td>240</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02169">TIGR02169</a></td><td>1164</td><td>20</td><td>81.6</td><td>0.74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SMC_prok_A</td><td>chromosome segregation protein SMC, primarily archaeal type. SMC (structural maintenance of chromosomes) proteins bind DNA and act in organizing and segregating chromosomes for partition. SMC proteins are found in bacteria, archaea, and eukaryotes. It is found in a single copy and is homodimeric in prokaryotes, but six paralogs (excluded from this family) are found in eukarotes, where SMC proteins are heterodimeric. This family represents the SMC protein of archaea and a few bacteria (Aquifex, Synechocystis, etc); the SMC of other bacteria is described by TIGR02168. The N- and C-terminal domains of this protein are well conserved, but the central hinge region is skewed in composition and highly divergent. </td></tr>
<tr><td>241</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2256">COG2256</a></td><td>436</td><td>161</td><td>81.4</td><td>2.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RarA</td><td>Replication-associated recombination protein RarA (DNA-dependent ATPase) </td></tr>
<tr><td>242</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09493">PRK09493</a></td><td>240</td><td>43</td><td>81.4</td><td>0.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>glnQ</td><td>glutamine ABC transporter ATP-binding protein; Reviewed</td></tr>
<tr><td>243</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03220">cd03220</a></td><td>224</td><td>35</td><td>81.3</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_KpsT_Wzt</td><td>ATP-binding cassette component of polysaccharide transport system. The KpsT/Wzt ABC transporter subfamily is involved in extracellular polysaccharide export. Among the variety of membrane-linked or extracellular polysaccharides excreted by bacteria, only capsular polysaccharides, lipopolysaccharides, and teichoic acids have been shown to be exported by ABC transporters. A typical system is made of a conserved integral membrane and an ABC. In addition to these proteins, capsular polysaccharide exporter systems require two 'accessory' proteins to perform their function: a periplasmic (E.coli) or a lipid-anchored outer membrane protein called OMA (Neisseria meningitidis and Haemophilus influenza) and a cytoplasmic membrane protein MPA2.</td></tr>
<tr><td>244</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02033">cd02033</a></td><td>329</td><td>203</td><td>81.3</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BchX</td><td>Chlorophyllide reductase converts chlorophylls into bacteriochlorophylls by reducing the chlorin B-ring. This family contains the X subunit of this three-subunit enzyme. Sequence and structure similarity between bchX, protochlorophyllide reductase L subunit (bchL and chlL) and nitrogenase Fe protein (nifH gene) suggest their functional similarity. Members of the BchX family serve as the unique electron donors to their respective catalytic subunits (bchN-bchB, bchY-bchZ and nitrogenase component 1). Mechanistically, they hydrolyze ATP and transfer electrons through a Fe4-S4 cluster.</td></tr>
<tr><td>245</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03114">cd03114</a></td><td>148</td><td>37</td><td>81.2</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArgK-like</td><td>The function of this protein family is unkown. The protein sequences are similar to the ArgK protein in E. coli. ArgK protein is a membrane ATPase which is required for transporting arginine, ornithine and lysine into the cells by the arginine and ornithine (AO system) and lysine, arginine and ornithine (LAO) transport systems.</td></tr>
<tr><td>246</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04291">TIGR04291</a></td><td>566</td><td>74</td><td>80.9</td><td>3.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>arsen_driv_ArsA</td><td>arsenical pump-driving ATPase. The broader family (TIGR00345) to which the current family belongs consists of transport-energizing ATPases, including to TRC40/GET3 family involved in post-translational insertion of protein C-terminal transmembrane anchors into membranes from the cyotosolic face. This family, however, is restricted to ATPases that energize pumps that export arsenite (or antimonite).</td></tr>
<tr><td>247</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02524">TIGR02524</a></td><td>358</td><td>120</td><td>80.9</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>dot_icm_DotB</td><td>Dot/Icm secretion system ATPase DotB. Members of this protein family are the DotB component of Dot/Icm secretion systems, as found in obligate intracellular pathogens Legionella pneumophila and Coxiella burnetii. While this system resembles type IV secretion systems and has been called a form of type IV, the liturature now seems to favor calling this the Dot/Icm system. This family is most closely related to TraJ proteins of plasmid transfer, rather than to proteins of other type IV secretion systems.</td></tr>
<tr><td>248</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13342">PRK13342</a></td><td>413</td><td>68</td><td>80.9</td><td>2.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13342</td><td>recombination factor protein RarA; Reviewed</td></tr>
<tr><td>249</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03229">cd03229</a></td><td>178</td><td>17</td><td>80.8</td><td>0.74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Class3</td><td>ATP-binding cassette domain of the binding protein-dependent transport systems. This class is comprised of all BPD (Binding Protein Dependent) systems that are largely represented in archaea and eubacteria and are primarily involved in scavenging solutes from the environment. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>250</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03259">cd03259</a></td><td>213</td><td>20</td><td>80.8</td><td>0.73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Carb_Solutes_like</td><td>ATP-binding cassette domain of the carbohydrate and solute transporters-like. This family is comprised of proteins involved in the transport of apparently unrelated solutes and proteins specific for di- and oligosaccharides and polyols. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides and more complex organic molecules. The nucleotide-binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>251</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00265">pfam00265</a></td><td>175</td><td>36</td><td>80.7</td><td>2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TK</td><td>Thymidine kinase. </td></tr>
<tr><td>252</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4778">COG4778</a></td><td>235</td><td>49</td><td>80.7</td><td>0.73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PhnL</td><td>Alpha-D-ribose 1-methylphosphonate 5-triphosphate synthase subunit PhnL </td></tr>
<tr><td>253</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00955">TIGR00955</a></td><td>617</td><td>184</td><td>80.7</td><td>3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3a01204</td><td>The Eye Pigment Precursor Transporter (EPP) Family protein. </td></tr>
<tr><td>254</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03005">TIGR03005</a></td><td>252</td><td>21</td><td>80.5</td><td>0.75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ectoine_ehuA</td><td>ectoine/hydroxyectoine ABC transporter, ATP-binding protein. Members of this family are the ATP-binding protein of a conserved four gene ABC transporter operon found next to ectoine unilization operons and ectoine biosynthesis operons. Ectoine is a compatible solute that protects enzymes from high osmolarity. It is released by some species in response to hypoosmotic shock, and it is taken up by a number of bacteria as a compatible solute or for consumption. This family shows strong sequence similiarity to a number of amino acid ABC transporter ATP-binding proteins.</td></tr>
<tr><td>255</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00142">pfam00142</a></td><td>269</td><td>187</td><td>80.4</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fer4_NifH</td><td>4Fe-4S iron sulfur cluster binding proteins, NifH/frxC family. </td></tr>
<tr><td>256</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03301">cd03301</a></td><td>213</td><td>89</td><td>80.3</td><td>0.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_MalK_N</td><td>The N-terminal ATPase domain of the maltose transporter, MalK. ATP binding cassette (ABC) proteins function from bacteria to human, mediating the translocation of substances into and out of cells or organelles. ABC transporters contain two transmembrane-spanning domains (TMDs) or subunits and two nucleotide binding domains (NBDs) or subunits that couple transport to the hydrolysis of ATP. In the maltose transport system, the periplasmic maltose binding protein (MBP) stimulates the ATPase activity of the membrane-associated transporter, which consists of two transmembrane subunits, MalF and MalG, and two copies of the ATP binding subunit, MalK, and becomes tightly bound to the transporter in the catalytic transition state, ensuring that maltose is passed to the transporter as ATP is hydrolyzed.</td></tr>
<tr><td>257</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02324">TIGR02324</a></td><td>224</td><td>17</td><td>80.3</td><td>0.78</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CP_lyasePhnL</td><td>phosphonate C-P lyase system protein PhnL. Members of this family are the PhnL protein of C-P lyase systems for utilization of phosphonates. These systems resemble phosphonatase-based systems in having a three component ABC transporter, where TIGR01097 is the permease, TIGR01098 is the phosphonates binding protein, and TIGR02315 is the ATP-binding cassette (ABC) protein. They differ, however, in having, typically, ten or more additional genes, many of which are believed to form a membrane-associated C-P lysase complex. This protein (PhnL) and the adjacent-encoded PhnK (TIGR02323) resemble transporter ATP-binding proteins but are suggested, based on mutatgenesis studies, to be part of this C-P lyase complex rather than part of a transporter per se.</td></tr>
<tr><td>258</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1702">COG1702</a></td><td>348</td><td>166</td><td>80.2</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PhoH</td><td>Phosphate starvation-inducible protein PhoH, predicted ATPase </td></tr>
<tr><td>259</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02525">TIGR02525</a></td><td>372</td><td>37</td><td>80.1</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>plasmid_TraJ</td><td>plasmid transfer ATPase TraJ. Members of this protein family are predicted ATPases associated with plasmid transfer loci in bacteria. This family is most similar to the DotB ATPase of a type-IV secretion-like system of obligate intracellular pathogens Legionella pneumophila and Coxiella burnetii (TIGR02524). </td></tr>
<tr><td>260</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00041">TIGR00041</a></td><td>195</td><td>46</td><td>79.9</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DTMP_kinase</td><td>dTMP kinase. Function: phosphorylation of DTMP to form DTDP in both de novo and salvage pathways of DTTP synthesis. Catalytic activity: ATP + thymidine 5'-phosphate = ADP + thymidine 5'-diphosphate. </td></tr>
<tr><td>261</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3842">COG3842</a></td><td>352</td><td>46</td><td>79.7</td><td>0.76</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PotA</td><td>ABC-type Fe3+/spermidine/putrescine transport systems, ATPase components </td></tr>
<tr><td>262</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03695">PRK03695</a></td><td>248</td><td>24</td><td>79.7</td><td>0.92</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK03695</td><td>vitamin B12-transporter ATPase; Provisional</td></tr>
<tr><td>263</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02223">pfam02223</a></td><td>186</td><td>42</td><td>79.2</td><td>2.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Thymidylate_kin</td><td>Thymidylate kinase. </td></tr>
<tr><td>264</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03299">cd03299</a></td><td>235</td><td>47</td><td>79.1</td><td>0.98</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_ModC_like</td><td>ATP-binding cassette domain similar to the molybdate transporter. Archaeal protein closely related to ModC. ModC is an ABC-type transporter and the ATPase component of a molybdate transport system that also includes the periplasmic binding protein ModA and the membrane protein ModB. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>265</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06696">PRK06696</a></td><td>223</td><td>41</td><td>78.9</td><td>2.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06696</td><td>uridine kinase; Validated</td></tr>
<tr><td>266</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03231">cd03231</a></td><td>201</td><td>21</td><td>78.9</td><td>0.93</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_CcmA_heme_exporter</td><td>Cytochrome c biogenesis ATP-binding export protein. CcmA, the ATP-binding component of the bacterial CcmAB transporter. The CCM family is involved in bacterial cytochrome c biogenesis. Cytochrome c maturation in E. coli requires the ccm operon, which encodes eight membrane proteins (CcmABCDEFGH). CcmE is a periplasmic heme chaperon that binds heme covalently and transfers it onto apocytochrome c in the presence of CcmF, CcmG, and CcmH. The CcmAB proteins represent an ABC transporter and the CcmCD proteins participate in heme transfer to CcmE.</td></tr>
<tr><td>267</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13477">PRK13477</a></td><td>512</td><td>21</td><td>78.9</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13477</td><td>bifunctional pantoate ligase/cytidylate kinase; Provisional</td></tr>
<tr><td>268</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03238">cd03238</a></td><td>176</td><td>37</td><td>78.6</td><td>0.93</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_UvrA</td><td>ATP-binding cassette domain of the excision repair protein UvrA. Nucleotide excision repair in eubacteria is a process that repairs DNA damage by the removal of a 12-13-mer oligonucleotide containing the lesion. Recognition and cleavage of the damaged DNA is a multistep ATP-dependent reaction that requires the UvrA, UvrB, and UvrC proteins. Both UvrA and UvrB are ATPases, with UvrA having two ATP binding sites, which have the characteristic signature of the family of ABC proteins, and UvrB having one ATP binding site that is structurally related to that of helicases.</td></tr>
<tr><td>269</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4088">COG4088</a></td><td>261</td><td>32</td><td>78.6</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Kti12</td><td>tRNA Uridine 5-carbamoylmethylation protein Kti12 (Killer toxin insensitivity protein) </td></tr>
<tr><td>270</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07667">PRK07667</a></td><td>193</td><td>31</td><td>78.0</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07667</td><td>uridine kinase; Provisional</td></tr>
<tr><td>271</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4185">COG4185</a></td><td>187</td><td>31</td><td>77.9</td><td>1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4185</td><td>Predicted ABC-type ATPase </td></tr>
<tr><td>272</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00017">TIGR00017</a></td><td>217</td><td>20</td><td>77.8</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cmk</td><td>cytidylate kinase. This family consists of cytidylate kinase, which catalyzes the phosphorylation of cytidine 5-monophosphate (dCMP) to cytidine 5 -diphosphate (dCDP) in the presence of ATP or GTP. UMP and dCMP can also act as acceptors. </td></tr>
<tr><td>273</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03217">cd03217</a></td><td>200</td><td>43</td><td>77.7</td><td>1.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_FeS_Assembly</td><td>ABC-type transport system involved in Fe-S cluster assembly, ATPase component. Biosynthesis of iron-sulfur clusters (Fe-S) depends on multi-protein systems. The SUF system of E. coli and Erwinia chrysanthemi is important for Fe-S biogenesis under stressful conditions. The SUF system is made of six proteins: SufC is an atypical cytoplasmic ABC-ATPase, which forms a complex with SufB and SufD; SufA plays the role of a scaffold protein for assembly of iron-sulfur clusters and delivery to target proteins; SufS is a cysteine desulfurase which mobilizes the sulfur atom from cysteine and provides it to the cluster; SufE has no associated function yet.</td></tr>
<tr><td>274</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11823">PRK11823</a></td><td>446</td><td>57</td><td>77.6</td><td>4.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11823</td><td>DNA repair protein RadA; Provisional</td></tr>
<tr><td>275</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0194">COG0194</a></td><td>191</td><td>26</td><td>77.4</td><td>1.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Gmk</td><td>Guanylate kinase </td></tr>
<tr><td>276</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4240">COG4240</a></td><td>300</td><td>32</td><td>77.0</td><td>2.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Tda10</td><td>Pantothenate kinase-related protein Tda10 (topoisomerase I damage affected protein) </td></tr>
<tr><td>277</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1119">COG1119</a></td><td>257</td><td>33</td><td>77.0</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ModF</td><td>ABC-type molybdenum transport system, ATPase component/photorepair protein PhrA </td></tr>
<tr><td>278</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03253">cd03253</a></td><td>236</td><td>19</td><td>76.8</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCC_ATM1_transporter</td><td>ATP-binding cassette domain of iron-sulfur clusters transporter, subfamily C. ATM1 is an ABC transporter that is expressed in the mitochondria. Although the specific function of ATM1 is unknown, its disruption results in the accumulation of excess mitochondrial iron, loss of mitochondrial cytochromes, oxidative damage to mitochondrial DNA, and decreased levels of cytosolic heme proteins. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>279</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01428">cd01428</a></td><td>194</td><td>35</td><td>76.6</td><td>1.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ADK</td><td>Adenylate kinase (ADK) catalyzes the reversible phosphoryl transfer from adenosine triphosphates (ATP) to adenosine monophosphates (AMP) and to yield adenosine diphosphates (ADP). This enzyme is required for the biosynthesis of ADP and is essential for homeostasis of adenosine phosphates.</td></tr>
<tr><td>280</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0237">COG0237</a></td><td>201</td><td>85</td><td>76.6</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CoaE</td><td>Dephospho-CoA kinase </td></tr>
<tr><td>281</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02572">pfam02572</a></td><td>172</td><td>39</td><td>76.6</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CobA_CobO_BtuR</td><td>ATP:corrinoid adenosyltransferase BtuR/CobO/CobP. This family consists of the BtuR, CobO, CobP proteins all of which are Cob(I)alamin adenosyltransferase, EC:2.5.1.17, involved in cobalamin (vitamin B12) biosynthesis. These enzymes catalyse the adenosylation reaction: ATP + cob(I)alamin + H2O <=> phosphate + diphosphate + adenosylcobalamin.</td></tr>
<tr><td>282</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03276">cd03276</a></td><td>198</td><td>28</td><td>76.4</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC6_euk</td><td>ATP-binding cassette domain of eukaryotic SM6 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>283</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02640">TIGR02640</a></td><td>262</td><td>217</td><td>76.4</td><td>2.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gas_vesic_GvpN</td><td>gas vesicle protein GvpN. Members of this family are the GvpN protein associated with the production of gas vesicles produced in some prokaryotes to give cells buoyancy. This family belongs to a larger family of ATPases (pfam07728). </td></tr>
<tr><td>284</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04095">TIGR04095</a></td><td>451</td><td>69</td><td>76.2</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>dnd_restrict_1</td><td>DNA phosphorothioation system restriction enzyme. The DNA phosphorothioate modification system dnd (DNA instability during electrophoresis) recently has been shown to provide a modification essential to a restriction system. This protein family was detected by Partial Phylogenetic Profiling as linked to dnd, and its members usually are clustered with the dndABCDE genes.</td></tr>
<tr><td>285</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03278">cd03278</a></td><td>197</td><td>23</td><td>76.2</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC_barmotin</td><td>ATP-binding cassette domain of barmotin, a member of the SMC protein family. Barmotin is a tight junction-associated protein expressed in rat epithelial cells which is thought to have an important regulatory role in tight junction barrier function. Barmotin belongs to the SMC protein family. SMC proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>286</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03260">cd03260</a></td><td>227</td><td>21</td><td>76.1</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_PstB_phosphate_transporter</td><td>ATP-binding cassette domain of the phosphate transport system. Phosphate uptake is of fundamental importance in the cell physiology of bacteria because phosphate is required as a nutrient. The Pst system of E. coli comprises four distinct subunits encoded by the pstS, pstA, pstB, and pstC genes. The PstS protein is a phosphate-binding protein located in the periplasmic space. PstA and PstC are hydrophobic and they form the transmembrane portion of the Pst system. PstB is the catalytic subunit, which couples the energy of ATP hydrolysis to the import of phosphate across cellular membranes through the Pst system, often referred as ABC-protein. PstB belongs to one of the largest superfamilies of proteins characterized by a highly conserved adenosine triphosphate (ATP) binding cassette (ABC), which is also a nucleotide binding domain (NBD).</td></tr>
<tr><td>287</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14247">PRK14247</a></td><td>250</td><td>25</td><td>76.0</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14247</td><td>phosphate ABC transporter ATP-binding protein; Provisional</td></tr>
<tr><td>288</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1196">COG1196</a></td><td>1163</td><td>27</td><td>75.9</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Smc</td><td>Chromosome segregation ATPase </td></tr>
<tr><td>289</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1123">COG1123</a></td><td>539</td><td>146</td><td>75.9</td><td>9.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GsiA</td><td>ABC-type glutathione transport system ATPase component, contains duplicated ATPase domain </td></tr>
<tr><td>290</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00603">TIGR00603</a></td><td>732</td><td>167</td><td>75.7</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rad25</td><td>DNA repair helicase rad25. All proteins in this family for which functions are known are DNA-DNA helicases used for the initiation of nucleotide excision repair and transacription as part of the TFIIH complex.This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University). </td></tr>
<tr><td>291</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4559">COG4559</a></td><td>259</td><td>141</td><td>75.3</td><td>1.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4559</td><td>ABC-type hemin transport system, ATPase component  </td></tr>
<tr><td>292</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13230">PRK13230</a></td><td>279</td><td>52</td><td>75.3</td><td>2.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13230</td><td>nitrogenase reductase-like protein; Reviewed</td></tr>
<tr><td>293</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1205">COG1205</a></td><td>851</td><td>134</td><td>75.2</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YprA</td><td>ATP-dependent helicase YprA,  contains C-terminal metal-binding DUF1998 domain </td></tr>
<tr><td>294</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02315">TIGR02315</a></td><td>243</td><td>173</td><td>75.2</td><td>2.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_phnC</td><td>phosphonate ABC transporter, ATP-binding protein. Phosphonates are a class of phosphorus-containing organic compound with a stable direct C-P bond rather than a C-O-P linkage. A number of bacterial species have operons, typically about 14 genes in size, with genes for ATP-dependent transport of phosphonates, degradation, and regulation of the expression of the system. Members of this protein family are the ATP-binding cassette component of tripartite ABC transporters of phosphonates. </td></tr>
<tr><td>295</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03411">TIGR03411</a></td><td>242</td><td>19</td><td>75.0</td><td>1.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>urea_trans_UrtD</td><td>urea ABC transporter, ATP-binding protein UrtD. Members of this protein family are ABC transporter ATP-binding subunits associated with urea transport and metabolism. This protein is found in a conserved five-gene transport operon typically found adjacent to urease genes. It was shown in Cyanobacteria that disruption leads to the loss of high-affinity urea transport activity. </td></tr>
<tr><td>296</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00300">PRK00300</a></td><td>205</td><td>28</td><td>75.0</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gmk</td><td>guanylate kinase; Provisional</td></tr>
<tr><td>297</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1121">COG1121</a></td><td>254</td><td>36</td><td>75.0</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ZnuC</td><td>ABC-type Mn2+/Zn2+ transport system, ATPase component </td></tr>
<tr><td>298</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03846">PRK03846</a></td><td>198</td><td>31</td><td>74.8</td><td>2.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK03846</td><td>adenylylsulfate kinase; Provisional</td></tr>
<tr><td>299</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03239">cd03239</a></td><td>178</td><td>19</td><td>74.8</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC_head</td><td>The SMC head domain belongs to the ATP-binding cassette superfamily. The structural maintenance of chromosomes (SMC) proteins are essential for successful chromosome transmission during replication and segregation of the genome in all organisms. SMCs are generally present as single proteins in bacteria, and as at least six distinct proteins in eukaryotes. The proteins range in size from approximately 110 to 170 kDa, and each has five distinct domains: amino- and carboxy-terminal globular domains, which contain sequences characteristic of ATPases, two coiled-coil regions separating the terminal domains , and a central flexible hinge. SMC proteins function together with other proteins in a range of chromosomal transactions, including chromosome condensation, sister-chromatid cohesion, recombination, DNA repair, and epigenetic silencing of gene expression.</td></tr>
<tr><td>300</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0563">COG0563</a></td><td>178</td><td>20</td><td>74.7</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Adk</td><td>Adenylate kinase or related kinase </td></tr>
<tr><td>301</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5635">COG5635</a></td><td>824</td><td>28</td><td>74.7</td><td>4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG5635</td><td>Predicted NTPase, NACHT family domain </td></tr>
<tr><td>302</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13538">PRK13538</a></td><td>204</td><td>17</td><td>74.7</td><td>1.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13538</td><td>cytochrome c biogenesis protein CcmA; Provisional</td></tr>
<tr><td>303</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02022">cd02022</a></td><td>179</td><td>108</td><td>74.6</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DPCK</td><td>Dephospho-coenzyme A kinase (DPCK, EC 2.7.1.24) catalyzes the phosphorylation of dephosphocoenzyme A (dCoA) to yield CoA, which is the final step in CoA biosynthesis.</td></tr>
<tr><td>304</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10247">PRK10247</a></td><td>225</td><td>55</td><td>74.5</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10247</td><td>putative ABC transporter ATP-binding protein YbbL; Provisional</td></tr>
<tr><td>305</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0593">COG0593</a></td><td>408</td><td>44</td><td>74.5</td><td>3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DnaA</td><td>Chromosomal replication initiation ATPase DnaA </td></tr>
<tr><td>306</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03224">cd03224</a></td><td>222</td><td>22</td><td>74.2</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_TM1139_LivF_branched</td><td>ATP-binding cassette domain of branched-chain amino acid transporter. LivF (TM1139) is part of the LIV-I bacterial ABC-type two-component transport system that imports neutral, branched-chain amino acids. The E. coli branched-chain amino acid transporter comprises a heterodimer of ABC transporters (LivF and LivG), a heterodimer of six-helix TM domains (LivM and LivH), and one of two alternative soluble periplasmic substrate binding proteins (LivK or LivJ). ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules.</td></tr>
<tr><td>307</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02928">TIGR02928</a></td><td>365</td><td>44</td><td>74.1</td><td>4.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02928</td><td>orc1/cdc6 family replication initiation protein. Members of this protein family are found exclusively in the archaea. This set of DNA binding proteins shows homology to the origin recognition complex subunit 1/cell division control protein 6 family in eukaryotes. Several members may be found in genome and interact with each other. </td></tr>
<tr><td>308</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0610">COG0610</a></td><td>962</td><td>44</td><td>74.1</td><td>3.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG0610</td><td>Type I site-specific restriction-modification system, R (restriction) subunit and related helicases ... </td></tr>
<tr><td>309</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01130">cd01130</a></td><td>186</td><td>22</td><td>73.8</td><td>2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VirB11-like_ATPase</td><td>Type IV secretory pathway component VirB11, and related ATPases. The homohexamer, VirB11 is one of eleven Vir proteins, which are required for T-pilus biogenesis and virulence in the transfer of T-DNA from the Ti (tumor-inducing) plasmid of bacterial to plant cells. The pilus is a fibrous cell surface organelle, which mediates adhesion between bacteria during conjugative transfer or between bacteria and host eukaryotic cells during infection. VirB11- related ATPases include the archaeal flagella biosynthesis protein and the pilus assembly proteins CpaF/TadA and TrbB.  This alignment contains the C-terminal domain, which is the ATPase.</td></tr>
<tr><td>310</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09401">PRK09401</a></td><td>1176</td><td>53</td><td>73.7</td><td>3.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09401</td><td>reverse gyrase; Reviewed</td></tr>
<tr><td>311</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02982">TIGR02982</a></td><td>220</td><td>37</td><td>73.6</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>heterocyst_DevA</td><td>ABC exporter ATP-binding subunit, DevA family. Members of this protein family are found mostly in the Cyanobacteria, but also in the Planctomycetes. Cyanobacterial examples are involved in heterocyst formation, by which some fraction of members of the colony undergo a developmental change and become capable of nitrogen fixation. The DevBCA proteins are thought export of either heterocyst-specific glycolipids or an enzyme essential for formation of the laminated layer found in heterocysts.</td></tr>
<tr><td>312</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03263">TIGR03263</a></td><td>179</td><td>36</td><td>73.3</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>guanyl_kin</td><td>guanylate kinase. Members of this family are the enzyme guanylate kinase, also called GMP kinase. This enzyme transfers a phosphate from ATP to GMP, yielding ADP and GDP. </td></tr>
<tr><td>313</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01189">TIGR01189</a></td><td>198</td><td>69</td><td>73.2</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ccmA</td><td>heme ABC exporter, ATP-binding protein CcmA. This model describes the cyt c biogenesis protein encoded by ccmA in bacteria. An exception is, an arabidopsis protein. Quite likely this is encoded by an organelle. Bacterial c-type cytocromes are located on the periplasmic side of the cytoplasmic membrane. Several gene products encoded in a locus designated as 'ccm' are implicated in the transport and assembly of the functional cytochrome C. This cluster includes genes: ccmA;B;C;D;E;F;G and H. The posttranslational pathway includes the transport of heme moiety, the secretion of the apoprotein and the covalent attachment of the heme with the apoprotein. The proteins ccmA and B represent an ABC transporter; ccmC and D participate in heme transfer to ccmE, which function as a periplasmic heme chaperone. The presence of ccmF, G and H is suggested to be obligatory for the final functional assembly of cytochrome c. </td></tr>
<tr><td>314</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09200">PRK09200</a></td><td>790</td><td>53</td><td>73.2</td><td>2.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09200</td><td>preprotein translocase subunit SecA; Reviewed</td></tr>
<tr><td>315</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10789">PRK10789</a></td><td>569</td><td>16</td><td>72.6</td><td>1.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10789</td><td>putative multidrug transporter membrane\ATP-binding components; Provisional</td></tr>
<tr><td>316</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03744">TIGR03744</a></td><td>893</td><td>83</td><td>72.5</td><td>3.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>traC_PFL_4706</td><td>conjugative transfer ATPase, PFL_4706 family. Members of this protein family are predicted ATP-binding proteins apparently associated with DNA conjugal transfer. Members are found both in plasmids and in bacterial chromosomal regions that appear to derive from integrative elements such as conjugative transposons. More distant homologs, outside the scope of this family, include type IV secretion/conjugal transfer proteins such as TraC, VirB4 and TrsE. The granularity of this protein family definition is chosen so as to represent one distinctive clade and act as a marker through which to define and recognize the class of mobile element it serves. </td></tr>
<tr><td>317</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0396">COG0396</a></td><td>251</td><td>76</td><td>72.5</td><td>1.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SufC</td><td>Fe-S cluster assembly ATPase SufC </td></tr>
<tr><td>318</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03293">cd03293</a></td><td>220</td><td>20</td><td>72.4</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_NrtD_SsuB_transporters</td><td>ATP-binding cassette domain of the nitrate and sulfonate transporters. NrtD and SsuB are the ATP-binding subunits of the bacterial ABC-type nitrate and sulfonate transport systems, respectively. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>319</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02037">cd02037</a></td><td>169</td><td>40</td><td>72.1</td><td>4.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MRP-like</td><td>MRP (Multiple Resistance and pH adaptation) is a homologue of the Fer4_NifH superfamily. Like the other members of the superfamily, MRP contains a ATP-binding domain at the N-termini. It is found in bacteria as a membrane-spanning protein and functions as a Na+/H+ antiporter.</td></tr>
<tr><td>320</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09140">pfam09140</a></td><td>261</td><td>26</td><td>71.3</td><td>6.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MipZ</td><td>ATPase MipZ. MipZ is an ATPase that forms a complex with the chromosome partitioning protein ParB near the chromosomal origin of replication. It is responsible for the temporal and spatial regulation of FtsZ ring formation.</td></tr>
<tr><td>321</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1118">COG1118</a></td><td>345</td><td>37</td><td>70.9</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CysA</td><td>ABC-type sulfate/molybdate transport systems, ATPase component </td></tr>
<tr><td>322</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4604">COG4604</a></td><td>252</td><td>62</td><td>70.8</td><td>5.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CeuD</td><td>ABC-type enterochelin transport system, ATPase component  </td></tr>
<tr><td>323</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2884">COG2884</a></td><td>223</td><td>90</td><td>70.8</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FtsE</td><td>ABC-type ATPase involved in cell division </td></tr>
<tr><td>324</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1419">COG1419</a></td><td>407</td><td>26</td><td>70.7</td><td>7.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FlhF</td><td>Flagellar biosynthesis GTPase FlhF  </td></tr>
<tr><td>325</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1124">COG1124</a></td><td>252</td><td>35</td><td>70.6</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DppF</td><td>ABC-type dipeptide/oligopeptide/nickel transport system, ATPase component </td></tr>
<tr><td>326</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03300">cd03300</a></td><td>232</td><td>19</td><td>70.6</td><td>1.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_PotA_N</td><td>ATP-binding cassette domain of the polyamine transporter. PotA is an ABC-type transporter and the ATPase component of the spermidine/putrescine-preferential uptake system consisting of PotA, -B, -C, and -D. PotA has two domains with the N-terminal domain containing the ATPase activity and the residues required for homodimerization with PotA and heterdimerization with PotB. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>327</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4962">COG4962</a></td><td>355</td><td>159</td><td>70.5</td><td>1.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CpaF</td><td>Pilus assembly protein, ATPase of CpaF family </td></tr>
<tr><td>328</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4181">COG4181</a></td><td>228</td><td>16</td><td>70.4</td><td>2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YbbA</td><td>Predicted ABC-type transport system involved in lysophospholipase L1 biosynthesis, ATPase component </td></tr>
<tr><td>329</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1127">COG1127</a></td><td>263</td><td>36</td><td>70.0</td><td>2.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MlaF</td><td>ABC-type transporter Mla maintaining outer membrane lipid asymmetry, ATPase component MlaF </td></tr>
<tr><td>330</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09302">PRK09302</a></td><td>509</td><td>50</td><td>69.9</td><td>6.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09302</td><td>circadian clock protein KaiC; Reviewed</td></tr>
<tr><td>331</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06761">PRK06761</a></td><td>282</td><td>28</td><td>69.9</td><td>3.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06761</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>332</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3451">COG3451</a></td><td>796</td><td>106</td><td>69.6</td><td>5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VirB4</td><td>Type IV secretory pathway, VirB4 component </td></tr>
<tr><td>333</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06762">PRK06762</a></td><td>166</td><td>81</td><td>69.4</td><td>6.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06762</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>334</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06745">pfam06745</a></td><td>231</td><td>36</td><td>69.4</td><td>5.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KaiC</td><td>KaiC. This family represents a conserved region within bacterial and archaeal proteins, most of which are hypothetical. More than one copy is sometimes found in each protein. This family includes KaiC, which is one of the Kai proteins among which direct protein-protein association may be a critical process in the generation of circadian rhythms in cyanobacteria.</td></tr>
<tr><td>335</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13166">pfam13166</a></td><td>713</td><td>23</td><td>69.2</td><td>3.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_13</td><td>AAA domain. This family of domains contain a P-loop motif that is characteristic of the AAA superfamily. Many of the proteins in this family are conjugative transfer proteins. This family includes the PrrC protein that is thought to be the active component of the anticodon nuclease.</td></tr>
<tr><td>336</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03296">cd03296</a></td><td>239</td><td>16</td><td>69.2</td><td>1.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_CysA_sulfate_importer</td><td>ATP-binding cassette domain of the sulfate transporter. Part of the ABC transporter complex cysAWTP involved in sulfate import. Responsible for energy coupling to the transport system. The complex is composed of two ATP-binding proteins (cysA), two transmembrane proteins (cysT and cysW), and a solute-binding protein (cysP). ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>337</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1066">COG1066</a></td><td>456</td><td>68</td><td>69.0</td><td>9.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Sms</td><td>Predicted ATP-dependent serine protease </td></tr>
<tr><td>338</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01078">pfam01078</a></td><td>207</td><td>38</td><td>68.9</td><td>2.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Mg_chelatase</td><td>Magnesium chelatase, subunit ChlI. Magnesium-chelatase is a three-component enzyme that catalyses the insertion of Mg2+ into protoporphyrin IX. This is the first unique step in the synthesis of (bacterio)chlorophyll. Due to this, it is thought that Mg-chelatase has an important role in channelling inter- mediates into the (bacterio)chlorophyll branch in response to conditions suitable for photosynthetic growth. ChlI and BchD have molecular weight between 38-42 kDa.</td></tr>
<tr><td>339</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00929">TIGR00929</a></td><td>785</td><td>87</td><td>68.7</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VirB4_CagE</td><td>type IV secretion/conjugal transfer ATPase, VirB4 family. Type IV secretion systems are found in Gram-negative pathogens. They export proteins, DNA, or complexes in different systems and are related to plasmid conjugation systems. This model represents related ATPases that include VirB4 in Agrobacterium tumefaciens (DNA export) CagE in Helicobacter pylori (protein export) and plasmid TraB (conjugation).</td></tr>
<tr><td>340</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03348">TIGR03348</a></td><td>1169</td><td>33</td><td>68.6</td><td>3.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VI_IcmF</td><td>type VI secretion protein IcmF. Members of this protein family are IcmF homologs and tend to be associated with type VI secretion systems. </td></tr>
<tr><td>341</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03219">cd03219</a></td><td>236</td><td>73</td><td>68.6</td><td>2.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Mj1267_LivG_branched</td><td>ATP-binding cassette component of branched chain amino acids transport system. The Mj1267/LivG ABC transporter subfamily is involved in the transport of the hydrophobic amino acids leucine, isoleucine and valine. MJ1267 is a branched-chain amino acid transporter with 29% similarity to both the LivF and LivG components of the E. coli branched-chain amino acid transporter. MJ1267 contains an insertion from residues 114 to 123 characteristic of LivG (Leucine-Isoleucine-Valine) homologs. The branched-chain amino acid transporter from E. coli comprises a heterodimer of ABCs (LivF and LivG), a heterodimer of six-helix TM domains (LivM and LivH), and one of two alternative soluble periplasmic substrate binding proteins (LivK or LivJ).</td></tr>
<tr><td>342</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1116">COG1116</a></td><td>248</td><td>18</td><td>68.6</td><td>2.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TauB</td><td>ABC-type nitrate/sulfonate/bicarbonate transport system, ATPase component </td></tr>
<tr><td>343</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03015">TIGR03015</a></td><td>269</td><td>22</td><td>68.4</td><td>7.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pepcterm_ATPase</td><td>putative secretion ATPase, PEP-CTERM locus subfamily. Members of this protein are marked as probable ATPases by the nucleotide binding P-loop motif GXXGXGKTT, a motif DEAQ similar to the DEAD/H box of helicases, and extensive homology to ATPases of MSHA-type pilus systems and to GspA proteins associated with type II protein secretion systems. </td></tr>
<tr><td>344</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07728">pfam07728</a></td><td>135</td><td>24</td><td>68.4</td><td>5.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_5</td><td>AAA domain (dynein-related subfamily). This Pfam entry includes some of the AAA proteins not detected by the pfam00004 model.</td></tr>
<tr><td>345</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10436">PRK10436</a></td><td>462</td><td>57</td><td>68.3</td><td>4.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10436</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>346</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1223">COG1223</a></td><td>368</td><td>46</td><td>68.2</td><td>4.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1223</td><td>Predicted ATPase, AAA+ superfamily </td></tr>
<tr><td>347</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0497">COG0497</a></td><td>557</td><td>56</td><td>68.1</td><td>6.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecN</td><td>DNA repair ATPase RecN </td></tr>
<tr><td>348</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14516">pfam14516</a></td><td>331</td><td>71</td><td>68.0</td><td>7.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_35</td><td>AAA-like domain. This family of proteins are part of the AAA superfamily.</td></tr>
<tr><td>349</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00131">PRK00131</a></td><td>175</td><td>63</td><td>68.0</td><td>3.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>aroK</td><td>shikimate kinase; Reviewed</td></tr>
<tr><td>350</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00958">TIGR00958</a></td><td>711</td><td>29</td><td>67.9</td><td>2.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3a01208</td><td>Conjugate Transporter-2 (CT2) Family protein. </td></tr>
<tr><td>351</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15516">pfam15516</a></td><td>159</td><td>74</td><td>67.8</td><td>7.8</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BpuSI_N</td><td>BpuSI N-terminal domain. This is the N-terminal (nuclease) domain of the BpuSI restriction endonuclease.</td></tr>
<tr><td>352</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01121">cd01121</a></td><td>372</td><td>74</td><td>67.7</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Sms</td><td>Sms (bacterial radA) DNA repair protein. This protein is not related to archael radA any more than is to other RecA-like NTPases. Sms has a role in recombination and recombinational repair and is responsible for the stabilization or processing of branched DNA molecules.</td></tr>
<tr><td>353</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01007">TIGR01007</a></td><td>204</td><td>27</td><td>67.3</td><td>4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>eps_fam</td><td>capsular exopolysaccharide family. This model describes the capsular exopolysaccharide proteins in bacteria. The exopolysaccharide gene cluster consists of several genes which encode a number of proteins which regulate the exoploysaccharide biosynthesis(EPS). Atleast 13 genes espA to espM in streptococcus species seem to direct the EPS proteins and all of which share high homology. Functional roles were characterized by gene disruption experiments which resulted in exopolysaccharide-deficient phenotypes. </td></tr>
<tr><td>354</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03783">TIGR03783</a></td><td>829</td><td>41</td><td>67.2</td><td>5.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Bac_Flav_CT_G</td><td>Bacteroides conjugation system ATPase, TraG family. Members of this family include the predicted ATPase, TraG, encoded by transfer region genes of conjugative transposons of Bacteroides, such as CTnDOT, found on the main chromosome. Members also include TraG homologs borne on plasmids in Bacteroides. The protein family is related to the conjugative transfer system ATPase VirB4. </td></tr>
<tr><td>355</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03221">cd03221</a></td><td>144</td><td>27</td><td>67.1</td><td>2.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCF_EF-3</td><td>ATP-binding cassette domain of elongation factor 3, subfamily F. Elongation factor 3 (EF-3) is a cytosolic protein required by fungal ribosomes for in vitro protein synthesis and for in vivo growth. EF-3 stimulates the binding of the EF-1: GTP: aa-tRNA ternary complex to the ribosomal A site by facilitated release of the deacylated tRNA from the E site. The reaction requires ATP hydrolysis. EF-3 contains two ATP nucleotide binding sequence (NBS) motifs. NBSI is sufficient for the intrinsic ATPase activity. NBSII is essential for the ribosome-stimulated functions.</td></tr>
<tr><td>356</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03298">cd03298</a></td><td>211</td><td>138</td><td>67.1</td><td>2.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_ThiQ_thiamine_transporter</td><td>ATP-binding cassette domain of the thiamine transport system. Part of the binding-protein-dependent transport system tbpA-thiPQ for thiamine and TPP. Probably responsible for the translocation of thiamine across the membrane. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>357</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1348">COG1348</a></td><td>278</td><td>192</td><td>67.1</td><td>5.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NifH</td><td>Nitrogenase subunit NifH, an ATPase </td></tr>
<tr><td>358</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03261">cd03261</a></td><td>235</td><td>45</td><td>67.0</td><td>3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Org_Solvent_Resistant</td><td>ATP-binding cassette transport system involved in resistant to organic solvents. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>359</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01918">cd01918</a></td><td>149</td><td>21</td><td>67.0</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HprK_C</td><td>HprK/P, the bifunctional histidine-containing protein kinase/phosphatase, controls the phosphorylation state of the phosphocarrier protein HPr and regulates the utilization of carbon sources by gram-positive bacteria. It catalyzes both the ATP-dependent phosphorylation of Ser-46 of HPr and its dephosphorylation by phosphorolysis. The latter reaction uses inorganic phosphate as substrate and produces pyrophosphate. Phosphoenolpyruvate carboxykinase (PEPCK) and the C-terminal catalytic domain of HprK/P are structurally similar with conserved active site residues suggesting these two phosphotransferases have related functions.  The HprK/P N-terminal domain is structurally similar to the N-terminal domains of the MurE and MurF amino acid ligases.</td></tr>
<tr><td>360</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11650">PRK11650</a></td><td>356</td><td>21</td><td>66.9</td><td>2.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ugpC</td><td>glycerol-3-phosphate transporter ATP-binding subunit; Provisional</td></tr>
<tr><td>361</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03235">cd03235</a></td><td>213</td><td>36</td><td>66.7</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_Metallic_Cations</td><td>ATP-binding cassette domain of the metal-type transporters. This family includes transporters involved in the uptake of various metallic cations such as iron, manganese, and zinc. The ATPases of this group of transporters are very similar to members of iron-siderophore uptake family suggesting that they share a common ancestor. The best characterized metal-type ABC transporters are the YfeABCD system of Y. pestis, the SitABCD system of Salmonella enterica serovar Typhimurium, and the SitABCD transporter of Shigella flexneri. Moreover other uncharacterized homologs of these metal-type transporters are mainly found in pathogens like Haemophilus or enteroinvasive E. coli isolates.</td></tr>
<tr><td>362</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03250">cd03250</a></td><td>204</td><td>32</td><td>66.6</td><td>3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCC_MRP_domain1</td><td>ATP-binding cassette domain 1 of multidrug resistance-associated protein, subfamily C. This subfamily is also known as MRP (multidrug resistance-associated protein). Some of the MRP members have five additional transmembrane segments in their N-terminus, but the function of these additional membrane-spanning domains is not clear. The MRP was found in the multidrug-resisting lung cancer cell in which p-glycoprotein was not overexpressed. MRP exports glutathione by drug stimulation, as well as, certain substrates in conjugated forms with anions, such as glutathione, glucuronate, and sulfate.</td></tr>
<tr><td>363</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1855">COG1855</a></td><td>604</td><td>40</td><td>66.4</td><td>4.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1855</td><td>Predicted ATPase, PilT family  </td></tr>
<tr><td>364</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03111">cd03111</a></td><td>106</td><td>36</td><td>66.3</td><td>5.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CpaE_like</td><td>This protein family consists of proteins similar to the cpaE protein of the Caulobacter pilus assembly and the orf4 protein of Actinobacillus pilus formation gene cluster. The function of these proteins are unkown. The Caulobacter pilus assembly contains 7 genes: pilA, cpaA, cpaB, cpaC, cpaD, cpaE and cpaF. These genes are clustered together on chromosome.</td></tr>
<tr><td>365</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03246">cd03246</a></td><td>173</td><td>14</td><td>66.2</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCC_Protease_Secretion</td><td>ATP-binding cassette domain of PrtD, subfamily C. This family represents the ABC component of the protease secretion system PrtD, a 60-kDa integral membrane protein sharing 37% identity with HlyB, the ABC component of the alpha-hemolysin secretion pathway, in the C-terminal domain. They export degradative enzymes by using a type I protein secretion system and lack an N-terminal signal peptide, but contain a C-terminal secretion signal. The Type I secretion apparatus is made up of three components, an ABC transporter, a membrane fusion protein (MFP), and an outer membrane protein (OMP). For the HlyA transporter complex, HlyB (ABC transporter) and HlyD (MFP) reside in the inner membrane of E. coli. The OMP component is TolC, which is thought to interact with the MFP to form a continuous channel across the periplasm from the cytoplasm to the exterior. HlyB belongs to the family of ABC transporters, which are ubiquitous, ATP-dependent transmembrane pumps or channels. The spectrum of transport substrates ranges from inorganic ions, nutrients such as amino acids, sugars, or peptides, hydrophobic drugs, to large polypeptides, such as HlyA.</td></tr>
<tr><td>366</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03243">cd03243</a></td><td>202</td><td>125</td><td>66.0</td><td>5.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_MutS_homologs</td><td>ATP-binding cassette domain of MutS homologs. The MutS protein initiates DNA mismatch repair by recognizing mispaired and unpaired bases embedded in duplex DNA and activating endo- and exonucleases to remove the mismatch. Members of the MutS family also possess a conserved ATPase activity that belongs to the ATP binding cassette (ABC) superfamily. MutS homologs (MSH) have been identified in most prokaryotic and all eukaryotic organisms examined. Prokaryotes have two homologs (MutS1 and MutS2), whereas seven MSH proteins (MSH1 to MSH7) have been identified in eukaryotes. The homodimer MutS1 and heterodimers MSH2-MSH3 and MSH2-MSH6 are primarily involved in mitotic mismatch repair, whereas MSH4-MSH5 is involved in resolution of Holliday junctions during meiosis. All members of the MutS family contain the highly conserved Walker A/B ATPase domain, and many share a common mechanism of action. MutS1, MSH2-MSH3, MSH2-MSH6, and MSH4-MSH5 dimerize to form sliding clamps, and recognition of specific DNA structures or lesions results in ADP/ATP exchange.</td></tr>
<tr><td>367</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06547">PRK06547</a></td><td>172</td><td>15</td><td>66.0</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06547</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>368</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11545">PRK11545</a></td><td>163</td><td>19</td><td>65.9</td><td>3.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gntK</td><td>gluconate kinase 1; Provisional</td></tr>
<tr><td>369</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13909">PRK13909</a></td><td>910</td><td>73</td><td>65.9</td><td>4.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13909</td><td>putative recombination protein RecB; Provisional</td></tr>
<tr><td>370</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3910">COG3910</a></td><td>233</td><td>175</td><td>65.4</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG3910</td><td>Predicted ATPase  </td></tr>
<tr><td>371</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13764">PRK13764</a></td><td>602</td><td>28</td><td>65.2</td><td>5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13764</td><td>ATPase; Provisional</td></tr>
<tr><td>372</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03689">TIGR03689</a></td><td>512</td><td>66</td><td>65.1</td><td>4.1</td><td>[&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pup_AAA</td><td>proteasome ATPase. In the Actinobacteria, as shown for Mycobacterium tuberculosis, some proteins are modified by ligation between an epsilon-amino group of a lysine side chain and the C-terminal carboxylate of the ubiquitin-like protein Pup. This modification leads to protein degradation by the archaeal-like proteasome found in the Actinobacteria. Members of this protein family belong to the AAA family of ATPases and tend to be clustered with the genes for Pup, the Pup ligase PafA, and structural components of the proteasome. This protein forms hexameric rings with ATPase activity. </td></tr>
<tr><td>373</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03918">PRK03918</a></td><td>880</td><td>27</td><td>65.0</td><td>4.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK03918</td><td>chromosome segregation protein; Provisional</td></tr>
<tr><td>374</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4619">COG4619</a></td><td>223</td><td>54</td><td>64.6</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FetA</td><td>ABC-type iron transport system FetAB, ATPase component </td></tr>
<tr><td>375</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09452">PRK09452</a></td><td>375</td><td>53</td><td>64.4</td><td>2.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>potA</td><td>putrescine/spermidine ABC transporter ATPase protein; Reviewed</td></tr>
<tr><td>376</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13833">PRK13833</a></td><td>323</td><td>63</td><td>64.4</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13833</td><td>conjugal transfer protein TrbB; Provisional</td></tr>
<tr><td>377</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03297">cd03297</a></td><td>214</td><td>19</td><td>64.3</td><td>3.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_ModC_molybdenum_transporter</td><td>ATP-binding cassette domain of the molybdenum transport system. ModC is an ABC-type transporter and the ATPase component of a molybdate transport system that also includes the periplasmic binding protein ModA and the membrane protein ModB. ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>378</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01125">cd01125</a></td><td>239</td><td>49</td><td>64.1</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>repA</td><td>Hexameric Replicative Helicase RepA.  RepA is encoded by a plasmid, which is found in most Gram negative bacteria. RepA is a 5'-3' DNA helicase which can utilize ATP, GTP and CTP to a lesser extent.</td></tr>
<tr><td>379</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03608">TIGR03608</a></td><td>206</td><td>18</td><td>63.7</td><td>3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>L_ocin_972_ABC</td><td>putative bacteriocin export ABC transporter, lactococcin 972 group. A gene pair with a fairly wide distribution consists of a polypeptide related to the lactococcin 972 (see TIGR01653) and multiple-membrane-spanning putative immunity protein (see TIGR01654). This model represents a small clade within the ABC transporters that regularly are found adjacent to these bacteriocin system gene pairs and are likely serve as export proteins. </td></tr>
<tr><td>380</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11701">PRK11701</a></td><td>258</td><td>134</td><td>63.4</td><td>3.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>phnK</td><td>phosphonate C-P lyase system protein PhnK; Provisional</td></tr>
<tr><td>381</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03257">cd03257</a></td><td>228</td><td>54</td><td>63.3</td><td>3.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_NikE_OppD_transporters</td><td>ATP-binding cassette domain of nickel/oligopeptides specific transporters. The ABC transporter subfamily specific for the transport of dipeptides, oligopeptides (OppD), and nickel (NikDE). The NikABCDE system of E. coli belongs to this family and is composed of the periplasmic binding protein NikA, two integral membrane components (NikB and NikC), and two ATPase (NikD and NikE). The NikABCDE transporter is synthesized under anaerobic conditions to meet the increased demand for nickel resulting from hydrogenase synthesis. The molecular mechanism of nickel uptake in many bacteria and most archaea is not known. Many other members of this ABC family are also involved in the uptake of dipeptides and oligopeptides. The oligopeptide transport system (Opp) is a five-component ABC transport composed of a membrane-anchored substrate binding proteins (SRP), OppA, two transmembrane proteins, OppB and OppC, and two ATP-binding domains, OppD and OppF.</td></tr>
<tr><td>382</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03213">cd03213</a></td><td>194</td><td>62</td><td>63.3</td><td>1.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCG_EPDR</td><td>Eye pigment and drug resistance transporter subfamily G of the ATP-binding cassette superfamily. ABCG transporters are involved in eye pigment (EP) precursor transport, regulation of lipid-trafficking mechanisms, and pleiotropic drug resistance (DR). DR is a well-described phenomenon occurring in fungi and shares several similarities with processes in bacteria and higher eukaryotes. Compared to other members of the ABC transporter subfamilies, the ABCG transporter family is composed of proteins that have an ATP-binding cassette domain at the N-terminus and a TM (transmembrane) domain at the C-terminus.</td></tr>
<tr><td>383</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00152">TIGR00152</a></td><td>190</td><td>132</td><td>63.3</td><td>4.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00152</td><td>dephospho-CoA kinase. This model produces scores in the range of 0-25 bits against adenylate, guanylate, uridine, and thymidylate kinases. </td></tr>
<tr><td>384</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03295">cd03295</a></td><td>242</td><td>45</td><td>63.1</td><td>3.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_OpuCA_Osmoprotection</td><td>ATP-binding cassette domain of the osmoprotectant transporter. OpuCA is a the ATP binding component of a bacterial solute transporter that serves a protective role to cells growing in a hyperosmolar environment. ABC (ATP-binding cassette) transporter nucleotide-binding domain; ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition, to the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>385</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06921">PRK06921</a></td><td>266</td><td>46</td><td>63.0</td><td>7.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06921</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>386</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04520">TIGR04520</a></td><td>268</td><td>146</td><td>62.9</td><td>7.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ECF_ATPase_1</td><td>energy-coupling factor transporter ATPase. Members of this family are ATP-binding cassette (ABC) proteins by homology, but belong to energy coupling factor (ECF) transport systems. The architecture in general is two ATPase subunits (or a double-length fusion protein), a T component, and a substrate capture (S) component that is highly variable, and may be interchangeable in genomes with only one T component. This model identifies many but not examples of the upstream member of the pair of ECF ATPases in Firmicutes and Mollicutes. </td></tr>
<tr><td>387</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4136">COG4136</a></td><td>213</td><td>43</td><td>62.9</td><td>3.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YnjD</td><td>ABC-type uncharacterized transport system YnjBCD, ATPase component </td></tr>
<tr><td>388</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00968">TIGR00968</a></td><td>237</td><td>16</td><td>62.9</td><td>3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3a0106s01</td><td>sulfate ABC transporter, ATP-binding protein. </td></tr>
<tr><td>389</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00064">PRK00064</a></td><td>361</td><td>41</td><td>62.7</td><td>4.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recF</td><td>recombination protein F; Reviewed</td></tr>
<tr><td>390</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0411">COG0411</a></td><td>250</td><td>62</td><td>62.7</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LivG</td><td>ABC-type branched-chain amino acid transport system, ATPase component </td></tr>
<tr><td>391</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13578">pfam13578</a></td><td>105</td><td>38</td><td>62.6</td><td>80</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_24</td><td>Methyltransferase domain. This family appears to be a methyltransferase domain.</td></tr>
<tr><td>392</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01968">TIGR01968</a></td><td>261</td><td>110</td><td>62.5</td><td>7.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>minD_bact</td><td>septum site-determining protein MinD. This model describes the bacterial and chloroplast form of MinD, a multifunctional cell division protein that guides correct placement of the septum. The homologous archaeal MinD proteins, with many archaeal genomes having two or more forms, are described by a separate model. </td></tr>
<tr><td>393</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00006">pfam00006</a></td><td>213</td><td>38</td><td>62.4</td><td>7.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ATP-synt_ab</td><td>ATP synthase alpha/beta family, nucleotide-binding domain. This family includes the ATP synthase alpha and beta subunits, the ATP synthase associated with flagella and the termination factor Rho.</td></tr>
<tr><td>394</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02040">cd02040</a></td><td>270</td><td>183</td><td>62.3</td><td>7.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NifH</td><td>NifH gene encodes component II (iron protein) of nitrogenase. Nitrogenase is responsible for the biological nitrogen fixation, i.e. reduction of molecular nitrogen to ammonia. NifH consists of two oxygen-sensitive metallosulfur proteins: the mollybdenum-iron (alternatively, vanadium-iron or iron-iron) protein (commonly referred to as component 1), and the iron protein (commonly referred to as component 2). The iron protein is a homodimer, with an Fe4S4 cluster bound between the subunits and two ATP-binding domains. It supplies energy by ATP hydrolysis, and transfers electrons from reduced ferredoxin or flavodoxin to component 1 for the reduction of molecular nitrogen to ammonia.</td></tr>
<tr><td>395</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00235">TIGR00235</a></td><td>207</td><td>20</td><td>62.1</td><td>6.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>udk</td><td>uridine kinase. Model contains a number of longer eukaryotic proteins and starts bringing in phosphoribulokinase hits at scores of 160 and below </td></tr>
<tr><td>396</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1117">COG1117</a></td><td>253</td><td>18</td><td>62.1</td><td>3.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PstB</td><td>ABC-type phosphate transport system, ATPase component </td></tr>
<tr><td>397</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00931">pfam00931</a></td><td>287</td><td>34</td><td>62.1</td><td>6.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NB-ARC</td><td>NB-ARC domain. </td></tr>
<tr><td>398</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00972">TIGR00972</a></td><td>247</td><td>14</td><td>62.1</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>3a0107s01c2</td><td>phosphate ABC transporter, ATP-binding protein. This model represents the ATP-binding protein of a family of ABC transporters for inorganic phosphate. In the model species Escherichia coli, a constitutive transporter for inorganic phosphate, with low affinity, is also present. The high affinity transporter that includes this polypeptide is induced when extracellular phosphate concentrations are low. The proteins most similar to the members of this family but not included appear to be amino acid transporters. </td></tr>
<tr><td>399</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09518">PRK09518</a></td><td>712</td><td>21</td><td>61.9</td><td>4.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09518</td><td>bifunctional cytidylate kinase/GTPase Der; Reviewed</td></tr>
<tr><td>400</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0703">COG0703</a></td><td>172</td><td>21</td><td>61.9</td><td>5.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AroK</td><td>Shikimate kinase </td></tr>
<tr><td>401</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0606">COG0606</a></td><td>490</td><td>94</td><td>61.8</td><td>3.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YifB</td><td>Predicted ATPase with chaperone activity </td></tr>
<tr><td>402</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0630">COG0630</a></td><td>312</td><td>39</td><td>61.8</td><td>5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VirB11</td><td>Type IV secretory pathway ATPase VirB11/Archaellum biosynthesis ATPase </td></tr>
<tr><td>403</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05841">pfam05841</a></td><td>122</td><td>29</td><td>61.8</td><td>4.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Apc15p</td><td>Apc15p protein. The anaphase-promoting complex (APC) is a conserved multi-subunit ubiquitin ligase required for the degradation of key cell cycle regulators Members of this family are components of the anaphase-promoting complex homologous to Apc15p.</td></tr>
<tr><td>404</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4525">COG4525</a></td><td>259</td><td>20</td><td>61.7</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TauB</td><td>ABC-type taurine transport system, ATPase component </td></tr>
<tr><td>405</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00485">pfam00485</a></td><td>196</td><td>26</td><td>61.7</td><td>5.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK</td><td>Phosphoribulokinase / Uridine kinase family. In Arabidopsis the region carries two binding domains, a phosphoribosylpyrophosphate-binding domain and, at the very C-terminus, a uracil-binding domain.</td></tr>
<tr><td>406</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03371">TIGR03371</a></td><td>246</td><td>31</td><td>61.7</td><td>6.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cellulose_yhjQ</td><td>cellulose synthase operon protein YhjQ. Members of this family are the YhjQ protein, found immediately upsteam of bacterial cellulose synthase (bcs) genes in a broad range of bacteria, including both copies of the bcs locus in Klebsiella pneumoniae. In several species it is seen clearly as part of the bcs operon. It is identified as a probable component of the bacterial cellulose metabolic process not only by gene location, but also by partial phylogenetic profiling, or Haft-Selengut algorithm (), based on a bacterial cellulose biosynthesis genome property profile. Cellulose plays an important role in biofilm formation and structural integrity in some bacteria. Mutants in yhjQ in Escherichia coli, show altered morphology an growth, but the function of YhjQ has not yet been determined. </td></tr>
<tr><td>407</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0464">COG0464</a></td><td>494</td><td>18</td><td>61.7</td><td>4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SpoVK</td><td>AAA+-type ATPase, SpoVK/Ycf46/Vps4 family </td></tr>
<tr><td>408</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03254">cd03254</a></td><td>229</td><td>69</td><td>61.5</td><td>3.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCC_Glucan_exporter_like</td><td>ATP-binding cassette domain of glucan transporter and related proteins, subfamily C. Glucan exporter ATP-binding protein. In A. tumefaciens cyclic beta-1, 2-glucan must be transported into the periplasmic space to exert its action as a virulence factor. This subfamily belongs to the MRP-like family and is involved in drug, peptide, and lipid export. The MRP-like family, similar to all ABC proteins, have a common four-domain core structure constituted by two membrane-spanning domains each composed of six transmembrane (TM) helices and two nucleotide-binding domains (NBD). ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>409</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3638">COG3638</a></td><td>258</td><td>45</td><td>61.5</td><td>3.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PhnC</td><td>ABC-type phosphate/phosphonate transport system, ATPase component </td></tr>
<tr><td>410</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03258">cd03258</a></td><td>233</td><td>88</td><td>61.2</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_MetN_methionine_transporter</td><td>ATP-binding cassette domain of methionine transporter. MetN (also known as YusC) is an ABC-type transporter encoded by metN of the metNPQ operon in Bacillus subtilis that is involved in methionine transport. Other members of this system include the MetP permease and the MetQ substrate binding protein. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>411</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12422">PRK12422</a></td><td>445</td><td>64</td><td>60.9</td><td>7.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK12422</td><td>chromosomal replication initiation protein; Provisional</td></tr>
<tr><td>412</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09087">PRK09087</a></td><td>226</td><td>14</td><td>60.9</td><td>3.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09087</td><td>hypothetical protein; Validated</td></tr>
<tr><td>413</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3840">COG3840</a></td><td>231</td><td>144</td><td>60.7</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ThiQ</td><td>ABC-type thiamine transport system, ATPase component </td></tr>
<tr><td>414</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01978">TIGR01978</a></td><td>243</td><td>75</td><td>60.5</td><td>4.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>sufC</td><td>FeS assembly ATPase SufC. SufC is part of the SUF system, shown in E. coli to consist of six proteins and believed to act in Fe-S cluster formation during oxidative stress. SufC forms a complex with SufB and SufD. SufC belongs to the ATP-binding cassette transporter family (pfam00005) but is no longer thought to be part of a transporter. The complex is reported as cytosolic () or associated with the membrane (). The SUF system also includes a cysteine desulfurase (SufS, enhanced by SufE) and a probable iron-sulfur cluster assembly scaffold protein, SufA. </td></tr>
<tr><td>415</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01077">PRK01077</a></td><td>451</td><td>29</td><td>60.5</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01077</td><td>cobyrinic acid a,c-diamide synthase; Validated</td></tr>
<tr><td>416</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04411">pfam04411</a></td><td>154</td><td>88</td><td>60.5</td><td>24</td><td>[&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF524</td><td>Protein of unknown function (DUF524). This domain has been identified as a member of the PD-(D/E)XK nuclease superfamily through transitive meta profile searches. The domain has two additional beta-strands inserted to the core fold after the first core alpha-helix.   It has been speculated that it could function as s methylation-dependent restriction.</td></tr>
<tr><td>417</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10917">PRK10917</a></td><td>681</td><td>41</td><td>60.4</td><td>9.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10917</td><td>ATP-dependent DNA helicase RecG; Provisional</td></tr>
<tr><td>418</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13670">cd13670</a></td><td>298</td><td>72</td><td>60.4</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_TRAP_Tp0957_like</td><td>Uncharacterized substrate-binding protein of the Tripartite ATP-independent  Periplasmic transporter family; the type 2 periplasmic-binding protein fold. This subfamily includes the putative periplasmic substrate-binding protein Tp0957 from Treponema pallidum, which is similar to TRAP transport systems such as SiaP (a sialic acid binding virulence factor) and TeaA (an ectoine binding protein). TRAP transporters are a large family of solute transporters ubiquitously found in bacteria and archaea. They are comprised of a periplasmic substrate-binding protein (SBP) and two unequally sized integral membrane components: a large transmembrane subunit involved in the translocation process and a smaller membrane of unknown function. The driving force of TRAP transporters is provided by electrochemical ion gradients (either protons or sodium ions) across the cytoplasmic membrane, rather than ATP hydrolysis. This substrate-binding domain belongs to the type 2 periplasmic binding fold protein superfamily (PBP2). The PBP2 proteins are typically comprised of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap.</td></tr>
<tr><td>419</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03256">cd03256</a></td><td>241</td><td>42</td><td>60.3</td><td>4.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_PhnC_transporter</td><td>ATP-binding cassette domain of the binding protein-dependent phosphonate transport system. Phosphonates are a class of organophosphorus compounds characterized by a chemically stable carbon-to-phosphorus (C-P) bond. Phosphonates are widespread among naturally occurring compounds in all kingdoms of wildlife, but only prokaryotic microorganisms are able to cleave this bond. Certain bacteria such as E. coli can use alkylphosphonates as a phosphorus source. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>420</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05416">PRK05416</a></td><td>288</td><td>15</td><td>60.3</td><td>4.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05416</td><td>glmZ(sRNA)-inactivating NTPase; Provisional</td></tr>
<tr><td>421</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00348">TIGR00348</a></td><td>667</td><td>181</td><td>59.9</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>hsdR</td><td>type I site-specific deoxyribonuclease, HsdR family. This gene is part of the type I restriction and modification system which is composed of three polypeptides R (restriction endonuclease), M (modification) and S (specificity). This group of enzymes recognize specific short DNA sequences and have an absolute requirement for ATP (or dATP) and S-adenosyl-L-methionine. They also catalyse the reactions of EC 2.1.1.72 and EC 2.1.1.73, with similar site specificity.(J. Mol. Biol. 271 (3), 342-348 (1997)). Members of this family are assumed to differ from each other in DNA site specificity. </td></tr>
<tr><td>422</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00081">PRK00081</a></td><td>194</td><td>143</td><td>59.8</td><td>6.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>coaE</td><td>dephospho-CoA kinase; Reviewed</td></tr>
<tr><td>423</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01580">pfam01580</a></td><td>201</td><td>35</td><td>59.5</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FtsK_SpoIIIE</td><td>FtsK/SpoIIIE family. FtsK has extensive sequence similarity to wide variety of proteins from prokaryotes and plasmids, termed the FtsK/SpoIIIE family. This domain contains a putative ATP binding P-loop motif. It is found in the FtsK cell division protein from Escherichia coli and the stage III sporulation protein E SpoIIIE, which has roles in regulation of prespore specific gene expression in B. subtilis. A mutation in FtsK causes a temperature sensitive block in cell division and it is involved in peptidoglycan synthesis or modification. The SpoIIIE protein is implicated in intercellular chromosomal DNA transfer.</td></tr>
<tr><td>424</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00889">PRK00889</a></td><td>175</td><td>31</td><td>59.3</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00889</td><td>adenylylsulfate kinase; Provisional</td></tr>
<tr><td>425</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03263">cd03263</a></td><td>220</td><td>35</td><td>59.1</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_subfamily_A</td><td>ATP-binding cassette domain of the lipid transporters, subfamily A. The ABCA subfamily mediates the transport of a variety of lipid compounds. Mutations of members of ABCA subfamily are associated with human genetic diseases, such as, familial high-density lipoprotein (HDL) deficiency, neonatal surfactant deficiency, degenerative retinopathies, and congenital keratinization disorders. The ABCA1 protein is involved in disorders of cholesterol transport and high-density lipoprotein (HDL) biosynthesis. The ABCA4 (ABCR) protein transports vitamin A derivatives in the outer segments of photoreceptor cells, and therefore, performs a crucial step in the visual cycle. The ABCA genes are not present in yeast. However, evolutionary studies of ABCA genes indicate that they arose as transporters that subsequently duplicated and that certain sets of ABCA genes were lost in different eukaryotic lineages.</td></tr>
<tr><td>426</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03873">TIGR03873</a></td><td>256</td><td>31</td><td>58.7</td><td>3.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420-0_ABC_ATP</td><td>proposed F420-0 ABC transporter, ATP-binding protein. This small clade of ABC-type transporter ATP-binding protein components is found as a three gene cassette along with a periplasmic substrate-binding protein (TIGR03868) and a permease (TIGR03869). The organisms containing this cassette are all Actinobacteria and all contain numerous genes requiring the coenzyme F420. This model was defined based on five such organisms, four of which are lacking all F420 biosynthetic capability save the final side-chain polyglutamate attachment step (via the gene cofE: TIGR01916). In Jonesia denitrificans DSM 20603 and marine actinobacterium PHSC20C1 this cassette is in an apparent operon with the cofE gene and, in PHSC20C1, also with a F420-dependent glucose-6-phosphate dehydrogenase (TIGR03554). Based on these observations we propose that this ATP-binding protein is a component of an F420-0 (that is, F420 lacking only the polyglutamate tail) transporter.</td></tr>
<tr><td>427</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4674">COG4674</a></td><td>249</td><td>28</td><td>58.6</td><td>3.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4674</td><td>ABC-type uncharacterized transport system, ATPase component  </td></tr>
<tr><td>428</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02323">TIGR02323</a></td><td>253</td><td>170</td><td>58.3</td><td>4.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CP_lyasePhnK</td><td>phosphonate C-P lyase system protein PhnK. Members of this family are the PhnK protein of C-P lyase systems for utilization of phosphonates. These systems resemble phosphonatase-based systems in having a three component ABC transporter, where TIGR01097 is the permease, TIGR01098 is the phosphonates binding protein, and TIGR02315 is the ATP-binding cassette (ABC) protein. They differ, however, in having, typically, ten or more additional genes, many of which are believed to form a membrane-associated complex. This protein (PhnK) and the adjacent-encoded PhnL resemble transporter ATP-binding proteins but are suggested, based on mutatgenesis studies, to be part of this complex rather than part of a transporter per se. </td></tr>
<tr><td>429</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07693">pfam07693</a></td><td>301</td><td>55</td><td>58.2</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KAP_NTPase</td><td>KAP family P-loop domain. The KAP (after Kidins220/ARMS and PifA) family of predicted NTPases are sporadically distributed across a wide phylogenetic range in bacteria and in animals. Many of the prokaryotic KAP NTPases are encoded in plasmids and tend to undergo disruption to form pseudogenes. A unique feature of all eukaryotic and certain bacterial KAP NTPases is the presence of two or four transmembrane helices inserted into the P-loop NTPase domain. These transmembrane helices anchor KAP NTPases in the membrane such that the P-loop domain is located on the intracellular side.</td></tr>
<tr><td>430</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03193">pfam03193</a></td><td>161</td><td>52</td><td>58.2</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF258</td><td>Protein of unknown function, DUF258. </td></tr>
<tr><td>431</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02702">pfam02702</a></td><td>211</td><td>32</td><td>58.2</td><td>8.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KdpD</td><td>Osmosensitive K+ channel His kinase sensor domain. This is a family of KdpD sensor kinase proteins that regulate the kdpFABC operon responsible for potassium transport. The aligned region corresponds to the N-terminal cytoplasmic part of the protein which may be the sensor domain responsible for sensing turgor pressure.</td></tr>
<tr><td>432</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01187">TIGR01187</a></td><td>325</td><td>14</td><td>58.0</td><td>3.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>potA</td><td>spermidine/putrescine ABC transporter ATP-binding subunit. This model describes spermidine/putrescine ABC transporter, ATP binding subunit in bacteria and its equivalents in archaea. This transport system belong to the larger ATP-Binding Cassette (ABC) transporter superfamily. The characteristic feature of these transporter is the obligatory coupling of ATP hydrolysis to substrate translocation. The minimal configuration of bacterial ABC transport system: an ATPase or ATP binding subunit; An integral membrane protein; a hydrophilic polypetpide, which likely functions as substrate binding protein. Polyamines like spermidine and putrescine play vital role in cell proliferation, differentiation, and ion homeostasis. The concentration of polyamines within the cell are regulated by biosynthesis, degradation and transport (uptake and efflux included). </td></tr>
<tr><td>433</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03271">cd03271</a></td><td>261</td><td>16</td><td>57.9</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_UvrA_II</td><td>ATP-binding cassette domain II of the excision repair protein UvrA. Nucleotide excision repair in eubacteria is a process that repairs DNA damage by the removal of a 12-13-mer oligonucleotide containing the lesion. Recognition and cleavage of the damaged DNA is a multistep ATP-dependent reaction that requires the UvrA, UvrB, and UvrC proteins. Both UvrA and UvrB are ATPases, with UvrA having two ATP binding sites, which have the characteristic signature of the family of ABC proteins and UvrB having one ATP binding site that is structurally related to that of helicases.</td></tr>
<tr><td>434</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00464">cd00464</a></td><td>154</td><td>21</td><td>57.8</td><td>7.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SK</td><td>Shikimate kinase (SK) is the fifth enzyme in the shikimate pathway, a seven-step biosynthetic pathway which converts erythrose-4-phosphate to chorismic acid, found in bacteria, fungi and plants. Chorismic acid is a important intermediate in the synthesis of aromatic compounds, such as aromatic amino acids, p-aminobenzoic acid, folate and ubiquinone. Shikimate kinase catalyses the phosphorylation of the 3-hydroxyl group of shikimic acid using ATP.</td></tr>
<tr><td>435</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07652">pfam07652</a></td><td>146</td><td>42</td><td>57.8</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Flavi_DEAD</td><td>Flavivirus DEAD domain. </td></tr>
<tr><td>436</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1134">COG1134</a></td><td>249</td><td>42</td><td>57.8</td><td>4.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TagH</td><td>ABC-type polysaccharide/polyol phosphate transport system, ATPase component </td></tr>
<tr><td>437</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1224">COG1224</a></td><td>450</td><td>72</td><td>57.6</td><td>6.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIP49</td><td>DNA helicase TIP49, TBP-interacting protein </td></tr>
<tr><td>438</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06995">PRK06995</a></td><td>484</td><td>46</td><td>57.6</td><td>5.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>flhF</td><td>flagellar biosynthesis regulator FlhF; Validated</td></tr>
<tr><td>439</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06414">pfam06414</a></td><td>194</td><td>30</td><td>57.6</td><td>9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Zeta_toxin</td><td>Zeta toxin. This family consists of several bacterial zeta toxin proteins. Zeta toxin is thought to be part of a postregulational killing system in bacteria. It relies on antitoxin/toxin systems that secure stable inheritance of low and medium copy number plasmids during cell division and kill cells that have lost the plasmid.</td></tr>
<tr><td>440</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10851">PRK10851</a></td><td>353</td><td>26</td><td>57.4</td><td>4.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10851</td><td>sulfate/thiosulfate transporter subunit; Provisional</td></tr>
<tr><td>441</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13647">PRK13647</a></td><td>274</td><td>148</td><td>57.1</td><td>3.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cbiO</td><td>cobalt transporter ATP-binding subunit; Provisional</td></tr>
<tr><td>442</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK04195">PRK04195</a></td><td>482</td><td>16</td><td>57.0</td><td>6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK04195</td><td>replication factor C large subunit; Provisional</td></tr>
<tr><td>443</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04521">TIGR04521</a></td><td>277</td><td>49</td><td>57.0</td><td>7.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ECF_ATPase_2</td><td>energy-coupling factor transporter ATPase. Members of this family are ATP-binding cassette (ABC) proteins by homology, but belong to energy coupling factor (ECF) transport systems. The architecture in general is two ATPase subunits (or a double-length fusion protein), a T component, and a substrate capture (S) component that is highly variable, and may be interchangeable in genomes with only one T component. This model identifies many but not examples of the downstream member of the pair of ECF ATPases in Firmicutes and Mollicutes. </td></tr>
<tr><td>444</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06068">pfam06068</a></td><td>395</td><td>94</td><td>56.9</td><td>6.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIP49</td><td>TIP49 C-terminus. This family consists of the C-terminal region of several eukaryotic and archaeal RuvB-like 1 (Pontin or TIP49a) and RuvB-like 2 (Reptin or TIP49b) proteins. The N-terminal domain contains the pfam00004 domain. In zebrafish, the liebeskummer (lik) mutation, causes development of hyperplastic embryonic hearts. lik encodes Reptin, a component of a DNA-stimulated ATPase complex. Beta-catenin and Pontin, a DNA-stimulated ATPase that is often part of complexes with Reptin, are in the same genetic pathways. The Reptin/Pontin ratio serves to regulate heart growth during development, at least in part via the beta-catenin pathway. TBP-interacting protein 49 (TIP49) was originally identified as a TBP-binding protein, and two related proteins are encoded by individual genes, tip49a and b. Although the function of this gene family has not been elucidated, they are supposed to play a critical role in nuclear events because they interact with various kinds of nuclear factors and have DNA helicase activities.TIP49a has been suggested to act as an autoantigen in some patients with autoimmune diseases.</td></tr>
<tr><td>445</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09580">PRK09580</a></td><td>248</td><td>96</td><td>56.8</td><td>4.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>sufC</td><td>cysteine desulfurase ATPase component; Reviewed</td></tr>
<tr><td>446</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13849">PRK13849</a></td><td>231</td><td>25</td><td>56.6</td><td>7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13849</td><td>putative crown gall tumor protein VirC1; Provisional</td></tr>
<tr><td>447</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00882">cd00882</a></td><td>161</td><td>15</td><td>56.4</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ras_like_GTPase</td><td>Rat sarcoma (Ras)-like superfamily of small guanosine triphosphatases (GTPases). Ras-like GTPase superfamily. The Ras-like superfamily of small GTPases consists of several families with an extremely high degree of structural and functional similarity. The Ras superfamily is divided into at least four families in eukaryotes: the Ras, Rho, Rab, and Sar1/Arf families. This superfamily also includes proteins like the GTP translation factors, Era-like GTPases, and G-alpha chain of the heterotrimeric G proteins. Members of the Ras superfamily regulate a wide variety of cellular functions: the Ras family regulates gene expression, the Rho family regulates cytoskeletal reorganization and gene expression, the Rab and Sar1/Arf families regulate vesicle trafficking, and the Ran family regulates nucleocytoplasmic transport and microtubule organization. The GTP translation factor family regulates initiation, elongation, termination, and release in translation, and the Era-like GTPase family regulates cell division, sporulation, and DNA replication. Members of the Ras superfamily are identified by the GTP binding site, which is made up of five characteristic sequence motifs, and the switch I and switch II regions.</td></tr>
<tr><td>448</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14723">PRK14723</a></td><td>767</td><td>79</td><td>56.0</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>flhF</td><td>flagellar biosynthesis regulator FlhF; Provisional</td></tr>
<tr><td>449</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4555">COG4555</a></td><td>245</td><td>114</td><td>55.9</td><td>9.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NatA</td><td>ABC-type Na+ transport system, ATPase component NatA </td></tr>
<tr><td>450</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03223">cd03223</a></td><td>166</td><td>20</td><td>55.7</td><td>6.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCD_peroxisomal_ALDP</td><td>ATP-binding cassette domain of peroxisomal transporter, subfamily D. Peroxisomal ATP-binding cassette transporter (Pat) is involved in the import of very long-chain fatty acids (VLCFA) into the peroxisome. The peroxisomal membrane forms a permeability barrier for a wide variety of metabolites required for and formed during fatty acid beta-oxidation. To communicate with the cytoplasm and mitochondria, peroxisomes need dedicated proteins to transport such hydrophilic molecules across their membranes. X-linked adrenoleukodystrophy (X-ALD) is caused by mutations in the ALD gene, which encodes ALDP (adrenoleukodystrophy protein ), a peroxisomal integral membrane protein that is a member of the ATP-binding cassette (ABC) transporter protein family. The disease is characterized by a striking and unpredictable variation in phenotypic expression. Phenotypes include the rapidly progressive childhood cerebral form (CCALD), the milder adult form, adrenomyeloneuropathy (AMN), and variants without neurologic involvement (i.e. asymptomatic).</td></tr>
<tr><td>451</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1797">COG1797</a></td><td>451</td><td>51</td><td>55.6</td><td>8.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CobB</td><td>Cobyrinic acid a,c-diamide synthase  </td></tr>
<tr><td>452</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00611">TIGR00611</a></td><td>365</td><td>21</td><td>55.6</td><td>7.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recf</td><td>recF protein. All proteins in this family for which functions are known are DNA binding proteins that assist the filamentation of RecA onto DNA for the initiation of recombination or recombinational repair. This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University). </td></tr>
<tr><td>453</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1123">COG1123</a></td><td>539</td><td>33</td><td>55.5</td><td>5.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GsiA</td><td>ABC-type glutathione transport system ATPase component, contains duplicated ATPase domain </td></tr>
<tr><td>454</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03248">cd03248</a></td><td>226</td><td>44</td><td>55.4</td><td>5.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABCC_TAP</td><td>ATP-binding cassette domain of the Transporter Associated with Antigen Processing, subfamily C. TAP (Transporter Associated with Antigen Processing) is essential for peptide delivery from the cytosol into the lumen of the endoplasmic reticulum (ER), where these peptides are loaded on major histocompatibility complex (MHC) I molecules. Loaded MHC I leave the ER and display their antigenic cargo on the cell surface to cytotoxic T cells. Subsequently, virus-infected or malignantly transformed cells can be eliminated. TAP belongs to the large family of ATP-binding cassette (ABC) transporters, which translocate a vast variety of solutes across membranes.</td></tr>
<tr><td>455</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2109">COG2109</a></td><td>198</td><td>47</td><td>55.3</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BtuR</td><td>ATP:corrinoid adenosyltransferase </td></tr>
<tr><td>456</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03274">cd03274</a></td><td>212</td><td>33</td><td>55.1</td><td>5.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC4_euk</td><td>ATP-binding cassette domain of eukaryotic SMC4 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>457</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13020">pfam13020</a></td><td>91</td><td>66</td><td>55.0</td><td>16</td><td>[&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3883</td><td>Domain of unknown function (DUF3883). This is a domain is uncharacterized. It is found on restriction endonucleases.</td></tr>
<tr><td>458</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03249">cd03249</a></td><td>238</td><td>15</td><td>55.0</td><td>5.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_MTABC3_MDL1_MDL2</td><td>ATP-binding cassette domain of a mitochondrial protein MTABC3 and related proteins. MTABC3 (also known as ABCB6) is a mitochondrial ATP-binding cassette protein involved in iron homeostasis and one of four ABC transporters expressed in the mitochondrial inner membrane, the other three being MDL1(ABC7), MDL2, and ATM1. In fact, the yeast MDL1 (multidrug resistance-like protein 1) and MDL2 (multidrug resistance-like protein 2) transporters are also included in this CD. MDL1 is an ATP-dependent permease that acts as a high-copy suppressor of ATM1 and is thought to have a role in resistance to oxidative stress. Interestingly, subfamily B is more closely related to the carboxyl-terminal component of subfamily C than the two halves of ABCC molecules are with one another.</td></tr>
<tr><td>459</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03275">cd03275</a></td><td>247</td><td>23</td><td>54.9</td><td>5.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC1_euk</td><td>ATP-binding cassette domain of eukaryotic SMC1 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>460</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11776">PRK11776</a></td><td>460</td><td>80</td><td>54.8</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11776</td><td>ATP-dependent RNA helicase DbpA; Provisional</td></tr>
<tr><td>461</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4107">COG4107</a></td><td>258</td><td>146</td><td>54.8</td><td>4.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PhnK</td><td>ABC-type phosphonate transport system, ATPase component </td></tr>
<tr><td>462</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13341">PRK13341</a></td><td>725</td><td>68</td><td>54.7</td><td>8.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13341</td><td>recombination factor protein RarA/unknown domain fusion protein; Reviewed</td></tr>
<tr><td>463</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03215">pfam03215</a></td><td>517</td><td>74</td><td>54.4</td><td>8.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rad17</td><td>Rad17 cell cycle checkpoint protein. </td></tr>
<tr><td>464</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02546">TIGR02546</a></td><td>422</td><td>22</td><td>53.8</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>III_secr_ATP</td><td>type III secretion apparatus H+-transporting two-sector ATPase. </td></tr>
<tr><td>465</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13479">pfam13479</a></td><td>201</td><td>64</td><td>53.7</td><td>4.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_24</td><td>AAA domain. This AAA domain is found in a wide variety of presumed phage proteins.</td></tr>
<tr><td>466</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14246">PRK14246</a></td><td>257</td><td>23</td><td>53.6</td><td>6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14246</td><td>phosphate ABC transporter ATP-binding protein; Provisional</td></tr>
<tr><td>467</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09549">PRK09549</a></td><td>407</td><td>56</td><td>53.5</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>mtnW</td><td>2,3-diketo-5-methylthiopentyl-1-phosphate enolase; Reviewed</td></tr>
<tr><td>468</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00618">TIGR00618</a></td><td>1042</td><td>22</td><td>53.4</td><td>8.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>sbcc</td><td>exonuclease SbcC. All proteins in this family for which functions are known are part of an exonuclease complex with sbcD homologs. This complex is involved in the initiation of recombination to regulate the levels of palindromic sequences in DNA. This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University). </td></tr>
<tr><td>469</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09984">PRK09984</a></td><td>262</td><td>18</td><td>53.4</td><td>5.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09984</td><td>phosphonate/organophosphate ester transporter subunit; Provisional</td></tr>
<tr><td>470</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03272">cd03272</a></td><td>243</td><td>29</td><td>53.3</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_SMC3_euk</td><td>ATP-binding cassette domain of eukaryotic SMC3 proteins. The structural maintenance of chromosomes (SMC) proteins are large (approximately 110 to 170 kDa), and each is arranged into five recognizable domains. Amino-acid sequence homology of SMC proteins between species is largely confined to the amino- and carboxy-terminal globular domains. The amino-terminal domain contains a 'Walker A' nucleotide-binding domain (GxxGxGKS/T, in the single-letter amino-acid code), which by mutational studies has been shown to be essential in several proteins. The carboxy-terminal domain contains a sequence (the DA-box) that resembles a 'Walker B' motif, and a motif with homology to the signature sequence of the ATP-binding cassette (ABC) family of ATPases. The sequence homology within the carboxy-terminal domain is relatively high within the SMC1-SMC4 group, whereas SMC5 and SMC6 show some divergence in both of these sequences. In eukaryotic cells, the proteins are found as heterodimers of SMC1 paired with SMC3, SMC2 with SMC4, and SMC5 with SMC6 (formerly known as Rad18).</td></tr>
<tr><td>471</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01854">cd01854</a></td><td>211</td><td>99</td><td>53.1</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YjeQ_EngC</td><td>Ribosomal interacting GTPase YjeQ/EngC, a circularly permuted subfamily of the Ras GTPases. YjeQ (YloQ in Bacillus subtilis) is a ribosomal small subunit-dependent GTPase; hence also known as RsgA. YjeQ is a late-stage ribosomal biogenesis factor involved in the 30S subunit maturation, and it represents a protein family whose members are broadly conserved in bacteria and have been shown to be essential to the growth of E. coli and B. subtilis. Proteins of the YjeQ family contain all sequence motifs typical of the vast class of P-loop-containing GTPases, but show a circular permutation, with a G4-G1-G3 pattern of motifs as opposed to the regular G1-G3-G4 pattern seen in most GTPases. All YjeQ family proteins display a unique domain architecture, which includes an N-terminal OB-fold RNA-binding domain, the central permuted GTPase domain, and a zinc knuckle-like C-terminal cysteine domain.</td></tr>
<tr><td>472</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1219">COG1219</a></td><td>408</td><td>37</td><td>52.8</td><td>2.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ClpX</td><td>ATP-dependent protease Clp, ATPase subunit </td></tr>
<tr><td>473</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09825">PRK09825</a></td><td>176</td><td>17</td><td>52.7</td><td>6.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>idnK</td><td>D-gluconate kinase; Provisional</td></tr>
<tr><td>474</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02016">TIGR02016</a></td><td>296</td><td>196</td><td>52.6</td><td>9.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BchX</td><td>chlorophyllide reductase iron protein subunit X. This model represents the X subunit of the three-subunit enzyme, (bacterio)chlorophyllide reductase. This enzyme is responsible for the reduction of the chlorin B-ring and is closely related to the protochlorophyllide reductase complex which reduces the D-ring. Both of these complexes in turn are homologous to nitrogenase. This subunit is homologous to the nitrogenase component II, or "iron" protein. </td></tr>
<tr><td>475</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03797">TIGR03797</a></td><td>686</td><td>17</td><td>52.5</td><td>6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NHLM_micro_ABC2</td><td>NHLM bacteriocin system ABC transporter, ATP-binding protein. Members of this protein family are ABC transporter ATP-binding subunits, part of a three-gene putative bacteriocin transport operon. The other subunits include another ATP-binding subunit (TIGR03796), which has an N-terminal leader sequence cleavage domain, and an HlyD homolog (TIGR03794). In a number of genomes, members of protein families related to nitrile hydratase alpha subunit or to nif11 have undergone paralogous family expansions, with members possessing a putative bacteriocin cleavage region ending with a classic Gly-Gly motif. Those sets of putative bacteriocins, members of this protein family and its partners TIGR03794 and TIGR03796, and cyclodehydratase/docking scaffold fusion proteins of thiazole/oxazole biosynthesis frequently show correlated species distribution and co-clustering within many of those genomes. </td></tr>
<tr><td>476</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07015">pfam07015</a></td><td>231</td><td>25</td><td>52.4</td><td>9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VirC1</td><td>VirC1 protein. This family consists of several bacterial VirC1 proteins. In Agrobacterium tumefaciens, a cis-active 24-base-pair sequence adjacent to the right border of the T-DNA, called overdrive, stimulates tumor formation by increasing the level of T-DNA processing. It is thought that the virC operon which enhances T-DNA processing probably does so because the VirC1 protein interacts with overdrive. It has now been shown that the virC1 gene product binds to overdrive but not to the right border of T-DNA.</td></tr>
<tr><td>477</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01037">cd01037</a></td><td>80</td><td>48</td><td>52.3</td><td>33</td><td>[&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Restriction_endonuclease_like</td><td>Superfamily of nucleases including Short Patch Repair (Vsr) Endonucleases, archaeal Holliday junction resolvases, MutH methy-directed DNA mismatch-repair endonucleases, and catalytic domains of many restriction endonucleases, such as EcoRI, BamHI, and FokI</td></tr>
<tr><td>478</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4598">COG4598</a></td><td>256</td><td>62</td><td>52.3</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HisP</td><td>ABC-type histidine transport system, ATPase component </td></tr>
<tr><td>479</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00098">PRK00098</a></td><td>298</td><td>75</td><td>52.3</td><td>8.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00098</td><td>GTPase RsgA; Reviewed</td></tr>
<tr><td>480</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11248">PRK11248</a></td><td>255</td><td>19</td><td>52.2</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tauB</td><td>taurine transporter ATP-binding subunit; Provisional</td></tr>
<tr><td>481</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10412">pfam10412</a></td><td>386</td><td>40</td><td>52.1</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TrwB_AAD_bind</td><td>Type IV secretion-system coupling protein DNA-binding domain. The plasmid conjugative coupling protein TrwB forms hexamers from six structurally very similar protomers. This hexamer contains a central channel running from the cytosolic pole (made up by the AADs) to the membrane pole ending at the transmembrane pore shaped by 12 transmembrane helices, rendering an overall mushroom-like structure. The TrwB_AAD (all-alpha domain) domain appears to be the DNA-binding domain of the structure. TrwB, a basic integral inner-membrane nucleoside-triphosphate-binding protein, is the structural prototype for the type IV secretion system coupling proteins, a family of proteins essential for macromolecular transport between cells and export.</td></tr>
<tr><td>482</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06919">cd06919</a></td><td>111</td><td>44</td><td>52.0</td><td>9.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Asp_decarbox</td><td>Aspartate alpha-decarboxylase or L-aspartate 1-decarboxylase, a pyruvoyl group-dependent  decarboxylase in beta-alanine production. Decarboxylation of aspartate is  the major route of beta-alanine production in bacteria, and is catalyzed  by the enzyme L-aspartate decarboxylase (ADC), EC:4.1.1.11 which  requires a pyruvoyl group for its activity. The pyruvoyl cofactor is  covalently bound to the enzyme. The protein is synthesized as a  proenzyme and cleaved via self-processing at Gly23-Ser24 to yield an  alpha chain (C-terminal fragment) and beta chain (N-terminal fragment),  and the pyruvoyl group. Beta-alanine is required for the biosynthesis of  pantothenate, in which the enzyme plays a critical regulatory role. The  active site of the tetrameric enzyme is located at the interface of two  subunits, with a Lysine and a Histidine from the beta chain of one  subunit forming the active site with residues from the alpha chain of  the adjacent subunit. This alignment model spans the precursor (or both  beta and alpha chains) of aspartate decarboxylase.</td></tr>
<tr><td>483</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02768">TIGR02768</a></td><td>744</td><td>124</td><td>52.0</td><td>6.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TraA_Ti</td><td>Ti-type conjugative transfer relaxase TraA. This protein contains domains distinctive of a single strand exonuclease (N-terminus, MobA/MobL, pfam03389) as well as a helicase domain (central region, homologous to the corresponding region of the F-type relaxase TraI, TIGR02760). This protein likely fills the same role as TraI(F), nicking (at the oriT site) and unwinding the coiled plasmid prior to conjugative transfer.</td></tr>
<tr><td>484</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05342">PRK05342</a></td><td>412</td><td>36</td><td>51.9</td><td>7.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>clpX</td><td>ATP-dependent protease ATP-binding subunit ClpX; Provisional</td></tr>
<tr><td>485</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02868">TIGR02868</a></td><td>530</td><td>22</td><td>51.8</td><td>6.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CydC</td><td>thiol reductant ABC exporter, CydC subunit. The gene pair cydCD encodes an ABC-family transporter in which each gene contains an N-terminal membrane-spanning domain (pfam00664) and a C-terminal ATP-binding domain (pfam00005). In E. coli these genes were discovered as mutants which caused the terminal heme-copper oxidase complex cytochrome bd to fail to assemble. Recent work has shown that the transporter is involved in export of redox-active thiol compounds such as cysteine and glutathione. The linkage to assembly of the cytochrome bd complex is further supported by the conserved operon structure found outside the gammaproteobacteria (cydABCD) containing both the transporter and oxidase genes components. The genes used as the seed members for this model are all either found in the gammproteobacterial context or the CydABCD context. All members of this family scoring above trusted at the time of its creation were from genomes which encode a cytochrome bd complex.</td></tr>
<tr><td>486</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1195">COG1195</a></td><td>363</td><td>37</td><td>51.1</td><td>9.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecF</td><td>Recombinational DNA repair ATPase RecF </td></tr>
<tr><td>487</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03992">PRK03992</a></td><td>389</td><td>20</td><td>50.7</td><td>8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK03992</td><td>proteasome-activating nucleotidase; Provisional</td></tr>
<tr><td>488</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03241">cd03241</a></td><td>276</td><td>46</td><td>50.4</td><td>9.8</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_RecN</td><td>ATP-binding cassette domain of RecN. RecN ATPase involved in DNA repair; similar to ABC (ATP-binding cassette) transporter nucleotide-binding domain; ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds including sugars, ions, peptides, and more complex organic molecules. The nucleotide binding domain shows the highest similarity between all members of the family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>489</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01287">TIGR01287</a></td><td>275</td><td>33</td><td>50.1</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>nifH</td><td>nitrogenase iron protein. This model describes nitrogenase (EC 1.18.6.1) iron protein, also called nitrogenase reductase or nitrogenase component II. This model includes molybdenum-iron nitrogenase reductase (nifH), vanadium-iron nitrogenase reductase (vnfH), and iron-iron nitrogenase reductase (anfH). The model excludes the homologous protein from the light-independent protochlorophyllide reductase. </td></tr>
<tr><td>490</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4178">COG4178</a></td><td>604</td><td>25</td><td>50.1</td><td>8.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YddA</td><td>ABC-type uncharacterized transport system, permease and ATPase components </td></tr>
<tr><td>491</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03270">cd03270</a></td><td>226</td><td>40</td><td>50.1</td><td>7.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_UvrA_I</td><td>ATP-binding cassette domain I of the excision repair protein UvrA. Nucleotide excision repair in eubacteria is a process that repairs DNA damage by the removal of a 12-13-mer oligonucleotide containing the lesion. Recognition and cleavage of the damaged DNA is a multistep ATP-dependent reaction that requires the UvrA, UvrB, and UvrC proteins. Both UvrA and UvrB are ATPases, with UvrA having two ATP binding sites, which have the characteristic signature of the family of ABC proteins, and UvrB having one ATP binding site that is structurally related to that of helicases.</td></tr>
<tr><td>492</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00382">TIGR00382</a></td><td>413</td><td>36</td><td>50.1</td><td>9.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>clpX</td><td>endopeptidase Clp ATP-binding regulatory subunit (clpX). A member of the ATP-dependent proteases, ClpX has ATP-dependent chaperone activity and is required for specific ATP-dependent proteolytic activities expressed by ClpPX. The gene is also found to be involved in stress tolerance in Bacillus subtilis and is essential for the efficient acquisition of genes specifying type IA and IB restriction. </td></tr>
<tr><td>493</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00158">pfam00158</a></td><td>168</td><td>40</td><td>50.0</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Sigma54_activat</td><td>Sigma-54 interaction domain. </td></tr>
<tr><td>494</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08774">pfam08774</a></td><td>96</td><td>22</td><td>50.0</td><td>38</td><td>[&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VRR_NUC</td><td>VRR-NUC domain. </td></tr>
<tr><td>495</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14955">PRK14955</a></td><td>397</td><td>134</td><td>50.0</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14955</td><td>DNA polymerase III subunits gamma and tau; Provisional</td></tr>
<tr><td>496</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07952">PRK07952</a></td><td>244</td><td>33</td><td>49.9</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07952</td><td>DNA replication protein DnaC; Validated</td></tr>
<tr><td>497</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12270">cd12270</a></td><td>74</td><td>35</td><td>49.9</td><td>7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM_MTHFSD</td><td>RNA recognition motif in vertebrate methenyltetrahydrofolate synthetase domain-containing proteins. This subfamily corresponds to methenyltetrahydrofolate synthetase domain (MTHFSD), a putative RNA-binding protein found in various vertebrate species. It contains an N-terminal 5-formyltetrahydrofolate cyclo-ligase domain and a C-terminal RNA recognition motif (RRM), also termed RBD (RNA binding domain) or RNP (ribonucleoprotein domain). The biological role of MTHFSD remains unclear. </td></tr>
<tr><td>498</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02538">TIGR02538</a></td><td>564</td><td>40</td><td>49.5</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>type_IV_pilB</td><td>type IV-A pilus assembly ATPase PilB. This model describes a protein of type IV pilus biogenesis designated PilB in Pseudomonas aeruginosa but PilF in Neisseria gonorrhoeae; the more common usage, reflected here, is PilB. This protein is an ATPase involved in protein export for pilin assembly and is closely related to GspE (TIGR02533) of type II secretion, also called the main terminal branch of the general secretion pathway. Note that type IV pilus systems are often divided into type IV-A and IV-B, with the latter group including bundle-forming pilus, mannose-sensitive hemagglutinin, etc. Members of this family are found in type IV-A systems. </td></tr>
<tr><td>499</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0523">COG0523</a></td><td>323</td><td>31</td><td>49.3</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YejR</td><td>GTPase, G3E family </td></tr>
<tr><td>500</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1936">COG1936</a></td><td>180</td><td>24</td><td>49.3</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fap7</td><td>Broad-specificity NMP kinase </td></tr>
</table>
</body>
