<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02579">TIGR02579</a></td><td>83</td><td>71</td><td>100.0</td><td>5.2E-36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------------]</td><td>cas_csx3</td><td>CRISPR-associated protein, Csx3 family. Members of this family are found encoded in CRISPR-associated (cas) gene clusters, near CRISPR repeats, in the genomes of several different thermophiles: Archaeoglobus fulgidus (archaeal), Aquifex aeolicus (Aquificae), Dictyoglomus thermophilum (Dictyoglomi), and a thermophilic Synechococcus (Cyanobacteria). It is not yet assigned to a specific CRISPR/cas subtype (hence the x designation csx3).</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09681">cd09681</a></td><td>83</td><td>71</td><td>100.0</td><td>5.2E-36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------------]</td><td>Csx3_III-U</td><td>CRISPR/Cas system-associated protein Csx3. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Small protein in some cases fused to Csx1 (COG1517) family domains</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09740">cd09740</a></td><td>84</td><td>71</td><td>100.0</td><td>1E-33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------------]</td><td>Csx3_III-U</td><td>CRISPR/Cas system-associated protein Csx3. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Small protein in some cases fused to Csx1 (COG1517) family domains</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09620">pfam09620</a></td><td>80</td><td>61</td><td>100.0</td><td>1.3E-32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------------]</td><td>Cas_csx3</td><td>CRISPR-associated protein (Cas_csx3). This entry is encoded in CRISPR-associated (cas) gene clusters, near CRISPR repeats, in the genomes of several different thermophiles: Archaeoglobus fulgidus (archaeal), Aquifex aeolicus (Aquificae), Dictyoglomus thermophilum (Dictyoglomi), and a thermophilic Synechococcus (Cyanobacteria). It is not yet assigned to a specific CRISPR/cas subtype (hence the x designation csx3).</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10887">PRK10887</a></td><td>443</td><td>35</td><td>82.3</td><td>1.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>glmM</td><td>phosphoglucosamine mutase; Provisional</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05802">cd05802</a></td><td>434</td><td>35</td><td>56.3</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>GlmM</td><td>GlmM is a bacterial phosphoglucosamine mutase (PNGM) that belongs to the alpha-D-phosphohexomutase superfamily. It is required for the interconversion of glucosamine-6-phosphate and glucosamine-1-phosphate in the biosynthetic pathway of UDP-N-acetylglucosamine, an essential precursor to components of the cell envelope.  In order to be active, GlmM must be phosphorylated, which can occur via autophosphorylation or by the Ser/Thr kinase StkP. GlmM functions in a classical ping-pong bi-bi mechanism with glucosamine-1,6-diphosphate as an intermediate.  Other members of the alpha-D-phosphohexomutase superfamily include phosphoglucosamine mutase (PNGM), phosphoacetylglucosamine mutase (PAGM), the bacterial phosphomannomutase ManB, and the bifunctional phosphomannomutase/phosphoglucomutase (PMM/PGM). Each of these enzymes has four domains with a centrally located active site formed by four loops, one from each domain. All four domains are included in this alignment model.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14314">PRK14314</a></td><td>450</td><td>35</td><td>53.2</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>glmM</td><td>phosphoglucosamine mutase; Provisional</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09887">cd09887</a></td><td>82</td><td>14</td><td>49.1</td><td>9.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------]</td><td>NGN_Arch</td><td>Archaeal N-Utilization Substance G (NusG) N-terminal (NGN) domain. The N-Utilization Substance G (NusG) protein and its eukaryotic homolog, Spt5, are involved in transcription elongation and termination. Transcription in archaea has a eukaryotic-type transcription apparatus, but contains bacterial-type transcription factors. NusG is one of the few archaeal transcription factors that has orthologs in both bacteria and eukaryotes. Archaeal NusG is similar to bacterial NusG, composed of an NGN domain and a Kyrpides Ouzounis and Woese (KOW) repeat. The eukaryotic ortholog, Spt5, is a large protein composed of an acidic N-terminus, an NGN domain, and multiple KOW motifs at its C-terminus. NusG was originally discovered as a N-dependent antitermination enhancing activity in Escherichia coli and has a variety of functions, such as being involved in RNA polymerase elongation and Rho-termination in bacteria. Archaeal NusG forms a complex with DNA-directed RNA polymerase subunit E (rpoE) that is similar to the Spt5-Spt4 complex in eukaryotes.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14147">pfam14147</a></td><td>52</td><td>11</td><td>48.9</td><td>9.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Spore_YhaL</td><td>Sporulation protein YhaL. This family of proteins is involved in sporulation. In B. subtilis its expression is regulated by the early mother-cell-specific transcription factor sigma-E.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02878">pfam02878</a></td><td>138</td><td>35</td><td>48.3</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>PGM_PMM_I</td><td>Phosphoglucomutase/phosphomannomutase, alpha/beta/alpha domain I. </td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01455">TIGR01455</a></td><td>443</td><td>53</td><td>45.4</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------------]</td><td>glmM</td><td>phosphoglucosamine mutase. This model describes GlmM, phosphoglucosamine mutase, also designated in MrsA and YhbF E. coli, UreC in Helicobacter pylori, and femR315 or FemD in Staphlococcus aureus. It converts glucosamine-6-phosphate to glucosamine-1-phosphate as part of the pathway toward UDP-N-acetylglucosamine for peptidoglycan and lipopolysaccharides. </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04903">cd04903</a></td><td>71</td><td>28</td><td>44.1</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ACT_LSD</td><td>C-terminal ACT domain of the L-serine dehydratase (LSD), iron-sulfur-dependent, beta subunit. The C-terminal ACT domain of the L-serine dehydratase (LSD), iron-sulfur-dependent, beta subunit, found in various bacterial anaerobes such as Clostridium, Bacillis, and Treponema species. These enzymes catalyze the deamination of L-serine, producing pyruvate and ammonia. Unlike the eukaryotic L-serine dehydratase, which requires the pyridoxal-5'-phosphate (PLP) cofactor, the prokaryotic L-serine dehydratase contains an </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13816">PRK13816</a></td><td>131</td><td>14</td><td>43.3</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;]</td><td>PRK13816</td><td>ribosome-binding factor A; Provisional</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03226">cd03226</a></td><td>205</td><td>13</td><td>40.1</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------]</td><td>ABC_cobalt_CbiO_domain2</td><td>Second domain of the ATP-binding cassette component of cobalt transport system. Domain II of the ABC component of a cobalt transport family found in bacteria, archaea, and eukaryota. The transition metal cobalt is an essential component of many enzymes and must be transported into cells in appropriate amounts when needed. The CbiMNQO family ABC transport system is involved in cobalt transport in association with the cobalamin (vitamin B12) biosynthetic pathways. Most cobalt (Cbi) transport systems possess a separate CbiN component, the cobalt-binding periplasmic protein, and they are encoded by the conserved gene cluster cbiMNQO. Both the CbiM and CbiQ proteins are integral cytoplasmic membrane proteins, and the CbiO protein has the linker peptide and the Walker A and B motifs commonly found in the ATPase components of the ABC-type transport systems.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1109">COG1109</a></td><td>464</td><td>35</td><td>36.7</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>ManB</td><td>Phosphomannomutase </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06129">pfam06129</a></td><td>109</td><td>20</td><td>34.1</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;]</td><td>Chordopox_G3</td><td>Chordopoxvirus G3 protein. This family consists of several Chordopoxvirus specific G3 proteins. The function of this family is unknown.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03711">cd03711</a></td><td>78</td><td>39</td><td>31.6</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Tet_C</td><td>Tet_C: C-terminus of ribosomal protection proteins Tet(M) and Tet(O). This domain has homology to the C terminal domains of the elongation factors EF-G and EF-2. Tet(M) and Tet(O) catalyze the release of tetracycline (Tc) from the ribosome in a GTP-dependent manner thereby mediating Tc resistance.  Tcs are broad-spectrum antibiotics.  Typical Tcs bind to the ribosome and inhibit the elongation phase of protein synthesis, by inhibiting the  occupation of site A by aminoacyl-tRNA.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04398">pfam04398</a></td><td>111</td><td>49</td><td>30.5</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF538</td><td>Protein of unknown function, DUF538. This family consists of several plant proteins of unknown function.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02420">cd02420</a></td><td>125</td><td>33</td><td>29.0</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_C39D</td><td>A sub-family of peptidase family C39. Peptidase family C39 mostly contains bacteriocin-processing endopeptidases from bacteria. The cysteine peptidases in family C39 cleave the "double-glycine" leader peptides from the precursors of various bacteriocins (mostly non-lantibiotic). The cleavage is mediated by the transporter as part of the secretion process. Bacteriocins are antibiotic proteins secreted by some species of bacteria that inhibit the growth of other bacterial species. The bacteriocin is synthesized as a precursor with an N-terminal leader peptide, and processing involves removal of the leader peptide by cleavage at a Gly-Gly bond, followed by translocation of the mature bacteriocin across the cytoplasmic membrane. Most endopeptidases of family C39 are N-terminal domains in larger proteins (ABC transporters) that serve both functions. The proposed protease active site is conserved in this sub-family.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00212">cd00212</a></td><td>78</td><td>32</td><td>28.8</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PTS_IIB_glc</td><td>PTS_IIB, PTS system, glucose/sucrose specific IIB subunit. The bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS) is a multi-protein system involved in the regulation of a variety of metabolic and transcriptional processes. This family is one of four structurally and functionally distinct group IIB PTS system cytoplasmic enzymes, necessary for the uptake of carbohydrates across the cytoplasmic membrane and their phosphorylation</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05352">pfam05352</a></td><td>281</td><td>26</td><td>28.8</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;]</td><td>Phage_connector</td><td>Phage Connector (GP10). The head-tail connector of bacteriophage 29 is composed of 12 36 kDa subunits with 12 fold symmetry. It is the central component of a rotary motor that packages the genomic dsDNA into pre-formed proheads. This motor consists of the head-tail connector, surrounded by a 29-encoded, 174-base, RNA and a viral ATPase protein.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11890">pfam11890</a></td><td>81</td><td>29</td><td>27.8</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3410</td><td>Domain of unknown function (DUF3410). This domain is functionally uncharacterized. This domain is found in bacteria. This presumed domain is about 90 amino acids in length. This domain is found associated with pfam02826, pfam00389. This domain has a conserved RRE sequence motif.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13304">pfam13304</a></td><td>144</td><td>12</td><td>27.3</td><td>33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------]</td><td>AAA_21</td><td>AAA domain. </td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01788">pfam01788</a></td><td>40</td><td>16</td><td>27.2</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PsbJ</td><td>PsbJ. This family consists of the photosystem II reaction centre protein PsbJ from plants and Cyanobacteria. In Synechocystis sp. PCC 6803 PsbJ regulates the number of photosystem II centres in thylakoid membranes, it is a predicted 4kDa protein with one membrane spanning domain.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00099">TIGR00099</a></td><td>256</td><td>55</td><td>27.1</td><td>2.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------------&nbsp;]</td><td>Cof-subfamily</td><td>Cof subfamily of IIB subfamily of haloacid dehalogenase superfamily. This subfamily of sequences falls within the Class-IIB subfamily (TIGR01484) of the Haloacid Dehalogenase superfamily of aspartate-nucleophile hydrolases. The use of the name "Cof" as an identifier here is arbitrary and refers to the E. coli Cof protein. This subfamily is notable for the large number of recent paralogs in many species. Listeria, for instance, has 12, Clostridium, Lactococcus and Streptococcus pneumoniae have 8 each, Enterococcus and Salmonella have 7 each, and Bacillus subtilus, Mycoplasma, Staphylococcus and E. coli have 6 each. This high degree of gene duplication is limited to the gamma proteobacteria and low-GC gram positive lineages. The profusion of genes in this subfamily is not coupled with a high degree of divergence, so it is impossible to determine an accurate phylogeny at the equivalog level. Considering the relationship of this subfamily to the other known members of the HAD-IIB subfamily (TIGR01484), sucrose and trehalose phosphatases and phosphomannomutase, it seems a reasonable hypothesis that these enzymes act on phosphorylated sugars. Possibly the diversification of genes in this subfamily represents the diverse sugars and polysaccharides that various bacteria find in their biological niches. The members of this subfamily are restricted almost exclusively to bacteria (one sequences from S. pombe scores above trusted, while another is between trusted and noise). It is notable that no archaea are found in this group, the closest relations to the archaea found here being two Deinococcus sequences. </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09120">PRK09120</a></td><td>275</td><td>50</td><td>26.4</td><td>43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09120</td><td>p-hydroxycinnamoyl CoA hydratase/lyase; Validated</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13539">PRK13539</a></td><td>207</td><td>11</td><td>25.9</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------]</td><td>PRK13539</td><td>cytochrome c biogenesis protein CcmA; Provisional</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03791">TIGR03791</a></td><td>291</td><td>22</td><td>25.7</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------]</td><td>TTQ_mauG</td><td>tryptophan tryptophylquinone biosynthesis enzyme MauG. Members of this protein family are the tryptophan tryptophylquinone biosynthesis (TTQ) enzyme MauG, as found in Methylobacterium extorquens and related species. This protein is required to complete the maturation of the TTQ cofactor in the methylamine dehydrogenase light (beta) chain.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14323">PRK14323</a></td><td>440</td><td>31</td><td>25.5</td><td>63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------]</td><td>glmM</td><td>phosphoglucosamine mutase; Provisional</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10682">pfam10682</a></td><td>220</td><td>31</td><td>25.5</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------]</td><td>UL40</td><td>Glycoprotein of human cytomegalovirus HHV-5. This is glycoprotein UL40 from human cytomegalovirus or herpesvirus 5. The signal sequence of the UL40 polypeptide contains an HLA-E ligand identical with HLA-Cw*0304. The first 37 residues of UL40, including this ligand, are predicted to encode a signal peptide. The virus thus prevents the lysis by NK (natural killer) cells of the cell it has invaded.</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08314">PRK08314</a></td><td>546</td><td>26</td><td>25.0</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08314</td><td>long-chain-fatty-acid--CoA ligase; Validated</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14315">PRK14315</a></td><td>448</td><td>35</td><td>24.8</td><td>96</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>glmM</td><td>phosphoglucosamine mutase; Provisional</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10006">pfam10006</a></td><td>69</td><td>32</td><td>24.4</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2249</td><td>Uncharacterized conserved protein (DUF2249). Members of this family of hypothetical bacterial proteins have no known function.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03221">cd03221</a></td><td>144</td><td>12</td><td>23.4</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------]</td><td>ABCF_EF-3</td><td>ATP-binding cassette domain of elongation factor 3, subfamily F. Elongation factor 3 (EF-3) is a cytosolic protein required by fungal ribosomes for in vitro protein synthesis and for in vivo growth. EF-3 stimulates the binding of the EF-1: GTP: aa-tRNA ternary complex to the ribosomal A site by facilitated release of the deacylated tRNA from the E site. The reaction requires ATP hydrolysis. EF-3 contains two ATP nucleotide binding sequence (NBS) motifs. NBSI is sufficient for the intrinsic ATPase activity. NBSII is essential for the ribosome-stimulated functions.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06072">PRK06072</a></td><td>248</td><td>45</td><td>23.2</td><td>1.4E+02</td><td>[&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06072</td><td>enoyl-CoA hydratase; Provisional</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03608">TIGR03608</a></td><td>206</td><td>12</td><td>22.8</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------]</td><td>L_ocin_972_ABC</td><td>putative bacteriocin export ABC transporter, lactococcin 972 group. A gene pair with a fairly wide distribution consists of a polypeptide related to the lactococcin 972 (see TIGR01653) and multiple-membrane-spanning putative immunity protein (see TIGR01654). This model represents a small clade within the ABC transporters that regularly are found adjacent to these bacteriocin system gene pairs and are likely serve as export proteins. </td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13872">cd13872</a></td><td>141</td><td>15</td><td>21.8</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CuRO_2_AAO_like_1</td><td>The second cupredoxin domain of plant pollen multicopper oxidase homologous to ascorbate oxidase. The proteins in this subfamily are expressed in plant pollen. They share homology to ascorbate oxidase and other members of the blue copper oxidase family. The expression of the protein is detected during germination and pollen tube growth. Ascorbate oxidase catalyzes the oxidation of ascorbic acid to dehydroascorbic acid. It is a member of the multicopper oxidase (MCO) family that couples oxidation of substrates with reduction of dioxygen to water. Although MCOs have diverse functions, majority of them have three cupredoxin domain repeats that include one mononuclear and one trinuclear copper center. The copper ions are bound in several sites: Type 1, Type 2, and/or Type 3. The ensemble of types 2 and 3 copper is called a trinuclear cluster. MCOs oxidize their substrate by accepting electrons at a mononuclear copper center and transferring them to the active site trinuclear copper center. The cupredoxin domain 2 of 3-domain MCOs has lost the ability to bind copper.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08559">PRK08559</a></td><td>153</td><td>44</td><td>21.5</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------]</td><td>nusG</td><td>transcription antitermination protein NusG; Validated</td></tr>
</table>
</body>
