<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02801">TIGR02801</a></td><td>129</td><td>68</td><td>34.6</td><td>58</td><td>[---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tolR</td><td>TolR protein. The model describes the inner membrane protein TolR, part of the TolR/TolQ complex that transduces energy from the proton-motive force, through TolA, to an outer membrane complex made up of TolB and Pal (peptidoglycan-associated lipoprotein). The complex is required to maintain outer membrane integrity, and defects may cause a defect in the import of some organic compounds in addition to the resulting morphologic. While several gene pairs homologous to talR and tolQ may be found in a single genome, but the scope of this model is set to favor finding only bone fide TolR, supported by operon structure as well as by score. </td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06183">pfam06183</a></td><td>65</td><td>22</td><td>23.6</td><td>79</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DinI</td><td>DinI-like family. This family of short proteins includes DNA-damage-inducible protein I (DinI) and related proteins. The SOS response, a set of cellular phenomena exhibited by eubacteria, is initiated by various causes that include DNA damage-induced replication arrest, and is positively regulated by the co- protease activity of RecA. Escherichia coli DinI, a LexA-regulated SOS gene product, shuts off the initiation of the SOS response when overexpressed in vivo. Biochemical and genetic studies indicated that DinI physically interacts with RecA to inhibit its co-protease activity. The structure of DinI is known.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK04148">PRK04148</a></td><td>134</td><td>30</td><td>23.3</td><td>41</td><td>[&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK04148</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00128">PRK00128</a></td><td>286</td><td>46</td><td>18.5</td><td>1.4E+02</td><td>[&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ipk</td><td>4-diphosphocytidyl-2-C-methyl-D-erythritol kinase; Provisional</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08898">PRK08898</a></td><td>394</td><td>16</td><td>14.8</td><td>63</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08898</td><td>coproporphyrinogen III oxidase; Provisional</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11289">cd11289</a></td><td>92</td><td>42</td><td>13.7</td><td>2.5E+02</td><td>[&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>gelsolin_S2_like</td><td>Gelsolin sub-domain 2-like domain found in gelsolin, severin, villin, and related proteins. Gelsolin repeats occur in gelsolin, severin, villin, advillin, villidin, supervillin, flightless, quail, fragmin, and other proteins, usually in several copies. They co-occur with villin headpiece domains, leucine-rich repeats, and several other domains. These gelsolin-related actin binding proteins (GRABPs) play regulatory roles in the assembly and disassembly of actin filaments; they are involved in F-actin capping, uncapping, severing, or the nucleation of actin filaments. Severing of actin filaments is Ca2+ dependent. Villins are also linked to generating bundles of F-actin with uniform filament polarity, which is most likely mediated by their extra villin headpiece domain. Many family members have also adopted functions in the nucleus, including the regulation of transcription. Supervillin, gelsolin, and flightless I are involved in intracellular signaling via nuclear hormone receptors. The gelsolin-like domain is distantly related to the actin depolymerizing domains found in cofilin and similar proteins.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06397">cd06397</a></td><td>82</td><td>36</td><td>13.5</td><td>1.3E+02</td><td>[&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PB1_UP1</td><td>Uncharacterized protein 1. The PB1 domain is a modular domain mediating specific protein-protein interaction which play a role in many critical cell processes, such as osteoclastogenesis, angiogenesis, early cardiovascular development, and cell polarity. A canonical PB1-PB1 interaction, which involves heterodimerization of two PB1 domain, is required for the formation of macromolecular signaling complexes ensuring specificity and fidelity during cellular signaling. The interaction between two PB1 domain depends on the type of PB1. There are three types of PB1 domains: type I which contains an OPCA motif, acidic aminoacid cluster, type II which contains a basic cluster, and type I/II which contains both an OPCA motif and a basic cluster. Interactions of PB1 domains with other protein domains have been described as noncanonical PB1-interactions.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03598">TIGR03598</a></td><td>179</td><td>64</td><td>13.2</td><td>2E+02</td><td>[&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GTPase_YsxC</td><td>ribosome biogenesis GTP-binding protein YsxC/EngB. Members of this protein family are a GTPase associated with ribosome biogenesis, typified by YsxC from Bacillus subutilis. The family is widely but not universally distributed among bacteria. Members commonly are called EngB based on homology to EngA, one of several other GTPases of ribosome biogenesis. Cutoffs as set find essentially all bacterial members, but also identify large numbers of eukaryotic (probably organellar) sequences. This protein is found in about 80 percent of bacterial genomes. </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05798">pfam05798</a></td><td>75</td><td>35</td><td>12.1</td><td>1.4E+02</td><td>[---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Phage_FRD3</td><td>Bacteriophage FRD3 protein. This family consists of bacteriophage FRD3 proteins.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01876">cd01876</a></td><td>170</td><td>63</td><td>12.0</td><td>2.6E+02</td><td>[&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YihA_EngB</td><td>YihA (EngB) GTPase family. The YihA (EngB) subfamily of GTPases is typified by the E. coli YihA, an essential protein involved in cell division control. YihA and its orthologs are small proteins that typically contain less than 200 amino acid residues and consists of the GTPase domain only (some of the eukaryotic homologs contain an N-terminal extension of about 120 residues that might be involved in organellar targeting). Homologs of yihA are found in most Gram-positive and Gram-negative pathogenic bacteria, with the exception of Mycobacterium tuberculosis. The broad-spectrum nature of YihA and its essentiality for cell viability in bacteria make it an attractive antibacterial target.</td></tr>
</table>
</body>
