<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03555">TIGR03555</a></td><td>325</td><td>284</td><td>100.0</td><td>2.1E-70</td><td>[&nbsp;------------------------------------------------]</td><td>F420_mer</td><td>5,10-methylenetetrahydromethanopterin reductase. Members of this protein family are 5,10-methylenetetrahydromethanopterin reductase, an F420-dependent enzyme of methanogenesis. It is restricted to the Archaea. </td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02271">PRK02271</a></td><td>325</td><td>282</td><td>100.0</td><td>1.3E-65</td><td>[-------------------------------------------------]</td><td>PRK02271</td><td>methylenetetrahydromethanopterin reductase; Provisional</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00296">pfam00296</a></td><td>260</td><td>195</td><td>99.9</td><td>7.1E-27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Bac_luciferase</td><td>Luciferase-like monooxygenase. </td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2141">COG2141</a></td><td>336</td><td>256</td><td>99.9</td><td>7.7E-24</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>SsuD</td><td>Flavin-dependent oxidoreductase, luciferase family (includes alkanesulfonate monooxygenase SsuD and methylene tetrahydromethanopterin reductase) </td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03619">TIGR03619</a></td><td>246</td><td>179</td><td>99.7</td><td>1.5E-17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_Rv2161c</td><td>probable F420-dependent oxidoreductase, Rv2161c family. Coenzyme F420 has a limited phylogenetic distribution, including methanogenic archaea, Mycobacterium tuberculosis and related species, Colwellia psychrerythraea 34H, Rhodopseudomonas palustris HaA2, and others. Partial phylogenetic profiling identifies protein subfamilies, within the larger family called luciferase-like monooxygenanases (pfam00296), that appear only in F420-positive genomes and are likely to be F420-dependent. This model describes a domain found in a distinctive subset of bacterial luciferase homologs, found only in F420-biosynthesizing members of the Actinobacteria. </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03560">TIGR03560</a></td><td>227</td><td>176</td><td>99.5</td><td>1.1E-14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_Rv1855c</td><td>probable F420-dependent oxidoreductase, Rv1855c family. Coenzyme F420 has a limited phylogenetic distribution, including methanogenic archaea, Mycobacterium tuberculosis and related species, Colwellia psychrerythraea 34H, Rhodopseudomonas palustris HaA2, and others. Partial phylogenetic profiling identifies protein subfamilies, within the larger family called luciferase-like monooxygenanases (pfam00296), that appear only in F420-positive genomes and are likely to be F420-dependent. This model describes one such subfamily, exemplified by Rv1855c from Mycobacterium tuberculosis. </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03842">TIGR03842</a></td><td>330</td><td>244</td><td>99.4</td><td>4.2E-13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>F420_CPS_4043</td><td>F420-dependent oxidoreductase, CPS_4043 family. This model represents a family of putative F420-dependent oxidoreductases, fairly closely related to 5,10-methylenetetrahydromethanopterin reductase (mer, TIGR03555), both within the bacterial luciferase-like monoxygenase (LLM) family. A fairly deep split (to about 40 % sequence identity) in the present family separates a strictly Actinobacterial clade from an alpha/beta/gamma-proteobacterial clade, in which the member is often the only apparent F420-dependent LLM family member. The specific function, and whether Actinobacterial and Proteobacterial clades differ in function, are unknown. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01097">cd01097</a></td><td>202</td><td>151</td><td>99.4</td><td>1.2E-12</td><td>[----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Tetrahydromethanopterin_reductase</td><td>N5,N10-methylenetetrahydromethanopterin reductase (Mer) catalyzes the reduction of N5,N10-methylenetetrahydromethanopterin with reduced coenzyme F420 to N5-methyltetrahydromethanopterin and oxidized coenzyme F420.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03557">TIGR03557</a></td><td>316</td><td>163</td><td>99.3</td><td>1.1E-12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_G6P_family</td><td>F420-dependent oxidoreductase, G6PDH family. Members of this protein family include F420-dependent glucose-6-phosphate dehydrogenases (TIGR03554) and related proteins. All members of this family come from species that synthesize coenzyme F420, with the exception of those that belong to TIGR03885, a clade within this family in which cofactor binding may instead be directed to FMN. </td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03559">TIGR03559</a></td><td>325</td><td>257</td><td>99.2</td><td>2E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;]</td><td>F420_Rv3520c</td><td>probable F420-dependent oxidoreductase, Rv3520c family. Members of this protein family are predicted to be oxidoreductases dependent on coenzyme F420. The family includes a single member in Mycobacterium tuberculosis (Rv3520c/MT3621) but four in Mycobacterium smegmatis. Prediction that this family is F420-dependent is based primarily on Partial Phylogenetic Profiling vs. F420 biosynthesis. </td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03857">TIGR03857</a></td><td>329</td><td>179</td><td>99.2</td><td>2E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_MSMEG_2249</td><td>probable F420-dependent oxidoreductase, MSMEG_2249 family. Coenzyme F420 has a limited phylogenetic distribution, including methanogenic archaea, Mycobacterium tuberculosis and related species, Colwellia psychrerythraea 34H, Rhodopseudomonas palustris HaA2, and others. Partial phylogenetic profiling identifies protein subfamilies, within the larger family called luciferase-like monooxygenanases (pfam00296), that appear only in F420-positive genomes and are likely to be F420-dependent. This model describes a distinctive subfamily, found only in F420-biosynthesizing members of the Actinobacteria of the bacterial luciferase-like monooxygenase (LLM) superfamily. </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04024">TIGR04024</a></td><td>330</td><td>243</td><td>99.1</td><td>6.7E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>F420_NP1902A</td><td>coenzyme F420-dependent oxidoreductase, NP1902A family. This subfamily of the luciferase-like monooxygenases is restricted to the order Halobacteriales. SIMBAL analysis strongly suggests this oxidoreductase binds coenzyme F420 rather than FMN. Occasional annotations of members of this family as N5,N10-methylenetetrahydromethanopterin reductase appear to represent overly aggressive transfer of annotation. </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04465">TIGR04465</a></td><td>364</td><td>199</td><td>99.1</td><td>9.7E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ArgArg_F420</td><td>TAT-translocated F420-dependent dehydrogenase, FGD2 family. Members of this family are F420-binding enzymes with a proven functional N-terminal twin-arginine translocation (TAT) signal. Members are homologous to the cytosolic F420-dependent glucose-6-phosphate dehydrogenase but do not share the same function.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04020">TIGR04020</a></td><td>341</td><td>165</td><td>99.0</td><td>2.3E-10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>seco_metab_LLM</td><td>natural product biosynthesis luciferase-like monooxygenase domain. This model describes a subfamily within the bacterial luciferase-like monooxygenase (LLM) family that regularly occurs within large non-ribosomal protein synthases/polyketide synthases, but also as small proteins. The LLM family includes members that bind either FMN or F420, and FMN is more likely in this case because many members are from species that lack F420 biosynthesis capability. An example member is the MupA protein of mupirocin biosynthesis in Pseudomonas fluorescens NCIMB 10586.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04027">TIGR04027</a></td><td>326</td><td>232</td><td>99.0</td><td>7.7E-11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;]</td><td>LLM_KPN_01858</td><td>putative FMN-dependent luciferase-like monooxygenase, KPN_01858 family. This protein family consists of luciferase-like monooxygenases (LLM), and include KPN_01858 from Klebsiella pneumoniae as a representative member. Most are from species that lack F420 biosynthesis, so the family is likely to bind FMN as its cofactor. This family is closely associated with a binding protein-dependent ABC transporter, suggesting a role in catabolism. </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03621">TIGR03621</a></td><td>295</td><td>223</td><td>99.0</td><td>3.8E-10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>F420_MSMEG_2516</td><td>probable F420-dependent oxidoreductase, MSMEG_2516 family. Coenzyme F420 is produced by methanogenic archaea, a number of the Actinomycetes (including Mycobacterium tuberculosis), and rare members of other lineages. The resulting information-rich phylogenetic profile identifies candidate F420-dependent oxidoreductases within the family of luciferase-like enzymes (pfam00296), where the species range for the subfamily encompasses many F420-positive genomes without straying beyond. This family is uncharacterized, and named for member MSMEG_2516 from Mycobacterium smegmatis. </td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03858">TIGR03858</a></td><td>337</td><td>255</td><td>99.0</td><td>6.4E-10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------------------]</td><td>LLM_2I7G</td><td>probable oxidoreductase, LLM family. This model describes a highly conserved, somewhat broadly distributed family withing the luciferase-like monooxygenase (LLM) superfamily. Most members are from species incapable of synthesizing coenzyme F420, bound by some members of the LLM superfamily. Members, therefore, are more likely to use FMN as a cofactor.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03571">TIGR03571</a></td><td>298</td><td>102</td><td>98.9</td><td>6.4E-10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>lucif_BA3436</td><td>luciferase-type oxidoreductase, BA3436 family. This family is a distinct subgroup among members of the luciferase monooxygenase domain family. The larger family contains both FMN-binding enzymes (luciferase, alkane monooxygenase) and F420-binding enzymes (methylenetetrahydromethanopterin reductase, secondary alcohol dehydrogenase, glucose-6-phosphate dehydrogenase). Although some members of the domain family bind coenzyme F420 rather than FMN, members of this family are from species that lack the genes for F420 biosynthesis. A crystal structure, but not function, is known (but unpublished) for the member from Bacillus cereus, PDB|2B81. </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03558">TIGR03558</a></td><td>323</td><td>230</td><td>98.9</td><td>1.3E-09</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>oxido_grp_1</td><td>luciferase family oxidoreductase, group 1. The Pfam domain family pfam00296 is named for luciferase-like monooxygenases, but the family also contains several coenzyme F420-dependent enzymes. This protein family represents a well-resolved clade within family pfam00296 and shows no restriction to coenzyme F420-positive species, unlike some other clades within pfam00296. </td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03617">TIGR03617</a></td><td>318</td><td>249</td><td>98.8</td><td>5.9E-09</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;]</td><td>F420_MSMEG_2256</td><td>probable F420-dependent oxidoreductase, MSMEG_2256 family. Coenzyme F420 has a limited phylogenetic distribution, including methanogenic archaea, Mycobacterium tuberculosis and related species, Colwellia psychrerythraea 34H, Rhodopseudomonas palustris HaA2, and others. Partial phylogenetic profiling identifies protein subfamilies, within the larger family called luciferase-like monooxygenanases (pfam00296), that appear only in F420-positive genomes and are likely to be F420-dependent. This model describes one such subfamily, exemplified by MSMEG_2256 from Mycobacterium smegmatis. </td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03554">TIGR03554</a></td><td>331</td><td>173</td><td>98.6</td><td>6.2E-08</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_G6P_DH</td><td>glucose-6-phosphate dehydrogenase (coenzyme-F420). This family consists of the F420-dependent glucose-6-phosphate dehydrogenase of Mycobacterium and Nocardia. It shows homology to several other F420-dependent enzymes rather than to the NAD or NADP-dependent glucose-6-phosphate dehydrogenases. </td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04021">TIGR04021</a></td><td>350</td><td>205</td><td>98.3</td><td>2.7E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------&nbsp;&nbsp;]</td><td>LLM_DMSO2_sfnG</td><td>dimethyl sulfone monooxygenase SfnG. This family of FMNH2-dependent members of the luciferase-like monooxygenase (LLM) family includes SfnG, a monooxygenase that converts dimethylsulphone (DMSO2) to methanesulphonate. This step can be followed immediately by methanesulfonate sulfonatase (an alkanesulfonate monooxygenase - see TIGR03565) for the FMNH2-dependent conversion an inorganic form. </td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03564">TIGR03564</a></td><td>265</td><td>243</td><td>98.3</td><td>1.1E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>F420_MSMEG_4879</td><td>F420-dependent oxidoreductase, MSMEG_4879 family. Coenzyme F420 is produced by methanogenic archaea, a number of the Actinomycetes (including Mycobacterium tuberculosis), and rare members of other lineages. The resulting information-rich phylogenetic profile identifies candidate F420-dependent oxidoreductases within the family of luciferase-like enzymes (pfam00296), where the species range for the subfamily encompasses many F420-positive genomes without straying beyond. This family is uncharacterized, and named for member MSMEG_4879 from Mycobacterium smegmatis. </td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01094">cd01094</a></td><td>244</td><td>175</td><td>98.3</td><td>7.5E-07</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Alkanesulfonate_monoxygenase</td><td>Alkanesulfonate monoxygenase is the monoxygenase of a two-component system that catalyzes the conversion of alkanesulfonates to the corresponding aldehyde and sulfite. Alkanesulfonate monoxygenase (SsuD) has an absolute requirement for reduced flavin mononucleotide (FMNH2), which is provided by the NADPH-dependent FMN oxidoreductase (SsuE).</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03841">TIGR03841</a></td><td>301</td><td>172</td><td>97.9</td><td>9.6E-06</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_Rv3093c</td><td>probable F420-dependent oxidoreductase, Rv3093c family. This model describes a small family of enzymes in the bacterial luciferase-like monooxygenase family, which includes F420-dependent enzymes such as N5,N10-methylenetetrahydromethanopterin reductase as well as FMN-dependent enzymes. All members of this family are from species that produce coenzyme F420; SIMBAL analysis suggests that members of this family bind F420 rather than FMN. </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03854">TIGR03854</a></td><td>290</td><td>174</td><td>97.9</td><td>2.3E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_MSMEG_3544</td><td>probable F420-dependent oxidoreductase, MSMEG_3544 family. Coenzyme F420 has a limited phylogenetic distribution, including methanogenic archaea, Mycobacterium tuberculosis and related species, Colwellia psychrerythraea 34H, Rhodopseudomonas palustris HaA2, and others. Partial phylogenetic profiling identifies protein subfamilies, within the larger family called luciferase-like monooxygenanases (pfam00296), that appear only in F420-positive genomes and are likely to be F420-dependent. This model describes a small family, closely related to other such families in the putative F420-binding region, exemplified by MSMEG_3544 in Mycobacterium smegmatis. </td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03565">TIGR03565</a></td><td>346</td><td>250</td><td>97.8</td><td>9.6E-05</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------------------&nbsp;]</td><td>alk_sulf_monoox</td><td>alkanesulfonate monooxygenase, FMNH(2)-dependent. Members of this protein family are monooxygenases that catalyze desulfonation of aliphatic sulfonates such as methane sulfonate. This enzyme uses reduced FMN, although various others members of the same luciferase-like monooxygenase family (pfam00296) are F420-dependent enzymes. </td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01096">cd01096</a></td><td>315</td><td>233</td><td>97.6</td><td>0.00019</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;]</td><td>Alkanal_monooxygenase</td><td>Alkanal monooxygenase are flavin monoxygenases. Molecular oxygen is activated by reaction with reduced flavin mononucleotide (FMNH2) and reacts with an aldehyde to yield the carboxylic acid, oxidized flavin (FMN) and a blue-green light. Bacterial luciferases are heterodimers made of alpha and beta subunits which are homologous. The single activer center is on the alpha subunit. The alpha subunit has a stretch of 30 amino acid residues that is not present in the beta subunit. The beta subunit does not contain the active site and is required for the formation of the fully active heterodimer. The beta subunit does not contribute anything directly to the active site. Its role is probably to stabilize the high quantum yield conformation of the alpha subunit through interactionbs across the subunit interface.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00719">PRK00719</a></td><td>378</td><td>130</td><td>97.1</td><td>0.0018</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK00719</td><td>alkanesulfonate monooxygenase; Provisional</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03856">TIGR03856</a></td><td>249</td><td>129</td><td>96.3</td><td>0.0014</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_MSMEG_2906</td><td>probable F420-dependent oxidoreductase, MSMEG_2906 family. This model describes a small family of enzymes in the bacterial luciferase-like monooxygenase family, which includes F420-dependent enzymes such as N5,N10-methylenetetrahydromethanopterin reductase as well as FMN-dependent enzymes. All members of this family are from species that produce coenzyme F420; SIMBAL analysis suggests that members of this family bind F420 rather than FMN. </td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03885">TIGR03885</a></td><td>315</td><td>159</td><td>95.6</td><td>0.034</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>flavin_revert</td><td>probable non-F420 flavinoid oxidoreductase. This model represents a clade of proteins within the larger subfamily TIGR03557. The parent model includes the F420-dependent glucose-6-phosphate dehydrogenase (TIGR03554) and many other proteins. Excepting the members of this family, all members of TIGR03557 occur in species capable of synthesizing coenzyme F420. All members of the seed alignment for this model are from species that lack F420 biosynthesis. It is suggested that members of this family bind FMN, or FO, or a novel flavinoid cofactor, but not F420 per se. </td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13424">pfam13424</a></td><td>78</td><td>61</td><td>95.2</td><td>0.068</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_12</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04036">TIGR04036</a></td><td>355</td><td>227</td><td>93.9</td><td>0.037</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>LLM_CE1758_fam</td><td>putative luciferase-like monooxygenase, FMN-dependent, CE1758 family. This tightly conserved subfamily of the bacterial luciferase-like monooxygenase (LLM) family, with members showing > 60 % pairwise sequence identity, includes proteins from both species with and species without the ability to make coenzyme F420. Therefore, the like cofactor is FMN rather than F420. The presence of three members in Kineococcus radiotolerans SRS30216 and two in Saccharopolyspora erythraea NRRL 2338 suggest closely related (subfamily) rather than exactly conserved (equivalog) function. Gene neighborhoods around members are not conserved. </td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03860">TIGR03860</a></td><td>422</td><td>242</td><td>93.3</td><td>0.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;]</td><td>FMN_nitrolo</td><td>FMN-dependent oxidoreductase, nitrilotriacetate monooxygenase family. This model represents a distinctive clade, in which all characterized members are FMN-binding, within the larger family of luciferase-like monooxygenases (LLM), among which there are both FMN- and F420-binding enzymes. A well-characterized member is nitrilotriacetate monooxygenase from Aminobacter aminovorans (Chelatobacter heintzii), where nitrilotriacetate is a chelating agent used in detergents. </td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13424">pfam13424</a></td><td>78</td><td>73</td><td>93.2</td><td>0.31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;]</td><td>TPR_12</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10508">PRK10508</a></td><td>333</td><td>69</td><td>88.6</td><td>0.027</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK10508</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03620">TIGR03620</a></td><td>278</td><td>133</td><td>86.5</td><td>0.39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>F420_MSMEG_4141</td><td>probable F420-dependent oxidoreductase, MSMEG_4141 family. Members of this protein family, related to F420-dependent oxidoreductases within the larger family of a bacterial luciferase (an FMN-dependent enzyme), occurs only within the small subset of species that synthesize F420. Most such proteins are from members of the Actinobacteria, but at least one species, Sphingomonas wittichii, belongs to the Alphaproteobacteria. </td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13374">pfam13374</a></td><td>42</td><td>34</td><td>79.7</td><td>0.53</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_10</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01095">cd01095</a></td><td>358</td><td>99</td><td>77.9</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Nitrilotriacetate_monoxgenase</td><td>nitrilotriacetate monoxygenase oxidizes nitrilotriacetate utilizing reduced flavin mononucleotide (FMNH2) and oxygen. The FMNH2 is provided by an NADH:flavin mononucleotide (FMN) oxidorductase that uses NADH to reduce FMN to FMNH2.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00515">pfam00515</a></td><td>34</td><td>27</td><td>69.6</td><td>1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_1</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00347">cd00347</a></td><td>90</td><td>64</td><td>59.0</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Flavin_utilizing_monoxygenases</td><td>Flavin-utilizing monoxygenases</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13176">pfam13176</a></td><td>36</td><td>30</td><td>57.9</td><td>3.9</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_7</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd15832">cd15832</a></td><td>278</td><td>84</td><td>56.7</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SNAP</td><td>Soluble N-ethylmaleimide-sensitive factor (NSF) Attachment Protein family. Members of the soluble NSF attachment protein (SNAP) family are involved in intracellular membrane trafficking, including vesicular transport between the endoplasmic reticulum and Golgi apparatus. Higher eukaryotes contain three isoforms of SNAPs: alpha, beta, and gamma. Alpha-SNAP is universally present in eukaryotes and acts as an adaptor protein between SNARE (integral membrane SNAP receptor) and NSF for recruitment to the 20S complex. Beta-SNAP is brain-specific and shares high sequence identity (about 85%) with alpha-SNAP. Gamma-SNAP is weakly related (about 20-25% identity) to the two other isoforms, and is ubiquitous. It may help regulate the activity of the 20S complex. The X-ray structures of vertebrate gamma-SNAP and yeast Sec17, a SNAP family member, show similar all-helical structures consisting of an N-terminal extended twisted sheet of four Tetratricopeptide repeat (TPR)-like helical hairpins and a C-terminal helical bundle.</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07592">cd07592</a></td><td>223</td><td>54</td><td>54.5</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BAR_Endophilin_A</td><td>The Bin/Amphiphysin/Rvs (BAR) domain of Endophilin-A. BAR domains are dimerization, lipid binding and curvature sensing modules found in many different proteins with diverse functions. Endophilins are accessory proteins, localized at synapses, which interact with the endocytic proteins, dynamin and synaptojanin. They are essential for synaptic vesicle formation from the plasma membrane. They interact with voltage-gated calcium channels, thus linking vesicle endocytosis to calcium regulation. They also play roles in virus budding, mitochondrial morphology maintenance, receptor-mediated endocytosis inhibition, and endosomal sorting. Endophilins contain an N-terminal N-BAR domain (BAR domain with an additional N-terminal amphipathic helix), followed by a variable region containing proline clusters, and a C-terminal SH3 domain. They are classified into two types, A and B. Vertebrates contain three endophilin-A isoforms. Endophilin-A proteins are enriched in the brain and play multiple roles in receptor-mediated endocytosis. They tubulate membranes and regulate calcium influx into neurons to trigger the activation of the endocytic machinery. They are also involved in the sorting of plasma membrane proteins, actin filament assembly, and the uncoating of clathrin-coated vesicles for fusion with endosomes. The BAR domains of endophilin-A1 and A3 form crescent-shaped dimers that can detect membrane curvature and drive membrane bending.</td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13181">pfam13181</a></td><td>34</td><td>26</td><td>53.7</td><td>5.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_8</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09490">PRK09490</a></td><td>1229</td><td>59</td><td>49.7</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>metH</td><td>B12-dependent methionine synthase; Provisional</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13441">PRK13441</a></td><td>180</td><td>86</td><td>48.8</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;]</td><td>PRK13441</td><td>F0F1 ATP synthase subunit delta; Provisional</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07615">cd07615</a></td><td>223</td><td>51</td><td>48.7</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BAR_Endophilin_A3</td><td>The Bin/Amphiphysin/Rvs (BAR) domain of Endophilin-A3. BAR domains are dimerization, lipid binding and curvature sensing modules found in many different proteins with diverse functions. Endophilins are accessory proteins localized at synapses that interacts with the endocytic proteins, dynamin and synaptojanin. They are essential for synaptic vesicle formation from the plasma membrane. They interact with voltage-gated calcium channels, thus linking vesicle endocytosis to calcium regulation. They also play roles in virus budding, mitochondrial morphology maintenance, receptor-mediated endocytosis inhibition, and endosomal sorting. Endophilins contain an N-terminal N-BAR domain (BAR domain with an additional N-terminal amphipathic helix), followed by a variable region containing proline clusters, and a C-terminal SH3 domain. They are classified into two types, A and B. Endophilin-A proteins are enriched in the brain and play multiple roles in receptor-mediated endocytosis. Endophilin-A3 (or endophilin-3) is also referred to as SH3P13 (SH3 domain containing protein 13) or SH3GL3 (SH3 domain containing Grb2-like protein 3). It regulates Arp2/3-dependent actin filament assembly during endocytosis. It binds N-WASP through its SH3 domain and enhances the ability of N-WASP to activate the Arp2/3 complex. Endophilin-A3 co-localizes with the vesicular glutamate transporter 1 (VGLUT1), and may play an important role in the synaptic release of glutamate.</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14938">pfam14938</a></td><td>282</td><td>85</td><td>48.4</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SNAP</td><td>Soluble NSF attachment protein, SNAP. The soluble NSF attachment protein (SNAP) proteins are involved in vesicular transport between the endoplasmic reticulum and Golgi apparatus. They act as adaptors between SNARE (integral membrane SNAP receptor) proteins and NSF (N-ethylmaleimide-sensitive factor). They are structurally similar to TPR repeats.</td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04053">pfam04053</a></td><td>431</td><td>119</td><td>45.0</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;]</td><td>Coatomer_WDAD</td><td>Coatomer WD associated region. This region is composed of WD40 repeats.</td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12919">pfam12919</a></td><td>472</td><td>114</td><td>41.6</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TcdA_TcdB</td><td>TcdA/TcdB catalytic glycosyltransferase domain. This domain represents the N-terminal glycosyltransferase from a set of toxins found in some bacteria. This domain in TcdB glycosylates the host RhoA protein.</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03612">TIGR03612</a></td><td>355</td><td>88</td><td>39.4</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RutA</td><td>pyrimidine utilization protein A. This protein is observed in operons extremely similar to that characterized in E. coli K-12 responsible for the import and catabolism of pyrimidines, primarily uracil. This protein is a member of the luciferase family defined by pfam00296 and is likely a FMN-dependent monoxygenase. </td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12381">PRK12381</a></td><td>406</td><td>63</td><td>38.8</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;]</td><td>PRK12381</td><td>bifunctional succinylornithine transaminase/acetylornithine transaminase; Provisional</td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00840">cd00840</a></td><td>186</td><td>65</td><td>37.5</td><td>67</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MPP_Mre11_N</td><td>Mre11 nuclease, N-terminal metallophosphatase domain. Mre11 (also known as SbcD in Escherichia coli) is a subunit of the MRX protein complex. This complex includes: Mre11, Rad50, and Xrs2/Nbs1, and plays a vital role in several nuclear processes including DNA double-strand break repair, telomere length maintenance, cell cycle checkpoint control, and meiotic recombination, in eukaryotes. During double-strand break repair, the MRX complex is required to hold the two ends of a broken chromosome together. In vitro studies show that Mre11 has 3'-5' exonuclease activity on dsDNA templates and endonuclease activity on dsDNA and ssDNA templates. In addition to the N-terminal phosphatase domain, the eukaryotic MRE11 members of this family have a C-terminal DNA binding domain (not included in this alignment model). MRE11-like proteins are found in prokaryotes and archaea was well as in eukaryotes. Mre11 belongs to the metallophosphatase (MPP) superfamily. MPPs are functionally diverse, but all share a conserved domain with an active site consisting of two metal ions (usually manganese, iron, or zinc) coordinated with octahedral geometry by a cage of histidine, aspartate, and asparagine residues. The MPP superfamily includes: Mre11/SbcD-like exonucleases, Dbr1-like RNA lariat debranching enzymes, YfcE-like phosphodiesterases, purple acid phosphatases (PAPs), YbbF-like UDP-2,3-diacylglucosamine hydrolases, and acid sphingomyelinases (ASMases). The conserved domain is a double beta-sheet sandwich with a di-metal active site made up of residues located at the C-terminal side of the sheets. This domain is thought to allow for productive metal coordination.</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07985">cd07985</a></td><td>235</td><td>120</td><td>37.2</td><td>87</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LPLAT_GPAT</td><td>Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: GPAT. Lysophospholipid acyltransferase (LPLAT) superfamily member: glycerol-3-phosphate 1-acyltransferase (GPAT, PlsB). LPLATs are acyltransferases of de novo and remodeling pathways of glycerophospholipid biosynthesis which catalyze the incorporation of an acyl group from either acylCoAs or acyl-acyl carrier proteins (acylACPs) into acceptors such as glycerol 3-phosphate, dihydroxyacetone phosphate or lyso-phosphatidic acid. This subgroup includes glycerol-3-phosphate 1-acyltransferase (GPAT, PlsB).</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00959">TIGR00959</a></td><td>428</td><td>91</td><td>36.9</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ffh</td><td>signal recognition particle protein. This model represents Ffh (Fifty-Four Homolog), the protein component that forms the bacterial (and organellar) signal recognition particle together with a 4.5S RNA. Ffh is a GTPase homologous to eukaryotic SRP54 and also to the GTPase FtsY (TIGR00064) that is the receptor for the signal recognition particle. </td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10140">pfam10140</a></td><td>359</td><td>26</td><td>36.4</td><td>39</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YukC</td><td>WXG100 protein secretion system (Wss), protein YukC. Members of this family of proteins include predicted membrane proteins homologous to YukC in B. subtilis. The YukC protein family would participate to the formation of a translocon required for the secretion of WXG100 proteins (pfam06013) in monoderm bacteria, the WXG100 protein secretion system (Wss). This family includes EssB in Staphylococcus aureus.</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02678">cd02678</a></td><td>75</td><td>55</td><td>35.1</td><td>99</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MIT_VPS4</td><td>MIT: domain contained within Microtubule Interacting and Trafficking molecules. This sub-family of MIT domains is found in intracellular protein transport proteins of the AAA-ATPase family. The molecular function of the MIT domain is unclear.</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02236">pfam02236</a></td><td>85</td><td>28</td><td>33.5</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Viral_DNA_bi</td><td>Viral DNA-binding protein, all alpha domain. This family represents a domain of the viral DNA- binding protein, a multi functional protein involved in DNA replication and transcription control.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14938">pfam14938</a></td><td>282</td><td>46</td><td>32.1</td><td>3.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SNAP</td><td>Soluble NSF attachment protein, SNAP. The soluble NSF attachment protein (SNAP) proteins are involved in vesicular transport between the endoplasmic reticulum and Golgi apparatus. They act as adaptors between SNARE (integral membrane SNAP receptor) proteins and NSF (N-ethylmaleimide-sensitive factor). They are structurally similar to TPR repeats.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11817">pfam11817</a></td><td>240</td><td>52</td><td>31.3</td><td>82</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Foie-gras_1</td><td>Foie gras liver health family 1. Mutating the gene foie gras in zebrafish has been shown to affect development; the mutants develop large, lipid-filled hepatocytes in the liver, resembling those in individuals with fatty liver disease. Foie-gras protein is long and has several well-defined domains though none of them has a known function. We have annotated this one as the first. The C-terminus of this region contains TPR repeats.</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03926">TIGR03926</a></td><td>377</td><td>26</td><td>28.5</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>T7_EssB</td><td>type VII secretion protein EssB. Members of this family are associated with type VII secretion of WXG100 family targets in the Firmicutes, but not in the Actinobacteria. This protein is designated YukC in Bacillus subtilis and EssB is Staphylococcus aureus. </td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01753">TIGR01753</a></td><td>140</td><td>36</td><td>27.9</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>flav_short</td><td>flavodoxin, short chain. Flavodoxins are small redox-active proteins with a flavin mononucleotide (FMN) prosthetic group. They can act in nitrogen fixation by nitrogenase, in sulfite reduction, and light-dependent NADP+ reduction in during photosynthesis, among other roles. This model describes the short chain type. Many of these are involved in sulfite reduction. </td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00777">cd00777</a></td><td>280</td><td>51</td><td>27.7</td><td>48</td><td>[---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AspRS_core</td><td>Asp tRNA synthetase (aspRS) class II core domain. Class II assignment is based upon its structure and the presence of three characteristic sequence motifs. AspRS is a homodimer, which attaches a specific amino acid to the 3' OH group of ribose of the appropriate tRNA. The catalytic core domain is primarily responsible for the ATP-dependent formation of the enzyme bound aminoacyl-adenylate. AspRS in this family differ from those found in the AsxRS family by a GAD insert in the core domain.</td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03103">PRK03103</a></td><td>409</td><td>41</td><td>27.1</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------]</td><td>PRK03103</td><td>DNA polymerase IV; Reviewed</td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03307">cd03307</a></td><td>326</td><td>96</td><td>27.0</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;]</td><td>Mta_CmuA_like</td><td>MtaA_CmuA_like family. MtaA/CmuA, also MtsA, or methyltransferase 2 (MT2) MT2-A and MT2-M isozymes, are methylcobamide:Coenzyme M methyltransferases, which play a role in metabolic pathways of methane formation from various substrates, such as methylated amines and methanol. Coenzyme M, 2-mercaptoethylsulfonate or CoM, is methylated during methanogenesis in a reaction catalyzed by three proteins. A methyltransferase methylates the corrinoid cofactor, which is bound to a second polypeptide, a corrinoid protein. The methylated corrinoid protein then serves as a substrate for MT2-A and related enzymes, which methylate CoM.</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12095">pfam12095</a></td><td>83</td><td>51</td><td>26.9</td><td>83</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------]</td><td>DUF3571</td><td>Protein of unknown function (DUF3571). This family of proteins is functionally uncharacterized. This protein is found in bacteria and eukaryotes. Proteins in this family are typically between 85 to 97 amino acids in length.</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00448">pfam00448</a></td><td>195</td><td>58</td><td>26.7</td><td>99</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SRP54</td><td>SRP54-type protein, GTPase domain. This family includes relatives of the G-domain of the SRP54 family of proteins.</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00029">pfam00029</a></td><td>107</td><td>14</td><td>25.8</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Connexin</td><td>Connexin. </td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09274">cd09274</a></td><td>121</td><td>23</td><td>25.7</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNase_HI_RT_Ty3</td><td>Ty3/Gypsy family of RNase HI in long-term repeat retroelements. Ribonuclease H (RNase H) enzymes are divided into two major families, Type 1 and Type 2, based on amino acid sequence similarities and biochemical properties. RNase H is an endonuclease that cleaves the RNA strand of an RNA/DNA hybrid in a sequence non-specific manner in the presence of divalent cations. RNase H is widely present in various organisms, including bacteria, archaea and eukaryotes. RNase HI has also been observed as adjunct domains to the reverse transcriptase gene in retroviruses, in long-term repeat (LTR)-bearing retrotransposons and non-LTR retrotransposons. RNase HI in LTR retrotransposons perform degradation of the original RNA template, generation of a polypurine tract (the primer for plus-strand DNA synthesis), and final removal of RNA primers from newly synthesized minus and plus strands. The catalytic residues for RNase H enzymatic activity, three aspartatic acids and one glutamic acid residue (DEDD), are unvaried across all RNase H domains. Phylogenetic patterns of RNase HI of LTR retroelements is classified into five major families, Ty3/Gypsy, Ty1/Copia, Bel/Pao, DIRS1 and the vertebrate retroviruses. Ty3/Gypsy family widely distributed among the genomes of plants, fungi and animals. RNase H inhibitors have been explored as an anti-HIV drug target because RNase H inactivation inhibits reverse transcription.</td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07719">pfam07719</a></td><td>34</td><td>25</td><td>25.6</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_2</td><td>Tetratricopeptide repeat. This Pfam entry includes outlying Tetratricopeptide-like repeats (TPR) that are not matched by pfam00515.</td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09325">pfam09325</a></td><td>236</td><td>96</td><td>25.2</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Vps5</td><td>Vps5 C terminal like. Vps5 is a sorting nexin that functions in membrane trafficking. This is the C terminal dimerization domain.</td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03517">cd03517</a></td><td>95</td><td>62</td><td>24.6</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Link_domain_CSPGs_modules_1_3</td><td>Link_domain_CSPGs_modules_1_3; this extracellular link domain is found in the first and third link modules of the chondroitin sulfate proteoglycan core protein (CSPG) aggrecan. In addition, it is found in the first link module of three other CSPGs: versican, neurocan, and brevican. The link domain is a hyaluronan (HA)-binding domain. CSPGs are characterized by an N-terminal globular domain (G1 domain) containing two contiguous link modules (modules 1 and 2). Both link modules of the G1 domain of aggrecan are involved in interaction with HA. In addition, aggrecan contains a second globular domain (G2) which contains link modules 3 and 4. G2 appears to lack HA-binding activity. In cartilage, aggrecan forms cartilage link protein stabilized aggregates with HA. These aggregates contribute to the tissue's load bearing properties. Aggregates having other CSPGs substituting for aggrecan may contribute to the structural integrity of many different tissues. Members of the vertebrate HPLN (hyaluronan/HA and proteoglycan binding link) protein family are physically linked adjacent to CSPG genes.</td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2264">COG2264</a></td><td>300</td><td>60</td><td>23.4</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PrmA</td><td>Ribosomal protein L11 methylase PrmA </td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02649">cd02649</a></td><td>306</td><td>70</td><td>22.5</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>nuc_hydro_CeIAG</td><td>nuc_hydro_CeIAG: Nucleoside hydrolases similar to the inosine-adenosine-guanosine-preferring nucleoside hydrolase from Caenorhabditis elegans.  Nucleoside hydrolases cleave the N-glycosidic bond in nucleosides generating ribose and the respective base. These enzymes vary in their substrate specificity. This group contains eukaryotic, bacterial and archeal proteins similar to the purine-preferring nucleoside hydrolase (IAG-NH) from C. elegans and the salivary purine nucleosidase from Aedes aegypti.  C. elegans IAG-NH exhibits a high affinity for the substrate analogue p-nitrophenylriboside (p-NPR). </td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06068">cd06068</a></td><td>144</td><td>25</td><td>22.4</td><td>42</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>H2MP_like-1</td><td>Putative </td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09840">pfam09840</a></td><td>190</td><td>46</td><td>22.3</td><td>2.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2067</td><td>Uncharacterized protein conserved in archaea (DUF2067). This domain, found in various archaeal proteins, has no known function.</td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09731">pfam09731</a></td><td>493</td><td>41</td><td>21.7</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;]</td><td>Mitofilin</td><td>Mitochondrial inner membrane protein. Mitofilin controls mitochondrial cristae morphology. Mitofilin is enriched in the narrow space between the inner boundary and the outer membranes, where it forms a homotypic interaction and assembles into a large multimeric protein complex. The first 78 amino acids contain a typical amino-terminal-cleavable mitochondrial presequence rich in positive-charged and hydroxylated residues and a membrane anchor domain. In addition, it has three centrally located coiled coil domains.</td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14391">cd14391</a></td><td>36</td><td>27</td><td>21.7</td><td>71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UBA_II_E2_UBCD4</td><td>UBA domain found in Drosophila melanogaster ubiquitin-conjugating enzyme E2-22 kDa (UbcD4) and similar proteins. UbcD4, also called ubiquitin carrier protein or ubiquitin-protein ligase, is a class II E2 ubiquitin-conjugating enzyme encoded by Drosophila E2 gene which is only expressed in pole cells in embryos. It is a putative E2 enzyme homologous to the Huntingtin interacting protein-2 (HIP2) of human. UbcD4 specifically interacts with the polyubiquitin-binding subunit of the proteasome. It contains a C-terminal ubiquitin-associated (UBA) domain in addition to an N-terminal catalytic ubiquitin-conjugating enzyme E2 (UBCc) domain.</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07626">pfam07626</a></td><td>68</td><td>38</td><td>21.4</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PSD3</td><td>Protein of unknown function (DUF1587). A region of similarity shared by several Rhodopirellula baltica cytochrome-like proteins that are predicted to be secreted. These proteins also match pfam07624.</td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04694">pfam04694</a></td><td>60</td><td>15</td><td>21.2</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Corona_3</td><td>Coronavirus ORF3 protein. </td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03246">TIGR03246</a></td><td>397</td><td>72</td><td>20.8</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;]</td><td>arg_catab_astC</td><td>succinylornithine transaminase family. Members of the seed alignment for this protein family are the enzyme succinylornithine transaminase (EC 2.6.1.81), which catalyzes the third of five steps in arginine succinyltransferase (AST) pathway, an ammonia-releasing pathway of arginine degradation. All seed alignment sequences are found within arginine succinyltransferase operons, and all proteins that score above 820.0 bits should function as succinylornithine transaminase. However, a number of sequences extremely closely related in sequence, found in different genomic contexts, are likely to act in different biological processes and may act on different substrates. This model is desigated subfamily rather than equivalog, pending further consideration, for this reason. </td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04674">pfam04674</a></td><td>273</td><td>33</td><td>20.4</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Phi_1</td><td>Phosphate-induced protein 1 conserved region. Family of conserved plant proteins. Conserved region identified in a phosphate-induced protein of unknown function.</td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04752">pfam04752</a></td><td>177</td><td>54</td><td>20.1</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ChaC</td><td>ChaC-like protein. The ChaC protein is thought to be associated with the putative ChaA Ca2+/H+ cation transport protein in Escherichia coli. Its function is not known. This family also includes homologues regions from several other bacterial and eukaryotic proteins.</td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0783">COG0783</a></td><td>156</td><td>33</td><td>20.0</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Dps</td><td>DNA-binding ferritin-like protein (oxidative damage protectant) </td></tr>
</table>
</body>
