<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01807">pfam01807</a></td><td>98</td><td>45</td><td>84.0</td><td>1.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>zf-CHC2</td><td>CHC2 zinc finger. This domain is principally involved in DNA binding in DNA primases.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07938">cd07938</a></td><td>274</td><td>49</td><td>82.6</td><td>2.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;]</td><td>DRE_TIM_HMGL</td><td>3-hydroxy-3-methylglutaryl-CoA lyase, catalytic TIM barrel domain. 3-hydroxy-3-methylglutaryl-CoA lyase (HMGL) catalyzes the cleavage of HMG-CoA to acetyl-CoA and acetoacetate, one of the terminal steps in ketone body generation and leucine degradation, and is a key enzyme in the pathway that supplies metabolic fuel to extrahepatic tissues.  Mutations in HMGL cause a human autosomal recessive disorder called primary metabolic aciduria that affects ketogenesis and leucine catabolism and can be fatal due to an inability to tolerate hypoglycemia.  HMGL has a TIM barrel domain with a catalytic center containing a divalent cation-binding site formed by a cluster of invariant residues that cap the core of the barrel.  The cleavage of HMG-CoA requires the presence of a divalent cation like Mg2+ or Mn2+, and the reaction is thought to involve general acid/base catalysis.  This family belongs to the DRE-TIM metallolyase superfamily.  DRE-TIM metallolyases include 2-isopropylmalate synthase (IPMS), alpha-isopropylmalate synthase (LeuA), 3-hydroxy-3-methylglutaryl-CoA lyase, homocitrate synthase, citramalate synthase, 4-hydroxy-2-oxovalerate aldolase, re-citrate synthase, transcarboxylase 5S, pyruvate carboxylase, AksA, and FrbC.  These members all share a conserved  triose-phosphate isomerase (TIM) barrel domain consisting of a core beta(8)-alpha(8) motif with the eight parallel beta strands forming an enclosed barrel surrounded by eight alpha helices.  The domain has a catalytic center containing a divalent cation-binding site formed by a cluster of invariant residues that cap the core of the barrel.  In addition, the catalytic site includes three invariant residues - an aspartate (D), an arginine (R), and a glutamate (E) - which is the basis for the domain name "DRE-TIM"td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14751">cd14751</a></td><td>376</td><td>36</td><td>78.4</td><td>2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_GacH</td><td>The periplasmic-binding component of the putative oligosacchride ABC transporter GacHFG; possesses type 2 periplasmic binding fold. This group represents the periplasmic component GacH of an ABC import system. GacH is identified as a maltose/maltodextrin-binding protein with a low affinity for acarbose. Members of this group belong to the type 2 periplasmic-binding fold superfamily. PBP2 proteins are comprised of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. The majority of PBP2 proteins function in the uptake of small soluble substrates in eubacteria and archaea. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two cytoplasmically located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14629">PRK14629</a></td><td>99</td><td>40</td><td>68.2</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14629</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0358">COG0358</a></td><td>568</td><td>47</td><td>60.0</td><td>8.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DnaG</td><td>DNA primase (bacterial type) </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01016">cd01016</a></td><td>276</td><td>32</td><td>59.0</td><td>6.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TroA</td><td>Metal binding protein TroA. These proteins have been shown to function as initial receptors in ABC transport of Zn2+ and possibly Fe3+ in many eubacterial species.  The TroA proteins belong to the TroA superfamily of periplasmic metal binding proteins that share a distinct fold and ligand binding mechanism. A typical TroA protein is comprised of two globular subdomains connected by a single helix and can bind the metal ion in the cleft between these domains. In addition, these proteins sometimes have a low complexity region containing a metal-binding histidine-rich motif (repetitive HDH sequence).</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02789">pfam02789</a></td><td>126</td><td>44</td><td>58.5</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_M17_N</td><td>Cytosol aminopeptidase family, N-terminal domain. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1469">COG1469</a></td><td>289</td><td>42</td><td>58.1</td><td>4.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;]</td><td>FolE2</td><td>GTP cyclohydrolase FolE2 </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01834">cd01834</a></td><td>191</td><td>19</td><td>55.1</td><td>91</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SGNH_hydrolase_like_2</td><td>SGNH_hydrolase subfamily. SGNH hydrolases are a diverse family of lipases and esterases. The tertiary fold of the enzyme is substantially different from that of the alpha/beta hydrolase family and unique among all known hydrolases; its active site closely resembles the Ser-His-Asp(Glu) triad found in other serine hydrolases.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09173">cd09173</a></td><td>159</td><td>13</td><td>54.5</td><td>6.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLDc_Nuc_like_unchar1_2</td><td>Putative catalytic domain, repeat 2, of uncharacterized hypothetical proteins similar to Nuc, an endonuclease from Salmonella typhimurium. Putative catalytic domain, repeat 2,  of uncharacterized hypothetical proteins, which show high sequence homology to the endonuclease from Salmonella typhimurium and vertebrate phospholipase D6. Nuc and PLD6 belong to the phospholipase D (PLD) superfamily. They contain a short conserved sequence motif, the HKD motif (H-x-K-x(4)-D, where x represents any amino acid residue), which characterizes the PLD superfamily and is essential for catalysis. Nuc and PLD6 utilize a two-step mechanism to cleave phosphodiester bonds: Upon substrate binding, the bond is first attacked by a histidine residue from one HKD motif to form a covalent phosphohistidine intermediate, which is then hydrolyzed by water with the aid of a second histidine residue from the other HKD motif in the opposite subunit. However, proteins in this subfamily have two HKD  motifs in a single polypeptide chain.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02878">pfam02878</a></td><td>138</td><td>14</td><td>52.1</td><td>7.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PGM_PMM_I</td><td>Phosphoglucomutase/phosphomannomutase, alpha/beta/alpha domain I. </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12973">pfam12973</a></td><td>91</td><td>12</td><td>52.0</td><td>8.1</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cupin_7</td><td>ChrR Cupin-like domain. Members of this family are part of the cupin superfamily. This family includes the transcriptional activator ChrR.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02643">cd02643</a></td><td>74</td><td>28</td><td>46.8</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>R3H_NF-X1</td><td>R3H domain of the X1 box binding protein (NF-X1) and related proteins. Human NF-X1 is a transcription factor that regulates the expression of class II major histocompatibility complex (MHC) genes. The Drosophila homolog shuttle craft (STC) has been shown to be a DNA- or RNA-binding protein required for proper axon guidance in the central nervous system and, the yeast homolog FAP1 encodes a dosage suppressor of rapamycin toxicity. The name of the R3H domain comes from the characteristic spacing of the most conserved arginine and histidine residues. The function of the domain is predicted to bind ssDNA or ssRNA in a sequence-specific manner.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14542">PRK14542</a></td><td>137</td><td>20</td><td>46.0</td><td>9.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14542</td><td>nucleoside diphosphate kinase; Provisional</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03385">pfam03385</a></td><td>390</td><td>62</td><td>46.0</td><td>7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF288</td><td>Protein of unknown function, DUF288. </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03595">TIGR03595</a></td><td>69</td><td>27</td><td>45.3</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Obg_CgtA_exten</td><td>Obg family GTPase CgtA, C-terminal extension. CgtA (see model TIGR02729) is a broadly conserved member of the obg family of GTPases associated with ribosome maturation. This model represents a unique C-terminal domain found in some but not all sequences of CgtA. This region is preceded, and may be followed, by a region of low-complexity sequence.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04092">cd04092</a></td><td>83</td><td>21</td><td>44.9</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>mtEFG2_II_like</td><td>mtEFG2_C: C-terminus of mitochondrial Elongation factor G2 (mtEFG2)-like proteins found in eukaryotes.  Eukaryotic cells harbor 2 protein synthesis systems: one localized in the cytoplasm, the other in the mitochondria. Most factors regulating mitochondrial protein synthesis are encoded by nuclear genes, translated in the cytoplasm, and then transported to the mitochondria. The eukaryotic system of elongation factor (EF) components is more complex than that in prokaryotes, with both cytoplasmic and mitochondrial elongation factors and multiple isoforms being expressed in certain species.  Eukaryotic EF-2 operates in the cytosolic protein synthesis machinery of eukaryotes, EF-Gs in protein synthesis in bacteria.  Eukaryotic mtEFG1 proteins show significant homology to bacterial EF-Gs.  No clear phenotype has been found for mutants in the yeast homologue of mtEFG2, MEF2.  There are two forms of mtEFG present in mammals (designated mtEFG1s and mtEFG2s) mtEFG1s are not present in this group.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07481">cd07481</a></td><td>264</td><td>29</td><td>44.2</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidases_S8_BacillopeptidaseF-like</td><td>Peptidase S8 family domain in BacillopeptidaseF-like proteins. Bacillus subtilis produces and secretes proteases and other types of exoenzymes at the end of the exponential phase of growth. The ones that make up this group is known as bacillopeptidase F, encoded by bpr,  a serine protease with high esterolytic activity which is inhibited by PMSF.  Like other members of the peptidases S8 family these have a Asp/His/Ser catalytic triad similar to that found in trypsin-like proteases, but do not share their three-dimensional structure and are not homologous to trypsin. The stability of these enzymes may be enhanced by calcium, some members have been shown to bind up to 4 ions via binding sites with different affinity.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09937">PRK09937</a></td><td>74</td><td>11</td><td>42.5</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09937</td><td>stationary phase/starvation inducible regulatory protein CspD; Provisional</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03012">cd03012</a></td><td>126</td><td>27</td><td>42.4</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TlpA_like_DipZ_like</td><td>TlpA-like family, DipZ-like subfamily; composed uncharacterized proteins containing a TlpA-like TRX domain. Some members show domain architectures similar to that of E. coli DipZ protein (also known as DsbD). The only eukaryotic members of the TlpA family belong to this subfamily. TlpA is a disulfide reductase known to have a crucial role in the biogenesis of cytochrome aa3.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14998">PRK14998</a></td><td>73</td><td>11</td><td>42.4</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14998</td><td>cold shock-like protein CspD; Provisional</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01178">cd01178</a></td><td>101</td><td>48</td><td>42.4</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;]</td><td>IPT_NFAT</td><td>IPT domain of the NFAT family of transcription factors. NFAT transcription complexes are a target of calcineurin, a calcium dependent phosphatase, and activate genes mainly involved in cell-cell-interaction.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01390">pfam01390</a></td><td>107</td><td>44</td><td>41.2</td><td>64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------]</td><td>SEA</td><td>SEA domain. Domain found in Sea urchin sperm protein, Enterokinase, Agrin (SEA). Proposed function of regulating or binding carbohydrate side chains. Recently a proteolytic activity has been shown for a SEA domain.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00750">pfam00750</a></td><td>345</td><td>45</td><td>39.5</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>tRNA-synt_1d</td><td>tRNA synthetases class I (R). Other tRNA synthetase sub-families are too dissimilar to be included. This family includes only arginyl tRNA synthetase.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09581">PRK09581</a></td><td>457</td><td>70</td><td>38.8</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pleD</td><td>response regulator PleD; Reviewed</td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09269">pfam09269</a></td><td>69</td><td>28</td><td>38.3</td><td>74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1967</td><td>Domain of unknown function (DUF1967). Members of this family contain a four-stranded beta sheet and three alpha helices flanked by an additional beta strand. They are predominantly found in the bacterial GTP-binding protein Obg, and are still functionally uncharacterized.</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13675">PRK13675</a></td><td>308</td><td>42</td><td>36.6</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;]</td><td>PRK13675</td><td>GTP cyclohydrolase; Provisional</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04088">cd04088</a></td><td>83</td><td>30</td><td>35.6</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>EFG_mtEFG_II</td><td>EFG_mtEFG_II: this subfamily represents the domain II of elongation factor G (EF-G) in bacteria and, the C-terminus of mitochondrial Elongation factor G1 (mtEFG1) and G2 (mtEFG2)_like proteins found in eukaryotes. During the process of peptide synthesis and tRNA site changes, the ribosome is moved along the mRNA a distance equal to one codon with the addition of each amino acid. In bacteria this translocation step is catalyzed by EF-G_GTP, which is hydrolyzed to provide the required energy. Thus, this action releases the uncharged tRNA from the P site and transfers the newly formed peptidyl-tRNA from the A site to the P site. Eukaryotic cells harbor 2 protein synthesis systems: one localized in the cytoplasm, the other in the mitochondria. Most factors regulating mitochondrial protein synthesis are encoded by nuclear genes, translated in the cytoplasm, and then transported to the mitochondria. The eukaryotic system of elongation factor (EF) components is more complex than that in prokaryotes, with both cytoplasmic and mitochondrial elongation factors and multiple isoforms being expressed in certain species.  mtEFG1 and mtEFG2 show significant homology to bacterial EF-Gs.  Mutants in yeast mtEFG1 have impaired mitochondrial protein synthesis, respiratory defects and a tendency to lose mitochondrial DNA. No clear phenotype has been found for mutants in the yeast homologue of mtEFG2, MEF2.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09176">cd09176</a></td><td>114</td><td>14</td><td>34.1</td><td>20</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLDc_unchar6</td><td>Putative catalytic domain of uncharacterized hypothetical proteins with one or two copies of the HKD motif. Putative catalytic domain of uncharacterized hypothetical proteins with similarity to phospholipase D (PLD, EC 3.1.4.4). PLD enzymes hydrolyze phospholipid phosphodiester bonds to yield phosphatidic acid and a free polar head group. They can also catalyze transphosphatidylation of phospholipids to acceptor alcohols. Members of this subfamily contain one or two copies of the HKD motif (H-x-K-x(4)-D, where x represents any amino acid residue) that characterizes the PLD superfamily.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03416">cd03416</a></td><td>101</td><td>35</td><td>33.7</td><td>83</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;]</td><td>CbiX_SirB_N</td><td>Sirohydrochlorin cobalt chelatase (CbiX) and sirohydrochlorin iron chelatase (SirB), N-terminal domain. SirB catalyzes the ferro-chelation of sirohydrochlorin to siroheme, the prosthetic group of sulfite and nitrite reductases. CbiX is a cobaltochelatase, responsible for the chelation of Co2+ into sirohydrochlorin, an important step in the vitamin B12 biosynthetic pathway. CbiX often contains a C-terminal histidine-rich region that may be important for metal delivery and/or storage, and may also contain an iron-sulfur center. Both are found in a wide range of bacteria. This subgroup also contains single domain proteins from archaea and bacteria which may represent the ancestral form of class II chelatases before domain duplication occurred.</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04176">cd04176</a></td><td>163</td><td>8</td><td>33.0</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rap2</td><td>Rap2 family GTPase consists of Rap2a, Rap2b, and Rap2c. The Rap2 subgroup is part of the Rap subfamily of the Ras family. It consists of Rap2a, Rap2b, and Rap2c. Both isoform 3 of the human mitogen-activated protein kinase kinase kinase kinase 4 (MAP4K4) and Traf2- and Nck-interacting kinase (TNIK) are putative effectors of Rap2 in mediating the activation of c-Jun N-terminal kinase (JNK) to regulate the actin cytoskeleton. In human platelets, Rap2 was shown to interact with the cytoskeleton by binding the actin filaments. In embryonic Xenopus development, Rap2 is necessary for the Wnt/beta-catenin signaling pathway. The Rap2 interacting protein 9 (RPIP9) is highly expressed in human breast carcinomas and correlates with a poor prognosis, suggesting a role for Rap2 in breast cancer oncogenesis. Rap2b, but not Rap2a, Rap2c, Rap1a, or Rap1b, is expressed in human red blood cells, where it is believed to be involved in vesiculation. A number of additional effector proteins for Rap2 have been identified, including the RalGEFs RalGDS, RGL, and Rlf, which also interact with Rap1 and Ras. Most Ras proteins contain a lipid modification site at the C-terminus, with a typical sequence motif CaaX, where a = an aliphatic amino acid and X = any amino acid. Lipid binding is essential for membrane attachment, a key feature of most Ras proteins. Due to the presence of truncated sequences in this CD, the lipid modification site is not available for annotation.</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14535">pfam14535</a></td><td>96</td><td>38</td><td>32.2</td><td>68</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AMP-binding_C_2</td><td>AMP-binding enzyme C-terminal domain. This is a small domain that is found C terminal to pfam00501. It has a central beta sheet core that is flanked by alpha helices.</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12686">pfam12686</a></td><td>227</td><td>46</td><td>32.1</td><td>76</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3800</td><td>Protein of unknown function (DUF3800). This family of proteins is functionally uncharacterized. This family of proteins is found in bacteria, archaea, eukaryotes and viruses. Proteins in this family are typically between 215 and 302 amino acids in length. There is a DE motif at the N-terminus and a QXXD motif at the C-terminus that may be functionally important.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13472">pfam13472</a></td><td>173</td><td>40</td><td>31.9</td><td>88</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Lipase_GDSL_2</td><td>GDSL-like Lipase/Acylhydrolase family. This family of presumed lipases and related enzymes are similar to pfam00657.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1190">COG1190</a></td><td>502</td><td>44</td><td>31.8</td><td>68</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;]</td><td>LysU</td><td>Lysyl-tRNA synthetase (class II) </td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02649">pfam02649</a></td><td>268</td><td>31</td><td>30.9</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GCHY-1</td><td>Type I GTP cyclohydrolase folE2. This is a family of prokaryotic proteins with type I GTP cyclohydrolase activity. GTP cyclohydrolase I is the first enzyme of the de novo tetrahydrofolate biosynthetic pathway present in bacteria, fungi, and plants, and encoded in Escherichia coli by the folE gene; it is also the first enzyme of the biopterin (BH4) pathway in Homo sapiens. The invariate, highly conserved glutamate residue at position 216 in Neisseria gonorrhoeae folE2 is likely to be the substrate ligand and the metal ligand is likely to be the cysteine at position 147. The enzyme is Zinc 2+ dependent.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08697">cd08697</a></td><td>185</td><td>37</td><td>30.4</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>C2_Dock-D</td><td>C2 domains found in Dedicator Of CytoKinesis (Dock) class C proteins. Dock-D is one of 4 classes of Dock family proteins.  The members here include: Dock9/Zizimin1, Dock10/Zizimin3, and Dock11/Zizimin2/ACG (activated Cdc42-associated GEF).  Dock-D are Cdc42-specific GEFs. In addition to the C2 domain (AKA Dock homology region (DHR)-1, CED-5, Dock180, MBC-zizimin homology (CZH) 1) and the DHR-2 (AKA CZH2, or Docker), which all Dock180-related proteins have, Dock-D members contain a functionally uncharacterized domain and a PH domain upstream of the C2 domain.  DHR-2 has the catalytic activity for Rac and/or Cdc42, but is structurally unrelated to the DH domain. The C2/DHR-1 domains of Dock180 and Dock4 have been shown to bind phosphatidylinositol-3, 4, 5-triphosphate (PtdIns(3,4,5)P3).  The PH domain broadly binds to phospholipids and is thought to be involved in targeting the plasma membrane.  The C2 domain was first identified in PKC. C2 domains fold into an 8-standed beta-sandwich that can adopt 2 structural arrangements: Type I and Type II, distinguished by a circular permutation involving their N- and C-terminal beta strands. Many C2 domains are Ca2+-dependent membrane-targeting modules that bind a wide variety of substances including bind phospholipids, inositol polyphosphates, and intracellular proteins.  Most C2 domain proteins are either signal transduction enzymes that contain a single C2 domain, such as protein kinase C, or membrane trafficking proteins which contain at least two C2 domains, such as synaptotagmin 1.  However, there are a few exceptions to this including RIM isoforms and some splice variants of piccolo/aczonin and intersectin which only have a single C2 domain.  C2 domains with a calcium binding region have negatively charged residues, primarily aspartates, that serve as ligands for calcium ions.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05350">cd05350</a></td><td>239</td><td>83</td><td>29.8</td><td>43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;]</td><td>SDR_c6</td><td>classical (c) SDR, subgroup 6. These proteins are members of the classical SDR family, with a canonical active site tetrad  and a fairly well conserved typical Gly-rich  NAD-binding motif. SDRs are a functionally diverse family of oxidoreductases that have a single domain with a structurally conserved Rossmann fold (alpha/beta folding pattern with a central beta-sheet), an NAD(P)(H)-binding region, and a structurally diverse C-terminal region. Classical SDRs are typically about 250 residues long, while extended SDRS are approximately 350 residues. Sequence identity between different SDR enzymes are typically in the 15-30% range, but the enzymes share the Rossmann fold NAD-binding motif and characteristic NAD-binding and catalytic sequence patterns. These enzymes have a 3-glycine N-terminal NAD(P)(H)-binding pattern (typically, TGxxxGxG in classical SDRs and TGxxGxxG in extended SDRs), while substrate binding is in the C-terminal region. A critical catalytic Tyr residue (Tyr-151, human 15-hydroxyprostaglandin dehydrogenase (15-PGDH) numbering), is often found in a conserved YXXXK pattern. In addition to the Tyr and Lys, there is often an upstream Ser (Ser-138, 15-PGDH numbering) and/or an Asn (Asn-107, 15-PGDH numbering) or additional Ser, contributing to the active site. Substrates for these enzymes include sugars, steroids, alcohols, and aromatic compounds. The standard reaction mechanism is a proton relay involving the conserved Tyr and Lys, as well as Asn (or Ser). Some SDR family members, including 17 beta-hydroxysteroid dehydrogenase contain an additional helix-turn-helix motif that is not generally found among SDRs.</td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09778">pfam09778</a></td><td>212</td><td>16</td><td>29.2</td><td>29</td><td>[&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Guanylate_cyc_2</td><td>Guanylylate cyclase. Members of this family of proteins catalyse the conversion of guanosine triphosphate (GTP) to 3',5'-cyclic guanosine monophosphate (cGMP) and pyrophosphate.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03956">TIGR03956</a></td><td>340</td><td>68</td><td>29.1</td><td>77</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;]</td><td>rSAM_HydE</td><td></td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08974">pfam08974</a></td><td>165</td><td>22</td><td>29.0</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1877</td><td>Domain of unknown function (DUF1877). This domain is found in a set of hypothetical bacterial proteins.</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10050">pfam10050</a></td><td>166</td><td>47</td><td>28.4</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2284</td><td>Predicted metal-binding protein (DUF2284). Members of this family of metal-binding hypothetical bacterial proteins have no known function.</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06138">pfam06138</a></td><td>130</td><td>35</td><td>28.3</td><td>83</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------]</td><td>Chordopox_E11</td><td>Chordopoxvirus E11 protein. This family consists of several Chordopoxvirus E11 proteins. The E11 gene of vaccinia virus encodes a 15-kDa polypeptide. Mutations in the E11 gene makes the virus temperature-sensitive due to either the fact that virus infectivity requires a threshold level of active E11 protein or that E11 function is conditionally essential.</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09117">cd09117</a></td><td>117</td><td>11</td><td>28.1</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLDc_Bfil_DEXD_like</td><td>Catalytic domain of type II restriction endonucleases BfiI and NgoFVII, and uncharacterized proteins with a DEAD domain. Catalytic domain of type II restriction endonucleases BfiI and NgoFVII, uncharacterized type III restriction endonuclease Res subunit, and uncharacterized DNA/RNA helicase superfamily II members. Proteins in this family are found mainly in prokaryotes. They contain one copy of the conserved HKD motif (H-x-K-x(4)-D, where x represents any amino acid residue) in a single polypeptide chain, and have been classified as members of the phospholipase D (PLD, EC 3.1.4.4) superfamily. BfiI consists of two discrete domains with distinct functions: an N-terminal catalytic domain with non-specific nuclease activity and dimerization function that is more closely related to Nuc, an EDTA-resistant nuclease from the phospholipase D (PLD) superfamily; and a C-terminal domain that specifically recognizes its target sequences, 5'-ACTGGG-3'. BfiI forms a functionally active homodimer which has two DNA-binding surfaces located at the C-terminal domains but only one active site, located at the dimer interface between the two N-terminal catalytic domains that contain the two HKD motifs from both subunits. BfiI utilizes a single active site to cut both DNA strands, which represents a novel mechanism for the scission of double-stranded DNA. It uses a histidine residue from the HKD motif in one subunit as the nucleophile for the cleavage of the target phosphodiester bond in both of the anti-parallel DNA strands, while the symmetrically-related histidine residue from the HKD motif of the opposite subunit acts as the proton donor/acceptor during both strand-scission events.</td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14523">pfam14523</a></td><td>101</td><td>31</td><td>28.0</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Syntaxin_2</td><td>Syntaxin-like protein. This domain includes syntaxin-like domains including from the Vam3p protein.</td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2260">COG2260</a></td><td>59</td><td>42</td><td>28.0</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Nop10</td><td>rRNA maturation protein Nop10, contains Zn-ribbon domain </td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2880">COG2880</a></td><td>67</td><td>52</td><td>27.9</td><td>2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG2880</td><td>Predicted DNA-binding protein, potential antitoxin AbrB/MazE fold </td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00517">pfam00517</a></td><td>204</td><td>20</td><td>27.5</td><td>64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GP41</td><td>Retroviral envelope protein. This family includes envelope protein from a variety of retroviruses. It includes the GP41 subunit of the envelope protein complex from human and simian immunodeficiency viruses (HIV and SIV) which mediate membrane fusion during viral entry. The family also includes bovine immunodeficiency virus, feline immunodeficiency virus and Equine infectious anaemia (EIAV). The family also includes the Gp36 protein from mouse mammary tumor virus (MMTV) and human endogenous retroviruses (HERVs).</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0680">COG0680</a></td><td>160</td><td>39</td><td>27.0</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;]</td><td>HyaD</td><td>Ni,Fe-hydrogenase maturation factor </td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01391">TIGR01391</a></td><td>415</td><td>45</td><td>26.7</td><td>68</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>dnaG</td><td>DNA primase, catalytic core. Members of this family are DNA primase, a ubiquitous bacteria protein. Most members of this family contain nearly two hundred additional residues C-terminal to the region represented here, but conservation between species is poor and the C-terminal region was not included in the seed alignment. This protein contains a CHC2 zinc finger (pfam01807) and a Toprim domain (pfam01751). </td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07454">pfam07454</a></td><td>264</td><td>38</td><td>26.3</td><td>96</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SpoIIP</td><td>Stage II sporulation protein P (SpoIIP). This family contains the bacterial stage II sporulation protein P (SpoIIP) (approximately 350 residues long). It has been shown that a block in polar cytokinesis in Bacillus subtilis is mediated partly by transcription of spoIID, spoIIM and spoIIP. This inhibition of polar division is involved in the locking in of asymmetry after the formation of a polar septum during sporulation. Engulfment in Bacillus subtilis is mediated by two complementary systems: the first includes the proteins SpoIID, SpoIIM and SpoIIP (DMP) which carry out the engulfment, and the second includes the SpoIIQ-SpoIIIAGH (Q-AH) zipper, that recruits other proteins to the septum in a second-phase of the engulfment. The course of events follows as the incorporation firstly of SpoIIB into the septum during division to serve directly or indirectly as a landmark for localizing SpoIIM and then SpoIIP and SpoIID to the septum. SpoIIP and SpoIID interact together to form part of the DMP complex. SpoIIP itself has been identified as an autolysin with peptidoglycan hydrolase activity.</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13674">PRK13674</a></td><td>271</td><td>41</td><td>26.3</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;]</td><td>PRK13674</td><td>putative GTP cyclohydrolase; Provisional</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00352">pfam00352</a></td><td>86</td><td>26</td><td>26.0</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TBP</td><td>Transcription factor TFIID (or TATA-binding protein, TBP). </td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14131">pfam14131</a></td><td>90</td><td>23</td><td>25.9</td><td>79</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4298</td><td>Domain of unknown function (DUF4298). This family of proteins is functionally uncharacterized. This family of proteins is found in bacteria and eukaryotes. Proteins in this family are typically between 94 and 105 amino acids in length. There are two completely conserved residues (Y and D) that may be functionally important.</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09141">cd09141</a></td><td>183</td><td>12</td><td>25.7</td><td>33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLDc_vPLD1_2_yPLD_like_2</td><td>Catalytic domain, repeat 2, of vertebrate phospholipases, PLD1 and PLD2, yeast PLDs, and similar proteins. Catalytic domain, repeat 2, of vertebrate phospholipases D (PLD1 and PLD2), yeast phospholipase D (PLD SPO14/PLD1), and other similar eukaryotic proteins. These PLD enzymes play a pivotal role in transmembrane signaling and cellular regulation. They hydrolyze the terminal phosphodiester bond of phospholipids resulting in the formation of phosphatidic acid and alcohols. Phosphatidic acid is an essential compound involved in signal transduction. PLDs also catalyze the transphosphatidylation of phospholipids to acceptor alcohols, by which various phospholipids can be synthesized. The vertebrate PLD1 and PLD2 are membrane associated phosphatidylinositol 4,5-bisphosphate (PIP2)-dependent enzymes that selectively hydrolyze phosphatidylcholine (PC). Protein cofactors and calcium may be required for their activation. Yeast SPO14/PLD1 is a calcium-independent PLD, which needs PIP2 for its activity. Instead of the regulatory calcium-dependent phospholipid-binding C2 domain in plants, most mammalian and yeast PLDs have adjacent Phox (PX) and the Pleckstrin homology (PH) domains at the N-terminus, which have been shown to mediate membrane targeting of the protein and are closely linked to polyphosphoinositide signaling. The PX and PH domains are also present in zeta-type PLD from Arabidopsis, which is more closely related to vertebrate PLDs than to other plant PLD types. In addition, this subfamily also includes some related proteins which have either PX-like or PH domains in their N-termini. Like other members of the PLD superfamily, the monomer of mammalian and yeast PLDs consists of two catalytic domains, each containing one copy of the conserved HKD motif (H-x-K-x(4)-D, where x represents any amino acid residue). Two HKD motifs from the two domains form a single active site. These PLDs utilize a common two-step ping-pong catalytic mechanism involving an enzyme-substrate intermediate to cleave phosphodiester bonds. The two histidine residues from the two HKD motifs play key roles in the catalysis. Upon substrate binding, a histidine residue from one HKD motif could function as the nucleophile, attacking the phosphodiester bond to create a covalent phosphohistidine intermediate, while the other histidine residue from the second HKD motif could serve as a general acid, stabilizing the leaving group.</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05265">pfam05265</a></td><td>60</td><td>9</td><td>25.6</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF723</td><td>Protein of unknown function (DUF723). This family contains several uncharacterized proteins from Neisseria meningitidis. These proteins may have a role in DNA-binding.</td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15466">pfam15466</a></td><td>140</td><td>15</td><td>25.5</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF4635</td><td>Domain of unknown function (DUF4635). This family of proteins is found in eukaryotes. Proteins in this family are typically between 120 and 154 amino acids in length. There are two conserved sequence motifs: LEQ and DLE.</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13645">pfam13645</a></td><td>176</td><td>7</td><td>25.2</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YkuD_2</td><td>L,D-transpeptidase catalytic domain. This family is related to pfam03734.</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00516">pfam00516</a></td><td>493</td><td>56</td><td>25.0</td><td>73</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;]</td><td>GP120</td><td>Envelope glycoprotein GP120. The entry of HIV requires interaction of viral GP120 with CD4 and a chemokine receptor on the cell surface.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06013">pfam06013</a></td><td>86</td><td>29</td><td>24.6</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>WXG100</td><td>Proteins of 100 residues with WXG. ESAT-6 is a small protein appears to be of fundamental importance in virulence and protective immunity in Mycobacterium tuberculosis. Homologues have been detected in other Gram-positive bacterial species. It may represent a novel secretion system potentially driven by the pfam01580 domains in the YukA-like proteins.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03089">cd03089</a></td><td>443</td><td>12</td><td>24.5</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PMM_PGM</td><td>The phosphomannomutase/phosphoglucomutase (PMM/PGM) bifunctional enzyme catalyzes the reversible conversion of 1-phospho to 6-phospho-sugars (e.g. between mannose-1-phosphate and mannose-6-phosphate or glucose-1-phosphate and glucose-6-phosphate) via a bisphosphorylated sugar intermediate. The reaction involves two phosphoryl transfers, with an intervening 180 degree reorientation of the reaction intermediate during catalysis. Reorientation of the intermediate occurs without dissociation from the active site of the enzyme and is thus, a simple example of processivity, as defined by multiple rounds of catalysis without release of substrate. Glucose-6-phosphate and glucose-1-phosphate are known to be utilized for energy metabolism and cell surface construction, respectively. PMM/PGM belongs to the alpha-D-phosphohexomutase superfamily which includes several related enzymes that catalyze a reversible intramolecular phosphoryl transfer on their sugar substrates. Other members of this superfamily include phosphoglucosamine mutase (PNGM), phosphoacetylglucosamine mutase (PAGM), the bacterial phosphomannomutase ManB, the bacterial phosphoglucosamine mutase GlmM, and the phosphoglucomutases (PGM1 and PGM2). Each of these enzymes has four domains with a centrally located active site formed by four loops, one from each domain. All four domains are included in this alignment model.</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01173">cd01173</a></td><td>254</td><td>74</td><td>24.3</td><td>3.2E+02</td><td>[&nbsp;&nbsp;&nbsp;------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pyridoxal_pyridoxamine_kinase</td><td>Pyridoxal kinase plays a key role in the synthesis of the active coenzyme pyridoxal-5'-phosphate  (PLP), by catalyzing the phosphorylation of the precursor vitamin B6  in the presence of Zn2+ and ATP. Mammals are unable to synthesize PLP de novo and require its precursors in the form of vitamin B6 (pyridoxal, pyridoxine, and pyridoxamine) from their diet. Pyridoxal kinase encoding genes are also found in many other species including yeast and bacteria.</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09791">pfam09791</a></td><td>48</td><td>23</td><td>24.3</td><td>81</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Oxidored-like</td><td>Oxidoreductase-like protein, N-terminal. Members of this family are found in the N terminal region of various oxidoreductase like proteins. Their exact function is, as yet, unknown.</td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1109">COG1109</a></td><td>464</td><td>34</td><td>24.2</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ManB</td><td>Phosphomannomutase </td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09754">pfam09754</a></td><td>220</td><td>45</td><td>24.1</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PAC2</td><td>PAC2 family. This PAC2 (Proteasome assembly chaperone) family of proteins is found in bacteria, archaea and eukaryotes. Proteins in this family are typically between 247 and 307 amino acids in length. These proteins function as a chaperone for the 26S proteasome. The 26S proteasome mediates ubiquitin-dependent proteolysis in eukaryotic cells. A number of studies including very recent ones have revealed that assembly of its 20S catalytic core particle is an ordered process that involves several conserved proteasome assembly chaperones (PACs). Two heterodimeric chaperones, PAC1-PAC2 and PAC3-PAC4, promote the assembly of rings composed of seven alpha subunits.</td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00499">TIGR00499</a></td><td>493</td><td>16</td><td>24.1</td><td>80</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>lysS_bact</td><td>lysyl-tRNA synthetase, eukaryotic and non-spirochete bacterial. This model represents the lysyl-tRNA synthetases that are class II amino-acyl tRNA synthetases. It includes all eukaryotic and most bacterial examples of the enzyme, but not archaeal or spirochete forms. </td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14717">PRK14717</a></td><td>107</td><td>18</td><td>24.0</td><td>53</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14717</td><td>putative glycine/sarcosine/betaine reductase complex protein A; Provisional</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01841">cd01841</a></td><td>174</td><td>83</td><td>24.0</td><td>3.4E+02</td><td>[&nbsp;&nbsp;---------------------------------------------&nbsp;&nbsp;]</td><td>NnaC_like</td><td>NnaC (CMP-NeuNAc synthetase) _like subfamily of SGNH_hydrolases, a diverse family of lipases and esterases. The tertiary fold of the enzyme is substantially different from that of the alpha/beta hydrolase family and unique among all known hydrolases; its active site closely resembles two of the three components of typical Ser-His-Asp(Glu) triad from other serine hydrolases. E. coli NnaC appears to be involved in polysaccharide synthesis.</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06824">pfam06824</a></td><td>403</td><td>38</td><td>23.5</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1237</td><td>Protein of unknown function (DUF1237). This family contains a number of hypothetical proteins of about 450 residues in length. Their function is unknown, and most are bacterial. However, structurally this family is part of the 6 hairpin glycosidase superfamily, suggesting a glycosyl hydrolase function.</td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00615">pfam00615</a></td><td>117</td><td>28</td><td>23.3</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RGS</td><td>Regulator of G protein signaling domain. RGS family members are GTPase-activating proteins for heterotrimeric G-protein alpha-subunits.</td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07424">pfam07424</a></td><td>165</td><td>47</td><td>23.2</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TrbM</td><td>TrbM. This family contains the bacterial protein TrbM (approximately 180 residues long). In Comamonas testosteroni T-2, TrbM is derived from the IncP1beta plasmid pTSA, which encodes the widespread genes for p-toluenesulfonate (TSA) degradation.</td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07130">cd07130</a></td><td>474</td><td>32</td><td>22.8</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ALDH_F7_AASADH</td><td>NAD+-dependent alpha-aminoadipic semialdehyde dehydrogenase, ALDH family members 7A1 and 7B. Alpha-aminoadipic semialdehyde dehydrogenase (AASADH, EC=1.2.1.31), also known as ALDH7A1, Antiquitin-1, ALDH7B, or delta-1-piperideine-6-carboxylate dehydrogenase (P6CDH), is a NAD+-dependent ALDH. Human ALDH7A1 is involved in the pipecolic acid pathway of lysine catabolism, catalyzing the oxidation of alpha-aminoadipic semialdehyde to alpha-aminoadipate.  Arabidopsis thaliana ALDH7B4 appears to be an osmotic-stress-inducible ALDH gene encoding a turgor-responsive or stress-inducible ALDH. The Streptomyces clavuligerus P6CDH appears to be involved in cephamycin biosynthesis, catalyzing the second stage of the two-step conversion of lysine to alpha-aminoadipic acid.  The ALDH7A1 enzyme and others in this group have been observed as tetramers, yet the bacterial P6CDH enzyme has been reported as a monomer.</td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04106">cd04106</a></td><td>162</td><td>27</td><td>22.8</td><td>94</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rab23_like</td><td>Rab GTPase family 23 (Rab23)-like. Rab23-like subfamily. Rab23 is a member of the Rab family of small GTPases. In mouse, Rab23 has been shown to function as a negative regulator in the sonic hedgehog (Shh) signaling pathway. Rab23 mediates the activity of Gli2 and Gli3, transcription factors that regulate Shh signaling in the spinal cord, primarily by preventing Gli2 activation in the absence of Shh ligand. Rab23 also regulates a step in the cytoplasmic signal transduction pathway that mediates the effect of Smoothened (one of two integral membrane proteins that are essential components of the Shh signaling pathway in vertebrates). In humans, Rab23 is expressed in the retina. Mice contain an isoform that shares 93% sequence identity with the human Rab23 and an alternative splicing isoform that is specific to the brain. This isoform causes the murine open brain phenotype, indicating it may have a role in the development of the central nervous system. GTPase activating proteins (GAPs) interact with GTP-bound Rab and accelerate the hydrolysis of GTP to GDP. Guanine nucleotide exchange factors (GEFs) interact with GDP-bound Rabs to promote the formation of the GTP-bound state. Rabs are further regulated by guanine nucleotide dissociation inhibitors (GDIs), which facilitate Rab recycling by masking C-terminal lipid binding and promoting cytosolic localization. Most Rab GTPases contain a lipid modification site at the C-terminus, with sequence motifs CC, CXC, or CCX. Lipid binding is essential for membrane attachment, a key feature of most Rab proteins. Due to the presence of truncated sequences in this CD, the lipid modification site is not available for annotation.</td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3113">COG3113</a></td><td>99</td><td>30</td><td>22.4</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MlaB</td><td>ABC-type transporter Mla maintaining outer membrane lipid asymmetry, MlaB component, contains STAS domain  </td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13124">pfam13124</a></td><td>40</td><td>14</td><td>22.3</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3963</td><td>Protein of unknown function (DUF3963). This family of proteins is functionally uncharacterized. This family of proteins is found in bacteria. Proteins in this family are typically between 42 and 85 amino acids in length. There is a conserved DIQKW sequence motif.</td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13793">pfam13793</a></td><td>117</td><td>17</td><td>22.1</td><td>66</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pribosyltran_N</td><td>N-terminal domain of ribose phosphate pyrophosphokinase. This family is frequently found N-terminal to the Pribosyltran, pfam00156.</td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0736">COG0736</a></td><td>127</td><td>22</td><td>20.9</td><td>47</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AcpS</td><td>Phosphopantetheinyl transferase (holo-ACP synthase) </td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13732">pfam13732</a></td><td>84</td><td>17</td><td>20.7</td><td>90</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;]</td><td>DUF4162</td><td>Domain of unknown function (DUF4162). This domain is found at the C-terminus of bacterial ABC transporter proteins. The function is not known.</td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07875">pfam07875</a></td><td>64</td><td>31</td><td>20.5</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Coat_F</td><td>Coat F domain. The Coat F proteins, which contribute to the Bacillales spore coat. It occurs multiple times in the genomes it is found in.</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09097">cd09097</a></td><td>329</td><td>60</td><td>20.2</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Deadenylase_CCR4</td><td>C-terminal deadenylase domain of CCR4 and related domains. This subfamily contains the C-terminal catalytic domain of the deadenylases, Saccharomyces cerevisiae Ccr4p and two vertebrate homologs (CCR4a and CCR4b), and related domains. CCR4 belongs to the large EEP (exonuclease/endonuclease/phosphatase) superfamily that contains functionally diverse enzymes that share a common catalytic mechanism of cleaving phosphodiester bonds. CCR4 is the major deadenylase subunit of the CCR4-NOT transcription complex, which contains two deadenylase subunits and several noncatalytic subunits. The other deadenylase subunit, Caf1 (called Pop2 in yeast), is a DEDD-type protein and does not belong in this superfamily. Saccharomyces cerevisiae CCR4 (or Ccr4p) is a 3'-5' poly(A) RNA and ssDNA exonuclease. It is the catalytic subunit of the yeast mRNA deadenylase (Ccr4p/Pop2p/Not complex). This complex participates in various ways in mRNA metabolism, including transcription initiation and elongation, and mRNA degradation. Ccr4p degrades both poly(A) and single-stranded DNA. There are two vertebrate homologs of Ccr4p, CCR4a (also called CCR4-NOT transcription complex subunit 6 or CNOT6) and CCR4b (also called CNOT6-like or CNOT6L), which independently associate with other components to form distinct CCR4-NOT multisubunit complexes. The nuclease domain of CNOT6 and CNOT6L exhibits Mg2+-dependent deadenylase activity, with specificity for poly (A) RNA as substrate. CCR4a is a component of P-bodies and is necessary for foci formation. CCR4b regulates p27/Kip1 mRNA levels, thereby influencing cell cycle progression. They both contribute to the prevention of cell death by regulating insulin-like growth factor-binding protein 5.</td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12093">cd12093</a></td><td>44</td><td>10</td><td>20.1</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TM_ErbB1</td><td>Transmembrane domain of Epidermal Growth Factor Receptor or ErbB1, a Protein Tyrosine Kinase. PTKs catalyze the transfer of the gamma-phosphoryl group from ATP to tyrosine (tyr) residues in protein substrates. EGFR (HER1, ErbB1) is a receptor PTK (RTK) containing an extracellular EGF-related ligand-binding region, a transmembrane (TM) helix, and a cytoplasmic region with a tyr kinase domain and a regulatory C-terminal tail. It is activated by ligand-induced dimerization, leading to the phosphorylation of tyr residues in the C-terminal tail, which serve as binding sites for downstream signaling molecules. Ligands for ErbB1 include EGF, heparin binding EGF-like growth factor (HBEGF), epiregulin, amphiregulin, TGFalpha, and betacellulin. Upon ligand binding, ErbB1 can form homo- or heterodimers with other EGFR/ErbB subfamily members. The TM domain not only serves as a membrane anchor, but also plays an important role in receptor dimerization and optimal activation. Mutations in the TM domain of ErbB1 have been associated with increased breast cancer risk. The ErbB1 signaling pathway is one of the most important pathways regulating cell proliferation, differentiation, survival, and growth. A number of monoclonal antibodies and small molecule inhibitors have been developed that target ErbB1, including the antibodies Cetuximab and Panitumumab, which are used in combination with other therapies for the treatment of colorectal cancer and non-small cell lung carcinoma (NSCLC). The small molecule inhibitors Gefitinib (Iressa) and Erlotinib (Tarceva), already used for NSCLC, are undergoing clinical trials for other types of cancer including gastrointestinal, breast, head and neck, and bladder.</td></tr>
</table>
</body>
