<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09623">pfam09623</a></td><td>225</td><td>153</td><td>98.7</td><td>1.6E-07</td><td>[----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_NE0113</td><td>CRISPR-associated protein NE0113 (Cas_NE0113). Members of this minor CRISPR-associated (Cas) protein family are encoded in cas gene clusters in Vibrio vulnificus YJ016, Nitrosomonas europaea ATCC 19718, Mannheimia succiniciproducens MBEL55E, and Verrucomicrobium spinosum.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09741">cd09741</a></td><td>219</td><td>131</td><td>98.5</td><td>1.9E-06</td><td>[--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as NE0113 family</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09723">cd09723</a></td><td>132</td><td>122</td><td>98.5</td><td>1.3E-06</td><td>[-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as csx13 family</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03642">TIGR03642</a></td><td>124</td><td>115</td><td>97.9</td><td>0.00014</td><td>[-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_csx14</td><td>CRISPR-associated protein, Csx14 family. This model describes a protein N-terminal protein sequence domain strictly associated with CRISPR and CRISPR-associated protein systems. This model and TIGR02584 identify two separate clades from a larger homology domain family, both CRISPR-associated, while other homologs are found that may not be. Members are found in bacteria that include Pelotomaculum thermopropionicum SI, Thermoanaerobacter tengcongensis MB4, and Roseiflexus sp. RS-1, and in archaea that include Thermoplasma volcanium, Picrophilus torridus, and Methanospirillum hungatei. The molecular function is unknown.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09880">cd09880</a></td><td>139</td><td>113</td><td>97.7</td><td>0.00015</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Smg5-Smg6-like</td><td>PIN domain. PIN (PilT N terminus) domain of nonsense-mediated decay (NMD) factors, Smg5 and Smg6, and homologs are included in this family. Smg5 and Smg6 are essential factors in NMD, a post-transcriptional regulatory pathway that recognizes and rapidly degrades mRNAs containing premature translation termination codons. In vivo, the Smg6 PIN domain elicits degradation of bound mRNAs, as well as, metal-ion dependent, degradation of single-stranded RNA, in vitro. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  Point mutation studies of the conserved aspartate residues in the catalytic center of the Smg6 PIN domain revealed that Smg6 is the endonuclease involved in human NMD. However, Smg5 lacks several of these key catalytic residues and does not degrade single-stranded RNA, in vivo. Eukaryotic Smg5 and Smg6-like PIN domains are present at the C-terminal end of telomerase activating proteins, Est1. Many of the bacterial homologs in this group have an N-terminal PIN domain and a C-terminal PhoH-like ATPase domain.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09651">pfam09651</a></td><td>136</td><td>111</td><td>97.5</td><td>0.0014</td><td>[&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_APE2256</td><td>CRISPR-associated protein (Cas_APE2256). This entry represents a conserved region of about 150 amino acids found in at least five archaeal and three bacterial species. These species all contain CRISPRs (Clustered Regularly Interspaced Short Palindromic Repeats). In six of eight species, the protein is encoded the vicinity of a CRISPR/Cas locus.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13638">pfam13638</a></td><td>130</td><td>37</td><td>97.4</td><td>0.00097</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_4</td><td>PIN domain. Members of this family of bacterial domains are predicted to be RNases (from similarities to 5'-exonucleases).</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09879">cd09879</a></td><td>117</td><td>110</td><td>97.2</td><td>0.0016</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_AF0591</td><td>PIN domain of Archaeoglobus fulgidus AF0591 protein and other similar archaeal homologs. PIN (PilT N terminus) domain of Archaeoglobus fulgidus AF0591 protein and other similar uncharacterized archaeal homologs are included in this family. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domains within this subgroup contain four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1412">COG1412</a></td><td>136</td><td>115</td><td>97.0</td><td>0.0032</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fcf1</td><td>rRNA-processing protein FCF1 </td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09742">cd09742</a></td><td>183</td><td>114</td><td>96.9</td><td>0.0064</td><td>[&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csm6_III-A</td><td>CRISPR/Cas system-associated protein Csm6. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; loosely associated with CRISPR/Cas systems; also known as APE2256 family</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09883">cd09883</a></td><td>147</td><td>42</td><td>96.6</td><td>0.0036</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_PhoHL-ATPase</td><td>PIN domain of bacterial Smg6-like homologs with PhoH-like ATPase domains. PIN (PilT N terminus) domain of Smg6-like bacterial proteins with C-terminal PhoH-like ATPase domains and other similar homologs are included in this family. Eukaryotic Smg5 and Smg6 nucleases are essential factors in nonsense-mediated mRNA decay (NMD), a post-transcriptional regulatory pathway that recognizes and rapidly degrades mRNAs containing premature translation termination codons. In vivo, the Smg6 PIN domain elicits degradation of bound mRNAs, as well as, metal ion dependent, degradation of single-stranded RNA, in vitro. These PIN domains are homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this subgroup contain four highly conserved acidic residues (putative metal-binding, active site residues). Many of the bacterial homologs in this group have an N-terminal PIN domain and a C-terminal PhoH-like ATPase domain and are predicted to be ATPases which are induced by phosphate starvation.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09455">pfam09455</a></td><td>372</td><td>89</td><td>96.0</td><td>0.12</td><td>[&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_DxTHG</td><td>CRISPR-associated (Cas) DxTHG family. CRISPR is a term for Clustered Regularly Interspaced Short Palidromic Repeats. A number of protein families appear only in association with these repeats and are designated Cas (CRISPR associated) proteins. The family describes Cas proteins of about 400 residues that include the motif </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09747">cd09747</a></td><td>378</td><td>90</td><td>95.5</td><td>0.084</td><td>[&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as Cas02710 family</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01884">TIGR01884</a></td><td>203</td><td>108</td><td>94.4</td><td>0.79</td><td>[-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_HTH</td><td>CRISPR locus-related DNA-binding protein. Most but not all examples of this family are associated with CRISPR loci, a combination of DNA repeats and characteristic proteins encoded near the repeat cluster. The C-terminal region of this protein is homologous to DNA-binding helix-turn-helix domains with predicted transcriptional regulatory activity. </td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09732">cd09732</a></td><td>221</td><td>114</td><td>93.9</td><td>0.5</td><td>[&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as TM1812 family</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09670">pfam09670</a></td><td>379</td><td>90</td><td>93.7</td><td>0.45</td><td>[&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cas_Cas02710</td><td>CRISPR-associated protein (Cas_Cas02710). Members of this family are found, exclusively in the vicinity of CRISPR repeats and other CRISPR-associated (cas) genes, in Methanothermobacter thermautotrophicus (Methanobacterium thermoformicicum), Thermus thermophilus (Deinococcus-Thermus), Chloroflexus aurantiacus (Chloroflexi), and Thermomicrobium roseum (Thermomicrobia).</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09878">cd09878</a></td><td>124</td><td>99</td><td>93.6</td><td>0.096</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_VirB11L-ATPase</td><td>PIN domain of the Methanocaldococcus Jannaschii Dsm 2661 protein, Thermococcus sibiricus MM 739 predicted ATPase, and other similar archaeal homologs. PIN (PilT N terminus) domain present N-terminal of AAA+, VirB11-like ATPases, as well as, the PIN domains of proteins from Methanopyrus kandleri AV19 and Thermococcus sibiricus MM 739, and other similar archaeal homologs are included in this family. Several members of this subfamily possess an AAA+, VirB11-like ATPase domain, flanked by PIN and KH nucleic acid-binding domains.  VirB11-ATPase is a type IV secretory pathway component required for T-pilus biogenesis and virulence. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domains within this subgroup contain four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4006">COG4006</a></td><td>278</td><td>151</td><td>92.6</td><td>0.47</td><td>[&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4006</td><td>CRISPR/Cas system-associated protein Csm6, COG1517 family </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09885">cd09885</a></td><td>182</td><td>46</td><td>90.9</td><td>0.13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Smg6</td><td>PIN domain of human telomerase-binding protein EST1, Smg6, and other similar eukaryotic homologs. Nonsense-mediated decay (NMD) factors, Smg5 and Smg6 are essential to the post-transcriptional regulatory pathway, NMD, which recognizes and rapidly degrades mRNAs containing premature translation termination codons. In vivo, the Smg6 PIN (PilT N terminus) domain elicits degradation of bound mRNAs, as well as, metal ion dependent, degradation of single-stranded RNA, in vitro. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domains within this subgroup contain four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.  Point mutation studies of the conserved aspartate residues in the catalytic center of the Smg6 PIN domain revealed that Smg6 is the endonuclease involved in human NMD. However, Smg5 lacks several of these key catalytic residues and does not degrade single-stranded RNA, in vivo. Eukaryotic Smg6 PIN domains are present at the C-terminal end of the telomerase activating proteins, Est1.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09728">cd09728</a></td><td>400</td><td>119</td><td>89.9</td><td>3.4</td><td>[------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as DxTHG family</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09686">cd09686</a></td><td>209</td><td>140</td><td>88.7</td><td>4.9</td><td>[---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as NE0113 family</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09655">cd09655</a></td><td>198</td><td>107</td><td>88.1</td><td>5.7</td><td>[------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CasRa_I-A</td><td>CRISPR/Cas system-associated transcriptional regulator CasRa. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Predicted transcriptional regulator of CRISPR/Cas system</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09002">pfam09002</a></td><td>379</td><td>98</td><td>87.5</td><td>3.1</td><td>[&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1887</td><td>Domain of unknown function (DUF1887). This domain is found in a set of hypothetical bacterial proteins.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02584">TIGR02584</a></td><td>209</td><td>140</td><td>87.4</td><td>6.3</td><td>[---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_NE0113</td><td>CRISPR-associated protein, NE0113 family. Members of this minor CRISPR-associated (Cas) protein family are found in cas gene clusters in Vibrio vulnificus YJ016, Nitrosomonas europaea ATCC 19718, Mannheimia succiniciproducens MBEL55E, and Verrucomicrobium spinosum.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1875">COG1875</a></td><td>436</td><td>47</td><td>87.1</td><td>0.55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YlaK</td><td>Predicted ribonuclease YlaK, contains NYN-type RNase and PhoH-family ATPase domains  </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09863">cd09863</a></td><td>125</td><td>101</td><td>87.0</td><td>2.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Smg6-like</td><td>PIN domain of human telomerase-binding protein EST1, Smg6, and ribosome assembly factor, Nob1, Archaeoglobus fulgidus AF0591 protein and other eukaryotic, bacterial, and archaeal homologs. PIN (PilT N terminus) domains of eukaryotic ribonucleases such as Smg5 and Smg6, essential factors in nonsense-mediated mRNA decay (NMD), and Nob1, a ribosome assembly factor critical in pre-rRNA processing, as well as, the uncharacterized archaeal Archaeoglobus fulgidus AF0591 protein and other eukaryotic, bacterial, and archaeal homologs are included in this family. Smg5 and Smg6 are essential factors in NMD, a post-transcriptional regulatory pathway that recognizes and rapidly degrades mRNAs containing premature translation termination codons. In vivo, the Smg6 PIN domain elicits degradation of bound mRNAs, as well as, metal-ion dependent, degradation of single-stranded RNA, in vitro. The Nob1 PIN domain binds the single-stranded cleavage site D at the 3'end of 18S rRNA. Recombinant Nob1 binds as a tetramer to pre-18S rRNA fragments containing cleavage site D and believed to cleave at this site.  These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, that is seen in FEN1-like PIN domains. PIN domains within this group typically contain three or four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09864">cd09864</a></td><td>143</td><td>116</td><td>86.3</td><td>6.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Fcf1</td><td>PIN domain of rRNA-processing protein, Fcf1 (Utp24, YDR339C), and other eukaryotic homologs. Fcf1/Utp24 (FAF1-copurifying factor 1/U three-associated protein 24) is an essential protein involved in pre-rRNA processing and 40S ribosomal subunit assembly. Component of the small subunit (SSU) processome, Fcf1 is an essential nucleolar protein that is required for processing of the 18S pre-rRNA at sites A0-A2. The Fcf1 protein was reported to interact with Pmc1p (vacuolar Ca2+ ATPase) and Cor1p (core subunit of the ubiquinol-cytochrome c reductase complex). The PIN (PilT N terminus) domain of this protein is a homolog of flap endonuclease-1 (FEN1)-like PIN domains, but apparently lack the H3TH domain or extensive arch/clamp region seen in the latter. PIN domains typically contain three or four conserved acidic residues (putative metal-binding, active site residues). The Fcf1 PIN domain subfamily has four of these putative active site residues and the Fcf1-Utp23 homolog PIN domain subfamily has three of them. Point mutation studies of the conserved acidic residues in the putative active site of Saccharomyces cerevisiae Fcf1 determined they were essential for pre-rRNA processing at sites A1 and A2, whereas the presence of the Fcf1 protein itself is also required for cleavage at site A0.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09862">cd09862</a></td><td>182</td><td>44</td><td>83.8</td><td>0.62</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_Rrp44</td><td>PIN domain of yeast exosome subunit Rrp44 endoribonuclease and other eukaryotic homologs. PIN (PilT N terminus) domain of the Saccharomyces cerevisiae exosome subunit Rrp44 (Ribosomal RNA-processing protein 44 or Protein Dis3 homolog) and other similar eukaryotic homologs are included in this family. The eukaryotic exosome is a conserved macromolecular complex responsible for many RNA-processing and RNA-degradation reactions. It is composed of nine core subunits that directly binds Rrp44. The Rrp44 nuclease is the catalytic subunit of the exosome and has endonuclease activity in the PIN domain and an exoribonuclease activity in its RNase II-like region. Rrp44 binding to the exosome is mediated mainly by the PIN domain and by subunits Rrp41-Rrp45, and binding predictions indicate that the PIN domain active site is positioned on the outer surface of the exosome. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains.  PIN domains within this subgroup contain four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule. Recombinant Rrp44 was shown to possess manganese-dependent endonuclease activity in vitro that was abolished by point mutations in the putative metal binding residues of its PIN domain.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13764">PRK13764</a></td><td>602</td><td>116</td><td>81.8</td><td>1.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13764</td><td>ATPase; Provisional</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13470">pfam13470</a></td><td>80</td><td>40</td><td>75.8</td><td>3.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_3</td><td>PIN domain. Members of this family of bacterial domains are predicted to be RNases (from similarities to 5'-exonucleases).</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01850">pfam01850</a></td><td>120</td><td>41</td><td>75.7</td><td>4.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN</td><td>PIN domain. </td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09660">cd09660</a></td><td>394</td><td>94</td><td>75.1</td><td>27</td><td>[&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as MJ1666 family</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1569">COG1569</a></td><td>142</td><td>41</td><td>73.7</td><td>8.6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1569</td><td>Predicted nucleic acid-binding protein, contains PIN domain  </td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05913">pfam05913</a></td><td>357</td><td>64</td><td>73.3</td><td>8.1</td><td>[&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF871</td><td>Bacterial protein of unknown function (DUF871). This family consists of several conserved hypothetical proteins from bacteria and archaea. The function of this family is unknown.</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02710">TIGR02710</a></td><td>380</td><td>91</td><td>70.9</td><td>9.4</td><td>[&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02710</td><td>CRISPR-associated protein, TIGR02710 family. Members of this family are found, exclusively in the vicinity of CRISPR repeats and other CRISPR-associated (cas) genes, in Methanothermobacter thermautotrophicus (Archaea), Thermus thermophilus (Deinococcus-Thermus), Chloroflexus aurantiacus (Chloroflexi), and Thermomicrobium roseum (Thermomicrobia).</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02619">TIGR02619</a></td><td>149</td><td>112</td><td>64.1</td><td>60</td><td>[&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02619</td><td>putative CRISPR-associated protein, APE2256 family. This model represents a conserved domain of about 150 amino acids found in at least five archaeal species and three bacterial species, exclusively in species with CRISPR (Clustered Regularly Interspaced Short Palidromic Repeats). In six of eight species, the member of this family is in the vicinity of a CRISPR/Cas locus.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01897">TIGR01897</a></td><td>410</td><td>95</td><td>63.5</td><td>45</td><td>[&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_MJ1666</td><td>CRISPR-associated protein, MJ1666 family. CRISPR is a term for Clustered, Regularly Interspaced Short Palidromic Repeats. A number of protein families appear only in association with these repeats and are designated Cas (CRISPR-Associated) proteins. This model describes a Cas protein about 400 residues in length, found mostly in the Archaea but also in Aquifex.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02739">pfam02739</a></td><td>169</td><td>90</td><td>63.3</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>5_3_exonuc_N</td><td>5'-3' exonuclease, N-terminal resolvase-like domain. </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01936">pfam01936</a></td><td>141</td><td>74</td><td>63.2</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NYN</td><td>NYN domain. These domains are found in the eukaryotic proteins typified by the Nedd4-binding protein 1 and the bacterial YacP-like proteins (Nedd4-BP1, YacP nucleases; NYN domains). The NYN domain shares a common protein fold with two other previously characterized groups of nucleases, namely the PIN (PilT N-terminal) and FLAP/5' --> 3' exonuclease superfamilies. These proteins share a common set of 4 acidic conserved residues that are predicted to constitute their active site. Based on the conservation of the acidic residues and structural elements Aravind and colleagues suggest that PIN and NYN domains are likely to bind only a single metal ion, unlike the FLAP/5' --> 3' exonuclease superfamily, which binds two metal ions. Based on conserved gene neighborhoods Aravind and colleagues infer that the bacterial members are likely to be components of the processome/degradsome that process tRNAs or ribosomal RNAs.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04900">pfam04900</a></td><td>101</td><td>85</td><td>60.9</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fcf1</td><td>Fcf1. Fcf1 is a nucleolar protein involved in pre-rRNA processing. Depletion of yeast Fcf1 and Fcf2 leads to a decrease in synthesis of the 18S rRNA and results in a deficit in 40S ribosomal subunits.</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13522">cd13522</a></td><td>368</td><td>108</td><td>60.7</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_ABC_oligosaccharides</td><td>The periplasmic-binding component of ABC transport systems specific for maltose and related oligosaccharides; possess type 2 periplasmic binding fold. This family represents the periplasmic binding component of ABC transport systems involved in uptake of oligosaccharides including maltose, trehalose, maltodextrin, and cyclodextrin. Members of this family belong to the type 2 periplasmic-binding fold superfamily. PBP2 is comprised of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. The majority of PBP2 proteins function in the uptake of small soluble substrates in eubacteria and archaea. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two cytoplasmically located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09694">cd09694</a></td><td>181</td><td>110</td><td>59.7</td><td>66</td><td>[&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csm6_III-A</td><td>CRISPR/Cas system-associated protein Csm6. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; loosely associated with CRISPR/Cas systems</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14826">pfam14826</a></td><td>162</td><td>70</td><td>59.1</td><td>44</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FACT-Spt16_Nlob</td><td>FACT complex subunit SPT16 N-terminal lobe domain. The FACT or facilitator of chromatin transcription complex binds to and alters the properties of nucleosomes. This family represents the N-terminal lobe of the NTD, or N-terminal domain, and acts as a protein-protein interaction domain presumably with partners outside of the FACT complex. Knockout of the whole NTD domain, 1-450 residues in UniProt:P32558, in yeast serves to tender the cells sensitive to DNA replication stress but is not lethal. The C-terminal half of NTD is structurally similar to aminopeptidases, and the most highly conserved surface residues line a cleft equivalent to the aminopeptidase substrate-binding site, family peptidase_M24, pfam00557.</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09702">cd09702</a></td><td>378</td><td>101</td><td>59.1</td><td>20</td><td>[&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as TIGR02710 family</td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02221">TIGR02221</a></td><td>218</td><td>119</td><td>57.9</td><td>56</td><td>[&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cas_TM1812</td><td>CRISPR-associated protein, TM1812 family. CRISPR is a term for Clustered Regularly Interspaced Short Palidromic Repeats. A number of protein families appear only in association with these repeats and are designated Cas (CRISPR associated) proteins. This family, represented by TM1812 of Thermotoga maritima, is found also in Vibrio vulnificus YJ016, Nitrosomonas europaea ATCC 19718, a large plasmid of Synechocystis sp. PCC 6803, and Fibrobacter succinogenes S85.</td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11297">cd11297</a></td><td>133</td><td>53</td><td>54.9</td><td>7.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LabA_like_N_1</td><td>Uncharacterized subfamily of N-terminal LabA-like domains. This N-terminal domain is found in a well conserved group of mainly bacterial proteins with no defined function, which contain a C-terminal LabA_like_C domain. LabA from Synechococcus elongatus PCC 7942, (which does not contain this C-terminal domain), has been shown to play a role in cyanobacterial circadian timing. The LabA-like C-terminal domains characteristic of this subfamily may be related to the LOTUS domain family (which also co-occurs with LabA-like N-terminal domains). The function of the N-terminal domain is unknown. LabA_like domains exhibit some similarity to the NYN domain, a distant relative of the PIN-domain nucleases.</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09852">cd09852</a></td><td>120</td><td>17</td><td>49.9</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_SF</td><td>PIN (PilT N terminus) domain: Superfamily. PIN_SF The PIN (PilT N terminus) domain belongs to a large nuclease superfamily with representatives from eukaryota, eubacteria, and archaea. PIN domains were originally named for their sequence similarity to the N-terminal domain of an annotated pili biogenesis protein, PilT, a domain fusion between a PIN-domain and a PilT ATPase domain.  The structural properties of the PIN domain indicate its putative active center, consisting of invariant acidic amino acid residues (putative metal-binding residues) is geometrically similar in the active center of structure-specific 5' nucleases (also known as Flap endonuclease-1-like), PIN-domain ribonucleases of eukaryotic rRNA editing proteins, and bacterial toxins of toxin-antitoxin (TA) operons. Seen here, are two major divisions in the PIN domain superfamily. The first major division, the structure-specific 5' nuclease family, is represented by FEN1, the 5'-3' exonuclease of DNA polymerase I, and T4 RNase H nuclease PIN domains. These 5' nucleases are involved in DNA replication, repair, and recombination. They are capable of both 5'-3' exonucleolytic activity and cleaving bifurcated DNA, in an endonucleolytic, structure-specific manner. Unique to FEN1-like nucleases, the PIN domain has a helical arch/clamp region (I domain) of variable length (approximately 16 to 800 residues) and, inserted within the C-terminal region of the PIN domain, a H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region. Both the H3TH domain (not included here) and the helical arch/clamp region are involved in DNA binding. With the exception of Mkt1, these nucleases have a carboxylate rich active site that is involved in binding essential divalent metal ion cofactors (Mg2+, Mn2+, Zn2+, or Co2+).  The second major division of the PIN domain superfamily, the VapC-Smg6 family, includes such eukaryotic ribonucleases as, Smg6, an essential factor in nonsense-mediated mRNA decay; Rrp44, the catalytic subunit of the exosome; and Nob1, a ribosome assembly factor critical in pre-rRNA processing. A large percentage of members in this family are bacterial ribonuclease toxins of TA operons such as Mycobacterium tuberculosis VapC and Neisseria gonorrhoeae FitB, as well as, archaeal homologs, Pyrobaculum aerophilum Pea0151 and P. aerophilum Pae2754. Also included are the eukaryotic Fcf1/ Utp24 (FAF1-copurifying factor 1/U three-associated protein 24) and Utp23-like proteins.  Components of the small subunit processome, Fcf1/Utp24 and Utp23 are essential proteins involved in pre-rRNA processing and 40S ribosomal subunit assembly.</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09668">cd09668</a></td><td>214</td><td>75</td><td>49.7</td><td>34</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as TM1812 family</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10910">cd10910</a></td><td>145</td><td>45</td><td>49.3</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>limkain_b1_N_like</td><td>N-terminal LabA-like domain of limkain b1 and similar proteins. This eukaryotic subfamily of LabA-like domains contains the N-terminal domain of human limkain b1, a human autoantigen, localized to a subset of ABCD3 and PXF marked peroxisomes. Limkain b1 may be a relatively common target of human autoantibodies reactive to cytoplasmic vesicle-like structures. Limkain b1 contains multiple copies of LOTUS domains and a conserved RNA recognition motif, its - and similar - domain architectures are shared by several members of this family, and a function of these architectures in RNA binding or RNA metabolism has been suggested. The function of the N-terminal domain is unknown. LabA_like domains exhibit some similarity to the NYN domain, a distant relative of the PIN-domain nucleases.</td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01261">pfam01261</a></td><td>202</td><td>24</td><td>46.7</td><td>35</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AP_endonuc_2</td><td>Xylose isomerase-like TIM barrel. This TIM alpha/beta barrel structure is found in xylose isomerase and in endonuclease IV (EC:3.1.21.2). This domain is also found in the N termini of bacterial myo-inositol catabolism proteins. These are involved in the myo-inositol catabolism pathway, and is required for growth on myo-inositol in Rhizobium leguminosarum bv. viciae.</td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06167">cd06167</a></td><td>151</td><td>45</td><td>40.8</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LabA_like</td><td>LabA_like proteins. A well conserved group of domains found in all three kingdoms of life, with no defined function. LabA, a member from Synechococcus elongatus PCC 7942, has been shown to play a role in cyanobacterial circadian timing. It is required for negative feedback regulation of the autokinase/autophosphatase KaiC, a central component of the circadian clock system. In particular, LabA seems necessary for KaiC-dependent repression of gene expression. Another member is the N-terminal domain of limkain b1, a human autoantigen associated with cytoplasmic vesicles. Curiously, a gene labeled NicB from Pseudomonas putida S16, which is described as a putative NADH-dependent hydroxylase involved in the microbial degradation of nicotine also falls in the LabA_like family. LabA_like domains exhibit some similarity to the NYN domain, a distant relative of the PIN-domain nucleases.</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06424">cd06424</a></td><td>315</td><td>113</td><td>38.2</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UGGPase</td><td>UGGPase catalyzes the synthesis of UDP-Glucose/UDP-Galactose. UGGPase: UDP-Galactose/Glucose Pyrophosphorylase catalyzes the reversible production of UDP-Glucose/UDP-Galactose and pyrophosphate (PPi) from Glucose-1-phosphate/Galactose-1-phosphate and UTP. Its dual substrate specificity distinguishes it from the single substrate enzyme UDP-glucose pyrophosphorylase. It may play a key role in the galactose metabolism in raffinose oligosaccharide (RFO) metabolizing plants. RFO raffinose is a major photoassimilate and is a galactosylderivative of sucrose (Suc) containing a galactose (Gal) moiety. Upon arriving at the sink tissue, the Gal moieties of the RFOs are initially removed by alpha-galactosidase and then are phosphorylated to Gal-1-P. Gal-1-P is converted to UDP-Gal. The UDP-Gal is further metabolized to UDP-Glc via an epimerase reaction. The UDP-Glc can be directly utilized in cell wall metabolism or in Suc synthesis. However, for the Suc synthesis UDP-Glc must be further metabolized to Glc-1-P. This can be carried out either by the UGPase in the reverse direction or by the dual substrate PPase itself operating in the reverse direction. According to the latter possibility, the three-step pathway of Gal-1-P to Glc-1-P could be carried out by a single PPase, functioning sequentially in reverse directions separated by the epimerase reaction.</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09671">cd09671</a></td><td>346</td><td>98</td><td>37.5</td><td>1.2E+02</td><td>[&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Csx1_III-U</td><td>CRISPR/Cas system-associated protein Csx1. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Protein of this family often fused to HTH domain; Some proteins could have an additional fusion with RecB-family nuclease domain; Core domain appears to have a Rossmann-like fold; loosely associated with CRISPR/Cas systems; also known as DxTHG family</td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08311">PRK08311</a></td><td>237</td><td>107</td><td>37.1</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08311</td><td>putative RNA polymerase sigma factor SigI; Reviewed</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06828">cd06828</a></td><td>373</td><td>63</td><td>36.8</td><td>94</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLPDE_III_DapDC</td><td>Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzyme Diaminopimelate Decarboxylase. Diaminopimelate decarboxylase (DapDC, EC 4.1.1.20) participates in the last step of lysine biosynthesis. It converts meso-2,6-diaminoheptanedioate to L-lysine. It is a fold type III PLP-dependent enzyme that contains an N-terminal PLP-binding TIM-barrel domain and a C-terminal beta-sandwich domain, similar to bacterial alanine racemases. DapDC exists as homodimers with active sites that lie at the interface between the TIM barrel domain of one subunit and the beta-sandwich domain of the other subunit. Homodimer formation and the presence of the PLP cofactor are required for catalytic activity.</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12427">pfam12427</a></td><td>23</td><td>13</td><td>33.4</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3665</td><td>Branched-chain amino acid aminotransferase. This domain family is found in bacteria, and is typically between 23 and 35 amino acids in length. The family is found in association with pfam01063. There is a conserved TRT sequence motif.</td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd02982">cd02982</a></td><td>103</td><td>68</td><td>32.9</td><td>2.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PDI_b'_family</td><td>Protein Disulfide Isomerase (PDIb') family, redox inactive TRX-like domain b'; composed of eukaryotic proteins involved in oxidative protein folding in the endoplasmic reticulum (ER) by acting as catalysts and folding assistants. Members of this family include PDI, calsequestrin and other PDI-related proteins like ERp72, ERp57 (or ERp60), ERp44, P5 and PDIR. PDI, ERp57, ERp72, P5 and PDIR are all oxidases, catalyzing the formation of disulfide bonds of newly synthesized polypeptides in the ER. They also exhibit reductase activity in acting as isomerases to correct any non-native disulfide bonds, as well as chaperone activity to prevent protein aggregation and facilitate the folding of newly synthesized proteins. These proteins contain multiple copies of a redox active TRX (a) domain containing a CXXC motif, and one or more redox inactive TRX-like (b) domains. The molecular structure of PDI is abb'a'. Also included in this family is the PDI-related protein ERp27, which contains only redox-inactive TRX-like (b and b') domains. The redox inactive domains are implicated in substrate recognition with the b' domain serving as the primary substrate binding site. Only the b' domain is necessary for the binding of small peptide substrates. In addition to the b' domain, other domains are required for the binding of larger polypeptide substrates. The b' domain is also implicated in chaperone activity.</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10967">cd10967</a></td><td>202</td><td>56</td><td>32.3</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CE4_GLA_like_6s</td><td>Putative catalytic NodB homology domain of gellan lyase and similar proteins. This family is represented by the extracellular polysaccharide-degrading enzyme, gellan lyase (gellanase, EC 4.2.2.-), from Bacillus sp. The enzyme acts on gellan exolytically and releases a tetrasaccharide of glucuronyl-glucosyl-rhamnosyl-glucose with unsaturated glucuronic acid at the nonreducing terminus. The family also includes many uncharacterized prokaryotic polysaccharide deacetylases, which show high sequence similarity to Bacillus sp. gellan lyase. Although their biological functions remain unknown, all members of the family contain a conserved domain with a 6-stranded beta/alpha barrel, which is similar to the catalytic NodB homology domain of rhizobial NodB-like proteins, belonging to the larger carbohydrate esterase 4 (CE4) superfamily.</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03693">pfam03693</a></td><td>81</td><td>50</td><td>32.0</td><td>50</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RHH_2</td><td>Uncharacterized protein family (UPF0156). This family of proteins are about 80 amino acids in length and their function is unknown. The proteins contain a conserved GRY motif. This family appears to be related to ribbon-helix-helix DNA-binding proteins.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03769">cd03769</a></td><td>134</td><td>54</td><td>31.9</td><td>78</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SR_IS607_transposase_like</td><td>Serine Recombinase (SR) family, IS607-like transposase subfamily, catalytic domain; members contain a DNA binding domain with homology to MerR/SoxR located N-terminal to the catalytic domain. Serine recombinases catalyze site-specific recombination of DNA molecules by a concerted, four-strand cleavage and rejoining mechanism which involves a transient phosphoserine linkage between DNA and the enzyme. They are functionally versatile and include resolvases, invertases, integrases, and transposases. This subfamily is composed of proteins that catalyze the transposition of insertion sequence (IS) elements such as IS607 from Helicobacter and IS1535 from Mycobacterium, and similar proteins from other bacteria and several archaeal species. IS elements are DNA segments that move to new sites in prokaryotic and eukaryotic genomes causing insertion mutations and gene rearrangements.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00739">cd00739</a></td><td>257</td><td>106</td><td>30.3</td><td>91</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DHPS</td><td>DHPS subgroup of Pterin binding enzymes. DHPS (dihydropteroate synthase), a functional homodimer, catalyzes the condensation of p-aminobenzoic acid (pABA) in the de novo biosynthesis of folate, which is an essential cofactor in both nucleic acid and protein biosynthesis. Prokaryotes (and some lower eukaryotes) must synthesize folate de novo, while higher eukaryotes are able to utilize dietary folate and therefore lack DHPS.  Sulfonamide drugs, which are substrate analogs of pABA, target DHPS.</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02095">TIGR02095</a></td><td>473</td><td>53</td><td>30.2</td><td>81</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>glgA</td><td>glycogen/starch synthase, ADP-glucose type. This family consists of glycogen (or starch) synthases that use ADP-glucose (EC 2.4.1.21), rather than UDP-glucose (EC 2.4.1.11) as in animals, as the glucose donor. This enzyme is found in bacteria and plants. Whether the name given is glycogen synthase or starch synthase depends on context, and therefore on substrate. </td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4956">COG4956</a></td><td>356</td><td>105</td><td>29.8</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG4956</td><td>Uncharacterized conserved protein YacL, contains PIN and TRAM domains </td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1855">COG1855</a></td><td>604</td><td>114</td><td>29.4</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1855</td><td>Predicted ATPase, PilT family  </td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13586">cd13586</a></td><td>367</td><td>106</td><td>29.4</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_Maltose_binding_like</td><td>The periplasmic-binding component of ABC transport systems specific for maltose and related polysaccharides; possess type 2 periplasmic binding fold. This subfamily represents the periplasmic binding component of ABC transport systems involved in uptake of polysaccharides including maltose, maltodextrin, and cyclodextrin. Members of this family belong to the type 2 periplasmic-binding fold superfamily. PBP2 is comprised of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. The majority of PBP2 proteins function in the uptake of small soluble substrates in eubacteria and archaea. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two cytoplasmically located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03083">cd03083</a></td><td>80</td><td>41</td><td>27.9</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TRX_Fd_NuoE_hoxF</td><td>TRX-like </td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00305">TIGR00305</a></td><td>114</td><td>97</td><td>27.8</td><td>2.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00305</td><td>putative toxin-antitoxin system toxin component, PIN family. This uncharacterized protein family, part of the PIN domain superfamily, is restricted to bacteria and archaea. A comprehensive in silico study of toxin-antitoxin systems by Makarova, et al. (2009) finds evidence this family represents the toxin-like component of one class of type 2 toxin-antitoxin systems. </td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08345">PRK08345</a></td><td>289</td><td>57</td><td>27.7</td><td>1.4E+02</td><td>[------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08345</td><td>cytochrome-c3 hydrogenase subunit gamma; Provisional</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09877">cd09877</a></td><td>127</td><td>101</td><td>27.3</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_YacL</td><td>PIN domain of Thermus Thermophilus Hb8, uncharacterized Bacillus subtilis YacL, and other bacterial homologs. PIN (PilT N terminus) domain of the conserved membrane protein of unknown function of Thermus Thermophilus Hb8, Bacillus subtilis YacL and other similar homologs are included in this family. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domain homologs within this group contain four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.  Proteins in this group have a C-terminal TRAM domain whose function is unknown but predicted to be a RNA-binding domain common to tRNA uracil methylation and adenine thiolation enzymes.</td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00593">TIGR00593</a></td><td>887</td><td>98</td><td>27.2</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pola</td><td>DNA polymerase I. All proteins in this family for which functions are known are DNA polymerases Many also have an exonuclease motif. This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University). </td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03230">cd03230</a></td><td>173</td><td>73</td><td>25.7</td><td>41</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ABC_DR_subfamily_A</td><td>ATP-binding cassette domain of the drug resistance transporter and related proteins, subfamily A. This family of ATP-binding proteins belongs to a multi-subunit transporter involved in drug resistance (BcrA and DrrA), nodulation, lipid transport, and lantibiotic immunity. In bacteria and archaea, these transporters usually include an ATP-binding protein and one or two integral membrane proteins. Eukaryotic systems of the ABCA subfamily display ABC domains that are quite similar to this family. The ATP-binding domain shows the highest similarity between all members of the ABC transporter family. ABC transporters are a subset of nucleotide hydrolases that contain a signature motif, Q-loop, and H-loop/switch region, in addition to, the Walker A motif/P-loop and Walker B motif commonly found in a number of ATP- and GTP-binding and hydrolyzing proteins.</td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02824">TIGR02824</a></td><td>325</td><td>137</td><td>25.3</td><td>2.6E+02</td><td>[&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>quinone_pig3</td><td>putative NAD(P)H quinone oxidoreductase, PIG3 family. Members of this family are putative quinone oxidoreductases that belong to the broader superfamily (modeled by Pfam pfam00107) of zinc-dependent alcohol (of medium chain length) dehydrogenases and quinone oxiooreductases. The alignment shows no motif of conserved Cys residues as are found in zinc-binding members of the superfamily, and members are likely to be quinone oxidoreductases instead. A member of this family in Homo sapiens, PIG3, is induced by p53 but is otherwise uncharacterized. </td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02549">TIGR02549</a></td><td>21</td><td>16</td><td>25.3</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CRISPR_DxTHG</td><td>CRISPR-associated DxTHG motif protein. This model describes a short region highly conserved between two otherwise substantially different CRISPR-associated (cas) proteins, TIGR02221 and TIGR01987. This region includes the motif </td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10911">cd10911</a></td><td>155</td><td>49</td><td>25.2</td><td>76</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LabA</td><td>LabA_like proteins. A well conserved group of bacterial and archaeal proteins with no defined function. LabA, a member from Synechococcus elongatus PCC 7942, has been shown to play a role in cyanobacterial circadian timing. It is required for negative feedback regulation of the autokinase/autophosphatase KaiC, a central component of the circadian clock system. In particular, LabA seems necessary for KaiC-dependent repression of gene expression. LabA_like domains exhibit some similarity to the NYN domain, a distant relative of the PIN-domain nucleases.</td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09854">cd09854</a></td><td>125</td><td>111</td><td>24.7</td><td>59</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_VapC-Smg6_family</td><td>PIN domains of VapC and Smg6 ribonucleases, ribosome assembly factor NOB1, rRNA-processing protein Fcf1, Archaeoglobus fulgidus AF0591 protein, and homologs. PIN (PilT N terminus) domains of such ribonucleases as the toxins of prokaryotic toxin/antitoxin operons FitAB and VapBC, as well as, eukaryotic ribonucleases such as Smg6, ribosome assembly factor NOB1, exosome subunit Rrp44 endoribonuclease and, rRNA-processing protein Fcf1, are included in this family. Also included are the PIN domains of the Pyrobaculum aerophilum Pea0151 and Archaeoglobus fulgidus AF0591 proteins and other similar archaeal homologs. These PIN domains are structural homologs of flap endonuclease-1 (FEN1)-like PIN domains, but lack the extensive arch/clamp region  and the H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region, seen in FEN1-like PIN domains. PIN domains typically contain three or four highly conserved acidic residues (putative metal-binding, active site residues) which cluster at the C-terminal end of the beta-sheet and form a negatively charged pocket near the center of the molecule.</td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01496">TIGR01496</a></td><td>257</td><td>52</td><td>23.4</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DHPS</td><td>dihydropteroate synthase. This model represents dihydropteroate synthase, the enzyme that catalyzes the second to last step in folic acid biosynthesis. The gene is usually designated folP (folic acid biosynthsis) or sul (sulfanilamide resistance). This model represents one branch of the family of pterin-binding enzymes (pfam00809) and of a cluster of dihydropteroate synthase and related enzymes (COG0294). Other members of pfam00809 and COG0294 are represented by model TIGR00284. </td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG3589">COG3589</a></td><td>360</td><td>64</td><td>21.9</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG3589</td><td>Uncharacterized protein </td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0297">COG0297</a></td><td>487</td><td>49</td><td>21.8</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GlgA</td><td>Glycogen synthase </td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04028">pfam04028</a></td><td>74</td><td>45</td><td>21.6</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF374</td><td>Domain of unknown function (DUF374). Bacterial domain of unknown function.</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03729">cd03729</a></td><td>42</td><td>11</td><td>21.5</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SOCS_ASB13</td><td>SOCS (suppressors of cytokine signaling) box of ASB13-like proteins. ASB family members have a C-terminal SOCS box and an N-terminal ankyrin-related sequence. The general function of the SOCS box is the recruitment of the ubiquitin-transferase system. The SOCS box interacts with Elongins B and C, Cullin-5 or Cullin-2, Rbx-1, and E2. Therefore, SOCS-box-containing proteins probably function as E3 ubiquitin ligases and mediate the degradation of proteins associated through their N-terminal regions.</td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0028">COG0028</a></td><td>550</td><td>74</td><td>21.5</td><td>4E+02</td><td>[---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>IlvB</td><td>Acetolactate synthase large subunit or other thiamine pyrophosphate-requiring enzyme </td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04665">pfam04665</a></td><td>241</td><td>61</td><td>21.1</td><td>2.6E+02</td><td>[------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pox_A32</td><td>Poxvirus A32 protein. The A32 protein is thought to be involved in viral DNA packaging.</td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0148">COG0148</a></td><td>423</td><td>68</td><td>21.0</td><td>2E+02</td><td>[&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Eno</td><td>Enolase </td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11017">pfam11017</a></td><td>314</td><td>48</td><td>20.8</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2855</td><td>Protein of unknown function (DUF2855). This family of proteins has no known function.</td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06836">PRK06836</a></td><td>394</td><td>94</td><td>20.6</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06836</td><td>aspartate aminotransferase; Provisional</td></tr>
<tr><td>86</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09859">cd09859</a></td><td>174</td><td>46</td><td>20.6</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PIN_53EXO</td><td>PIN domain of the 5'-3' exonuclease of Taq DNA polymerase I and homologs. The 5'-3' exonuclease (53EXO) PIN (PilT N terminus) domain of multi-domain DNA polymerase I and single domain protein homologs are included in this family.  Taq contains a polymerase domain for synthesizing a new DNA strand and a 53EXO PIN domain for cleaving RNA primers or damaged DNA strands. Taq's 53EXO PIN domain recognizes and endonucleolytically cleaves a structure-specific DNA substrate that has a bifurcated downstream duplex and an upstream template-primer duplex that overlaps the downstream duplex by 1 bp. The 53EXO PIN domain cleaves the unpaired 5'-arm of the overlap flap DNA substrate. 5'-3' exonucleases are members of the structure-specific, 5' nuclease family that catalyzes hydrolysis of DNA duplex-containing nucleic acid structures during DNA replication, repair, and recombination. These nucleases contain a PIN domain with a helical arch/clamp region (I domain) of variable length (approximately 16 residues in 53EXO PIN domains) and a H3TH (helix-3-turn-helix) domain, an atypical helix-hairpin-helix-2-like region.  Both the H3TH domain (not included here) and the helical arch/clamp region are involved in DNA/RNA binding. The active site includes a set of conserved acidic residues that are essential for binding divalent metal ions required for nuclease activity.</td></tr>
<tr><td>87</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06745">pfam06745</a></td><td>231</td><td>67</td><td>20.5</td><td>4.5E+02</td><td>[&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>KaiC</td><td>KaiC. This family represents a conserved region within bacterial and archaeal proteins, most of which are hypothetical. More than one copy is sometimes found in each protein. This family includes KaiC, which is one of the Kai proteins among which direct protein-protein association may be a critical process in the generation of circadian rhythms in cyanobacteria.</td></tr>
<tr><td>88</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08266">cd08266</a></td><td>342</td><td>101</td><td>20.4</td><td>5.7E+02</td><td>[&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Zn_ADH_like1</td><td>Alcohol dehydrogenases of the MDR family. This group contains proteins related to the zinc-dependent  alcohol dehydrogenases. However, while the group has structural zinc site characteristic of these enzymes, it lacks the consensus site for a catalytic zinc. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.   Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation. ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form.  The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide. A GxGxxG motif after the first mononucleotide contact half allows the close contact of the coenzyme with the ADH backbone. The N-terminal catalytic domain has a distant homology to GroES. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria),  and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site, and a structural zinc in a lobe of the catalytic domain. NAD(H)-binding occurs in the cleft between the catalytic  and coenzyme-binding domains at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding. In human ADH catalysis, the zinc ion helps coordinate the alcohol, followed by deprotonation of  a histidine, the ribose of NAD, a serine, then the alcohol, which allows the transfer of a hydride to NAD+, creating NADH and a zinc-bound aldehyde or ketone. In yeast and some bacteria, the active site zinc binds an aldehyde, polarizing it, and leading to the reverse reaction.</td></tr>
<tr><td>89</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08763">pfam08763</a></td><td>32</td><td>15</td><td>20.3</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ca_chan_IQ</td><td>Voltage gated calcium channel IQ domain. Voltage gated calcium channels control cellular calcium entry in response to changes in membrane potential. The isoleucine-glutamine (IQ) motif in the voltage gated calcium channel IQ domain interacts with hydrophobic pockets of Ca2+/calmodulin. The interaction regulates two self-regulatory calcium dependent feedback mechanism, calcium dependent inactivation (CDI), and calcium-dependent facilitation (CDF).</td></tr>
<tr><td>90</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01048">TIGR01048</a></td><td>414</td><td>67</td><td>20.2</td><td>1.9E+02</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>lysA</td><td>diaminopimelate decarboxylase. This family consists of diaminopimelate decarboxylase, an enzyme which catalyzes the conversion of diaminopimelic acid into lysine during the last step of lysine biosynthesis. </td></tr>
</table>
</body>
