<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12010">pfam12010</a></td><td>129</td><td>48</td><td>72.3</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------&nbsp;&nbsp;]</td><td>DUF3502</td><td>Domain of unknown function (DUF3502). This presumed domain is functionally uncharacterized. This domain is found in bacteria. This domain is about 140 amino acids in length. This domain is found associated with pfam01547.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11558">cd11558</a></td><td>169</td><td>48</td><td>60.5</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>W2_eIF2B_epsilon</td><td>C-terminal W2 domain of eukaryotic translation initiation factor 2B epsilon. eIF2B is a heteropentameric complex which functions as a guanine nucleotide exchange factor in the recycling of eIF-2 during the initiation of translation in eukaryotes. The epsilon and gamma subunits are sequence similar and both are essential in yeast. Epsilon appears to be the catalytically active subunit, with gamma enhancing its activity. The C-terminal domain of the eIF2B epsilon subunit contains bipartite motifs rich in acidic and aromatic residues, which are responsible for the interaction with eIF2. The structure of the domain resembles that of a set of concatenated HEAT repeats.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14150">PRK14150</a></td><td>193</td><td>44</td><td>59.6</td><td>19</td><td>[&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK14150</td><td>heat shock protein GrpE; Provisional</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11533">cd11533</a></td><td>75</td><td>35</td><td>59.2</td><td>5.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;]</td><td>NTP-PPase_Af0060_like</td><td>Nucleoside Triphosphate Pyrophosphohydrolase (EC 3.6.1.8) MazG-like domain found in uncharacterized protein from Archaeoglobus fulgidus (Af0060) and its bacterial homologs. This family includes an uncharacterized protein from Archaeoglobus fulgidus (Af0060) and its homologs from bacteria. Although its biological role remains unclear, Af0060 shows high sequence similarity to the dimeric 2-deoxyuridine 5'-triphosphate nucleotidohydrolase (dUTP pyrophosphatase or dUTPase) and NTP-PPase MazG proteins. However, unlike typical tandem-domain MazG proteins, members in this family consist of a single MazG-like domain that contains a well conserved divalent ion-binding motif EXX</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15601">pfam15601</a></td><td>133</td><td>37</td><td>59.0</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;]</td><td>Imm42</td><td>Immunity protein 42. A predicted immunity protein with an alpha+beta fold and conserved tyrosine and tryptophan residues. Proteins containing this domain are present in bacterial polymorphic toxin systems as an immediate gene neighbor of the toxin gene, which usually contains toxin domains of the Tox-REase-10 family.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14298">cd14298</a></td><td>38</td><td>26</td><td>56.1</td><td>18</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UBA2_scUBP14_like</td><td>UBA2 domain found in Saccharomyces cerevisiae ubiquitin carboxyl-terminal hydrolase 14 (scUBP14) and similar proteins. scUBP14, also called deubiquitinating enzyme 14, glucose-induced degradation protein 6, ubiquitin thioesterase 14, or ubiquitin-specific-processing protease 14, is the yeast ortholog of human Isopeptidase T (USP5), a deubiquitinating enzyme known to bind the 29-linked polyubiquitin chains. scUBP14 has been identified as a K29-linked polyubiquitin binding protein as well. It is involved in K29-linked polyubiquitin metabolism by binding to the 29-linked Ub4 resin and serving as an internal positive control in budding yeast. Members in this family contain two tandem ubiquitin-association (UBA) domains. This model corresponds to the UBA2 domain.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02985">TIGR02985</a></td><td>161</td><td>26</td><td>54.9</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Sig70_bacteroi1</td><td>RNA polymerase sigma-70 factor, Bacteroides expansion family 1. This group of sigma factors are members of the sigma-70 family (TIGR02937) and are found primarily in the genus Bacteroides. This family appears to have resulted from a lineage-specific expansion as B. thetaiotaomicron VPI-5482, Bacteroides forsythus ATCC 43037, Bacteroides fragilis YCH46 and Bacteroides fragilis NCTC 9343 contain 25, 12, 24 and 23 members, respectively. There are currentlyonly two known members of this family outside of the Bacteroides, in Rhodopseudomonas and Bradyrhizobium.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00191">pfam00191</a></td><td>66</td><td>40</td><td>52.3</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Annexin</td><td>Annexin. This family of annexins also includes giardin that has been shown to function as an annexin.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06654">PRK06654</a></td><td>181</td><td>22</td><td>52.2</td><td>14</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;]</td><td>fliL</td><td>flagellar basal body-associated protein FliL; Reviewed</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13274">pfam13274</a></td><td>108</td><td>20</td><td>51.2</td><td>22</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------]</td><td>DUF4065</td><td>Protein of unknown function (DUF4065). This family of proteins is functionally uncharacterized. This family of proteins is found in bacteria, archaea and viruses. Proteins in this family are typically between 155 and 202 amino acids in length.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4086">COG4086</a></td><td>299</td><td>19</td><td>48.8</td><td>65</td><td>[&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YpuA</td><td>Uncharacterized protein YpuA, DUF1002 family </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13877">PRK13877</a></td><td>114</td><td>64</td><td>48.3</td><td>43</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13877</td><td>conjugal transfer relaxosome component TraJ; Provisional</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00440">PRK00440</a></td><td>319</td><td>61</td><td>47.2</td><td>64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rfc</td><td>replication factor C small subunit; Reviewed</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15368">pfam15368</a></td><td>170</td><td>56</td><td>45.3</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;&nbsp;]</td><td>BioT2</td><td>Spermatogenesis family BioT2. BioT2 is a family of eukaryotic proteins expressed only in the testes. BioT2 is found abundantly in five types of murine cancer cell lines, suggesting it plays a role in testes development as well as tumorigenesis.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00644">TIGR00644</a></td><td>539</td><td>21</td><td>44.2</td><td>19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------]</td><td>recJ</td><td>single-stranded-DNA-specific exonuclease RecJ. All proteins in this family are 5'-3' single-strand DNA exonucleases. These proteins are used in some aspects of mismatch repair, recombination, and recombinational repair. </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01003">cd01003</a></td><td>229</td><td>26</td><td>42.5</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------]</td><td>PBP2_YckB</td><td>Substrate binding domain of an ABC cystine transporter; the type 2 periplasmic binding protein fold. Periplasmic cystine-binding domain (YckB) of an ATP-binding cassette (ABC) transporter from Bacillus subtilis and its related proteins. Cystine is an oxidized dimeric form of cysteine that is required for optimal bacterial growth. In Bacillus subtilis, three ABC transporters, TcyJKLMN (YtmJKLMN), TcyABC (YckKJI), and YxeMNO are involved in uptake of cystine. Likewise, three uptake systems were identified in Salmonella enterica serovar Typhimurium, while in Escherichia coli, two transport systems seem to be involved in cystine uptake.  Moreover, L-cystine limitation was shown to prevent virulence of Neisseria gonorrhoeae; thus, its L-cystine solute receptor (Ngo0372) may be suited as target for an antimicrobial vaccine. The cystine receptor belongs to the type 2 periplasmic binding fold protein superfamily (PBP2). The PBP2 proteins are typically comprises of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two receptor cytoplasmically-located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12221">pfam12221</a></td><td>45</td><td>19</td><td>42.4</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;]</td><td>HflK_N</td><td>Bacterial membrane protein N terminal. This domain is found in bacteria. This domain is typically between 65 to 81 amino acids in length. This domain is found associated with pfam01145. This domain is the N terminal of the bacterial membrane protein HflK. HflK complexes with HflC to form a membrane protease which is modulated by the GTPase HflX. The N terminal domain of HflK is the membrane spanning region which anchors the protein in the bacterial membrane.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00196">pfam00196</a></td><td>58</td><td>27</td><td>42.1</td><td>26</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GerE</td><td>Bacterial regulatory proteins, luxR family. </td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07606">cd07606</a></td><td>202</td><td>22</td><td>41.2</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------]</td><td>BAR_SFC_plant</td><td>The Bin/Amphiphysin/Rvs (BAR) domain of the plant protein SCARFACE (SFC). BAR domains are dimerization, lipid binding and curvature sensing modules found in many different proteins with diverse functions including organelle biogenesis, membrane trafficking or remodeling, and cell division and migration. The plant protein SCARFACE (SFC), also called VAscular Network 3 (VAN3), is a plant ACAP (ArfGAP with Coiled-coil, ANK repeat and PH domain containing protein), an Arf GTPase Activating Protein (GAP) that plays a role in the trafficking of auxin efflux regulators from the plasma membrane to the endosome. It is required for the normal vein patterning in leaves. SCF contains an N-terminal BAR domain, followed by a Pleckstrin Homology (PH) domain, an Arf GAP domain, and C-terminal ankyrin (ANK) repeats. BAR domains form dimers that bind to membranes, induce membrane bending and curvature, and may also be involved in protein-protein interactions.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10153">cd10153</a></td><td>88</td><td>19</td><td>41.2</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;]</td><td>RcnR-FrmR-like_DUF156</td><td>Transcriptional regulators RcnR and FrmR, and related domains; this domain family was previously known as part of DUF156. This domain family includes various transcriptional regulators that respond to different stressors. It includes Escherichia coli RncR (formally known as YohL, nickel and cobalt-sensitive), and E. coli FrmR (formally known as YaiN, formaldehyde sensitive). Escherichia coli RncR represses expression of the gene encoding the nickel and cobalt-efflux protein RcnA; RcnA may act through modulating NikR, to repress the NIkABCDE nickel transporter. In vitro, purified RncR binds to the rncA promoter DNA fragment in the absence of Ni2+ or Co2+, and the affinity of RncR for this promoter is reduced in the presence of excess nickel.  Escherichia coli FrmR regulates the formaldehyde degradation frmRAB operon. This family belongs to a larger superfamily that includes CsoRs (copper-sensitive operon repressors). CsoRs form homotetramers (dimer of dimers). In Mycobacterium tuberculosis CsoR, within each dimer, two Cys residues on opposite subunits, along with a His residue, bind the Cu(I) ion (forming a triagonal S2N coordination complex, C-H-C). These residues are conserved in the majority of members of this superfamily. In this family, however, not all these residues are conserved; in E.coli RcnR and FrmR there is a His or a Thr instead of the second Cys (C-H-H or C-H-T) respectively. For E. coli FrmR, an N-terminal His residue, not conserved in all members of this family, is also involved in metal binding (H-C-H-H). A conserved Tyr and a Glu residue that facilitate allosteric regulation of DNA binding for CsoRs are poorly conserved in this family.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13518">pfam13518</a></td><td>52</td><td>30</td><td>41.0</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_28</td><td>Helix-turn-helix domain. This helix-turn-helix domain is often found in transposases and is likely to be DNA-binding.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2771">COG2771</a></td><td>65</td><td>27</td><td>41.0</td><td>56</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CsgD</td><td>DNA-binding transcriptional regulator, CsgD family </td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd10157">cd10157</a></td><td>85</td><td>25</td><td>40.9</td><td>33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;]</td><td>BsCsoR-like_DUF156</td><td>Bacillus subtilis copper-sensitive operon repressor (BsCsoR), and related domains; this family was previously known as part of DUF156. This domain family includes Bacillus subtilis CsoR (BsCsoR). CsoRs are Cu(I)-inducible, and regulate the expression of genes involved in copper homeostasis. BsCsoR regulates the copZA operon which encodes the copper chaperone CopZ, and the copper efflux P-type ATPase CopA. This family belongs to a larger superfamily that contains various transcriptional regulators that respond to different stressors such as Cu(I), Ni(I), sulfite, and formaldehyde, and includes Mycobacterium tuberculosis CsoR (MtCsoR), Thermus thermophilus CsoR, and Staphylococcus aureus CsoR. The latter three proteins do not belong to this family. CsoRs regulate the expression of genes involved in copper homeostasis. CsoRs form homotetramers (dimer of dimers). In MtCsoR, within each dimer, two Cys residues on opposite subunits, along with a His residue, bind the Cu(I) ion (forming a triagonal S2N coordination complex, C-H-C). These residues are conserved in the majority of members of this superfamily, including this family, and the conserved Tyr and a Glu residue that facilitate allosteric regulation of DNA binding for CsoRs are also well conserved.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02923">TIGR02923</a></td><td>343</td><td>79</td><td>39.0</td><td>1.5E+02</td><td>[&nbsp;&nbsp;---------------------------------------------&nbsp;&nbsp;]</td><td>AhaC</td><td>ATP synthase A1, C subunit. The A1/A0 ATP synthase is homologous to the V-type (V1/V0, vacuolar) ATPase, but functions in the ATP synthetic direction as does the F1/F0 ATPase of bacteria. The C subunit is part of the hydrophilic A1 "stalk" complex (AhaABCDEFG), which is the site of ATP generation and is coupled to the membrane-embedded proton translocating A0 complex.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06371">pfam06371</a></td><td>187</td><td>43</td><td>38.0</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Drf_GBD</td><td>Diaphanous GTPase-binding Domain. This domain is bound to by GTP-attached Rho proteins, leading to activation of the Drf protein.</td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02020">pfam02020</a></td><td>77</td><td>29</td><td>37.9</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>W2</td><td>eIF4-gamma/eIF5/eIF2-epsilon. This domain of unknown function is found at the C-terminus of several translation initiation factors.</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07719">pfam07719</a></td><td>34</td><td>21</td><td>37.3</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_2</td><td>Tetratricopeptide repeat. This Pfam entry includes outlying Tetratricopeptide-like repeats (TPR) that are not matched by pfam00515.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK12791">PRK12791</a></td><td>131</td><td>40</td><td>37.2</td><td>64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>flbT</td><td>flagellar biosynthesis repressor FlbT; Reviewed</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15265">pfam15265</a></td><td>514</td><td>14</td><td>37.0</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FAM196</td><td>FAM196 family. This protein family is a domain of unknown function. This family of proteins is found in eukaryotes and are typically between 441 and 534 amino acids in length.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13174">pfam13174</a></td><td>33</td><td>17</td><td>36.5</td><td>47</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;]</td><td>TPR_6</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam15337">pfam15337</a></td><td>97</td><td>47</td><td>34.6</td><td>57</td><td>[&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Vasculin</td><td>Vascular protein family Vasculin-like 1. GC-rich promoter-binding protein 1-like 1 or Vasculin-like protein family 1, is likely to be a transcription factor. The domain family is found in eukaryotes, and is approximately 90 amino acids in length.</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12703">cd12703</a></td><td>81</td><td>43</td><td>34.3</td><td>51</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM4_ROD1</td><td>RNA recognition motif 4 in vertebrate regulator of differentiation 1 (Rod1). This subgroup corresponds to the RRM4 of ROD1 coding protein Rod1, a mammalian polypyrimidine tract binding protein (PTB) homolog of a regulator of differentiation in the fission yeast Schizosaccharomyces pombe, where the nrd1 gene encodes an RNA binding protein that negatively regulates the onset of differentiation. ROD1 is predominantly expressed in hematopoietic cells or organs. It might play a role controlling differentiation in mammals. Rod1 contains four repeats of RNA recognition motifs (RRM), also known as RBD (RNA binding domain) or RNP (ribonucleoprotein domain) and does have RNA binding activities. </td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03186">pfam03186</a></td><td>295</td><td>34</td><td>33.7</td><td>60</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CobD_Cbib</td><td>CobD/Cbib protein. This family includes CobD proteins from a number of bacteria, in Salmonella this protein is called Cbib. Salmonella CobD is a different protein. This protein is involved in cobalamin biosynthesis and is probably an enzyme responsible for the conversion of adenosylcobyric acid to adenosylcobinamide or adenosylcobinamide phosphate.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1937">COG1937</a></td><td>89</td><td>61</td><td>33.6</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------&nbsp;&nbsp;]</td><td>FrmR</td><td>DNA-binding transcriptional regulator, FrmR family </td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07575">pfam07575</a></td><td>564</td><td>49</td><td>33.0</td><td>58</td><td>[&nbsp;&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Nucleopor_Nup85</td><td>Nup85 Nucleoporin. A family of nucleoporins conserved from yeast to human. THe nuclear pore complex is a large assembly composed of two essential complexes: the heptameric Nup84 complex and the heteromeric Nic96-containing complex. The Nup84 complex is composed of one copy each of Nup84, Nup85, Nup120, Nup133, Nup145C, Sec13, and Seh1. The structure of a complex of Nup85 and Seh1 was solved. The N-terminus of Nup85 is inserted and forms a three-stranded blade that completes the Seh1 6-bladed beta-propeller in trans. Following its N-terminal insertion blade, Nup85 forms a compact cuboid structure composed of 20 helices, with two distinct modules, referred to as crown and trunk.</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01678">cd01678</a></td><td>738</td><td>26</td><td>32.6</td><td>38</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PFL1</td><td>Pyruvate formate lyase 1. Pyruvate formate lyase catalyzes a key step in anaerobic glycolysis, the conversion of pyruvate and CoenzymeA to formate and acetylCoA. The PFL mechanism involves an unusual radical cleavage of pyruvate in which two cysteines and one glycine form radicals that are required for catalysis. PFL has a ten-stranded alpha/beta barrel domain that is structurally similar to those of all three ribonucleotide reductase (RNR) classes as well as benzylsuccinate synthase and B12-independent glycerol dehydratase.</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08463">pfam08463</a></td><td>162</td><td>51</td><td>32.0</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>EcoEI_R_C</td><td>EcoEI R protein C-terminal. The restriction enzyme EcoEI recognizes 5'-GAGN(7)ATGC-3' and is composed of the three proteins R, M, and S. The domain described here is found at the C-terminus of the R protein (HsdR) which is required for both nuclease and ATPase activity.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2522">COG2522</a></td><td>119</td><td>53</td><td>31.9</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------------&nbsp;&nbsp;]</td><td>COG2522</td><td>Predicted transcriptional regulator  </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01878">TIGR01878</a></td><td>97</td><td>30</td><td>31.1</td><td>60</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;]</td><td>cas_Csa5</td><td>CRISPR type I-A/APERN-associated protein Csa5. CRISPR is a term for Clustered, Regularly Interspaced Short Palidromic Repeats. A number of protein families appear only in association with these repeats and are designated Cas (CRISPR-Associated) proteins. This model represents a minor family of Cas protein found in the (all archaeal) APERN subtype of CRISPR/Cas locus, so the family is designated Csa5, for CRISPR/Cas Subtype Protein 5.</td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11561">cd11561</a></td><td>157</td><td>40</td><td>30.8</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>W2_eIF5</td><td>C-terminal W2 domain of eukaryotic translation initiation factor 5. eIF5 functions as a GTPase acceleration protein (GAP), as well as a GDP dissociation inhibitor (GDI) during translational initiation in eukaryotes. The structure of this C-terminal domain resembles that of a set of concatenated HEAT repeats.</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12701">cd12701</a></td><td>76</td><td>33</td><td>30.8</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM4_PTBP1</td><td>RNA recognition motif 4 in vertebrate polypyrimidine tract-binding protein 1 (PTB). This subgroup corresponds to the RRM4 of PTB, also known as 58 kDa RNA-binding protein PPTB-1 or heterogeneous nuclear ribonucleoprotein I (hnRNP I), an important negative regulator of alternative splicing in mammalian cells. PTB also functions at several other aspects of mRNA metabolism, including mRNA localization, stabilization, polyadenylation, and translation. PTB contains four RNA recognition motifs (RRM), also known as RBD (RNA binding domain) or RNP (ribonucleoprotein domain). RRM1 and RRM2 are independent from each other and separated by flexible linkers. By contrast, there is an unusual and conserved interdomain interaction between RRM3 and RRM4. It is widely held that only RRMs 3 and 4 are involved in RNA binding and RRM2 mediates PTB homodimer formation. However, new evidence shows that the RRMs 1 and 2 also contribute substantially to RNA binding. Moreover, PTB may not always dimerize to repress splicing. It is a monomer in solution. </td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12295">pfam12295</a></td><td>181</td><td>33</td><td>30.0</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Symplekin_C</td><td>Symplekin tight junction protein C terminal. This domain family is found in eukaryotes, and is approximately 180 amino acids in length. There is a single completely conserved residue P that may be functionally important. Symplekn has been localized, by light and electron microscopy, to the plaque associated with the cytoplasmic face of the tight junction-containing zone (zonula occludens) of polar epithelial cells and of Sertoli cells of testis. However, both the mRNA and the protein can also be detected in a wide range of cell types that do not form tight junctions. Careful analyses have revealed that the protein occurs in all these diverse cells in the nucleoplasm, and only in those cells forming tight junctions is it recruited, partly but specifically, to the plaque structure of the zonula occludens.</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2197">COG2197</a></td><td>211</td><td>44</td><td>29.7</td><td>89</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CitB</td><td>DNA-binding response regulator, NarL/FixJ family, contains REC and HTH domains </td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01209">PRK01209</a></td><td>312</td><td>37</td><td>29.6</td><td>78</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>cobD</td><td>cobalamin biosynthesis protein; Provisional</td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13181">pfam13181</a></td><td>34</td><td>16</td><td>29.3</td><td>57</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TPR_8</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14364">cd14364</a></td><td>40</td><td>16</td><td>29.1</td><td>36</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CUE_ASCC2</td><td>CUE domain found in activating signal cointegrator 1 complex subunit 2 (ASCC2) and similar proteins. ASCC2, also called ASC-1 complex subunit p100 or Trip4 complex subunit p100, together with ASCC1 (also called p50) and ASCC3 (also called p300), form the activating signal cointegrator complex (ASCC). ASCC plays an essential role in activating protein 1 (AP-1), serum response factor (SRF), and nuclear factor kappaB (NF-kappaB) transactivation. It acts as a transcriptional coactivator of nuclear receptors and regulates the transrepression between nuclear receptors and either AP-1 or NF-kappaB in vivo. Members in this family all contain a CUE domain.</td></tr>
<tr><td>47</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09820">pfam09820</a></td><td>279</td><td>63</td><td>28.8</td><td>64</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------&nbsp;]</td><td>AAA-ATPase_like</td><td>Predicted AAA-ATPase. This family contains many hypothetical bacterial proteins. This family was previously the N-terminal part of the Pfam DUF1703 (pfam08011) family before it was split into two. This region is predicted to be an AAA-ATPase domain.</td></tr>
<tr><td>48</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14276">pfam14276</a></td><td>121</td><td>47</td><td>27.9</td><td>1.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;]</td><td>DUF4363</td><td>Domain of unknown function (DUF4363). This family of proteins is found in bacteria. Proteins in this family are approximately 120 amino acids in length.</td></tr>
<tr><td>49</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13428">pfam13428</a></td><td>44</td><td>19</td><td>27.9</td><td>55</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;]</td><td>TPR_14</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>50</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2390">COG2390</a></td><td>321</td><td>44</td><td>27.9</td><td>1.3E+02</td><td>[&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DeoR</td><td>DNA-binding transcriptional regulator LsrR, DeoR family </td></tr>
<tr><td>51</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14325">cd14325</a></td><td>55</td><td>37</td><td>27.2</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;]</td><td>UBA_RNF31</td><td>UBA domain found in E3 ubiquitin-protein ligase RING finger protein 31 and similar proteins. RNF31, also called HOIL-1-interacting protein (HOIP), or zinc in-between-RING-finger ubiquitin-associated domain protein, together with HOIL-1 and SHARPIN, forms the E3-ligase complex (also known as linear-ubiquitin-chain assembly complex LUBAC) that regulates NF-kappaB activity and apoptosis. RNF31 contains a central ubiquitin-associated (UBA) domain that is responsible for the interaction with the N-terminal ubiquitin-like domain (UBL) of HOIL-1L. In addition, RNF31 can interact with the atypical mammalian orphan receptor DAX-1, trigger DAX-1 ubiquitination and stabilization, and participate in repressing steroidogenic gene expression.</td></tr>
<tr><td>52</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09388">pfam09388</a></td><td>45</td><td>25</td><td>27.0</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SpoOE-like</td><td>Spo0E like sporulation regulatory protein. Spore formation is an extreme response to starvation and can also be a component of disease transmission. Sporulation is controlled by an expanded two-component system where starvation signals result in sensor kinase activation and phosphorylation of the master sporulation response regulator Spo0A. Phosphatases such as Spo0E dephosphorylate Spo0A thereby inhibiting sporulation. This is a family of Spo0E-like phosphatases. The structure of a Bacillus anthracis member of this family has revealed an anti-parallel alpha-helical structure.</td></tr>
<tr><td>53</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08900">pfam08900</a></td><td>217</td><td>34</td><td>26.8</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1845</td><td>Domain of unknown function (DUF1845). This family of proteins are functionally uncharacterized.</td></tr>
<tr><td>54</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08241">pfam08241</a></td><td>95</td><td>16</td><td>26.8</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;]</td><td>Methyltransf_11</td><td>Methyltransferase domain. Members of this family are SAM dependent methyltransferases.</td></tr>
<tr><td>55</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12939">cd12939</a></td><td>43</td><td>40</td><td>26.7</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LEM_emerin</td><td>LEM (Lap2/Emerin/Man1) domain found in emerin. This CD corresponds to the LEM domain that is critical for binding to lamin A/C and is also involved in interaction with the DNA binding protein barrier-to-autointegration factor (BAF). Emerin is an inner nuclear membrane protein that occurs in four differently phosphorylated forms and plays a role in cell cycle-dependent events. It is absent from the inner nuclear membrane in most patients with X-linked muscular dystrophy. Emerin interacts with A-type and B-type lamins. It contains an N-terminal LEM domain followed by a poly-serine segment, a region rich in hydrophobic amino acids comprising the nuclear localization signal (NLS) followed by another poly-serine segment, and a C-terminal transmembrane region.</td></tr>
<tr><td>56</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03118">pfam03118</a></td><td>59</td><td>30</td><td>26.6</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNA_pol_A_CTD</td><td>Bacterial RNA polymerase, alpha chain C terminal domain. The alpha subunit of RNA polymerase consists of two independently folded domains, referred to as amino-terminal and carboxyl terminal domains. The amino terminal domain is involved in the interaction with the other subunits of the RNA polymerase. The carboxyl-terminal domain interacts with the DNA and activators. The amino acid sequence of the alpha subunit is conserved in prokaryotic and chloroplast RNA polymerases. There are three regions of particularly strong conservation, two in the amino-terminal and one in the carboxyl- terminal.</td></tr>
<tr><td>57</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03780">pfam03780</a></td><td>106</td><td>25</td><td>26.4</td><td>99</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Asp23</td><td>Asp23 family. The alkaline shock protein Asp23 was identified as an alkaline shock protein that was expressed in a sigmaB-dependent manner in Staphylococcus aureus.</td></tr>
<tr><td>58</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07452">PRK07452</a></td><td>326</td><td>62</td><td>26.3</td><td>1E+02</td><td>[&nbsp;--------------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07452</td><td>DNA polymerase III subunit delta; Validated</td></tr>
<tr><td>59</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06170">cd06170</a></td><td>57</td><td>28</td><td>26.3</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LuxR_C_like</td><td>C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many are active as homodimers. Many are two domain proteins in which the DNA binding property of the C-terminal DNA binding domain is modulated by modifications of the N-terminal domain.  For example in the case of Lux R which participates in the regulation of gene expression in response to fluctuations in cell-population density (quorum-sensing), a signaling molecule, the pheromone Acyl HSL (N-acyl derivatives of homoserine lactone), binds to the N-terminal domain and leads to LuxR dimerization.  For others phophorylation of the N-terminal domain leads to multimerization, for example Escherichia coli NarL and Sinorhizobium melilot FixJ. NarL controls gene expression of many respiratory-related operons when environmental nitrate or nitrite is present under anerobic conditions. FixJ is involved in the transcriptional activation of nitrogen fixation genes. The group also includes small proteins which lack an N-terminal signaling domain, such as Bacillus subtilis GerE.  GerE is dimeric and acts in conjunction with sigmaK as an activator or a repressor modulating the expression of various genes in particular those encoding the spore-coat. These LuxR family regulators may share a similar organization of their target binding sites. For example the LuxR dimer binds the lux box, a 20bp inverted repeat, GerE dimers bind two 12bp consensus sequences in inverted orientation having the central four bases overlap, and the NarL dimer binds two 7bp inverted repeats separated by 2 bp.</td></tr>
<tr><td>60</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08776">pfam08776</a></td><td>40</td><td>21</td><td>26.1</td><td>79</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;]</td><td>VASP_tetra</td><td>VASP tetramerization domain. Vasodilator-stimulated phosphoprotein (VASP) is an actin cytoskeletal regulatory protein. This region corresponds to the tetramerization domain which forms a right handed alpha helical coiled coil structure.</td></tr>
<tr><td>61</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07361">pfam07361</a></td><td>102</td><td>25</td><td>25.8</td><td>79</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cytochrom_B562</td><td>Cytochrome b562. This family contains the bacterial cytochrome b562. This forms a four-helix bundle that non-covalently binds a single heme prosthetic group..</td></tr>
<tr><td>62</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02572">pfam02572</a></td><td>172</td><td>52</td><td>25.4</td><td>53</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CobA_CobO_BtuR</td><td>ATP:corrinoid adenosyltransferase BtuR/CobO/CobP. This family consists of the BtuR, CobO, CobP proteins all of which are Cob(I)alamin adenosyltransferase, EC:2.5.1.17, involved in cobalamin (vitamin B12) biosynthesis. These enzymes catalyse the adenosylation reaction: ATP + cob(I)alamin + H2O <=> phosphate + diphosphate + adenosylcobalamin.</td></tr>
<tr><td>63</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08775">pfam08775</a></td><td>127</td><td>51</td><td>25.4</td><td>2.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------------------&nbsp;&nbsp;&nbsp;]</td><td>ParB</td><td>ParB family. ParB is a component of the par system which mediates accurate DNA partition during cell division. It recognizes A-box and B-box DNA motifs. ParB forms an asymmetric dimer with 2 extended helix-turn-helix (HTH) motifs that bind to A-boxes. The HTH motifs emanate from a beta sheet coiled coil DNA binding module. Both DNA binding elements are free to rotate around a flexible linker, this enables them to bind to complex arrays of A- and B-box elements on adjacent DNA arms of the looped partition site.</td></tr>
<tr><td>64</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01221">PRK01221</a></td><td>312</td><td>16</td><td>25.1</td><td>40</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01221</td><td>putative deoxyhypusine synthase; Provisional</td></tr>
<tr><td>65</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK15180">PRK15180</a></td><td>831</td><td>44</td><td>24.9</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;]</td><td>PRK15180</td><td>Vi polysaccharide biosynthesis protein TviD; Provisional</td></tr>
<tr><td>66</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK09958">PRK09958</a></td><td>204</td><td>24</td><td>24.5</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK09958</td><td>DNA-binding transcriptional activator EvgA; Provisional</td></tr>
<tr><td>67</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14718">pfam14718</a></td><td>70</td><td>32</td><td>24.4</td><td>83</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;]</td><td>SLT_L</td><td>Soluble lytic murein transglycosylase L domain. Soluble lytic murein transglycosylase (SLT) consists of three domains, an N-terminal U domain, an L domain (linker domain) and a C-terminal domain (C). The L domain may be involved in the interaction of the enzyme with peptidoglycan.</td></tr>
<tr><td>68</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04601">cd04601</a></td><td>110</td><td>57</td><td>24.3</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CBS_pair_IMPDH</td><td>This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domains in the inosine 5' monophosphate dehydrogenase (IMPDH) protein.  IMPDH is an essential enzyme that catalyzes the first step unique to GTP synthesis, playing a key role in the regulation of cell proliferation and differentiation. CBS is a small domain originally identified in cystathionine beta-synthase and subsequently found in a wide range of different proteins. CBS domains usually come in tandem repeats, which associate to form a so-called Bateman domain or a CBS pair which is reflected in this model. The interface between the two CBS domains forms a cleft that is a potential ligand binding site. The CBS pair coexists with a variety of other functional domains. It has been proposed that the CBS domain may play a regulatory role, although its exact function is unknown. Mutations of conserved residues within this domain in IMPDH have been associated with retinitis pigmentosa.</td></tr>
<tr><td>69</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01381">pfam01381</a></td><td>55</td><td>38</td><td>24.2</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_3</td><td>Helix-turn-helix. This large family of DNA binding helix-turn helix proteins includes Cro and CI. Within the Neisseria gonorrhoeae phage associated protein NGO0477, the full protein fold incorporates a helix-turn-helix motif, but the function of this member is unlikely to be that of a DNA-binding regulator, the function of most other members, so is not necessarily characteristic of the whole family.</td></tr>
<tr><td>70</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07729">pfam07729</a></td><td>125</td><td>23</td><td>24.1</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FCD</td><td>FCD domain. This domain is the C-terminal ligand binding domain of many members of the GntR family. This domain probably binds to a range of effector molecules that regulate the transcription of genes through the action of the N-terminal DNA-binding domain pfam00392. This domain is found in E. coli nanR and dgoR that are regulators of sugar biosynthesis operons. It is also in the known structure of FadR where it binds to acyl-coA, the domain is alpha helical. This family has been named as FCD for (FadR C-terminal Domain).</td></tr>
<tr><td>71</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04652">pfam04652</a></td><td>312</td><td>22</td><td>23.9</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF605</td><td>Vta1 like. Vta1 (VPS20-associated protein 1) is a positive regulator of Vps4. Vps4 is an ATPase that is required in the multivesicular body (MVB) sorting pathway to dissociate the endosomal sorting complex required for transport (ESCRT). Vta1 promotes correct assembly of Vps4 and stimulates its ATPase activity through its conserved Vta1/SBP1/LIP5 region.</td></tr>
<tr><td>72</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00298">TIGR00298</a></td><td>216</td><td>32</td><td>23.8</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR00298</td><td>2-phosphosulfolactate phosphatase. 2-phosphosulfolactate phosphatase catalyzes the sulfonation of phosphoenolpyruvate to form 2-phospho-3-sulfolactate, the second step in coenzyme M biosynthesis. Coenzyme M is the terminal methyl carrier in methanogenesis. </td></tr>
<tr><td>73</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0634">COG0634</a></td><td>178</td><td>23</td><td>23.6</td><td>99</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------]</td><td>HptA</td><td>Hypoxanthine-guanine phosphoribosyltransferase </td></tr>
<tr><td>74</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00482">pfam00482</a></td><td>125</td><td>13</td><td>22.4</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>T2SF</td><td>Type II secretion system (T2SS), protein F. The original family covered both the regions found by the current model. The splitting of the family has allowed the related FlaJ_arch (archaeal FlaJ family) to be merged with it. Proteins with this domain in form a platform for the machiney of the Type II secretion system, as well as the Type 4 pili and the archaeal flagella. This domain seems to show some similarity to PF00664 but this may just be due to similarities in the TM helices (personal obs: C Yeats).</td></tr>
<tr><td>75</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13384">pfam13384</a></td><td>50</td><td>37</td><td>22.4</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_23</td><td>Homeodomain-like domain. </td></tr>
<tr><td>76</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd15479">cd15479</a></td><td>329</td><td>72</td><td>21.9</td><td>93</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------------------&nbsp;&nbsp;&nbsp;]</td><td>fMyo4p_CBD</td><td>cargo binding domain of fungal myosin 4. Yeast myosin V travels along actin cables, actin filaments that are bundled by fimbrin, in the presence of tropomyosin. This is in contrast to the other vertebrate class V  myosins. Like other class V myosins, fungal myosin 2 and 4 contain a C-terminal cargo binding domain. In case of Myo4 it has been shown to bind to the adapter protein She3p (Swi5p-dependent HO expression 3), which in turn anchors myosin 4 to its cargos, zip-coded mRNP (messenger ribonucleoprotein particles) and tER (tubular endoplasmic reticulum).</td></tr>
<tr><td>77</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam08064">pfam08064</a></td><td>107</td><td>21</td><td>21.7</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UME</td><td>UME (NUC010) domain. This domain is characteristic of UVSB PI-3 kinase, MEI-41 and ESR1.</td></tr>
<tr><td>78</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11528">cd11528</a></td><td>114</td><td>37</td><td>21.4</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;]</td><td>NTP-PPase_MazG_Nterm</td><td>Nucleoside Triphosphate Pyrophosphohydrolase (EC 3.6.1.8) N-terminal tandem-domain of MazG proteins from Escherichia coli and bacterial homologs. MazG is a NTP-PPase that hydrolyzes all canonical NTPs into their corresponding nucleoside monophosphates and pyrophosphate. The prototype of this family is MazG proteins from Escherichia coli (EcMazG) that represents the most abundant form consisting two sequence-related domains in tandem, this family corresponding to the N-terminal MazG-like domain. EcMazG functions as a regulator of cellular response to starvation by lowering the cellular concentration of guanosine 3',5'-bispyrophosphate (ppGpp). EcMazG exists as a dimer; each monomer contains two tandem MazG-like domains with similarly folded globular structures. However, only the C-terminal domain has well-ordered active site and exhibits an NTPase activity responsible for the regulation of bacterial cell survival under nutritional stress. Divalent ions, such as Mg2+ or Mn2+, are required for activity; however, this domain does not exhibit an NTPase activity despite containing structural features such as the EEXX(E/D) motif and key basic catalytic residues responsible for nucleotide pyrophosphohydrolysis activity. It is suggested that the N-terminal domain of EcMazG might have a house-cleaning function by hydrolyzing noncanonical NTPs whose incorporation into the nascent DNA leads to increased mutagenesis and DNA damage.</td></tr>
<tr><td>79</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14316">cd14316</a></td><td>37</td><td>25</td><td>21.2</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UBA2_UBAP1_like</td><td>UBA2 domain found in ubiquitin-associated protein 1 (UBAP-1) and similar proteins. UBAP-1, also called nasopharyngeal carcinoma-associated gene 20 protein, is a ubiquitously expressed protein that may play an important role in the ubiquitin pathway and cell progression. It co-localizes with TDP-43 proteins in neuronal cytoplasmic inclusions and acts as a genetic risk factor for frontotemporal lobar degeneration (FTLD). Moreover, UBAP-1, together with VPS37A, forms an endosome-specific endosomal sorting complexes I required for transport (ESCRT-I) complex that displays a restricted cellular function, ubiquitin-dependent endosomal sorting and multivesicular body (MVB) biogenesis. UBAP-1 contains an N-terminal UBAP-1-MVB12-associated (UMA) domain, and two tandem ubiquitin-associated (UBA) domains that may be responsible for the binding of ubiquitin-conjugating enzymes. This model corresponds to UBA2 domain.</td></tr>
<tr><td>80</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07553">pfam07553</a></td><td>49</td><td>22</td><td>21.2</td><td>58</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;]</td><td>Lipoprotein_Ltp</td><td>Host cell surface-exposed lipoprotein. This is a family of lipoproteins that is involved in superinfection exclusion. Proteins in this family have been shown to act at the stage of DNA release from the phage head into the cell.</td></tr>
<tr><td>81</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13432">pfam13432</a></td><td>65</td><td>23</td><td>21.2</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;]</td><td>TPR_16</td><td>Tetratricopeptide repeat. </td></tr>
<tr><td>82</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00583">cd00583</a></td><td>210</td><td>26</td><td>21.1</td><td>1E+02</td><td>[&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MutH_Sau3AI</td><td>MutH is a 28kD endonuclease involved in methyl-directed DNA mismatch repair in gram negative bacteria. MutH is both sequence-specific and methylation-specific, introducing a nick in the unmethylated strand of a hemi-methylated d(GATC) DNA duplex.   MutH is homologous to the type II restriction endonuclease Sau3AI which also recognizes the d(GATC) sequence however, Sau3AI cleaves both strands regardless of their methylation state. The active form of MutH is monomeric while that of Sau3AI is homodimeric. In addition to MutH, MutS, involved in mismatch recognition, and MutL, involved in mediating the interactions between MutH and MutS, are essential  in initiating mismatch repair in Escherichia coli.</td></tr>
<tr><td>83</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13709">cd13709</a></td><td>227</td><td>15</td><td>21.0</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_YxeM</td><td>Substrate binding domain of an ABC transporter YxeMNO; the type 2 periplasmic binding protein fold. This group contains cystine-binding domain (YxeM) of a periplasmic receptor-dependent ATP-binding cassette transporter and its closely related proteins. Cystine is an oxidized dimeric form of cysteine that is required for optimal bacterial growth. In Bacillus subtilis, three ABC transporters, TcyJKLMN (YtmJKLMN), TcyABC (YckKJI), and YxeMNO are involved in uptake of cystine. Likewise, three uptake systems were identified in Salmonella enterica serovar Typhimurium, while in Escherichia coli, two transport systems seem to be involved in cystine uptake. Moreover, L-cystine limitation was shown to prevent virulence of Neisseria gonorrhoeae; thus, its L-cystine solute receptor (Ngo0372) may be suited as target for an antimicrobial vaccine. The cystine receptor belongs to the type 2 periplasmic binding fold protein superfamily (PBP2). The PBP2 proteins are typically comprised of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two receptor cytoplasmically-located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>84</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam05205">pfam05205</a></td><td>106</td><td>33</td><td>20.9</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;]</td><td>COMPASS-Shg1</td><td>COMPASS (Complex proteins associated with Set1p) component shg1. The Shg1 subunit is one of the eight subunits of the COMPASS complex, complex associated with SET1, conserved in yeasts and in other eukaryotes up to humans. It is associated with the region of the Set1 protein that is N-terminal to the C-terminus, ie Set1-560-900. The function of Shg1 seems to be to slightly inhibit histone 3 lysine 4 (H3K4) di- and tri-methylation, and it is a pioneer protein. The COMPASS complex functions to methylate the fourth lysine of Histone 3 and for silencing of genes close to the telomeres of chromosomes.</td></tr>
<tr><td>85</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11365">pfam11365</a></td><td>97</td><td>17</td><td>20.6</td><td>99</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------]</td><td>DUF3166</td><td>Protein of unknown function (DUF3166). This eukaryotic family of proteins has no known function.</td></tr>
<tr><td>86</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4977">COG4977</a></td><td>328</td><td>46</td><td>20.5</td><td>2.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GlxA</td><td>Transcriptional regulator GlxA family, contains an amidase domain and an AraC-type DNA-binding HTH domain  </td></tr>
<tr><td>87</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14275">cd14275</a></td><td>37</td><td>32</td><td>20.4</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UBA_EF-Ts</td><td>UBA domain found in elongation factor Ts (EF-Ts) from bacteria, chloroplasts and mitochondria of eukaryotes. EF-Ts functions as a nucleotide exchange factor in the functional cycle of EF-Tu, another translation elongation factor that facilitates the binding of aminoacylated transfer RNAs (aminoacyl-tRNA) to the ribosomal A site as a ternary complex with guanosine triphosphate during the elongation cycle of protein biosynthesis, and then catalyzes the hydrolysis of GTP and release itself in GDP-bound form. EF-Ts forms complex with EF-Tu and catalyzes the nucleotide exchange reaction promoting the formation of EF-Tu in GTP-bound form from EF-Tu in GDP-bound form. EF-Ts from Thermus thermophiles is shorter than EF-Ts from Escherichia coli, but it has higher thermostability. The mitochondrial translational EF-Ts from chloroplasts and mitochondria display high similarity to the bacterial EF-Ts. The majority of family members contain one ubiquitin-associated (UBA) domain, but some family members from plants harbor two tandem UBA domains.</td></tr>
<tr><td>88</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13626">cd13626</a></td><td>219</td><td>13</td><td>20.3</td><td>74</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_Cystine_like</td><td>Substrate binding domain of cystine ABC transporters; the type 2 periplasmic binding protein fold. Cystine-binding domain of periplasmic receptor-dependent ATP-binding cassette (ABC) transporters.  Cystine is an oxidized dimeric form of cysteine that is required for optimal bacterial growth. In Bacillus subtilis, three ABC transporters, TcyJKLMN (YtmJKLMN), TcyABC (YckKJI), and YxeMNO are involved in uptake of cystine. Also, three uptake systems were identified in Salmonella enterica serovar Typhimurium, while in Escherichia coli, two transport systems seem to be involved in cystine uptake. Moreover, L-cystine limitation was shown to prevent virulence of Neisseria gonorrhoeae; thus, its L-cystine solute receptor (Ngo0372) may be suited as target for an antimicrobial vaccine. The cystine receptor belongs to the type 2 periplasmic binding fold protein superfamily (PBP2). The PBP2 proteins are typically comprised of two globular subdomains connected by a flexible hinge and bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two receptor cytoplasmically-located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>89</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05182">PRK05182</a></td><td>310</td><td>44</td><td>20.1</td><td>77</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------&nbsp;&nbsp;]</td><td>PRK05182</td><td>DNA-directed RNA polymerase subunit alpha; Provisional</td></tr>
<tr><td>90</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14338">cd14338</a></td><td>45</td><td>18</td><td>20.1</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UBA_SIK</td><td>UBA domain found in salt-inducible kinase SIK1, SIK2, SIK3 and similar proteins. Salt-inducible kinase SIK1, SIK2, SIK3 are serine/threonine kinases that belong to the AMP-activated protein kinases (AMPK) family involved in the regulation of metabolism during energy stress. SIK1, also called serine/threonine-protein kinase SNF1-like kinase 1 (SNF1LK), is required for myogenic differentiation. It is degraded by the proteasome in myoblasts which is regulated by cAMP signaling. Moreover, SIK1 acts as a class II histone deacetylase (HDAC) kinase, triggering the cytoplasmic export of the HDACs and activation of myocyte enhancer factor 2 (MEF2)-dependent transcription. It also regulates transcription through inhibitory phosphorylation of a family of cAMP responsive element binding protein (CREB) coactivators, called TORCs/CRTCs. In addition, SIK1 links LKB1 to p53-dependent anoikis and suppresses metastasis. It is also involved in a cell sodium-sensing network that regulates active sodium transport through a calcium-dependent process. SIK2, also called Qin-induced kinase or serine/threonine-protein kinase SNF1-like kinase 2 (SNF1LK2), plays an important role in the insulin-signaling pathway during adipocyte differentiation, as well as in autophagy progression. Moreover, SIK2 plays a critical role in neuronal survival and modulates cAMP responsive element binding protein (CREB)-mediated gene expression in response to hormones and nutrients. SIK2 acts as a critical determinant in autophagy progression. In addition, SIK2 localizes at the centrosome and functions as a centrosome kinase required for bipolar mitotic spindle formation. It is involved in the initiation of mitosis, and regulates the localization of the centrosome linker protein, C-Nap1, through S2392 phosphorylation. SIK3, also called salt-inducible kinase 3 or serine/threonine-protein kinase QSK, acts as a novel energy regulator that modulates cholesterol and bile acid metabolism by coupling with retinoid metabolism. It also play an essential role in facilitating chondrocyte hypertrophy during skeletogenesis and growth plate maintenance. Members in this family contain an N-terminal protein kinase catalytic domain followed by an ubiquitin-associated (UBA) domain. </td></tr>
</table>
</body>
