<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13949">pfam13949</a></td><td>292</td><td>107</td><td>94.6</td><td>3.2</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ALIX_LYPXL_bnd</td><td>ALIX V-shaped domain binding to HIV. The binding of the LYPxL motif of late HIV p6Gag and EIAV p9Gag to this domain is necessary for viral budding.This domain is generally central between an N-terminal Bro1 domain, pfam03097 and a C-terminal proline-rich domain. The retroviruses thus used this domain to hijack the ESCRT system of the cell.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00996">TIGR00996</a></td><td>291</td><td>143</td><td>89.9</td><td>6</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Mtu_fam_mce</td><td>virulence factor Mce family protein. Members of this paralogous family are found as six tandem homologous proteins in the same orientation per cassette, in four separate cassettes in Mycobacterium tuberculosis. The six members of each cassette represent six subfamilies. One subfamily includes the protein mce (mycobacterial cell entry), a virulence protein required for invasion of non-phagocytic cells. </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam13514">pfam13514</a></td><td>1112</td><td>223</td><td>82.9</td><td>71</td><td>[&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AAA_27</td><td>AAA domain. This domain is found in a number of double-strand DNA break proteins. This domain contains a P-loop motif.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02168">TIGR02168</a></td><td>1179</td><td>20</td><td>77.2</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SMC_prok_B</td><td>chromosome segregation protein SMC, common bacterial type. SMC (structural maintenance of chromosomes) proteins bind DNA and act in organizing and segregating chromosomes for partition. SMC proteins are found in bacteria, archaea, and eukaryotes. This family represents the SMC protein of most bacteria. The smc gene is often associated with scpB (TIGR00281) and scpA genes, where scp stands for segregation and condensation protein. SMC was shown (in Caulobacter crescentus) to be induced early in S phase but present and bound to DNA throughout the cell cycle. </td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1463">COG1463</a></td><td>359</td><td>53</td><td>76.2</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MlaD</td><td>ABC-type transporter Mla maintaining outer membrane lipid asymmetry, periplasmic component MlaD </td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0497">COG0497</a></td><td>557</td><td>65</td><td>71.8</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RecN</td><td>DNA repair ATPase RecN </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04740">pfam04740</a></td><td>204</td><td>125</td><td>62.8</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>LXG</td><td>LXG domain of WXG superfamily. This domain is present is the N-terminal region of a group of polymorphic toxin proteins in bacteria. It is predicted to use Type VII secretion pathway to mediate export of bacterial toxins.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09304">pfam09304</a></td><td>95</td><td>59</td><td>62.7</td><td>33</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Cortex-I_coil</td><td>Cortexillin I, coiled coil. Members of this family are predominantly found in the actin-bundling protein Cortexillin I from Dictyostelium discoideum. They adopt a structure consisting of an 18-heptad-repeat alpha-helical coiled-coil, and are a prerequisite for the assembly of Cortexillin I.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02791">TIGR02791</a></td><td>220</td><td>18</td><td>57.9</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>VirB5</td><td>P-type DNA transfer protein VirB5. The VirB5 protein is involved in the type IV DNA secretion systems typified by the Agrobacterium Ti plasmid vir system where it interacts with several other proteins essential for proper pilus formation. VirB5 is homologous to the IncN (N-type) conjugation system protein TraC as well as the P-type protein TrbJ and the F-type protein TraE.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12128">pfam12128</a></td><td>1198</td><td>43</td><td>57.5</td><td>3.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3584</td><td>Protein of unknown function (DUF3584). This protein is found in bacteria and eukaryotes. Proteins in this family are typically between 943 to 1234 amino acids in length. This family contains a P-loop motif suggesting it is a nucleotide binding protein. It may be involved in replication.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10498">pfam10498</a></td><td>355</td><td>34</td><td>54.0</td><td>2.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>IFT57</td><td>Intra-flagellar transport protein 57. Eukaryotic cilia and flagella are specialized organelles found at the periphery of cells of diverse organisms. Intra-flagellar transport (IFT) is required for the assembly and maintenance of eukaryotic cilia and flagella, and consists of the bidirectional movement of large protein particles between the base and the distal tip of the organelle. IFT particles contain multiple copies of two distinct protein complexes, A and B, which contain at least 6 and 11 protein subunits. IFT57 is part of complex B but is not, however, required for the core subunits to stay associated. This protein is known as Huntington-interacting protein-1 in humans.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04211">TIGR04211</a></td><td>198</td><td>58</td><td>50.5</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH3_and_anchor</td><td>SH3 domain protein. Members of this protein family have a signal peptide, a strongly conserved SH3 domain, a variable region, and then a C-terminal hydrophobic transmembrane alpha helix region.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03007">TIGR03007</a></td><td>498</td><td>119</td><td>49.6</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>pepcterm_ChnLen</td><td>polysaccharide chain length determinant protein, PEP-CTERM locus subfamily. Members of this protein family belong to the family of polysaccharide chain length determinant proteins (pfam02706). All are found in species that encode the PEP-CTERM/exosortase system predicted to act in protein sorting in a number of Gram-negative bacteria, and are found near the epsH homolog that is the putative exosortase gene. </td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01024">PRK01024</a></td><td>503</td><td>55</td><td>47.7</td><td>7.3</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01024</td><td>Na(+)-translocating NADH-quinone reductase subunit B; Provisional</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03836">pfam03836</a></td><td>139</td><td>30</td><td>47.3</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RasGAP_C</td><td>RasGAP C-terminus. </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04211">TIGR04211</a></td><td>198</td><td>60</td><td>46.9</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH3_and_anchor</td><td>SH3 domain protein. Members of this protein family have a signal peptide, a strongly conserved SH3 domain, a variable region, and then a C-terminal hydrophobic transmembrane alpha helix region.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00606">TIGR00606</a></td><td>1311</td><td>89</td><td>45.7</td><td>3.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>rad50</td><td>rad50. All proteins in this family for which functions are known are involvedin recombination, recombinational repair, and/or non-homologous end joining.They are components of an exonuclease complex with MRE11 homologs. This family is distantly related to the SbcC family of bacterial proteins.This family is based on the phylogenomic analysis of JA Eisen (1999, Ph.D. Thesis, Stanford University).</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04711">pfam04711</a></td><td>76</td><td>58</td><td>44.8</td><td>42</td><td>[&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ApoA-II</td><td>Apolipoprotein A-II (ApoA-II). Apolipoprotein A-II (ApoA-II) is the second major apolipoprotein of high density lipoprotein in human plasma.  Mature ApoA-II is present as a dimer of two 77-amino acid chains joined by a disulphide bridge. ApoA-II regulates many steps in HDL metabolizm, and its role in coronary heart disease is unclear. In bovine serum, the ApoA-II homologue is present in almost free form.   Bovine ApoA-II shows antimicrobial activity against Escherichia coli and yeasts in phosphate buffered saline (PBS).</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04102">pfam04102</a></td><td>68</td><td>51</td><td>40.7</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SlyX</td><td>SlyX. The SlyX protein has no known function. It is short less than 80 amino acids and is found close to the slyD gene. The SlyX protein has a conserved PPH(Y/W) motif at its C-terminus. The protein may be a coiled-coil structure.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11637">PRK11637</a></td><td>428</td><td>45</td><td>39.8</td><td>4.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK11637</td><td>AmiB activator; Provisional</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14003">pfam14003</a></td><td>65</td><td>32</td><td>38.1</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YlbE</td><td>YlbE-like protein. The YlbE-like protein family includes the B. subtilis protein YlbE, which is functionally uncharacterized. This family of cytosolic proteins is found in bacteria. Proteins in this family are approximately 80 amino acids in length. There is a conserved WYR sequence motif.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd06616">cd06616</a></td><td>291</td><td>69</td><td>36.8</td><td>25</td><td>[&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PKc_MKK4</td><td>Catalytic domain of the dual-specificity Protein Kinase, Mitogen-activated protein Kinase Kinase 4. PKs catalyze the transfer of the gamma-phosphoryl group from ATP to serine/threonine (ST) or tyrosine residues on protein substrates. MKK4 is a dual-specificity PK that phosphorylates and activates the downstream targets, c-Jun N-terminal kinase (JNK) and p38 MAPK, on specific threonine and tyrosine residues. JNK and p38 are collectively known as stress-activated MAPKs, as they are activated in response to a variety of environmental stresses and pro-inflammatory cytokines. Their activation is associated with the induction of cell death. Mice deficient in MKK4 die during embryogenesis and display anemia, severe liver hemorrhage, and abnormal hepatogenesis. MKK4 may also play roles in the immune system and in cardiac hypertrophy. It plays a major role in cancer as a tumor and metastasis suppressor. Under certain conditions, MKK4 is pro-oncogenic. The MKK4 subfamily is part of a larger superfamily that includes the catalytic domains of other STKs, protein tyrosine kinases, RIO kinases, aminoglycoside phosphotransferase, choline kinase, and phosphoinositide 3-kinase.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02449">TIGR02449</a></td><td>65</td><td>54</td><td>35.8</td><td>2.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TIGR02449</td><td>TIGR02449 family protein. Members of this family are small proteins, typically 73 amino acids in length, with single copies in each of several Proteobacteria, including Xylella fastidiosa, Pseudomonas aeruginosa, and Xanthomonas campestris. The function is unknown.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09375">pfam09375</a></td><td>276</td><td>111</td><td>34.4</td><td>3.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Peptidase_M75</td><td>Imelysin. The imelysin peptidase was first identified in Pseudomonas aeruginosa. The active site residues have not been identified. However, His201 and Glu204 are completely conserved in the family and occur in an HXXE motif that is also found in family M14.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12938">pfam12938</a></td><td>238</td><td>55</td><td>34.3</td><td>81</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;]</td><td>M_domain</td><td>M domain of GW182. </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02168">TIGR02168</a></td><td>1179</td><td>44</td><td>32.1</td><td>7.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SMC_prok_B</td><td>chromosome segregation protein SMC, common bacterial type. SMC (structural maintenance of chromosomes) proteins bind DNA and act in organizing and segregating chromosomes for partition. SMC proteins are found in bacteria, archaea, and eukaryotes. This family represents the SMC protein of most bacteria. The smc gene is often associated with scpB (TIGR00281) and scpA genes, where scp stands for segregation and condensation protein. SMC was shown (in Caulobacter crescentus) to be induced early in S phase but present and bound to DNA throughout the cell cycle. </td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11887">pfam11887</a></td><td>270</td><td>60</td><td>30.1</td><td>3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3407</td><td>Protein of unknown function (DUF3407). This family of proteins are functionally uncharacterized. This protein is found in bacteria. Proteins in this family are typically between 360 to 454 amino acids in length. This protein is found associated with pfam02470.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07428">PRK07428</a></td><td>288</td><td>60</td><td>30.1</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07428</td><td>nicotinate-nucleotide pyrophosphorylase; Provisional</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12128">pfam12128</a></td><td>1198</td><td>47</td><td>28.8</td><td>9.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3584</td><td>Protein of unknown function (DUF3584). This protein is found in bacteria and eukaryotes. Proteins in this family are typically between 943 to 1234 amino acids in length. This family contains a P-loop motif suggesting it is a nucleotide binding protein. It may be involved in replication.</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07743">pfam07743</a></td><td>77</td><td>35</td><td>27.1</td><td>2.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HSCB_C</td><td>HSCB C-terminal oligomerization domain. This domain is the HSCB C-terminal oligomerization domain and is found on co-chaperone proteins.</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13386">PRK13386</a></td><td>236</td><td>71</td><td>25.1</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fliH</td><td>flagellar assembly protein H; Provisional</td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04880">pfam04880</a></td><td>166</td><td>44</td><td>24.4</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NUDE_C</td><td>NUDE protein, C-terminal conserved region. This family represents the C-terminal conserved region of the NUDE proteins. NUDE proteins are involved in nuclear migration.</td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00634">TIGR00634</a></td><td>563</td><td>43</td><td>23.7</td><td>5.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recN</td><td>DNA repair protein RecN. All proteins in this family for which functions are known are ATP binding proteins involved in the initiation of recombination and recombinational repair. </td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01843">TIGR01843</a></td><td>423</td><td>114</td><td>23.1</td><td>7.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>type_I_hlyD</td><td>type I secretion membrane fusion protein, HlyD family. Type I secretion is an ABC transport process that exports proteins, without cleavage of any signal sequence, from the cytosol to extracellular medium across both inner and outer membranes. The secretion signal is found in the C-terminus of the transported protein. This model represents the adaptor protein between the ATP-binding cassette (ABC) protein of the inner membrane and the outer membrane protein, and is called the membrane fusion protein. This model selects a subfamily closely related to HlyD; it is defined narrowly and excludes, for example, colicin V secretion protein CvaA and multidrug efflux proteins. </td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04899">pfam04899</a></td><td>70</td><td>40</td><td>22.1</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;]</td><td>MbeD_MobD</td><td>MbeD/MobD like. The MbeD and MobD proteins are plasmid encoded, and are involved in the plasmids mobilisation and transfer in the presence of conjugative plasmids.</td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02156">TIGR02156</a></td><td>289</td><td>49</td><td>22.0</td><td>65</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PA_CoA_Oxy1</td><td>phenylacetate-CoA oxygenase, PaaG subunit. Phenylacetate-CoA oxygenase is comprised of a five gene complex responsible for the hydroxylation of phenylacetate-CoA (PA-CoA) as the second catabolic step in phenylacetic acid (PA) degradation. Although the exact function of this enzyme has not been determined, it has been shown to be required for phenylacetic acid degradation and has been proposed to function in a multicomponent oxygenase acting on phenylacetate-CoA. </td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12226">pfam12226</a></td><td>230</td><td>40</td><td>21.9</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Astro_capsid_p</td><td>Turkey astrovirus capsid protein. This family of proteins is found in viruses. Proteins in this family are typically between 241 and 261 amino acids in length. These proteins are capsid proteins from various astrovirus strains.</td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07627">cd07627</a></td><td>216</td><td>61</td><td>21.8</td><td>6.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BAR_Vps5p</td><td>The Bin/Amphiphysin/Rvs (BAR) domain of yeast Sorting Nexin Vps5p. BAR domains are dimerization, lipid binding and curvature sensing modules found in many different proteins with diverse functions. Sorting nexins (SNXs) are Phox homology (PX) domain containing proteins that are involved in regulating membrane traffic and protein sorting in the endosomal system. SNXs differ from each other in their lipid-binding specificity, subcellular localization and specific function in the endocytic pathway. A subset of SNXs also contain BAR domains. The PX-BAR structural unit determines the specific membrane targeting of SNXs. Vsp5p is the yeast counterpart of human SNX1 and is part of the retromer complex, which functions in the endosome-to-Golgi retrieval of vacuolar protein sorting receptor Vps10p, the Golgi-resident membrane protein A-ALP, and endopeptidase Kex2. BAR domains form dimers that bind to membranes, induce membrane bending and curvature, and may also be involved in protein-protein interactions.</td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00634">TIGR00634</a></td><td>563</td><td>161</td><td>20.7</td><td>1E+03</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>recN</td><td>DNA repair protein RecN. All proteins in this family for which functions are known are ATP binding proteins involved in the initiation of recombination and recombinational repair. </td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14338">pfam14338</a></td><td>92</td><td>34</td><td>20.6</td><td>62</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Mrr_N</td><td>Mrr N-terminal domain. This domain is found at the N-terminus of the Mrr restriction endonuclease catalytic domain, pfam04471. Fold recognition analysis predicts that it is a diverged member of the winged helix variant of helix turn helix proteins. It may play a role in DNA sequence recognition.</td></tr>
</table>
</body>
