<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09766">cd09766</a></td><td>101</td><td>87</td><td>99.8</td><td>1.5E-19</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;]</td><td>Csx15_I-U</td><td>CRISPR/Cas system-associated protein Csx15. CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and associated Cas proteins comprise a system for heritable host defense by prokaryotic cells against phage and other foreign DNA; Small protein loosely associated with CRISPR/Cas systems; some are fused to AAA ATPase domain, also known as TTE2665 family</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK06725">PRK06725</a></td><td>570</td><td>72</td><td>63.0</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK06725</td><td>acetolactate synthase 3 catalytic subunit; Validated</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11567">cd11567</a></td><td>76</td><td>23</td><td>60.7</td><td>17</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YciH_like</td><td>Homologs of eIF1/SUI1 including Escherichia coli YciH. Members of the eIF1/SUI1 (eukaryotic initiation factor 1) family are found in eukaryotes, archaea, and some bacteria; eukaryotic members are understood to play an important role in accurate initiator codon recognition during translation initiation. The function of non-eukaryotic family members is unclear. Escherichia coli YciH is a non-essential protein and was reported to be able to perform some of the functions of IF3 in prokaryotic initiation.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08243">cd08243</a></td><td>320</td><td>62</td><td>56.5</td><td>32</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>quinone_oxidoreductase_like_1</td><td>Quinone oxidoreductase (QOR). NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.  The medium chain alcohol dehydrogenase family (MDR) have a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form. The N-terminal region typically has an all-beta catalytic domain. These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03334">cd03334</a></td><td>261</td><td>49</td><td>55.1</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Fab1_TCP</td><td>TCP-1 like domain of the eukaryotic phosphatidylinositol 3-phosphate (PtdIns3P) 5-kinase Fab1.  Fab1p is important for vacuole size regulation, presumably by modulating PtdIns(3,5)P2 effector activity. In the human homolog p235/PIKfyve deletion of this domain leads to loss of catalytic activity. However no exact function this domain has been defined. In general, chaperonins are involved in productive folding of proteins.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08273">cd08273</a></td><td>331</td><td>60</td><td>53.0</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;]</td><td>MDR8</td><td>Medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family. This group is a member of the medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, but lacks the zinc-binding sites of the zinc-dependent alcohol dehydrogenases. The medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, which contains the zinc-dependent alcohol dehydrogenase (ADH-Zn) and related proteins, is a diverse group of proteins related to the first identified member, class I mammalian ADH.  MDRs display a broad range of activities and are distinguished from the smaller short chain dehydrogenases (~ 250 amino acids vs. the ~ 350 amino acids of the MDR).  The MDR proteins have 2 domains: a C-terminal NAD(P)-binding Rossmann fold domain of a beta-alpha form and an N-terminal catalytic domain with distant homology to GroES.  The MDR group contains a host of activities, including the founding alcohol dehydrogenase (ADH), quinone reductase, sorbitol dehydrogenase, formaldehyde dehydrogenase, butanediol DH, ketose reductase, cinnamyl reductase, and numerous others. The zinc-dependent alcohol dehydrogenases (ADHs) catalyze the  NAD(P)(H)-dependent interconversion of alcohols to aldehydes or ketones.  Active site zinc has a catalytic role, while structural zinc aids in stability.  ADH-like proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and generally have 2 tightly bound zinc atoms per subunit. The active site zinc is coordinated by a histidine, two cysteines, and a water molecule. The second zinc seems to play a structural role, affects subunit interactions, and is typically coordinated by 4 cysteines.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10443">pfam10443</a></td><td>428</td><td>22</td><td>51.7</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RNA12</td><td>RNA12 protein. This family includes RNA12 from S. cerevisiae. That protein contains an RRM domain. This region is C-terminal to that and includes a P-loop motif suggesting this region binds to NTP. The RNA12 proteins is involved in pre-rRNA maturation.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03156">TIGR03156</a></td><td>351</td><td>21</td><td>49.9</td><td>8.9</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GTP_HflX</td><td>GTP-binding protein HflX. This protein family is one of a number of homologous small, well-conserved GTP-binding proteins with pleiotropic effects. Bacterial members are designated HflX, following the naming convention in Escherichia coli where HflX is encoded immediately downstream of the RNA chaperone Hfq, and immediately upstream of HflKC, a membrane-associated protease pair with an important housekeeping function. Over large numbers of other bacterial genomes, the pairing with hfq is more significant than with hflK and hlfC. The gene from Homo sapiens in this family has been named PGPL (pseudoautosomal GTP-binding protein-like). </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03411">cd03411</a></td><td>159</td><td>30</td><td>46.4</td><td>45</td><td>[&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ferrochelatase_N</td><td>Ferrochelatase, N-terminal domain: Ferrochelatase (protoheme ferrolyase or HemH) is the terminal enzyme of the heme biosynthetic pathway. It catalyzes the insertion of ferrous iron into the protoporphyrin IX ring yielding protoheme. This enzyme is ubiquitous in nature and widely distributed in bacteria and eukaryotes. Recently, some archaeal members have been identified. The oligomeric state of these enzymes varies depending on the presence of a dimerization motif at the C-terminus.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08978">PRK08978</a></td><td>548</td><td>72</td><td>45.9</td><td>89</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08978</td><td>acetolactate synthase 2 catalytic subunit; Reviewed</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05758">PRK05758</a></td><td>177</td><td>32</td><td>45.3</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK05758</td><td>F0F1 ATP synthase subunit delta; Validated</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00217">pfam00217</a></td><td>243</td><td>61</td><td>40.1</td><td>30</td><td>[&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ATP-gua_Ptrans</td><td>ATP:guanido phosphotransferase, C-terminal catalytic domain. The substrate binding site is located in the cleft between N and C-terminal domains, but most of the catalytic residues are found in the larger C-terminal domain.</td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07451">PRK07451</a></td><td>115</td><td>36</td><td>39.1</td><td>57</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07451</td><td>translation initiation factor Sui1; Validated</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09419">pfam09419</a></td><td>166</td><td>54</td><td>34.7</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PGP_phosphatase</td><td>Mitochondrial PGP phosphatase. This is a family of proteins that acts as a mitochondrial phosphatase in cardiolipin biosynthesis. Cardiolipin is a unique dimeric phosphoglycerolipid predominantly present in mitochondrial membranes. The inverted phosphatase motif includes the highly conserved DKD triad.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR00236">TIGR00236</a></td><td>365</td><td>27</td><td>34.7</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>wecB</td><td>UDP-N-acetylglucosamine 2-epimerase. This cytosolic enzyme converts UDP-N-acetyl-D-glucosamine to UDP-N-acetyl-D-mannosamine. In E. coli, this is the first step in the pathway of enterobacterial common antigen biosynthesis.Members of this orthology group have many gene symbols, often reflecting the overall activity of the pathway and/or operon that includes it. Symbols include epsC (exopolysaccharide C) in Burkholderia solanacerum, cap8P (type 8 capsule P) in Staphylococcus aureus, and nfrC in an older designation based on the effects of deletion on phage N4 adsorption. Epimerase activity was also demonstrated in a bifunctional rat enzyme, for which the N-terminal domain appears to be orthologous. The set of proteins found above the suggested cutoff includes E. coli WecB in one of two deeply branched clusters and the rat UDP-N-acetylglucosamine 2-epimerase domain in the other. </td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06506">pfam06506</a></td><td>169</td><td>71</td><td>33.3</td><td>2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PrpR_N</td><td>Propionate catabolism activator. This domain is found at the N terminus of several sigma54- dependent transcriptional activators including PrpR, which activates catabolism of propionate.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12014">cd12014</a></td><td>62</td><td>23</td><td>32.4</td><td>12</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH3_RIM-BP_1</td><td>First Src homology 3 domain of Rab3-interacting molecules (RIMs) binding proteins. RIMs binding proteins (RBPs, RIM-BPs) associate with calcium channels present in photoreceptors, neurons, and hair cells; they interact simultaneously with specific calcium channel subunits, and active zone proteins, RIM1 and RIM2. RIMs are part of the matrix at the presynaptic active zone and are associated with synaptic vesicles through their interaction with the small GTPase Rab3. RIM-BPs play a role in regulating synaptic transmission by serving as adaptors and linking calcium channels with the synaptic vesicle release machinery. RIM-BPs contain three SH3 domains and two to three fibronectin III repeats. Invertebrates contain one, while vertebrates contain at least two RIM-BPs, RIM-BP1 and RIM-BP2. RIM-BP1 is also called peripheral-type benzodiazapine receptor associated protein 1 (PRAX-1). Mammals contain a third protein, RIM-BP3. RIM-BP1 and RIM-BP2 are predominantly expressed in the brain where they display overlapping but distinct expression patterns, while RIM-BP3 is almost exclusively expressed in the testis and is essential in spermiogenesis. The SH3 domains of RIM-BPs bind to the PxxP motifs of RIM1, RIM2, and L-type (alpha1D) and N-type (alpha1B) calcium channel subunits. SH3 domains are protein interaction domains that bind to proline-rich ligands with moderate affinity and selectivity. They play versatile and diverse roles in the cell including the regulation of enzymes, changing the subcellular localization of signaling pathway components, and mediating the formation of multiprotein complex assemblies.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12323">pfam12323</a></td><td>46</td><td>19</td><td>32.3</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_OrfB_IS605</td><td>Helix-turn-helix domain. This is the N terminal helix-turn-helix domain of Transposase_2 pfam01385.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10925">pfam10925</a></td><td>59</td><td>13</td><td>31.7</td><td>28</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF2680</td><td>Protein of unknown function (DUF2680). Members in this family of proteins are annotated as yckD however currently no function is known.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd07385">cd07385</a></td><td>224</td><td>23</td><td>31.4</td><td>65</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MPP_YkuE_C</td><td>Bacillus subtilis YkuE and related proteins, C-terminal metallophosphatase domain. YkuE is an uncharacterized Bacillus subtilis protein with a C-terminal metallophosphatase domain and an N-terminal twin-arginine (RR) motif. An RR-signal peptide derived from the Bacillus subtilis YkuE protein can direct Tat-dependent secretion of agarase in Streptomyces lividans. This is an indication that YkuE is transported by the Bacillus subtilis Tat (Twin-arginine translocation) pathway machinery.  YkuE belongs to the metallophosphatase (MPP) superfamily.  MPPs are functionally diverse, but all share a conserved domain with an active site consisting of two metal ions (usually manganese, iron, or zinc) coordinated with octahedral geometry by a cage of histidine, aspartate, and asparagine residues. The MPP superfamily includes: Mre11/SbcD-like exonucleases, Dbr1-like RNA lariat debranching enzymes, YfcE-like phosphodiesterases, purple acid phosphatases (PAPs), YbbF-like UDP-2,3-diacylglucosamine hydrolases, and acid sphingomyelinases (ASMases).  The conserved domain is a double beta-sheet sandwich with a di-metal active site made up of residues located at the C-terminal side of the sheets. This domain is thought to allow for productive metal coordination.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08296">cd08296</a></td><td>333</td><td>26</td><td>31.3</td><td>69</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CAD_like</td><td>Cinnamyl alcohol dehydrogenases (CAD). Cinnamyl alcohol dehydrogenases (CAD), members of the medium chain dehydrogenase/reductase family, reduce cinnamaldehydes to cinnamyl alcohols in the last step of monolignal metabolism in plant cells walls. CAD binds 2 zinc ions and is NADPH- dependent. CAD family members are also found in non-plant species, e.g. in yeast where they have an aldehyde reductase activity. The medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, which contains the zinc-dependent alcohol dehydrogenase (ADH-Zn) and related proteins, is a diverse group of proteins related to the first identified member, class I mammalian ADH.  MDRs display a broad range of activities and are distinguished from the smaller short chain dehydrogenases (~ 250 amino acids vs. the ~ 350 amino acids of the MDR).  The MDR proteins have 2 domains: a C-terminal NAD(P) binding-Rossmann fold domain of a beta-alpha form and an N-terminal catalytic domain with distant homology to GroES.  The MDR group contains a host of activities, including the founding alcohol dehydrogenase (ADHs), quinone reductase, sorbitol dehydrogenase, formaldehyde dehydrogenase, butanediol DH, ketose reductase, cinnamyl reductase, and numerous others. The zinc-dependent alcohol dehydrogenases (ADHs) catalyze the  NAD(P)(H)-dependent interconversion of alcohols to aldehydes or ketones.  Active site zinc has a catalytic role, while structural zinc aids in stability.  ADH-like proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and generally have 2 tightly bound zinc atoms per subunit. The active site zinc is coordinated by a histidine, two cysteines, and a water molecule. The second zinc seems to play a structural role, affects subunit interactions, and is typically coordinated by 4 cysteines.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08268">cd08268</a></td><td>328</td><td>61</td><td>31.2</td><td>1.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MDR2</td><td>Medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family. This group is a member of the medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, but lacks the zinc-binding sites of the zinc-dependent alcohol dehydrogenases. The medium chain dehydrogenases/reductase (MDR)/zinc-dependent alcohol dehydrogenase-like family, which contains the zinc-dependent alcohol dehydrogenase (ADH-Zn) and related proteins, is a diverse group of proteins related to the first identified member, class I mammalian ADH.  MDRs display a broad range of activities and are distinguished from the smaller short chain dehydrogenases (~ 250 amino acids vs. the ~ 350 amino acids of the MDR).  The MDR proteins have 2 domains: a C-terminal NAD(P)-binding Rossmann fold domain of a beta-alpha form and an N-terminal catalytic domain with distant homology to GroES.  The MDR group contains a host of activities, including the founding alcohol dehydrogenase (ADH), quinone reductase, sorbitol dehydrogenase, formaldehyde dehydrogenase, butanediol DH, ketose reductase, cinnamyl reductase, and numerous others. The zinc-dependent alcohol dehydrogenases (ADHs) catalyze the  NAD(P)(H)-dependent interconversion of alcohols to aldehydes or ketones.  Active site zinc has a catalytic role, while structural zinc aids in stability.  ADH-like proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and generally have 2 tightly bound zinc atoms per subunit. The active site zinc is coordinated by a histidine, two cysteines, and a water molecule. The second zinc seems to play a structural role, affects subunit interactions, and is typically coordinated by 4 cysteines.</td></tr>
<tr><td>23</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04501">cd04501</a></td><td>183</td><td>48</td><td>30.9</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SGNH_hydrolase_like_4</td><td>Members of the SGNH-hydrolase superfamily, a diverse family of lipases and esterases. The tertiary fold of the enzyme is substantially different from that of the alpha/beta hydrolase family and unique among all known hydrolases; its active site closely resembles the Ser-His-Asp(Glu) triad from other serine hydrolases, but may lack the carboxlic acid.</td></tr>
<tr><td>24</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09346">pfam09346</a></td><td>123</td><td>24</td><td>30.6</td><td>38</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SMI1_KNR4</td><td>SMI1 / KNR4 family (SUKH-1). Proteins in this family are involved in the regulation of 1,3-beta-glucan synthase activity and cell-wall formation. Genome contextual information showed that SMI1 are primary immunity proteins in bacterial toxin systems.</td></tr>
<tr><td>25</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0712">COG0712</a></td><td>178</td><td>34</td><td>30.4</td><td>21</td><td>[&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>AtpH</td><td>FoF1-type ATP synthase, delta subunit </td></tr>
<tr><td>26</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13430">PRK13430</a></td><td>271</td><td>32</td><td>30.4</td><td>25</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13430</td><td>F0F1 ATP synthase subunit delta; Provisional</td></tr>
<tr><td>27</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08449">cd08449</a></td><td>197</td><td>43</td><td>28.7</td><td>85</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP2_XapR</td><td>The C-terminal substrate binding domain of LysR-type transcriptional regulator XapR involved in xanthosine catabolism, contains the type 2 periplasmic binding fold. In Escherichia coli, XapR is a positive regulator for the expression of xapA gene, encoding xanthosine phosphorylase, and xapB gene, encoding a polypeptide similar to the nucleotide transport protein NupG. As an operon, the expression of both xapA and xapB is fully dependent on the presence of both XapR and the inducer xanthosine. Expression of the xapR is constitutive but not auto-regulated, unlike many other LysR family proteins. This substrate-binding domain shows significant homology to the type 2 periplasmic binding proteins (PBP2), which are responsible for the uptake of a variety of substrates such as phosphate, sulfate, polysaccharides, lysine/arginine/ornithine, and histidine. The PBP2 bind their ligand in the cleft between these domains in a manner resembling a Venus flytrap. After binding their specific ligand with high affinity, they can interact with a cognate membrane transport complex comprised of two integral membrane domains and two cytoplasmically located ATPase domains. This interaction triggers the ligand translocation across the cytoplasmic membrane energized by ATP hydrolysis.</td></tr>
<tr><td>28</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd05282">cd05282</a></td><td>323</td><td>59</td><td>26.9</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ETR_like</td><td>2-enoyl thioester reductase-like. 2-enoyl thioester reductase (ETR) catalyzes the NADPH-dependent conversion of trans-2-enoyl acyl carrier protein/coenzyme A (ACP/CoA) to acyl-(ACP/CoA) in fatty acid synthesis. 2-enoyl thioester reductase activity has been linked in Candida tropicalis as essential in maintaining mitiochondrial respiratory function. This ETR family is a part of the medium chain dehydrogenase/reductase family, but lack the zinc coordination sites characteristic of the alcohol dehydrogenases in this family. NAD(P)(H)-dependent oxidoreductases are the major enzymes in the interconversion of alcohols and aldehydes, or ketones.  Alcohol dehydrogenase in the liver converts ethanol and NAD+ to acetaldehyde and NADH, while in yeast and some other microorganisms ADH catalyzes the conversion acetaldehyde to ethanol in alcoholic fermentation.   ADH is a member of the medium chain alcohol dehydrogenase family (MDR), which has a NAD(P)(H)-binding domain in a Rossmann fold of a beta-alpha form.  The NAD(H)-binding region is comprised of 2 structurally similar halves, each of which contacts a mononucleotide.  The N-terminal catalytic domain has a distant homology  to GroES.  These proteins typically form dimers (typically higher plants, mammals) or tetramers (yeast, bacteria), and have 2 tightly bound zinc atoms per subunit, a catalytic zinc at the active site and a structural zinc in a lobe of the catalytic domain.  NAD(H) binding occurs in the cleft between the catalytic and coenzyme-binding domains at the active site, and coenzyme binding induces a conformational closing of this cleft. Coenzyme binding typically precedes and contributes to substrate binding.  Candida tropicalis enoyl thioester reductase (Etr1p) catalyzes the NADPH-dependent reduction of trans-2-enoyl thioesters in mitochondrial fatty acid synthesis. Etr1p forms homodimers with each subunit containing a nucleotide-binding Rossmann fold domain and a catalytic domain.</td></tr>
<tr><td>29</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01565">PRK01565</a></td><td>394</td><td>63</td><td>26.3</td><td>1.7E+02</td><td>[&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01565</td><td>thiamine biosynthesis protein ThiI; Provisional</td></tr>
<tr><td>30</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07085">pfam07085</a></td><td>105</td><td>10</td><td>25.7</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DRTGG</td><td>DRTGG domain. This presumed domain is about 120 amino acids in length. It is found associated with CBS domains pfam00571, as well as the CbiA domain pfam01656. The function of this domain is unknown. It is named the DRTGG domain after some of the most conserved residues. This domain may be very distantly related to a pair of CBS domains. There are no significant sequence similarities, but its length and association with CBS domains supports this idea (Bateman A, pers. obs.).</td></tr>
<tr><td>31</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12518">cd12518</a></td><td>80</td><td>33</td><td>25.0</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM_SRSF11</td><td>RNA recognition motif in serine/arginine-rich splicing factor 11 (SRSF11) and similar proteins. This subgroup corresponds to the RRM of SRSF11, also termed arginine-rich 54 kDa nuclear protein (SRp54 or p54), which belongs to a family of proteins containing regions rich in serine-arginine dipeptides (SR proteins family). It is involved in bridge-complex formation and splicing by mediating protein-protein interactions across either introns or exons. SRSF11 has been identified as a tau exon 10 splicing repressor. It interacts with a purine-rich element in exon 10, and suppresses exon 10 inclusion by antagonizing Tra2beta, an SR-domain-containing protein that enhances exon 10 inclusion. SRSF11 is a unique SR family member and may regulate the alternative splicing in a tissue- and substrate-dependent manner. It can directly interact with the U2 auxiliary factor 65-kDa subunit (U2AF65), a protein associated with the 3' splice site. In addition, unlike the typical SR proteins, SRSF11 associates with other SR proteins but not with the U1 small nuclear ribonucleoprotein U1-70K or the U2 auxiliary factor 35-kDa subunit (U2AF35). SREK1 has unique properties in regulating alternative splicing of different pre-mRNAs; it promotes the use of the distal 5' splice site in E1A pre-mRNA alternative splicing. It also inhibits cryptic splice site selection on the beta-globin pre-mRNA containing competing 5' splice sites. SREK1 contains an RNA recognition motif (RRM), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains), and one serine-arginine (SR)-rich domains (SR domains). </td></tr>
<tr><td>32</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01965">cd01965</a></td><td>428</td><td>66</td><td>25.0</td><td>1.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Nitrogenase_MoFe_beta_like</td><td>Nitrogenase_MoFe_beta_like: Nitrogenase MoFe protein, beta subunit_like. The nitrogenase enzyme catalyzes the ATP-dependent reduction of dinitrogen (N2) to ammonia.  This group contains the beta subunits of component 1 of the three known genetically distinct types of nitrogenase systems: a molybdenum-dependent  nitrogenase (Mo-nitrogenase), a vanadium-dependent nitrogenase (V-nitrogenase), and an iron-only nitrogenase (Fe-nitrogenase). These nitrogenase systems consist of component 1 (MoFe protein, VFe protein or, FeFe protein respectively) and, component 2 (Fe protein). The most widespread and best characterized of these systems is the Mo-nitrogenase. MoFe is an alpha2beta2 tetramer, the alternative nitrogenases are alpha2beta2delta2 hexamers having  alpha and beta subunits similar to the alpha and beta subunits of MoFe. For MoFe, each alphabeta pair contains one P-cluster (at the alphabeta interface) and, one molecule of iron molybdenum cofactor (FeMoco) contained within the alpha subunit. The Fe protein contains, a single </td></tr>
<tr><td>33</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01973">pfam01973</a></td><td>170</td><td>41</td><td>25.0</td><td>96</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MAF_flag10</td><td>Protein of unknown function DUF115. This family of archaebacterial proteins has no known function.</td></tr>
<tr><td>34</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK10401">PRK10401</a></td><td>346</td><td>72</td><td>24.7</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;]</td><td>PRK10401</td><td>DNA-binding transcriptional regulator GalS; Provisional</td></tr>
<tr><td>35</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0028">COG0028</a></td><td>550</td><td>69</td><td>23.7</td><td>1.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>IlvB</td><td>Acetolactate synthase large subunit or other thiamine pyrophosphate-requiring enzyme </td></tr>
<tr><td>36</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK14360">PRK14360</a></td><td>450</td><td>68</td><td>23.2</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>glmU</td><td>bifunctional N-acetylglucosamine-1-phosphate uridyltransferase/glucosamine-1-phosphate acetyltransferase; Provisional</td></tr>
<tr><td>37</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2121">COG2121</a></td><td>214</td><td>76</td><td>22.6</td><td>2E+02</td><td>[&nbsp;&nbsp;---------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG2121</td><td>Uncharacterized conserved protein, lysophospholipid acyltransferase (LPLAT) superfamily  </td></tr>
<tr><td>38</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG2262">COG2262</a></td><td>411</td><td>21</td><td>22.4</td><td>61</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HflX</td><td>50S ribosomal subunit-associated GTPase HflX </td></tr>
<tr><td>39</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG5634">COG5634</a></td><td>223</td><td>12</td><td>21.8</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>YukJ</td><td>Uncharacterized protein YukJ, DUF2278 family </td></tr>
<tr><td>40</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd11804">cd11804</a></td><td>52</td><td>13</td><td>21.4</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>SH3_GRB2_like_N</td><td>N-terminal Src homology 3 domain of Growth factor receptor-bound protein 2 (GRB2) and related proteins. This family includes the adaptor protein GRB2 and related proteins including Drosophila melanogaster Downstream of receptor kinase (DRK), Caenorhabditis elegans Sex muscle abnormal protein 5 (Sem-5), GRB2-related adaptor protein (GRAP), GRAP2, and similar proteins. Family members contain an N-terminal SH3 domain, a central SH2 domain, and a C-terminal SH3 domain. GRB2/Sem-5/DRK is a critical signaling molecule that regulates the Ras pathway by linking tyrosine kinases to the Ras guanine nucleotide releasing protein Sos (son of sevenless), which converts Ras to the active GTP-bound state. GRAP2 plays an important role in T cell receptor (TCR) signaling by promoting the formation of the SLP-76:LAT complex, which couples the TCR to the Ras pathway. GRAP acts as a negative regulator of T cell receptor (TCR)-induced lymphocyte proliferation by downregulating the signaling to the Ras/ERK pathway. The N-terminal SH3 domain of GRB2 binds to Sos and Sos-derived proline-rich peptides. SH3 domains are protein interaction domains that bind to proline-rich ligands with moderate affinity and selectivity, preferentially to PxxP motifs. They play versatile and diverse roles in the cell including the regulation of enzymes, changing the subcellular localization of signaling pathway components, and mediating the formation of multiprotein complex assemblies.</td></tr>
<tr><td>41</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK11172">PRK11172</a></td><td>267</td><td>79</td><td>21.3</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>dkgB</td><td>2,5-diketo-D-gluconate reductase B; Provisional</td></tr>
<tr><td>42</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07282">PRK07282</a></td><td>566</td><td>72</td><td>21.3</td><td>2.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK07282</td><td>acetolactate synthase catalytic subunit; Reviewed</td></tr>
<tr><td>43</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04187">cd04187</a></td><td>181</td><td>41</td><td>21.2</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DPM1_like_bac</td><td>Bacterial DPM1_like enzymes are related to eukaryotic DPM1. A family of  bacterial enzymes related to eukaryotic DPM1; Although the mechanism of eukaryotic enzyme is well studied, the mechanism of the  bacterial enzymes is not well understood. The eukaryotic DPM1 is the catalytic subunit of eukaryotic Dolichol-phosphate mannose (DPM) synthase. DPM synthase is required for synthesis of the glycosylphosphatidylinositol (GPI) anchor, N-glycan precursor, protein O-mannose, and C-mannose. The enzyme has three subunits, DPM1, DPM2 and DPM3. DPM is synthesized from dolichol phosphate and GDP-Man on the cytosolic surface of the ER membrane by DPM synthase and then is flipped onto the luminal side and used as a donor substrate. This protein family belongs to Glycosyltransferase 2 superfamily.</td></tr>
<tr><td>44</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00316">cd00316</a></td><td>399</td><td>71</td><td>21.1</td><td>2.7E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Oxidoreductase_nitrogenase</td><td>The nitrogenase enzyme system catalyzes the ATP-dependent reduction of dinitrogen to ammonia.  This group contains both alpha and beta subunits of component 1 of the three known genetically distinct types of nitrogenase systems: a molybdenum-dependent  nitrogenase (Mo-nitrogenase), a vanadium-dependent nitrogenase (V-nitrogenase), and an iron-only nitrogenase (Fe-nitrogenase) and, both subunits of Protochlorophyllide (Pchlide) reductase and chlorophyllide (chlide) reductase. The nitrogenase systems consist of component 1 (MoFe protein, VFe protein or, FeFe protein respectively) and, component 2 (Fe protein). The most widespread and best characterized nitrogenase is the Mo-nitrogenase. MoFe is an alpha2beta2 tetramer, the alternative nitrogenases are alpha2beta2delta2 hexamers whose alpha and beta subunits are similar to the alpha and beta subunits of MoFe. For MoFe, each alphabeta pair contains one P-cluster (at the alphabeta interface) and, one molecule of iron molybdenum cofactor (FeMoco) contained within the alpha subunit. The Fe protein contains a single </td></tr>
<tr><td>45</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4809">COG4809</a></td><td>466</td><td>76</td><td>20.8</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pfk2</td><td>Archaeal ADP-dependent phosphofructokinase/glucokinase  </td></tr>
<tr><td>46</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01821">cd01821</a></td><td>198</td><td>29</td><td>20.7</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Rhamnogalacturan_acetylesterase_like</td><td>Rhamnogalacturan_acetylesterase_like subgroup of SGNH-hydrolases. Rhamnogalacturan acetylesterase removes acetyl esters from rhamnogalacturonan substrates, and renders them susceptible to degradation by rhamnogalacturonases. Rhamnogalacturonans are highly branched regions in pectic polysaccharides, consisting of repeating -(1,2)-L-Rha-(1,4)-D-GalUA disaccharide units, with many rhamnose residues substituted by neutral oligosaccharides such as arabinans, galactans and arabinogalactans. Extracellular enzymes participating in the degradation of plant cell wall polymers, such as Rhamnogalacturonan acetylesterase, would typically be found in saprophytic and plant pathogenic fungi and bacteria.</td></tr>
</table>
</body>
