<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04899">pfam04899</a></td><td>70</td><td>35</td><td>84.0</td><td>3.7</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MbeD_MobD</td><td>MbeD/MobD like. The MbeD and MobD proteins are plasmid encoded, and are involved in the plasmids mobilisation and transfer in the presence of conjugative plasmids.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12631">cd12631</a></td><td>84</td><td>15</td><td>62.5</td><td>4.4</td><td>[&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM1_CELF1_2_Bruno</td><td>RNA recognition motif 1 in CUGBP Elav-like family member CELF-1, CELF-2, Drosophila melanogaster Bruno protein and similar proteins. This subgroup corresponds to the RRM1 of CELF-1, CELF-2 and Bruno protein. CELF-1 (also termed BRUNOL-2, or CUG-BP1, or EDEN-BP) and CELF-2 (also termed BRUNOL-3, or ETR-3, or CUG-BP2, or NAPOR) belong to the CUGBP1 and ETR-3-like factors (CELF) or BRUNOL (Bruno-like) family of RNA-binding proteins that have been implicated in regulation of pre-mRNA splicing, and control of mRNA translation and deadenylation. CELF-1 is strongly expressed in all adult and fetal tissues tested. The human CELF-1 is a nuclear and cytoplasmic RNA-binding protein that regulates multiple aspects of nuclear and cytoplasmic mRNA processing, with implications for onset of type 1 myotonic dystrophy (DM1), a neuromuscular disease associated with an unstable CUG triplet expansion in the 3'-UTR (3'-untranslated region) of the DMPK (myotonic dystrophy protein kinase) gene; it preferentially targets UGU-rich mRNA elements. It has been shown to bind to a Bruno response element, a cis-element involved in translational control of oskar mRNA in Drosophila, and share sequence similarity to Bruno, the Drosophila protein that mediates this process. The Xenopus homolog embryo deadenylation element-binding protein (EDEN-BP) mediates sequence-specific deadenylation of Eg5 mRNA. It binds specifically to the EDEN motif in the 3'-untranslated regions of maternal mRNAs and targets these mRNAs for deadenylation and translational repression. CELF-1 contain three highly conserved RNA recognition motifs (RRMs), also known as RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains): two consecutive RRMs (RRM1 and RRM2) situated in the N-terminal region followed by a linker region and the third RRM (RRM3) close to the C-terminus of the protein. The two N-terminal RRMs of EDEN-BP are necessary for the interaction with EDEN as well as a part of the linker region (between RRM2 and RRM3). Oligomerization of EDEN-BP is required for specific mRNA deadenylation and binding. CELF-2 is expressed in all tissues at some level, but highest in brain, heart, and thymus. It has been implicated in the regulation of nuclear and cytoplasmic RNA processing events, including alternative splicing, RNA editing, stability and translation. CELF-2 shares high sequence identity with CELF-1, but shows different binding specificity; it binds preferentially to sequences with UG repeats and UGUU motifs. It has been shown to bind to a Bruno response element, a cis-element involved in translational control of oskar mRNA in Drosophila, and share sequence similarity to Bruno, the Drosophila protein that mediates this process. It also binds to the 3'-UTR of cyclooxygenase-2 messages, affecting both translation and mRNA stability, and binds to apoB mRNA, regulating its C to U editing. CELF-2 also contains three highly conserved RRMs. It binds to RNA via the first two RRMs, which are also important for localization in the cytoplasm. The splicing activation or repression activity of CELF-2 on some specific substrates is mediated by RRM1/RRM2. Both, RRM1 and RRM2 of CELF-2, can activate cardiac troponin T (cTNT) exon 5 inclusion. In addition, CELF-2 possesses a typical arginine and lysine-rich nuclear localization signal (NLS) in the C-terminus, within RRM3. This subgroup also includes Drosophila melanogaster Bruno protein, which plays a central role in regulation of Oskar (Osk) expression in flies. It mediates repression by binding to regulatory Bruno response elements (BREs) in the Osk mRNA 3' UTR. The full-length Bruno protein contains three RRMs, two located in the N-terminal half of the protein and the third near the C-terminus, separated by a linker region. </td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK13680">PRK13680</a></td><td>117</td><td>21</td><td>44.5</td><td>6.5</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK13680</td><td>hypothetical protein; Provisional</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04711">pfam04711</a></td><td>76</td><td>22</td><td>39.4</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>ApoA-II</td><td>Apolipoprotein A-II (ApoA-II). Apolipoprotein A-II (ApoA-II) is the second major apolipoprotein of high density lipoprotein in human plasma.  Mature ApoA-II is present as a dimer of two 77-amino acid chains joined by a disulphide bridge. ApoA-II regulates many steps in HDL metabolizm, and its role in coronary heart disease is unclear. In bovine serum, the ApoA-II homologue is present in almost free form.   Bovine ApoA-II shows antimicrobial activity against Escherichia coli and yeasts in phosphate buffered saline (PBS).</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd14395">cd14395</a></td><td>50</td><td>14</td><td>36.4</td><td>15</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>UBA_BIRC4_8</td><td>UBA domain found in E3 ubiquitin-protein ligase XIAP, baculoviral IAP repeat-containing protein 8 (BIRC8) and similar proteins. XIAP, also called baculoviral IAP repeat-containing protein 4 (BIRC4), IAP-like protein (ILP), inhibitor of apoptosis protein 3 (IAP-3), or X-linked inhibitor of apoptosis protein (X-linked IAP), is a potent suppressor of apoptosis that directly inhibits specific members of the caspase family of cysteine proteases, including caspase-3, -7, and -9. It promotes proteasomal degradation of caspase-3 and enhances its anti-apoptotic effect in Fas-induced cell death. The ubiquitin-protein ligase (E3) activity of XIAP also exhibits in the ubiquitination of second mitochondria-derived activator of caspases (Smac). The mitochondrial proteins, Smac/DIABLO and Omi/HtrA2, can inhibit the antiapoptotic activity of XIAP. XIAP has also been implicated in several intracellular signaling cascades involved in the cellular response to stress, such as the c-Jun N-terminal kinase (JNK) pathway, the nuclear factor-kappaB (NF-kappaB) pathway, and the transforming growth factor-beta (TGF-beta) pathway. Moreover, XIAP can regulate copper homeostasis through interacting with MURR1. BIRC8, also called inhibitor of apoptosis-like protein 2 (IAP-like protein 2 or ILP-2), or testis-specific inhibitor of apoptosis, is a tissue-specific homolog of E3 ubiquitin-protein ligase XIAP. It has been implicated in the control of apoptosis in the testis by direct inhibition of caspase 9. Both XIAP and BIRC8 contain three N-terminal baculoviral IAP repeat (BIR) domains, a ubiquitin-association (UBA) domain and a RING domain at the carboxyl terminus.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12361">cd12361</a></td><td>77</td><td>22</td><td>35.3</td><td>23</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM1_2_CELF1-6_like</td><td>RNA recognition motif 1 and 2 in CELF/Bruno-like family of RNA binding proteins and plant flowering time control protein FCA. This subfamily corresponds to the RRM1 and RRM2 domains of the CUGBP1 and ETR-3-like factors (CELF) as well as plant flowering time control protein FCA. CELF, also termed BRUNOL (Bruno-like) proteins, is a family of structurally related RNA-binding proteins involved in regulation of pre-mRNA splicing in the nucleus, and control of mRNA translation and deadenylation in the cytoplasm. The family contains six members: CELF-1 (also known as BRUNOL-2, CUG-BP1, NAPOR, EDEN-BP), CELF-2 (also known as BRUNOL-3, ETR-3, CUG-BP2, NAPOR-2), CELF-3 (also known as BRUNOL-1, TNRC4, ETR-1, CAGH4, ER DA4), CELF-4 (BRUNOL-4), CELF-5 (BRUNOL-5) and CELF-6 (BRUNOL-6). They all contain three highly conserved RNA recognition motifs (RRMs), also known as RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains): two consecutive RRMs (RRM1 and RRM2) situated in the N-terminal region followed by a linker region and the third RRM (RRM3) close to the C-terminus of the protein. The low sequence conservation of the linker region is highly suggestive of a large variety in the co-factors that associate with the various CELF family members. Based on both, sequence similarity and function, the CELF family can be divided into two subfamilies, the first containing CELFs 1 and 2, and the second containing CELFs 3, 4, 5, and 6. The different CELF proteins may act through different sites on at least some substrates. Furthermore, CELF proteins may interact with each other in varying combinations to influence alternative splicing in different contexts. This subfamily also includes plant flowering time control protein FCA that functions in the posttranscriptional regulation of transcripts involved in the flowering process. FCA contains two RRMs, and a WW protein interaction domain.  </td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12297">cd12297</a></td><td>78</td><td>15</td><td>33.5</td><td>26</td><td>[&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM2_Prp24</td><td>RNA recognition motif 2 in fungal pre-messenger RNA splicing protein 24 (Prp24) and similar proteins. This subfamily corresponds to the RRM2 of Prp24, also termed U4/U6 snRNA-associated-splicing factor PRP24 (U4/U6 snRNP), an RNA-binding protein with four well conserved RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains). It facilitates U6 RNA base-pairing with U4 RNA during spliceosome assembly. Prp24 specifically binds free U6 RNA primarily with RRMs 1 and 2 and facilitates pairing of U6 RNA bases with U4 RNA bases. Additionally, it may also be involved in dissociation of the U4/U6 complex during spliceosome activation. </td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam14288">pfam14288</a></td><td>112</td><td>20</td><td>32.9</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>FKS1_dom1</td><td>1,3-beta-glucan synthase subunit FKS1, domain-1. The FKS1_dom1 domain is likely to be the 'Class I' region just N-terminal to the first set of transmembrane helices that is involved in 1,3-beta-glucan synthesis itself. This family is found on proteins with family Glucan_synthase, pfam02364.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam10281">pfam10281</a></td><td>38</td><td>23</td><td>29.4</td><td>50</td><td>[&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ish1</td><td>Putative stress-responsive nuclear envelope protein. This family of proteins found in fungi is a putative stress-responsive nuclear envelope protein Ish1.</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12633">cd12633</a></td><td>80</td><td>15</td><td>28.0</td><td>33</td><td>[&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM1_FCA</td><td>RNA recognition motif 1 in plant flowering time control protein FCA and similar proteins. This subgroup corresponds to the RRM1 of FCA, a gene controlling flowering time in Arabidopsis, encoding a flowering time control protein that functions in the posttranscriptional regulation of transcripts involved in the flowering process. FCA contains two RNA recognition motifs (RRMs), also known as RBDs (RNA binding domains) or RNP (ribonucleoprotein domains), and a WW protein interaction domain. </td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01841">pfam01841</a></td><td>108</td><td>25</td><td>27.8</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Transglut_core</td><td>Transglutaminase-like superfamily. This family includes animal transglutaminases and other bacterial proteins of unknown function. Sequence conservation in this superfamily primarily involves three motifs that centre around conserved cysteine, histidine, and aspartate residues that form the catalytic triad in the structurally characterized transglutaminase, the human blood clotting factor XIIIa'. On the basis of the experimentally demonstrated activity of the Methanobacterium phage pseudomurein endoisopeptidase, it is proposed that many, if not all, microbial homologues of the transglutaminases are proteases and that the eukaryotic transglutaminases have evolved from an ancestral protease.</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04402">cd04402</a></td><td>192</td><td>24</td><td>25.6</td><td>72</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RhoGAP_ARHGAP20</td><td>RhoGAP_ARHGAP20: RhoGAP (GTPase-activator protein </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04608">cd04608</a></td><td>124</td><td>61</td><td>24.2</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CBS_pair_PALP_assoc</td><td>This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domains associated with the pyridoxal-phosphate (PALP) dependent enzyme domain upstream.   The vitamin B6 complex comprises pyridoxine, pyridoxal, and pyridoxamine, as well as the 5'-phosphate esters of pyridoxal (PALP) and pyridoxamine, the last two being the biologically active coenzyme derivatives.  The members of the PALP family are principally involved in the biosynthesis of amino acids and amino acid-derived metabolites, but they are also found in the biosynthetic pathways of amino sugars and other amine-containing compounds.  CBS is a small domain originally identified in cystathionine beta-synthase and subsequently found in a wide range of different proteins. CBS domains usually come in tandem repeats, which associate to form a so-called Bateman domain or a CBS pair which is reflected in this model. The interface between the two CBS domains forms a cleft that is a potential ligand binding site. The CBS pair coexists with a variety of other functional domains.  It has been proposed that the CBS domain may play a regulatory role, although its exact function is unknown.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08917">cd08917</a></td><td>118</td><td>35</td><td>22.0</td><td>69</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TrHb2_O</td><td>Truncated hemoglobins (TrHbs, 2/2Hb, 2/2 globins); group 2 (O). The M- and S families exhibit the canonical secondary structure of hemoglobins, a 3-over-3 alpha-helical sandwich structure (3/3 Mb-fold), built by eight alpha-helical segments. Truncated hemoglobins (TrHbs, 2/2Hb, or 2/2 globins) or the T family globins adopt a 2-on-2 alpha-helical sandwich structure, resulting from extensive and complex modifications of the canonical 3-on-3 alpha-helical sandwich that are distributed throughout the whole protein molecule. TrHbs are classified into three main groups based on their structural properties: TrHb1s (N), TrHb2s (O) and TrHb3s (P). TrHb2s include the dimeric Arabidopsis thaliana TrHb2 AtGLB3. GLB3 is likely to have a function distinct from other plant globins: it exhibits a low O2 affinity, an unusual concentration-independent binding of O2 and CO, and does not respond to any of the treatments that induce plant 3-on-3 globins. Other TrHb2's include Bacillus subtilis trHb (Bs-trHb) which exhibits an extremely high oxygen affinity, and Pseudoalteromonas haloplanktis PhHbO (encoded by the PSHAa0030 gene) which appears to be involved in oxidative and nitrosative stress resistance.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam02944">pfam02944</a></td><td>37</td><td>28</td><td>21.6</td><td>1.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>BESS</td><td>BESS motif. The BESS motif is named after the proteins in which it is found (BEAF, Suvar(3)7 and Stonewall). The motif is 40 amino acid residues long and is composed of two predicted alpha helices. Based on the protein in which it is found and the presence of conserved positively charged residues it is predicted to be a DNA binding domain. This domain appears to be specific to drosophila.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01277">TIGR01277</a></td><td>213</td><td>80</td><td>21.5</td><td>37</td><td>[&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>thiQ</td><td>thiamine ABC transporter, ATP-binding protein. This model describes the energy-transducing ATPase subunit ThiQ of the ThiBPQ thiamine (and thiamine pyrophosphate) ABC transporter in several Proteobacteria. This protein is found so far only in Proteobacteria, and is found in complete genomes only if the ThiB and ThiP subunits are also found. </td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12375">cd12375</a></td><td>77</td><td>15</td><td>21.0</td><td>59</td><td>[&nbsp;&nbsp;-----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RRM1_Hu_like</td><td>RNA recognition motif 1 in the Hu proteins family, Drosophila sex-lethal (SXL), and similar proteins. This subfamily corresponds to the RRM1 of Hu proteins and SXL. The Hu proteins family represents a group of RNA-binding proteins involved in diverse biological processes. Since the Hu proteins share high homology with the Drosophila embryonic lethal abnormal vision (ELAV) protein, the Hu family is sometimes referred to as the ELAV family. Drosophila ELAV is exclusively expressed in neurons and is required for the correct differentiation and survival of neurons in flies. The neuronal members of the Hu family include Hu-antigen B (HuB or ELAV-2 or Hel-N1), Hu-antigen C (HuC or ELAV-3 or PLE21), and Hu-antigen D (HuD or ELAV-4), which play important roles in neuronal differentiation, plasticity and memory. HuB is also expressed in gonads. Hu-antigen R (HuR or ELAV-1 or HuA) is ubiquitously expressed Hu family member. It has a variety of biological functions mostly related to the regulation of cellular response to DNA damage and other types of stress. Hu proteins perform their cytoplasmic and nuclear molecular functions by coordinately regulating functionally related mRNAs. In the cytoplasm, Hu proteins recognize and bind to AU-rich RNA elements (AREs) in the 3' untranslated regions (UTRs) of certain target mRNAs, such as GAP-43, vascular epithelial growth factor (VEGF), the glucose transporter GLUT1, eotaxin and c-fos, and stabilize those ARE-containing mRNAs. They also bind and regulate the translation of some target mRNAs, such as neurofilament M, GLUT1, and p27. In the nucleus, Hu proteins function as regulators of polyadenylation and alternative splicing. Each Hu protein contains three RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains). RRM1 and RRM2 may cooperate in binding to an ARE. RRM3 may help to maintain the stability of the RNA-protein complex, and might also bind to poly(A) tails or be involved in protein-protein interactions. This family also includes the sex-lethal protein (SXL) from Drosophila melanogaster. SXL governs sexual differentiation and X chromosome dosage compensation in flies. It induces female-specific alternative splicing of the transformer (tra) pre-mRNA by binding to the tra uridine-rich polypyrimidine tract at the non-sex-specific 3' splice site during the sex-determination process. SXL binds to its own pre-mRNA and promotes female-specific alternative splicing. It contains an N-terminal Gly/Asn-rich domain that may be responsible for the protein-protein interaction, and tandem RRMs that show high preference to bind single-stranded, uridine-rich target RNA transcripts. </td></tr>
</table>
</body>
