<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01539">cd01539</a></td><td>303</td><td>52</td><td>47.3</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PBP1_GGBP</td><td>Periplasmic glucose/galactose-binding protein (GGBP) involved in chemotaxis towards, and active transport of, glucose and galactose in various bacterial species. Periplasmic glucose/galactose-binding protein (GGBP) involved in chemotaxis towards, and active transport of, glucose and galactose in various bacterial species. GGBP is a member of the pentose/hexose sugar-binding protein family of the type I periplasmic binding protein superfamily which consists of two alpha/beta globular domains connected by a three-stranded hinge. This Venus flytrap-like domain undergoes transition from an open to a closed conformational state upon ligand binding. Moreover, the periplasmic GGBP is homologous to the ligand-binding domain of eukaryotic receptors such as glutamate receptor (GluR) and DNA-binding transcriptional repressors such as LacI and GalR.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd03825">cd03825</a></td><td>365</td><td>83</td><td>47.0</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GT1_wcfI_like</td><td>This family is most closely related to the GT1 family of glycosyltransferases. wcfI in Bacteroides fragilis has been shown to be involved in the capsular polysaccharide biosynthesis.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04424">TIGR04424</a></td><td>286</td><td>80</td><td>45.3</td><td>95</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>metallo_McbB</td><td>McbB family protein. This family includes the partially characterized zinc metalloprotein McbB, part of the maturase system for microcin B17, a thiazole/oxazole-modified microcin (TOMM). Other members of this family belong to very different gene neighborhoods. The Cys residues that act as zinc ligands are conserved in most members, but for rare members are jointly absent.</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam12593">pfam12593</a></td><td>40</td><td>21</td><td>38.9</td><td>11</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>McyA_C</td><td>Microcystin synthetase C terminal. This domain family is found in bacteria, and is approximately 40 amino acids in length. The family is found in association with pfam08242, pfam00501. There is a conserved YAN sequence motif. Microcystins form a large family of small cyclic heptapeptides harbouring extensive modifications in amino acid residue composition and functional group chemistry. These peptide hepatotoxins contain a range of non-proteinogenic amino acids and unusual peptide bonds, and are typically N-methylated. They are synthesized on large enzyme complexes consisting of non-ribosomal peptide synthetases and polyketide synthases. This family is made up of the C terminal of microcystin synthetase, one of the proteins involved in this synthesis pathway.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12095">cd12095</a></td><td>39</td><td>30</td><td>38.5</td><td>31</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TM_ErbB3</td><td>Transmembrane domain of ErbB3, a Protein Tyrosine Kinase. ErbB3 (HER3) is a member of the EGFR (HER, ErbB) subfamily of proteins, which are receptor PTKs (RTKs) containing an extracellular EGF-related ligand-binding region, a transmembrane (TM) helix, and a cytoplasmic region with a tyr kinase domain and a regulatory C-terminal tail. ErbB receptors are activated by ligand-induced dimerization, leading to the phosphorylation of tyr residues in the C-terminal tail, which serve as binding sites for downstream signaling molecules. ErbB3 contains an impaired tyr kinase domain, which lacks crucial residues for catalytic activity against exogenous substrates but is still able to bind ATP and autophosphorylate. ErbB3 binds the neuregulin ligands, NRG1 and NRG2, and it relies on its heterodimerization partners for activity following ligand binding. The ErbB2-ErbB3 heterodimer constitutes a high affinity co-receptor capable of potent mitogenic signaling. The TM domain not only serves as a membrane anchor, but also plays an important role in receptor dimerization and optimal activation. Mutations in the TM domain of ErbB receptors have been associated with increased breast cancer risk. ErbB3 participates in a signaling pathway involved in the proliferation, survival, adhesion, and motility of tumor cells.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11346">pfam11346</a></td><td>42</td><td>25</td><td>31.5</td><td>90</td><td>[&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3149</td><td>Protein of unknown function (DUF3149). This bacterial family of proteins has no known function.</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR03023">TIGR03023</a></td><td>450</td><td>99</td><td>29.8</td><td>3.8E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>WcaJ_sugtrans</td><td>Undecaprenyl-phosphate glucose phosphotransferase. This family of proteins encompasses the E. coli WcaJ protein involved in colanic acid biosynthesis, the Methylobacillus EpsB protein involved in methanolan biosynthesis, as well as the GumD protein involved in the biosynthesis of xanthan. All of these are closely related to the well-characterized WbaP (formerly RfbP) protein, which is the first enzyme in O-antigen biosynthesis in Salmonella typhimurium. The enzyme transfers galactose from UDP-galactose (NOTE: not glucose) to a polyprenyl carrier (utilizing the highly conserved C-terminal sugar transferase domain, pfam02397) a reaction which takes place at the cytoplasmic face of the inner membrane. The N-terminal hydrophobic domain is then believed to facilitate the "flippase" function of transferring the liposaccharide unit from the cytoplasmic face to the periplasmic face of the inner membrane. Most of these genes are found within large operons dedicated to the production of complex exopolysaccharides such as the enterobacterial O-antigen. Colanic acid biosynthesis utilizes a glucose-undecaprenyl carrier, knockout of EpsB abolishes incorporation of UDP-glucose into the lipid phase, and the C-terminal portion of GumD has been shown to be responsible for the glucosyl-1-transferase activity.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd09124">cd09124</a></td><td>126</td><td>39</td><td>27.2</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PLDc_like_TrmB_middle</td><td>Middle phospholipase D-like domain of the transcriptional regulator TrmB and similar proteins. Middle phospholipase D (PLD)-like domain of the transcriptional regulator TrmB and similar proteins. TrmB acts as a bifunctional sugar-sensing transcriptional regulator which controls two operons encoding maltose/trehalose and maltodextrin ABC transporters of Pyrococcus fruiosus. It  functions as a dimer. Full length TrmB includes an N-terminal DNA-binding domain, a C-terminal sugar-binding domain and middle region that has been named as a PLD-like domain. The middle domain displays homology to PLD enzymes, which contain one or two HKD motifs (H-x-K-x(4)-D, where x represents any amino acid residue) per chain. The HKD motif characterizes the PLD superfamily. Due to the lack of key residues related to PLD activity in the PLD-like domain, members of this subfamily are unlikely to carry PLD activity.</td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK01395">PRK01395</a></td><td>104</td><td>47</td><td>27.1</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK01395</td><td>V-type ATP synthase subunit F; Provisional</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd08659">cd08659</a></td><td>365</td><td>44</td><td>26.7</td><td>48</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>M20_ArgE_DapE_like</td><td>Peptidase M20 acetylornithine deacetylase/succinyl-diaminopimelate desuccinylase (ArgE/DapE)-like. Peptidase M20 acetylornithine deacetylase/succinyl-diaminopimelate desuccinylase (ArgE/DapE) like family of enzymes catalyze analogous reactions and share a common activator, the metal ion (usually Co2+ or Zn2+). ArgE catalyzes a broad range of substrates, including N-acetylornithine, alpha-N-acetylmethionine and alpha-N-formylmethionine, while DapE catalyzes the hydrolysis of N-succinyl-L,L-diaminopimelate (L,L-SDAP) to L,L-diaminopimelate and succinate. Proteins in this family are mostly bacterial and have been inferred by homology as being related to both, ArgE and DapE. This family also includes N-acetyl-L-citrulline deacetylase (ACDase; acetylcitrulline deacetylase), a unique, novel enzyme found in Xanthomonas campestris, a plant pathogen, in which N-acetyl-L-ornithine is the substrate for transcarbamoylation reaction, and the product is N-acetyl-L-citrulline. Thus, in the arginine biosynthesis pathway, ACDase subsequently catalyzes the hydrolysis of N-acetyl-L-citrulline to acetate and L-citrulline.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0391">COG0391</a></td><td>323</td><td>38</td><td>24.7</td><td>54</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CofD</td><td>Archaeal 2-phospho-L-lactate transferase/Bacterial gluconeogenesis factor, CofD/UPF0052 family </td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG0347">COG0347</a></td><td>112</td><td>35</td><td>23.9</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>GlnK</td><td>Nitrogen regulatory protein PII </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG1099">COG1099</a></td><td>254</td><td>117</td><td>23.8</td><td>4.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>COG1099</td><td>Predicted metal-dependent hydrolase, TIM-barrel fold  </td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd00242">cd00242</a></td><td>136</td><td>15</td><td>23.8</td><td>44</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Ecotin</td><td>Protease Inhibitor Ecotin; homodimeric protease inhibitor. Protease Inhibitor Ecotin; homodimeric protease inhibitor which binds two chymotrypsin-like serine proteases to form a heterotetramer. Found in bacterial periplasm. Inhibits a broad range of serine proteases including collagenase, trypsin, chymotrypsin, elastase, and factor Xa but not thrombin. Inhibition mechanism involves binding at two different protease contact sites: the primary and secondary binding sites. Primary site loops of ecotin bind to the active site of target proteases in a substrate-like manner with the P1 residue in ecotin mimicking the interactions of a canonical P1 substrate residue.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam03709">pfam03709</a></td><td>111</td><td>54</td><td>23.0</td><td>1.9E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>OKR_DC_1_N</td><td>Orn/Lys/Arg decarboxylase, N-terminal domain. This domain has a flavodoxin-like fold, and is termed the "wing" domain because of its position in the overall 3D structure.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK03600">PRK03600</a></td><td>134</td><td>32</td><td>22.6</td><td>52</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>nrdI</td><td>ribonucleotide reductase stimulatory protein; Reviewed</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR02764">TIGR02764</a></td><td>191</td><td>36</td><td>22.1</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>spore_ybaN_pdaB</td><td>polysaccharide deacetylase family sporulation protein PdaB. This model describes the YbaN protein family, also called PdaB and SpoVIE, of Gram-positive bacteria. Although ybaN null mutants have only a mild sporulation defect, ybaN/ytrI double mutants show drastically reducted sporulation efficiencies. This synthetic defect suggests the role of this sigmaE-controlled gene in sporulation had been masked by functional redundancy. Members of this family are homologous to a characterized polysaccharide deacetylase; the exact function this protein family is unknown. </td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01421">cd01421</a></td><td>187</td><td>32</td><td>21.9</td><td>95</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>IMPCH</td><td>Inosine monophosphate cyclohydrolase domain. This is the N-terminal domain in the purine biosynthesis pathway protein ATIC (purH). The bifunctional ATIC protein contains a C-terminal  ATIC formylase domain that formylates 5-aminoimidazole-4-carboxamide-ribonucleotide. The IMPCH domain then converts the formyl-5-aminoimidazole-4-carboxamide-ribonucleotide to inosine monophosphate. This is the final step in de novo purine production.</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04106">pfam04106</a></td><td>194</td><td>67</td><td>21.8</td><td>16</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>APG5</td><td>Autophagy protein Apg5. Apg5 is directly required for the import of aminopeptidase I via the cytoplasm-to-vacuole targeting pathway.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK05574">PRK05574</a></td><td>340</td><td>34</td><td>21.5</td><td>3.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;----------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>holA</td><td>DNA polymerase III subunit delta; Reviewed</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR04391">TIGR04391</a></td><td>36</td><td>23</td><td>20.8</td><td>93</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>CcmD_alt_fam</td><td>CcmD family protein. Members of this protein family are small (typically less than 50 amino acids in length), with the first half highly hydrophobic like transmembrane alpha helices and containing a nearly invariant tyrosine residue. Members from the Desulfovibrionales appear in the position of ccmD of system I c-type cytochrome biogenesis operons (see pfam04995). This family and pfam04995 appear very similar in sequence properties, but the very low level of actual sequence identify makes it unclear that the similarity reflects homology per se.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam01403">pfam01403</a></td><td>434</td><td>50</td><td>20.7</td><td>1.3E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;]</td><td>Sema</td><td>Sema domain. The Sema domain occurs in semaphorins, which are a large family of secreted and transmembrane proteins, some of which function as repellent signals during axon guidance. Sema domains also occur in the hepatocyte growth factor receptor and human plexin A-3.</td></tr>
</table>
</body>
