<html xmlns="http://www.w3.org/1999/xhtml">
<head> 
    <link rel="stylesheet" type="text/css" href="style.css"/> 
</head>

<body>
<table>
<tr><th>match no.</th><th>target id</th><th>target length</th><th>alignment length</th><th>probability</th><th>E-value</th><th>coverage</th><th>match description</th></tr>
<tr><td>1</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04169">cd04169</a></td><td>268</td><td>38</td><td>74.3</td><td>0.96</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>RF3</td><td>Release Factor 3 (RF3) protein involved in the terminal step of translocation in bacteria. Peptide chain release factor 3 (RF3) is a protein involved in the termination step of translation in bacteria. Termination occurs when class I release factors (RF1 or RF2) recognize the stop codon at the A-site of the ribosome and activate the release of the nascent polypeptide. The class II release factor RF3 then initiates the release of the class I RF from the ribosome. RF3 binds to the RF/ribosome complex in the inactive (GDP-bound) state. GDP/GTP exchange occurs, followed by the release of the class I RF. Subsequent hydrolysis of GTP to GDP triggers the release of RF3 from the ribosome. RF3 also enhances the efficiency of class I RFs at less preferred stop codons and at stop codons in weak contexts.</td></tr>
<tr><td>2</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09278">pfam09278</a></td><td>65</td><td>51</td><td>48.5</td><td>48</td><td>[&nbsp;----------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>MerR-DNA-bind</td><td>MerR, DNA binding. Members of this family of DNA-binding domains are predominantly found in the prokaryotic transcriptional regulator MerR. They adopt a structure consisting of a core of three alpha helices, with an architecture that is similar to that of the 'winged helix' fold.</td></tr>
<tr><td>3</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK00741">PRK00741</a></td><td>526</td><td>38</td><td>47.9</td><td>5.4</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>prfC</td><td>peptide chain release factor 3; Provisional</td></tr>
<tr><td>4</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd12947">cd12947</a></td><td>94</td><td>28</td><td>42.2</td><td>37</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>NOPS_p54nrb</td><td>NOPS domain, including C-terminal coiled-coil region, in 54 kDa nuclear RNA- and DNA-binding protein (p54nrb) and similar proteins. The family contains a DBHS domain (for Drosophila behavior, human splicing), which comprises two conserved RNA recognition motifs (RRMs), also termed RBDs (RNA binding domains) or RNPs (ribonucleoprotein domains), and a charged protein-protein interaction NOPS (NONA and PSP1) domain. This model corresponds to the NOPS domain, with a long helical C-terminal extension, found in p54nrb, also termed non-POU domain-containing octamer-binding protein (NONO), or 55 kDa nuclear protein (NMT55), or DNA-binding p52/p100 complex 52 kDa subunit. It is a multi-functional protein involved in numerous nuclear processes including transcriptional regulation, splicing, DNA unwinding, nuclear retention of hyperedited double-stranded RNA, viral RNA processing, control of cell proliferation, and circadian rhythm maintenance. p54nrb is ubiquitously expressed and highly conserved in vertebrates. It binds both single- and double-stranded RNA and DNA, and also possesses inherent carbonic anhydrase activity. p54nrb forms a heterodimer with paraspeckle component 1 (PSPC1 or PSP1), localizing to paraspeckles in an RNA-dependent manner. It also forms a heterodimer with polypyrimidine tract-binding protein-associated-splicing factor (PSF). The NOPS domain specifically binds to the second RNA recognition motif (RRM2) domain of the partner DBHS protein via a substantial interaction surface. Its highly conserved C-terminal residues are critical for functional DBHS dimerization while the highly conserved C-terminal helical extension, forming a right-handed antiparallel heterodimeric coiled-coil, is essential for paraspeckle localization to subnuclear bodies.</td></tr>
<tr><td>5</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06064">pfam06064</a></td><td>98</td><td>21</td><td>37.2</td><td>29</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Gam</td><td>Host-nuclease inhibitor protein Gam. The Gam protein inhibits RecBCD nuclease and is found in both bacteria and bacteriophage.</td></tr>
<tr><td>6</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK02506">PRK02506</a></td><td>310</td><td>37</td><td>36.7</td><td>25</td><td>[-------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK02506</td><td>dihydroorotate dehydrogenase 1A; Reviewed</td></tr>
<tr><td>7</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13357">cd13357</a></td><td>100</td><td>26</td><td>35.8</td><td>23</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PH-GRAM_MTMR2_insect-like</td><td>Myotubularian related 2 protein (MTMR2) Pleckstrin Homology-Glucosyltransferases, Rab-like GTPase activators and Myotubularins (PH-GRAM) domain. MTMR2 is a member of the myotubularin protein phosphatase gene family. MTMR2 binds to phosphoinositide lipids through its PH-GRAM domain, and can hydrolyze phosphatidylinositol(3)-phosphate and phosphatidylinositol(3,5)-biphosphate in vitro. Mutations in MTMR2 are a cause of Charcot-Marie-Tooth disease type 4B, an autosomal recessive demyelinating neuropathy. The protein can self-associate and form heteromers with MTMR5 and MTMR12. MTMR2 contains a N-terminal PH-GRAM domain, a Rac-induced recruitment domain (RID) domain, an active PTP domain, a SET-interaction domain, a coiled-coil region, and a C-terminal PDZ domain. Myotubularin-related proteins are a subfamily of protein tyrosine phosphatases (PTPs) that dephosphorylate D3-phosphorylated inositol lipids. Mutations in this family cause the human neuromuscular disorders myotubular myopathy and type 4B Charcot-Marie-Tooth syndrome. 6 of the 13 MTMRs (MTMRs 5, 9-13) contain naturally occurring substitutions of residues required for catalysis by PTP family enzymes. Although these proteins are predicted to be enzymatically inactive, they are thought to function as antagonists of endogenous phosphatase activity or interaction modules. Most MTMRs contain a N-terminal PH-GRAM domain, a Rac-induced recruitment domain (RID) domain, a PTP domain (which may be active or inactive), a SET-interaction domain, and a C-terminal coiled-coil region. In addition some members contain DENN domain N-terminal to the PH-GRAM domain and FYVE, PDZ, and PH domains C-terminal to the coiled-coil region. The GRAM domain, found in myotubularins, glucosyltransferases, and other putative membrane-associated proteins, is part of a larger motif with a pleckstrin homology (PH) domain fold. The PH domain family possesses multiple functions including the ability to bind phosphoinositides via its beta1/beta2, beta3/beta4, and beta6/beta7 connecting loops and to other proteins. However, no phosphoinositide binding sites have been found for the MTMRs to date. Members in this cd include Drosophila, sea urchins, mosquitos, bees, ticks, and anemones.</td></tr>
<tr><td>8</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=COG4108">COG4108</a></td><td>528</td><td>38</td><td>35.7</td><td>13</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PrfC</td><td>Peptide chain release factor RF-3 </td></tr>
<tr><td>9</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK07720">PRK07720</a></td><td>146</td><td>39</td><td>31.3</td><td>76</td><td>[&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>fliJ</td><td>flagellar biosynthesis chaperone; Validated</td></tr>
<tr><td>10</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd15478">cd15478</a></td><td>375</td><td>21</td><td>30.3</td><td>49</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Myo5a_CBD</td><td>Cargo binding domain of myosin 5a. Class V myosins are well studied unconventional myosins, represented by three paralogs (Myo5a,b,c) in vertebrates.  Their C-terminal cargo binding domains (CBDs) are important for the binding of a diverse set of cargos, including membrane vesicles, organelles, proteins and mRNA. They interact with several adaptor proteins, in case of Myo5a-CBD, melanophilin (MLPH), Rab interacting lysosomal protein-like 2 (RILPL2), and granuphilin. Mutations in human Myo5a (many of which map to the cargo binding domain) lead to Griscelli syndrome, a severe neurological disease.</td></tr>
<tr><td>11</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK08507">PRK08507</a></td><td>275</td><td>40</td><td>29.9</td><td>71</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK08507</td><td>prephenate dehydrogenase; Validated</td></tr>
<tr><td>12</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=TIGR01618">TIGR01618</a></td><td>220</td><td>18</td><td>29.3</td><td>27</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>phage_P_loop</td><td>phage nucleotide-binding protein. This model represents an uncharacterized family of proteins from a number of phage of Gram-positive bacteria. This protein contains a P-loop motif, G/A-X-X-G-X-G-K-T near its amino end. The function of this protein is unknown. </td></tr>
<tr><td>13</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd04770">cd04770</a></td><td>123</td><td>48</td><td>29.2</td><td>1.5E+02</td><td>[&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_HMRTR</td><td>Helix-Turn-Helix DNA binding domain of Heavy Metal Resistance transcription regulators. Helix-turn-helix (HTH) heavy metal resistance transcription regulators (HMRTR): MerR1 (mercury), CueR (copper),  CadR (cadmium),  PbrR (lead), ZntR (zinc), and other related proteins. These transcription regulators mediate responses to heavy metal stress in eubacteria. They belong to the MerR superfamily of transcription regulators that promote transcription of various stress regulons by reconfiguring the operator sequence located between the -35 and -10 promoter elements. A typical MerR regulator is comprised of two distinct domains that harbor the regulatory (effector-binding) site and the active (DNA-binding) site. Their N-terminal domains are homologous and contain a DNA-binding winged HTH motif, while the C-terminal domains are often dissimilar and bind specific coactivator molecules such as metal ions, drugs, and organic substrates.</td></tr>
<tr><td>14</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam11864">pfam11864</a></td><td>467</td><td>32</td><td>28.8</td><td>41</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;---------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF3384</td><td>Domain of unknown function (DUF3384). This domain is functionally uncharacterized. This domain is found in eukaryotes. This presumed domain is typically between 422 to 486 amino acids in length. This domain is found associated with pfam02145.</td></tr>
<tr><td>15</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam07131">pfam07131</a></td><td>61</td><td>20</td><td>28.7</td><td>46</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>DUF1382</td><td>Protein of unknown function (DUF1382). This family consists of several hypothetical Escherichia coli and bacteriophage lambda-like proteins of around 60 residues in length. The function of this family is unknown. Structural modelling suggests this domain may bind nucleic acids.</td></tr>
<tr><td>16</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd01279">cd01279</a></td><td>98</td><td>33</td><td>26.9</td><td>1.1E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>HTH_HspR-like</td><td>Helix-Turn-Helix DNA binding domain of HspR-like transcription regulators. Helix-turn-helix (HTH) transcription regulator HspR and related proteins, N-terminal domain. Heat shock protein regulators (HspR) have been shown to regulate expression of specific regulons in response to high temperature or high osmolarity in Streptomyces and Helicobacter, respectively. These proteins share the N-terminal DNA binding domain with other transcription regulators of the MerR superfamily that promote transcription by reconfiguring the spacer between the -35 and -10 promoter elements. A typical MerR regulator is comprised of two distinct domains that harbor the regulatory (effector-binding) site and the active (DNA-binding) site. Their conserved N-terminal domains contain predicted winged HTH motifs that mediate DNA binding, while the dissimilar C-terminal domains bind specific coactivator molecules.</td></tr>
<tr><td>17</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam06070">pfam06070</a></td><td>777</td><td>35</td><td>24.6</td><td>75</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-----------------------&nbsp;&nbsp;]</td><td>Herpes_UL32</td><td>Herpesvirus large structural phosphoprotein UL32. The large phosphorylated protein (UL32-like) of herpes viruses is the polypeptide most frequently reactive in immuno-blotting analyses with antisera when compared with other viral proteins.</td></tr>
<tr><td>18</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=PRK15365">PRK15365</a></td><td>107</td><td>39</td><td>24.3</td><td>1.6E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;---------------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>PRK15365</td><td>type III secretion system chaperone SseA; Provisional</td></tr>
<tr><td>19</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09059">pfam09059</a></td><td>81</td><td>27</td><td>23.9</td><td>1.4E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>TyeA</td><td>TyeA. Members of this family are composed of two pairs of parallel alpha-helices, and interact with the bacterial protein YopN via hydrophobic residues located on the helices. Association of TyeA with the C terminus of YopN is accompanied by conformational changes in both polypeptides that create order out of disorder: the resulting structure then serves as an impediment to type III secretion of YopN.</td></tr>
<tr><td>20</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=cd13746">cd13746</a></td><td>115</td><td>12</td><td>22.9</td><td>10</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Sir4p-SID_like</td><td>The SID domain of Saccharomyces cerevisiae silent information regulator 4, a Sir2p interaction domain; and related domains. Saccharomyces cerevisiae Sir2p, Sir3p, and Sir4p form a heterotrimeric complex which binds chromatin and represses transcription at the homothallic mating type (HM) loci and at subtelomeric regions. This domain model spans residues 742-893 of Sir4p. Sir4p forms a stable heterodimer with Sir2p, mediated by Sir4p residues included in this domain, and a pocket between Sir2p's catalytic domain and its non-conserved N-terminus. Sir4p also interacts with an array of additional factors, including Yku80p, a subunit of the telomeric Ku complex (Yku70p-Yku80p), which binds two sites within Sir4p, one at the N-terminus and one in the C-terminal residues, 731-1358. Other interaction factors include Esc1p (Establishes silent chromatin 1) which binds the Sir4p PAD domain (partitioning and anchoring domain, residues 950-1262), and Sir3p, Yku70p, and Rap1p (Repressor Activator Protein) which bind in its C-terminal coiled-coil (residues 1257-1358). Other Sir4p interacting factors include the Ty5 retrotransposon. Additional roles for Sir4p include roles in DNA repair, and in aging. A SIR4 mutant having a truncated Sir4p lacking a C-terminal coiled-coil domain, has an extended mean life span; deletion of the SIR4 gene leads to a decreased mean life span.</td></tr>
<tr><td>21</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam09036">pfam09036</a></td><td>71</td><td>40</td><td>21.6</td><td>1.5E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;--------------------------------&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Bcr-Abl_Oligo</td><td>Bcr-Abl oncoprotein oligomerization domain. The Bcr-Abl oncoprotein oligomerization domain consists of a short N-terminal helix (alpha-1), a flexible loop and a long C-terminal helix (alpha-2). Together these form an N-shaped structure, with the loop allowing the two helices to assume a parallel orientation. The monomeric domains associate into a dimer through the formation of an antiparallel coiled coil between the alpha-2 helices and domain swapping of two alpha-1 helices, where one alpha-1 helix swings back and packs against the alpha-2 helix from the second monomer. Two dimers then associate into a tetramer. The oligomerization domain is essential for the oncogenicity of the Bcr-Abl protein.</td></tr>
<tr><td>22</td><td><a href="http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam04508">pfam04508</a></td><td>23</td><td>20</td><td>21.1</td><td>1.2E+02</td><td>[&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;------------&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]</td><td>Pox_A_type_inc</td><td>Viral A-type inclusion protein repeat. The repeat is found in the A-type inclusion protein of the Poxvirus family.</td></tr>
</table>
</body>
